<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG03242.html">HOG03242&emsp;&emsp;</a><br> 
																	HOG03243&emsp;&emsp;<br> 
																	<a href="HOG03244.html">HOG03244&emsp;&emsp;OSW4/6 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG03243.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG03243_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG03243_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG03243_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG07143<br>EOG8DFN7X<br><br>sce:MET4<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Leucine-zipper transcriptional activator; responsible for regulation of sulfur amino acid pathway; requires different combinations of auxiliary factors Cbf1p, Met28p, Met31p and Met32p; feedforward loop exists in the regulation of genes controlled by Met4p and Met32p; can be ubiquitinated by ubiquitin ligase SCF-Met30p, is either degraded or maintained in an inactive state; regulates degradation of its own DNA-binding cofactors by targeting them to SCF-Met30p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1549123" target=_blank>Thomas D, <i>et al.</i> (1992 Apr)</a>. MET4, a leucine zipper protein, and centromere-binding factor 1 are both required for transcriptional activation of sulfur metabolism in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446029" target=_blank>Mountain HA, <i>et al.</i> (1993 Jan)</a>. The general amino acid control regulates MET4, which encodes a methionine-pathway-specific transcriptional activator of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8524217" target=_blank>Thomas D, <i>et al.</i> (1995 Dec)</a>. Met30p, a yeast transcriptional inhibitor that responds to S-adenosylmethionine, is an essential protein with WD40 repeats.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7799928" target=_blank>Kuras L, <i>et al.</i> (1995 Jan)</a>. Functional analysis of Met4, a yeast transcriptional activator responsive to S-adenosylmethionine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8665859" target=_blank>Kuras L, <i>et al.</i> (1996 May 15)</a>. A heteromeric complex containing the centromere binding factor 1 and two basic leucine zipper factors, Met4 and Met28, mediates the transcription activation of yeast sulfur metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171357" target=_blank>Kuras L, <i>et al.</i> (1997 May 1)</a>. Assembly of a bZIP-bHLH transcription activation complex: formation of the yeast Cbf1-Met4-Met28 complex is regulated through Met28 stimulation of Cbf1 DNA binding.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9799240" target=_blank>Blaiseau PL, <i>et al.</i> (1998 Nov 2)</a>. Multiple transcriptional activation complexes tether the yeast activator Met4 to DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10637232" target=_blank>Rouillon A, <i>et al.</i> (2000 Jan 17)</a>. Feedback-regulated degradation of the transcriptional activator Met4 is triggered by the SCF(Met30 )complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15689486" target=_blank>Yen JL, <i>et al.</i> (2005 Apr)</a>. The yeast ubiquitin ligase SCFMet30 regulates heavy metal response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15883825" target=_blank>Brunson LE, <i>et al.</i> (2005 Jun)</a>. Identification of residues in the WD-40 repeat motif of the F-box protein Met30p required for interaction with its substrate Met4p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07143_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07143_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07143_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG07143_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG03243.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>