<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01155.html">HOG01155&emsp;&emsp;peptidase S10 family</a><br> 
																	HOG01156&emsp;&emsp;<br> 
																	<a href="HOG01157.html">HOG01157&emsp;&emsp;helicase family. SKI2 subfamily</a><br><br> 
																	<a href="../sequences/hog/HOG01156.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG01156_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG01156_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="HOG01156_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02831<br>EOG8X69Q9<br><br>sce:HSP104<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Disaggregase; heat shock protein that cooperates with Ydj1p (Hsp40) and Ssa1p (Hsp70) to refold and reactivate previously denatured, aggregated proteins; responsive to stresses including: heat, ethanol, and sodium arsenite; involved in [PSI+] propagation; protein becomes more abundant and forms cytoplasmic foci in response to DNA replication stress; potentiated Hsp104p variants decrease TDP-43 proteotoxicity by eliminating its cytoplasmic aggregation<br><br>
			<br><b>PomBase Description</b><br>heat shock protein Hsp104 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATPase activity, misfolded protein binding activity, role in cellular response to misfolded protein, protein refolding, protein unfolding and cytosol, nuclear envelope localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2188365" target=_blank>Sanchez Y, <i>et al.</i> (1990 Jun 1)</a>. HSP104 required for induced thermotolerance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1896074" target=_blank>Parsell DA, <i>et al.</i> (1991 Sep 19)</a>. Hsp104 is a highly conserved protein with two essential nucleotide-binding sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1600951" target=_blank>Sanchez Y, <i>et al.</i> (1992 Jun)</a>. Hsp104 is required for tolerance to many forms of stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8407824" target=_blank>Sanchez Y, <i>et al.</i> (1993 Oct)</a>. Genetic evidence for a functional relationship between Hsp104 and Hsp70.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984243" target=_blank>Parsell DA, <i>et al.</i> (1994 Dec 1)</a>. Protein disaggregation mediated by heat-shock protein Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8308017" target=_blank>Parsell DA, <i>et al.</i> (1994 Feb 11)</a>. Saccharomyces cerevisiae Hsp104 protein. Purification and characterization of ATP-induced structural changes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7754373" target=_blank>Chernoff YO, <i>et al.</i> (1995 May 12)</a>. Role of the chaperone protein Hsp104 in propagation of the yeast prion-like factor [psi+].<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8643570" target=_blank>Lindquist S, <i>et al.</i> (1996 May 28)</a>. Heat-shock protein 104 expression is sufficient for thermotolerance in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9534180" target=_blank>Schirmer EC, <i>et al.</i> (1998)</a>. Purification and properties of Hsp104 from yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674429" target=_blank>Glover JR, <i>et al.</i> (1998 Jul 10)</a>. Hsp104, Hsp70, and Hsp40: a novel chaperone system that rescues previously aggregated proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9703962" target=_blank>Fujita K, <i>et al.</i> (1998 Jul 30)</a>. Hsp104 responds to heat and oxidative stress with different intracellular localization in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9624144" target=_blank>Schirmer EC, <i>et al.</i> (1998 Jun 19)</a>. The ATPase activity of Hsp104, effects of environmental conditions and mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10467108" target=_blank>Kawai R, <i>et al.</i> (1999 Mar)</a>. Direct evidence for the intracellular localization of Hsp104 in Saccharomyces cerevisiae by immunoelectron microscopy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073991" target=_blank>Moriyama H, <i>et al.</i> (2000 Dec)</a>. [URE3] prion propagation in Saccharomyces cerevisiae: requirement for chaperone Hsp104 and curing by overexpressed chaperone Ydj1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10678178" target=_blank>Sondheimer N, <i>et al.</i> (2000 Jan)</a>. Rnq1: an epigenetic modifier of protein function in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158570" target=_blank>Schirmer EC, <i>et al.</i> (2001 Jan 30)</a>. Subunit interactions influence the biochemical and biological properties of Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11375656" target=_blank>Jung G, <i>et al.</i> (2001 Jul)</a>. Guanidine hydrochloride inhibits Hsp104 activity in vivo: a possible explanation for its effect in curing yeast prions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11442834" target=_blank>Ferreira PC, <i>et al.</i> (2001 Jun)</a>. The elimination of the yeast [PSI+] prion by guanidine hydrochloride is the result of Hsp104 inactivation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11604493" target=_blank>Abbas-Terki T, <i>et al.</i> (2001 Nov)</a>. Hsp104 interacts with Hsp90 cochaperones in respiring yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11983167" target=_blank>Cashikar AG, <i>et al.</i> (2002 Apr)</a>. Defining a pathway of communication from the C-terminal peptide binding domain to the N-terminal ATPase domain in a AAA protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11967066" target=_blank>Grably MR, <i>et al.</i> (2002 Apr)</a>. HSF and Msn2/4p can exclusively or cooperatively activate the yeast HSP104 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101251" target=_blank>Ness F, <i>et al.</i> (2002 Aug)</a>. Guanidine hydrochloride inhibits the generation of prion "seeds" but not prion protein aggregation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11782421" target=_blank>Hattendorf DA, <i>et al.</i> (2002 Jan 15)</a>. Cooperative kinetics of both Hsp104 ATPase domains and interdomain communication revealed by AAA sensor-1 mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12105276" target=_blank>Jung G, <i>et al.</i> (2002 Jul 23)</a>. Amino acid residue 184 of yeast Hsp104 chaperone is critical for prion-curing by guanidine, prion propagation, and thermotolerance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11867765" target=_blank>Hattendorf DA, <i>et al.</i> (2002 Mar 5)</a>. Analysis of the AAA sensor-2 motif in the C-terminal ATPase domain of Hsp104 with a site-specific fluorescent probe of nucleotide binding.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14507919" target=_blank>Kryndushkin DS, <i>et al.</i> (2003 Dec 5)</a>. Yeast [PSI+] prion aggregates are formed by small Sup35 polymers fragmented by Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14557538" target=_blank>Hitchcock AL, <i>et al.</i> (2003 Oct 28)</a>. A subset of membrane-associated proteins is ubiquitinated in response to mutations in the endoplasmic reticulum degradation machinery.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15049822" target=_blank>Seppä L, <i>et al.</i> (2004 Apr)</a>. Upregulation of the Hsp104 chaperone at physiological temperature during recovery from thermal insult.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14668331" target=_blank>Grimminger V, <i>et al.</i> (2004 Feb 27)</a>. The prion curing agent guanidinium chloride specifically inhibits ATP hydrolysis by Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15128736" target=_blank>Lum R, <i>et al.</i> (2004 Jul 9)</a>. Evidence for an unfolding/threading mechanism for protein disaggregation by Saccharomyces cerevisiae Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15155912" target=_blank>Shorter J, <i>et al.</i> (2004 Jun 18)</a>. Hsp104 catalyzes formation and elimination of self-replicating Sup35 prion conformers.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978213" target=_blank>Schirmer EC, <i>et al.</i> (2004 May)</a>. Dominant gain-of-function mutations in Hsp104p reveal crucial roles for the middle region.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15845535" target=_blank>Cashikar AG, <i>et al.</i> (2005 Jun 24)</a>. A chaperone pathway in protein disaggregation. Hsp26 alters the nature of protein aggregates to facilitate reactivation by Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15843375" target=_blank>Haslbeck M, <i>et al.</i> (2005 Jun 24)</a>. Disassembling protein aggregates in the yeast cytosol. The cooperation of Hsp26 with Ssa1 and Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135516" target=_blank>Bösl B, <i>et al.</i> (2005 Nov 18)</a>. Substrate binding to the molecular chaperone Hsp104 and its regulation by nucleotides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885031" target=_blank>Shorter J, <i>et al.</i> (2006 Aug 4)</a>. Destruction or potentiation of different prions catalyzed by similar Hsp104 remodeling activities.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16570324" target=_blank>Narayanan S, <i>et al.</i> (2006 May)</a>. Yeast prion-protein, sup35, fibril formation proceeds by addition and substraction of oligomers.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160044" target=_blank>Wendler P, <i>et al.</i> (2007 Dec 28)</a>. Atypical AAA+ subunit packing creates an expanded cavity for disaggregation by the protein-remodeling factor Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17259993" target=_blank>Doyle SM, <i>et al.</i> (2007 Feb)</a>. Asymmetric deceleration of ClpB or Hsp104 ATPase activity unleashes protein-remodeling activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17253904" target=_blank>Satpute-Krishnan P, <i>et al.</i> (2007 Feb)</a>. Hsp104-dependent remodeling of prion complexes mediates protein-only inheritance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17543332" target=_blank>Schaupp A, <i>et al.</i> (2007 Jul 20)</a>. Processing of proteins by the molecular chaperone Hsp104.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17367387" target=_blank>Kurahashi H, <i>et al.</i> (2007 Mar)</a>. Channel mutations in Hsp104 hexamer distinctively affect thermotolerance and prion-specific propagation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18312264" target=_blank>Tessarz P, <i>et al.</i> (2008 Apr)</a>. Substrate threading through the central pore of the Hsp104 chaperone as a common mechanism for protein disaggregation and prion propagation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18756251" target=_blank>Kaganovich D, <i>et al.</i> (2008 Aug 28)</a>. Misfolded proteins partition between two distinct quality control compartments.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18197703" target=_blank>Mackay RG, <i>et al.</i> (2008 Feb 19)</a>. The C-terminal extension of Saccharomyces cerevisiae Hsp104 plays a role in oligomer assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17973656" target=_blank>Tkach JM, <i>et al.</i> (2008 Jan)</a>. Nucleocytoplasmic trafficking of the molecular chaperone Hsp104 in unstressed and heat-shocked cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21091509" target=_blank>Colabardini AC, <i>et al.</i> (2010 Dec)</a>. Involvement of the Aspergillus nidulans protein kinase C with farnesol tolerance is related to the unfolded protein response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25865884" target=_blank>Egan MJ, <i>et al.</i> (2015 Apr 14)</a>. Cytoplasmic dynein is required for the spatial organization of protein aggregates in filamentous fungi.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02831_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02831_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02831_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02831_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01156_phobius.html#FOG02831">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02832<br>EOG8X69Q9<br><br>sce:HSP78<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Oligomeric mitochondrial matrix chaperone; cooperates with Ssc1p in mitochondrial thermotolerance after heat shock; able to prevent the aggregation of misfolded proteins as well as resolubilize protein aggregates<br><br>
			<br><b>PomBase Description</b><br>mitochondrial heatshock protein Hsp78 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, nucleoside-triphosphatase activity, nucleotide binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413229" target=_blank>Leonhardt SA, <i>et al.</i> (1993 Oct)</a>. HSP78 encodes a yeast mitochondrial heat shock protein in the Clp family of ATP-dependent proteases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7500331" target=_blank>Moczko M, <i>et al.</i> (1995 Dec 8)</a>. The mitochondrial ClpB homolog Hsp78 cooperates with matrix Hsp70 in maintenance of mitochondrial function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7628444" target=_blank>Schmitt M, <i>et al.</i> (1995 Jul 17)</a>. Hsp78, a Clp homologue within mitochondria, can substitute for chaperone functions of mt-hsp70.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830768" target=_blank>Schmitt M, <i>et al.</i> (1996 Sep)</a>. The molecular chaperone Hsp78 confers compartment-specific thermotolerance to mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11734006" target=_blank>Krzewska J, <i>et al.</i> (2001 Dec 7)</a>. Importance of two ATP-binding sites for oligomerization, ATPase activity and chaperone function of mitochondrial Hsp78 protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11231020" target=_blank>Krzewska J, <i>et al.</i> (2001 Jan 26)</a>. Mitochondrial Hsp78, a member of the Clp/Hsp100 family in Saccharomyces cerevisiae, cooperates with Hsp70 in protein refolding.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12023279" target=_blank>Germaniuk A, <i>et al.</i> (2002 Aug 2)</a>. A bichaperone (Hsp70-Hsp78) system restores mitochondrial DNA synthesis following thermal inactivation of Mip1p polymerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12237310" target=_blank>Rottgers K, <i>et al.</i> (2002 Nov 29)</a>. The ClpB homolog Hsp78 is required for the efficient degradation of proteins in the mitochondrial matrix.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16339719" target=_blank>Seppä L, <i>et al.</i> (2005 Dec)</a>. Regulation and recovery of functions of Saccharomyces cerevisiae chaperone BiP/Kar2p after thermal insult.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16545993" target=_blank>Lewandowska A, <i>et al.</i> (2006 Feb)</a>. Hsp78 chaperone functions in restoration of mitochondrial network following heat stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16460754" target=_blank>von Janowsky B, <i>et al.</i> (2006 Mar 31)</a>. The disaggregation activity of the mitochondrial ClpB homolog Hsp78 maintains Hsp70 function during heat stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23766336" target=_blank>Coradetti ST, <i>et al.</i> (2013 Aug)</a>. Analysis of a conserved cellulase transcriptional regulator reveals inducer-independent production of cellulolytic enzymes in Neurospora crassa.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02832_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02832_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02832_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02832_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01156_phobius.html#FOG02832">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG01156.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>