<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01081.html">HOG01081&emsp;&emsp;major facilitator superfamily. Sugar transporter (TC 2.A.1.1) family</a><br> 
																	HOG01082&emsp;&emsp;<br> 
																	<a href="HOG01083.html">HOG01083&emsp;&emsp;TrkH potassium transport family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01082.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01082_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01082_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG01082_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02630<br>EOG8K0P3B<br><br>sce:SUL1;SUL2<br><br>Genes: 41
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>High affinity sulfate permease of the SulP anion transporter family; sulfate uptake is mediated by specific sulfate transporters Sul1p and Sul2p, which control the concentration of endogenous activated sulfate intermediates|High affinity sulfate permease; sulfate uptake is mediated by specific sulfate transporters Sul1p and Sul2p, which control the concentration of endogenous activated sulfate intermediates<br><br>
			<br><b>PomBase Description</b><br>sulfate transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted secondary active sulfate transmembrane transporter activity, sulfate transmembrane transporter activity and role in sulfate transport, transmembrane transport|Putative sulfate permease<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5857597" target=_blank>Käfer E, <i>et al.</i> (1965 Jul)</a>. Origins of translocations in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5671427" target=_blank>Arst HN Jr, <i>et al.</i> (1968 Jul 20)</a>. Genetic analysis of the first steps of sulphate metabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4934393" target=_blank>Gravel RA, <i>et al.</i> (1970 Dec)</a>. Genetic and accumulation studies in sulfite-requiring mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4558448" target=_blank>Jansen GJ, <i>et al.</i> (1972)</a>. Mutator activity in uvs mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4570678" target=_blank>Lukaszkiewicz Z, <i>et al.</i> (1972)</a>. Mutations increasing the specificity of the sulphate permease in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4581424" target=_blank>Clutterbuck AJ, <i>et al.</i> (1973 Jun)</a>. Gene symbols in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=765839" target=_blank>Lukaszkiewicz Z, <i>et al.</i> (1976 Jan 29)</a>. Hyper-repressible operator-type mutant in sulphate permease gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=395416" target=_blank>Roberts T, <i>et al.</i> (1979)</a>. Allele specific, gene unspecific suppressors in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765129" target=_blank>Paszewski A, <i>et al.</i> (1994)</a>. Sulphur metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7616962" target=_blank>Smith FW, <i>et al.</i> (1995 Jun 25)</a>. Isolation of a cDNA from Saccharomyces cerevisiae that encodes a high affinity sulphate transporter at the plasma membrane.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7677785" target=_blank>Jin YH, <i>et al.</i> (1995 Sep 14)</a>. Characterization of SFP2, a putative sulfate permease gene of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9055073" target=_blank>Cherest H, <i>et al.</i> (1997 Mar)</a>. Molecular characterization of two high affinity sulfate transporters in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572125" target=_blank>van de Kamp M, <i>et al.</i> (1999 Dec)</a>. Sulfate transport in Penicillium chrysogenum: cloning and characterization of the sutA and sutB genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12890030" target=_blank>Natorff R, <i>et al.</i> (2003 Aug)</a>. The Aspergillus nidulans metR gene encodes a bZIP protein which activates transcription of sulphur metabolism genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17223367" target=_blank>Piłsyk S, <i>et al.</i> (2007 Aug)</a>. Sulfate transport in Aspergillus nidulans: a novel gene encoding alternative sulfate transporter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20955810" target=_blank>Brzywczy J, <i>et al.</i> (2011 Feb)</a>. Novel mutations reveal two important regions in Aspergillus nidulans transcriptional activator MetR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26210809" target=_blank>Dzikowska A, <i>et al.</i> (2015 Dec 1)</a>. KAEA (SUDPRO), a member of the ubiquitous KEOPS/EKC protein complex, regulates the arginine catabolic pathway and the expression of several other genes in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02630_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02630_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02630_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02630_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>41 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01082.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>