<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG06660.html">HOG06660&emsp;&emsp;</a><br> 
																	HOG06661&emsp;&emsp;<br> 
																	<a href="HOG06667.html">HOG06667&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG06661.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG06661_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG06661_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG06661_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG06661_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG10874<br>EOG8RJDM4<br><br>sce:MMS2<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin-conjugating enzyme variant; involved in error-free postreplication repair; forms a heteromeric complex with Ubc13p, an active ubiquitin-conjugating enzyme; cooperates with chromatin-associated RING finger proteins, Rad18p and Rad5p; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme Mms2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ubiquitin conjugating enzyme activity, ubiquitin-protein transferase activity and role in free ubiquitin chain polymerization, postreplication repair, protein K63-linked ubiquitination<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9576943" target=_blank>Broomfield S, <i>et al.</i> (1998 May 12)</a>. MMS2, encoding a ubiquitin-conjugating-enzyme-like protein, is a member of the yeast error-free postreplication repair pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9705497" target=_blank>Xiao W, <i>et al.</i> (1998 Sep 1)</a>. The products of the yeast MMS2 and two human homologs (hMMS2 and CROC-1) define a structurally and functionally conserved Ubc-like protein family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10089880" target=_blank>Hofmann RM, <i>et al.</i> (1999 Mar 5)</a>. Noncanonical MMS2-encoded ubiquitin-conjugating enzyme functions in assembly of novel polyubiquitin chains for DNA repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11440714" target=_blank>VanDemark AP, <i>et al.</i> (2001 Jun 15)</a>. Molecular insights into polyubiquitin chain assembly: crystal structure of the Mms2/Ubc13 heterodimer.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12531016" target=_blank>Brown M, <i>et al.</i> (2002 Nov 3)</a>. Structural and functional conservation of error-free DNA postreplication repair in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998526" target=_blank>Goldman GH, <i>et al.</i> (2004 Apr)</a>. Aspergillus nidulans as a model system to characterize the DNA damage response in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16641370" target=_blank>Frampton J, <i>et al.</i> (2006 Jul)</a>. Postreplication repair and PCNA modification in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17515930" target=_blank>Kai M, <i>et al.</i> (2007 Jun)</a>. Rad3-dependent phosphorylation of the checkpoint clamp regulates repair-pathway choice.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20453833" target=_blank>Coulon S, <i>et al.</i> (2010 Jun 16)</a>. Rad8Rad5/Mms2-Ubc13 ubiquitin ligase complex controls translesion synthesis in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20176980" target=_blank>Dolan WP, <i>et al.</i> (2010 May)</a>. Fission yeast Hsk1 (Cdc7) kinase is required after replication initiation for induced mutagenesis and proper response to DNA alkylation damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22064477" target=_blank>Groocock LM, <i>et al.</i> (2012 Jan)</a>. The RecQ4 orthologue Hrq1 is critical for DNA interstrand cross-link repair and genome stability in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25293972" target=_blank>Mastro TL, <i>et al.</i> (2014 Dec)</a>. Increased meiotic crossovers and reduced genome stability in absence of Schizosaccharomyces pombe Rad16 (XPF).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27473316" target=_blank>Ranatunga NS, <i>et al.</i> (2016 Oct 13)</a>. Characterization of a Novel MMS-Sensitive Allele of Schizosaccharomyces pombe mcm4.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG10874_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG10874_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG10874_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG10874_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG06661_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG06661.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>