<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01165.html">HOG01165&emsp;&emsp;WrbA family</a><br> 
																	HOG01166&emsp;&emsp;<br> 
																	<a href="HOG01167.html">HOG01167&emsp;&emsp;snRNP Sm proteins family; snRNP Sm proteins family. SmF/LSm6 subfamily; snRNP SmB/SmN family; snRNP core protein family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01166.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01166_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01166_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01166_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02856<br>EOG8TQJRS<br><br>sce:SNF2<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Catalytic subunit of the SWI/SNF chromatin remodeling complex; involved in transcriptional regulation; contains DNA-stimulated ATPase activity; functions interdependently in transcriptional activation with Snf5p and Snf6p<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent DNA helicase Snf21|ATP-dependent DNA helicase Snf22<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular response to biotic stimulus, cellular response to neutral pH and filamentous growth of a population of unicellular organisms in response to biotic stimulus, more<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1901413" target=_blank>Laurent BC, <i>et al.</i> (1991 Apr 1)</a>. Functional interdependence of the yeast SNF2, SNF5, and SNF6 proteins in transcriptional activation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1886612" target=_blank>Yoshimoto H, <i>et al.</i> (1991 Aug)</a>. The GAM1/SNF2 gene of Saccharomyces cerevisiae encodes a highly charged nuclear protein required for transcription of the STA1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7608126" target=_blank>Kodaki T, <i>et al.</i> (1995 Feb)</a>. The SNF2/SWI2/GAM1/TYE3/RIC1 gene is involved in the coordinate regulation of phospholipid synthesis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871545" target=_blank>Richmond E, <i>et al.</i> (1996 Oct 1)</a>. Functional analysis of the DNA-stimulated ATPase domain of yeast SWI2/SNF2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8896271" target=_blank>Chéret G, <i>et al.</i> (1996 Sep)</a>. DNA sequence analysis of the VPH1-SNF2 region on chromosome XV of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524530" target=_blank>Smith CL, <i>et al.</i> (2003 Feb)</a>. Structural analysis of the yeast SWI/SNF chromatin remodeling complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02856_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02856_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02856_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02856_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01166_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02857<br>EOG8TQJRS<br><br>sce:ISW1;ISW2<br><br>Genes: 56
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATPase subunit of imitation-switch (ISWI) class chromatin remodelers; with Ioc3p forms Isw1a complex involved in repression of transcription initiation; with Ioc2p and Ioc4p forms Isw1b complex involved in regulation of transcription elongation; Isw1b recruited to ORFs by H3K36 methylation and acts with Chd1p to prevent trans-histone exchange over coding regions; Isw1p import into nucleus depends on C-terminal bipartite nuclear targeting signal KRIR X19 KKAK|ATP-dependent DNA translocase involved in chromatin remodeling; ATPase component that, with Itc1p, forms a complex required for repression of a-specific genes, INO1, and early meiotic genes during mitotic growth; the Isw2 complex exhibits basal levels of chromatin binding throughout the genome as well as target-specific chromatin interactions; targeted by Ume6p- and Sua7p-dependent DNA looping to many loci genome-wide<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have DNA translocase activity, chromatin binding, single-stranded DNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10090725" target=_blank>Tsukiyama T, <i>et al.</i> (1999 Mar 15)</a>. Characterization of the imitation switch subfamily of ATP-dependent chromatin-remodeling factors in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11081629" target=_blank>Goldmark JP, <i>et al.</i> (2000 Oct 27)</a>. The Isw2 chromatin remodeling complex represses early meiotic genes upon recruitment by Ume6p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238944" target=_blank>Gelbart ME, <i>et al.</i> (2001 Mar)</a>. Interactions of Isw2 chromatin remodeling complex with nucleosomal arrays: analyses using recombinant yeast histones and immobilized templates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238381" target=_blank>Kent NA, <i>et al.</i> (2001 Mar 1)</a>. In vivo chromatin remodeling by yeast ISWI homologs Isw1p and Isw2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11533234" target=_blank>Fazzio TG, <i>et al.</i> (2001 Oct)</a>. Widespread collaboration of Isw2 and Sin3-Rpd3 chromatin remodeling complexes in transcriptional repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489850" target=_blank>Sugiyama M, <i>et al.</i> (2001 Sep)</a>. The Saccharomyces cerevisiae Isw2p-Itc1p complex represses INO1 expression and maintains cell morphology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14663094" target=_blank>Nobile CJ, <i>et al.</i> (2003 Dec)</a>. Genetic control of chlamydospore formation in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12482963" target=_blank>Vary JC Jr, <i>et al.</i> (2003 Jan)</a>. Yeast Isw1p forms two separable complexes in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14622597" target=_blank>Morillon A, <i>et al.</i> (2003 Nov 14)</a>. Isw1 chromatin remodeling ATPase coordinates transcription elongation and termination by RNA polymerase II.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15024052" target=_blank>McConnell AD, <i>et al.</i> (2004 Apr)</a>. Histone fold protein Dls1p is required for Isw2-dependent chromatin remodeling in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14673157" target=_blank>Iida T, <i>et al.</i> (2004 Jan)</a>. Noncompetitive counteractions of DNA polymerase epsilon and ISW2/yCHRAC for epigenetic inheritance of telomere position effect in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16339080" target=_blank>Enjalbert B, <i>et al.</i> (2006 Feb)</a>. Role of the Hog1 stress-activated protein kinase in the global transcriptional response to stress in the fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21592964" target=_blank>Singh RP, <i>et al.</i> (2011 Jul 15)</a>. Cap2-HAP complex is a critical transcriptional regulator that has dual but contrasting roles in regulation of iron homeostasis in Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02857_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02857_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02857_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02857_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01166_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02858<br>EOG8TQJRS<br><br>sce:CHD1<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Chromatin remodeler that regulates various aspects of transcription; acts in in conjunction with Isw1b to regulate chromatin structure and maintain chromatin integrity during transcription elongation by RNAP II by preventing trans-histone exchange over coding regions; contains a chromo domain, a helicase domain and a DNA-binding domain; component of both the SAGA and SLIK complexes<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent DNA helicase Hrp1|ATP-dependent DNA helicase Hrp3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent DNA helicase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10026213" target=_blank>Grant PA, <i>et al.</i> (1999 Feb 26)</a>. Expanded lysine acetylation specificity of Gcn5 in native complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10811623" target=_blank>Tran HG, <i>et al.</i> (2000 May 15)</a>. The chromo domain protein chd1p from budding yeast is an ATP-dependent chromatin-modifying factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446794" target=_blank>Pray-Grant MG, <i>et al.</i> (2002 Dec)</a>. The novel SLIK histone acetyltransferase complex functions in the yeast retrograde response pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12242279" target=_blank>Krogan NJ, <i>et al.</i> (2002 Oct)</a>. RNA polymerase II elongation factors of Saccharomyces cerevisiae: a targeted proteomics approach.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12682017" target=_blank>Simic R, <i>et al.</i> (2003 Apr 15)</a>. Chromatin remodeling protein Chd1 interacts with transcription elongation factors and localizes to transcribed genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12872131" target=_blank>Peng J, <i>et al.</i> (2003 Aug)</a>. A proteomics approach to understanding protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14585955" target=_blank>Robinson KM, <i>et al.</i> (2003 Nov)</a>. Replication-independent assembly of nucleosome arrays in a novel yeast chromatin reconstitution system involves antisilencing factor Asf1p and chromodomain protein Chd1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15260971" target=_blank>Wu PY, <i>et al.</i> (2004 Jul 23)</a>. Molecular architecture of the S. cerevisiae SAGA complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15647753" target=_blank>Pray-Grant MG, <i>et al.</i> (2005 Jan 27)</a>. Chd1 chromodomain links histone H3 methylation with SAGA- and SLIK-dependent acetylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16606615" target=_blank>Stockdale C, <i>et al.</i> (2006 Jun 16)</a>. Analysis of nucleosome repositioning by yeast ISWI and Chd1 chromatin remodeling complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16468993" target=_blank>Xella B, <i>et al.</i> (2006 Mar)</a>. The ISWI and CHD1 chromatin remodelling activities influence ADH2 expression and chromatin organization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17259992" target=_blank>Jones HS, <i>et al.</i> (2007 Feb)</a>. RNA polymerase I in yeast transcribes dynamic nucleosomal rDNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17098252" target=_blank>Okuda M, <i>et al.</i> (2007 Jan 26)</a>. Structural polymorphism of chromodomains in Chd1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17433364" target=_blank>Flanagan JF, <i>et al.</i> (2007 Jun 1)</a>. Molecular implications of evolutionary differences in CHD double chromodomains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17949749" target=_blank>Ferreira H, <i>et al.</i> (2007 Nov 30)</a>. Histone modifications influence the action of Snf2 family remodelling enzymes by different mechanisms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17620414" target=_blank>Biswas D, <i>et al.</i> (2007 Sep)</a>. Chd1 and yFACT act in opposition in regulating transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576814" target=_blank>Warner MH, <i>et al.</i> (2007 Sep)</a>. Rtf1 is a multifunctional component of the Paf1 complex that regulates gene expression by directing cotranscriptional histone modification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245327" target=_blank>Biswas D, <i>et al.</i> (2008 Feb)</a>. A role for Chd1 and Set2 in negatively regulating DNA replication in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02858_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02858_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02858_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02858_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01166_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02859<br>EOG8TQJRS<br><br>sce:IRC5<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative ATPase containing the DEAD/H helicase-related sequence motif; null mutant displays increased levels of spontaneous Rad52p foci<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in mitotic recombination<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12844361" target=_blank>Brachat S, <i>et al.</i> (2003)</a>. Reinvestigation of the Saccharomyces cerevisiae genome annotation by comparison to the genome of a related fungus: Ashbya gossypii.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18085829" target=_blank>Alvaro D, <i>et al.</i> (2007 Dec)</a>. Genome-wide analysis of Rad52 foci reveals diverse mechanisms impacting recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02859_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02859_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02859_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02859_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01166_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG02860<br>EOG8TQJRS<br><br>sce:INO80<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATPase and nucleosome spacing factor; subunit of complex containing actin and actin-related proteins that has chromatin remodeling activity and 3' to 5' DNA helicase activity in vitro; promotes nucleosome shifts in the 3 prime direction; has a role in modulating stress gene transcription<br><br>
			<br><b>PomBase Description</b><br>SNF2 family ATP-dependent 3' to 5' DNA helicase Ino80<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in chromatin remodeling and Ino80 complex localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10077188" target=_blank>Shiratori A, <i>et al.</i> (1999 Feb)</a>. Systematic identification, classification, and characterization of the open reading frames which encode novel helicase-related proteins in Saccharomyces cerevisiae by gene disruption and Northern analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10361278" target=_blank>Ebbert R, <i>et al.</i> (1999 May)</a>. The product of the SNF2/SWI2 paralogue INO80 of Saccharomyces cerevisiae required for efficient expression of various yeast structural genes is part of a high-molecular-weight protein complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10952318" target=_blank>Shen X, <i>et al.</i> (2000 Aug 3)</a>. A chromatin remodelling complex involved in transcription and DNA processing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12434012" target=_blank>Steger DJ, <i>et al.</i> (2003 Jan 3)</a>. Regulation of chromatin remodeling by inositol polyphosphates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12887900" target=_blank>Shen X, <i>et al.</i> (2003 Jul)</a>. Involvement of actin-related proteins in ATP-dependent chromatin remodeling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15607974" target=_blank>Morrison AJ, <i>et al.</i> (2004 Dec 17)</a>. INO80 and gamma-H2AX interaction links ATP-dependent chromatin remodeling to DNA damage repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15607975" target=_blank>van Attikum H, <i>et al.</i> (2004 Dec 17)</a>. Recruitment of the INO80 complex by H2A phosphorylation links ATP-dependent chromatin remodeling with DNA double-strand break repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19040720" target=_blank>Shevchenko A, <i>et al.</i> (2008)</a>. Chromatin Central: towards the comparative proteome by accurate mapping of the yeast proteomic environment.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19933844" target=_blank>Hogan CJ, <i>et al.</i> (2010 Feb)</a>. Fission yeast Iec1-ino80-mediated nucleosome eviction regulates nucleotide and phosphate metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22144913" target=_blank>Duncan CD, <i>et al.</i> (2011 Dec)</a>. Widespread cotranslational formation of protein complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26167880" target=_blank>Lipp JJ, <i>et al.</i> (2015 Aug)</a>. SR protein kinases promote splicing of nonconsensus introns.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02860_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02860_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02860_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02860_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01166_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG02861<br>EOG8TQJRS<br><br>sce:SWR1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Swi2/Snf2-related ATPase; structural component of the SWR1 complex, which exchanges histone variant H2AZ (Htz1p) for chromatin-bound histone H2A; relocalizes to the cytosol in response to hypoxia; chronological aging factor that mediates lifespan extension by dietary restriction<br><br>
			<br><b>PomBase Description</b><br>SNF2 family ATP-dependent DNA helicase Swr1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have structural molecule activity, role in histone exchange, regulation of phenotypic switching by regulation of transcription from RNA polymerase II promoter and Swr1 complex, cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9571240" target=_blank>Kitamura K, <i>et al.</i> (1998 May)</a>. Fission yeast Ste9, a homolog of Hct1/Cdh1 and Fizzy-related, is a novel negative regulator of cell cycle progression during G1-phase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10921878" target=_blank>Yamaguchi S, <i>et al.</i> (2000 Aug 1)</a>. Fission yeast Fizzy-related protein srw1p is a G(1)-specific promoter of mitotic cyclin B degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14690608" target=_blank>Krogan NJ, <i>et al.</i> (2003 Dec)</a>. A Snf2 family ATPase complex required for recruitment of the histone H2A variant Htz1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645854" target=_blank>Mizuguchi G, <i>et al.</i> (2004 Jan 16)</a>. ATP-driven exchange of histone H2AZ variant catalyzed by SWR1 chromatin remodeling complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15045029" target=_blank>Kobor MS, <i>et al.</i> (2004 May)</a>. A protein complex containing the conserved Swi2/Snf2-related ATPase Swr1p deposits histone variant H2A.Z into euchromatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15353583" target=_blank>Krogan NJ, <i>et al.</i> (2004 Sep 14)</a>. Regulation of chromosome stability by the histone H2A variant Htz1, the Swr1 chromatin remodeling complex, and the histone acetyltransferase NuA4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19040720" target=_blank>Shevchenko A, <i>et al.</i> (2008)</a>. Chromatin Central: towards the comparative proteome by accurate mapping of the yeast proteomic environment.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19160458" target=_blank>Anders A, <i>et al.</i> (2008 Dec)</a>. Improved tools for efficient mapping of fission yeast genes: identification of microtubule nucleation modifier mod22-1 as an allele of chromatin- remodelling factor gene swr1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18931302" target=_blank>Dixon SJ, <i>et al.</i> (2008 Oct 28)</a>. Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19911051" target=_blank>Buchanan L, <i>et al.</i> (2009 Nov)</a>. The Schizosaccharomyces pombe JmjC-protein, Msc1, prevents H2A.Z localization in centromeric and subtelomeric chromatin domains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19910462" target=_blank>Hou H, <i>et al.</i> (2010 Jan 15)</a>. Histone variant H2A.Z regulates centromere silencing and chromosome segregation in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20858896" target=_blank>Qiu X, <i>et al.</i> (2010 Nov 19)</a>. Activity of a C-terminal plant homeodomain (PHD) of Msc1 is essential for function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21901086" target=_blank>Sadeghi L, <i>et al.</i> (2011 Aug)</a>. Podbat: a novel genomic tool reveals Swr1-independent H2A.Z incorporation at gene coding sequences through epigenetic meta-analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25210736" target=_blank>Abenza JF, <i>et al.</i> (2014)</a>. Dynamics of cell shape inheritance in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24687850" target=_blank>Tapia-Alveal C, <i>et al.</i> (2014 Jun)</a>. H2A.Z-dependent regulation of cohesin dynamics on chromosome arms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojardín L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26510788" target=_blank>George AA, <i>et al.</i> (2015 Dec)</a>. Escape from Mitotic Arrest: An Unexpected Connection Between Microtubule Dynamics and Epigenetic Regulation of Centromeric Chromatin in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02861_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02861_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02861_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02861_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01166_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG02862<br>EOG8TQJRS<br><br>sce:STH1<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATPase component of the RSC chromatin remodeling complex; required for expression of early meiotic genes; promotes base excision repair in chromatin; essential helicase-related protein homologous to Snf2p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1549132" target=_blank>Laurent BC, <i>et al.</i> (1992 Apr)</a>. An essential Saccharomyces cerevisiae gene homologous to SNF2 encodes a helicase-related protein in a new family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1396591" target=_blank>Tsuchiya E, <i>et al.</i> (1992 Nov)</a>. The Saccharomyces cerevisiae NPS1 gene, a novel CDC gene which encodes a 160 kDa nuclear protein involved in G2 phase control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8980231" target=_blank>Cairns BR, <i>et al.</i> (1996 Dec 27)</a>. RSC, an essential, abundant chromatin-remodeling complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9121424" target=_blank>Treich I, <i>et al.</i> (1997 Apr)</a>. Interaction of a Swi3 homolog with Sth1 provides evidence for a Swi/Snf-related complex with an essential function in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9154831" target=_blank>Cao Y, <i>et al.</i> (1997 Jun)</a>. Sfh1p, a component of a novel chromatin-remodeling complex, is required for cell cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9799253" target=_blank>Du J, <i>et al.</i> (1998 Nov)</a>. Sth1p, a Saccharomyces cerevisiae Snf2p/Swi2p homolog, is an essential ATPase in RSC and differs from Snf/Swi in its interactions with histones and chromatin-associated proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10320476" target=_blank>Yukawa M, <i>et al.</i> (1999 Feb)</a>. Nps1/Sth1p, a component of an essential chromatin-remodeling complex of Saccharomyces cerevisiae, is required for the maximal expression of early meiotic genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10025404" target=_blank>Lorch Y, <i>et al.</i> (1999 Feb 5)</a>. Histone octamer transfer by a chromatin-remodeling complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10329629" target=_blank>Moreira JM, <i>et al.</i> (1999 May 17)</a>. Transcriptional repression of the yeast CHA1 gene requires the chromatin-remodeling complex RSC.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10619019" target=_blank>Cairns BR, <i>et al.</i> (1999 Nov)</a>. Two functionally distinct forms of the RSC nucleosome-remodeling complex, containing essential AT hook, BAH, and bromodomains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12183366" target=_blank>Saha A, <i>et al.</i> (2002 Aug 15)</a>. Chromatin remodeling by RSC involves ATP-dependent DNA translocation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12072455" target=_blank>Chai B, <i>et al.</i> (2002 Jun)</a>. Yeast RSC function is required for organization of the cellular cytoskeleton via an alternative PKC1 pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12697820" target=_blank>Hsu JM, <i>et al.</i> (2003 May)</a>. The yeast RSC chromatin-remodeling complex is required for kinetochore function in chromosome segregation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15506919" target=_blank>Lee KK, <i>et al.</i> (2004 Dec)</a>. Proteomic analysis of chromatin-modifying complexes in Saccharomyces cerevisiae identifies novel subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16273108" target=_blank>Deutschbauer AM, <i>et al.</i> (2005 Dec)</a>. Quantitative trait loci mapped to single-nucleotide resolution in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204215" target=_blank>Wilson B, <i>et al.</i> (2006 Feb)</a>. The RSC chromatin remodeling complex bears an essential fungal-specific protein module with broad functional roles.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02862_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02862_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02862_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02862_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01166_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01166.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>