<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01167.html">HOG01167&emsp;&emsp;snRNP Sm proteins family; snRNP Sm proteins family. SmF/LSm6 subfamily; snRNP SmB/SmN family; snRNP core protein family</a><br> 
																	HOG01168&emsp;&emsp;<br> 
																	<a href="HOG01169.html">HOG01169&emsp;&emsp;PPP phosphatase family. PP-5 (PP-T) subfamily; SGT family</a><br><br> 
																	<a href="../sequences/hog/HOG01168.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG01168_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG01168_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="HOG01168_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02882<br>EOG8QBZPJ<br><br>sce:NAM8;NGR1<br><br>Genes: 54
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>RNA binding protein, component of the U1 snRNP protein; mutants are defective in meiotic recombination and in formation of viable spores, involved in the formation of DSBs through meiosis-specific splicing of REC107 pre-mRNA; Nam8p regulon embraces the meiotic pre-mRNAs of REC107, HFM1, AMA1 SPO22 and PCH2; the putative RNA binding domains RRM2 and RRM3 are required for Nam8p meiotic function|RNA binding protein that negatively regulates growth rate; interacts with the 3' UTR of the mitochondrial porin (POR1) mRNA and enhances its degradation; overexpression impairs mitochondrial function; interacts with Dhh1p to mediate POR1 mRNA decay; expressed in stationary phase<br><br>
			<br><b>PomBase Description</b><br>mRNA processing factor<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have mRNA binding activity, role in arginine catabolic process, positive regulation of cellular response to oxidative stress, positive regulation of nuclear-transcribed mRNA poly(A) tail shortening and cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1603056" target=_blank>Ekwall K, <i>et al.</i> (1992 May)</a>. The NAM8 gene in Saccharomyces cerevisiae encodes a protein with putative RNA binding motifs and acts as a suppressor of mitochondrial splicing deficiencies when overexpressed.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8325883" target=_blank>Lee FJ, <i>et al.</i> (1993 Jul 15)</a>. An RNA-binding protein gene (RBP1) of Saccharomyces cerevisiae encodes a putative glucose-repressible protein containing two RNA recognition motifs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15470237" target=_blank>Kupfer DM, <i>et al.</i> (2004 Oct)</a>. Introns and splicing elements of five diverse fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17719249" target=_blank>Olszewska A, <i>et al.</i> (2007 Dec)</a>. Arginine catabolism in Aspergillus nidulans is regulated by the rrmA gene coding for the RNA-binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23841692" target=_blank>Krol K, <i>et al.</i> (2013 Sep)</a>. RrmA regulates the stability of specific transcripts in response to both nitrogen source and oxidative stress.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02882_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02882_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02882_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02882_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02882">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02883<br>EOG8QBZPJ<br><br>sce:PAB1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Poly(A) binding protein; part of the 3'-end RNA-processing complex, mediates interactions between the 5' cap structure and the 3' mRNA poly(A) tail, involved in control of poly(A) tail length, interacts with translation factor eIF-4G; stimulates, but is not required for the deadenylation activity of the Pan2p-Pan3p poly(A)-ribonuclease complex<br><br>
			<br><b>PomBase Description</b><br>mRNA export shuttling protein<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have mRNA binding, poly(A) binding, ribonuclease inhibitor activity and role in regulation of nuclear-transcribed mRNA poly(A) tail shortening, regulation of translational initiation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537727" target=_blank>Adam SA, <i>et al.</i> (1986 Aug)</a>. mRNA polyadenylate-binding protein: gene isolation and sequencing and identification of a ribonucleoprotein consensus sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3518950" target=_blank>Sachs AB, <i>et al.</i> (1986 Jun 20)</a>. A single gene from yeast for both nuclear and cytoplasmic polyadenylate-binding proteins: domain structure and expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3313012" target=_blank>Sachs AB, <i>et al.</i> (1987 Sep)</a>. A single domain of yeast poly(A)-binding protein is necessary and sufficient for RNA binding and cell viability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2565532" target=_blank>Bernstein P, <i>et al.</i> (1989 Feb)</a>. The poly(A)-poly(A)-binding protein complex is a major determinant of mRNA stability in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2673535" target=_blank>Sachs AB, <i>et al.</i> (1989 Sep 8)</a>. The poly(A) binding protein is required for poly(A) shortening and 60S ribosomal subunit-dependent translation initiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1675426" target=_blank>Burd CG, <i>et al.</i> (1991 Jul)</a>. The multiple RNA-binding domains of the mRNA poly(A)-binding protein have different RNA-binding activities.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413212" target=_blank>Anderson JT, <i>et al.</i> (1993 Oct)</a>. PUB1 is a major nuclear and cytoplasmic polyadenylated RNA-binding protein in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8127652" target=_blank>Cockell M, <i>et al.</i> (1994 Jan 11)</a>. The yeast protein encoded by PUB1 binds T-rich single stranded DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557393" target=_blank>Caponigro G, <i>et al.</i> (1995 Oct 1)</a>. Multiple functions for the poly(A)-binding protein in mRNA decapping and deadenylation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8978035" target=_blank>Marhoul JF, <i>et al.</i> (1996 Dec)</a>. Aspergillus fabM encodes an essential product that is related to poly(A)-binding proteins and activates development when overexpressed.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003792" target=_blank>Tarun SZ Jr, <i>et al.</i> (1996 Dec 16)</a>. Association of the yeast poly(A) tail binding protein with translation initiation factor eIF-4G.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9298649" target=_blank>Garrels JI, <i>et al.</i> (1997 Aug)</a>. Proteome studies of Saccharomyces cerevisiae: identification and characterization of abundant proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256432" target=_blank>Tarun SZ Jr, <i>et al.</i> (1997 Aug 19)</a>. Translation initiation factor eIF4G mediates in vitro poly(A) tail-dependent translation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199303" target=_blank>Amrani N, <i>et al.</i> (1997 Jul)</a>. Yeast Pab1 interacts with Rna15 and participates in the control of the poly(A) tail length in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9223284" target=_blank>Minvielle-Sebastia L, <i>et al.</i> (1997 Jul 22)</a>. The major yeast poly(A)-binding protein is associated with cleavage factor IA and functions in premessenger RNA 3'-end formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9193001" target=_blank>Deardorff JA, <i>et al.</i> (1997 May 30)</a>. Differential effects of aromatic and charged residue substitutions in the RNA binding domains of the yeast poly(A)-binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819425" target=_blank>Mangus DA, <i>et al.</i> (1998 Dec)</a>. Pbp1p, a factor interacting with Saccharomyces cerevisiae poly(A)-binding protein, regulates polyadenylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9418852" target=_blank>Kessler SH, <i>et al.</i> (1998 Jan)</a>. RNA recognition motif 2 of yeast Pab1p is required for its functional interaction with eukaryotic translation initiation factor 4G.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9702200" target=_blank>Wells SE, <i>et al.</i> (1998 Jul)</a>. Circularization of mRNA by eukaryotic translation initiation factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9784497" target=_blank>Coller JM, <i>et al.</i> (1998 Oct 15)</a>. mRNA stabilization by poly(A) binding protein is independent of poly(A) and requires translation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10357826" target=_blank>Otero LJ, <i>et al.</i> (1999 Jun 1)</a>. The yeast poly(A)-binding protein Pab1p stimulates in vitro poly(A)-dependent and cap-dependent translation by distinct mechanisms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10944120" target=_blank>Vilela C, <i>et al.</i> (2000 Aug 15)</a>. The eukaryotic mRNA decapping protein Dcp1 interacts physically and functionally with the eIF4F translation initiation complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279042" target=_blank>Horton LE, <i>et al.</i> (2001 Apr 27)</a>. The yeast hsp70 homologue Ssa is required for translation and interacts with Sis1 and Pab1 on translating ribosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741542" target=_blank>Tharun S, <i>et al.</i> (2001 Nov)</a>. Targeting an mRNA for decapping: displacement of translation factors and association of the Lsm1p-7p complex on deadenylated yeast mRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11940585" target=_blank>Kozlov G, <i>et al.</i> (2002 Jun 21)</a>. Solution structure of the orphan PABC domain from Saccharomyces cerevisiae poly(A)-binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11889048" target=_blank>Tucker M, <i>et al.</i> (2002 Mar 15)</a>. Ccr4p is the catalytic subunit of a Ccr4p/Pop2p/Notp mRNA deadenylase complex in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11971964" target=_blank>Cosson B, <i>et al.</i> (2002 May)</a>. Poly(A)-binding protein acts in translation termination via eukaryotic release factor 3 interaction and does not influence [PSI(+)] propagation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923185" target=_blank>Hosoda N, <i>et al.</i> (2003 Oct 3)</a>. Translation termination factor eRF3 mediates mRNA decay through the regulation of deadenylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15169912" target=_blank>Mangus DA, <i>et al.</i> (2004 Jun)</a>. Positive and negative regulation of poly(A) nuclease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15356266" target=_blank>Trautwein M, <i>et al.</i> (2004 Nov)</a>. Arf1p provides an unexpected link between COPI vesicles and mRNA in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15525991" target=_blank>Amrani N, <i>et al.</i> (2004 Nov 4)</a>. A faux 3'-UTR promotes aberrant termination and triggers nonsense-mediated mRNA decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15337765" target=_blank>Kobayashi T, <i>et al.</i> (2004 Oct 29)</a>. The GTP-binding release factor eRF3 as a key mediator coupling translation termination to mRNA decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15769879" target=_blank>Brune C, <i>et al.</i> (2005 Apr)</a>. Yeast poly(A)-binding protein Pab1 shuttles between the nucleus and the cytoplasm and functions in mRNA export.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15630021" target=_blank>Dunn EF, <i>et al.</i> (2005 Jan 1)</a>. Yeast poly(A)-binding protein, Pab1, and PAN, a poly(A) nuclease complex recruited by Pab1, connect mRNA biogenesis to export.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16809762" target=_blank>Keeling KM, <i>et al.</i> (2006 Jul)</a>. Tpa1p is part of an mRNP complex that influences translation termination, mRNA deadenylation, and mRNA turnover in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919853" target=_blank>Oh YT, <i>et al.</i> (2010 Mar)</a>. Proteomic analysis of early phase of conidia germination in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23865587" target=_blank>Low JK, <i>et al.</i> (2013 Sep 6)</a>. Analysis of the proteome of Saccharomyces cerevisiae for methylarginine.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02883_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02883_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02883_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02883_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02883">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02884<br>EOG8QBZPJ<br><br>sce:HRP1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of cleavage factor I; cleavage factor I is a five-subunit complex required for the cleavage and polyadenylation of pre-mRNA 3' ends; RRM-containing heteronuclear RNA binding protein and hnRNPA/B family member that binds to poly (A) signal sequences; required for genome stability<br><br>
			<br><b>PomBase Description</b><br>mRNA cleavage factor complex subunit Msi2<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8770588" target=_blank>Henry M, <i>et al.</i> (1996 Jan)</a>. Potential RNA binding proteins in Saccharomyces cerevisiae identified as suppressors of temperature-sensitive mutations in NPL3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9334319" target=_blank>Kessler MM, <i>et al.</i> (1997 Oct 1)</a>. Hrp1, a sequence-specific RNA-binding protein that shuttles between the nucleus and the cytoplasm, is required for mRNA 3'-end formation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9857200" target=_blank>Minvielle-Sebastia L, <i>et al.</i> (1998 Dec 15)</a>. Control of cleavage site selection during mRNA 3' end formation by a yeast hnRNP.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9776761" target=_blank>Chen S, <i>et al.</i> (1998 Nov 1)</a>. A specific RNA-protein interaction at yeast polyadenylation efficiency elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10882134" target=_blank>González CI, <i>et al.</i> (2000 Mar)</a>. The yeast hnRNP-like protein Hrp1/Nab4 marks a transcript for nonsense-mediated mRNA decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11344258" target=_blank>Gross S, <i>et al.</i> (2001 May 22)</a>. Five subunits are required for reconstitution of the cleavage and polyadenylation activities of Saccharomyces cerevisiae cleavage factor I.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12192043" target=_blank>Hammell CM, <i>et al.</i> (2002 Sep)</a>. Coupling of termination, 3' processing, and mRNA export.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02884_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02884_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02884_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02884_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02884">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02885<br>EOG8QBZPJ<br><br>sce:NOP4<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Nucleolar protein; essential for processing and maturation of 27S pre-rRNA and large ribosomal subunit biogenesis; constituent of 66S pre-ribosomal particles; contains four RNA recognition motifs (RRMs)<br><br>
			<br><b>PomBase Description</b><br>RNA-binding protein Nop4 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular response to drug and nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8039506" target=_blank>Bergès T, <i>et al.</i> (1994 Jul 1)</a>. Synthetic lethality with fibrillarin identifies NOP77p, a nucleolar protein required for pre-rRNA processing and modification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8039505" target=_blank>Sun C, <i>et al.</i> (1994 Jul 1)</a>. The yeast NOP4 gene product is an essential nucleolar protein required for pre-rRNA processing and accumulation of 60S ribosomal subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19596074" target=_blank>Hagiwara D, <i>et al.</i> (2009 Nov)</a>. Transcriptional profiling for Aspergillusnidulans HogA MAPK signaling pathway in response to fludioxonil and osmotic stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23297348" target=_blank>Chen JS, <i>et al.</i> (2013 May)</a>. Comprehensive proteomics analysis reveals new substrates and regulators of the fission yeast clp1/cdc14 phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02885_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02885_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02885_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02885_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02885">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG02886<br>EOG8QBZPJ<br><br>sce:NOP13<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Nucleolar protein found in preribosomal complexes; contains an RNA recognition motif (RRM); relative distribution to the nucleolus increases upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>RNA-binding protein Rnp24<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8598051" target=_blank>VanHoy RW, <i>et al.</i> (1996 Mar)</a>. Molecular analysis of a novel schizosaccharomyces pombe gene containing two RNP consensus-sequence RNA-binding domains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11452019" target=_blank>Wu K, <i>et al.</i> (2001 Jul 15)</a>. Nucleolar protein Nop12p participates in synthesis of 25S rRNA in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23297348" target=_blank>Chen JS, <i>et al.</i> (2013 May)</a>. Comprehensive proteomics analysis reveals new substrates and regulators of the fission yeast clp1/cdc14 phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02886_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02886_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02886_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02886_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02886">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG02887<br>EOG8QBZPJ<br><br>sce:NOP12<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Nucleolar protein involved in pre-25S rRNA processing; also involved in biogenesis of large 60S ribosomal subunit; contains an RNA recognition motif (RRM); binds to Ebp2; similar to Nop13p and Nsr1p<br><br>
			<br><b>PomBase Description</b><br>RNA-binding protein Nop12 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11452019" target=_blank>Wu K, <i>et al.</i> (2001 Jul 15)</a>. Nucleolar protein Nop12p participates in synthesis of 25S rRNA in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23297348" target=_blank>Chen JS, <i>et al.</i> (2013 May)</a>. Comprehensive proteomics analysis reveals new substrates and regulators of the fission yeast clp1/cdc14 phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02887_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02887_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02887_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02887_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02887">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG02888<br>EOG8QBZPJ<br><br>sce:IST3<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Component of the U2 snRNP; required for the first catalytic step of splicing and for spliceosomal assembly; interacts with Rds3p and is required for Mer1p-activated splicing; diploid mutants have a specific defect in MATa1 pre-mRNA splicing which leads to haploid gene expression in diploids<br><br>
			<br><b>PomBase Description</b><br>RNA-binding protein Cwf29<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted nucleic acid binding, nucleotide binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=776230" target=_blank>Guiard B, <i>et al.</i> (1976)</a>. Complete amino acid sequence of the heme-binding core in bakers' yeast cytochrome b2 (L-(+)-lactate dehydrogenase).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287609" target=_blank>Gottschalk A, <i>et al.</i> (2001 May)</a>. A novel yeast U2 snRNP protein, Snu17p, is required for the first catalytic step of splicing and for progression of spliceosome assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11884590" target=_blank>Ohi MD, <i>et al.</i> (2002 Apr)</a>. Proteomics analysis reveals stable multiprotein complexes in both fission and budding yeasts containing Myb-related Cdc5p/Cef1p, novel pre-mRNA splicing factors, and snRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11804584" target=_blank>Stevens SW, <i>et al.</i> (2002 Jan)</a>. Composition and functional characterization of the yeast spliceosomal penta-snRNP.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14517302" target=_blank>Wang Q, <i>et al.</i> (2003 Oct)</a>. Rds3p is required for stable U2 snRNP recruitment to the splicing apparatus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973223" target=_blank>Spingola M, <i>et al.</i> (2004)</a>. Mer1p is a modular splicing factor whose function depends on the conserved U2 snRNP protein Snu17p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15565172" target=_blank>Dziembowski A, <i>et al.</i> (2004 Dec 8)</a>. Proteomic analysis identifies a new complex required for nuclear pre-mRNA retention and splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755920" target=_blank>Carnahan RH, <i>et al.</i> (2005 Mar)</a>. Dim1p is required for efficient splicing and export of mRNA encoding lid1p, a component of the fission yeast anaphase-promoting complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28821619" target=_blank>Burr R, <i>et al.</i> (2017 Sep 29)</a>. Dsc E3 ligase localization to the Golgi requires the ATPase Cdc48 and cofactor Ufd1 for activation of sterol regulatory element-binding protein in fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02888_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02888_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02888_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02888_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02888">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG02889<br>EOG8QBZPJ<br><br>sce:PUB1<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Poly (A)+ RNA-binding protein; abundant mRNP-component protein that binds mRNA and is required for stability of many mRNAs; component of glucose deprivation induced stress granules, involved in P-body-dependent granule assembly; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have mRNA binding, poly(U) RNA binding activity and role in nuclear-transcribed mRNA catabolic process, nonsense-mediated decay, regulation of mRNA stability, stress granule assembly<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8341595" target=_blank>Ripmaster TL, <i>et al.</i> (1993 Jul 11)</a>. A protein containing conserved RNA-recognition motifs is associated with ribosomal subunits in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413212" target=_blank>Anderson JT, <i>et al.</i> (1993 Oct)</a>. PUB1 is a major nuclear and cytoplasmic polyadenylated RNA-binding protein in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413213" target=_blank>Matunis MJ, <i>et al.</i> (1993 Oct)</a>. PUB1: a major yeast poly(A)+ RNA-binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8127652" target=_blank>Cockell M, <i>et al.</i> (1994 Jan 11)</a>. The yeast protein encoded by PUB1 binds T-rich single stranded DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9298649" target=_blank>Garrels JI, <i>et al.</i> (1997 Aug)</a>. Proteome studies of Saccharomyces cerevisiae: identification and characterization of abundant proteins.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02889_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02889_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02889_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02889_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02889">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG02890<br>EOG8QBZPJ<br><br>sce:HSH49<br><br>Genes: 30
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>U2-snRNP associated splicing factor; similar to the mammalian splicing factor SAP49; proposed to function as a U2-snRNP assembly factor along with Hsh155p and binding partner Cus1p; contains two RNA recognition motifs (RRM)<br><br>
			<br><b>PomBase Description</b><br>U2 snRNP-associated RNA-binding protein Sap49<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have U2 snRNP, cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8598051" target=_blank>VanHoy RW, <i>et al.</i> (1996 Mar)</a>. Molecular analysis of a novel schizosaccharomyces pombe gene containing two RNP consensus-sequence RNA-binding domains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14517302" target=_blank>Wang Q, <i>et al.</i> (2003 Oct)</a>. Rds3p is required for stable U2 snRNP recruitment to the splicing apparatus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755920" target=_blank>Carnahan RH, <i>et al.</i> (2005 Mar)</a>. Dim1p is required for efficient splicing and export of mRNA encoding lid1p, a component of the fission yeast anaphase-promoting complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22064476" target=_blank>Shao W, <i>et al.</i> (2012 Jan)</a>. A U1-U2 snRNP interaction network during intron definition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02890_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02890_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02890_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02890_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02890">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG02891<br>EOG8QBZPJ<br><br>sce:SNP1<br><br>Genes: 30
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Component of U1 snRNP required for mRNA splicing via spliceosome; substrate of the arginine methyltransferase Hmt1p; may interact with poly(A) polymerase to regulate polyadenylation; homolog of human U1 70K protein; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>U1 snRNP-associated protein Usp101<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have U1 snRNP, cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1714384" target=_blank>Smith V, <i>et al.</i> (1991 Sep)</a>. Cloning of a yeast U1 snRNP 70K protein homologue: functional conservation of an RNA-binding domain between humans and yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1387202" target=_blank>Kao HY, <i>et al.</i> (1992 Aug 11)</a>. The yeast homolog of the U1 snRNP protein 70K is encoded by the SNP1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8643384" target=_blank>Hilleren PJ, <i>et al.</i> (1995)</a>. The RRM domain is dispensable for yeast U1-70K function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565787" target=_blank>Hilleren PJ, <i>et al.</i> (1995 Nov)</a>. The amino-terminal domain of yeast U1-70K is necessary and sufficient for function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9012791" target=_blank>Neubauer G, <i>et al.</i> (1997 Jan 21)</a>. Identification of the proteins of the yeast U1 small nuclear ribonucleoprotein complex by mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9630245" target=_blank>Gottschalk A, <i>et al.</i> (1998 Apr)</a>. A comprehensive biochemical and genetic analysis of the yeast U1 snRNP reveals five novel proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10072386" target=_blank>Zhang D, <i>et al.</i> (1999 Mar 1)</a>. Identification of eight proteins that cross-link to pre-mRNA in the yeast commitment complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11425851" target=_blank>Awasthi S, <i>et al.</i> (2001 Aug 17)</a>. New roles for the Snp1 and Exo84 proteins in yeast pre-mRNA splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11804584" target=_blank>Stevens SW, <i>et al.</i> (2002 Jan)</a>. Composition and functional characterization of the yeast spliceosomal penta-snRNP.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02891_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02891_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02891_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02891_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02891">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG02892<br>EOG8QBZPJ<br><br>sce:SGN1<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic RNA-binding protein; contains an RNA recognition motif (RRM); may have a role in mRNA translation, as suggested by genetic interactions with genes encoding proteins involved in translational initiation<br><br>
			<br><b>PomBase Description</b><br>poly(A) binding protein Pab2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have poly(A) binding activity, role in nuclear-transcribed mRNA poly(A) tail shortening and Mmi1 nuclear focus complex, cytoplasm, nuclear chromatin, nucleolus, nucleoplasm, polysome localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=776230" target=_blank>Guiard B, <i>et al.</i> (1976)</a>. Complete amino acid sequence of the heme-binding core in bakers' yeast cytochrome b2 (L-(+)-lactate dehydrogenase).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10764794" target=_blank>Winstall E, <i>et al.</i> (2000 Jul 21)</a>. The Saccharomyces cerevisiae RNA-binding protein Rbp29 functions in cytoplasmic mRNA metabolism.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02892_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02892_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02892_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02892_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02892">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG02893<br>EOG8QBZPJ<br><br>sce:RNA15<br><br>Genes: 28
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Component of the cleavage and polyadenylation factor I (CF I); CF 1, composed of the CF 1A complex (Rna14p, Rna15p, Clp1p, Pcf11p) and Hrp1, is involved in cleavage and polyadenylation of mRNA 3' ends; interacts with the A-rich polyadenylation signal in complex with Rna14p and Hrp1p; mutant displays reduced transcription elongation in the G-less-based run-on (GLRO) assay; required for gene looping and maintenance of genome stability<br><br>
			<br><b>PomBase Description</b><br>RNA-binding protein Rna15<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, mRNA cleavage and polyadenylation specificity factor complex localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1674817" target=_blank>Minvielle-Sebastia L, <i>et al.</i> (1991 Jun)</a>. Mutations in the yeast RNA14 and RNA15 genes result in an abnormal mRNA decay rate; sequence analysis reveals an RNA-binding domain in the RNA15 protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7992054" target=_blank>Minvielle-Sebastia L, <i>et al.</i> (1994 Dec 9)</a>. RNA14 and RNA15 proteins as components of a yeast pre-mRNA 3'-end processing factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900210" target=_blank>Kessler MM, <i>et al.</i> (1996 Oct 25)</a>. Purification of the Saccharomyces cerevisiae cleavage/polyadenylation factor I. Separation into two components that are required for both cleavage and polyadenylation of mRNA 3' ends.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199303" target=_blank>Amrani N, <i>et al.</i> (1997 Jul)</a>. Yeast Pab1 interacts with Rna15 and participates in the control of the poly(A) tail length in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9234674" target=_blank>Feuermann M, <i>et al.</i> (1997 Jul)</a>. The characterization of two new clusters of duplicated genes suggests a 'Lego' organization of the yeast Saccharomyces cerevisiae chromosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11689695" target=_blank>Gross S, <i>et al.</i> (2001 Dec)</a>. Rna15 interaction with the A-rich yeast polyadenylation signal is an essential step in mRNA 3'-end formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11344258" target=_blank>Gross S, <i>et al.</i> (2001 May 22)</a>. Five subunits are required for reconstitution of the cleavage and polyadenylation activities of Saccharomyces cerevisiae cleavage factor I.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20512112" target=_blank>Yamanaka S, <i>et al.</i> (2010 Jul 7)</a>. Importance of polyadenylation in the selective elimination of meiotic mRNAs in growing S. pombe cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513198" target=_blank>Paulson AR, <i>et al.</i> (2012 Jun)</a>. Crystal structure of the Rna14-Rna15 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02893_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02893_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02893_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02893_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02893">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG02894<br>EOG8QBZPJ<br><br>sce:absent<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02894_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02894_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02894_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02894_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02894">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG02895<br>EOG8HT79J<br>EOG8QBZPJ<br><br>sce:PES4<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Poly(A) binding protein, suppressor of DNA polymerase epsilon mutation; PES4 has a paralog, MIP6, that arose from the whole genome duplication<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02895_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02895_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02895_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02895_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02895">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG02896<br>EOG8QBZPJ<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted nucleic acid binding, nucleotide binding activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02896_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02896_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02896_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02896_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02896">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG02897<br>EOG8QBZPJ<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Glycine-rich RNA-binding protein<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02897_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02897_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02897_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02897_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02897">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG02898<br>EOG8QBZPJ<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>RNA-binding protein Csx1|RNA-binding protein involved in histone acetylation|U1 snRNP-associated protein Usp109|mRNA cleavage and polyadenylation specificity factor complex subunit Ctf1<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11389847" target=_blank>Aranda A, <i>et al.</i> (2001 May)</a>. Transcriptional termination factors for RNA polymerase II in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14617822" target=_blank>Roguev A, <i>et al.</i> (2004 Feb)</a>. A comparative analysis of an orthologous proteomic environment in the yeasts Saccharomyces cerevisiae and Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20537132" target=_blank>Han TX, <i>et al.</i> (2010)</a>. Global fitness profiling of fission yeast deletion strains by barcode sequencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21169418" target=_blank>Henry TC, <i>et al.</i> (2011 Feb)</a>. Systematic screen of Schizosaccharomyces pombe deletion collection uncovers parallel evolution of the phosphate signal transduction pathway in yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24945319" target=_blank>Vanoosthuyse V, <i>et al.</i> (2014 Jun)</a>. CPF-associated phosphatase activity opposes condensin-mediated chromosome condensation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547512" target=_blank>Estill M, <i>et al.</i> (2015 May)</a>. Dissection of the PHO pathway in Schizosaccharomyces pombe using epistasis and the alternate repressor adenine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28567704" target=_blank>Sonkar A, <i>et al.</i> (2017 May 31)</a>. Fission yeast Ctf1, a cleavage and polyadenylation factor subunit is required for the maintenance of genomic integrity.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02898_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02898_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02898_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02898_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02898">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG02899<br>EOG8HT79J<br><br>sce:MIP6<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative RNA-binding protein; interacts with Mex67p, which is a component of the nuclear pore involved in nuclear mRNA export; MIP6 has a paralog, PES4, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9214641" target=_blank>Segref A, <i>et al.</i> (1997 Jun 2)</a>. Mex67p, a novel factor for nuclear mRNA export, binds to both poly(A)+ RNA and nuclear pores.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02899_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02899_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02899_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02899_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01168_phobius.html#FOG02899">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG01168.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>