<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00594.html">HOG00594&emsp;&emsp;flavoprotein pyridine nucleotide cytochrome reductase family; globin family. Two-domain flavohemoproteins subfamily</a><br> 
																	HOG00595&emsp;&emsp;<br> 
																	<a href="HOG00596.html">HOG00596&emsp;&emsp;metallo-dependent hydrolases superfamily. Allantoinase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00595.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00595_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00595_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00595_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00595_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01418<br>EOG8VX0PK<br><br>sce:absent<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>uricase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Urate oxidase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5832832" target=_blank>Darlington AJ, <i>et al.</i> (1965 May 8)</a>. Biochemical and genetical studies of purine breakdown in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5583651" target=_blank>Scazzocchio C, <i>et al.</i> (1967 Dec 18)</a>. The genetic control of xanthine dehydrogenase and urate oxidase synthess in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6025432" target=_blank>Darlington AJ, <i>et al.</i> (1967 Mar)</a>. Use of analogues and the substrate-sensitivity of mutants in analysis of purine uptake and breakdown in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5680611" target=_blank>Darlington AJ, <i>et al.</i> (1968 Sep 24)</a>. Evidence for an alternative pathway of xanthine oxidation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5680610" target=_blank>Scazzocchio C, <i>et al.</i> (1968 Sep 24)</a>. The induction and repression of the enzymes of purine breakdown in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5475567" target=_blank>Arst HN Jr, <i>et al.</i> (1970)</a>. Molybdate metabolism in Aspergillus nidulans. I. Mutations affecting nitrate reductase and-or xanthine dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226863" target=_blank>Oestreicher N, <i>et al.</i> (1993 Nov 5)</a>. Sequence, regulation, and mutational analysis of the gene encoding urate oxidase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8224862" target=_blank>Oestreicher N, <i>et al.</i> (1993 Oct 15)</a>. Characterisation, cloning and integrative properties of the gene encoding urate oxidase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765126" target=_blank>Scazzocchio C, <i>et al.</i> (1994)</a>. The purine degradation pathway, genetics, biochemistry and regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7729421" target=_blank>Su√°rez T, <i>et al.</i> (1995 Apr 3)</a>. The sequence and binding specificity of UaY, the specific regulator of the purine utilization pathway in Aspergillus nidulans, suggest an evolutionary relationship with the PPR1 protein of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7602582" target=_blank>Oestreicher N, <i>et al.</i> (1995 Jun 16)</a>. A single amino acid change in a pathway-specific transcription factor results in differing degrees of constitutivity, hyperinducibility and derepression of several structural genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8569680" target=_blank>Platt A, <i>et al.</i> (1996 Jan 15)</a>. Mutational analysis of the C-terminal region of AREA, the transcription factor mediating nitrogen metabolite repression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10613888" target=_blank>Cubero B, <i>et al.</i> (2000 Jan)</a>. Metabolite repression and inducer exclusion in the proline utilization gene cluster of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11810243" target=_blank>Ribard C, <i>et al.</i> (2001 Dec)</a>. The oxpA5 mutation of Aspergillus nidulans is an allele of adB, the gene encoding adenylosuccinate synthetase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16889413" target=_blank>Forgue P, <i>et al.</i> (2006 Aug)</a>. NMR metabolic profiling of Aspergillus nidulans to monitor drug and protein activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16963627" target=_blank>Morozov IY, <i>et al.</i> (2006 Nov)</a>. Nonsense-mediated mRNA decay mutation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055231" target=_blank>Oestreicher N, <i>et al.</i> (2008 May)</a>. The nadA gene of Aspergillus nidulans, encoding adenine deaminase, is subject to a unique regulatory pattern.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21419234" target=_blank>Gournas C, <i>et al.</i> (2011 Aug)</a>. Completing the purine utilisation pathway of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22835028" target=_blank>Saykhedkar S, <i>et al.</i> (2012 Jul 26)</a>. A time course analysis of the extracellular proteome of Aspergillus nidulans growing on sorghum stover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633078" target=_blank>Sun X, <i>et al.</i> (2013 May)</a>. PyrG is required for maintaining stable cellular uracil level and normal sporulation pattern under excess uracil stress in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01418_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01418_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01418_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01418_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00595_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01419<br>EOG8VX0PK<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Uricase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01419_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01419_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01419_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01419_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00595_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00595.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>