<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01209.html">HOG01209&emsp;&emsp;RGC1 family</a><br> 
																	HOG01210&emsp;&emsp;<br> 
																	<a href="HOG01211.html">HOG01211&emsp;&emsp;RNF10 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01210.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01210_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01210_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03290<br>EOG8BNZT0<br>EOG8H44K8<br>EOG8QJQ4P<br>EOG8T76J9<br>EOG8VQ852<br><br>sce:CHA4<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>DNA binding transcriptional activator; mediates serine/threonine activation of the catabolic L-serine (L-threonine) deaminase (CHA1); Zinc-finger protein with Zn[2]-Cys[6] fungal-type binuclear cluster domain<br><br>
			<br><b>PomBase Description</b><br>transcription factor (predicted)|zn(2)-C6 fungal-type DNA-binding transcription factor, truncated<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted DNA binding, RNA polymerase II transcription factor activity, sequence-specific DNA binding, zinc ion binding activity and role in regulation of transcription, DNA-templated, transcription, DNA-templated|Has domain(s) with predicted DNA binding, RNA polymerase II transcription factor activity, sequence-specific DNA binding, zinc ion binding activity and role in regulation of transcription, DNA-templated, transcription, DNA-templated<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8889513" target=_blank>Holmberg S, <i>et al.</i> (1996 Oct)</a>. Cha4p of Saccharomyces cerevisiae activates transcription via serine/threonine response elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21270388" target=_blank>Bitton DA, <i>et al.</i> (2011 Apr)</a>. Augmented annotation of the Schizosaccharomyces pombe genome reveals additional genes required for growth and viability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21511999" target=_blank>Rhind N, <i>et al.</i> (2011 May 20)</a>. Comparative functional genomics of the fission yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23950735" target=_blank>Sun LL, <i>et al.</i> (2013)</a>. Global analysis of fission yeast mating genes reveals new autophagy factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695302" target=_blank>Vachon L, <i>et al.</i> (2013 Aug)</a>. Functional characterization of fission yeast transcription factors by overexpression analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26615217" target=_blank>Hu W, <i>et al.</i> (2015 Nov 27)</a>. Bulk Segregant Analysis Reveals the Genetic Basis of a Natural Trait Variation in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03290_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03290_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03290_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03290_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>14 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03291<br>EOG8H44K8<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted DNA binding, RNA polymerase II transcription factor activity, sequence-specific DNA binding, zinc ion binding activity and role in regulation of transcription, DNA-templated, transcription, DNA-templated<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03291_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03291_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03291_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03291_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03292<br>EOG8T76J9<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have protein domain specific binding, transcription regulatory region DNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4382427" target=_blank>Pateman JA, <i>et al.</i> (1967 Jul)</a>. Genetic and biochemical studies of nitrate reduction in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5788705" target=_blank>Arst HN Jr, <i>et al.</i> (1969 Jun)</a>. Methylammonium resistance in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5475567" target=_blank>Arst HN Jr, <i>et al.</i> (1970)</a>. Molybdate metabolism in Aspergillus nidulans. I. Mutations affecting nitrate reductase and-or xanthine dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4581424" target=_blank>Clutterbuck AJ, <i>et al.</i> (1973 Jun)</a>. Gene symbols in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4274591" target=_blank>Hankinson O, <i>et al.</i> (1974 Apr 25)</a>. Regulation of the pentose phosphate pathway in the fungus Aspergillus nidulans. The effect of growth with nitrate.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=129029" target=_blank>Davis RH, <i>et al.</i> (1975)</a>. Compartmentation and regulation of fungal metabolism: genetic approaches.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=163669" target=_blank>Hankinson O, <i>et al.</i> (1975 Jan)</a>. Regulation of mannitol-1-phosphate dehydrogenase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8697" target=_blank>Cove DJ, <i>et al.</i> (1976 Jul 23)</a>. Chlorate toxicity in Aspergillus nidulans. Studies of mutants altered in nitrate assimilation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17826" target=_blank>Dunn-Coleman NS, <i>et al.</i> (1977 Apr 29)</a>. In vivo and in vitro studies of nitrate reductase regulation in Asperillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=157059" target=_blank>Garrett RH, <i>et al.</i> (1978)</a>. Nitrate assimilation in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=347305" target=_blank>Rand KN, <i>et al.</i> (1978 Apr 20)</a>. Mutations in nirA gene of Aspergillus nidulans and nitrogen metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=389305" target=_blank>Cove DJ, <i>et al.</i> (1979 Aug)</a>. Genetic studies of nitrate assimilation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=375022" target=_blank>Dunn-Coleman NS, <i>et al.</i> (1979 Mar 9)</a>. The regulation of hexokinase and phosphoglucomutase activity in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24185869" target=_blank>Tollervey DW, <i>et al.</i> (1981 Sep)</a>. Mutations to constitutivity and derepression are separate and separable in a regulatory gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6350263" target=_blank>Brownlee AG, <i>et al.</i> (1983 Sep)</a>. Nitrate uptake in Aspergillus nidulans and involvement of the third gene of the nitrate assimilation gene cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6101115" target=_blank>Arst HN Jr, <i>et al.</i> (1984 Sep)</a>. Regulation of gene expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2869649" target=_blank>Wiame JM, <i>et al.</i> (1985)</a>. Nitrogen catabolite repression in yeasts and filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2652389" target=_blank>Davis MA, <i>et al.</i> (1989 Jan)</a>. Regulatory genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2381424" target=_blank>Debets AJ, <i>et al.</i> (1990 May)</a>. Genetic analysis of Aspergillus niger: isolation of chlorate resistance mutants, their use in mitotic mapping and evidence for an eighth linkage group.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1829047" target=_blank>Hawker KL, <i>et al.</i> (1991 Apr)</a>. Heterologous expression and regulation of the Neurospora crassa nit-4 pathway-specific regulatory gene for nitrate assimilation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1756977" target=_blank>Daboussi MJ, <i>et al.</i> (1991 Dec 20)</a>. Heterologous expression of the Aspergillus nidulans regulatory gene nirA in Fusarium oxysporum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1990284" target=_blank>Burger G, <i>et al.</i> (1991 Feb)</a>. Molecular cloning and functional characterization of the pathway-specific regulatory gene nirA, which controls nitrate assimilation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1986367" target=_blank>Unkles SE, <i>et al.</i> (1991 Jan 1)</a>. crnA encodes a nitrate transporter in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1922075" target=_blank>Burger G, <i>et al.</i> (1991 Nov)</a>. nirA, the pathway-specific regulatory gene of nitrate assimilation in Aspergillus nidulans, encodes a putative GAL4-type zinc finger protein and contains four introns in highly conserved regions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840634" target=_blank>Yuan GF, <i>et al.</i> (1991 Nov)</a>. nit-4, a pathway-specific regulatory gene of Neurospora crassa, encodes a protein with a putative binuclear zinc DNA-binding domain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1504607" target=_blank>Scazzocchio C, <i>et al.</i> (1992)</a>. Control of gene expression in the catabolic pathways of Aspergillus nidulans: a personal and biased account.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1538701" target=_blank>Hawker KL, <i>et al.</i> (1992 Feb)</a>. Nitrate reductase and nitrite reductase transcript levels in various mutants of Aspergillus nidulans: confirmation of autogenous regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8257101" target=_blank>Marzluf GA, <i>et al.</i> (1993)</a>. Regulation of sulfur and nitrogen metabolism in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765124" target=_blank>Kinghorn JR, <i>et al.</i> (1994)</a>. Inorganic nitrogen assimilation: molecular aspects.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8121403" target=_blank>Gems D, <i>et al.</i> (1994 Feb)</a>. An 'instant gene bank' method for gene cloning by mutant complementation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565720" target=_blank>Punt PJ, <i>et al.</i> (1995 Oct)</a>. The intergenic region between the divergently transcribed niiA and niaD genes of Aspergillus nidulans contains multiple NirA binding sites which act bidirectionally.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8662212" target=_blank>Chang PK, <i>et al.</i> (1996 Jun)</a>. Characterization of the Aspergillus parasiticus niaD and niiA gene cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9290251" target=_blank>Todd RB, <i>et al.</i> (1997 Jun)</a>. Evolution of a fungal regulatory gene family: the Zn(II)2Cys6 binuclear cluster DNA binding motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9488449" target=_blank>Strauss J, <i>et al.</i> (1998 Mar)</a>. The regulator of nitrate assimilation in ascomycetes is a dimer which binds a nonrepeated, asymmetrical sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394902" target=_blank>Snoeijers SS, <i>et al.</i> (1999 Jun)</a>. Transcription of the avirulence gene Avr9 of the fungal tomato pathogen Cladosporium fulvum is regulated by a GATA-type transcription factor in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10075929" target=_blank>Muro-Pastor MI, <i>et al.</i> (1999 Mar 15)</a>. The GATA factor AreA is essential for chromatin remodelling in a eukaryotic bidirectional promoter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11707396" target=_blank>Unkles SE, <i>et al.</i> (2001 Nov 15)</a>. Apparent genetic redundancy facilitates ecological plasticity for nitrate transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972792" target=_blank>Narendja F, <i>et al.</i> (2002 Apr)</a>. Nitrate and the GATA factor AreA are necessary for in vivo binding of NirA, the pathway-specific transcriptional activator of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12092813" target=_blank>Takaya N, <i>et al.</i> (2002 May)</a>. Transcriptional control of nitric oxide reductase gene (CYP55) in the fungal denitrifier Fusarium oxysporum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15075269" target=_blank>Muro-Pastor MI, <i>et al.</i> (2004 Apr)</a>. A paradoxical mutant GATA factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16390440" target=_blank>Berger H, <i>et al.</i> (2006 Jan)</a>. The GATA factor AreA regulates localization and in vivo binding site occupancy of the nitrate activator NirA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020584" target=_blank>Caddick MX, <i>et al.</i> (2006 Oct)</a>. Opposing signals differentially regulate transcript stability in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116695" target=_blank>Bernreiter A, <i>et al.</i> (2007 Feb)</a>. Nuclear export of the transcription factor NirA is a regulatory checkpoint for nitrate induction in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18024100" target=_blank>Wang Y, <i>et al.</i> (2008 Feb)</a>. Nitrite transport is mediated by the nitrite-specific high-affinity NitA transporter and by nitrate transporters NrtA, NrtB in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673441" target=_blank>Berger H, <i>et al.</i> (2008 Sep)</a>. Dissecting individual steps of nitrogen transcription factor cooperation in the Aspergillus nidulans nitrate cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20969648" target=_blank>Schinko T, <i>et al.</i> (2010 Nov)</a>. Transcriptome analysis of nitrate assimilation in Aspergillus nidulans reveals connections to nitric oxide metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22038601" target=_blank>Zhou S, <i>et al.</i> (2012 Jan)</a>. Heme-biosynthetic porphobilinogen deaminase protects Aspergillus nidulans from nitrosative stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23454548" target=_blank>Schinko T, <i>et al.</i> (2013 May)</a>. Pseudo-constitutivity of nitrate-responsive genes in nitrate reductase mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23973960" target=_blank>Scazzocchio C, <i>et al.</i> (2013 Sep-Oct)</a>. In praise of erroneous hypotheses.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25506342" target=_blank>Tudzynski B, <i>et al.</i> (2014)</a>. Nitrogen regulation of fungal secondary metabolism in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26132230" target=_blank>Gallmetzer A, <i>et al.</i> (2015 Jul)</a>. Reversible Oxidation of a Conserved Methionine in the Nuclear Export Sequence Determines Subcellular Distribution and Activity of the Fungal Nitrate Regulator NirA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26353949" target=_blank>Marcos AT, <i>et al.</i> (2016 Jan)</a>. Nitric oxide synthesis by nitrate reductase is regulated during development in Aspergillus.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03292_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03292_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03292_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03292_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03293<br>EOG8BNZT0<br>EOG8T76J9<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>transcription factor Cha4 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted DNA binding, RNA polymerase II transcription factor activity, sequence-specific DNA binding, zinc ion binding activity and role in regulation of transcription, DNA-templated, transcription, DNA-templated<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20404084" target=_blank>Kaufmann I, <i>et al.</i> (2010 Jul)</a>. Transcriptional activation of the general amino acid permease gene per1 by the histone deacetylase Clr6 Is regulated by Oca2 kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695302" target=_blank>Vachon L, <i>et al.</i> (2013 Aug)</a>. Functional characterization of fission yeast transcription factors by overexpression analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03293_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03293_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03293_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03293_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03294<br>EOG8T76J9<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted DNA binding, RNA polymerase II transcription factor activity, sequence-specific DNA binding, zinc ion binding activity and role in regulation of transcription, DNA-templated, transcription, DNA-templated|Has domain(s) with predicted DNA binding, zinc ion binding activity, role in transcription, DNA-templated and nucleus localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03294_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03294_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03294_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03294_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03295<br>EOG8T76J9<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03295_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03295_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03295_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03295_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG03296<br>EOG8T76J9<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted DNA binding, RNA polymerase II transcription factor activity, sequence-specific DNA binding, zinc ion binding activity and role in regulation of transcription, DNA-templated, transcription, DNA-templated|Ortholog(s) have role in negative regulation of transcription from RNA polymerase II promoter, regulation of nitrogen utilization and nucleus localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03296_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03296_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03296_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03296_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG03297<br>EOG8BNZT0<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03297_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03297_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03297_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03297_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01210.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>