<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01783.html">HOG01783&emsp;&emsp;</a><br> 
																	HOG01784&emsp;&emsp;<br> 
																	<a href="HOG01785.html">HOG01785&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01784.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01784_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01784_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG04948<br>EOG8FN2ZS<br><br>sce:CDC48<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>AAA ATPase; subunit of polyubiquitin-selective segregase complex involved in ERAD, cell wall integrity during heat stress, mitotic spindle disassembly; subunit of complex involved in mitochondria-associated degradation; role in mobilizing membrane bound transcription factors by regulated ubiquitin/proteasome-dependent processing, in macroautophagy, PMN, RAD, ribophagy, homotypic ER membrane fusion, disassembly of Met30p from SCF complex; functional ortholog of human p97/VCP<br><br>
			<br><b>PomBase Description</b><br>AAA family ATPase involved in ubiquitin-mediated protein degradation Cdc48<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATPase activity, protein phosphatase type 1 regulator activity, ubiquitin binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6749598" target=_blank>Moir D, <i>et al.</i> (1982 Apr)</a>. Cold-sensitive cell-division-cycle mutants of yeast: isolation, properties, and pseudoreversion studies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1860879" target=_blank>Fröhlich KU, <i>et al.</i> (1991 Aug)</a>. Yeast cell cycle protein CDC48p shows full-length homology to the mammalian protein VCP and is a member of a protein family involved in secretion, peroxisome formation, and gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11740563" target=_blank>Ye Y, <i>et al.</i> (2001 Dec 6)</a>. The AAA ATPase Cdc48/p97 and its partners transport proteins from the ER into the cytosol.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11733065" target=_blank>Rape M, <i>et al.</i> (2001 Nov 30)</a>. Mobilization of processed, membrane-tethered SPT23 transcription factor by CDC48(UFD1/NPL4), a ubiquitin-selective chaperone.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11598205" target=_blank>Hitchcock AL, <i>et al.</i> (2001 Oct)</a>. The conserved npl4 protein complex mediates proteasome-dependent membrane-bound transcription factor activation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11813000" target=_blank>Jarosch E, <i>et al.</i> (2002 Feb)</a>. Protein dislocation from the ER requires polyubiquitination and the AAA-ATPase Cdc48.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847109" target=_blank>Braun S, <i>et al.</i> (2002 Feb 15)</a>. Role of the ubiquitin-selective CDC48(UFD1/NPL4 )chaperone (segregase) in ERAD of OLE1 and other substrates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12872131" target=_blank>Peng J, <i>et al.</i> (2003 Aug)</a>. A proteomics approach to understanding protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15258615" target=_blank>Schuberth C, <i>et al.</i> (2004 Aug)</a>. Shp1 and Ubx2 are adaptors of Cdc48 involved in ubiquitin-dependent protein degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16427015" target=_blank>Rumpf S, <i>et al.</i> (2006 Jan 20)</a>. Functional division of substrate processing cofactors of the ubiquitin-selective Cdc48 chaperone.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16873066" target=_blank>Carvalho P, <i>et al.</i> (2006 Jul 28)</a>. Distinct ubiquitin-ligase complexes define convergent pathways for the degradation of ER proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21070972" target=_blank>Heo JM, <i>et al.</i> (2010 Nov 12)</a>. A stress-responsive system for mitochondrial protein degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21148305" target=_blank>Tran JR, <i>et al.</i> (2011 Feb 18)</a>. A Cdc48p-associated factor modulates endoplasmic reticulum-associated degradation, cell stress, and ubiquitinated protein homeostasis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23178123" target=_blank>Brandman O, <i>et al.</i> (2012 Nov 21)</a>. A ribosome-bound quality control complex triggers degradation of nascent peptides and signals translation stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23479637" target=_blank>Defenouillère Q, <i>et al.</i> (2013 Mar 26)</a>. Cdc48-associated complex bound to 60S particles is required for the clearance of aberrant translation products.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24261871" target=_blank>Matsuda R, <i>et al.</i> (2014 Jan)</a>. Protein quality control systems associated with no-go and nonstop mRNA surveillance in yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04948_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04948_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04948_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04948_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG04949<br>EOG8FN2ZS<br><br>sce:AFG2<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATPase of the CDC48/PAS1/SEC18 (AAA) family, forms a hexameric complex; is essential for pre-60S maturation and release of several preribosome maturation factors; releases Rlp24p from purified pre-60S particles in vitro; target of the ribosomal biosynthesis inhibitor diazaborine; may be involved in degradation of aberrant mRNAs<br><br>
			<br><b>PomBase Description</b><br>ribosome biogenesis factor recycling AAA family ATPase (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8109176" target=_blank>Thorsness PE, <i>et al.</i> (1993 Nov)</a>. AFG2, an essential gene in yeast, encodes a new member of the Sec18p, Pas1p, Cdc48p, TBP-1 family of putative ATPases.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04949_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04949_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04949_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04949_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG04950<br>EOG86Q58M<br>EOG8FN2ZS<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, nucleoside-triphosphatase activity, nucleotide binding activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04950_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04950_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04950_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04950_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01784.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>