<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00197.html">HOG00197&emsp;&emsp;triosephosphate isomerase family</a><br> 
																	HOG00198&emsp;&emsp;<br> 
																	<a href="HOG00200.html">HOG00200&emsp;&emsp;phosphoglycerate kinase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00198.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00198_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00198_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00198_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00198_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00285<br>EOG8DFN4R<br>TDH3<br>sce:TDH3<br><br>Genes: 39
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NAD-Glyceraldehyde-3-phosphate dehydrogenase<br><br>
			<br><b>SGD Description</b><br>Glyceraldehyde-3-phosphate dehydrogenase (GAPDH), isozyme 3; involved in glycolysis and gluconeogenesis; tetramer that catalyzes the reaction of glyceraldehyde-3-phosphate to 1,3 bis-phosphoglycerate; detected in the cytoplasm and cell wall; GAPDH-derived antimicrobial peptides secreted by S. cerevisiae are active against a wide variety of wine-related yeasts and bacteria; binds AU-rich RNA<br><br>
			<br><b>PomBase Description</b><br>glyceraldehyde 3-phosphate dehydrogenase Gpd3|glyceraldehyde-3-phosphate dehydrogenase Tdh1<br><br>
			<br><b>AspGD Description</b><br>Glyceraldehyde-3-phosphate dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11946592" target=_blank>Jones GM, <i>et al.</i> (1972 May 1)</a>. Glyceraldehyde 3-phosphate dehydrogenase: Amino acid sequence of enzyme from baker's yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=385592" target=_blank>Holland JP, <i>et al.</i> (1979 Oct 10)</a>. The primary structure of a glyceraldehyde-3-phosphate dehydrogenase gene from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3127274" target=_blank>Penttilä M, <i>et al.</i> (1987)</a>. A versatile transformation system for the cellulolytic filamentous fungus Trichoderma reesei.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3066699" target=_blank>Punt PJ, <i>et al.</i> (1988 Sep 15)</a>. Isolation and characterization of the glyceraldehyde-3-phosphate dehydrogenase gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2674943" target=_blank>Smith TL, <i>et al.</i> (1989 Sep)</a>. Disparate evolution of yeasts and filamentous fungi indicated by phylogenetic analysis of glyceraldehyde-3-phosphate dehydrogenase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2688930" target=_blank>de Ruiter-Jacobs YM, <i>et al.</i> (1989 Sep)</a>. A gene transfer system based on the homologous pyrG gene and efficient expression of bacterial genes in Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1366900" target=_blank>Archer DB, <i>et al.</i> (1990 Aug)</a>. Hen egg white lysozyme expressed in, and secreted from, Aspergillus niger is correctly processed and folded.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2377477" target=_blank>Shuster JR, <i>et al.</i> (1990 Jul 25)</a>. Kluyveromyces lactis glyceraldehyde-3-phosphate dehydrogenase and alcohol dehydrogenase-1 genes are linked and divergently transcribed.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2121607" target=_blank>Punt PJ, <i>et al.</i> (1990 Sep 1)</a>. Functional elements in the promoter region of the Aspergillus nidulans gpdA gene encoding glyceraldehyde-3-phosphate dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1367494" target=_blank>Punt PJ, <i>et al.</i> (1991 Jan)</a>. Intracellular and extracellular production of proteins in Aspergillus under the control of expression signals of the highly expressed Aspergillus nidulans gpdA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1367201" target=_blank>Hanegraaf PP, <i>et al.</i> (1991 Mar)</a>. Construction and physiological characterization of glyceraldehyde-3-phosphate dehydrogenase overproducing transformants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1745237" target=_blank>Lloyd AT, <i>et al.</i> (1991 Nov)</a>. Codon usage in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452034" target=_blank>Templeton MD, <i>et al.</i> (1992 Dec 1)</a>. Cloning and molecular characterization of the glyceraldehyde-3-phosphate dehydrogenase-encoding gene and cDNA from the plant pathogenic fungus Glomerella cingulata.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1398125" target=_blank>Punt PJ, <i>et al.</i> (1992 Oct 12)</a>. An upstream activating sequence from the Aspergillus nidulans gpdA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765105" target=_blank>Sánchez-Torres P, <i>et al.</i> (1994 Jun)</a>. Development of a transformation system for Trichoderma longibrachiatum and its use for constructing multicopy transformants for the egl1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747954" target=_blank>Glass NL, <i>et al.</i> (1995 Apr)</a>. Development of primer sets designed for use with the PCR to amplify conserved genes from filamentous ascomycetes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7737086" target=_blank>Norbeck J, <i>et al.</i> (1995 Jan)</a>. Gene linkage of two-dimensional polyacrylamide gel electrophoresis resolved proteins from isogene families in Saccharomyces cerevisiae by microsequencing of in-gel trypsin generated peptides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7789794" target=_blank>Punt PJ, <i>et al.</i> (1995 May 26)</a>. A mini-promoter lacZ gene fusion for the analysis of fungal transcription control sequences.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260938" target=_blank>Gil-Navarro I, <i>et al.</i> (1997 Aug)</a>. The glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase of Candida albicans is a surface antigen.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9603839" target=_blank>Redkar RJ, <i>et al.</i> (1998 Jun)</a>. Transcriptional activation of the Aspergillus nidulans gpdA promoter by osmotic signals.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18473559" target=_blank>Villanueva A, <i>et al.</i> (1999 Sep)</a>. Apparent mRNA instability in Aspergillus nidulans and Aspergillus terreus of a heterologous cDNA encoding the major capsid antigen of Rotavirus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11502169" target=_blank>Grandier-Vazeille X, <i>et al.</i> (2001 Aug 21)</a>. Yeast mitochondrial dehydrogenases are associated in a supramolecular complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12207217" target=_blank>Melin P, <i>et al.</i> (2002 Aug)</a>. Proteome analysis of Aspergillus nidulans reveals proteins associated with the response to the antibiotic concanamycin A, produced by Streptomyces species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12223193" target=_blank>Nikolaev I, <i>et al.</i> (2002 Oct)</a>. Heterologous expression of the Aspergillus nidulans alcR-alcA system in Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12684372" target=_blank>Greene AV, <i>et al.</i> (2003 Apr)</a>. A circadian oscillator in Aspergillus spp. regulates daily development and gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15620689" target=_blank>Mitsuzawa H, <i>et al.</i> (2005 Jan 3)</a>. Glyceraldehyde-3-phosphate dehydrogenase and actin associate with RNA polymerase II and interact with its Rpb7 subunit.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16699756" target=_blank>Lubertozzi D, <i>et al.</i> (2006 Oct)</a>. Marker and promoter effects on heterologous expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17258477" target=_blank>Kim Y, <i>et al.</i> (2007 Sep)</a>. Proteome map of Aspergillus nidulans during osmoadaptation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17995919" target=_blank>Helber N, <i>et al.</i> (2008)</a>. Expression of the fluorescence markers DsRed and GFP fused to a nuclear localization signal in the arbuscular mycorrhizal fungus Glomus intraradices.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18406331" target=_blank>Morigasaki S, <i>et al.</i> (2008 Apr 11)</a>. Glycolytic enzyme GAPDH promotes peroxide stress signaling through multistep phosphorelay to a MAPK cascade.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19525025" target=_blank>Schmidt-Heydt M, <i>et al.</i> (2009 Jul 31)</a>. Expression of a gfp gene in Penicillium nordicum under control of the promoter of the ochratoxin A polyketide synthase gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19093150" target=_blank>Liu H, <i>et al.</i> (2009 Mar)</a>. Overexpression of the chitosanase gene in Fusarium solani via Agrobacterium tumefaciens-mediated transformation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19682301" target=_blank>Paul SK, <i>et al.</i> (2009 Sep)</a>. A large complex mediated by Moc1, Moc2 and Cpc2 regulates sexual differentiation in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20946853" target=_blank>Morigasaki S, <i>et al.</i> (2010)</a>. Two-component signaling to the stress MAP kinase cascade in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19756689" target=_blank>Spirek M, <i>et al.</i> (2010 Feb)</a>. SUMOylation is required for normal development of linear elements and wild-type meiotic recombination in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20403971" target=_blank>Keller C, <i>et al.</i> (2010 Jun)</a>. Proteomic and functional analysis of the noncanonical poly(A) polymerase Cid14.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22403715" target=_blank>Bernal M, <i>et al.</i> (2012)</a>. Regulation of fission yeast morphogenesis by PP2A activator pta2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22792266" target=_blank>Etxebeste O, <i>et al.</i> (2012)</a>. GmcA is a putative glucose-methanol-choline oxidoreductase required for the induction of asexual development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22869137" target=_blank>Liu Q, <i>et al.</i> (2012 Aug 1)</a>. Preliminary crystallographic analysis of glyceraldehyde-3-phosphate dehydrogenase 3 from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23122962" target=_blank>Drogat J, <i>et al.</i> (2012 Nov 29)</a>. Cdk11-cyclinL controls the assembly of the RNA polymerase II mediator complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23738021" target=_blank>Zhou X, <i>et al.</i> (2013)</a>. A genome-wide screening of potential target genes to enhance the antifungal activity of micafungin in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23154970" target=_blank>Szilágyi M, <i>et al.</i> (2013 Jan)</a>. Transcriptome changes initiated by carbon starvation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24279325" target=_blank>Ravin NV, <i>et al.</i> (2013 Nov 27)</a>. Genome sequence and analysis of methylotrophic yeast Hansenula polymorpha DL1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24255738" target=_blank>Morigasaki S, <i>et al.</i> (2013 Sep 1)</a>. Phosphorelay-dependent and -independent regulation of MAPKKK by the Mcs4 response regulator in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23994319" target=_blank>Chang PK, <i>et al.</i> (2013 Sep-Oct)</a>. Aspergillus flavus VelB acts distinctly from VeA in conidiation and may coordinate with FluG to modulate sclerotial production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24818994" target=_blank>Jongjitwimol J, <i>et al.</i> (2014)</a>. The S. pombe translation initiation factor eIF4G is Sumoylated and associates with the SUMO protease Ulp2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24634168" target=_blank>Bernal M, <i>et al.</i> (2014 Jun)</a>. Proteome-wide search for PP2A substrates in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24442611" target=_blank>Chen W, <i>et al.</i> (2014 Mar)</a>. Endogenous U2·U5·U6 snRNA complexes in S. pombe are intron lariat spliceosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25373780" target=_blank>Graml V, <i>et al.</i> (2014 Oct 27)</a>. A genomic Multiprocess survey of machineries that control and link cell shape, microtubule organization, and cell-cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25009287" target=_blank>Deng L, <i>et al.</i> (2014 Sep 1)</a>. Megadalton-node assembly by binding of Skb1 to the membrane anchor Slf1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25852654" target=_blank>Gressler M, <i>et al.</i> (2015)</a>. A new high-performance heterologous fungal expression system based on regulatory elements from the Aspergillus terreus terrein gene cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojardín L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26167880" target=_blank>Lipp JJ, <i>et al.</i> (2015 Aug)</a>. SR protein kinases promote splicing of nonconsensus introns.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00285_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00285_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00285_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00285_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00198_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00286<br>EOG8DFN4R<br>TDH1<br>sce:TDH1;TDH2<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NAD-Glyceraldehyde-3-phosphate dehydrogenase ohnolog<br><br>
			<br><b>SGD Description</b><br>Glyceraldehyde-3-phosphate dehydrogenase (GAPDH), isozyme 1; involved in glycolysis and gluconeogenesis; tetramer that catalyzes the reaction of glyceraldehyde-3-phosphate to 1,3 bis-phosphoglycerate; detected in the cytoplasm and cell wall; protein abundance increases in response to DNA replication stress; GAPDH-derived antimicrobial peptides secreted by S. cerevisiae are active against a wide variety of wine-related yeasts and bateria|Glyceraldehyde-3-phosphate dehydrogenase (GAPDH), isozyme 2; involved in glycolysis and gluconeogenesis; tetramer that catalyzes reaction of glyceraldehyde-3-phosphate to 1,3 bis-phosphoglycerate; detected in cytoplasm and cell wall; protein abundance increases in response to DNA replication stress; GAPDH-derived antimicrobial peptides are active against a wide variety of wine-related yeasts and bateria; TDH2 has a paralog, TDH3, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6244283" target=_blank>Holland JP, <i>et al.</i> (1980 Mar 25)</a>. Structural comparison of two nontandemly repeated yeast glyceraldehyde-3-phosphate dehydrogenase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6833300" target=_blank>Holland JP, <i>et al.</i> (1983 Apr 25)</a>. Homologous nucleotide sequences at the 5' termini of messenger RNAs synthesized from the yeast enolase and glyceraldehyde-3-phosphate dehydrogenase gene families. The primary structure of a third yeast glyceraldehyde-3-phosphate dehydrogenase gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1789010" target=_blank>Mountain HA, <i>et al.</i> (1991 Nov)</a>. TDH2 is linked to MET3 on chromosome X of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7737086" target=_blank>Norbeck J, <i>et al.</i> (1995 Jan)</a>. Gene linkage of two-dimensional polyacrylamide gel electrophoresis resolved proteins from isogene families in Saccharomyces cerevisiae by microsequencing of in-gel trypsin generated peptides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9509572" target=_blank>Norbeck J, <i>et al.</i> (1997 Dec)</a>. Two-dimensional electrophoretic separation of yeast proteins using a non-linear wide range (pH 3-10) immobilized pH gradient in the first dimension; reproducibility and evidence for isoelectric focusing of alkaline (pI &amp;gt; 7) proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10930071" target=_blank>Kadokura T, <i>et al.</i> (2000 Jun)</a>. Divergence of glyceraldehyde-3-phosphate dehydrogenase isozymes in Saccharomyces cerevisiae complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11502169" target=_blank>Grandier-Vazeille X, <i>et al.</i> (2001 Aug 21)</a>. Yeast mitochondrial dehydrogenases are associated in a supramolecular complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00286_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00286_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00286_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00286_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00198_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00287<br>EOG8DFN4R<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NAD-Glyceraldehyde-3-phosphate dehydrogenase paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00285<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00287_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00287_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00287_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00287_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00198_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00288<br>EOG8DFN4R<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NAD-Glyceraldehyde-3-phosphate dehydrogenase paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00285<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00288_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00288_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00288_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00288_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00198_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00289<br><br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NAD-Glyceraldehyde-3-phosphate dehydrogenase paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00285<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00289_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00289_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00289_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00289_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00198_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00290<br>EOG8DFN4R<br>GDP1<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NAD(P)-Glyceraldehyde-3-phosphate dehydrogenase paralog<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00290_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00290_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00290_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00290_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00198_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00198.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>