<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01158.html">HOG01158&emsp;&emsp;CRISP family</a><br> 
																	HOG01159&emsp;&emsp;<br> 
																	<a href="HOG01160.html">HOG01160&emsp;&emsp;DEAD box helicase family. DDX19/DBP5 subfamily; DEAD box helicase family. DDX48/FAL1 subfamily; DEAD box helicase family. DDX6/DHH1 subfamily; DEAD box helicase family. eIF4A subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01159.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01159_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01159_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02840<br>EOG86Q57K<br><br>sce:SKI2<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ski complex component and putative RNA helicase; mediates 3'-5' RNA degradation by the cytoplasmic exosome; null mutants have superkiller phenotype of increased viral dsRNAs and are synthetic lethal with mutations in 5'-3' mRNA decay; mutations in the human ortholog, SKIV2L, causes Syndromic diarrhea/Trichohepatoenteric (SD/THE) syndrome<br><br>
			<br><b>PomBase Description</b><br>Ski complex RNA helicase Ski2 (predicted)|TRAMP complex ATP-dependent RNA helicase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=363683" target=_blank>Toh-E A, <i>et al.</i> (1978 Dec)</a>. Chromosomal superkiller mutants of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6371496" target=_blank>Ridley SP, <i>et al.</i> (1984 Apr)</a>. Superkiller mutations in Saccharomyces cerevisiae suppress exclusion of M2 double-stranded RNA by L-A-HN and confer cold sensitivity in the presence of M and L-A-HN.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8321235" target=_blank>Widner WR, <i>et al.</i> (1993 Jul)</a>. Evidence that the SKI antiviral system of Saccharomyces cerevisiae acts by blocking expression of viral mRNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7816623" target=_blank>Lygerou Z, <i>et al.</i> (1994 Dec 11)</a>. The yeast BDF1 gene encodes a transcription factor involved in the expression of a broad class of genes including snRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7739552" target=_blank>Johnson AW, <i>et al.</i> (1995 May)</a>. Synthetic lethality of sep1 (xrn1) ski2 and sep1 (xrn1) ski3 mutants of Saccharomyces cerevisiae is independent of killer virus and suggests a general role for these genes in translation control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7739557" target=_blank>Masison DC, <i>et al.</i> (1995 May)</a>. Decoying the cap- mRNA degradation system by a double-stranded RNA virus and poly(A)- mRNA surveillance by a yeast antiviral system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9482746" target=_blank>Anderson JS, <i>et al.</i> (1998 Mar 2)</a>. The 3' to 5' degradation of yeast mRNAs is a general mechanism for mRNA turnover that requires the SKI2 DEVH box protein and 3' to 5' exonucleases of the exosome complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10922069" target=_blank>Searfoss AM, <i>et al.</i> (2000 Aug 1)</a>. 3' poly(A) is dispensable for translation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10611222" target=_blank>van Hoof A, <i>et al.</i> (2000 Jan)</a>. Yeast exosome mutants accumulate 3'-extended polyadenylated forms of U4 small nuclear RNA and small nucleolar RNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10744028" target=_blank>Brown JT, <i>et al.</i> (2000 Mar)</a>. The yeast antiviral proteins Ski2p, Ski3p, and Ski8p exist as a complex in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11720286" target=_blank>Brown JT, <i>et al.</i> (2001 Nov)</a>. A cis-acting element known to block 3' mRNA degradation enhances expression of polyA-minus mRNA in wild-type yeast cells and phenocopies a ski mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532933" target=_blank>Araki Y, <i>et al.</i> (2001 Sep 3)</a>. Ski7p G protein interacts with the exosome and the Ski complex for 3'-to-5' mRNA decay in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14671320" target=_blank>Kushner DB, <i>et al.</i> (2003 Dec 23)</a>. Systematic, genome-wide identification of host genes affecting replication of a positive-strand RNA virus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12769863" target=_blank>Mitchell P, <i>et al.</i> (2003 May)</a>. An NMD pathway in yeast involving accelerated deadenylation and exosome-mediated 3'--&amp;gt;5' degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15044803" target=_blank>Aloy P, <i>et al.</i> (2004 Mar 26)</a>. Structure-based assembly of protein complexes in yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02840_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02840_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02840_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02840_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02841<br>EOG86Q57K<br><br>sce:MTR4<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATP-dependent 3'-5' RNA helicase of the DExD/H family; involved in nuclear RNA processing and degradation both as a component of TRAMP complex and in TRAMP-independent processes; TRAMP unwinds RNA duplexes, with Mtr4p unwinding activity stimulated by Pap2p/Air2p but not dependent on ongoing polyadenylation; contains an arch domain, with two coiled-coil arms/stalks and a globular fist/KOW domain, which has RNA binding activity and is required for 5.8S rRNA processing<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase, TRAMP complex subunit Mtr4<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in regulation of circadian rhythm and TRAMP complex localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8756671" target=_blank>Liang S, <i>et al.</i> (1996 Sep)</a>. A DEAD-box-family protein is required for nucleocytoplasmic transport of yeast mRNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9463390" target=_blank>de la Cruz J, <i>et al.</i> (1998 Feb 16)</a>. Dob1p (Mtr4p) is a putative ATP-dependent RNA helicase required for the 3' end formation of 5.8S rRNA in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15828860" target=_blank>Vanácová S, <i>et al.</i> (2005 Jun)</a>. A new yeast poly(A) polymerase complex involved in RNA quality control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18681941" target=_blank>Savoldi M, <i>et al.</i> (2008 Oct)</a>. Farnesol induces the transcriptional accumulation of the Aspergillus nidulans Apoptosis-Inducing Factor (AIF)-like mitochondrial oxidoreductase.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02841_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02841_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02841_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02841_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01159.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>