<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01182.html">HOG01182&emsp;&emsp;small GTPase superfamily. Rho family; small GTPase superfamily. Rho family. CDC42 subfamily</a><br> 
																	HOG01183&emsp;&emsp;<br> 
																	<a href="HOG01184.html">HOG01184&emsp;&emsp;small GTPase superfamily. Arf family; small GTPase superfamily. SAR1 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01183.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01183_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01183_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03067<br>EOG87PVQN<br>EOG8K3JB5<br><br>sce:NUP116;NUP100<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>FG-nucleoporin component of central core of the nuclear pore complex; contributes directly to nucleocytoplasmic transport and maintenance of the nuclear pore complex (NPC) permeability barrier; forms a stable association with Nup82p, Gle2p and two other FG-nucleoporins (Nsp1p and Nup159p); NUP116 has a paralog, NUP100, that arose from the whole genome duplication|FG-nucleoporin component of central core of the nuclear pore complex; contributes directly to nucleocytoplasmic transport and maintenance of the nuclear pore complex (NPC) permeability barrier and is involved in gene tethering at the nuclear periphery; NUP100 has a paralog, NUP116, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>nucleoporin Nup98 and Nup96<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular response to DNA damage stimulus and nuclear membrane, nuclear periphery, nuclear pore outer ring, spindle pole body localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1464327" target=_blank>Wimmer C, <i>et al.</i> (1992 Dec)</a>. A new subclass of nucleoporins that functionally interact with nuclear pore protein NSP1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385442" target=_blank>Wente SR, <i>et al.</i> (1992 Nov)</a>. A new family of yeast nuclear pore complex proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8044840" target=_blank>Fabre E, <i>et al.</i> (1994 Jul 29)</a>. Nup145p is required for nuclear export of mRNA and binds homopolymeric RNA in vitro via a novel conserved motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8557738" target=_blank>Iovine MK, <i>et al.</i> (1995 Dec)</a>. The GLFG repetitive region of the nucleoporin Nup116p interacts with Kap95p, an essential yeast nuclear import factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8524308" target=_blank>Sharma K, <i>et al.</i> (1996 Jan)</a>. Yeast nucleoporin mutants are defective in pre-tRNA splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9463388" target=_blank>Bailer SM, <i>et al.</i> (1998 Feb 16)</a>. Nup116p and nup100p are interchangeable through a conserved motif which constitutes a docking site for the mRNA transport factor gle2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9891088" target=_blank>Seedorf M, <i>et al.</i> (1999 Feb)</a>. Interactions between a nuclear transporter and a subset of nuclear pore complex proteins depend on Ran GTPase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10891509" target=_blank>Ho AK, <i>et al.</i> (2000 Aug)</a>. Assembly and preferential localization of Nup116p on the cytoplasmic face of the nuclear pore complex by interaction with Nup82p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10952996" target=_blank>Strässer K, <i>et al.</i> (2000 Aug 21)</a>. Binding of the Mex67p/Mtr2p heterodimer to FXFG, GLFG, and FG repeat nucleoporins is essential for nuclear mRNA export.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10801828" target=_blank>Bailer SM, <i>et al.</i> (2000 Aug 4)</a>. Nup116p associates with the Nup82p-Nsp1p-Nup159p nucleoporin complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10684247" target=_blank>Rout MP, <i>et al.</i> (2000 Feb 21)</a>. The yeast nuclear pore complex: composition, architecture, and transport mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11071906" target=_blank>Stage-Zimmermann T, <i>et al.</i> (2000 Nov)</a>. Factors affecting nuclear export of the 60S ribosomal subunit in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11387327" target=_blank>Allen NP, <i>et al.</i> (2001 Aug 3)</a>. Proteomic analysis of nucleoporin interacting proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11689687" target=_blank>Bailer SM, <i>et al.</i> (2001 Dec)</a>. The Nsp1p carboxy-terminal domain is organized into functionally distinct coiled-coil regions required for assembly of nucleoporin subcomplexes and nucleocytoplasmic transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104765" target=_blank>Strawn LA, <i>et al.</i> (2001 Mar 2)</a>. The GLFG regions of Nup116p and Nup100p serve as binding sites for both Kap95p and Mex67p at the nuclear pore complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12543930" target=_blank>Allen NP, <i>et al.</i> (2002 Dec)</a>. Deciphering networks of protein interactions at the nuclear pore complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372823" target=_blank>Bayliss R, <i>et al.</i> (2002 Dec 27)</a>. GLFG and FxFG nucleoporins bind to overlapping sites on importin-beta.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12376568" target=_blank>Tange Y, <i>et al.</i> (2002 Nov 15)</a>. An evolutionarily conserved fission yeast protein, Ned1, implicated in normal nuclear morphology and chromosome stability, interacts with Dis3, Pim1/RCC1 and an essential nucleoporin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791264" target=_blank>Suntharalingam M, <i>et al.</i> (2003 Jun)</a>. Peering through the pore: nuclear pore complex structure, assembly, and function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12604785" target=_blank>Denning DP, <i>et al.</i> (2003 Mar 4)</a>. Disorder in the nuclear pore complex: the FG repeat regions of nucleoporins are natively unfolded.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12917401" target=_blank>Pyhtila B, <i>et al.</i> (2003 Oct 24)</a>. A gradient of affinity for the karyopherin Kap95p along the yeast nuclear pore complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15116432" target=_blank>Chen XQ, <i>et al.</i> (2004 Apr 30)</a>. Identification of genes encoding putative nucleoporins and transport factors in the fission yeast Schizosaccharomyces pombe: a deletion analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15226438" target=_blank>Baï SW, <i>et al.</i> (2004 Jul)</a>. The fission yeast Nup107-120 complex functionally interacts with the small GTPase Ran/Spi1 and is required for mRNA export, nuclear pore distribution, and proper cell division.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039779" target=_blank>Strawn LA, <i>et al.</i> (2004 Mar)</a>. Minimal nuclear pore complexes define FG repeat domains essential for transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16157682" target=_blank>Thon G, <i>et al.</i> (2005 Dec)</a>. The Clr7 and Clr8 directionality factors and the Pcu4 cullin mediate heterochromatin formation in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17203074" target=_blank>McGuire AT, <i>et al.</i> (2007 Jan 24)</a>. Cex1p is a novel cytoplasmic component of the Saccharomyces cerevisiae nuclear tRNA export machinery.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20970342" target=_blank>Asakawa H, <i>et al.</i> (2010 Nov 9)</a>. Virtual breakdown of the nuclear envelope in fission yeast meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22544723" target=_blank>Sampathkumar P, <i>et al.</i> (2012 Aug)</a>. Atomic structure of the nuclear pore complex targeting domain of a Nup116 homologue from the yeast, Candida glabrata.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22955883" target=_blank>Bilokapic S, <i>et al.</i> (2012 Sep 18)</a>. Molecular basis for Nup37 and ELY5/ELYS recruitment to the nuclear pore complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24637836" target=_blank>Asakawa H, <i>et al.</i> (2014 Mar-Apr)</a>. Characterization of nuclear pore complex components in fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26615217" target=_blank>Hu W, <i>et al.</i> (2015 Nov 27)</a>. Bulk Segregant Analysis Reveals the Genetic Basis of a Natural Trait Variation in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27736299" target=_blank>Moris N, <i>et al.</i> (2016 Nov 16)</a>. A genome-wide screen to identify genes controlling the rate of entry into mitosis in fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03067_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03067_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03067_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03067_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03068<br>EOG8K3JB5<br><br>sce:NUP57<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>FG-nucleoporin component of central core of the nuclear pore complex; contributes directly to nucleocytoplasmic transport and maintenance of the nuclear pore complex (NPC) permeability barrier; found in stable complex with Nic96p and two other FG-nucleoproteins (Nsp1p and Nup49p)<br><br>
			<br><b>PomBase Description</b><br>nucleoporin Nup44<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nuclear periphery, nuclear pore localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7828598" target=_blank>Grandi P, <i>et al.</i> (1995 Jan 3)</a>. Functional interaction of Nic96p with a core nucleoporin complex consisting of Nsp1p, Nup49p and a novel protein Nup57p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9017593" target=_blank>Schlaich NL, <i>et al.</i> (1997 Jan)</a>. In vitro reconstitution of a heterotrimeric nucleoporin complex consisting of recombinant Nsp1p, Nup49p, and Nup57p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9725905" target=_blank>Bucci M, <i>et al.</i> (1998 Sep)</a>. A novel fluorescence-based genetic strategy identifies mutants of Saccharomyces cerevisiae defective for nuclear pore complex assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10684247" target=_blank>Rout MP, <i>et al.</i> (2000 Feb 21)</a>. The yeast nuclear pore complex: composition, architecture, and transport mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11387327" target=_blank>Allen NP, <i>et al.</i> (2001 Aug 3)</a>. Proteomic analysis of nucleoporin interacting proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11689687" target=_blank>Bailer SM, <i>et al.</i> (2001 Dec)</a>. The Nsp1p carboxy-terminal domain is organized into functionally distinct coiled-coil regions required for assembly of nucleoporin subcomplexes and nucleocytoplasmic transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791264" target=_blank>Suntharalingam M, <i>et al.</i> (2003 Jun)</a>. Peering through the pore: nuclear pore complex structure, assembly, and function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12604785" target=_blank>Denning DP, <i>et al.</i> (2003 Mar 4)</a>. Disorder in the nuclear pore complex: the FG repeat regions of nucleoporins are natively unfolded.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15116432" target=_blank>Chen XQ, <i>et al.</i> (2004 Apr 30)</a>. Identification of genes encoding putative nucleoporins and transport factors in the fission yeast Schizosaccharomyces pombe: a deletion analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039779" target=_blank>Strawn LA, <i>et al.</i> (2004 Mar)</a>. Minimal nuclear pore complexes define FG repeat domains essential for transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16987955" target=_blank>Osmani AH, <i>et al.</i> (2006 Dec)</a>. Systematic deletion and mitotic localization of the nuclear pore complex proteins of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20970342" target=_blank>Asakawa H, <i>et al.</i> (2010 Nov 9)</a>. Virtual breakdown of the nuclear envelope in fission yeast meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695164" target=_blank>Das J, <i>et al.</i> (2013 May 21)</a>. Cross-species protein interactome mapping reveals species-specific wiring of stress response pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24637836" target=_blank>Asakawa H, <i>et al.</i> (2014 Mar-Apr)</a>. Characterization of nuclear pore complex components in fission yeast Schizosaccharomyces pombe.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03068_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03068_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03068_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03068_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03069<br>EOG8K3JB5<br><br>sce:NUP145<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Essential protein with distinct roles in two nuclear pore subcomplexes; catalyzes its own proteolytic cleavage in vivo to generate a C-terminal fragment that is a structural component of the Nup84p subcomplex (with roles in NPC biogenesis and localization of genes to the nuclear periphery), and an N-terminal fragment that is one of several FG-nucleoporins within the NPC central core directly responsible for nucleocytoplasmic transport; homologous to human NUP98<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8044840" target=_blank>Fabre E, <i>et al.</i> (1994 Jul 29)</a>. Nup145p is required for nuclear export of mRNA and binds homopolymeric RNA in vitro via a novel conserved motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195299" target=_blank>Wente SR, <i>et al.</i> (1994 Jun)</a>. NUP145 encodes a novel yeast glycine-leucine-phenylalanine-glycine (GLFG) nucleoporin required for nuclear envelope structure.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8524308" target=_blank>Sharma K, <i>et al.</i> (1996 Jan)</a>. Yeast nucleoporin mutants are defective in pre-tRNA splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9305650" target=_blank>Teixeira MT, <i>et al.</i> (1997 Aug 15)</a>. Two functionally distinct domains generated by in vivo cleavage of Nup145p: a novel biogenesis pathway for nucleoporins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542288" target=_blank>Teixeira MT, <i>et al.</i> (1999 Nov 5)</a>. Self-catalyzed cleavage of the yeast nucleoporin Nup145p precursor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10684247" target=_blank>Rout MP, <i>et al.</i> (2000 Feb 21)</a>. The yeast nuclear pore complex: composition, architecture, and transport mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10638763" target=_blank>Galy V, <i>et al.</i> (2000 Jan 6)</a>. Nuclear pore complexes in the organization of silent telomeric chromatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11823431" target=_blank>Lutzmann M, <i>et al.</i> (2002 Feb 1)</a>. Modular self-assembly of a Y-shaped multiprotein complex from seven nucleoporins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791264" target=_blank>Suntharalingam M, <i>et al.</i> (2003 Jun)</a>. Peering through the pore: nuclear pore complex structure, assembly, and function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12604785" target=_blank>Denning DP, <i>et al.</i> (2003 Mar 4)</a>. Disorder in the nuclear pore complex: the FG repeat regions of nucleoporins are natively unfolded.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039779" target=_blank>Strawn LA, <i>et al.</i> (2004 Mar)</a>. Minimal nuclear pore complexes define FG repeat domains essential for transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160040" target=_blank>Hsia KC, <i>et al.</i> (2007 Dec 28)</a>. Architecture of a coat for the nuclear pore membrane.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03069_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03069_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03069_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03069_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03070<br>EOG8K3JB5<br><br>sce:OM45<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mitochondrial outer membrane protein of unknown function; major constituent of the outer membrane, located on the outer (cytosolic) face; interacts with porin (Por1p) and with Om14p; imported via the presequence pathway involving the TOM and TIM23 complexes, then assembled in the outer membrane by Mim1p; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2687271" target=_blank>Yaffe MP, <i>et al.</i> (1989 Dec 15)</a>. The major 45-kDa protein of the yeast mitochondrial outer membrane is not essential for cell growth or mitochondrial function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11502169" target=_blank>Grandier-Vazeille X, <i>et al.</i> (2001 Aug 21)</a>. Yeast mitochondrial dehydrogenases are associated in a supramolecular complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21987634" target=_blank>Hoppins S, <i>et al.</i> (2011 Oct 17)</a>. A mitochondrial-focused genetic interaction map reveals a scaffold-like complex required for inner membrane organization in mitochondria.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03070_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03070_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03070_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03070_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03071<br>EOG8K3JB5<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03071_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03071_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03071_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03071_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03072<br>EOG8K3JB5<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03072_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03072_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03072_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03072_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG03073<br>EOG8K3JB5<br>EOG8TX988<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative alcohol dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03073_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03073_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03073_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03073_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG03074<br>EOG8K3JB5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03074_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03074_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03074_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03074_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG03075<br>EOG8K3JB5<br><br>sce:absent<br><br>Genes: 13
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>Schizosaccharomyces specific protein<br><br>
			<br><b>AspGD Description</b><br>Ortholog of Neosartorya fischeri NRRL 181 : NFIA_114510, Aspergillus wentii : Aspwe1_0173818, Aspwe1_0175438, Aspergillus clavatus NRRL 1 : ACLA_065760 and Aspergillus sydowii : Aspsy1_0049549|Ortholog of A. nidulans FGSC A4 : AN9521, AN0790, A. fumigatus Af293 : Afu5g03750, A. oryzae RIB40 : AO090102000585 and Aspergillus wentii : Aspwe1_0174463<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11595734" target=_blank>Lane S, <i>et al.</i> (2001 Dec 28)</a>. DNA array studies demonstrate convergent regulation of virulence factors by Cph1, Cph2, and Efg1 in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15273122" target=_blank>Karababa M, <i>et al.</i> (2004 Aug)</a>. Comparison of gene expression profiles of Candida albicans azole-resistant clinical isolates and laboratory strains exposed to drugs inducing multidrug transporters.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15554973" target=_blank>Bensen ES, <i>et al.</i> (2004 Dec)</a>. Transcriptional profiling in Candida albicans reveals new adaptive responses to extracellular pH and functions for Rim101p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15470256" target=_blank>Oberholzer U, <i>et al.</i> (2004 Oct)</a>. Functional characterization of myosin I tail regions in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16339080" target=_blank>Enjalbert B, <i>et al.</i> (2006 Feb)</a>. Role of the Hog1 stress-activated protein kinase in the global transcriptional response to stress in the fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18509269" target=_blank>Dib L, <i>et al.</i> (2008 Jun)</a>. The Candida albicans Ddr48 protein is essential for filamentation, stress response, and confers partial antifungal drug resistance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19252121" target=_blank>Hao B, <i>et al.</i> (2009 Apr)</a>. Candida albicans RFX2 encodes a DNA binding protein involved in DNA damage responses, morphogenesis, and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19364863" target=_blank>Shirtliff ME, <i>et al.</i> (2009 Jun)</a>. Farnesol-induced apoptosis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20388546" target=_blank>Alonso-Monge R, <i>et al.</i> (2010 Jul)</a>. The Sko1 protein represses the yeast-to-hypha transition and regulates the oxidative stress response in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20167299" target=_blank>Hernáez ML, <i>et al.</i> (2010 May 7)</a>. Identification of Candida albicans exposed surface proteins in vivo by a rapid proteomic approach.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20870877" target=_blank>Synnott JM, <i>et al.</i> (2010 Nov)</a>. Regulation of the hypoxic response in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21392133" target=_blank>Herrero S, <i>et al.</i> (2011 May)</a>. The Aspergillus nidulans CENP-E kinesin motor KipA interacts with the fungal homologue of the centromere-associated protein CENP-H at the kinetochore.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22523369" target=_blank>Cleary IA, <i>et al.</i> (2012 Jun)</a>. Investigating the function of Ddr48p in Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03075_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03075_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03075_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03075_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01183.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>