<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00039.html">HOG00039&emsp;&emsp;eukaryotic ribosomal protein eL38 family</a><br> 
																	HOG00041&emsp;&emsp;<br> 
																	<a href="HOG00044.html">HOG00044&emsp;&emsp;eukaryotic ribosomal protein eL42 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00041.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00041_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00041_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00041_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00056<br>EOG863XZV<br>RPL40<br>sce:RPL40A;RPL40B<br><br>Genes: 28
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>RPL40 Large ribosome protein subunit; not present in all yeasts<br><br>
			<br><b>SGD Description</b><br>Ubiquitin-ribosomal 60S subunit protein L40A fusion protein; cleaved to yield ubiquitin and ribosomal protein L40A; ubiquitin may facilitate assembly of the ribosomal protein into ribosomes; homologous to mammalian ribosomal protein L40, no bacterial homolog; RPL40A has a paralog, RPL40B, that arose from the whole genome duplication; relative distribution to the nucleus increases upon DNA replication stress|Ubiquitin-ribosomal 60S subunit protein L40B fusion protein; cleaved to yield ubiquitin and ribosomal protein L40B; ubiquitin may facilitate assembly of the ribosomal protein into ribosomes; homologous to mammalian ribosomal protein L40, no bacterial homolog; RPL40B has a paralog, RPL40A, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>ribosomal-ubiquitin fusion protein Ubi1 (predicted)|ribosomal-ubiquitin fusion protein Ubi2<br><br>
			<br><b>AspGD Description</b><br>Cytoplasmic ubiquitin / ribosomal fusion protein<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11788722" target=_blank>Elder RT, <i>et al.</i> (2002 Jan 15)</a>. Involvement of rhp23, a Schizosaccharomyces pombe homolog of the human HHR23A and Saccharomyces cerevisiae RAD23 nucleotide excision repair genes, in cell cycle control and protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19264558" target=_blank>Deshpande GP, <i>et al.</i> (2009 May 1)</a>. Screening a genome-wide S. pombe deletion library identifies novel genes and pathways involved in genome stability maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23738021" target=_blank>Zhou X, <i>et al.</i> (2013)</a>. A genome-wide screening of potential target genes to enhance the antifungal activity of micafungin in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24454826" target=_blank>Fang Y, <i>et al.</i> (2014)</a>. E3 ubiquitin ligase Pub1 is implicated in endocytosis of a GPI-anchored protein Ecm33 in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25373780" target=_blank>Graml V, <i>et al.</i> (2014 Oct 27)</a>. A genomic Multiprocess survey of machineries that control and link cell shape, microtubule organization, and cell-cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28765280" target=_blank>Kampmeyer C, <i>et al.</i> (2017 Sep 15)</a>. The exocyst subunit Sec3 is regulated by a protein quality control pathway.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00056_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00056_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00056_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00056_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00057<br>EOG863XZV<br>RS27<br>sce:RPS31<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>RS27 Small ribosome protein subunit<br><br>
			<br><b>SGD Description</b><br>Fusion protein cleaved to yield ribosomal protein S31 and ubiquitin; ubiquitin may facilitate assembly of the ribosomal protein into ribosomes; interacts genetically with translation factor eIF2B; homologous to mammalian ribosomal protein S27A, no bacterial homolog<br><br>
			<br><b>PomBase Description</b><br>ribosomal-ubiquitin fusion protein Ubi3 (predicted)|ribosomal-ubiquitin fusion protein Ubi5 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Protein similar to ubiquitin UBI1 of A. nidulans<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9559554" target=_blank>Planta RJ, <i>et al.</i> (1998 Mar 30)</a>. The list of cytoplasmic ribosomal proteins of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601260" target=_blank>Arnold RJ, <i>et al.</i> (1999 Dec 24)</a>. The action of N-terminal acetyltransferases on yeast ribosomal proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10684969" target=_blank>Noventa-Jordão MA, <i>et al.</i> (2000 Feb 29)</a>. Molecular characterization of ubiquitin genes from Aspergillus nidulans: mRNA expression on different stress and growth conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10669872" target=_blank>Bao WG, <i>et al.</i> (2000 Mar 15)</a>. The ubiquitin-encoding genes of Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11788722" target=_blank>Elder RT, <i>et al.</i> (2002 Jan 15)</a>. Involvement of rhp23, a Schizosaccharomyces pombe homolog of the human HHR23A and Saccharomyces cerevisiae RAD23 nucleotide excision repair genes, in cell cycle control and protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12237855" target=_blank>Tallada VA, <i>et al.</i> (2002 Sep 30)</a>. Genome-wide search of Schizosaccharomyces pombe genes causing overexpression-mediated cell cycle defects.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20980660" target=_blank>Armache JP, <i>et al.</i> (2010 Nov 16)</a>. Cryo-EM structure and rRNA model of a translating eukaryotic 80S ribosome at 5.5-A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096102" target=_blank>Ben-Shem A, <i>et al.</i> (2011 Dec 16)</a>. The structure of the eukaryotic ribosome at 3.0 Å resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21247416" target=_blank>Gatti L, <i>et al.</i> (2011 Jan 19)</a>. Ubiquitin-proteasome genes as targets for modulation of cisplatin sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22806344" target=_blank>Ryuko S, <i>et al.</i> (2012 Aug)</a>. Genome-wide screen reveals novel mechanisms for regulating cobalt uptake and detoxification in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22650761" target=_blank>Young BD, <i>et al.</i> (2012 Jun 26)</a>. Identification of methylated proteins in the yeast small ribosomal subunit: a role for SPOUT methyltransferases in protein arginine methylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24200810" target=_blank>Fernández IS, <i>et al.</i> (2013 Nov 15)</a>. Molecular architecture of a eukaryotic translational initiation complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24454826" target=_blank>Fang Y, <i>et al.</i> (2014)</a>. E3 ubiquitin ligase Pub1 is implicated in endocytosis of a GPI-anchored protein Ecm33 in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24524803" target=_blank>Ban N, <i>et al.</i> (2014 Feb)</a>. A new system for naming ribosomal proteins.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00057_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00057_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00057_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00057_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00058<br>EOG863XZV<br>UBI4<br>sce:UBI4<br><br>Genes: 25
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Essential for the cellular stress response<br><br>
			<br><b>SGD Description</b><br>Ubiquitin; becomes conjugated to proteins, marking them for selective degradation via the ubiquitin-26S proteasome system; essential for the cellular stress response; encoded as a polyubiquitin precursor comprised of 5 head-to-tail repeats; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>ubiquitin<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN2000/ubi4, A. fumigatus Af293 : Afu4g10350/ubiD, A. oryzae RIB40 : AO090003001182/ubiB, Aspergillus wentii : Aspwe1_0047475 and Aspergillus sydowii : Aspsy1_0053937<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095120" target=_blank>Ozkaynak E, <i>et al.</i> (1984 Dec 13-19)</a>. The yeast ubiquitin gene: head-to-tail repeats encoding a polyubiquitin precursor protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038523" target=_blank>Ozkaynak E, <i>et al.</i> (1987 May)</a>. The yeast ubiquitin genes: a family of natural gene fusions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7615550" target=_blank>Johnson ES, <i>et al.</i> (1995 Jul 21)</a>. A proteolytic pathway that recognizes ubiquitin as a degradation signal.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862120" target=_blank>Spence J, <i>et al.</i> (1995 Mar)</a>. A ubiquitin mutant with specific defects in DNA repair and multiubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8674973" target=_blank>Lee DW, <i>et al.</i> (1996 Apr 15)</a>. Quantitative analysis of gene expression in sexual structures of Aspergillus nidulans by sequencing of 3'-directed cDNA clones.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10406793" target=_blank>Johnston SC, <i>et al.</i> (1999 Jul 15)</a>. Structural basis for the specificity of ubiquitin C-terminal hydrolases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10684969" target=_blank>Noventa-Jordão MA, <i>et al.</i> (2000 Feb 29)</a>. Molecular characterization of ubiquitin genes from Aspergillus nidulans: mRNA expression on different stress and growth conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10669872" target=_blank>Bao WG, <i>et al.</i> (2000 Mar 15)</a>. The ubiquitin-encoding genes of Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11591345" target=_blank>Hamilton KS, <i>et al.</i> (2001 Oct)</a>. Structure of a conjugating enzyme-ubiquitin thiolester intermediate reveals a novel role for the ubiquitin tail.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12628920" target=_blank>Shih SC, <i>et al.</i> (2003 Mar 17)</a>. A ubiquitin-binding motif required for intramolecular monoubiquitylation, the CUE domain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12787503" target=_blank>Kang RS, <i>et al.</i> (2003 May 30)</a>. Solution structure of a CUE-ubiquitin complex reveals a conserved mode of ubiquitin binding.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18923548" target=_blank>Silva EM, <i>et al.</i> (2008 Oct)</a>. Identification of genes differentially expressed in a strain of the mold Aspergillus nidulans carrying a loss-of-function mutation in the palA gene.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00058_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00058_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00058_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00058_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00059<br>EOG863XZV<br>RUB1.00<br>sce:RUB1.00<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Ubiquitin-like protein with similarity to mammalian NEDD8<br><br>
			<br><b>SGD Description</b><br>Ubiquitin-like protein with similarity to mammalian NEDD8; conjugation (neddylation) substrates include the cullins Cdc53p, Rtt101p, and Cul3p; activated by Ula1p and Uba3p (E1 enzyme pair); conjugation mediated by Ubc12p (E2 enzyme)<br><br>
			<br><b>PomBase Description</b><br>ubiquitin-like protein modifier Ned8<br><br>
			<br><b>AspGD Description</b><br>Putative ubiquitin-like protein; repressed by growth on starch and lactate<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9531531" target=_blank>Lammer D, <i>et al.</i> (1998 Apr 1)</a>. Modification of yeast Cdc53p by the ubiquitin-related protein rub1p affects function of the SCFCdc4 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9545234" target=_blank>Liakopoulos D, <i>et al.</i> (1998 Apr 15)</a>. A novel protein modification pathway related to the ubiquitin system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14519104" target=_blank>Laplaza JM, <i>et al.</i> (2004 Jan 15)</a>. Saccharomyces cerevisiae ubiquitin-like protein Rub1 conjugates to cullin proteins Rtt101 and Cul3 in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15988528" target=_blank>Kurz T, <i>et al.</i> (2005 Jun 30)</a>. The conserved protein DCN-1/Dcn1p is required for cullin neddylation in C. elegans and S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22329854" target=_blank>von Zeska Kress MR, <i>et al.</i> (2012 Mar)</a>. The COP9 signalosome counteracts the accumulation of cullin SCF ubiquitin E3 RING ligases during fungal development.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00059_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00059_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00059_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00059_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00060<br>EOG84F4WC<br>CUZ1<br>sce:CUZ1<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Protein with a role in the ubiquitin-proteasome pathway<br><br>
			<br><b>SGD Description</b><br>Protein with a role in the ubiquitin-proteasome pathway; interacts with ubiquitinated protein, Cdc48p and the proteasomal regulatory particle; may protect cells from trivalent metalloid induced proteotoxicity; contains a PACE promoter element and is co-regulated with proteasome subunit genes; AN1-type zinc finger protein, with DHHC and ubiquitin-like domains (UBL); ortholog of ZFAND1, a human gene linked to cancer; protein abundance increases under DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24121501" target=_blank>Sá-Moura B, <i>et al.</i> (2013 Nov 22)</a>. A conserved protein with AN1 zinc finger and ubiquitin-like domains modulates Cdc48 (p97) function in the ubiquitin-proteasome pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24297164" target=_blank>Hanna J, <i>et al.</i> (2014 Jan 17)</a>. Cuz1/Ynl155w, a zinc-dependent ubiquitin-binding protein, protects cells from metalloid-induced proteotoxicity.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00060_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00060_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00060_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00060_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00061<br>EOG84F4WC<br>EOG863XZV<br>TMC1<br>sce:TMC1<br><br>Genes: 27
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>AN1-type zinc finger protein of unknown function<br><br>
			<br><b>SGD Description</b><br>AN1-type zinc finger protein of unknown function; may protect cells from trivalent metalloid induced proteotoxicity; contains a PACE promoter element, a transcriptional profile similar to CUZ1 and RPN2, and decreased expression in an RPN4 mutant; induced by nitrogen limitation and weak acid; ortholog of human AIRAP, which stimulates proteasome activity in response to arsenic; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>zf-AN1 type zinc finger protein<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24297164" target=_blank>Hanna J, <i>et al.</i> (2014 Jan 17)</a>. Cuz1/Ynl155w, a zinc-dependent ubiquitin-binding protein, protects cells from metalloid-induced proteotoxicity.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00061_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00061_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00061_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00061_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00062<br>EOG863XZV<br>EOG8GQNP2<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>RNA polymerase I core factor complex subunit Rrn7<br><br>
			<br><b>PomBase Description</b><br>RNA polymerase I core factor complex subunit Rrn7<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in transcription from RNA polymerase I promoter and RNA polymerase I transcription factor complex, cytosol localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00062_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00062_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00062_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00062_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG00063<br>EOG863XZV<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00063_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00063_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00063_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00063_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00041_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00041.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>