<html>
<head><title>HOG01200</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01199.html">HOG01199&emsp;&emsp;multi antimicrobial extrusion (MATE) (TC 2.A.66.1) family; tyrosyl-DNA phosphodiesterase family</a><br> 
																	HOG01200&emsp;&emsp;<br> 
																	<a href="HOG01201.html">HOG01201&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01200.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01200_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01200_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG01200.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03218<br>EOG8G4FBZ<br><br>sce:ZAP1<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Zinc-regulated transcription factor; binds to zinc-responsive promoters to induce transcription of certain genes in presence of zinc, represses other genes in low zinc; regulates its own transcription; contains seven zinc-finger domains<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular zinc ion homeostasis, positive regulation of transcription, DNA-templated<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17542914" target=_blank>Moreno MA, <i>et al.</i> (2007 Jun)</a>. The regulation of zinc homeostasis by the ZafA transcriptional activator is essential for Aspergillus fumigatus virulence.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03218_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03218_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03218_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03218_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03219<br>EOG89W117<br>EOG8G4FBZ<br><br>sce:MIG1<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Transcription factor involved in glucose repression; sequence specific DNA binding protein containing two Cys2His2 zinc finger motifs; regulated by the SNF1 kinase and the GLC7 phosphatase; regulates filamentous growth along with Mig2p in response to glucose depletion; activated in stochastic pulses of nuclear localization, shuttling between cytosol and nucleus depending on external glucose levels and its phosphorylation state<br><br>
			<br><b>PomBase Description</b><br>transcription factor Scr1<br><br>
			<br><b>AspGD Description</b><br>Transcriptional regulator of the carbon catabolite repression; contains two Cys2His2 zinc finger motifs<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=168071" target=_blank>Bailey C, <i>et al.</i> (1975 Feb 21)</a>. Carbon catabolite repression in Aspergillos nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23491" target=_blank>Kelly JM, <i>et al.</i> (1977 Nov 4)</a>. Increased and decreased sensitivity to carbon catabolite repression of enzymes of acetate metabolism in mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6123545" target=_blank>Kelly JM, <i>et al.</i> (1982 Jan)</a>. The regulation of NADP-linked isocitrate dehydrogenase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24186375" target=_blank>Tollervey DW, <i>et al.</i> (1982 Oct)</a>. Domain-wide, locus-specific suppression of nitrogen metabolite repressed mutations in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6101115" target=_blank>Arst HN Jr, <i>et al.</i> (1984 Sep)</a>. Regulation of gene expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2834622" target=_blank>Lockington R, <i>et al.</i> (1987 Nov)</a>. Regulation of alcR, the positive regulatory gene of the ethanol utilization regulon of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2685573" target=_blank>Katz ME, <i>et al.</i> (1989 Dec)</a>. Isolation and analysis of the acetate regulatory gene, facB, from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2652389" target=_blank>Davis MA, <i>et al.</i> (1989 Jan)</a>. Regulatory genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2571070" target=_blank>Sandeman RA, <i>et al.</i> (1989 Jul)</a>. Isolation of the facA (acetyl-coenzyme A synthetase) and acuE (malate synthase) genes of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2673558" target=_blank>Dowzer CE, <i>et al.</i> (1989 Jun)</a>. Cloning of the creA gene from Aspergillus nidulans: a gene involved in carbon catabolite repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2535501" target=_blank>Dean RA, <i>et al.</i> (1989 Mar)</a>. Production of cell wall-degrading enzymes by Aspergillus nidulans: a model system for fungal pathogenesis of plants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2201871" target=_blank>Arst HN Jr, <i>et al.</i> (1990 May)</a>. An inversion truncating the creA gene of Aspergillus nidulans results in carbon catabolite derepression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2167835" target=_blank>Nehlin JO, <i>et al.</i> (1990 Sep)</a>. Yeast MIG1 repressor is related to the mammalian early growth response and Wilms' tumour finger proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1367013" target=_blank>Felenbok B, <i>et al.</i> (1991 Jan)</a>. The ethanol utilization regulon of Aspergillus nidulans: the alcA-alcR system as a tool for the expression of recombinant proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1922072" target=_blank>Dowzer CE, <i>et al.</i> (1991 Nov)</a>. Analysis of the creA gene, a regulator of carbon catabolite repression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1504607" target=_blank>Scazzocchio C, <i>et al.</i> (1992)</a>. Control of gene expression in the catabolic pathways of Aspergillus nidulans: a personal and biased account.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1625576" target=_blank>Espeso EA, <i>et al.</i> (1992 Jun)</a>. Carbon catabolite repression can account for the temporal pattern of expression of a penicillin biosynthetic gene in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1398125" target=_blank>Punt PJ, <i>et al.</i> (1992 Oct 12)</a>. An upstream activating sequence from the Aspergillus nidulans gpdA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119589" target=_blank>Davis MA, <i>et al.</i> (1993)</a>. Fungal catabolic gene regulation: molecular genetic analysis of the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8359691" target=_blank>Drysdale MR, <i>et al.</i> (1993 Aug 25)</a>. The Aspergillus niger carbon catabolite repressor encoding gene, creA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437566" target=_blank>Sophianopoulou V, <i>et al.</i> (1993 Jan)</a>. Operator derepressed mutations in the proline utilisation gene cluster of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7763860" target=_blank>Hintz WE, <i>et al.</i> (1993 Jul)</a>. A glucose-derepressed promoter for expression of heterologous products in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483416" target=_blank>Kulmburg P, <i>et al.</i> (1993 Mar)</a>. Specific binding sites in the alcR and alcA promoters of the ethanol regulon for the CREA repressor mediating carbon catabolite repression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8404862" target=_blank>Espeso EA, <i>et al.</i> (1993 Oct)</a>. pH regulation is a major determinant in expression of a fungal penicillin biosynthetic gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7872837" target=_blank>De Lucas JR, <i>et al.</i> (1994)</a>. Glucose-induced inactivation of isocitrate lyase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765123" target=_blank>Felenbok B, <i>et al.</i> (1994)</a>. Alcohol metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7847883" target=_blank>Hynes MJ, <i>et al.</i> (1994)</a>. Regulatory circuits of the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765127" target=_blank>Scazzocchio C, <i>et al.</i> (1994)</a>. The proline utilisation pathway, history and beyond.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7872840" target=_blank>van der Veen P, <i>et al.</i> (1994)</a>. Extracellular arabinases in Aspergillus nidulans: the effect of different cre mutations on enzyme levels.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8121406" target=_blank>Bowyer P, <i>et al.</i> (1994 Feb)</a>. Regulation of the expression of the isocitrate lyase gene (acuD) of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7764306" target=_blank>Wey TT, <i>et al.</i> (1994 Jan)</a>. Molecular cloning and sequence analysis of the cellobiohydrolase I gene from Trichoderma koningii G-39.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8313886" target=_blank>Cubero B, <i>et al.</i> (1994 Jan 15)</a>. Two different, adjacent and divergent zinc finger binding sites are necessary for CREA-mediated carbon catabolite repression in the proline gene cluster of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8028581" target=_blank>De Lucas JR, <i>et al.</i> (1994 Jun 15)</a>. Analysis of the regulation of the Aspergillus nidulans acuD gene, encoding isocitrate lyase, by construction of a hybrid promoter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8143847" target=_blank>Espeso EA, <i>et al.</i> (1994 Mar 28)</a>. In vitro binding of the two-finger repressor CreA to several consensus and non-consensus sites at the ipnA upstream region is context dependent.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7988872" target=_blank>González R, <i>et al.</i> (1994 Oct 1)</a>. Transcriptional regulation of the Trichoderma longibrachiatum egl1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076597" target=_blank>Mathieu M, <i>et al.</i> (1994 Sep 1)</a>. The Aspergillus nidulans CREA protein mediates glucose repression of the ethanol regulon at various levels through competition with the ALCR-specific transactivator.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7788717" target=_blank>Ho MC, <i>et al.</i> (1995 Jan)</a>. Sequence analysis of the Aspergillus nidulans pectate lyase pelA gene and evidence for binding of promoter regions to CREA, a regulator of carbon catabolite repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7888172" target=_blank>Sophianopoulou V, <i>et al.</i> (1995 Jan)</a>. Amino acid transporters of lower eukaryotes: regulation, structure and topogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7635218" target=_blank>Fillinger S, <i>et al.</i> (1995 Jul 24)</a>. The basal level of transcription of the alc genes in the ethanol regulon in Aspergillus nidulans is controlled both by the specific transactivator AlcR and the general carbon catabolite repressor CreA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7614370" target=_blank>Kawasaki L, <i>et al.</i> (1995 Mar)</a>. Aspergillus nidulans mutants affected in acetate metabolism isolated as lipid nonutilizers.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521952" target=_blank>Strauss J, <i>et al.</i> (1995 Nov 27)</a>. Cre1, the carbon catabolite repressor protein from Trichoderma reesei.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7496542" target=_blank>van der Veen P, <i>et al.</i> (1995 Sep)</a>. An extreme creA mutation in Aspergillus nidulans has severe effects on D-glucose utilization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7672126" target=_blank>Cassart JP, <i>et al.</i> (1995 Sep 4)</a>. The MIG1 repressor from Kluyveromyces lactis: cloning, sequencing and functional analysis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8625430" target=_blank>Mernitz G, <i>et al.</i> (1996 Apr)</a>. Endoglucanase II (EGII) of Penicillium janthinellum: cDNA sequence, heterologous expression and promotor analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8628232" target=_blank>Katz ME, <i>et al.</i> (1996 Apr 10)</a>. Mutations affecting extracellular protease production in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8753653" target=_blank>Reymond-Cotton P, <i>et al.</i> (1996 Aug)</a>. Expression of the Sclerotinia sclerotiorum polygalacturonase pg1 gene: possible involvement of CREA in glucose catabolite repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8978090" target=_blank>Takashima S, <i>et al.</i> (1996 Dec 15)</a>. Analysis of Cre1 binding sites in the Trichoderma reesei cbh1 upstream region.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8598280" target=_blank>Lee J, <i>et al.</i> (1996 Jan 1)</a>. Regulation of beta-glucosidase biosynthesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8736527" target=_blank>Fillinger S, <i>et al.</i> (1996 May)</a>. A newly identified gene cluster in Aspergillus nidulans comprises five novel genes localized in the alc region that are controlled both by the specific transactivator AlcR and the general carbon-catabolite repressor CreA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987852" target=_blank>Kato M, <i>et al.</i> (1996 Nov)</a>. Sequence-specific binding sites in the Taka-amylase A G2 promoter for the CreA repressor mediating carbon catabolite repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8864218" target=_blank>Shroff RA, <i>et al.</i> (1996 Sep)</a>. Analysis of mutations in the creA gene involved in carbon catabolite repression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9344629" target=_blank>Shroff RA, <i>et al.</i> (1997 Aug)</a>. Null alleles of creA, the regulator of carbon catabolite repression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044292" target=_blank>Murphy RL, <i>et al.</i> (1997 Feb)</a>. Identification of amdX, a new Cys-2-His-2 (C2H2) zinc-finger gene involved in the regulation of the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9066765" target=_blank>Fernández-Cañón JM, <i>et al.</i> (1997 Jan)</a>. The phenylacetic acid uptake system of Aspergillus nidulans is under a creA-independent model of catabolic repression which seems to be mediated by acetyl-CoA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9211795" target=_blank>Screen S, <i>et al.</i> (1997 Jun)</a>. Carbon regulation of the cuticle-degrading enzyme PR1 from Metarhizium anisopliae may involve a trans-acting DNA-binding protein CRR1, a functional equivalent of the Aspergillus nidulans CREA protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9228741" target=_blank>Ruijter GJ, <i>et al.</i> (1997 Jun 15)</a>. Carbon repression in Aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9184237" target=_blank>Gonzalez R, <i>et al.</i> (1997 May 15)</a>. The integration of nitrogen and carbon catabolite repression in Aspergillus nidulans requires the GATA factor AreA and an additional positive-acting element, ADA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308182" target=_blank>Ruijter GJ, <i>et al.</i> (1997 Sep)</a>. Isolation of Aspergillus niger creA mutants and effects of the mutations on expression of arabinases and L-arabinose catabolic enzymes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9546179" target=_blank>Pérez-González JA, <i>et al.</i> (1998 Apr)</a>. Molecular cloning and transcriptional regulation of the Aspergillus nidulans xlnD gene encoding a beta-xylosidase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829933" target=_blank>Stemple CJ, <i>et al.</i> (1998 Dec)</a>. The facC gene of Aspergillus nidulans encodes an acetate-inducible carnitine acetyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9972263" target=_blank>Takashima S, <i>et al.</i> (1998 Dec)</a>. Isolation of the creA gene from the cellulolytic fungus Humicola grisea and analysis of CreA binding sites upstream from the cellulase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9497366" target=_blank>Panozzo C, <i>et al.</i> (1998 Mar 13)</a>. The CreA repressor is the sole DNA-binding protein responsible for carbon catabolite repression of the alcA gene in Aspergillus nidulans via its binding to a couple of specific sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10577470" target=_blank>Agger T, <i>et al.</i> (1999)</a>. Genetically structured modeling of protein production in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216870" target=_blank>Strauss J, <i>et al.</i> (1999 Apr)</a>. The function of CreA, the carbon catabolite repressor of Aspergillus nidulans, is regulated at the transcriptional and post-transcriptional level.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10361038" target=_blank>de la Serna I, <i>et al.</i> (1999 Apr)</a>. Carbon regulation of ribosomal genes in Neurospora crassa occurs by a mechanism which does not require Cre-1, the homologue of the Aspergillus carbon catabolite repressor, CreA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987120" target=_blank>Orejas M, <i>et al.</i> (1999 Jan)</a>. Carbon catabolite repression of the Aspergillus nidulans xlnA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10422579" target=_blank>Martín JF, <i>et al.</i> (1999 Jan-Feb)</a>. Penicillin and cephalosporin biosynthesis: mechanism of carbon catabolite regulation of penicillin production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10403374" target=_blank>Vautard G, <i>et al.</i> (1999 Jun 18)</a>. The glucose repressor CRE1 from Sclerotinia sclerotiorum is functionally related to CREA from Aspergillus nidulans but not to the Mig proteins from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10079330" target=_blank>Dzikowska A, <i>et al.</i> (1999 Mar)</a>. Cloning, characterisation and regulation of the ornithine transaminase (otaA) gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217508" target=_blank>Gielkens M, <i>et al.</i> (1999 Mar)</a>. The abfB gene encoding the major alpha-L-arabinofuranosidase of Aspergillus nidulans: nucleotide sequence, regulation and construction of a disrupted strain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10320578" target=_blank>Hutchings H, <i>et al.</i> (1999 May)</a>. The multiply-regulated gabA gene encoding the GABA permease of Aspergillus nidulans: a score of exons.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760169" target=_blank>Mathieu M, <i>et al.</i> (2000 Apr)</a>. In vivo studies of upstream regulatory cis-acting elements of the alcR gene encoding the transactivator of the ethanol regulon in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11102357" target=_blank>Katz ME, <i>et al.</i> (2000 Dec)</a>. The Aspergillus nidulans xprF gene encodes a hexokinase-like protein involved in the regulation of extracellular proteases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10613888" target=_blank>Cubero B, <i>et al.</i> (2000 Jan)</a>. Metabolite repression and inducer exclusion in the proline utilization gene cluster of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689158" target=_blank>Tudzynski B, <i>et al.</i> (2000 Mar 1)</a>. Carbon catabolite repression in plant pathogenic fungi: isolation and characterization of the Gibberella fujikuroi and Botrytis cinerea creA genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10853770" target=_blank>Vautard-Mey G, <i>et al.</i> (2000 May)</a>. Mutation of a putative AMPK phosphorylation site abolishes the repressor activity but not the nuclear targeting of the fungal glucose regulator CRE1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11131024" target=_blank>Abdallah BM, <i>et al.</i> (2000 Nov)</a>. Glucose does not activate the plasma-membrane-bound H+-ATPase but affects pmaA transcript abundance in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11010922" target=_blank>Bautista LF, <i>et al.</i> (2000 Oct)</a>. Antisense silencing of the creA gene in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11550794" target=_blank>Felenbok B, <i>et al.</i> (2001)</a>. Ethanol catabolism in Aspergillus nidulans: a model system for studying gene regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11277624" target=_blank>Jones IG, <i>et al.</i> (2001 Feb)</a>. ADHII in Aspergillus nidulans is induced by carbon starvation stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11318101" target=_blank>Tani S, <i>et al.</i> (2001 Feb)</a>. Characterization of the amyR gene encoding a transcriptional activator for the amylase genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11234963" target=_blank>Agger T, <i>et al.</i> (2001 Jan)</a>. alpha-Amylase production in high cell density submerged cultivation of Aspergillus oryzae and A. nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139496" target=_blank>Fraser JA, <i>et al.</i> (2001 Jan)</a>. The formamidase gene of Aspergillus nidulans: regulation by nitrogen metabolite repression and transcriptional interference by an overlapping upstream gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11442830" target=_blank>Lockington RA, <i>et al.</i> (2001 Jun)</a>. Carbon catabolite repression in Aspergillus nidulans involves deubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11260466" target=_blank>Hicks J, <i>et al.</i> (2001 Mar)</a>. RcoA has pleiotropic effects on Aspergillus nidulans cellular development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160081" target=_blank>Orejas M, <i>et al.</i> (2001 Mar)</a>. The wide-domain carbon catabolite repressor CreA indirectly controls expression of the Aspergillus nidulans xlnB gene, encoding the acidic endo-beta-(1,4)-xylanase X(24).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11102439" target=_blank>Flipphi M, <i>et al.</i> (2001 Mar 9)</a>. Regulation of the aldehyde dehydrogenase gene (aldA) and its role in the control of the coinducer level necessary for induction of the ethanol utilization pathway in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12109156" target=_blank>Karaffa L, <i>et al.</i> (2002)</a>. Carbon catabolite repression in the regulation of beta-galactosidase activity in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11916668" target=_blank>Hasper AA, <i>et al.</i> (2002 Apr)</a>. EglC, a new endoglucanase from Aspergillus niger with major activity towards xyloglucan.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12471499" target=_blank>Fekete E, <i>et al.</i> (2002 Dec)</a>. Regulation of formation of the intracellular beta-galactosidase activity of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741859" target=_blank>Hynes MJ, <i>et al.</i> (2002 Jan)</a>. Regulation of the acuF gene, encoding phosphoenolpyruvate carboxykinase in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11689252" target=_blank>Agger T, <i>et al.</i> (2002 Jan 18)</a>. Physiological characterisation of recombinant Aspergillus nidulans strains with different creA genotypes expressing A. oryzae alpha-amylase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11848676" target=_blank>Fraser JA, <i>et al.</i> (2002 Mar)</a>. The genes gmdA, encoding an amidase, and bzuA, encoding a cytochrome P450, are required for benzamide utilization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12409103" target=_blank>Lockington RA, <i>et al.</i> (2002 Nov)</a>. Regulation by carbon and nitrogen sources of a family of cellulases in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12242504" target=_blank>de Vries RP, <i>et al.</i> (2002 Sep)</a>. Regulation of the alpha-glucuronidase-encoding gene ( aguA) from Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12485344" target=_blank>Rao U, <i>et al.</i> (2003)</a>. Characterization of AnRP-mediated negative regulation of the xylanase gene, cgxA, from Chaetomium gracile in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12825351" target=_blank>Tonukari NJ, <i>et al.</i> (2003 Apr)</a>. Isolation of the carbon catabolite repressor (CREA) gene from the plant-pathogenic fungus Cochliobolus carbonum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12519784" target=_blank>Flipphi M, <i>et al.</i> (2003 Apr 4)</a>. Onset of carbon catabolite repression in Aspergillus nidulans. Parallel involvement of hexokinase and glucokinase in sugar signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12904552" target=_blank>MacCabe AP, <i>et al.</i> (2003 Aug)</a>. Glucose uptake in germinating Aspergillus nidulans conidia: involvement of the creA and sorA genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12520371" target=_blank>Vautard-Mey G, <i>et al.</i> (2003 Feb)</a>. Carbon and pH modulate the expression of the fungal glucose repressor encoding genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620254" target=_blank>Dzikowska A, <i>et al.</i> (2003 Mar)</a>. Specific induction and carbon/nitrogen repression of arginine catabolism gene of Aspergillus nidulans--functional in vivo analysis of the otaA promoter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14507380" target=_blank>Gómez D, <i>et al.</i> (2003 Oct)</a>. Multiple GATA sites: protein binding and physiological relevance for the regulation of the proline transporter gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15719552" target=_blank>Brakhage AA, <i>et al.</i> (2004)</a>. Regulation of penicillin biosynthesis in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14717659" target=_blank>Vankuyk PA, <i>et al.</i> (2004 Apr 15)</a>. Aspergillus niger mstA encodes a high-affinity sugar/H+ symporter which is regulated in response to extracellular pH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14871945" target=_blank>García I, <i>et al.</i> (2004 Feb)</a>. Chromatin rearrangements in the prnD-prnB bidirectional promoter: dependence on transcription factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12920487" target=_blank>Prathumpai W, <i>et al.</i> (2004 Feb)</a>. The effect of CreA in glucose and xylose catabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14732266" target=_blank>Sims AH, <i>et al.</i> (2004 Feb)</a>. Use of expressed sequence tag analysis and cDNA microarrays of the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15158274" target=_blank>Ilyés H, <i>et al.</i> (2004 Jun 1)</a>. CreA-mediated carbon catabolite repression of beta-galactosidase formation in Aspergillus nidulans is growth rate dependent.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15849404" target=_blank>Kato M, <i>et al.</i> (2005 Apr)</a>. An overview of the CCAAT-box binding factor in filamentous fungi: assembly, nuclear translocation, and transcriptional enhancement.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15813742" target=_blank>Mathieu M, <i>et al.</i> (2005 Apr)</a>. Patterns of nucleosomal organization in the alc regulon of Aspergillus nidulans: roles of the AlcR transcriptional activator and the CreA global repressor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950156" target=_blank>Choi CJ, <i>et al.</i> (2005 Jul)</a>. Isolation and characterization of the Aspergillus nidulans eglC gene encoding a putative beta-1,3-endoglucanase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000711" target=_blank>David H, <i>et al.</i> (2005 Jul)</a>. CreA influences the metabolic fluxes of Aspergillus nidulans during growth on glucose and xylose.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17107606" target=_blank>David H, <i>et al.</i> (2006)</a>. Metabolic network driven analysis of genome-wide transcription data from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16698295" target=_blank>Mogensen J, <i>et al.</i> (2006 Aug)</a>. Transcription analysis using high-density micro-arrays of Aspergillus nidulans wild-type and creA mutant during growth on glucose or ethanol.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16418173" target=_blank>Forment JV, <i>et al.</i> (2006 Mar 31)</a>. Identification of the mstE gene encoding a glucose-inducible, low affinity glucose transporter in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17018254" target=_blank>Emri T, <i>et al.</i> (2006 Oct)</a>. Glucose-mediated repression of autolysis and conidiogenesis in Emericella nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17031028" target=_blank>Nakamura T, <i>et al.</i> (2006 Oct)</a>. Expression profile of amylolytic genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16697631" target=_blank>Kanda S, <i>et al.</i> (2008)</a>. Ambient pH signaling regulates expression of the serine protease gene (spr1) in pine wilt nematode-trapping fungus, Monacrosporium megalosporum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18975147" target=_blank>Emri T, <i>et al.</i> (2008 Dec)</a>. Regulation of autolysis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18512059" target=_blank>Katz ME, <i>et al.</i> (2008 Jul)</a>. The interaction of induction, repression and starvation in the regulation of extracellular proteases in Aspergillus nidulans: evidence for a role for CreA in the response to carbon starvation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18420433" target=_blank>Tamayo EN, <i>et al.</i> (2008 Jun)</a>. CreA mediates repression of the regulatory gene xlnR which controls the production of xylanolytic enzymes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18063396" target=_blank>Roy P, <i>et al.</i> (2008 May)</a>. CreA-mediated repression in Aspergillus nidulans does not require transcriptional auto-regulation, regulated intracellular localisation or degradation of CreA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18661134" target=_blank>Díaz J, <i>et al.</i> (2008 Sep)</a>. Functional analysis of the endoxylanase B (xynB) promoter from Penicillium purpurogenum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19444441" target=_blank>Meijer S, <i>et al.</i> (2009 Aug)</a>. Physiological characterisation of acuB deletion in Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20173036" target=_blank>Murray SL, <i>et al.</i> (2010 Apr)</a>. Metabolic and developmental effects resulting from deletion of the citA gene encoding citrate synthase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23100816" target=_blank>Szilágyi M, <i>et al.</i> (2010 Mar)</a>. MeaB-dependent nutrition sensing regulates autolysis in carbon starved Aspergillus nidulans cultures.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21953444" target=_blank>Szilágyi M, <i>et al.</i> (2011 Dec)</a>. Extracellular proteinase formation in carbon starving Aspergillus nidulans cultures--physiological function and regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22142781" target=_blank>Colabardini AC, <i>et al.</i> (2012 Feb)</a>. Molecular characterization of the Aspergillus nidulans fbxA encoding an F-box protein involved in xylanase induction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22353731" target=_blank>Tamayo-Ramos JA, <i>et al.</i> (2012 Feb 21)</a>. L-rhamnose induction of Aspergillus nidulans α-L-rhamnosidase genes is glucose repressed via a CreA-independent mechanism acting at the level of inducer uptake.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22306112" target=_blank>Sarkar A, <i>et al.</i> (2012 Jul 31)</a>. Differential expression of silent polyketide biosynthesis gene clusters in chemostat cultures of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22445777" target=_blank>Fekete E, <i>et al.</i> (2012 Jun)</a>. Identification of a permease gene involved in lactose utilisation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22252265" target=_blank>Murakoshi Y, <i>et al.</i> (2012 Jun)</a>. Comparison and characterization of α-amylase inducers in Aspergillus nidulans based on nuclear localization of AmyR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22300944" target=_blank>Macios M, <i>et al.</i> (2012 Mar)</a>. The GATA factors AREA and AREB together with the co-repressor NMRA, negatively regulate arginine catabolism in Aspergillus nidulans in response to nitrogen and carbon source.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173087" target=_blank>Georgakopoulos P, <i>et al.</i> (2012 Nov)</a>. SAGA complex components and acetate repression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23800192" target=_blank>Brown NA, <i>et al.</i> (2013 Jun 25)</a>. Functional characterisation of the non-essential protein kinases and phosphatases regulating Aspergillus nidulans hydrolytic enzyme production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24751997" target=_blank>Forment JV, <i>et al.</i> (2014)</a>. High-affinity glucose transport in Aspergillus nidulans is mediated by the products of two related but differentially expressed genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24818689" target=_blank>Cepeda-García C, <i>et al.</i> (2014 Aug)</a>. Direct involvement of the CreA transcription factor in penicillin biosynthesis and expression of the pcbAB gene in Penicillium chrysogenum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24125468" target=_blank>Esperón P, <i>et al.</i> (2014 Dec)</a>. In vitro and in silico analysis of the Aspergillus nidulans DNA-CreA repressor interactions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26690721" target=_blank>de Assis LJ, <i>et al.</i> (2015)</a>. Aspergillus nidulans protein kinase A plays an important role in cellulase production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25889113" target=_blank>Cupertino FB, <i>et al.</i> (2015 Apr)</a>. Regulation of glycogen metabolism by the CRE-1, RCO-1 and RCM-1 proteins in Neurospora crassa. The role of CRE-1 as the central transcriptional regulator.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25519188" target=_blank>Spitzmüller Z, <i>et al.</i> (2015 Mar)</a>. γ-Glutamyl transpeptidase (GgtA) of Aspergillus nidulans is not necessary for bulk degradation of glutathione.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26666972" target=_blank>Bi F, <i>et al.</i> (2016 Oct)</a>. Carbon regulation of environmental pH by secreted small molecules that modulate pathogenicity in phytopathogenic fungi.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03219_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03219_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03219_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03219_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03220<br>EOG8WH74F<br><br>sce:absent<br><br>Genes: 19
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Transcription factor that binds IME1 Upstream Activation Signal (UAS)ru; COM2 transcription is regulated by Haa1p, Sok2p and Zap1p transcriptional activators; may bind the IME1 promoter under all growth conditions to negatively regulate its transcription in the absence of a positive regulator that binds more effectively; repressor activity may depend on phosphorylation by PKA; C. albicans homolog (MNL1) plays a role in adaptation to stress<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular response to calcium ion, cellular response to heat, cellular response to oxidative stress, positive regulation of cellular response to oxidative stress and cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11918796" target=_blank>Han KH, <i>et al.</i> (2002 Mar)</a>. Osmotic stress-coupled maintenance of polar growth in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15470239" target=_blank>Nicholls S, <i>et al.</i> (2004 Oct)</a>. Msn2- and Msn4-like transcription factors play no obvious roles in the stress responses of the fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18653474" target=_blank>Ramsdale M, <i>et al.</i> (2008 Oct)</a>. MNL1 regulates weak acid-induced stress responses of the fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18703157" target=_blank>Miskei M, <i>et al.</i> (2009 Mar)</a>. Annotation of stress-response proteins in the aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22091779" target=_blank>Dyer PS, <i>et al.</i> (2012 Jan)</a>. Sexual development and cryptic sexuality in fungi: insights from Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24532783" target=_blank>Lee MK, <i>et al.</i> (2014 May)</a>. NsdD is a key repressor of asexual development in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03220_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03220_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03220_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03220_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03221<br>EOG8G4FBZ<br>EOG8WH74F<br><br>sce:MSN2;MSN4<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Stress-responsive transcriptional activator; activated in stochastic pulses of nuclear localization in response to various stress conditions; binds DNA at stress response elements of responsive genes; relative distribution to nucleus increases upon DNA replication stress|Stress-responsive transcriptional activator; activated in stochastic pulses of nuclear localization in response to various stress conditions; binds DNA at stress response elements of responsive genes, inducing gene expression; involved in diauxic shift<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8321194" target=_blank>Estruch F, <i>et al.</i> (1993 Jul)</a>. Two homologous zinc finger genes identified by multicopy suppression in a SNF1 protein kinase mutant of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8641288" target=_blank>Martínez-Pastor MT, <i>et al.</i> (1996 May 1)</a>. The Saccharomyces cerevisiae zinc finger proteins Msn2p and Msn4p are required for transcriptional induction through the stress response element (STRE).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12090248" target=_blank>Kaida D, <i>et al.</i> (2002 Jun)</a>. Yeast Whi2 and Psr1-phosphatase form a complex and regulate STRE-mediated gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12732613" target=_blank>Jacquet M, <i>et al.</i> (2003 May 12)</a>. Oscillatory nucleocytoplasmic shuttling of the general stress response transcriptional activators Msn2 and Msn4 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27618436" target=_blank>Piskacek M, <i>et al.</i> (2016)</a>. The 9aaTAD Transactivation Domains: From Gal4 to p53.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03221_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03221_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03221_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03221_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03222<br>EOG8D7WPT<br>EOG8G4FBZ<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03222_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03222_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03222_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03222_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03223<br>EOG8WH74F<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted nucleic acid binding activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03223_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03223_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03223_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03223_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG03224<br>EOG8G4FBZ<br>EOG8X0KCX<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03224_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03224_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03224_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03224_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG03225<br>EOG8D7WPT<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03225_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03225_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03225_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03225_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG03226<br>EOG8G4FBZ<br><br>sce:MIG2<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Zinc finger transcriptional repressor; cooperates with Mig1p in glucose-induced gene repression; under low glucose conditions relocalizes to mitochondrion, where it interacts with Ups1p, antagonizes mitochondrial fission factor Dnm1p, indicative of a role in mitochondrial fusion or regulating morphology; regulates filamentous growth in response to glucose depletion; activated in stochastic pulses of nuclear localization in response to low glucose<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8813766" target=_blank>Kail M, <i>et al.</i> (1996 Jun 30)</a>. Lambda clone B22 contains a 7676 bp genomic fragment of Saccharomyces cerevisiae chromosome VII spanning the VAM7-SPM2 intergenic region and containing three novel transcribed open reading frames.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8756637" target=_blank>Lutfiyya LL, <i>et al.</i> (1996 Sep)</a>. Two zinc-finger-containing repressors are responsible for glucose repression of SUC2 expression.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03226_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03226_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03226_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03226_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG03227<br>EOG8WH74F<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03227_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03227_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03227_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03227_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG03228<br>EOG8G4FBZ<br><br>sce:MIG3<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Transcriptional regulator; partially nonfunctional in S288C strains but has a major role in catabolite repression and ethanol response in some other strains; involved in response to toxic agents; phosphorylation by Snf1p or the Mec1p pathway inactivates Mig3p, allowing induction of damage response genesenvironment<br><br>
			<br><b>PomBase Description</b><br>transcription factor Rsv1<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9372444" target=_blank>Hao Z, <i>et al.</i> (1997 Oct)</a>. A zinc finger protein required for stationary phase viability in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12161108" target=_blank>Lombardía LJ, <i>et al.</i> (2002 Aug)</a>. Genome-wide analysis of yeast transcription upon calcium shortage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14993292" target=_blank>Dubacq C, <i>et al.</i> (2004 Mar)</a>. The protein kinase Snf1 is required for tolerance to the ribonucleotide reductase inhibitor hydroxyurea.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17927811" target=_blank>Mata J, <i>et al.</i> (2007)</a>. Transcriptional regulatory network for sexual differentiation in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695302" target=_blank>Vachon L, <i>et al.</i> (2013 Aug)</a>. Functional characterization of fission yeast transcription factors by overexpression analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26167880" target=_blank>Lipp JJ, <i>et al.</i> (2015 Aug)</a>. SR protein kinases promote splicing of nonconsensus introns.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25411338" target=_blank>Saitoh S, <i>et al.</i> (2015 Jan 15)</a>. Mechanisms of expression and translocation of major fission yeast glucose transporters regulated by CaMKK/phosphatases, nuclear shuttling, and TOR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28342076" target=_blank>Pataki E, <i>et al.</i> (2017 Jun)</a>. Schizosaccharomyces pombe rsv1 Transcription Factor and its Putative Homologues Preserved their Functional Homology and are Evolutionarily Conserved.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28945192" target=_blank>Guydosh NR, <i>et al.</i> (2017 Sep 25)</a>. Regulated Ire1-dependent mRNA decay requires no-go mRNA degradation to maintain endoplasmic reticulum homeostasis in &lt;i&gt;S. pombe&lt;/i&gt;.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03228_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03228_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03228_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03228_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG03229<br>EOG8WH74F<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>transcription factor Hsr1<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18003976" target=_blank>Chen D, <i>et al.</i> (2008 Jan)</a>. Multiple pathways differentially regulate global oxidative stress responses in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21340088" target=_blank>Arita Y, <i>et al.</i> (2011 May)</a>. Microarray-based target identification using drug hypersensitive fission yeast expressing ORFeome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22806344" target=_blank>Ryuko S, <i>et al.</i> (2012 Aug)</a>. Genome-wide screen reveals novel mechanisms for regulating cobalt uptake and detoxification in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22212525" target=_blank>Ohtsuka H, <i>et al.</i> (2012 Jan)</a>. Chronological lifespan extension by Ecl1 family proteins depends on Prr1 response regulator in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695302" target=_blank>Vachon L, <i>et al.</i> (2013 Aug)</a>. Functional characterization of fission yeast transcription factors by overexpression analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03229_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03229_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03229_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03229_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01200_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01200.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>