<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG03516.html">HOG03516&emsp;&emsp;class I-like SAM-binding methyltransferase superfamily. Protein arginine N-methyltransferase family</a><br> 
																	HOG03517&emsp;&emsp;<br> 
																	<a href="HOG03518.html">HOG03518&emsp;&emsp;BLI1 family</a><br><br> 
																	<a href="../sequences/hog/HOG03517.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG03517_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG03517_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG03517_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG03517_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG07516<br>EOG88SF81<br><br>sce:SKI3<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ski complex component and TPR protein; mediates 3'-5' RNA degradation by the cytoplasmic exosome; null mutants have superkiller phenotype of increased viral dsRNAs and are synthetic lethal with mutations in 5'-3' mRNA decay; mutations in the human ortholog, TTC37, causes Syndromic diarrhea/Trichohepatoenteric (SD/THE) syndrome<br><br>
			<br><b>PomBase Description</b><br>Ski complex TPR repeat subunit Ski3 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=363683" target=_blank>Toh-E A, <i>et al.</i> (1978 Dec)</a>. Chromosomal superkiller mutants of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6371496" target=_blank>Ridley SP, <i>et al.</i> (1984 Apr)</a>. Superkiller mutations in Saccharomyces cerevisiae suppress exclusion of M2 double-stranded RNA by L-A-HN and confer cold sensitivity in the presence of M and L-A-HN.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2660461" target=_blank>Rhee SK, <i>et al.</i> (1989 May-Jun)</a>. Structure and nuclear localization signal of the SKI3 antiviral protein of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404612" target=_blank>Sikorski RS, <i>et al.</i> (1990 Jan 26)</a>. A repeating amino acid motif in CDC23 defines a family of proteins and a new relationship among genes required for mitosis and RNA synthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7739552" target=_blank>Johnson AW, <i>et al.</i> (1995 May)</a>. Synthetic lethality of sep1 (xrn1) ski2 and sep1 (xrn1) ski3 mutants of Saccharomyces cerevisiae is independent of killer virus and suggests a general role for these genes in translation control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7739557" target=_blank>Masison DC, <i>et al.</i> (1995 May)</a>. Decoying the cap- mRNA degradation system by a double-stranded RNA virus and poly(A)- mRNA surveillance by a yeast antiviral system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9482746" target=_blank>Anderson JS, <i>et al.</i> (1998 Mar 2)</a>. The 3' to 5' degradation of yeast mRNAs is a general mechanism for mRNA turnover that requires the SKI2 DEVH box protein and 3' to 5' exonucleases of the exosome complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10744028" target=_blank>Brown JT, <i>et al.</i> (2000 Mar)</a>. The yeast antiviral proteins Ski2p, Ski3p, and Ski8p exist as a complex in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11720286" target=_blank>Brown JT, <i>et al.</i> (2001 Nov)</a>. A cis-acting element known to block 3' mRNA degradation enhances expression of polyA-minus mRNA in wild-type yeast cells and phenocopies a ski mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532933" target=_blank>Araki Y, <i>et al.</i> (2001 Sep 3)</a>. Ski7p G protein interacts with the exosome and the Ski complex for 3'-to-5' mRNA decay in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14671320" target=_blank>Kushner DB, <i>et al.</i> (2003 Dec 23)</a>. Systematic, genome-wide identification of host genes affecting replication of a positive-strand RNA virus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15044803" target=_blank>Aloy P, <i>et al.</i> (2004 Mar 26)</a>. Structure-based assembly of protein complexes in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22624651" target=_blank>Navarro FJ, <i>et al.</i> (2012 May 24)</a>. A systematic screen reveals new elements acting at the G2/M cell cycle control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG07516_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG07516_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG07516_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG07516_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG03517_phobius.html#FOG07516">Phobius transmembrane predictions</a>
										<br>13 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG07517<br>EOG88SF81<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG07517_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG07517_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG07517_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG07517_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG03517_phobius.html#FOG07517">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG03517.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>