<html>
<head><title>HOG00599</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>

																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG00599.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01424<br>EOG8PNVZJ<br><br>sce:absent<br><br>Genes: 53
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>acetamidase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative amidase/acetamidase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01424_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01424_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01424_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01424_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01425<br>EOG8PNVZJ<br><br>sce:AMD2<br><br>Genes: 44
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative amidase<br><br>
			<br><b>PomBase Description</b><br>fatty acid amide hydrolase Fah1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative amidase/acetamidase|Putative amidase/acetamidase|Putative amidase/acetamidase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4097637" target=_blank>Hynes MJ, <i>et al.</i> (1970)</a>. The genetic analysis of regulation of amidase synthesis in Aspergillus nidulans. II. Mutants resistant to fluoroacetamide.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1089206" target=_blank>Hynes MJ, <i>et al.</i> (1975 Jan 17)</a>. A cis-dominant regulatory mutation affecting enzyme induction in the eukaryote Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1104771" target=_blank>Hynes MJ, <i>et al.</i> (1975 Nov)</a>. Amide utilization in Aspergillus nidulans: evidence for a third amidase enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=778641" target=_blank>Arst HN Jr, <i>et al.</i> (1976 Jul 15)</a>. Integrator gene in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19418" target=_blank>Hynes MJ, <i>et al.</i> (1977 Sep)</a>. Induction of the acetamidase of Aspergillus nidulans by acetate metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=353500" target=_blank>Hynes MJ, <i>et al.</i> (1978 Apr 25)</a>. Multiple independent control mechanisms affecting the acetamidase of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=368568" target=_blank>Hynes MJ, <i>et al.</i> (1978 Oct 25)</a>. An "up-promotor" mutation affecting the acetamidase of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=378765" target=_blank>Hynes MJ, <i>et al.</i> (1979 Mar)</a>. Fine-structure mapping of the acetamidase structural gene and its controlling region in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6759305" target=_blank>Hynes MJ, <i>et al.</i> (1982 Oct)</a>. A cis-dominant mutation in aspergillus nidulans affecting the expression of the amdS gene in the presence of mutations in the unlinked gene, amdA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6353203" target=_blank>Hynes MJ, <i>et al.</i> (1983 Aug)</a>. Isolation of genomic clones containing the amdS gene of Aspergillus nidulans and their use in the analysis of structural and regulatory mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6101115" target=_blank>Arst HN Jr, <i>et al.</i> (1984 Sep)</a>. Regulation of gene expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3916805" target=_blank>Atkinson PW, <i>et al.</i> (1985)</a>. Identification of the aciA gene controlled by the amdA regulatory gene in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3916728" target=_blank>Wernars K, <i>et al.</i> (1985)</a>. Gene amplification in Aspergillus nidulans by transformation with vectors containing the amdS gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3894007" target=_blank>Kelly JM, <i>et al.</i> (1985 Feb)</a>. Transformation of Aspergillus niger by the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3543620" target=_blank>Wernars K, <i>et al.</i> (1986 Nov)</a>. Genetic analysis of Aspergillus nidulans AmdS+ transformants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2945991" target=_blank>Fincham J, <i>et al.</i> (1986 Oct 30-Nov 5)</a>. Genetics of filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3551935" target=_blank>Hynes MJ, <i>et al.</i> (1986 Sep)</a>. The amdS gene of Aspergillus nidulans: control by multiple regulatory signals.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3131026" target=_blank>Beri RK, <i>et al.</i> (1987)</a>. Transformation of Penicillium chrysogenum using the Aspergillus nidulans amdS gene as a dominant selective marker.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3036667" target=_blank>Corrick CM, <i>et al.</i> (1987)</a>. The nucleotide sequence of the amdS gene of Aspergillus nidulans and the molecular characterization of 5' mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2835171" target=_blank>Kelly JM, <i>et al.</i> (1987)</a>. Multiple copies of the amdS gene of Aspergillus nidulans cause titration of trans-acting regulatory proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3127274" target=_blank>Penttil√§ M, <i>et al.</i> (1987)</a>. A versatile transformation system for the cellulolytic filamentous fungus Trichoderma reesei.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038698" target=_blank>Rodriguez RJ, <i>et al.</i> (1987)</a>. Selectable genes for transformation of the fungal plant pathogen Glomerella cingulata f. sp. phaseoli (Colletotrichum lindemuthianum).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3312958" target=_blank>Wernars K, <i>et al.</i> (1987 Aug)</a>. Cotransformation of Aspergillus nidulans: a tool for replacing fungal genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3131191" target=_blank>Kolar M, <i>et al.</i> (1988)</a>. Transformation of Penicillium chrysogenum using dominant selection markers and expression of an Escherichia coli lacZ fusion gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3062382" target=_blank>Andrianopoulos A, <i>et al.</i> (1988 Aug)</a>. Cloning and analysis of the positively acting regulatory gene amdR from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3043184" target=_blank>Hynes MJ, <i>et al.</i> (1988 Jun)</a>. Identification of the sites of action for regulatory genes controlling the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2838387" target=_blank>Davis MA, <i>et al.</i> (1988 Mar 31)</a>. An amdS-lacZ fusion for studying gene regulation in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2652389" target=_blank>Davis MA, <i>et al.</i> (1989 Jan)</a>. Regulatory genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670668" target=_blank>Diallinas G, <i>et al.</i> (1989 Jun)</a>. A gene coding for the uric acid-xanthine permease of Aspergillus nidulans: inactivational cloning, characterization, and sequence of a cis-acting mutation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2505051" target=_blank>Richardson IB, <i>et al.</i> (1989 May)</a>. Cloning and molecular characterisation of the amdR controlled gatA gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2263500" target=_blank>Chang TH, <i>et al.</i> (1990 Dec 11)</a>. Identification of a putative amidase gene in yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187153" target=_blank>Katz ME, <i>et al.</i> (1990 Feb)</a>. A mutation affecting amdS expression in Aspergillus nidulans contains a triplication of a cis-acting regulatory sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2274031" target=_blank>Debets AJ, <i>et al.</i> (1990 Jul)</a>. Genetic analysis of amdS transformants of Aspergillus niger and their use in chromosome mapping.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2277644" target=_blank>Debets AJ, <i>et al.</i> (1990 Nov)</a>. An electrophoretic karyotype of Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840550" target=_blank>Gomi K, <i>et al.</i> (1991 Dec 1)</a>. Cloning and molecular characterization of the acetamidase-encoding gene (amdS) from Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2041742" target=_blank>van Heeswijck R, <i>et al.</i> (1991 May 25)</a>. The amdR product and a CCAAT-binding factor bind to adjacent, possibly overlapping DNA sequences in the promoter region of the Aspergillus nidulans amdS gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1745237" target=_blank>Lloyd AT, <i>et al.</i> (1991 Nov)</a>. Codon usage in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1494342" target=_blank>Yamashiro CT, <i>et al.</i> (1992 Dec)</a>. A dominant selectable marker that is meiotically stable in Neurospora crassa: the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452021" target=_blank>Wang XW, <i>et al.</i> (1992 Dec 1)</a>. Structural and functional analysis of the amdR regulatory gene of Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729609" target=_blank>Richardson IB, <i>et al.</i> (1992 Jan)</a>. Molecular characterization of the lam locus and sequences involved in regulation by the AmdR protein of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1465107" target=_blank>Saleeba JA, <i>et al.</i> (1992 Nov)</a>. Characterization of the amdA-regulated aciA gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1435733" target=_blank>Littlejohn TG, <i>et al.</i> (1992 Oct)</a>. Analysis of the site of action of the amdR product for regulation of the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1398125" target=_blank>Punt PJ, <i>et al.</i> (1992 Oct 12)</a>. An upstream activating sequence from the Aspergillus nidulans gpdA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119589" target=_blank>Davis MA, <i>et al.</i> (1993)</a>. Fungal catabolic gene regulation: molecular genetic analysis of the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8257101" target=_blank>Marzluf GA, <i>et al.</i> (1993)</a>. Regulation of sulfur and nitrogen metabolism in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765128" target=_blank>Hynes MJ, <i>et al.</i> (1994)</a>. Regulation of acetamide utilization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7847883" target=_blank>Hynes MJ, <i>et al.</i> (1994)</a>. Regulatory circuits of the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7915967" target=_blank>Varga J, <i>et al.</i> (1994 Apr)</a>. Assignment of RFLP, RAPD and isoenzyme markers to Aspergillus nidulans chromosomes, using chromosome-substituted segregants of a hybrid of A. nidulans and A. quadrilineatus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862093" target=_blank>Bonnefoy N, <i>et al.</i> (1995 Jan 20)</a>. Yeast proteins can activate expression through regulatory sequences of the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596297" target=_blank>Lints R, <i>et al.</i> (1995 Mar)</a>. The positively acting amdA gene of Aspergillus nidulans encodes a protein with two C2H2 zinc-finger motifs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655534" target=_blank>Davis MA, <i>et al.</i> (1996 Jun)</a>. The tamA gene of Aspergillus nidulans contains a putative zinc cluster motif which is not required for gene function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9126617" target=_blank>Andrianopoulos A, <i>et al.</i> (1997 Feb)</a>. The amdA regulatory gene of Aspergillus nidulans: characterization of gain-of-function mutations and identification of binding sites for the gene product.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044292" target=_blank>Murphy RL, <i>et al.</i> (1997 Feb)</a>. Identification of amdX, a new Cys-2-His-2 (C2H2) zinc-finger gene involved in the regulation of the amdS gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236780" target=_blank>Bird D, <i>et al.</i> (1997 Jun)</a>. Gene targeting is locus dependent in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9108273" target=_blank>Kato M, <i>et al.</i> (1997 Mar 26)</a>. An Aspergillus nidulans nuclear protein, AnCP, involved in enhancement of Taka-amylase A gene expression, binds to the CCAAT-containing taaG2, amdS, and gatA promoters.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197408" target=_blank>Todd RB, <i>et al.</i> (1997 May 20)</a>. The acetate regulatory gene facB of Aspergillus nidulans encodes a Zn(II)2Cys6 transcriptional activator.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367656" target=_blank>Todd RB, <i>et al.</i> (1997 Oct)</a>. Molecular characterization of mutants of the acetate regulatory gene facB of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524126" target=_blank>Todd RB, <i>et al.</i> (1998 Apr 1)</a>. FacB, the Aspergillus nidulans activator of acetate utilization genes, binds dissimilar DNA sequences.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829933" target=_blank>Stemple CJ, <i>et al.</i> (1998 Dec)</a>. The facC gene of Aspergillus nidulans encodes an acetate-inducible carnitine acetyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10052139" target=_blank>Kanemori Y, <i>et al.</i> (1999 Jan)</a>. Insertion analysis of putative functional elements in the promoter region of the Aspergillus oryzae Taka-amylase A gene (amyB) using a heterologous Aspergillus nidulans amdS-lacZ fusion gene system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9858535" target=_blank>Steidl S, <i>et al.</i> (1999 Jan)</a>. AnCF, the CCAAT binding complex of Aspergillus nidulans, contains products of the hapB, hapC, and hapE genes and is required for activation by the pathway-specific regulatory gene amdR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10441450" target=_blank>Brakhage AA, <i>et al.</i> (1999 Jul-Aug)</a>. HAP-Like CCAAT-binding complexes in filamentous fungi: implications for biotechnology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10490592" target=_blank>Narendja FM, <i>et al.</i> (1999 Oct)</a>. AnCF, the CCAAT binding complex of Aspergillus nidulans, is essential for the formation of a DNase I-hypersensitive site in the 5' region of the amdS gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10471703" target=_blank>Small AJ, <i>et al.</i> (1999 Sep)</a>. The TamA protein fused to a DNA-binding domain can recruit AreA, the major nitrogen regulatory protein, to activate gene expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10779325" target=_blank>Espeso EA, <i>et al.</i> (2000 May)</a>. On the mechanism by which alkaline pH prevents expression of an acid-expressed gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11243777" target=_blank>Steidl S, <i>et al.</i> (2001 Mar 2)</a>. The Aspergillus nidulans multimeric CCAAT binding complex AnCF is negatively autoregulated via its hapB subunit gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11683268" target=_blank>Han S, <i>et al.</i> (2001 Oct)</a>. Complex control of the developmental regulatory locus brlA in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11848676" target=_blank>Fraser JA, <i>et al.</i> (2002 Mar)</a>. The genes gmdA, encoding an amidase, and bzuA, encoding a cytochrome P450, are required for benzamide utilization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15045526" target=_blank>Michielse CB, <i>et al.</i> (2004 Jun)</a>. The Aspergillus nidulans amdS gene as a marker for the identification of multicopy T-DNA integration events in Agrobacterium-mediated transformation of Aspergillus awamori.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15849404" target=_blank>Kato M, <i>et al.</i> (2005 Apr)</a>. An overview of the CCAAT-box binding factor in filamentous fungi: assembly, nuclear translocation, and transcriptional enhancement.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16215172" target=_blank>Todd RB, <i>et al.</i> (2005 Oct)</a>. Nuclear accumulation of the GATA factor AreA in response to complete nitrogen starvation by regulation of nuclear export.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17107606" target=_blank>David H, <i>et al.</i> (2006)</a>. Metabolic network driven analysis of genome-wide transcription data from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16698295" target=_blank>Mogensen J, <i>et al.</i> (2006 Aug)</a>. Transcription analysis using high-density micro-arrays of Aspergillus nidulans wild-type and creA mutant during growth on glucose or ethanol.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020584" target=_blank>Caddick MX, <i>et al.</i> (2006 Oct)</a>. Opposing signals differentially regulate transcript stability in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17854403" target=_blank>Wong KH, <i>et al.</i> (2007 Oct)</a>. Transcriptional control of nmrA by the bZIP transcription factor MeaB reveals a new level of nitrogen regulation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245853" target=_blank>Damveld RA, <i>et al.</i> (2008 Feb)</a>. A novel screening method for cell wall mutants in Aspergillus niger identifies UDP-galactopyranose mutase as an important protein in fungal cell wall biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21166768" target=_blank>Ganatra MB, <i>et al.</i> (2011 Mar)</a>. A set of aspartyl protease-deficient strains for improved expression of heterologous proteins in Kluyveromyces lactis.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01425_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01425_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01425_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01425_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG01426<br>EOG8PNVZJ<br><br>sce:absent<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative amidase/acetamidase|Putative amidase/acetamidase|Has domain(s) with predicted carbon-nitrogen ligase activity, with glutamine as amido-N-donor activity|Putative amidase/acetamidase|Putative amidase/acetamidase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16698295" target=_blank>Mogensen J, <i>et al.</i> (2006 Aug)</a>. Transcription analysis using high-density micro-arrays of Aspergillus nidulans wild-type and creA mutant during growth on glucose or ethanol.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01426_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01426_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01426_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01426_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG01427<br>EOG8866V8<br>EOG8V41Q9<br>EOG8X3FH5<br><br>sce:VBA3;VBA5<br><br>Genes: 76
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>amino acid transporter into [v]<br><br>
			<br><b>SGD Description</b><br>Permease of basic amino acids in the vacuolar membrane; VBA3 has a paralog, VBA5, that arose from a segmental duplication|Plasma membrane protein of the Major Facilitator Superfamily (MFS); involved in amino acid uptake and drug sensitivity; VBA5 has a paralog, VBA3, that arose from a segmental duplication<br><br>
			<br><b>PomBase Description</b><br>transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted transporter activity, role in transmembrane transport and integral component of membrane localization|Putative MFS drug transporter|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have endoplasmic reticulum localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have endoplasmic reticulum localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15572352" target=_blank>Shimazu M, <i>et al.</i> (2005 Feb 11)</a>. A family of basic amino acid transporters of the vacuolar membrane from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22329759" target=_blank>Lo HC, <i>et al.</i> (2012 Mar 14)</a>. Two separate gene clusters encode the biosynthetic pathway for the meroterpenoids austinol and dehydroaustinol in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23264642" target=_blank>Garzia A, <i>et al.</i> (2013 Feb)</a>. Transcriptional changes in the transition from vegetative cells to asexual development in the model fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24463365" target=_blank>Rallis C, <i>et al.</i> (2014 Feb 15)</a>. Systematic screen for mutants resistant to TORC1 inhibition in fission yeast reveals genes involved in cellular ageing and growth.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01427_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01427_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01427_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01427_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>76 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG01428<br>EOG8866V8<br>EOG8QBZQH<br><br>sce:absent<br><br>Genes: 28
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01428_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01428_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01428_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01428_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>28 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG01429<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 11
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function; expression repressed by tunicamycin and DTT|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have role in arginine transport and plasma membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24690493" target=_blank>Colabardini AC, <i>et al.</i> (2014 Apr 1)</a>. Functional characterization of a xylose transporter in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01429_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01429_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01429_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01429_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>11 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG01430<br>EOG8866V8<br>EOG8V41Q9<br><br>sce:SGE1<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Plasma membrane multidrug transporter; member of the major facilitator superfamily; acts as an extrusion permease; partial multicopy suppressor of gal11 mutations<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8349103" target=_blank>Amakasu H, <i>et al.</i> (1993 Jul)</a>. Isolation and characterization of SGE1: a yeast gene that partially suppresses the gal11 mutation in multiple copies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8058039" target=_blank>Ehrenhofer-Murray AE, <i>et al.</i> (1994 Aug 2)</a>. The Saccharomyces cerevisiae SGE1 gene product: a novel drug-resistance protein within the major facilitator superfamily.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9271105" target=_blank>Jacquot C, <i>et al.</i> (1997 Aug)</a>. The Saccharomyces cerevisiae MFS superfamily SGE1 gene confers resistance to cationic dyes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9483795" target=_blank>Ehrenhofer-Murray AE, <i>et al.</i> (1998 Jan 15)</a>. The Sge1 protein of Saccharomyces cerevisiae is a membrane-associated multidrug transporter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01430_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01430_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01430_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01430_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG01431<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01431_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01431_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01431_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01431_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG01432<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01432_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01432_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01432_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01432_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG01433<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have role in asperfuranone biosynthetic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01433_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01433_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01433_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01433_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG01434<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have extracellular region localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22835028" target=_blank>Saykhedkar S, <i>et al.</i> (2012 Jul 26)</a>. A time course analysis of the extracellular proteome of Aspergillus nidulans growing on sorghum stover.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01434_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01434_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01434_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01434_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG01435<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01435_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01435_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01435_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01435_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG01436<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01436_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01436_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01436_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01436_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG01437<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 28
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular response to drug, hexose transport, pathogenesis<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633078" target=_blank>Sun X, <i>et al.</i> (2013 May)</a>. PyrG is required for maintaining stable cellular uracil level and normal sporulation pattern under excess uracil stress in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01437_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01437_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01437_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01437_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>28 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG01438<br>EOG8X3FH5<br><br>sce:TPO1<br><br>Genes: 57
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Polyamine transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; recognizes spermine, putrescine, and spermidine; catalyzes uptake of polyamines at alkaline pH and excretion at acidic pH; during oxidative stress exports spermine, spermidine from the cell, which controls timing of expression of stress-responsive genes; phosphorylation enhances activity and sorting to the plasma membrane<br><br>
			<br><b>PomBase Description</b><br>MFS family transmembrane transporter Mfs3|spermidine family transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have drug transmembrane transporter activity and role in cellular response to biotic stimulus, drug transmembrane transport, fluconazole transport, peptide transport, spermidine transport|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have spermidine transmembrane transporter activity, spermine transmembrane transporter activity and role in polyamine transmembrane transport, putrescine transport, spermidine transport, spermine transport<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9920864" target=_blank>Tomitori H, <i>et al.</i> (1999 Feb 5)</a>. Identification of a gene for a polyamine transport protein in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11171066" target=_blank>Tomitori H, <i>et al.</i> (2001 Feb 1)</a>. Multiple polyamine transport systems on the vacuolar membrane in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11470516" target=_blank>do Valle Matta MA, <i>et al.</i> (2001 Jul 11)</a>. Novel target genes of the yeast regulator Pdr1p: a contribution of the TPO1 gene in resistance to quinidine and other drugs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12562762" target=_blank>Albertsen M, <i>et al.</i> (2003 Apr 11)</a>. Localization and function of the yeast multidrug transporter Tpo1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524434" target=_blank>Kim H, <i>et al.</i> (2003 Mar 21)</a>. Topology models for 37 Saccharomyces cerevisiae membrane proteins based on C-terminal reporter fusions and predictions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13679573" target=_blank>Shepard KA, <i>et al.</i> (2003 Sep 30)</a>. Widespread cytoplasmic mRNA transport in yeast: identification of 22 bud-localized transcripts using DNA microarray analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066780" target=_blank>Aranda A, <i>et al.</i> (2004 Apr)</a>. Exposure of Saccharomyces cerevisiae to acetaldehyde induces sulfur amino acid metabolism and polyamine transporter genes, which depend on Met4p and Haa1p transcription factors, respectively.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15637075" target=_blank>Uemura T, <i>et al.</i> (2005 Mar 11)</a>. Characteristics of the polyamine transporter TPO1 and regulation of its activity and cellular localization by phosphorylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17362938" target=_blank>Mima S, <i>et al.</i> (2007 Apr 3)</a>. Identification of the TPO1 gene in yeast, and its human orthologue TETRAN, which cause resistance to NSAIDs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21832151" target=_blank>Kashiwazaki J, <i>et al.</i> (2011 Oct)</a>. Endocytosis is essential for dynamic translocation of a syntaxin 1 orthologue during fission yeast meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22840777" target=_blank>Kawashima SA, <i>et al.</i> (2012 Jul 27)</a>. Analyzing fission yeast multidrug resistance mechanisms to develop a genetically tractable model system for chemical biology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23050226" target=_blank>Chen Z, <i>et al.</i> (2012 Oct)</a>. A genetic screen to discover pathways affecting cohesin function in Schizosaccharomyces pombe identifies chromatin effectors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23264642" target=_blank>Garzia A, <i>et al.</i> (2013 Feb)</a>. Transcriptional changes in the transition from vegetative cells to asexual development in the model fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01438_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01438_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01438_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01438_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>57 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG01439<br>EOG8X3FH5<br><br>sce:TPO2;TPO3<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Polyamine transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; specific for spermine; localizes to the plasma membrane; transcription of TPO2 is regulated by Haa1p; TPO2 has a paralog, TPO3, that arose from the whole genome duplication|Polyamine transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; specific for spermine; localizes to the plasma membrane; TPO3 has a paralog, TPO2, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>spermine family transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have spermine transmembrane transporter activity, role in cellular response to drug, spermine transport and fungal-type vacuole membrane, plasma membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11171066" target=_blank>Tomitori H, <i>et al.</i> (2001 Feb 1)</a>. Multiple polyamine transport systems on the vacuolar membrane in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12562762" target=_blank>Albertsen M, <i>et al.</i> (2003 Apr 11)</a>. Localization and function of the yeast multidrug transporter Tpo1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066780" target=_blank>Aranda A, <i>et al.</i> (2004 Apr)</a>. Exposure of Saccharomyces cerevisiae to acetaldehyde induces sulfur amino acid metabolism and polyamine transporter genes, which depend on Met4p and Haa1p transcription factors, respectively.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633078" target=_blank>Sun X, <i>et al.</i> (2013 May)</a>. PyrG is required for maintaining stable cellular uracil level and normal sporulation pattern under excess uracil stress in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01439_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01439_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01439_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01439_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>32 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG01440<br>EOG8X3FH5<br><br>sce:TPO4<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Polyamine transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; recognizes spermine, putrescine, and spermidine; localizes to the plasma membrane<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8896271" target=_blank>Ch√©ret G, <i>et al.</i> (1996 Sep)</a>. DNA sequence analysis of the VPH1-SNF2 region on chromosome XV of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9593122" target=_blank>Delling U, <i>et al.</i> (1998 May)</a>. Identification of Saccharomyces cerevisiae genes conferring resistance to quinoline ring-containing antimalarial drugs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11171066" target=_blank>Tomitori H, <i>et al.</i> (2001 Feb 1)</a>. Multiple polyamine transport systems on the vacuolar membrane in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12562762" target=_blank>Albertsen M, <i>et al.</i> (2003 Apr 11)</a>. Localization and function of the yeast multidrug transporter Tpo1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066780" target=_blank>Aranda A, <i>et al.</i> (2004 Apr)</a>. Exposure of Saccharomyces cerevisiae to acetaldehyde induces sulfur amino acid metabolism and polyamine transporter genes, which depend on Met4p and Haa1p transcription factors, respectively.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01440_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01440_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01440_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01440_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>26 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG01441<br>EOG8X3FH5<br><br>sce:FLR1<br><br>Genes: 61
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Plasma membrane transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; involved in efflux of fluconazole, diazaborine, benomyl, methotrexate, and other drugs; expression induced in cells treated with the mycotoxin patulin; relocalizes from nucleus to plasma membrane upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>MFS family transmembrane transporter Mfs1|spermine family transmembrane transporter Caf5<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have Golgi apparatus localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2062311" target=_blank>Fling ME, <i>et al.</i> (1991 Jun)</a>. Analysis of a Candida albicans gene that encodes a novel mechanism for resistance to benomyl and methotrexate.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8031026" target=_blank>Ben-Yaacov R, <i>et al.</i> (1994 Apr)</a>. Candida albicans gene encoding resistance to benomyl and methotrexate is a multidrug resistance gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7726508" target=_blank>Goldway M, <i>et al.</i> (1995 Feb)</a>. Multidrug resistance in Candida albicans: disruption of the BENr gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8891134" target=_blank>Sanglard D, <i>et al.</i> (1996 Oct)</a>. Susceptibilities of Candida albicans multidrug transporter mutants to various antifungal agents and other metabolic inhibitors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9235926" target=_blank>Alarco AM, <i>et al.</i> (1997 Aug 1)</a>. AP1-mediated multidrug resistance in Saccharomyces cerevisiae requires FLR1 encoding a transporter of the major facilitator superfamily.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9046086" target=_blank>Goffeau A, <i>et al.</i> (1997 Jan)</a>. Multidrug-resistant transport proteins in yeast: complete inventory and phylogenetic characterization of yeast open reading frames with the major facilitator superfamily.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210670" target=_blank>White TC, <i>et al.</i> (1997 Jul)</a>. Increased mRNA levels of ERG16, CDR, and MDR1 correlate with increases in azole resistance in Candida albicans isolates from a patient infected with human immunodeficiency virus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9797228" target=_blank>Lopez-Ribot JL, <i>et al.</i> (1998 Nov)</a>. Distinct patterns of gene expression associated with development of fluconazole resistance in serial candida albicans isolates from human immunodeficiency virus-infected patients with oropharyngeal candidiasis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9756759" target=_blank>Marr KA, <i>et al.</i> (1998 Oct)</a>. Rapid, transient fluconazole resistance in Candida albicans is associated with increased mRNA levels of CDR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922230" target=_blank>Alarco AM, <i>et al.</i> (1999 Feb)</a>. The bZip transcription factor Cap1p is involved in multidrug resistance and oxidative stress response in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844673" target=_blank>Wirsching S, <i>et al.</i> (2000 May)</a>. Targeted gene disruption in Candida albicans wild-type strains: the role of the MDR1 gene in fluconazole resistance of clinical Candida albicans isolates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11568478" target=_blank>Kohli A, <i>et al.</i> (2001 Sep)</a>. Specificity of drug transport mediated by CaMDR1: a major facilitator of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14688046" target=_blank>Lee MK, <i>et al.</i> (2004 Feb)</a>. Drug resistance genes and trailing growth in Candida albicans isolates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15980350" target=_blank>Harry JB, <i>et al.</i> (2005 Jul)</a>. Drug-induced regulation of the MDR1 promoter in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16569853" target=_blank>Hiller D, <i>et al.</i> (2006 Apr)</a>. Overexpression of the MDR1 gene is sufficient to confer increased resistance to toxic compounds in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17041190" target=_blank>Riggle PJ, <i>et al.</i> (2006 Dec)</a>. Transcriptional regulation of MDR1, encoding a drug efflux determinant, in fluconazole-resistant Candida albicans strains through an Mcm1p binding site.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17159223" target=_blank>Rognon B, <i>et al.</i> (2006 Dec)</a>. Identification of promoter elements responsible for the regulation of MDR1 from Candida albicans, a major facilitator transporter involved in azole resistance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16801405" target=_blank>Hiller D, <i>et al.</i> (2006 Jul)</a>. Multiple cis-acting sequences mediate upregulation of the MDR1 efflux pump in a fluconazole-resistant clinical Candida albicans isolate.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17325226" target=_blank>Cheng S, <i>et al.</i> (2007 May)</a>. A Candida albicans petite mutant strain with uncoupled oxidative phosphorylation overexpresses MDR1 and has diminished susceptibility to fluconazole and voriconazole.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17983269" target=_blank>Morschh√§user J, <i>et al.</i> (2007 Nov)</a>. The transcription factor Mrr1p controls expression of the MDR1 efflux pump and mediates multidrug resistance in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18238892" target=_blank>Vogel M, <i>et al.</i> (2008 Mar)</a>. Rifampicin induces MDR1 expression in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18390649" target=_blank>Znaidi S, <i>et al.</i> (2008 May)</a>. Genomewide location analysis of Candida albicans Upc2p, a regulator of sterol metabolism and azole drug resistance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21136999" target=_blank>Hoehamer CF, <i>et al.</i> (2009 Aug)</a>. Proteomic analysis of Mrr1p- and Tac1p-associated differential protein expression in azole-resistant clinical isolates of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19395663" target=_blank>Znaidi S, <i>et al.</i> (2009 Jun)</a>. Identification of the Candida albicans Cap1p regulon.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19420894" target=_blank>Zhang H, <i>et al.</i> (2009 May)</a>. Mechanism of action of tetrandrine, a natural inhibitor of Candida albicans drug efflux pumps.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19527793" target=_blank>Chen CG, <i>et al.</i> (2009 Sep)</a>. Rep1p negatively regulating MDR1 efflux pump involved in drug resistance in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20348384" target=_blank>Basso LR Jr, <i>et al.</i> (2010 Jun)</a>. Fluconazole transport into Candida albicans secretory vesicles by the membrane proteins Cdr1p, Cdr2p, and Mdr1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685320" target=_blank>Schubert S, <i>et al.</i> (2011 Aug)</a>. Functional dissection of a Candida albicans zinc cluster transcription factor, the multidrug resistance regulator Mrr1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21343453" target=_blank>Mogavero S, <i>et al.</i> (2011 May)</a>. Differential requirement of the transcription factor Mcm1 for activation of the Candida albicans multidrug efflux pump MDR1 by its regulators Mrr1 and Cap1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21402859" target=_blank>Schubert S, <i>et al.</i> (2011 May)</a>. Regulation of efflux pump expression and drug resistance by the transcription factors Mrr1, Upc2, and Cap1 in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22615278" target=_blank>Sasse C, <i>et al.</i> (2012 Aug)</a>. Inducible and constitutive activation of two polymorphic promoter alleles of the Candida albicans multidrug efflux pump MDR1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22587419" target=_blank>Mandal A, <i>et al.</i> (2012 Aug 1)</a>. A key structural domain of the Candida albicans Mdr1 protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22840777" target=_blank>Kawashima SA, <i>et al.</i> (2012 Jul 27)</a>. Analyzing fission yeast multidrug resistance mechanisms to develop a genetically tractable model system for chemical biology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23527892" target=_blank>Zhang X, <i>et al.</i> (2013 Jun)</a>. Molecular mechanisms underlying the tetrandrine-mediated reversal of the fluconazole resistance of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24051054" target=_blank>Morio F, <i>et al.</i> (2013 Nov)</a>. Deciphering azole resistance mechanisms with a focus on transcription factor-encoding genes TAC1, MRR1 and UPC2 in a set of fluconazole-resistant clinical isolates of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24723295" target=_blank>Shafreen RM, <i>et al.</i> (2014 Aug)</a>. Inhibition of Candida albicans virulence factors by novel levofloxacin derivatives.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24936593" target=_blank>Ram√≠rez-Zavala B, <i>et al.</i> (2014 Sep)</a>. SAGA/ADA complex subunit Ada2 is required for Cap1- but not Mrr1-mediated upregulation of the Candida albicans multidrug efflux pump MDR1.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01441_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01441_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01441_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01441_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>61 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9e6c3" class="col">
			FOG01442<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have endoplasmic reticulum, plasma membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01442_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01442_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01442_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01442_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>36 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dbc4e1" class="col">
			FOG01443<br>EOG8X3FH5<br><br>sce:YHK8<br><br>Genes: 30
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Presumed antiporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; expression of gene is up-regulated in cells exhibiting reduced susceptibility to azoles<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted transporter activity, role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted transporter activity, role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12697649" target=_blank>Barker KS, <i>et al.</i> (2003 May)</a>. Identification of genes differentially expressed in association with reduced azole susceptibility in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16630275" target=_blank>Gbelska Y, <i>et al.</i> (2006 May)</a>. Evolution of gene families: the multidrug resistance transporter genes in five related yeast species.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01443_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01443_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01443_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01443_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>30 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fad7ad" class="col">
			FOG01444<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 23
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>spermidine family transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01444_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01444_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01444_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01444_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>23 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb9" class="col">
			FOG01445<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>MFS family transmembrane transporter Mfs2 (predicted)|amino acid transmembrane transporter (predicted)|transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22235339" target=_blank>Lando D, <i>et al.</i> (2012)</a>. The S. pombe histone H2A dioxygenase Ofd2 regulates gene expression during hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01445_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01445_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01445_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01445_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>17 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dfc9ba" class="col">
			FOG01446<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23766336" target=_blank>Coradetti ST, <i>et al.</i> (2013 Aug)</a>. Analysis of a conserved cellulase transcriptional regulator reveals inducer-independent production of cellulolytic enzymes in Neurospora crassa.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01446_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01446_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01446_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01446_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>12 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8d4ea" class="col">
			FOG01447<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative fructose transporter<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01447_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01447_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01447_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01447_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eeadad" class="col">
			FOG01448<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23264642" target=_blank>Garzia A, <i>et al.</i> (2013 Feb)</a>. Transcriptional changes in the transition from vegetative cells to asexual development in the model fungus Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01448_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01448_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01448_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01448_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bacde4" class="col">
			FOG01449<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559404" target=_blank>Christians JK, <i>et al.</i> (2011 Apr 29)</a>. Quantitative trait locus (QTL) mapping reveals a role for unstudied genes in Aspergillus virulence.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01449_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01449_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01449_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01449_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c1e3bb" class="col">
			FOG01450<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01450_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01450_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01450_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01450_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7bcdd" class="col">
			FOG01451<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01451_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01451_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01451_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01451_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d2a2" class="col">
			FOG01452<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01452_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01452_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01452_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01452_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb0" class="col">
			FOG01453<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01453_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01453_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01453_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01453_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dac1b0" class="col">
			FOG01454<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01454_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01454_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01454_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01454_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cee7" class="col">
			FOG01455<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22170104" target=_blank>Terabayashi Y, <i>et al.</i> (2012 Jan)</a>. Conserved and specific responses to hypoxia in Aspergillus oryzae and Aspergillus nidulans determined by comparative transcriptomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01455_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01455_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01455_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01455_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eca3a4" class="col">
			FOG01456<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01456_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01456_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01456_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01456_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b2c7e1" class="col">
			FOG01457<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01457_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01457_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01457_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01457_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bae0b3" class="col">
			FOG01458<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01458_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01458_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01458_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01458_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d2b4d9" class="col">
			FOG01459<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted transporter activity, role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01459_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01459_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01459_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01459_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cc97" class="col">
			FOG01460<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01460_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01460_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01460_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01460_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcffa6" class="col">
			FOG01461<br>EOG8K3JBP<br><br>sce:QDR1<br><br>Genes: 46
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Multidrug transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; involved in spore wall assembly; sequence similarity to DTR1 and QDR3, and the triple mutant dtr1 qdr1 qdr3 exhibits reduced dityrosine fluorescence relative to the single mutants; required for resistance to quinidine, ketoconazole, fluconazole, and barban; QDR1 has a paralog, AQR1, that arose from the whole genome duplication<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in biofilm formation, lipid homeostasis, membrane raft assembly and membrane raft localization|Ortholog(s) have drug transmembrane transporter activity, role in ascospore wall assembly, drug transmembrane transport and plasma membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have cation transmembrane transporter activity, drug transmembrane transporter activity, role in copper ion export, drug transmembrane transport, potassium ion import and integral component of plasma membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11302822" target=_blank>Nunes PA, <i>et al.</i> (2001 May)</a>. Resistance and adaptation to quinidine in Saccharomyces cerevisiae: role of QDR1 (YIL120w), encoding a plasma membrane transporter of the major facilitator superfamily required for multidrug resistance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11737641" target=_blank>Murad AM, <i>et al.</i> (2001 Nov)</a>. Transcript profiling in Candida albicans reveals new cellular functions for the transcriptional repressors CaTup1, CaMig1 and CaNrg1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12397174" target=_blank>Lan CY, <i>et al.</i> (2002 Nov 12)</a>. Metabolic specialization associated with phenotypic switching in Candidaalbicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15917516" target=_blank>Liu TT, <i>et al.</i> (2005 Jun)</a>. Genome-wide expression profiling of the response to azole, polyene, echinocandin, and pyrimidine antifungal agents in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19055746" target=_blank>Gaur M, <i>et al.</i> (2008 Dec 3)</a>. MFS transportome of the human pathogenic yeast Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21592964" target=_blank>Singh RP, <i>et al.</i> (2011 Jul 15)</a>. Cap2-HAP complex is a critical transcriptional regulator that has dual but contrasting roles in regulation of iron homeostasis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23613980" target=_blank>Palige K, <i>et al.</i> (2013)</a>. Global transcriptome sequencing identifies chlamydospore specific markers in Candida albicans and Candida dubliniensis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24621232" target=_blank>Shah AH, <i>et al.</i> (2014 Jun 1)</a>. Novel role of a family of major facilitator transporters in biofilm development and virulence of Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01461_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01461_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01461_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01461_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>46 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d6baa7" class="col">
			FOG01462<br>EOG8K3JBP<br><br>sce:QDR2<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Plasma membrane transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; exports copper; has broad substrate specificity and can transport many mono- and divalent cations; transports a variety of drugs and is required for resistance to quinidine, barban, cisplatin, and bleomycin; contributes to potassium homeostasis; expression is regulated by copper<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15215105" target=_blank>Vargas RC, <i>et al.</i> (2004 Jul)</a>. Saccharomyces cerevisiae multidrug transporter Qdr2p (Yil121wp): localization and function as a quinidine resistance determinant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17189489" target=_blank>Vargas RC, <i>et al.</i> (2007 Feb)</a>. Saccharomyces cerevisiae multidrug resistance transporter Qdr2 is implicated in potassium uptake, providing a physiological advantage to quinidine-stressed cells.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01462_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01462_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01462_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01462_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f7c8e5" class="col">
			FOG01463<br>EOG8K3JBP<br><br>sce:AQR1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Plasma membrane transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; confers resistance to short-chain monocarboxylic acids and quinidine; involved in the excretion of excess amino acids; AQR1 has a paralog, QDR1, that arose from the whole genome duplication; relocalizes from plasma membrane to cytoplasm upon DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11922628" target=_blank>Tenreiro S, <i>et al.</i> (2002 Apr 5)</a>. AQR1 gene (ORF YNL065w) encodes a plasma membrane transporter of the major facilitator superfamily that confers resistance to short-chain monocarboxylic acids and quinidine in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01463_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01463_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01463_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01463_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ea999a" class="col">
			FOG01464<br>EOG8K3JBP<br>EOG8X3FH5<br><br>sce:DTR1<br><br>Genes: 13
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative dityrosine transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; required for spore wall synthesis; sequence similarity to QDR1 and QDR3, and the triple mutant dtr1 qdr1 qdr3 exhibits reduced dityrosine fluorescence relative to the single mutants; expressed during sporulation<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12455697" target=_blank>Felder T, <i>et al.</i> (2002 Oct)</a>. Dtrlp, a multidrug resistance transporter of the major facilitator superfamily, plays an essential role in spore wall maturation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755916" target=_blank>Iwamoto MA, <i>et al.</i> (2005 Mar)</a>. Saccharomyces cerevisiae Sps1p regulates trafficking of enzymes required for spore wall synthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16554438" target=_blank>Nakanishi H, <i>et al.</i> (2006 Apr 1)</a>. Phospholipase D and the SNARE Sso1p are necessary for vesicle fusion during sporulation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17645731" target=_blank>Morishita M, <i>et al.</i> (2007 Sep)</a>. Snc1p v-SNARE transport to the prospore membrane during yeast sporulation is dependent on endosomal retrieval pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18676951" target=_blank>Morishita M, <i>et al.</i> (2008 Oct)</a>. Sorting signals within the Saccharomyces cerevisiae sporulation-specific dityrosine transporter, Dtr1p, C terminus promote Golgi-to-prospore membrane transport.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01464_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01464_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01464_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01464_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>13 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#aac1de" class="col">
			FOG01465<br>EOG8K3JBP<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01465_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01465_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01465_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01465_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b3dcab" class="col">
			FOG01466<br>EOG8K3JBP<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative multidrug transporter<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01466_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01466_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01466_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01466_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cdacd5" class="col">
			FOG01467<br>EOG8X3FH5<br><br>sce:QDR3<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Multidrug transporter of the major facilitator superfamily; member of the 12-spanner drug:H(+) antiporter DHA1 family; has a role in polyamine homeostasis; involved in spore wall asembly; sequence similarity to DTR1 and QDR1, and the triple mutant dtr1 qdr1 qdr3 exhibits reduced dityrosine fluorescence relative to the single mutants; expression is upregulated under polyamine stress; required for resistance to quinidine, barban, cisplatin, and bleomycin<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have drug transmembrane transporter activity, polyamine transmembrane transporter activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15649438" target=_blank>Tenreiro S, <i>et al.</i> (2005 Feb 18)</a>. The yeast multidrug transporter Qdr3 (Ybr043c): localization and role as a determinant of resistance to quinidine, barban, cisplatin, and bleomycin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19055746" target=_blank>Gaur M, <i>et al.</i> (2008 Dec 3)</a>. MFS transportome of the human pathogenic yeast Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21592964" target=_blank>Singh RP, <i>et al.</i> (2011 Jul 15)</a>. Cap2-HAP complex is a critical transcriptional regulator that has dual but contrasting roles in regulation of iron homeostasis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24621232" target=_blank>Shah AH, <i>et al.</i> (2014 Jun 1)</a>. Novel role of a family of major facilitator transporters in biofilm development and virulence of Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01467_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01467_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01467_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01467_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>34 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8c78c" class="col">
			FOG01468<br>EOG8DFN43<br>EOG8PNVZJ<br>EOG8S4MZQ<br><br>sce:absent<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01468_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01468_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01468_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01468_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>36 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff9d" class="col">
			FOG01469<br>EOG84F4S9<br>EOG88W9HJ<br>EOG8PNVZJ<br>EOG8QRFKH<br>EOG8X3FH5<br>EOG8XWDDF<br><br>sce:absent<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have cell cortex localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01469_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01469_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01469_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01469_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>32 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d2b39e" class="col">
			FOG01470<br>EOG8547FH<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 27
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Putative efflux pump<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01470_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01470_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01470_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01470_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>27 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6c2e2" class="col">
			FOG01471<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 16
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01471_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01471_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01471_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01471_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>16 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e88f90" class="col">
			FOG01472<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 15
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>MFS multidrug transporter|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01472_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01472_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01472_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01472_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>15 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a2bbdb" class="col">
			FOG01473<br>EOG8PNVZJ<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative amidase/acetamidase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01473_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01473_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01473_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01473_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#abd9a3" class="col">
			FOG01474<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>pyridoxal family transmembrane transporter (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22806344" target=_blank>Ryuko S, <i>et al.</i> (2012 Aug)</a>. Genome-wide screen reveals novel mechanisms for regulating cobalt uptake and detoxification in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01474_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01474_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01474_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01474_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c8a4d1" class="col">
			FOG01475<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>MFS family transmembrane transporter (predicted)|copper transmembrane transporter, meiotic Mfc1|transmembrane transporter (predicted)|truncated C terminal region of membrane transporter<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19264558" target=_blank>Deshpande GP, <i>et al.</i> (2009 May 1)</a>. Screening a genome-wide S. pombe deletion library identifies novel genes and pathways involved in genome stability maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20537132" target=_blank>Han TX, <i>et al.</i> (2010)</a>. Global fitness profiling of fission yeast deletion strains by barcode sequencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21828039" target=_blank>Beaudoin J, <i>et al.</i> (2011 Sep 30)</a>. Mfc1 is a novel forespore membrane copper transporter in meiotic and sporulating cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22840777" target=_blank>Kawashima SA, <i>et al.</i> (2012 Jul 27)</a>. Analyzing fission yeast multidrug resistance mechanisms to develop a genetically tractable model system for chemical biology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24569997" target=_blank>Plante S, <i>et al.</i> (2014 Apr 4)</a>. Characterization of Schizosaccharomyces pombe copper transporter proteins in meiotic and sporulating cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27558664" target=_blank>Guo L, <i>et al.</i> (2016 Oct 13)</a>. Global Fitness Profiling Identifies Arsenic and Cadmium Tolerance Mechanisms in Fission Yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01475_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01475_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01475_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01475_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f7c181" class="col">
			FOG01476<br>EOG8866V8<br>EOG8CFXRZ<br>EOG8D51DN<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01476_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01476_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01476_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01476_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff93" class="col">
			FOG01477<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01477_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01477_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01477_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01477_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cdab94" class="col">
			FOG01478<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01478_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01478_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01478_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01478_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f5bcdf" class="col">
			FOG01479<br>EOG8547FH<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01479_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01479_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01479_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01479_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e68586" class="col">
			FOG01480<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01480_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01480_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01480_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01480_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#9ab5d8" class="col">
			FOG01481<br>EOG8M0CHK<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01481_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01481_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01481_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01481_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a4d69b" class="col">
			FOG01482<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01482_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01482_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01482_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01482_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c39ccd" class="col">
			FOG01483<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01483_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01483_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01483_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01483_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6bc76" class="col">
			FOG01484<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>high-affinity import carrier for pyridoxine, pyridoxal, and pyridoxamine Bsu1|pyridoxamine/pyridoxine/pyridoxal transmembrane transporter (predicted)<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01484_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01484_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01484_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01484_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff8a" class="col">
			FOG01485<br>EOG8DFN43<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01485_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01485_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01485_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01485_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9a48b" class="col">
			FOG01486<br>EOG8PNVZJ<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative amidase/acetamidase|Putative amidase/acetamidase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01486_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01486_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01486_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01486_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4b6dc" class="col">
			FOG01487<br>EOG8V41Q9<br><br>sce:AZR1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Plasma membrane transporter of the major facilitator superfamily; involved in resistance to azole drugs such as ketoconazole and fluconazole<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11113970" target=_blank>Tenreiro S, <i>et al.</i> (2000 Dec)</a>. Expression of the AZR1 gene (ORF YGR224w), encoding a plasma membrane transporter of the major facilitator superfamily, is required for adaptation to acetic acid and resistance to azoles in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529331" target=_blank>Hikkel I, <i>et al.</i> (2003 Mar 28)</a>. A general strategy to uncover transcription factor properties identifies a new regulator of drug resistance in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01487_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01487_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01487_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01487_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e47c7d" class="col">
			FOG01488<br>EOG8866V8<br><br>sce:absent<br><br>Genes: 14
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Ortholog(s) have extracellular region localization|Ortholog(s) have role in gliotoxin biosynthetic process|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization||Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22835028" target=_blank>Saykhedkar S, <i>et al.</i> (2012 Jul 26)</a>. A time course analysis of the extracellular proteome of Aspergillus nidulans growing on sorghum stover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23264642" target=_blank>Garzia A, <i>et al.</i> (2013 Feb)</a>. Transcriptional changes in the transition from vegetative cells to asexual development in the model fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24690493" target=_blank>Colabardini AC, <i>et al.</i> (2014 Apr 1)</a>. Functional characterization of a xylose transporter in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01488_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01488_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01488_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01488_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>14 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#92b0d5" class="col">
			FOG01489<br>EOG8K3JBP<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01489_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01489_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01489_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01489_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#9dd393" class="col">
			FOG01490<br>EOG8DFN43<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01490_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01490_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01490_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01490_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bf95ca" class="col">
			FOG01491<br>EOG8QRFKH<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01491_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01491_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01491_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01491_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6b76b" class="col">
			FOG01492<br>EOG8X3FH5<br><br>sce:absent<br><br>Genes: 30
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted transporter activity, role in transmembrane transport and integral component of membrane, plasma membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Protein of unknown function|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization|Has domain(s) with predicted role in transmembrane transport and integral component of membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01492_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01492_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01492_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01492_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00599_phobius.pdf">Phobius transmembrane predictions</a>
										<br>29 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<!--link to gblock-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00599.png"
	</div>
	<div id="footer">Footer</div>
</div>

</body></html>