<html>
<head><title>HOG01180</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01179.html">HOG01179&emsp;&emsp;TRAFAC class myosin-kinesin ATPase superfamily. Kinesin family; TRAFAC class myosin-kinesin ATPase superfamily. Kinesin family. BimC subfamily; TRAFAC class myosin-kinesin ATPase superfamily. Kinesin family. Kinesin II subfamily; TRAFAC class myosin-kinesin ATPase superfamily. Kinesin family. NCD subfamily</a><br> 
																	HOG01180&emsp;&emsp;<br> 
																	<a href="HOG01181.html">HOG01181&emsp;&emsp;NHP6 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01180.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01180_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01180_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG01180.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03039<br>EOG89W0WM<br>EOG8VT4D0<br>EOG8X69QP<br><br>sce:MSS116<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mitochondrial transcription elongation factor; DEAD-box protein; required for efficient splicing of mitochondrial Group I and II introns; non-polar RNA helicase that also facilities strand annealing; promotes RNA folding by stabilizing an early assembly intermediate<br><br>
			<br><b>PomBase Description</b><br>mitochondrial ATP-dependent RNA helicase Mss116 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have mitochondrion localization|Ortholog(s) have ATP-dependent RNA helicase activity, RNA strand annealing activity, mRNA binding activity and role in Group I intron splicing, Group II intron splicing, RNA folding, transcription elongation from mitochondrial promoter<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2535893" target=_blank>SÃ©raphin B, <i>et al.</i> (1989 Jan 5)</a>. Mitochondrial splicing requires a protein from a novel helicase family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7567443" target=_blank>Niemer I, <i>et al.</i> (1995 Sep 11)</a>. Overexpression of DEAD box protein pMSS116 promotes ATP-dependent splicing of a yeast group II intron in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12402239" target=_blank>Minczuk M, <i>et al.</i> (2002 Nov)</a>. Overexpressed yeast mitochondrial putative RNA helicase Mss116 partially restores proper mtRNA metabolism in strains lacking the Suv3 mtRNA helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15618406" target=_blank>Huang HR, <i>et al.</i> (2005 Jan 4)</a>. The splicing of yeast mitochondrial group I and group II introns requires a DEAD-box protein with RNA chaperone function.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03039_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03039_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03039_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03039_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03040<br>EOG8VT4D0<br><br>sce:DBP10<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative ATP-dependent RNA helicase of the DEAD-box protein family; constituent of 66S pre-ribosomal particles; essential protein involved in ribosome biogenesis<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Dbp10 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10871363" target=_blank>Burger F, <i>et al.</i> (2000 Jun 15)</a>. Dbp10p, a putative RNA helicase from Saccharomyces cerevisiae, is required for ribosome biogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100437" target=_blank>Horsey EW, <i>et al.</i> (2004 May)</a>. Role of the yeast Rrp1 protein in the dynamics of pre-ribosome maturation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26167880" target=_blank>Lipp JJ, <i>et al.</i> (2015 Aug)</a>. SR protein kinases promote splicing of nonconsensus introns.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26644575" target=_blank>Halim A, <i>et al.</i> (2015 Dec 22)</a>. Discovery of a nucleocytoplasmic O-mannose glycoproteome in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03040_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03040_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03040_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03040_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03041<br>EOG8VT4D0<br><br>sce:HCA4<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>DEAD box RNA helicase; component of the SSU; interacts with Bfr2p and Enp2p; high-copy number suppression of a U14 snoRNA processing mutant suggests an involvement in 18S rRNA synthesis<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Hca4 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2406722" target=_blank>Chang TH, <i>et al.</i> (1990 Feb)</a>. Identification of five putative yeast RNA helicase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199348" target=_blank>Liang WQ, <i>et al.</i> (1997 Jul)</a>. The rRNA-processing function of the yeast U14 small nucleolar RNA can be rescued by a conserved RNA helicase-like protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16209945" target=_blank>Kos M, <i>et al.</i> (2005 Oct 7)</a>. The Putative RNA Helicase Dbp4p Is Required for Release of the U14 snoRNA from Preribosomes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18833290" target=_blank>Bohnsack MT, <i>et al.</i> (2008 Dec)</a>. Quantitative analysis of snoRNA association with pre-ribosomes and release of snR30 by Rok1 helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18975973" target=_blank>Garcia I, <i>et al.</i> (2008 Nov 25)</a>. Differential RNA-dependent ATPase activities of four rRNA processing yeast DEAD-box proteins.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03041_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03041_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03041_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03041_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03042<br>EOG8VT4D0<br><br>sce:DBP7<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative ATP-dependent RNA helicase of the DEAD-box family; involved in ribosomal biogenesis; required at post-transcriptional step for efficient retrotransposition; essential for growth under anaerobic conditions<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Dbp7 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582098" target=_blank>Daugeron MC, <i>et al.</i> (1998 May)</a>. Dbp7p, a putative ATP-dependent RNA helicase from Saccharomyces cerevisiae, is required for 60S ribosomal subunit assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125055" target=_blank>Walsh DW, <i>et al.</i> (2002 Aug)</a>. Genomic differences between Candida glabrata and Saccharomyces cerevisiae around the MRPL28 and GCN3 loci.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15937140" target=_blank>Chibana H, <i>et al.</i> (2005 Aug)</a>. Sequence finishing and gene mapping for Candida albicans chromosome 7 and syntenic analysis against the Saccharomyces cerevisiae genome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03042_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03042_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03042_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03042_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03043<br>EOG8VT4D0<br><br>sce:DBP8<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATPase, putative RNA helicase of the DEAD-box family; component of 90S preribosome complex involved in production of 18S rRNA and assembly of 40S small ribosomal subunit; ATPase activity stimulated by association with Esf2p<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Dbp8 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222764" target=_blank>Daugeron MC, <i>et al.</i> (2001 Mar 1)</a>. Characterization and mutational analysis of yeast Dbp8p, a putative RNA helicase involved in ribosome biogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16772403" target=_blank>Granneman S, <i>et al.</i> (2006)</a>. The nucleolar protein Esf2 interacts directly with the DExD/H box RNA helicase, Dbp8, to stimulate ATP hydrolysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03043_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03043_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03043_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03043_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03044<br>EOG8VT4D0<br><br>sce:DRS1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Nucleolar DEAD-box protein required for ribosome assembly and function; including synthesis of 60S ribosomal subunits; constituent of 66S pre-ribosomal particles<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Ddx27/Drs1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1454790" target=_blank>Ripmaster TL, <i>et al.</i> (1992 Dec 1)</a>. A putative ATP-dependent RNA helicase involved in Saccharomyces cerevisiae ribosome assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8247005" target=_blank>Ripmaster TL, <i>et al.</i> (1993 Dec)</a>. DRS1 to DRS7, novel genes required for ribosome assembly and function in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11911362" target=_blank>Adams CC, <i>et al.</i> (2002 Feb)</a>. Saccharomyces cerevisiae nucleolar protein Nop7p is necessary for biogenesis of 60S ribosomal subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100437" target=_blank>Horsey EW, <i>et al.</i> (2004 May)</a>. Role of the yeast Rrp1 protein in the dynamics of pre-ribosome maturation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25963819" target=_blank>Lucena R, <i>et al.</i> (2015 May 11)</a>. Nucleocytoplasmic transport in the midzone membrane domain controls yeast mitotic spindle disassembly.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03044_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03044_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03044_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03044_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG03045<br>EOG8VT4D0<br><br>sce:HAS1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATP-dependent RNA helicase; involved in the biogenesis of 40S and 60S ribosome subunits; localizes to both the nuclear periphery and nucleolus; highly enriched in nuclear pore complex fractions; constituent of 66S pre-ribosomal particles<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Has1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent RNA helicase activity, RNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9398665" target=_blank>Nislow C, <i>et al.</i> (1997 Dec)</a>. SET1, a yeast member of the trithorax family, functions in transcriptional silencing and diverse cellular processes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10684247" target=_blank>Rout MP, <i>et al.</i> (2000 Feb 21)</a>. The yeast nuclear pore complex: composition, architecture, and transport mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14574415" target=_blank>Ubersax JA, <i>et al.</i> (2003 Oct 23)</a>. Targets of the cyclin-dependent kinase Cdk1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15049817" target=_blank>Emery B, <i>et al.</i> (2004 Apr)</a>. Has1p, a member of the DEAD-box family, is required for 40S ribosomal subunit biogenesis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15242642" target=_blank>Mnaimneh S, <i>et al.</i> (2004 Jul 9)</a>. Exploration of essential gene functions via titratable promoter alleles.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100437" target=_blank>Horsey EW, <i>et al.</i> (2004 May)</a>. Role of the yeast Rrp1 protein in the dynamics of pre-ribosome maturation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15718299" target=_blank>Rocak S, <i>et al.</i> (2005)</a>. Characterization of the ATPase and unwinding activities of the yeast DEAD-box protein Has1p and the analysis of the roles of the conserved motifs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15769876" target=_blank>De Marchis ML, <i>et al.</i> (2005 Apr)</a>. Rrp15p, a novel component of pre-ribosomal particles required for 60S ribosome subunit maturation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23297348" target=_blank>Chen JS, <i>et al.</i> (2013 May)</a>. Comprehensive proteomics analysis reveals new substrates and regulators of the fission yeast clp1/cdc14 phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25963819" target=_blank>Lucena R, <i>et al.</i> (2015 May 11)</a>. Nucleocytoplasmic transport in the midzone membrane domain controls yeast mitotic spindle disassembly.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03045_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03045_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03045_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03045_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG03046<br>EOG8VT4D0<br><br>sce:RRP3<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein involved in rRNA processing; required for maturation of the 35S primary transcript of pre-rRNA and for cleavage leading to mature 18S rRNA; homologous to eIF-4a, which is a DEAD box RNA-dependent ATPase with helicase activity<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Rrp3 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8774901" target=_blank>O'Day CL, <i>et al.</i> (1996 Aug 15)</a>. 18S rRNA processing requires the RNA helicase-like protein Rrp3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16449634" target=_blank>Granneman S, <i>et al.</i> (2006 Feb)</a>. Comprehensive mutational analysis of yeast DEXD/H box RNA helicases required for small ribosomal subunit synthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18975973" target=_blank>Garcia I, <i>et al.</i> (2008 Nov 25)</a>. Differential RNA-dependent ATPase activities of four rRNA processing yeast DEAD-box proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19596074" target=_blank>Hagiwara D, <i>et al.</i> (2009 Nov)</a>. Transcriptional profiling for Aspergillusnidulans HogA MAPK signaling pathway in response to fludioxonil and osmotic stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23153376" target=_blank>Garcia I, <i>et al.</i> (2012 Dec 18)</a>. Duplex destabilization by four ribosomal DEAD-box proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03046_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03046_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03046_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03046_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG03047<br>EOG8VT4D0<br><br>sce:SPB4<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative ATP-dependent RNA helicase; nucleolar protein required for synthesis of 60S ribosomal subunits at a late step in the pathway; sediments with 66S pre-ribosomes in sucrose gradients<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Spb4 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in assembly of large subunit precursor of preribosome, maturation of LSU-rRNA, ribosomal large subunit assembly and 90S preribosome, nucleolus, nucleoplasm, preribosome, large subunit precursor localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2408148" target=_blank>Sachs AB, <i>et al.</i> (1990 Mar 2)</a>. Translation initiation and ribosomal biogenesis: involvement of a putative rRNA helicase and RPL46.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9769101" target=_blank>de la Cruz J, <i>et al.</i> (1998 Oct)</a>. Spb4p, an essential putative RNA helicase, is required for a late step in the assembly of 60S ribosomal subunits in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559404" target=_blank>Christians JK, <i>et al.</i> (2011 Apr 29)</a>. Quantitative trait locus (QTL) mapping reveals a role for unstudied genes in Aspergillus virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21825077" target=_blank>GarcÃ­a-GÃ³mez JJ, <i>et al.</i> (2011 Oct)</a>. Dynamics of the putative RNA helicase Spb4 during ribosome assembly in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22735702" target=_blank>Talkish J, <i>et al.</i> (2012 Sep 1)</a>. Hierarchical recruitment into nascent ribosomes of assembly factors required for 27SB pre-rRNA processing in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874617" target=_blank>Ohmayer U, <i>et al.</i> (2013)</a>. Studies on the assembly characteristics of large subunit ribosomal proteins in S. cerevisae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25579579" target=_blank>GarcÃ­a-GÃ³mez JJ, <i>et al.</i> (2015)</a>. Dynamics of the Spb4 interactome monitored by affinity purification.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03047_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03047_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03047_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03047_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01180_phobius.pdf">Phobius transmembrane predictions</a>
										<br>16 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01180.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>