<html>
<head><title>HOG00210</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00209.html">HOG00209&emsp;&emsp;TPP enzyme family</a><br> 
																	HOG00210&emsp;&emsp;<br> 
																	<a href="HOG00216.html">HOG00216&emsp;&emsp;aldehyde dehydrogenase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00210.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00210_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00210_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00210_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG00210.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00320<br>EOG84XH04<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase, mostly closely related to ancestral ADH in Fungi.<br><br>
			<br><b>Suggested Analysis</b><br>Appears to be most closely related to ancestral ADH but further studies required.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00320_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00320_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00320_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00320_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00321<br>EOG84XH04<br>EOG8XD27N<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized ADH paralog<br><br>
			<br><b>AspGD Description</b><br>Putative Zn-containing alcohol dehydrogenase; repressed by growth on starch and lactate|Putative alcohol dehydrogenase<br><br>
			<br><b>Suggested Analysis</b><br>Appears to have a converved insertition in msa for ang<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765123" target=_blank>Felenbok B, <i>et al.</i> (1994)</a>. Alcohol metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7496541" target=_blank>Sealy-Lewis HM, <i>et al.</i> (1995 Sep)</a>. Substrate specificity of nine NAD(+)-dependent alcohol dehydrogenases in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821658" target=_blank>Hunter GD, <i>et al.</i> (1996 Jan)</a>. The cloning and sequencing of the alcB gene, coding for alcohol dehydrogenase II, in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11277624" target=_blank>Jones IG, <i>et al.</i> (2001 Feb)</a>. ADHII in Aspergillus nidulans is induced by carbon starvation stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19784673" target=_blank>Salazar M, <i>et al.</i> (2009 Dec)</a>. Uncovering transcriptional regulation of glycerol metabolism in Aspergilli through genome-wide gene expression data analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21811407" target=_blank>Grahl N, <i>et al.</i> (2011 Jul)</a>. In vivo hypoxia and a fungal alcohol dehydrogenase influence the pathogenesis of invasive pulmonary aspergillosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173087" target=_blank>Georgakopoulos P, <i>et al.</i> (2012 Nov)</a>. SAGA complex components and acetate repression in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00321_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00321_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00321_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00321_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00322<br>EOG84XH04<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized ADH paralog<br><br>
			<br><b>AspGD Description</b><br>Putative alcohol dehydrogenase<br><br>
			<br><b>Suggested Analysis</b><br>Appears to have conserved motifs between sai and arx<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00322_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00322_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00322_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00322_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00323<br>EOG8XD27N<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized ADH paralog<br><br>
			<br><b>AspGD Description</b><br>Putative alcohol dehydrogenase<br><br>
			<br><b>Suggested Analysis</b><br>Appears to have conserved motifs between tre and ang<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00323_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00323_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00323_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00323_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00324<br>EOG8XD27N<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized ADH paralog<br><br>
			<br><b>AspGD Description</b><br>Putative alcohol dehydrogenase|Putative alcohol dehydrogenase|Putative alcohol dehydrogenase<br><br>
			<br><b>Suggested Analysis</b><br>Probe impact of unconserved motif at ~450 in msa<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22170104" target=_blank>Terabayashi Y, <i>et al.</i> (2012 Jan)</a>. Conserved and specific responses to hypoxia in Aspergillus oryzae and Aspergillus nidulans determined by comparative transcriptomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00324_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00324_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00324_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00324_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00325<br>EOG84XH04<br>ADH1<br>sce:ADH1<br><br>Genes: 37
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase, cytoplasmic<br><br>
			<br><b>Features</b><br>[c];NAD<br><br>
			<br><b>SGD Description</b><br>Alcohol dehydrogenase; fermentative isozyme active as homo- or heterotetramers; required for the reduction of acetaldehyde to ethanol, the last step in the glycolytic pathway; ADH1 has a paralog, ADH5, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>alcohol dehydrogenase Adh1<br><br>
			<br><b>AspGD Description</b><br>Alcohol-dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=320000" target=_blank>Jörnvall H, <i>et al.</i> (1977 Feb)</a>. The primary structure of yeast alcohol dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=395416" target=_blank>Roberts T, <i>et al.</i> (1979)</a>. Allele specific, gene unspecific suppressors in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6985717" target=_blank>Williamson VM, <i>et al.</i> (1980 Jan 10)</a>. Isolation of the structural gene for alcohol dehydrogenase by genetic complementation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6279086" target=_blank>Young T, <i>et al.</i> (1982)</a>. The alcohol dehydrogenase genes of the yeast, Saccharomyces cerevisiae: isolation, structure, and regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6277922" target=_blank>Bennetzen JL, <i>et al.</i> (1982 Mar 25)</a>. The primary structure of the Saccharomyces cerevisiae gene for alcohol dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6132393" target=_blank>Pateman JA, <i>et al.</i> (1983 Feb 22)</a>. Regulation of alcohol dehydrogenase (ADH) and aldehyde dehydrogenase (AldDH) in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6340750" target=_blank>Martinelli SD, <i>et al.</i> (1983 Jan)</a>. Diagnosis of nonsense mutations in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6294096" target=_blank>Russell PR, <i>et al.</i> (1983 Jan 10)</a>. The primary structure of the alcohol dehydrogenase gene from the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24177792" target=_blank>Sealy-Lewis HM, <i>et al.</i> (1984 May)</a>. Regulation of two alcohol dehydrogenases in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3158573" target=_blank>Lockington RA, <i>et al.</i> (1985)</a>. Cloning and characterization of the ethanol utilization regulon in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3158502" target=_blank>Doy CH, <i>et al.</i> (1985 Apr)</a>. Genomic clones of Aspergillus nidulans containing alcA, the structural gene for alcohol dehydrogenase and alcR, a regulatory gene for ethanol metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2998782" target=_blank>McKnight GL, <i>et al.</i> (1985 Aug)</a>. Identification and molecular analysis of a third Aspergillus nidulans alcohol dehydrogenase gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3156582" target=_blank>Creaser EH, <i>et al.</i> (1985 Jan 15)</a>. Purification and preliminary characterization of alcohol dehydrogenase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2945991" target=_blank>Fincham J, <i>et al.</i> (1986 Oct 30-Nov 5)</a>. Genetics of filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3297923" target=_blank>Gwynne DI, <i>et al.</i> (1987)</a>. Comparison of the cis-acting control regions of two coordinately controlled genes involved in ethanol utilization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2834622" target=_blank>Lockington R, <i>et al.</i> (1987 Nov)</a>. Regulation of alcR, the positive regulatory gene of the ethanol utilization regulon of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2663191" target=_blank>Jones IG, <i>et al.</i> (1989 Feb)</a>. Chromosomal mapping and gene disruption of the ADHIII gene in aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2673931" target=_blank>Waring RB, <i>et al.</i> (1989 Jun 30)</a>. Characterization of an inducible expression system in Aspergillus nidulans using alcA and tubulin-coding genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2178787" target=_blank>Jones IG, <i>et al.</i> (1990 Jan)</a>. Chromosomal mapping of an alcC disruption with respect to amdA in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2196567" target=_blank>Adams TH, <i>et al.</i> (1990 Jul)</a>. Developmental repression of growth and gene expression in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2274033" target=_blank>Kelly JM, <i>et al.</i> (1990 Jul)</a>. Alcohol dehydrogenase III in Aspergillus nidulans is anaerobically induced and post-transcriptionally regulated.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2245474" target=_blank>Sealy-Lewis HM, <i>et al.</i> (1990 Jul)</a>. The identification of mutations in Aspergillus nidulans that lead to increased levels of ADHII.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2190430" target=_blank>Saliola M, <i>et al.</i> (1990 May-Jun)</a>. The alcohol dehydrogenase system in the yeast, Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1367013" target=_blank>Felenbok B, <i>et al.</i> (1991 Jan)</a>. The ethanol utilization regulon of Aspergillus nidulans: the alcA-alcR system as a tool for the expression of recombinant proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1745237" target=_blank>Lloyd AT, <i>et al.</i> (1991 Nov)</a>. Codon usage in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452033" target=_blank>Ward PP, <i>et al.</i> (1992 Dec 1)</a>. An inducible expression system for the production of human lactoferrin in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1369734" target=_blank>Morris NR, <i>et al.</i> (1992 Jan)</a>. Mitotic gold in a mold: Aspergillus genetics and the biology of mitosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400424" target=_blank>Kulmburg P, <i>et al.</i> (1992 Oct 15)</a>. Specific binding sites for the activator protein, ALCR, in the alcA promoter of the ethanol regulon of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7763860" target=_blank>Hintz WE, <i>et al.</i> (1993 Jul)</a>. A glucose-derepressed promoter for expression of heterologous products in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483416" target=_blank>Kulmburg P, <i>et al.</i> (1993 Mar)</a>. Specific binding sites in the alcR and alcA promoters of the ethanol regulon for the CREA repressor mediating carbon catabolite repression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765123" target=_blank>Felenbok B, <i>et al.</i> (1994)</a>. Alcohol metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8277945" target=_blank>Sequeval D, <i>et al.</i> (1994 Jan)</a>. Relationship between zinc content and DNA-binding activity of the DNA-binding motif of the transcription factor ALCR in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076597" target=_blank>Mathieu M, <i>et al.</i> (1994 Sep 1)</a>. The Aspergillus nidulans CREA protein mediates glucose repression of the ethanol regulon at various levels through competition with the ALCR-specific transactivator.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076813" target=_blank>Hawkins AR, <i>et al.</i> (1994 Sep 2)</a>. Evolution of transcription-regulating proteins by enzyme recruitment: molecular models for nitrogen metabolite repression and ethanol utilisation in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7737086" target=_blank>Norbeck J, <i>et al.</i> (1995 Jan)</a>. Gene linkage of two-dimensional polyacrylamide gel electrophoresis resolved proteins from isogene families in Saccharomyces cerevisiae by microsequencing of in-gel trypsin generated peptides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7635218" target=_blank>Fillinger S, <i>et al.</i> (1995 Jul 24)</a>. The basal level of transcription of the alc genes in the ethanol regulon in Aspergillus nidulans is controlled both by the specific transactivator AlcR and the general carbon catabolite repressor CreA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7496541" target=_blank>Sealy-Lewis HM, <i>et al.</i> (1995 Sep)</a>. Substrate specificity of nine NAD(+)-dependent alcohol dehydrogenases in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8772381" target=_blank>Landsman D, <i>et al.</i> (1996 Aug)</a>. Histone H1 in Saccharomyces cerevisiae: a double mystery solved?<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8736527" target=_blank>Fillinger S, <i>et al.</i> (1996 May)</a>. A newly identified gene cluster in Aspergillus nidulans comprises five novel genes localized in the alc region that are controlled both by the specific transactivator AlcR and the general carbon-catabolite repressor CreA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9339892" target=_blank>Lee SK, <i>et al.</i> (1997 Aug 31)</a>. Inducible expression of yeast mitochondrial citrate synthase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9046096" target=_blank>Ushinsky SC, <i>et al.</i> (1997 Feb)</a>. Histone H1 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9278448" target=_blank>Panozzo C, <i>et al.</i> (1997 Sep 5)</a>. The zinc binuclear cluster activator AlcR is able to bind to single sites but requires multiple repeated sites for synergistic activation of the alcA gene in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9546172" target=_blank>Cho JY, <i>et al.</i> (1998 Apr)</a>. Pichia stipitis genes for alcohol dehydrogenase with fermentative and respiratory functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9487526" target=_blank>Caddick MX, <i>et al.</i> (1998 Feb)</a>. An ethanol inducible gene switch for plants used to manipulate carbon metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9487515" target=_blank>Gatz C, <i>et al.</i> (1998 Feb)</a>. An intoxicating switch for plant transgene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9497366" target=_blank>Panozzo C, <i>et al.</i> (1998 Mar 13)</a>. The CreA repressor is the sole DNA-binding protein responsible for carbon catabolite repression of the alcA gene in Aspergillus nidulans via its binding to a couple of specific sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802210" target=_blank>Passoth V, <i>et al.</i> (1998 Oct)</a>. Molecular cloning of alcohol dehydrogenase genes of the yeast Pichia stipitis and identification of the fermentative ADH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10577470" target=_blank>Agger T, <i>et al.</i> (1999)</a>. Genetically structured modeling of protein production in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216870" target=_blank>Strauss J, <i>et al.</i> (1999 Apr)</a>. The function of CreA, the carbon catabolite repressor of Aspergillus nidulans, is regulated at the transcriptional and post-transcriptional level.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10092669" target=_blank>Nikolaev I, <i>et al.</i> (1999 Apr 2)</a>. Unique DNA binding specificity of the binuclear zinc AlcR activator of the ethanol utilization pathway in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10096079" target=_blank>Nikolaev I, <i>et al.</i> (1999 Feb)</a>. A single amino acid, outside the AlcR zinc binuclear cluster, is involved in DNA binding and in transcriptional regulation of the alc genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760169" target=_blank>Mathieu M, <i>et al.</i> (2000 Apr)</a>. In vivo studies of upstream regulatory cis-acting elements of the alcR gene encoding the transactivator of the ethanol regulon in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11010922" target=_blank>Bautista LF, <i>et al.</i> (2000 Oct)</a>. Antisense silencing of the creA gene in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11550794" target=_blank>Felenbok B, <i>et al.</i> (2001)</a>. Ethanol catabolism in Aspergillus nidulans: a model system for studying gene regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11102439" target=_blank>Flipphi M, <i>et al.</i> (2001 Mar 9)</a>. Regulation of the aldehyde dehydrogenase gene (aldA) and its role in the control of the coinducer level necessary for induction of the ethanol utilization pathway in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12702265" target=_blank>Leskovac V, <i>et al.</i> (2002 Dec)</a>. The three zinc-containing alcohol dehydrogenases from baker's yeast, Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11988072" target=_blank>Flipphi M, <i>et al.</i> (2002 May 15)</a>. Characteristics of physiological inducers of the ethanol utilization (alc) pathway in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12223193" target=_blank>Nikolaev I, <i>et al.</i> (2002 Oct)</a>. Heterologous expression of the Aspergillus nidulans alcR-alcA system in Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12519784" target=_blank>Flipphi M, <i>et al.</i> (2003 Apr 4)</a>. Onset of carbon catabolite repression in Aspergillus nidulans. Parallel involvement of hexokinase and glucokinase in sugar signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12560092" target=_blank>Junker BH, <i>et al.</i> (2003 Jan 30)</a>. In plants the alc gene expression system responds more rapidly following induction with acetaldehyde than with ethanol.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12750323" target=_blank>Boase NA, <i>et al.</i> (2003 May)</a>. Molecular characterization and analysis of the acrB gene of Aspergillus nidulans: a gene identified by genetic interaction as a component of the regulatory network that includes the CreB deubiquitination enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14516763" target=_blank>Romero B, <i>et al.</i> (2003 Nov)</a>. The Aspergillus nidulans alcA promoter drives tightly regulated conditional gene expression in Aspergillus fumigatus permitting validation of essential genes in this human pathogen.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12919319" target=_blank>Flipphi M, <i>et al.</i> (2003 Sep)</a>. Relationships between the ethanol utilization (alc) pathway and unrelated catabolic pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14732266" target=_blank>Sims AH, <i>et al.</i> (2004 Feb)</a>. Use of expressed sequence tag analysis and cDNA microarrays of the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15071756" target=_blank>Toews MW, <i>et al.</i> (2004 Jun)</a>. Establishment of mRFP1 as a fluorescent marker in Aspergillus nidulans and construction of expression vectors for high-throughput protein tagging using recombination in vitro (GATEWAY).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15040954" target=_blank>Sakurai M, <i>et al.</i> (2004 Mar)</a>. A distinct type of alcohol dehydrogenase, adh4+, complements ethanol fermentation in an adh1-deficient strain of Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15813742" target=_blank>Mathieu M, <i>et al.</i> (2005 Apr)</a>. Patterns of nucleosomal organization in the alc regulon of Aspergillus nidulans: roles of the AlcR transcriptional activator and the CreA global repressor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15588991" target=_blank>Zarrin M, <i>et al.</i> (2005 Jan)</a>. A rapid method for promoter exchange in Aspergillus nidulans using recombinant PCR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17107606" target=_blank>David H, <i>et al.</i> (2006)</a>. Metabolic network driven analysis of genome-wide transcription data from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16531087" target=_blank>Flipphi M, <i>et al.</i> (2006 Apr)</a>. Functional analysis of alcS, a gene of the alc cluster in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16698295" target=_blank>Mogensen J, <i>et al.</i> (2006 Aug)</a>. Transcription analysis using high-density micro-arrays of Aspergillus nidulans wild-type and creA mutant during growth on glucose or ethanol.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16984401" target=_blank>Bromley M, <i>et al.</i> (2006 Nov)</a>. The Aspergillus fumigatus cellobiohydrolase B (cbhB) promoter is tightly regulated and can be exploited for controlled protein expression and RNAi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16699756" target=_blank>Lubertozzi D, <i>et al.</i> (2006 Oct)</a>. Marker and promoter effects on heterologous expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17470056" target=_blank>Sakvarelidze L, <i>et al.</i> (2007 Jul)</a>. Coupling the GAL4 UAS system with alcR for versatile cell type-specific chemically inducible gene expression in Arabidopsis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054105" target=_blank>García I, <i>et al.</i> (2008 Dec)</a>. Roles of the Aspergillus nidulans homologues of Tup1 and Ssn6 in chromatin structure and cell viability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18684775" target=_blank>Kennedy PJ, <i>et al.</i> (2008 Nov)</a>. A genome-wide screen of genes involved in cadmium tolerance in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19053082" target=_blank>Shimizu M, <i>et al.</i> (2009 Jan)</a>. Proteomic analysis of Aspergillus nidulans cultured under hypoxic conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19430462" target=_blank>Rissland OS, <i>et al.</i> (2009 Jun)</a>. Decapping is preceded by 3' uridylation in a novel pathway of bulk mRNA turnover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19171936" target=_blank>Sato I, <i>et al.</i> (2009 Mar 20)</a>. The glutathione system of Aspergillus nidulans involves a fungus-specific glutathione S-transferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20495807" target=_blank>Robellet X, <i>et al.</i> (2010 Aug)</a>. Gene silencing of transgenes inserted in the Aspergillus nidulans alcM and/or alcS loci.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20403971" target=_blank>Keller C, <i>et al.</i> (2010 Jun)</a>. Proteomic and functional analysis of the noncanonical poly(A) polymerase Cid14.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20536828" target=_blank>Roux AE, <i>et al.</i> (2010 Jun)</a>. A screen for genes involved in respiration control and longevity in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20605454" target=_blank>Coudreuse D, <i>et al.</i> (2010 Jun 22)</a>. A gene-specific requirement of RNA polymerase II CTD phosphorylation for sexual differentiation in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21811407" target=_blank>Grahl N, <i>et al.</i> (2011 Jul)</a>. In vivo hypoxia and a fungal alcohol dehydrogenase influence the pathogenesis of invasive pulmonary aspergillosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22403715" target=_blank>Bernal M, <i>et al.</i> (2012)</a>. Regulation of fission yeast morphogenesis by PP2A activator pta2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22170104" target=_blank>Terabayashi Y, <i>et al.</i> (2012 Jan)</a>. Conserved and specific responses to hypoxia in Aspergillus oryzae and Aspergillus nidulans determined by comparative transcriptomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173087" target=_blank>Georgakopoulos P, <i>et al.</i> (2012 Nov)</a>. SAGA complex components and acetate repression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633078" target=_blank>Sun X, <i>et al.</i> (2013 May)</a>. PyrG is required for maintaining stable cellular uracil level and normal sporulation pattern under excess uracil stress in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24003116" target=_blank>Corkins ME, <i>et al.</i> (2013 Sep 17)</a>. Zinc finger protein Loz1 is required for zinc-responsive regulation of gene expression in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24907261" target=_blank>Cánovas D, <i>et al.</i> (2014 Aug)</a>. The histone acetyltransferase GcnE (GCN5) plays a central role in the regulation of Aspergillus asexual development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25043916" target=_blank>Martins I, <i>et al.</i> (2014 Jul 21)</a>. Elucidating how the saprophytic fungus Aspergillus nidulans uses the plant polyester suberin as carbon source.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24634168" target=_blank>Bernal M, <i>et al.</i> (2014 Jun)</a>. Proteome-wide search for PP2A substrates in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24831008" target=_blank>Ehrensberger KM, <i>et al.</i> (2014 Jun 27)</a>. The double zinc finger domain and adjacent accessory domain from the transcription factor loss of zinc sensing 1 (loz1) are necessary for DNA binding and zinc sensing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25118249" target=_blank>Jiang H, <i>et al.</i> (2014 Nov)</a>. Putative PmrA and PmcA are important for normal growth, morphogenesis and cell wall integrity, but not for viability in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26152728" target=_blank>Mathiassen SG, <i>et al.</i> (2015 Aug 21)</a>. A Two-step Protein Quality Control Pathway for a Misfolded DJ-1 Variant in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25645080" target=_blank>Hill TW, <i>et al.</i> (2015 Feb)</a>. A mutation in the converter subdomain of Aspergillus nidulans MyoB blocks constriction of the actomyosin ring in cytokinesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25963819" target=_blank>Lucena R, <i>et al.</i> (2015 May 11)</a>. Nucleocytoplasmic transport in the midzone membrane domain controls yeast mitotic spindle disassembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00325_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00325_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00325_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00325_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00326<br>EOG84XH04<br>ADH2<br>sce:ADH2<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cytoplasmic alcohol dehydrogenase II, required for ethanol consumption.<br><br>
			<br><b>Features</b><br>[c];NAD<br><br>
			<br><b>Parent</b><br>paralog:FOG00325<br><br>
			<br><b>SGD Description</b><br>Glucose-repressible alcohol dehydrogenase II; catalyzes the conversion of ethanol to acetaldehyde; involved in the production of certain carboxylate esters; regulated by ADR1<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1105150" target=_blank>Ciriacy M, <i>et al.</i> (1975)</a>. Genetics of alcohol dehydrogenase in Saccharomyces cerevisiae. II. Two loci controlling synthesis of the glucose-repressible ADH II.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1101965" target=_blank>Wiesenfeld M, <i>et al.</i> (1975 Oct 20)</a>. Multiple forms of mitochondrial alcohol dehydrogenase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6279086" target=_blank>Young T, <i>et al.</i> (1982)</a>. The alcohol dehydrogenase genes of the yeast, Saccharomyces cerevisiae: isolation, structure, and regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6337160" target=_blank>Russell DW, <i>et al.</i> (1983 Feb 25)</a>. Nucleotide sequence of the yeast alcohol dehydrogenase II gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1588917" target=_blank>Shain DH, <i>et al.</i> (1992 Apr)</a>. Evolution of the alcohol dehydrogenase (ADH) genes in yeast: characterization of a fourth ADH in Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1406268" target=_blank>Mazzoni C, <i>et al.</i> (1992 Aug)</a>. Ethanol-induced and glucose-insensitive alcohol dehydrogenase activity in the yeast Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9298649" target=_blank>Garrels JI, <i>et al.</i> (1997 Aug)</a>. Proteome studies of Saccharomyces cerevisiae: identification and characterization of abundant proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12702265" target=_blank>Leskovac V, <i>et al.</i> (2002 Dec)</a>. The three zinc-containing alcohol dehydrogenases from baker's yeast, Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15864308" target=_blank>Thomson JM, <i>et al.</i> (2005 Jun)</a>. Resurrecting ancestral alcohol dehydrogenases from yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00326_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00326_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00326_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00326_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG00327<br>EOG84XH04<br>ADH2.2<br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cytoplasmic alcohol dehydrogenase, required for ethanol consumption.<br><br>
			<br><b>Features</b><br>[c];NAD<br><br>
			<br><b>Parent</b><br>paralog:FOG00325<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9546172" target=_blank>Cho JY, <i>et al.</i> (1998 Apr)</a>. Pichia stipitis genes for alcohol dehydrogenase with fermentative and respiratory functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802210" target=_blank>Passoth V, <i>et al.</i> (1998 Oct)</a>. Molecular cloning of alcohol dehydrogenase genes of the yeast Pichia stipitis and identification of the fermentative ADH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12489125" target=_blank>Passoth V, <i>et al.</i> (2003 Jan 15)</a>. Analysis of the hypoxia-induced ADH2 promoter of the respiratory yeast Pichia stipitis reveals a new mechanism for sensing of oxygen limitation in yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00327_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00327_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00327_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00327_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG00328<br>EOG84XH04<br>ADH3<br>sce:ADH3<br><br>Genes: 27
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial alcohol dehydrogenase isozyme III<br><br>
			<br><b>Features</b><br>[m];NAD<br><br>
			<br><b>SGD Description</b><br>Mitochondrial alcohol dehydrogenase isozyme III; involved in the shuttling of mitochondrial NADH to the cytosol under anaerobic conditions and ethanol production<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1101965" target=_blank>Wiesenfeld M, <i>et al.</i> (1975 Oct 20)</a>. Multiple forms of mitochondrial alcohol dehydrogenase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=782914" target=_blank>Brandt WF, <i>et al.</i> (1976 Jun 15)</a>. The occurrence of histone H3 and H4 in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=334592" target=_blank>Wiesenfeld M, <i>et al.</i> (1977)</a>. Inactivation and re-activation of mitochondrial alcohol dehydrogenase from baker's yeast [proceedings].<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7035169" target=_blank>Brandt WF, <i>et al.</i> (1982 Jan)</a>. The primary structure of yeast histone H3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6355483" target=_blank>Smith MM, <i>et al.</i> (1983 Sep 25)</a>. DNA sequences of yeast H3 and H4 histone genes from two non-allelic gene sets encode identical H3 and H4 proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2943982" target=_blank>Young ET, <i>et al.</i> (1985 Nov)</a>. Isolation and DNA sequence of ADH3, a nuclear gene encoding the mitochondrial isozyme of alcohol dehydrogenase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823079" target=_blank>Williamson VM, <i>et al.</i> (1987 Sep)</a>. Homology of Saccharomyces cerevisiae ADH4 to an iron-activated alcohol dehydrogenase from Zymomonas mobilis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538971" target=_blank>Stark MJ, <i>et al.</i> (1989 Jan-Feb)</a>. Cloning and analysis of the Kluyveromyces lactis TRP1 gene: a chromosomal locus flanked by genes encoding inorganic pyrophosphatase and histone H3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2193925" target=_blank>Drewke C, <i>et al.</i> (1990 Jul)</a>. Ethanol formation in adh0 mutants reveals the existence of a novel acetaldehyde-reducing activity in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1872030" target=_blank>Saliola M, <i>et al.</i> (1991 May-Jun)</a>. Two genes encoding putative mitochondrial alcohol dehydrogenases are present in the yeast Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9391047" target=_blank>Keener J, <i>et al.</i> (1997 Dec 9)</a>. Histones H3 and H4 are components of upstream activation factor required for the high-level transcription of yeast rDNA by RNA polymerase I.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9606197" target=_blank>Zhang W, <i>et al.</i> (1998 Jun 1)</a>. Essential and redundant functions of histone acetylation revealed by mutation of target lysines and loss of the Gcn5p acetyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10082517" target=_blank>Clarke AS, <i>et al.</i> (1999 Apr)</a>. Esa1p is an essential histone acetyltransferase required for cell cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10777603" target=_blank>Waterborg JH, <i>et al.</i> (2000 Apr 28)</a>. Steady-state levels of histone acetylation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10975519" target=_blank>Hsu JY, <i>et al.</i> (2000 Aug 4)</a>. Mitotic phosphorylation of histone H3 is governed by Ipl1/aurora kinase and Glc7/PP1 phosphatase in budding yeast and nematodes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10911986" target=_blank>Lo WS, <i>et al.</i> (2000 Jun)</a>. Phosphorylation of serine 10 in histone H3 is functionally linked in vitro and in vivo to Gcn5-mediated acetylation at lysine 14.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940011" target=_blank>Bakker BM, <i>et al.</i> (2000 Sep)</a>. The mitochondrial alcohol dehydrogenase Adh3p is involved in a redox shuttle in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11545749" target=_blank>Suka N, <i>et al.</i> (2001 Aug)</a>. Highly specific antibodies determine histone acetylation site usage in yeast heterochromatin and euchromatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751634" target=_blank>Briggs SD, <i>et al.</i> (2001 Dec 15)</a>. Histone H3 lysine 4 methylation is mediated by Set1 and required for cell growth and rDNA silencing in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742990" target=_blank>Roguev A, <i>et al.</i> (2001 Dec 17)</a>. The Saccharomyces cerevisiae Set1 complex includes an Ash2 homologue and methylates histone 3 lysine 4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11152939" target=_blank>Bakker BM, <i>et al.</i> (2001 Jan)</a>. Stoichiometry and compartmentation of NADH metabolism in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566884" target=_blank>White CL, <i>et al.</i> (2001 Sep 17)</a>. Structure of the yeast nucleosome core particle reveals fundamental changes in internucleosome interactions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12152067" target=_blank>Briggs SD, <i>et al.</i> (2002 Aug 1)</a>. Gene silencing: trans-histone regulatory pathway in chromatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12097318" target=_blank>Lacoste N, <i>et al.</i> (2002 Aug 23)</a>. Disruptor of telomeric silencing-1 is a chromatin-specific histone H3 methyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12702265" target=_blank>Leskovac V, <i>et al.</i> (2002 Dec)</a>. The three zinc-containing alcohol dehydrogenases from baker's yeast, Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752412" target=_blank>Nagy PL, <i>et al.</i> (2002 Jan 8)</a>. A trithorax-group complex purified from Saccharomyces cerevisiae is required for methylation of histone H3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12080090" target=_blank>Ng HH, <i>et al.</i> (2002 Jun 15)</a>. Lysine methylation within the globular domain of histone H3 by Dot1 is important for telomeric silencing and Sir protein association.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11839797" target=_blank>Strahl BD, <i>et al.</i> (2002 Mar)</a>. Set2 is a nucleosomal histone H3-selective methyltransferase that mediates transcriptional repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12353038" target=_blank>Santos-Rosa H, <i>et al.</i> (2002 Sep 26)</a>. Active genes are tri-methylated at K4 of histone H3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12574507" target=_blank>Ng HH, <i>et al.</i> (2003 Feb 18)</a>. Lysine-79 of histone H3 is hypomethylated at silenced loci in yeast and mammalian cells: a potential mechanism for position-effect variegation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845608" target=_blank>Boa S, <i>et al.</i> (2003 Jul 15)</a>. Saccharomyces cerevisiae Set1p is a methyltransferase specific for lysine 4 of histone H3 and is required for efficient gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773564" target=_blank>Krogan NJ, <i>et al.</i> (2003 Jun)</a>. Methylation of histone H3 by Set2 in Saccharomyces cerevisiae is linked to transcriptional elongation by RNA polymerase II.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12629047" target=_blank>Xiao T, <i>et al.</i> (2003 Mar 1)</a>. Phosphorylation of RNA polymerase II CTD regulates H3 methylation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12917322" target=_blank>Landry J, <i>et al.</i> (2003 Sep)</a>. Set2-catalyzed methylation of histone H3 represses basal expression of GAL4 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15059259" target=_blank>Fay JC, <i>et al.</i> (2004)</a>. Population genetic variation in gene expression is associated with phenotypic variation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15798214" target=_blank>Kizer KO, <i>et al.</i> (2005 Apr)</a>. A novel domain in Set2 mediates RNA polymerase II interaction and couples histone H3 K36 methylation with transcript elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16015338" target=_blank>Masumoto H, <i>et al.</i> (2005 Jul 14)</a>. A role for cell-cycle-regulated histone H3 lysine 56 acetylation in the DNA damage response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15888442" target=_blank>Ozdemir A, <i>et al.</i> (2005 Jul 15)</a>. Characterization of lysine 56 of histone H3 as an acetylation site in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15949446" target=_blank>Morillon A, <i>et al.</i> (2005 Jun 10)</a>. Dynamic lysine methylation on histone H3 defines the regulatory phase of gene transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15632126" target=_blank>Giannattasio M, <i>et al.</i> (2005 Mar 18)</a>. The DNA damage checkpoint response requires histone H2B ubiquitination by Rad6-Bre1 and H3 methylation by Dot1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15719021" target=_blank>Lo WS, <i>et al.</i> (2005 Mar 9)</a>. Histone H3 phosphorylation can promote TBP recruitment through distinct promoter-specific mechanisms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16260619" target=_blank>Hyland EM, <i>et al.</i> (2005 Nov)</a>. Insights into the role of histone H3 and histone H4 core modifiable residues in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16122352" target=_blank>Liu CL, <i>et al.</i> (2005 Oct)</a>. Single-nucleosome mapping of histone modifications in S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16185711" target=_blank>Dehé PM, <i>et al.</i> (2005 Oct 28)</a>. Histone H3 lysine 4 mono-methylation does not require ubiquitination of histone H2B.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16168379" target=_blank>Schneider J, <i>et al.</i> (2005 Sep 16)</a>. Molecular regulation of histone H3 trimethylation by COMPASS and the regulation of gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17194708" target=_blank>Garcia BA, <i>et al.</i> (2007 Mar 9)</a>. Organismal differences in post-translational modifications in histones H3 and H4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17189264" target=_blank>Morris SA, <i>et al.</i> (2007 Mar 9)</a>. Identification of histone H3 lysine 36 acetylation as a highly conserved histone modification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17320445" target=_blank>Tsubota T, <i>et al.</i> (2007 Mar 9)</a>. Histone H3-K56 acetylation is catalyzed by histone chaperone-dependent complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19113941" target=_blank>Zhang K, <i>et al.</i> (2009 Feb)</a>. Identification and verification of lysine propionylation and butyrylation in yeast core histones using PTMap software.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22389435" target=_blank>Xie Z, <i>et al.</i> (2012 May)</a>. Lysine succinylation and lysine malonylation in histones.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27105113" target=_blank>Goudarzi A, <i>et al.</i> (2016 Apr 21)</a>. Dynamic Competing Histone H4 K5K8 Acetylation and Butyrylation Are Hallmarks of Highly Active Gene Promoters.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27089029" target=_blank>Andrews FH, <i>et al.</i> (2016 Jun)</a>. The Taf14 YEATS domain is a reader of histone crotonylation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00328_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00328_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00328_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00328_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG00329<br>EOG84XH04<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase, cytoplasmic, independent evolution in Peziz.<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00329_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00329_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00329_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00329_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG00330<br>EOG84XH04<br>ADH5<br>sce:ADH5<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase isoenzyme V, ohnolog<br><br>
			<br><b>Parent</b><br>ohnolog:FOG00325<br><br>
			<br><b>SGD Description</b><br>Alcohol dehydrogenase isoenzyme V; involved in ethanol production; ADH5 has a paralog, ADH1, that arose from the whole genome duplication<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00330_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00330_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00330_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00330_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG00331<br>EOG84XH04<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase isoenzyme, yli paralogs<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00331_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00331_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00331_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00331_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG00332<br>EOG84XH04<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase isoenzyme, ang paralogs close to budding yeasts<br><br>
			<br><b>AspGD Description</b><br>|Putative alcohol dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00332_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00332_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00332_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00332_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG00333<br>EOG84XH04<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00333_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00333_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00333_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00333_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG00334<br>EOG8XD27N<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00334_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00334_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00334_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00334_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG00335<br><br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase isoenzyme<br><br>
			<br><b>Suggested Analysis</b><br>Conserved motifs from sai to bin<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00335_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00335_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00335_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00335_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG00336<br><br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase isoenzyme<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00336_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00336_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00336_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00336_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG00337<br><br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol dehydrogenase isoenzyme<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00337_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00337_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00337_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00337_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9e6c3" class="col">
			FOG00338<br>EOG8MPG6T<br><br>sce:absent<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NADP-alcohol dehydrogenase<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity, transferase activity, transferring acyl groups other than amino-acyl groups, zinc ion binding activity and role in oxidation-reduction process|Putative alcohol dehydrogenase (catalyzed by glycerol dehydrogenase II (NADP+)|Has domain(s) with predicted oxidoreductase activity, transferase activity, transferring acyl groups other than amino-acyl groups, zinc ion binding activity and role in oxidation-reduction process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22170104" target=_blank>Terabayashi Y, <i>et al.</i> (2012 Jan)</a>. Conserved and specific responses to hypoxia in Aspergillus oryzae and Aspergillus nidulans determined by comparative transcriptomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23472095" target=_blank>Colabardini AC, <i>et al.</i> (2013)</a>. Functional characterization of Aspergillus nidulans ypkA, a homologue of the mammalian kinase SGK.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00338_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00338_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00338_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00338_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dbc4e1" class="col">
			FOG00339<br>EOG8MPG6T<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NADP-alcohol dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00339_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00339_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00339_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00339_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fad7ad" class="col">
			FOG00340<br>EOG8MPG6T<br>ADH6<br>sce:ADH6<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NADP-alcohol dehydrogenase VI: The highest k(cat)/K(m) values were found with pentanal>veratraldehyde > hexanal > 3-methylbutanal >cinnamaldehyde; Aldehydes exhibited 50-12000 times higher catalytic efficiency than the corresponding alcohols<br><br>
			<br><b>SGD Description</b><br>NADPH-dependent medium chain alcohol dehydrogenase; has broad substrate specificity; member of the cinnamyl family of alcohol dehydrogenases; may be involved in fusel alcohol synthesis or in aldehyde tolerance; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742541" target=_blank>Larroy C, <i>et al.</i> (2002 Jan 1)</a>. Characterization of the Saccharomyces cerevisiae YMR318C (ADH6) gene product as a broad specificity NADPH-dependent alcohol dehydrogenase: relevance in aldehyde reduction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12554944" target=_blank>Valencia E, <i>et al.</i> (2003 Feb)</a>. Crystallization and preliminary X-ray analysis of NADP(H)-dependent alcohol dehydrogenases from Saccharomyces cerevisiae and Rana perezi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12604208" target=_blank>Larroy C, <i>et al.</i> (2003 Feb 1)</a>. Properties and functional significance of Saccharomyces cerevisiae ADHVI.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00340_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00340_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00340_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00340_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb9" class="col">
			FOG00341<br>EOG8MPG6T<br>ADH7<br>sce:ADH7<br><br>Genes: 15
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NADP-alcohol dehydrogenase VII<br><br>
			<br><b>SGD Description</b><br>NADPH-dependent medium chain alcohol dehydrogenase; has broad substrate specificity; member of the cinnamyl family of alcohol dehydrogenases; may be involved in fusel alcohol synthesis or in aldehyde tolerance<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12423374" target=_blank>Larroy C, <i>et al.</i> (2002 Nov)</a>. Characterization of a Saccharomyces cerevisiae NADP(H)-dependent alcohol dehydrogenase (ADHVII), a member of the cinnamyl alcohol dehydrogenase family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00341_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00341_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00341_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00341_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dfc9ba" class="col">
			FOG00342<br>EOG8MPG6T<br><br>sce:absent<br><br>Genes: 14
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00342_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00342_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00342_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00342_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8d4ea" class="col">
			FOG00343<br>EOG8MPG6T<br><br>sce:absent<br><br>Genes: 23
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00343_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00343_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00343_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00343_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>10 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eeadad" class="col">
			FOG00344<br>EOG8MPG6T<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00344_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00344_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00344_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00344_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bacde4" class="col">
			FOG00345<br>EOG8MPG6T<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00345_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00345_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00345_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00345_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c1e3bb" class="col">
			FOG00346<br><br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00346_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00346_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00346_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00346_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00210_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00210.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>