<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01180.html">HOG01180&emsp;&emsp;DEAD box helicase family. DDX10/DBP4 subfamily; DEAD box helicase family. DDX18/HAS1 subfamily; DEAD box helicase family. DDX27/DRS1 subfamily; DEAD box helicase family. DDX31/DBP7 subfamily; DEAD box helicase family. DDX47/RRP3 subfamily; DEAD box helicase family. DDX49/DBP8 subfamily; DEAD box helicase family. DDX54/DBP10 subfamily; DEAD box helicase family. DDX55/SPB4 subfamily</a><br> 
																	HOG01181&emsp;&emsp;<br> 
																	<a href="HOG01182.html">HOG01182&emsp;&emsp;small GTPase superfamily. Rho family; small GTPase superfamily. Rho family. CDC42 subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01181.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01181_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01181_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03048<br>EOG8TQJWB<br><br>sce:NHP6B;NHP6A<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>High-mobility group (HMG) protein; binds to and remodels nucleosomes; involved in recruiting FACT and other chromatin remodelling complexes to the chromosomes; functionally redundant with Nhp6Ap; required for transcriptional initiation fidelity of some tRNA genes; homologous to mammalian HMGB1 and HMGB2; NHP6B has a paralog, NHP6A, that arose from the whole genome duplication|High-mobility group (HMG) protein; binds to and remodels nucleosomes; involved in recruiting FACT and other chromatin remodelling complexes to chromosomes; functionally redundant with Nhp6Bp; required for transcriptional initiation fidelity of some tRNA genes; homologous to mammalian HMGB1 and HMGB2; NHP6A has a paralog, NHP6B, that arose from the whole genome duplication; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>High-mobility group non-histone chromatin protein (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have DNA binding, bending, sequence-specific DNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2406250" target=_blank>Kolodrubetz D, <i>et al.</i> (1990 Feb 25)</a>. Duplicated NHP6 genes of Saccharomyces cerevisiae encode proteins homologous to bovine high mobility group protein 1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1339437" target=_blank>Tercero JC, <i>et al.</i> (1992 Oct 5)</a>. Localized mutagenesis and evidence for post-transcriptional regulation of MAK3. A putative N-acetyltransferase required for double-stranded RNA virus propagation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7721780" target=_blank>Paull TT, <i>et al.</i> (1995 Apr 14)</a>. DNA looping by Saccharomyces cerevisiae high mobility group proteins NHP6A/B. Consequences for nucleoprotein complex assembly and chromatin condensation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8946917" target=_blank>Paull TT, <i>et al.</i> (1996 Nov 1)</a>. Yeast HMG proteins NHP6A/B potentiate promoter-specific transcriptional activation in vivo and assembly of preinitiation complexes in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9468494" target=_blank>Yen YM, <i>et al.</i> (1998 Feb 20)</a>. Determinants of DNA binding and bending by the Saccharomyces cerevisiae high mobility group protein NHP6A that are important for its biological activities. Role of the unique N terminus and putative intercalating methionine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10228169" target=_blank>Allain FH, <i>et al.</i> (1999 May 4)</a>. Solution structure of the HMG protein NHP6A and its interaction with DNA reveals the structural determinants for non-sequence-specific binding.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118215" target=_blank>Moreira JM, <i>et al.</i> (2000 Dec 15)</a>. Chromatin-mediated transcriptional regulation by the yeast architectural factors NHP6A and NHP6B.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11239460" target=_blank>Kruppa M, <i>et al.</i> (2001 Feb)</a>. Nhp6, an HMG1 protein, functions in SNR6 transcription by RNA polymerase III in S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11422939" target=_blank>Yen YM, <i>et al.</i> (2001 Jul)</a>. Nuclear localization of the Saccharomyces cerevisiae HMG protein NHP6A occurs by a Ran-independent nonclassical pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11432837" target=_blank>Formosa T, <i>et al.</i> (2001 Jul 2)</a>. Spt16-Pob3 and the HMG protein Nhp6 combine to form the nucleosome-binding factor SPN.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11313475" target=_blank>Brewster NK, <i>et al.</i> (2001 May)</a>. A bipartite yeast SSRP1 analog comprised of Pob3 and Nhp6 proteins modulates transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287614" target=_blank>Lopez S, <i>et al.</i> (2001 May)</a>. High-mobility-group proteins NHP6A and NHP6B participate in activation of the RNA polymerase III SNR6 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12381320" target=_blank>Masse JE, <i>et al.</i> (2002 Oct 18)</a>. The S. cerevisiae architectural HMGB protein NHP6A complexed with DNA: DNA and protein conformational changes upon binding.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14585989" target=_blank>Mason PB, <i>et al.</i> (2003 Nov)</a>. The FACT complex travels with elongating RNA polymerase II and is important for the fidelity of transcriptional initiation in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12952948" target=_blank>Ruone S, <i>et al.</i> (2003 Nov 14)</a>. Multiple Nhp6 molecules are required to recruit Spt16-Pob3 to form yFACT complexes and to reorganize nucleosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14739930" target=_blank>Kim M, <i>et al.</i> (2004 Jan 28)</a>. Transitions in RNA polymerase II elongation complexes at the 3' ends of genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15082784" target=_blank>Rhoades AR, <i>et al.</i> (2004 May)</a>. Structural features of nucleosomes reorganized by yeast FACT and its HMG box component, Nhp6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15504049" target=_blank>Skoko D, <i>et al.</i> (2004 Nov 2)</a>. Micromechanical analysis of the binding of DNA-bending proteins HMGB1, NHP6A, and HU reveals their ability to form highly stable DNA-protein complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15987999" target=_blank>Biswas D, <i>et al.</i> (2005 Jul)</a>. The yeast FACT complex has a role in transcriptional initiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16407207" target=_blank>Kassavetis GA, <i>et al.</i> (2006 Mar 17)</a>. Nhp6 is a transcriptional initiation fidelity factor for RNA polymerase III transcription in vitro and in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17244705" target=_blank>Juneau K, <i>et al.</i> (2007 Jan 30)</a>. High-density yeast-tiling array reveals previously undiscovered introns and extensive regulation of meiotic splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25156107" target=_blank>Karácsony Z, <i>et al.</i> (2014 Oct)</a>. A dually located multi-HMG-box protein of Aspergillus nidulans has a crucial role in conidial and ascospore germination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26582768" target=_blank>Gal C, <i>et al.</i> (2016 Jan)</a>. Abo1, a conserved bromodomain AAA-ATPase, maintains global nucleosome occupancy and organisation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03048_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03048_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03048_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03048_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03049<br>EOG8TQJWB<br><br>sce:NHP10<br><br>Genes: 23
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Non-essential INO80 chromatin remodeling complex subunit; preferentially binds DNA ends, protecting them from exonucleatic cleavage; deletion affects telomere maintenance via recombination; related to mammalian high mobility group proteins<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969238" target=_blank>Lu J, <i>et al.</i> (1996 Dec 27)</a>. Characterization of a high mobility group 1/2 homolog in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12887900" target=_blank>Shen X, <i>et al.</i> (2003 Jul)</a>. Involvement of actin-related proteins in ATP-dependent chromatin remodeling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03049_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03049_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03049_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03049_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03050<br>EOG8TQJWB<br><br>sce:HMO1<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Chromatin associated high mobility group (HMG) family member; involved in compacting, bending, bridging and looping DNA; rDNA-binding component that regulates transcription from RNA polymerase I promoters; regulates start site selection of ribosomal protein genes via RNA polymerase II promoters; role in genome maintenance; associates with a 5'-3' DNA helicase and Fpr1p, a prolyl isomerase; relocalizes to the cytosol in response to hypoxia<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969238" target=_blank>Lu J, <i>et al.</i> (1996 Dec 27)</a>. Characterization of a high mobility group 1/2 homolog in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049913" target=_blank>Dolinski KJ, <i>et al.</i> (1999 Mar)</a>. Hmo1p, a high mobility group 1/2 homolog, genetically and physically interacts with the yeast FKBP12 prolyl isomerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12509247" target=_blank>Alekseev SY, <i>et al.</i> (2002 Apr 29)</a>. HSM2 (HMO1) gene participates in mutagenesis control in yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12374750" target=_blank>Gadal O, <i>et al.</i> (2002 Oct 15)</a>. Hmo1, an HMG-box protein, belongs to the yeast ribosomal DNA transcription system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20231876" target=_blank>Lavoie H, <i>et al.</i> (2010 Mar 9)</a>. Evolutionary tinkering with conserved components of a transcriptional regulatory network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22265407" target=_blank>Nobile CJ, <i>et al.</i> (2012 Jan 20)</a>. A recently evolved transcriptional network controls biofilm development in Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03050_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03050_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03050_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03050_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03051<br>EOG8TQJWB<br><br>sce:IXR1<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Transcriptional repressor that regulates hypoxic genes during normoxia; involved in the aerobic repression of genes such as COX5b, TIR1, and HEM13; binds DNA intrastrand cross-links formed by cisplatin; HMG (high mobility group box) domain containing protein which binds and bends cisplatin-modified DNA, blocking excision repair; IXR1 has a paralog, ABF2, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8342024" target=_blank>Brown SJ, <i>et al.</i> (1993 Jul 30)</a>. Ixr1, a yeast protein that binds to platinated DNA and confers sensitivity to cisplatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8041793" target=_blank>Lambert JR, <i>et al.</i> (1994 Jul 19)</a>. The ORD1 gene encodes a transcription factor involved in oxygen regulation and is identical to IXR1, a gene that confers cisplatin sensitivity to Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03051_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03051_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03051_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03051_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03052<br>EOG87M0GG<br>EOG8TQJWB<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03052_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03052_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03052_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03052_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03053<br>EOG8D51J6<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have DNA binding, bending, double-stranded DNA binding, four-way junction DNA binding, sequence-specific DNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25156107" target=_blank>Karácsony Z, <i>et al.</i> (2014 Oct)</a>. A dually located multi-HMG-box protein of Aspergillus nidulans has a crucial role in conidial and ascospore germination.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03053_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03053_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03053_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03053_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG03054<br>EOG8TQJWB<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03054_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03054_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03054_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03054_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG03055<br>EOG8D51J6<br>EOG8TQJWB<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03055_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03055_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03055_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03055_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG03056<br>EOG8TQJWB<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>transcription factor Ste11<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03056_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03056_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03056_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03056_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG03057<br>EOG8TQJWB<br><br>sce:ABF2<br><br>Genes: 16
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mitochondrial DNA-binding protein; involved in mitochondrial DNA replication and recombination, member of HMG1 DNA-binding protein family; activity may be regulated by protein kinase A phosphorylation; ABF2 has a paralog, IXR1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>HMG box protein Hmo1|cytosine-mismatch binding protein 1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in ascospore formation, asexual sporulation resulting in formation of a cellular spore and mitochondrion, nucleus localization|Ortholog of A. nidulans FGSC A4 : AN3667, A. fumigatus Af293 : Afu4g12410, A. oryzae RIB40 : AO090009000453, Aspergillus wentii : Aspwe1_0166557 and Aspergillus sydowii : Aspsy1_1055875<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1881919" target=_blank>Diffley JF, <i>et al.</i> (1991 Sep 1)</a>. A close relative of the nuclear, chromosomal high-mobility group protein HMG1 in yeast mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9804804" target=_blank>Fleck O, <i>et al.</i> (1998 Nov 13)</a>. The high mobility group domain protein Cmb1 of Schizosaccharomyces pombe binds to cytosines in base mismatches and opposite chemically altered guanines.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9742088" target=_blank>Cho JH, <i>et al.</i> (1998 Oct)</a>. A novel DNA-binding protein bound to the mitochondrial inner membrane restores the null mutation of mitochondrial histone Abf2p in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11399082" target=_blank>Sassoon J, <i>et al.</i> (2001 Jun 22)</a>. Biochemical characterization of the structure-specific DNA-binding protein Cmb1 from Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12617726" target=_blank>Kunz C, <i>et al.</i> (2003 Jun 1)</a>. Mutagenesis of the HMGB (high-mobility group B) protein Cmb1 (cytosine-mismatch binding 1) of Schizosaccharomyces pombe: effects on recognition of DNA mismatches and damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22119525" target=_blank>Singh NS, <i>et al.</i> (2011 Dec 6)</a>. SIN-inhibitory phosphatase complex promotes Cdc11p dephosphorylation and propagates SIN asymmetry in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24945319" target=_blank>Vanoosthuyse V, <i>et al.</i> (2014 Jun)</a>. CPF-associated phosphatase activity opposes condensin-mediated chromosome condensation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25156107" target=_blank>Karácsony Z, <i>et al.</i> (2014 Oct)</a>. A dually located multi-HMG-box protein of Aspergillus nidulans has a crucial role in conidial and ascospore germination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26297166" target=_blank>Karácsony Z, <i>et al.</i> (2015 Oct)</a>. Further characterization of the role of the mitochondrial high-mobility group box protein in the intracellular redox environment of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26628015" target=_blank>Zhang X, <i>et al.</i> (2015 Oct 9)</a>. Characterization of Tamoxifen as an Antifungal Agent Using the Yeast Schizosaccharomyces Pombe Model Organism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03057_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03057_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03057_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03057_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01181_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01181.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>