<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01295.html">HOG01295&emsp;&emsp;CSM3 family</a><br> 
																	HOG01296&emsp;&emsp;<br> 
																	<a href="HOG01297.html">HOG01297&emsp;&emsp;peptidase M18 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01296.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01296_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01296_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01296_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03725<br>EOG8ZW3T8<br><br>sce:TEF1; TEF2<br><br>Genes: 45
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>translation elongation factor EF-1 alpha Ef1a-a|translation elongation factor EF-1 alpha Ef1a-b|translation elongation factor EF-1 alpha Ef1a-c<br><br>
			<br><b>AspGD Description</b><br>Protein similar to elongation factor 1 alpha<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6383821" target=_blank>Nagata S, <i>et al.</i> (1984 Aug)</a>. Polypeptide chain elongation factor 1 alpha (EF-1 alpha) from yeast: nucleotide sequence of one of the two genes for EF-1 alpha from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6396088" target=_blank>Schirmaier F, <i>et al.</i> (1984 Dec 20)</a>. Identification of two genes coding for the translation elongation factor EF-1 alpha of S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982849" target=_blank>Cottrelle P, <i>et al.</i> (1985 Mar 10)</a>. Cloning, nucleotide sequence, and expression of one of two genes coding for yeast elongation factor 1 alpha.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3882705" target=_blank>Thiele D, <i>et al.</i> (1985 Mar 10)</a>. Elongation factor 1 alpha from Saccharomyces cerevisiae. Rapid large-scale purification and molecular characterization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3026912" target=_blank>Nagashima K, <i>et al.</i> (1986)</a>. Structure of the two genes coding for polypeptide chain elongation factor 1 alpha (EF-1 alpha) from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3066688" target=_blank>Sandbaken MG, <i>et al.</i> (1988 Dec)</a>. Mutations in elongation factor EF-1 alpha affect the frequency of frameshifting and amino acid misincorporation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3214489" target=_blank>Miyazaki M, <i>et al.</i> (1988 Mar)</a>. Peptide elongation factor 1 from yeasts: purification and biochemical characterization of peptide elongation factors 1 alpha and 1 beta (gamma) from Saccharomyces carlsbergensis and Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2180914" target=_blank>Sundstrom P, <i>et al.</i> (1990 Apr)</a>. Sequence analysis and expression of the two genes for elongation factor 1 alpha from the dimorphic yeast Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8476932" target=_blank>Cavallius J, <i>et al.</i> (1993 Apr 21)</a>. Characterization of yeast EF-1 alpha: non-conservation of post-translational modifications.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916691" target=_blank>Schaaff-Gerstenschläger I, <i>et al.</i> (1993 Oct 1)</a>. TKL2, a second transketolase gene of Saccharomyces cerevisiae. Cloning, sequence and deletion analysis of the gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8107673" target=_blank>Steiner S, <i>et al.</i> (1994 Feb)</a>. Sequence and promoter analysis of the highly expressed TEF gene of the filamentous fungus Ashbya gossypii.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9509572" target=_blank>Norbeck J, <i>et al.</i> (1997 Dec)</a>. Two-dimensional electrophoretic separation of yeast proteins using a non-linear wide range (pH 3-10) immobilized pH gradient in the first dimension; reproducibility and evidence for isoelectric focusing of alkaline (pI &amp;gt; 7) proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125210" target=_blank>Yanagihara C, <i>et al.</i> (1997 Mar 17)</a>. Association of elongation factor 1 alpha and ribosomal protein L3 with the proline-rich region of yeast adenylyl cyclase-associated protein CAP.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9099890" target=_blank>Mita K, <i>et al.</i> (1997 Mar 18)</a>. Comprehensive cloning of Schizosaccharomyces pombe genes encoding translation elongation factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9591785" target=_blank>Umikawa M, <i>et al.</i> (1998 Apr 16)</a>. Interaction of Rho1p target Bni1p with F-actin-binding elongation factor 1alpha: implication in Rho1p-regulated reorganization of the actin cytoskeleton in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852145" target=_blank>Gangwani L, <i>et al.</i> (1998 Dec 14)</a>. Interaction of ZPR1 with translation elongation factor-1alpha in proliferating cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990316" target=_blank>Kovalchuke O, <i>et al.</i> (1998 Dec 15)</a>. Competition and cooperation amongst yeast elongation factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802206" target=_blank>Müller S, <i>et al.</i> (1998 Oct)</a>. Comparison of expression systems in the yeasts Saccharomyces cerevisiae, Hansenula polymorpha, Klyveromyces lactis, Schizosaccharomyces pombe and Yarrowia lipolytica. Cloning of two novel promoters from Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9786872" target=_blank>Cavallius J, <i>et al.</i> (1998 Oct 30)</a>. Site-directed mutagenesis of yeast eEF1A. Viable mutants with altered nucleotide specificity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10593605" target=_blank>Rasmussen C, <i>et al.</i> (1999)</a>. Cloning of a Schizosaccharomyces pombe homologue of elongation factor 1 alpha by two-hybrid selection of calmodulin-binding proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10514524" target=_blank>Carr-Schmid A, <i>et al.</i> (1999 Oct 15)</a>. Mutations in a GTP-binding motif of eukaryotic elongation factor 1A reduce both translational fidelity and the requirement for nucleotide exchange.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10526238" target=_blank>Suda M, <i>et al.</i> (1999 Sep)</a>. Overproduction of elongation factor 1alpha, an essential translational component, causes aberrant cell morphology by affecting the control of growth polarity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766739" target=_blank>Grosshans H, <i>et al.</i> (2000 Apr 1)</a>. An aminoacylation-dependent nuclear tRNA export pathway in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11106763" target=_blank>Andersen GR, <i>et al.</i> (2000 Nov)</a>. Structural basis for nucleotide exchange and competition with tRNA in the yeast elongation factor complex eEF1A:eEF1Balpha.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10973948" target=_blank>Zobel-Thropp P, <i>et al.</i> (2000 Nov 24)</a>. A novel post-translational modification of yeast elongation factor 1A. Methylesterification at the C terminus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11290701" target=_blank>Munshi R, <i>et al.</i> (2001 Apr)</a>. Overexpression of translation elongation factor 1A affects the organization and function of the actin cytoskeleton in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11373622" target=_blank>Andersen GR, <i>et al.</i> (2001 Jun)</a>. Crystal structures of nucleotide exchange intermediates in the eEF1A-eEF1Balpha complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748053" target=_blank>Kurtzman CP, <i>et al.</i> (2003 Jun)</a>. Phylogenetic relationships among yeasts of the 'Saccharomyces complex' determined from multigene sequence analyses.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12755685" target=_blank>Shenton D, <i>et al.</i> (2003 Sep 1)</a>. Protein S-thiolation targets glycolysis and protein synthesis in response to oxidative stress in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175657" target=_blank>Bachand F, <i>et al.</i> (2004 Jul 7)</a>. PRMT3 is a ribosomal protein methyltransferase that affects the cellular levels of ribosomal subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601860" target=_blank>Chuang SM, <i>et al.</i> (2005 Jan)</a>. Proteasome-mediated degradation of cotranslationally damaged proteins involves translation elongation factor 1A.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17203074" target=_blank>McGuire AT, <i>et al.</i> (2007 Jan 24)</a>. Cex1p is a novel cytoplasmic component of the Saccharomyces cerevisiae nuclear tRNA export machinery.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18391219" target=_blank>Dewez M, <i>et al.</i> (2008 Apr 8)</a>. The conserved Wobble uridine tRNA thiolase Ctu1-Ctu2 is required to maintain genome integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18086883" target=_blank>Calvert ME, <i>et al.</i> (2008 Feb)</a>. Phosphorylation by casein kinase 2 regulates Nap1 localization and function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20510667" target=_blank>Lipson RS, <i>et al.</i> (2010 Aug 15)</a>. Two novel methyltransferases acting upon eukaryotic elongation factor 1A in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849502" target=_blank>Visweswaraiah J, <i>et al.</i> (2011 Oct 21)</a>. Evidence that eukaryotic translation elongation factor 1A (eEF1A) binds the Gcn2 protein C terminus and inhibits Gcn2 activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522802" target=_blank>Couttas TA, <i>et al.</i> (2012 Apr)</a>. Methylation of translation-associated proteins in Saccharomyces cerevisiae: Identification of methylated lysines and their methyltransferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24279325" target=_blank>Ravin NV, <i>et al.</i> (2013 Nov 27)</a>. Genome sequence and analysis of methylotrophic yeast Hansenula polymorpha DL1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24818994" target=_blank>Jongjitwimol J, <i>et al.</i> (2014)</a>. The S. pombe translation initiation factor eIF4G is Sumoylated and associates with the SUMO protease Ulp2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25446118" target=_blank>Dzialo MC, <i>et al.</i> (2014 Dec 12)</a>. A new type of protein lysine methyltransferase trimethylates Lys-79 of elongation factor 1A.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24634168" target=_blank>Bernal M, <i>et al.</i> (2014 Jun)</a>. Proteome-wide search for PP2A substrates in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24517342" target=_blank>Hart-Smith G, <i>et al.</i> (2014 Mar 7)</a>. Stoichiometry of Saccharomyces cerevisiae lysine methylation: insights into non-histone protein lysine methyltransferase activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26115316" target=_blank>Jakobsson ME, <i>et al.</i> (2015)</a>. Saccharomyces cerevisiae Eukaryotic Elongation Factor 1A (eEF1A) Is Methylated at Lys-390 by a METTL21-Like Methyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26545399" target=_blank>Hamey JJ, <i>et al.</i> (2016 Jan)</a>. Novel N-terminal and Lysine Methyltransferases That Target Translation Elongation Factor 1A in Yeast and Human.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27168121" target=_blank>Chen JS, <i>et al.</i> (2016 Sep)</a>. Discovery of genes involved in mitosis, cell division, cell wall integrity and chromosome segregation through construction of Schizosaccharomyces pombe deletion strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03725_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03725_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03725_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03725_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01296_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03726<br>EOG8ZW3T8<br><br>sce:SUP35<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Translation termination factor eRF3; has a role in mRNA deadenylation and decay; altered protein conformation creates the [PSI(+)] prion that modifies cellular fitness, alters translational fidelity by affecting reading frame selection, and results in a nonsense suppressor phenotype; many stress-response genes are repressed in the presence of [PSI(+)]<br><br>
			<br><b>PomBase Description</b><br>translation release factor class II eRF3<br><br>
			<br><b>AspGD Description</b><br>Suppressor protein 2<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3556215" target=_blank>Kushnirov VV, <i>et al.</i> (1987 May 11)</a>. Localization of possible functional domains in sup2 gene product of the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2841115" target=_blank>Kikuchi Y, <i>et al.</i> (1988 Apr)</a>. A yeast gene required for the G1-to-S transition encodes a protein containing an A-kinase target site and GTPase domain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3280807" target=_blank>Wilson PG, <i>et al.</i> (1988 Feb 20)</a>. SUF12 suppressor protein of yeast. A fusion protein related to the EF-1 family of elongation factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047009" target=_blank>Kushnirov VV, <i>et al.</i> (1988 Jun 15)</a>. Nucleotide sequence of the SUP2 (SUP35) gene of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556078" target=_blank>Stansfield I, <i>et al.</i> (1995 Sep 1)</a>. The products of the SUP45 (eRF1) and SUP35 genes interact to mediate translation termination in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8978027" target=_blank>Derkatch IL, <i>et al.</i> (1996 Dec)</a>. Genesis and variability of [PSI] prion factors in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670813" target=_blank>Paushkin SV, <i>et al.</i> (1996 Jun 17)</a>. Propagation of the yeast prion-like [psi+] determinant is mediated by oligomerization of the SUP35-encoded polypeptide chain release factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335589" target=_blank>Derkatch IL, <i>et al.</i> (1997 Oct)</a>. Genetic and environmental factors affecting the de novo appearance of the [PSI+] prion in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11511345" target=_blank>Derkatch IL, <i>et al.</i> (2001 Jul 27)</a>. Prions affect the appearance of other prions: the story of [PIN(+)].<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11430816" target=_blank>Nakayashiki T, <i>et al.</i> (2001 Jun)</a>. Yeast [PSI+] "prions" that are crosstransmissible and susceptible beyond a species barrier through a quasi-prion state.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11971964" target=_blank>Cosson B, <i>et al.</i> (2002 May)</a>. Poly(A)-binding protein acts in translation termination via eukaryotic release factor 3 interaction and does not influence [PSI(+)] propagation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923185" target=_blank>Hosoda N, <i>et al.</i> (2003 Oct 3)</a>. Translation termination factor eRF3 mediates mRNA decay through the regulation of deadenylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15029195" target=_blank>King CY, <i>et al.</i> (2004 Mar 18)</a>. Protein-only transmission of three yeast prion strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15029196" target=_blank>Tanaka M, <i>et al.</i> (2004 Mar 18)</a>. Conformational variations in an infectious protein determine prion strain differences.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15525991" target=_blank>Amrani N, <i>et al.</i> (2004 Nov 4)</a>. A faux 3'-UTR promotes aberrant termination and triggers nonsense-mediated mRNA decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15337765" target=_blank>Kobayashi T, <i>et al.</i> (2004 Oct 29)</a>. The GTP-binding release factor eRF3 as a key mediator coupling translation termination to mRNA decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15944695" target=_blank>Nelson R, <i>et al.</i> (2005 Jun 9)</a>. Structure of the cross-beta spine of amyloid-like fibrils.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170131" target=_blank>Shewmaker F, <i>et al.</i> (2006 Dec 26)</a>. Amyloid of the prion domain of Sup35p has an in-register parallel beta-sheet structure.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16809762" target=_blank>Keeling KM, <i>et al.</i> (2006 Jul)</a>. Tpa1p is part of an mRNP complex that influences translation termination, mRNA deadenylation, and mRNA turnover in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19917766" target=_blank>Lancaster AK, <i>et al.</i> (2010 Feb)</a>. The spontaneous appearance rate of the yeast prion [PSI+] and its implications for the evolution of the evolvability properties of the [PSI+] system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24727290" target=_blank>Liu W, <i>et al.</i> (2014 Apr 11)</a>. In Aspergillus nidulans the suppressors suaA and suaC code for release factors eRF1 and eRF3 and suaD codes for a glutamine tRNA.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03726_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03726_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03726_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03726_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01296_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03727<br>EOG8ZW3T8<br><br>sce:TUF1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mitochondrial translation elongation factor Tu; comprises both GTPase and guanine nucleotide exchange factor activities, while these activities are found in separate proteins in S. pombe and humans<br><br>
			<br><b>PomBase Description</b><br>mitochondrial translation elongation factor EF-Tu Tuf1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have GTPase activity, translation elongation factor activity, role in mitochondrial translational elongation and mitochondrion localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6353412" target=_blank>Nagata S, <i>et al.</i> (1983 Oct)</a>. Molecular cloning and sequence determination of the nuclear gene coding for mitochondrial elongation factor Tu of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3905388" target=_blank>Myers AM, <i>et al.</i> (1985 Aug)</a>. Mitochondrial protein synthesis is required for maintenance of intact mitochondrial genomes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7593000" target=_blank>Branda SS, <i>et al.</i> (1995 Nov 10)</a>. Prediction and identification of new natural substrates of the yeast mitochondrial intermediate peptidase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03727_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03727_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03727_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03727_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01296_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03728<br>EOG8ZW3T8<br><br>sce:HBS1<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>GTPase with similarity to translation release factors; together with binding partner Dom34p, facilitates ribosomal subunit dissociation and peptidyl-tRNA release when translation is stalled, particularly in 3' UTRs; genetically implicated in mRNA no-go decay; HBS1 has a paralog, SKI7, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>elongation factor 1 alpha related protein Hbs1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1394434" target=_blank>Nelson RJ, <i>et al.</i> (1992 Oct 2)</a>. The translation machinery and 70 kd heat shock protein cooperate in protein synthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11909951" target=_blank>Carr-Schmid A, <i>et al.</i> (2002 Apr)</a>. Novel G-protein complex whose requirement is linked to the translational status of the cell.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16554824" target=_blank>Doma MK, <i>et al.</i> (2006 Mar 23)</a>. Endonucleolytic cleavage of eukaryotic mRNAs with stalls in translation elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20890290" target=_blank>Chen L, <i>et al.</i> (2010 Oct)</a>. Structure of the Dom34-Hbs1 complex and implications for no-go decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28945192" target=_blank>Guydosh NR, <i>et al.</i> (2017 Sep 25)</a>. Regulated Ire1-dependent mRNA decay requires no-go mRNA degradation to maintain endoplasmic reticulum homeostasis in &lt;i&gt;S. pombe&lt;/i&gt;.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03728_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03728_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03728_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03728_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01296_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01296.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>