<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01170.html">HOG01170&emsp;&emsp;small GTPase superfamily. Rab family</a><br> 
																	HOG01171&emsp;&emsp;<br> 
																	<a href="HOG01172.html">HOG01172&emsp;&emsp;TRAFAC class myosin-kinesin ATPase superfamily. Myosin family; protein kinase superfamily. CAMK Ser/Thr protein kinase family. CHEK2 subfamily; protein kinase superfamily. CAMK Ser/Thr protein kinase family. CaMK subfamily; protein kinase superfamily. CAMK Ser/Thr protein kinase family. NIM1 subfamily; protein kinase superfamily. CAMK Ser/Thr protein kinase family. SNF1 subfamily; protein kinase superfamily. Ser/Thr protein kinase family; protein kinase superfamily. Ser/Thr protein kinase family. Aurora subfamily; protein kinase superfamily. Ser/Thr protein kinase family. CDC5/Polo subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01171.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01171_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01171_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02930<br>EOG8P8D3R<br><br>sce:CDC34<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin-conjugating enzyme (E2); catalytic subunit of SCF ubiquitin-protein ligase complex (together with Skp1p, Rbx1p, Cdc53p, and an F-box protein) that regulates cell cycle progression by targeting key substrates for degradation; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have protein homodimerization activity, ubiquitin protein ligase activity, ubiquitin-protein transferase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842867" target=_blank>Goebl MG, <i>et al.</i> (1988 Sep 9)</a>. The yeast cell cycle gene CDC34 encodes a ubiquitin-conjugating enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312054" target=_blank>Drury LS, <i>et al.</i> (1997 Oct 1)</a>. The Cdc4/34/53 pathway targets Cdc6p for proteolysis in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9499404" target=_blank>Patton EE, <i>et al.</i> (1998 Mar 1)</a>. Cdc53 is a scaffold protein for multiple Cdc34/Skp1/F-box proteincomplexes that regulate cell division and methionine biosynthesis in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10385629" target=_blank>Seol JH, <i>et al.</i> (1999 Jun 15)</a>. Cdc53/cullin and the essential Hrt1 RING-H2 subunit of SCF define a ubiquitin ligase module that activates the E2 enzyme Cdc34.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02930_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02930_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02930_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02930_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02931<br>EOG8P8D3R<br><br>sce:UBC4;UBC5<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin-conjugating enzyme (E2); key E2 partner with Ubc1p for the anaphase-promoting complex (APC); mediates degradation of abnormal or excess proteins, including calmodulin and histone H3; regulates levels of DNA Polymerase-{alpha} to promote efficient and accurate DNA replication; interacts with many SCF ubiquitin protein ligases; component of the cellular stress response; UBC4 has a paralog, UBC5, that arose from the whole genome duplication|Ubiquitin-conjugating enzyme; mediates selective degradation of short-lived, abnormal, or excess proteins, including histone H3; central component of the cellular stress response; expression is heat inducible; protein abundance increases in response to DNA replication stress; UBC5 has a paralog, UBC4, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme E2 Ubc4/UbcP1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have APC-Cdc20 complex activity, protein binding, bridging, ubiquitin binding, ubiquitin conjugating enzyme activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2154373" target=_blank>Seufert W, <i>et al.</i> (1990 Feb)</a>. Ubiquitin-conjugating enzymes UBC4 and UBC5 mediate selective degradation of short-lived and abnormal proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8268156" target=_blank>Cook WJ, <i>et al.</i> (1993 Dec 21)</a>. Tertiary structures of class I ubiquitin-conjugating enzymes are highly conserved: crystal structure of yeast Ubc4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11788821" target=_blank>Reggiori F, <i>et al.</i> (2002 Feb)</a>. A transmembrane ubiquitin ligase required to sort membrane proteins into multivesicular bodies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17550898" target=_blank>Williams C, <i>et al.</i> (2007 Aug 3)</a>. A conserved cysteine is essential for Pex4p-dependent ubiquitination of the peroxisomal import receptor Pex5p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02931_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02931_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02931_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02931_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02932<br>EOG8P8D3R<br><br>sce:UBC8<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin-conjugating enzyme that regulates gluconeogenesis; negatively regulates gluconeogenesis by mediating the glucose-induced ubiquitination of fructose-1,6-bisphosphatase (FBPase); cytoplasmic enzyme that catalyzes the ubiquitination of histones in vitro<br><br>
			<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme E2 Ubc8 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted acid-amino acid ligase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1869573" target=_blank>Qin S, <i>et al.</i> (1991 Aug 15)</a>. Cloning and characterization of a Saccharomyces cerevisiae gene encoding a new member of the ubiquitin-conjugating protein family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8132613" target=_blank>Kaiser P, <i>et al.</i> (1994 Mar 25)</a>. A human ubiquitin-conjugating enzyme homologous to yeast UBC8.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10811607" target=_blank>Schüle T, <i>et al.</i> (2000 May 15)</a>. Ubc8p functions in catabolite degradation of fructose-1, 6-bisphosphatase in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12686616" target=_blank>Regelmann J, <i>et al.</i> (2003 Apr)</a>. Catabolite degradation of fructose-1,6-bisphosphatase in the yeast Saccharomyces cerevisiae: a genome-wide screen identifies eight novel GID genes and indicates the existence of two degradation pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23416107" target=_blank>Olsen SK, <i>et al.</i> (2013 Mar 7)</a>. Structure of a ubiquitin E1-E2 complex: insights to E1-E2 thioester transfer.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27168121" target=_blank>Chen JS, <i>et al.</i> (2016 Sep)</a>. Discovery of genes involved in mitosis, cell division, cell wall integrity and chromosome segregation through construction of Schizosaccharomyces pombe deletion strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28162934" target=_blank>Lv Z, <i>et al.</i> (2017 Feb 16)</a>. S. pombe Uba1-Ubc15 Structure Reveals a Novel Regulatory Mechanism of Ubiquitin E2 Activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02932_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02932_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02932_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02932_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02933<br>EOG8P8D3R<br><br>sce:RAD6<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin-conjugating enzyme (E2); involved in postreplication repair as a heterodimer with Rad18p, regulation of K63 polyubiquitination in response to oxidative stress, DSBR and checkpoint control as a heterodimer with Bre1p, ubiquitin-mediated N-end rule protein degradation as a heterodimer with Ubr1p, as well as endoplasmic reticulum-associated protein degradation (ERAD) with Ubr1p in the absence of canonical ER membrane ligases<br><br>
			<br><b>PomBase Description</b><br>Rad6 homolog, ubiquitin conjugating enzyme E2 Rhp6<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have single-stranded DNA binding, single-stranded DNA-dependent ATPase activity, ubiquitin conjugating enzyme activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7038392" target=_blank>Montelone BA, <i>et al.</i> (1981)</a>. Recombination and mutagenesis in rad6 mutants of Saccharomyces cerevisiae: evidence for multiple functions of the RAD6 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3881753" target=_blank>Reynolds P, <i>et al.</i> (1985 Jan)</a>. RAD6 gene of Saccharomyces cerevisiae encodes a protein containing a tract of 13 consecutive aspartates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3306404" target=_blank>Jentsch S, <i>et al.</i> (1987 Sep 10-16)</a>. The yeast DNA repair gene RAD6 encodes a ubiquitin-conjugating enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2157209" target=_blank>Sung P, <i>et al.</i> (1990 Apr)</a>. Mutation of cysteine-88 in the Saccharomyces cerevisiae RAD6 protein abolishes its ubiquitin-conjugating activity and its various biological functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2179869" target=_blank>Madura K, <i>et al.</i> (1990 Feb 25)</a>. Expression of the Saccharomyces cerevisiae DNA repair gene RAD6 that encodes a ubiquitin conjugating enzyme, increases in response to DNA damage and in meiosis but remains constant during the mitotic cell cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2065660" target=_blank>Sung P, <i>et al.</i> (1991 Aug)</a>. Yeast RAD6 encoded ubiquitin conjugating enzyme mediates protein degradation dependent on the N-end-recognizing E3 enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1651502" target=_blank>Dohmen RJ, <i>et al.</i> (1991 Aug 15)</a>. The N-end rule is mediated by the UBC2(RAD6) ubiquitin-conjugating enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8436296" target=_blank>Watkins JF, <i>et al.</i> (1993 Feb)</a>. The extremely conserved amino terminus of RAD6 ubiquitin-conjugating enzyme is essential for amino-end rule-dependent protein degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7926769" target=_blank>Bailly V, <i>et al.</i> (1994 Apr 1)</a>. Specific complex formation between yeast RAD6 and RAD18 proteins: a potential mechanism for targeting RAD6 ubiquitin-conjugating activity to DNA damage sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9234674" target=_blank>Feuermann M, <i>et al.</i> (1997 Jul)</a>. The characterization of two new clusters of duplicated genes suggests a 'Lego' organization of the yeast Saccharomyces cerevisiae chromosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9343433" target=_blank>Huang H, <i>et al.</i> (1997 Nov)</a>. The ubiquitin-conjugating enzyme Rad6 (Ubc2) is required for silencing in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9287349" target=_blank>Bailly V, <i>et al.</i> (1997 Sep 12)</a>. Yeast DNA repair proteins Rad6 and Rad18 form a heterodimer that has ubiquitin conjugating, DNA binding, and ATP hydrolytic activities.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9497353" target=_blank>Worthylake DK, <i>et al.</i> (1998 Mar 13)</a>. Crystal structure of the Saccharomyces cerevisiae ubiquitin-conjugating enzyme Rad6 at 2.6 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10581257" target=_blank>Xie Y, <i>et al.</i> (1999 Dec 1)</a>. The E2-E3 interaction in the N-end rule pathway: the RING-H2 finger of E3 is required for the synthesis of multiubiquitin chain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10880451" target=_blank>Ulrich HD, <i>et al.</i> (2000 Jul 3)</a>. Two RING finger proteins mediate cooperation between ubiquitin-conjugating enzymes in DNA repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712706" target=_blank>Leng P, <i>et al.</i> (2000 Mar)</a>. Rad6p represses yeast-hypha morphogenesis in the human fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077605" target=_blank>Sun ZW, <i>et al.</i> (2002 Jul 4)</a>. Ubiquitination of histone H2B regulates H3 methylation and gene silencing in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12226657" target=_blank>Hoege C, <i>et al.</i> (2002 Sep 12)</a>. RAD6-dependent DNA repair is linked to modification of PCNA by ubiquitin and SUMO.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15388802" target=_blank>de Padula M, <i>et al.</i> (2004)</a>. The post-replication repair RAD18 and RAD6 genes are involved in the prevention of spontaneous mutations caused by 7,8-dihydro-8-oxoguanine in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14752010" target=_blank>Kao CF, <i>et al.</i> (2004 Jan 15)</a>. Rad6 plays a role in transcriptional activation through ubiquitylation of histone H2B.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15632065" target=_blank>Xiao T, <i>et al.</i> (2005 Jan)</a>. Histone H2B ubiquitylation is associated with elongating RNA polymerase II.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16247017" target=_blank>Zhang H, <i>et al.</i> (2005 Nov 1)</a>. The error-free component of the RAD6/RAD18 DNA damage tolerance pathway of budding yeast employs sister-strand recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16307922" target=_blank>Wood A, <i>et al.</i> (2005 Nov 23)</a>. The Bur1/Bur2 complex is required for histone H2B monoubiquitination by Rad6/Bre1 and histone methylation by COMPASS.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02933_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02933_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02933_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02933_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG02934<br>EOG8P8D3R<br><br>sce:UBC1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin-conjugating enzyme; key E2 partner with Ubc4p for the anaphase-promoting complex (APC); mediates selective degradation of short-lived and abnormal proteins; plays a role in vesicle biogenesis and ER-associated protein degradation (ERAD); component of the cellular stress response; protein abundance increases in response to DNA replication stress key E2 partner with Ubc4p for the anaphase-promoting complex (APC)<br><br>
			<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme Ubc1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ubiquitin-protein transferase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265617" target=_blank>Seufert W, <i>et al.</i> (1990 Dec)</a>. UBC1 encodes a novel member of an essential subfamily of yeast ubiquitin-conjugating enzymes involved in protein degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10878801" target=_blank>Friedlander R, <i>et al.</i> (2000 Jul)</a>. A regulatory link between ER-associated protein degradation and the unfolded-protein response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11146622" target=_blank>Bays NW, <i>et al.</i> (2001 Jan)</a>. Hrd1p/Der3p is a membrane-anchored ubiquitin ligase required for ER-associated degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11591345" target=_blank>Hamilton KS, <i>et al.</i> (2001 Oct)</a>. Structure of a conjugating enzyme-ubiquitin thiolester intermediate reveals a novel role for the ubiquitin tail.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15328341" target=_blank>Merkley N, <i>et al.</i> (2004 Nov 5)</a>. Solution structure of the flexible class II ubiquitin-conjugating enzyme Ubc1 provides insights for polyubiquitin chain assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02934_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02934_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02934_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02934_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG02935<br>EOG8P8D3R<br><br>sce:UBC13<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>E2 ubiquitin-conjugating enzyme; involved in the error-free DNA postreplication repair pathway; interacts with Mms2p to assemble ubiquitin chains at the Ub Lys-63 residue; DNA damage triggers redistribution from the cytoplasm to the nucleus<br><br>
			<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme E2 Ubc13<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ubiquitin conjugating enzyme activity, ubiquitin-protein transferase activity and role in UV-damage excision repair, free ubiquitin chain polymerization, postreplication repair, protein K63-linked ubiquitination<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576256" target=_blank>Matuschewski K, <i>et al.</i> (1996 Feb 2)</a>. Identification of a novel family of ubiquitin-conjugating enzymes with distinct amino-terminal extensions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11440714" target=_blank>VanDemark AP, <i>et al.</i> (2001 Jun 15)</a>. Molecular insights into polyubiquitin chain assembly: crystal structure of the Mms2/Ubc13 heterodimer.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12531016" target=_blank>Brown M, <i>et al.</i> (2002 Nov 3)</a>. Structural and functional conservation of error-free DNA postreplication repair in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12951513" target=_blank>Onishi M, <i>et al.</i> (2003 Aug)</a>. Isolation of suppressor mutants of phosphatidylinositol 3-phosphate 5-kinase deficient cells in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998526" target=_blank>Goldman GH, <i>et al.</i> (2004 Apr)</a>. Aspergillus nidulans as a model system to characterize the DNA damage response in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16641370" target=_blank>Frampton J, <i>et al.</i> (2006 Jul)</a>. Postreplication repair and PCNA modification in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20537132" target=_blank>Han TX, <i>et al.</i> (2010)</a>. Global fitness profiling of fission yeast deletion strains by barcode sequencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20453833" target=_blank>Coulon S, <i>et al.</i> (2010 Jun 16)</a>. Rad8Rad5/Mms2-Ubc13 ubiquitin ligase complex controls translesion synthesis in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20176980" target=_blank>Dolan WP, <i>et al.</i> (2010 May)</a>. Fission yeast Hsk1 (Cdc7) kinase is required after replication initiation for induced mutagenesis and proper response to DNA alkylation damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21247416" target=_blank>Gatti L, <i>et al.</i> (2011 Jan 19)</a>. Ubiquitin-proteasome genes as targets for modulation of cisplatin sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21183410" target=_blank>Vejrup-Hansen R, <i>et al.</i> (2011 Mar 7)</a>. Schizosaccharomyces pombe Mms1 channels repair of perturbed replication into Rhp51 independent homologous recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22064477" target=_blank>Groocock LM, <i>et al.</i> (2012 Jan)</a>. The RecQ4 orthologue Hrq1 is critical for DNA interstrand cross-link repair and genome stability in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24265825" target=_blank>Køhler JB, <i>et al.</i> (2013)</a>. Concerted action of the ubiquitin-fusion degradation protein 1 (Ufd1) and Sumo-targeted ubiquitin ligases (STUbLs) in the DNA-damage response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24192486" target=_blank>Fontebasso Y, <i>et al.</i> (2013 Dec)</a>. The conserved Fanconi anemia nuclease Fan1 and the SUMO E3 ligase Pli1 act in two novel Pso2-independent pathways of DNA interstrand crosslink repair in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23416107" target=_blank>Olsen SK, <i>et al.</i> (2013 Mar 7)</a>. Structure of a ubiquitin E1-E2 complex: insights to E1-E2 thioester transfer.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25293972" target=_blank>Mastro TL, <i>et al.</i> (2014 Dec)</a>. Increased meiotic crossovers and reduced genome stability in absence of Schizosaccharomyces pombe Rad16 (XPF).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24687850" target=_blank>Tapia-Alveal C, <i>et al.</i> (2014 Jun)</a>. H2A.Z-dependent regulation of cohesin dynamics on chromosome arms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26365378" target=_blank>Liu XM, <i>et al.</i> (2015 Sep 17)</a>. ESCRTs Cooperate with a Selective Autophagy Receptor to Mediate Vacuolar Targeting of Soluble Cargos.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27151298" target=_blank>Álvarez V, <i>et al.</i> (2016 May 6)</a>. Orderly progression through S-phase requires dynamic ubiquitylation and deubiquitylation of PCNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27473316" target=_blank>Ranatunga NS, <i>et al.</i> (2016 Oct 13)</a>. Characterization of a Novel MMS-Sensitive Allele of Schizosaccharomyces pombe mcm4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28162934" target=_blank>Lv Z, <i>et al.</i> (2017 Feb 16)</a>. S. pombe Uba1-Ubc15 Structure Reveals a Novel Regulatory Mechanism of Ubiquitin E2 Activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28718400" target=_blank>Yadav RK, <i>et al.</i> (2017 Jul 18)</a>. Histone H3G34R mutation causes replication stress, homologous recombination defects and genomic instability in &lt;i&gt;S. pombe&lt;/i&gt;.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28481910" target=_blank>Daigaku Y, <i>et al.</i> (2017 May)</a>. PCNA ubiquitylation ensures timely completion of unperturbed DNA replication in fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02935_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02935_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02935_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02935_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG02936<br>EOG8P8D3R<br><br>sce:UBC9<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>SUMO-conjugating enzyme involved in the Smt3p conjugation pathway; nuclear protein required for S- and M-phase cyclin degradation and mitotic control; involved in proteolysis mediated by the anaphase-promoting complex cyclosome (APCC)<br><br>
			<br><b>PomBase Description</b><br>SUMO conjugating enzyme E2 Hus5<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have SUMO transferase activity, role in hyphal growth, mitotic spindle elongation, protein sumoylation and condensed nuclear chromosome, cytosol, nuclear heterochromatin localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7800043" target=_blank>Seufert W, <i>et al.</i> (1995 Jan 5)</a>. Role of a ubiquitin-conjugating enzyme in degradation of S- and M-phase cyclins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9341106" target=_blank>Johnson ES, <i>et al.</i> (1997 Oct 24)</a>. Ubc9p is the conjugating enzyme for the ubiquitin-like protein Smt3p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11587849" target=_blank>Takahashi Y, <i>et al.</i> (2001 Sep 19)</a>. A novel factor required for the SUMO1/Smt3 conjugation of yeast septins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11572779" target=_blank>Johnson ES, <i>et al.</i> (2001 Sep 21)</a>. An E3-like factor that promotes SUMO conjugation to the yeast septins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24279728" target=_blank>Harting R, <i>et al.</i> (2013 Dec)</a>. Interplay of the fungal sumoylation network for control of multicellular development.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02936_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02936_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02936_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02936_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG02937<br>EOG8P8D3R<br><br>sce:UBC7<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin conjugating enzyme; involved in the ER-associated protein degradation pathway; requires Cue1p for recruitment to the ER membrane; proposed to be involved in chromatin assembly<br><br>
			<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme E2 Ubc7/UbcP3<br><br>
			<br><b>AspGD Description</b><br>Putative ubiquitin conjugating enzyme; expression enhanced by maltose<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327148" target=_blank>Vassal A, <i>et al.</i> (1992 Sep 24)</a>. QRI8, a novel ubiquitin-conjugating enzyme in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8381213" target=_blank>Jungmann J, <i>et al.</i> (1993 Jan 28)</a>. Resistance to cadmium mediated by ubiquitin-dependent proteolysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8393731" target=_blank>Chen P, <i>et al.</i> (1993 Jul 30)</a>. Multiple ubiquitin-conjugating enzymes participate in the in vivo degradation of the yeast MAT alpha 2 repressor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9172777" target=_blank>Yamazaki RK, <i>et al.</i> (1996 Feb)</a>. Bacterial expression of the Saccharomyces cerevisiae ubiquitin-conjugating enzyme Ubc7.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9388185" target=_blank>Biederer T, <i>et al.</i> (1997 Dec 5)</a>. Role of Cue1p in ubiquitination and degradation at the ER surface.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9048545" target=_blank>Cook WJ, <i>et al.</i> (1997 Feb 18)</a>. Crystal structure of a class I ubiquitin conjugating enzyme (Ubc7) from Saccharomyces cerevisiae at 2.9 angstroms resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9695950" target=_blank>Johnson PR, <i>et al.</i> (1998 Jul 24)</a>. Degradation signal masking by heterodimerization of MATalpha2 and MATa1 blocks their mutual destruction by the ubiquitin-proteasome pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11146622" target=_blank>Bays NW, <i>et al.</i> (2001 Jan)</a>. Hrd1p/Der3p is a membrane-anchored ubiquitin ligase required for ER-associated degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11390656" target=_blank>Gardner RG, <i>et al.</i> (2001 Jul)</a>. In vivo action of the HRD ubiquitin ligase complex: mechanisms of endoplasmic reticulum quality control and sterol regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11641273" target=_blank>Swanson R, <i>et al.</i> (2001 Oct 15)</a>. A conserved ubiquitin ligase of the nuclear envelope/endoplasmic reticulum that functions in both ER-associated and Matalpha2 repressor degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870366" target=_blank>Sims AH, <i>et al.</i> (2005 May)</a>. Transcriptome analysis of recombinant protein secretion by Aspergillus nidulans and the unfolded-protein response in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16179953" target=_blank>Neuber O, <i>et al.</i> (2005 Oct)</a>. Ubx2 links the Cdc48 complex to ER-associated protein degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16873066" target=_blank>Carvalho P, <i>et al.</i> (2006 Jul 28)</a>. Distinct ubiquitin-ligase complexes define convergent pathways for the degradation of ER proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17310239" target=_blank>Ravid T, <i>et al.</i> (2007 Apr)</a>. Autoregulation of an E2 enzyme by ubiquitin-chain assembly on its catalytic residue.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02937_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02937_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02937_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02937_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG02938<br>EOG8P8D3R<br><br>sce:PEX4<br><br>Genes: 30
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Peroxisomal ubiquitin conjugating enzyme; required for peroxisomal matrix protein import and peroxisome biogenesis<br><br>
			<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme E2 Ubc16 (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1326082" target=_blank>Wiebel FF, <i>et al.</i> (1992 Sep 3)</a>. The Pas2 protein essential for peroxisome biogenesis is related to ubiquitin-conjugating enzymes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17550898" target=_blank>Williams C, <i>et al.</i> (2007 Aug 3)</a>. A conserved cysteine is essential for Pex4p-dependent ubiquitination of the peroxisomal import receptor Pex5p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21247416" target=_blank>Gatti L, <i>et al.</i> (2011 Jan 19)</a>. Ubiquitin-proteasome genes as targets for modulation of cisplatin sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02938_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02938_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02938_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02938_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG02939<br>EOG8P8D3R<br><br>sce:UBC11<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin-conjugating enzyme; most similar in sequence to Xenopus ubiquitin-conjugating enzyme E2-C, but not a true functional homolog of this E2; unlike E2-C, not required for the degradation of mitotic cyclin Clb2<br><br>
			<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme E2-C, Ubc11/UbcP4<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have APC-Cdc20 complex activity, ubiquitin conjugating enzyme activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9675819" target=_blank>Townsley FM, <i>et al.</i> (1998 Jun 15)</a>. Functional analysis of the Saccharomyces cerevisiae UBC11 gene.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02939_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02939_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02939_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02939_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG02940<br>EOG8P8D3R<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme E2 Ubc14 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative ubiquitin conjugating enzyme; induced by growth on starch and lactate<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21247416" target=_blank>Gatti L, <i>et al.</i> (2011 Jan 19)</a>. Ubiquitin-proteasome genes as targets for modulation of cisplatin sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23416107" target=_blank>Olsen SK, <i>et al.</i> (2013 Mar 7)</a>. Structure of a ubiquitin E1-E2 complex: insights to E1-E2 thioester transfer.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26890608" target=_blank>Gaspa L, <i>et al.</i> (2016)</a>. A functional genome-wide genetic screening identifies new pathways controlling the G1/S transcriptional wave.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28162934" target=_blank>Lv Z, <i>et al.</i> (2017 Feb 16)</a>. S. pombe Uba1-Ubc15 Structure Reveals a Novel Regulatory Mechanism of Ubiquitin E2 Activity.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02940_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02940_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02940_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02940_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG02941<br>EOG8P8D3R<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>ubiquitin conjugating enzyme E2 Ubc15<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in chromatin silencing at silent mating-type cassette<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12417737" target=_blank>Choi ES, <i>et al.</i> (2002 Dec)</a>. Two ubiquitin-conjugating enzymes, Rhp6 and UbcX, regulate heterochromatin silencing in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870366" target=_blank>Sims AH, <i>et al.</i> (2005 May)</a>. Transcriptome analysis of recombinant protein secretion by Aspergillus nidulans and the unfolded-protein response in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23050226" target=_blank>Chen Z, <i>et al.</i> (2012 Oct)</a>. A genetic screen to discover pathways affecting cohesin function in Schizosaccharomyces pombe identifies chromatin effectors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25165823" target=_blank>Tsutsui Y, <i>et al.</i> (2014 Aug)</a>. Multiple regulation of Rad51-mediated homologous recombination by fission yeast Fbh1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28162934" target=_blank>Lv Z, <i>et al.</i> (2017 Feb 16)</a>. S. pombe Uba1-Ubc15 Structure Reveals a Novel Regulatory Mechanism of Ubiquitin E2 Activity.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02941_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02941_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02941_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02941_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG02942<br>EOG8P8D3R<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted acid-amino acid ligase activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02942_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02942_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02942_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02942_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01171_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01171.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>