<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00244.html">HOG00244&emsp;&emsp;short-chain dehydrogenases/reductases (SDR) family</a><br> 
																	HOG00245&emsp;&emsp;<br> 
																	<a href="HOG00246.html">HOG00246&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00245.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00245_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00245_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00245_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00245_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00558<br>EOG83R252<br>GPD1<br>sce:GPD1<br><br>Genes: 27
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cytoplasmic NAD-dependent glycerol-3-phosphate dehydrogenase<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>SGD Description</b><br>NAD-dependent glycerol-3-phosphate dehydrogenase; key enzyme of glycerol synthesis, essential for growth under osmotic stress; expression regulated by high-osmolarity glycerol response pathway; protein abundance increases in response to DNA replication stress; constitutively inactivated via phosphorylation by the protein kinases YPK1 and YPK2, dephosphorylation increases catalytic activity; GPD1 has a paralog, GPD2, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>glycerol-3-phosphate dehydrogenase Gpd1|glycerol-3-phosphate dehydrogenase Gpd2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have glycerol-3-phosphate dehydrogenase [NAD+] activity and role in NADH oxidation, cell-abiotic substrate adhesion, glycerol biosynthetic process, intracellular accumulation of glycerol<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1676389" target=_blank>Sleep D, <i>et al.</i> (1991 May 15)</a>. Cloning and characterisation of the Saccharomyces cerevisiae glycerol-3-phosphate dehydrogenase (GUT2) promoter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934860" target=_blank>Larsson K, <i>et al.</i> (1993 Dec)</a>. A gene encoding sn-glycerol 3-phosphate dehydrogenase (NAD+) complements an osmosensitive mutant of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8196651" target=_blank>Albertyn J, <i>et al.</i> (1994 Jun)</a>. GPD1, which encodes glycerol-3-phosphate dehydrogenase, is essential for growth under osmotic stress in Saccharomyces cerevisiae, and its expression is regulated by the high-osmolarity glycerol response pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961476" target=_blank>Wang HT, <i>et al.</i> (1994 Nov)</a>. Cloning, sequence, and disruption of the Saccharomyces diastaticus DAR1 gene encoding a glycerol-3-phosphate dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8825100" target=_blank>Ohmiya R, <i>et al.</i> (1995 Dec)</a>. Osmoregulation of fission yeast: cloning of two distinct genes encoding glycerol-3-phosphate dehydrogenase, one of which is responsible for osmotolerance for growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7498541" target=_blank>Aiba H, <i>et al.</i> (1995 Dec 4)</a>. The osmo-inducible gpd1+ gene is a target of the signaling pathway involving Wis1 MAP-kinase kinase in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8935650" target=_blank>Norbeck J, <i>et al.</i> (1996 Mar 15)</a>. Protein expression during exponential growth in 0.7 M NaCl medium of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704325" target=_blank>Yamada H, <i>et al.</i> (1996 May)</a>. Construction and characterization of a deletion mutant of gpd2 that encodes an isozyme of NADH-dependent glycerol-3-phosphate dehydrogenase in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123696" target=_blank>Fillinger S, <i>et al.</i> (2001 Jan)</a>. Molecular and physiological characterization of the NAD-dependent glycerol 3-phosphate dehydrogenase in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12823816" target=_blank>de Vries RP, <i>et al.</i> (2003 Jul)</a>. Glycerol dehydrogenase, encoded by gldB is essential for osmotolerance in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15210723" target=_blank>Valadi A, <i>et al.</i> (2004 Sep 17)</a>. Distinct intracellular localization of Gpd1p and Gpd2p, the two yeast isoforms of NAD+-dependent glycerol-3-phosphate dehydrogenase, explains their different contributions to redox-driven glycerol production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773992" target=_blank>Aguilera J, <i>et al.</i> (2005 Apr)</a>. The HOG MAP kinase pathway is required for the induction of methylglyoxal-responsive genes and determines methylglyoxal resistance in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17420584" target=_blank>Hagiwara D, <i>et al.</i> (2007 Apr)</a>. The SskA and SrrA response regulators are implicated in oxidative stress responses of hyphae and asexual spores in the phosphorelay signaling network of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18703157" target=_blank>Miskei M, <i>et al.</i> (2009 Mar)</a>. Annotation of stress-response proteins in the aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20625380" target=_blank>Liu NN, <i>et al.</i> (2010 Aug)</a>. A genome-wide screen for Schizosaccharomyces pombe deletion mutants that affect telomere length.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20131067" target=_blank>Balázs A, <i>et al.</i> (2010 Mar)</a>. AtfA bZIP-type transcription factor regulates oxidative and osmotic stress responses in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27887640" target=_blank>Malecki M, <i>et al.</i> (2016 Nov 25)</a>. Functional and regulatory profiling of energy metabolism in fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00558_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00558_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00558_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00558_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00245_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00559<br>EOG83R252<br>GPD2<br>sce:GPD2<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial NAD-dependent glycerol-3-phosphate dehydrogenase<br><br>
			<br><b>Features</b><br>[m]<br><br>
			<br><b>Parent</b><br>paralog:FOG00558<br><br>
			<br><b>SGD Description</b><br>NAD-dependent glycerol 3-phosphate dehydrogenase; expression is controlled by an oxygen-independent signaling pathway required to regulate metabolism under anoxic conditions; located in cytosol and mitochondria; constitutively active but is inactivated via phosphorylation by energy-stress responsive kinase SNF1; GPD2 has a paralog, GPD1, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476212" target=_blank>Eriksson P, <i>et al.</i> (1995 Jul)</a>. Cloning and characterization of GPD2, a second gene encoding sn-glycerol 3-phosphate dehydrogenase (NAD+) in Saccharomyces cerevisiae, and its comparison with GPD1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8979347" target=_blank>Björkqvist S, <i>et al.</i> (1997 Jan)</a>. Physiological response to anaerobicity of glycerol-3-phosphate dehydrogenase mutants of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171333" target=_blank>Ansell R, <i>et al.</i> (1997 May 1)</a>. The two isoenzymes for yeast NAD+-dependent glycerol 3-phosphate dehydrogenase encoded by GPD1 and GPD2 have distinct roles in osmoadaptation and redox regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15210723" target=_blank>Valadi A, <i>et al.</i> (2004 Sep 17)</a>. Distinct intracellular localization of Gpd1p and Gpd2p, the two yeast isoforms of NAD+-dependent glycerol-3-phosphate dehydrogenase, explains their different contributions to redox-driven glycerol production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00559_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00559_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00559_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00559_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00245_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00560<br>EOG83R252<br>GPD3<br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial NAD-dependent glycerol-3-phosphate dehydrogenase<br><br>
			<br><b>Features</b><br>[m]<br><br>
			<br><b>Parent</b><br>paralog:FOG00558<br><br>
			<br><b>AspGD Description</b><br>Glycerol 3-phosphate dehydrogenase (NAD+ dependent)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17617716" target=_blank>Furukawa K, <i>et al.</i> (2007 Jul)</a>. Novel reporter gene expression systems for monitoring activation of the Aspergillus nidulans HOG pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18838792" target=_blank>Hagiwara D, <i>et al.</i> (2008 Oct)</a>. Characterization of bZip-type transcription factor AtfA with reference to stress responses of conidia of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18703157" target=_blank>Miskei M, <i>et al.</i> (2009 Mar)</a>. Annotation of stress-response proteins in the aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19596074" target=_blank>Hagiwara D, <i>et al.</i> (2009 Nov)</a>. Transcriptional profiling for Aspergillusnidulans HogA MAPK signaling pathway in response to fludioxonil and osmotic stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20131067" target=_blank>Balázs A, <i>et al.</i> (2010 Mar)</a>. AtfA bZIP-type transcription factor regulates oxidative and osmotic stress responses in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00560_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00560_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00560_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00560_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00245_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00245.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>