<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG02947.html">HOG02947&emsp;&emsp;</a><br> 
																	HOG02948&emsp;&emsp;<br> 
																	<a href="HOG02949.html">HOG02949&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG02948.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG02948_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG02948_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG02948_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG02948_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG06732<br>EOG8JM66K<br><br>sce:HDA1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative catalytic subunit of a class II histone deacetylase complex; role in azole resistance via Hsp90p, and in the heat shock response; Hda1p interacts with the Hda2p-Hda3p subcomplex to form an active tetramer; deletion increases histone H2B, H3 and H4 acetylation; other members of the HDA1 histone deacetylase complex are Hda2p and Hda3p<br><br>
			<br><b>PomBase Description</b><br>histone deacetylase (class II) Clr3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have histone deacetylase activity (H3-K14 specific), histone deacetylase activity (H3-K9 specific) activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8962081" target=_blank>Rundlett SE, <i>et al.</i> (1996 Dec 10)</a>. HDA1 and RPD3 are members of distinct yeast histone deacetylase complexes that regulate silencing and transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443097" target=_blank>Srikantha T, <i>et al.</i> (2001 Aug)</a>. The histone deacetylase genes HDA1 and RPD3 play distinct roles in regulation of high-frequency phenotypic switching in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11404352" target=_blank>Klar AJ, <i>et al.</i> (2001 Jun)</a>. A histone deacetylation inhibitor and mutant promote colony-type switching of the human pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12853613" target=_blank>Trojer P, <i>et al.</i> (2003 Jul 15)</a>. Histone deacetylases in fungi: novel members, new facts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12798685" target=_blank>Tebarth B, <i>et al.</i> (2003 Jun 20)</a>. Adaptation of the Efg1p morphogenetic pathway in Candida albicans by negative autoregulation and PKA-dependent repression of the EFG1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16215180" target=_blank>Tribus M, <i>et al.</i> (2005 Oct)</a>. HdaA, a major class 2 histone deacetylase of Aspergillus nidulans, affects growth under conditions of oxidative stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616629" target=_blank>Shwab EK, <i>et al.</i> (2007 Sep)</a>. Histone deacetylase activity regulates chemical diversity in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20730094" target=_blank>Zacchi LF, <i>et al.</i> (2010 Aug 13)</a>. HOS2 and HDA1 encode histone deacetylases with opposing roles in Candida albicans morphogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20724388" target=_blank>Palmer JM, <i>et al.</i> (2010 Dec)</a>. Telomere position effect is regulated by heterochromatin-associated proteins and NkuA in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19940017" target=_blank>Tribus M, <i>et al.</i> (2010 Jan 15)</a>. A novel motif in fungal class 1 histone deacetylases is essential for growth and development of Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20219692" target=_blank>Shaaban M, <i>et al.</i> (2010 May)</a>. Involvement of transposon-like elements in penicillin gene cluster regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20433937" target=_blank>Brakhage AA, <i>et al.</i> (2011 Jan)</a>. Fungal secondary metabolites - strategies to activate silent gene clusters.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21811397" target=_blank>Lu Y, <i>et al.</i> (2011 Jul)</a>. Hyphal development in Candida albicans requires two temporally linked changes in promoter chromatin for initiation and maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22536157" target=_blank>Lu Y, <i>et al.</i> (2012)</a>. A GATA transcription factor recruits Hda1 in response to reduced Tor1 signaling to establish a hyphal chromatin state in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23041319" target=_blank>Robbins N, <i>et al.</i> (2012 Oct 25)</a>. Lysine deacetylases Hda1 and Rpd3 regulate Hsp90 function thereby governing fungal drug resistance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25980340" target=_blank>Hedtke M, <i>et al.</i> (2015 Aug)</a>. Light-dependent gene activation in Aspergillus nidulans is strictly dependent on phytochrome and involves the interplay of phytochrome and white collar-regulated histone H3 acetylation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06732_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06732_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06732_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG06732_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG02948_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG06733<br>EOG8JM66K<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06733_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06733_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06733_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG06733_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG02948_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG02948.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>