<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00562.html">HOG00562&emsp;&emsp;APS kinase family; FMP10 family; sulfate adenylyltransferase family</a><br> 
																	HOG00564&emsp;&emsp;<br> 
																	<a href="HOG00565.html">HOG00565&emsp;&emsp;NADPH--cytochrome P450 reductase family; NADPH-dependent diflavin oxidoreductase NDOR1 family; flavodoxin family; flavoprotein pyridine nucleotide cytochrome reductase family; nitrite and sulfite reductase 4Fe-4S domain family</a><br><br> 
																	<a href="../sequences/hog/HOG00564.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG00564_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG00564_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG00564_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG00564_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01359<br>EOG8573RM<br><br>sce:MET16<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>3'-phosphoadenylsulfate reductase; reduces 3'-phosphoadenylyl sulfate to adenosine-3',5'-bisphosphate and free sulfite using reduced thioredoxin as cosubstrate, involved in sulfate assimilation and methionine metabolism<br><br>
			<br><b>PomBase Description</b><br>phosphoadenosine phosphosulfate reductase<br><br>
			<br><b>AspGD Description</b><br>PAPS reductase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5671427" target=_blank>Arst HN Jr, <i>et al.</i> (1968 Jul 20)</a>. Genetic analysis of the first steps of sulphate metabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4934393" target=_blank>Gravel RA, <i>et al.</i> (1970 Dec)</a>. Genetic and accumulation studies in sulfite-requiring mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4581424" target=_blank>Clutterbuck AJ, <i>et al.</i> (1973 Jun)</a>. Gene symbols in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203779" target=_blank>Thomas D, <i>et al.</i> (1990 Sep 15)</a>. Gene-enzyme relationship in the sulfate assimilation pathway of Saccharomyces cerevisiae. Study of the 3'-phosphoadenylylsulfate reductase structural gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765129" target=_blank>Paszewski A, <i>et al.</i> (1994)</a>. Sulphur metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7770049" target=_blank>Borges-Walmsley MI, <i>et al.</i> (1995 May 20)</a>. Isolation and characterisation of genes for sulphate activation and reduction in Aspergillus nidulans: implications for evolution of an allosteric control region by gene duplication.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11057445" target=_blank>Schierová M, <i>et al.</i> (2000 Oct)</a>. Sulfate assimilation in Aspergillus terreus: analysis of genes encoding ATP-sulfurylase and PAPS-reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16436428" target=_blank>Fujita Y, <i>et al.</i> (2006 Feb)</a>. Homocysteine accumulation causes a defect in purine biosynthesis: further characterization of Schizosaccharomyces pombe methionine auxotrophs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17622533" target=_blank>Ma Y, <i>et al.</i> (2007 Aug)</a>. Six new amino acid-auxotrophic markers for targeted gene integration and disruption in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630328" target=_blank>Breakspear A, <i>et al.</i> (2007 Sep)</a>. Aspergillus nidulans conidiation genes dewA, fluG, and stuA are differentially regulated in early vegetative growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18665268" target=_blank>Zuin A, <i>et al.</i> (2008 Jul 30)</a>. Mitochondrial dysfunction increases oxidative stress and decreases chronological life span in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18684775" target=_blank>Kennedy PJ, <i>et al.</i> (2008 Nov)</a>. A genome-wide screen of genes involved in cadmium tolerance in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20955810" target=_blank>Brzywczy J, <i>et al.</i> (2011 Feb)</a>. Novel mutations reveal two important regions in Aspergillus nidulans transcriptional activator MetR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21169418" target=_blank>Henry TC, <i>et al.</i> (2011 Feb)</a>. Systematic screen of Schizosaccharomyces pombe deletion collection uncovers parallel evolution of the phosphate signal transduction pathway in yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24118096" target=_blank>García-Santamarina S, <i>et al.</i> (2013 Dec)</a>. Methionine sulphoxide reductases revisited: free methionine as a primary target of H₂O₂stress in auxotrophic fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27558664" target=_blank>Guo L, <i>et al.</i> (2016 Oct 13)</a>. Global Fitness Profiling Identifies Arsenic and Cadmium Tolerance Mechanisms in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01359_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01359_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01359_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01359_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00564_phobius.html#FOG01359">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG00564.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>