<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01552.html">HOG01552&emsp;&emsp;NST1 family</a><br> 
																	HOG01553&emsp;&emsp;<br> 
																	<a href="HOG01554.html">HOG01554&emsp;&emsp;TAF12 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01553.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01553_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01553_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01553_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG04426<br>EOG8V9S66<br><br>sce:CDC53<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cullin; structural protein of SCF complexes (which also contain Skp1p, Cdc34p, Hrt1p and an F-box protein) involved in ubiquitination; SCF promotes the G1-S transition by targeting G1 cyclins and the Cln-CDK inhibitor Sic1p for degradation<br><br>
			<br><b>PomBase Description</b><br>cullin 1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in SCF-dependent proteasomal ubiquitin-dependent protein catabolic process, negative regulation of mitotic DNA replication initiation and cytosol, nuclear SCF ubiquitin ligase complex localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7813440" target=_blank>Kornitzer D, <i>et al.</i> (1994 Dec 15)</a>. Regulated degradation of the transcription factor Gcn4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8943317" target=_blank>Mathias N, <i>et al.</i> (1996 Dec)</a>. Cdc53p acts in concert with Cdc4p and Cdc34p to control the G1-to-S-phase transition and identifies a conserved family of proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312054" target=_blank>Drury LS, <i>et al.</i> (1997 Oct 1)</a>. The Cdc4/34/53 pathway targets Cdc6p for proteolysis in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9346239" target=_blank>Feldman RM, <i>et al.</i> (1997 Oct 17)</a>. A complex of Cdc4p, Skp1p, and Cdc53p/cullin catalyzes ubiquitination of the phosphorylated CDK inhibitor Sic1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9346238" target=_blank>Skowyra D, <i>et al.</i> (1997 Oct 17)</a>. F-box proteins are receptors that recruit phosphorylated substrates to the SCF ubiquitin-ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9499404" target=_blank>Patton EE, <i>et al.</i> (1998 Mar 1)</a>. Cdc53 is a scaffold protein for multiple Cdc34/Skp1/F-box proteincomplexes that regulate cell division and methionine biosynthesis in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9736614" target=_blank>Jaquenoud M, <i>et al.</i> (1998 Sep 15)</a>. The Cdc42p effector Gic2p is targeted for ubiquitin-dependent degradation by the SCFGrr1 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10213692" target=_blank>Skowyra D, <i>et al.</i> (1999 Apr 23)</a>. Reconstitution of G1 cyclin ubiquitination with complexes containing SCFGrr1 and Rbx1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10385629" target=_blank>Seol JH, <i>et al.</i> (1999 Jun 15)</a>. Cdc53/cullin and the essential Hrt1 RING-H2 subunit of SCF define a ubiquitin ligase module that activates the E2 enzyme Cdc34.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11283612" target=_blank>Seol JH, <i>et al.</i> (2001 Apr)</a>. Skp1 forms multiple protein complexes, including RAVE, a regulator of V-ATPase assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14747994" target=_blank>Kus BM, <i>et al.</i> (2004 Feb 15)</a>. Functional interaction of 13 yeast SCF complexes with a set of yeast E2 enzymes in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16421250" target=_blank>Koepp DM, <i>et al.</i> (2006 Apr)</a>. The F-box protein Dia2 regulates DNA replication.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19942853" target=_blank>Siergiejuk E, <i>et al.</i> (2009 Dec 16)</a>. Cullin neddylation and substrate-adaptors counteract SCF inhibition by the CAND1-like protein Lag2 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19763088" target=_blank>Liu Y, <i>et al.</i> (2009 Nov 4)</a>. A longevity protein, Lag2, interacts with SCF complex and regulates SCF function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22329854" target=_blank>von Zeska Kress MR, <i>et al.</i> (2012 Mar)</a>. The COP9 signalosome counteracts the accumulation of cullin SCF ubiquitin E3 RING ligases during fungal development.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04426_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04426_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04426_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04426_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01553_phobius.pdf">Phobius transmembrane predictions</a>
										<br>8 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG04427<br>EOG8V9S66<br><br>sce:CUL3<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ubiquitin-protein ligase; forms a complex with Elc1p that polyubiquitylates monoubiquitylated RNA polymerase II to trigger its proteolysis; cullin family member with similarity to Cdc53p and human CUL3<br><br>
			<br><b>PomBase Description</b><br>cullin 3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in hyphal growth and Cul3-RING ubiquitin ligase complex, cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990507" target=_blank>Kominami K, <i>et al.</i> (1998 Nov)</a>. Two F-box/WD-repeat proteins Pop1 and Pop2 form hetero- and homo-complexes together with cullin-1 in the fission yeast SCF (Skp1-Cullin-1-F-box) ubiquitin ligase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11504566" target=_blank>Zhou C, <i>et al.</i> (2001)</a>. The fission yeast COP9/signalosome is involved in cullin modification by ubiquitin-related Ned8p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12718879" target=_blank>Zhou C, <i>et al.</i> (2003 Apr)</a>. Fission yeast COP9/signalosome suppresses cullin activity through recruitment of the deubiquitylating enzyme Ubp12p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12676951" target=_blank>Michel JJ, <i>et al.</i> (2003 Jun 20)</a>. A role for Saccharomyces cerevisiae Cul8 ubiquitin ligase in proper anaphase progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14527422" target=_blank>Geyer R, <i>et al.</i> (2003 Sep)</a>. BTB/POZ domain proteins are putative substrate adaptors for cullin 3 ubiquitin ligases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15769255" target=_blank>Zhou L, <i>et al.</i> (2005 Jul 15)</a>. Nep1, a Schizosaccharomyces pombe deneddylating enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16252005" target=_blank>Liu C, <i>et al.</i> (2005 Nov 16)</a>. Transactivation of Schizosaccharomyces pombe cdt2+ stimulates a Pcu4-Ddb1-CSN ubiquitin ligase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17296727" target=_blank>Ribar B, <i>et al.</i> (2007 Apr)</a>. ELA1 and CUL3 are required along with ELC1 for RNA polymerase II polyubiquitylation and degradation in DNA-damaged yeast cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18931302" target=_blank>Dixon SJ, <i>et al.</i> (2008 Oct 28)</a>. Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19264558" target=_blank>Deshpande GP, <i>et al.</i> (2009 May 1)</a>. Screening a genome-wide S. pombe deletion library identifies novel genes and pathways involved in genome stability maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20537132" target=_blank>Han TX, <i>et al.</i> (2010)</a>. Global fitness profiling of fission yeast deletion strains by barcode sequencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20625380" target=_blank>Liu NN, <i>et al.</i> (2010 Aug)</a>. A genome-wide screen for Schizosaccharomyces pombe deletion mutants that affect telomere length.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21247416" target=_blank>Gatti L, <i>et al.</i> (2011 Jan 19)</a>. Ubiquitin-proteasome genes as targets for modulation of cisplatin sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22329854" target=_blank>von Zeska Kress MR, <i>et al.</i> (2012 Mar)</a>. The COP9 signalosome counteracts the accumulation of cullin SCF ubiquitin E3 RING ligases during fungal development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23950735" target=_blank>Sun LL, <i>et al.</i> (2013)</a>. Global analysis of fission yeast mating genes reveals new autophagy factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24192486" target=_blank>Fontebasso Y, <i>et al.</i> (2013 Dec)</a>. The conserved Fanconi anemia nuclease Fan1 and the SUMO E3 ligase Pli1 act in two novel Pso2-independent pathways of DNA interstrand crosslink repair in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27887640" target=_blank>Malecki M, <i>et al.</i> (2016 Nov 25)</a>. Functional and regulatory profiling of energy metabolism in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27558664" target=_blank>Guo L, <i>et al.</i> (2016 Oct 13)</a>. Global Fitness Profiling Identifies Arsenic and Cadmium Tolerance Mechanisms in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04427_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04427_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04427_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04427_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01553_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG04428<br>EOG8V9S66<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>cullin 4<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular protein localization, chromatin silencing at centromere, chromatin silencing at silent mating-type cassette and chromatin silencing at telomere, more<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10880460" target=_blank>Osaka F, <i>et al.</i> (2000 Jul 3)</a>. Covalent modifier NEDD8 is essential for SCF ubiquitin-ligase in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10759889" target=_blank>Ding DQ, <i>et al.</i> (2000 Mar)</a>. Large-scale screening of intracellular protein localization in living fission yeast cells by the use of a GFP-fusion genomic DNA library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12695334" target=_blank>Liu C, <i>et al.</i> (2003 May 1)</a>. Cop9/signalosome subunits and Pcu4 regulate ribonucleotide reductase by both checkpoint-dependent and -independent mechanisms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805471" target=_blank>Holmberg C, <i>et al.</i> (2005 Apr 1)</a>. Ddb1 controls genome stability and meiosis in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16157682" target=_blank>Thon G, <i>et al.</i> (2005 Dec)</a>. The Clr7 and Clr8 directionality factors and the Pcu4 cullin mediate heterochromatin formation in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16024659" target=_blank>Horn PJ, <i>et al.</i> (2005 Jul 15)</a>. A Rik1-associated, cullin-dependent E3 ubiquitin ligase is essential for heterochromatin formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15769255" target=_blank>Zhou L, <i>et al.</i> (2005 Jul 15)</a>. Nep1, a Schizosaccharomyces pombe deneddylating enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17114925" target=_blank>Hong EJ, <i>et al.</i> (2005 Jul-Sep)</a>. A cullin E3 ubiquitin ligase complex associates with Rik1 and the Clr4 histone H3-K9 methyltransferase and is required for RNAi-mediated heterochromatin formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16252005" target=_blank>Liu C, <i>et al.</i> (2005 Nov 16)</a>. Transactivation of Schizosaccharomyces pombe cdt2+ stimulates a Pcu4-Ddb1-CSN ubiquitin ligase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16127433" target=_blank>Jia S, <i>et al.</i> (2005 Oct)</a>. Ubiquitin ligase component Cul4 associates with Clr4 histone methyltransferase to assemble heterochromatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16407326" target=_blank>Hansen KR, <i>et al.</i> (2006)</a>. Evolutionary-conserved telomere-linked helicase genes of fission yeast are repressed by silencing factors, RNAi components and the telomere-binding protein Taz1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18794354" target=_blank>Fukumoto Y, <i>et al.</i> (2008 Nov)</a>. Schizosaccharomyces pombe Ddb1 recruits substrate-specific adaptor proteins through a novel protein motif, the DDB-box.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211136" target=_blank>Bayne EH, <i>et al.</i> (2010 Mar 5)</a>. Stc1: a critical link between RNAi and chromatin modification required for heterochromatin integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21060862" target=_blank>Shanker S, <i>et al.</i> (2010 Oct 28)</a>. Continuous requirement for the Clr4 complex but not RNAi for centromeric heterochromatin assembly in fission yeast harboring a disrupted RITS complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21183410" target=_blank>Vejrup-Hansen R, <i>et al.</i> (2011 Mar 7)</a>. Schizosaccharomyces pombe Mms1 channels repair of perturbed replication into Rhp51 independent homologous recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21340088" target=_blank>Arita Y, <i>et al.</i> (2011 May)</a>. Microarray-based target identification using drug hypersensitive fission yeast expressing ORFeome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22329854" target=_blank>von Zeska Kress MR, <i>et al.</i> (2012 Mar)</a>. The COP9 signalosome counteracts the accumulation of cullin SCF ubiquitin E3 RING ligases during fungal development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25090107" target=_blank>White SA, <i>et al.</i> (2014)</a>. The RFTS domain of Raf2 is required for Cul4 interaction and heterochromatin integrity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24449894" target=_blank>Kuscu C, <i>et al.</i> (2014 Feb 4)</a>. CRL4-like Clr4 complex in Schizosaccharomyces pombe depends on an exposed surface of Dos1 for heterochromatin silencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26988418" target=_blank>Wang J, <i>et al.</i> (2016 Apr 1)</a>. The proper connection between shelterin components is required for telomeric heterochromatin assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27168121" target=_blank>Chen JS, <i>et al.</i> (2016 Sep)</a>. Discovery of genes involved in mitosis, cell division, cell wall integrity and chromosome segregation through construction of Schizosaccharomyces pombe deletion strains.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04428_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04428_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04428_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04428_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01553_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01553.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>