<html>
<head><title>HOG01052</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01051.html">HOG01051&emsp;&emsp;ENY2 family; archaeal/bacterial/fungal opsin family; oligopeptide OPT transporter family</a><br> 
																	HOG01052&emsp;&emsp;<br> 
																	<a href="HOG01053.html">HOG01053&emsp;&emsp;cation transport ATPase (P-type) (TC 3.A.3) family. Type IV subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01052.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01052_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01052_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG01052_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG01052.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02487<br>EOG87PVP7<br><br>sce:PRP16<br><br>Genes: 24
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>RNA degradation<br><br>
			<br><b>SGD Description</b><br>DEAH-box RNA helicase involved in second catalytic step of splicing and in exon ligation; exhibits ATP-dependent RNA unwinding activity; mediates the release of Yju2p and Cwc25p in the second step; in the absence of ATP, stabilizes the binding of Cwc25p to the spliceosome in the first catalytic step; missense mutation in human ortholog DHX38 associated with early-onset retinitis pigmentosa<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Prp16<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have U2-type spliceosomal complex localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2138057" target=_blank>Burgess S, <i>et al.</i> (1990 Mar 9)</a>. A putative ATP binding protein influences the fidelity of branchpoint recognition in yeast splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1825134" target=_blank>Schwer B, <i>et al.</i> (1991 Feb 7)</a>. PRP16 is an RNA-dependent ATPase that interacts transiently with the spliceosome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02487_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02487_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02487_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02487_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02488<br>EOG87PVP7<br><br>sce:PRP22<br><br>Genes: 24
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>RNA degradation<br><br>
			<br><b>SGD Description</b><br>DEAH-box RNA-dependent ATPase/ATP-dependent RNA helicase; associates with lariat intermediates before the second catalytic step of splicing; mediates ATP-dependent mRNA release from the spliceosome and unwinds RNA duplexes; required for proofreading the exon ligation reaction<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Prp22<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have U2-type spliceosomal complex localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1992352" target=_blank>Company M, <i>et al.</i> (1991 Feb 7)</a>. Requirement of the RNA helicase-like protein PRP22 for release of messenger RNA from spliceosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11884590" target=_blank>Ohi MD, <i>et al.</i> (2002 Apr)</a>. Proteomics analysis reveals stable multiprotein complexes in both fission and budding yeasts containing Myb-related Cdc5p/Cef1p, novel pre-mRNA splicing factors, and snRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755920" target=_blank>Carnahan RH, <i>et al.</i> (2005 Mar)</a>. Dim1p is required for efficient splicing and export of mRNA encoding lid1p, a component of the fission yeast anaphase-promoting complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21340088" target=_blank>Arita Y, <i>et al.</i> (2011 May)</a>. Microarray-based target identification using drug hypersensitive fission yeast expressing ORFeome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24014766" target=_blank>Livesay SB, <i>et al.</i> (2013 Nov)</a>. Structural and functional characterization of the N terminus of Schizosaccharomyces pombe Cwf10.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25274039" target=_blank>Bayne EH, <i>et al.</i> (2014)</a>. A systematic genetic screen identifies new factors  influencing centromeric heterochromatin integrity  in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24874881" target=_blank>Kallgren SP, <i>et al.</i> (2014)</a>. The proper splicing of RNAi factors is critical for pericentric heterochromatin assembly in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24442611" target=_blank>Chen W, <i>et al.</i> (2014 Mar)</a>. Endogenous U2·U5·U6 snRNA complexes in S. pombe are intron lariat spliceosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26167880" target=_blank>Lipp JJ, <i>et al.</i> (2015 Aug)</a>. SR protein kinases promote splicing of nonconsensus introns.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02488_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02488_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02488_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02488_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02489<br>EOG87PVP7<br><br>sce:PRP43<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>RNA degradation<br><br>
			<br><b>SGD Description</b><br>RNA helicase in the DEAH-box family; functions in both RNA polymerase I and polymerase II transcript metabolism; catalyzes removal of U2, U5, and U6 snRNPs from the postsplicing lariat-intron ribonucleoprotein complex; required for efficient biogenesis of both small- and large-subunit rRNAs; acts with Sqs1p to promote 20S to 18S rRNA processing catalyzed by endonuclease Nob1p<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Prp43<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent RNA helicase activity, mRNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9342317" target=_blank>Arenas JE, <i>et al.</i> (1997 Oct 28)</a>. Prp43: An RNA helicase-like factor involved in spliceosome disassembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16357217" target=_blank>Tsai RT, <i>et al.</i> (2005 Dec 15)</a>. Spliceosome disassembly catalyzed by Prp43 and its associated components Ntr1 and Ntr2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16880513" target=_blank>Boon KL, <i>et al.</i> (2006 Aug)</a>. Yeast ntr1/spp382 mediates prp43 function in postspliceosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22730331" target=_blank>Nie M, <i>et al.</i> (2012 Aug 24)</a>. Dual recruitment of Cdc48 (p97)-Ufd1-Npl4 ubiquitin-selective segregase by small ubiquitin-like modifier protein (SUMO) and ubiquitin in SUMO-targeted ubiquitin ligase-mediated genome stability functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24081329" target=_blank>Grenier St-Sauveur V, <i>et al.</i> (2013 Dec)</a>. Poly(A) tail-mediated gene regulation by opposing roles of Nab2 and Pab2 nuclear poly(A)-binding proteins in pre-mRNA decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24014766" target=_blank>Livesay SB, <i>et al.</i> (2013 Nov)</a>. Structural and functional characterization of the N terminus of Schizosaccharomyces pombe Cwf10.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24210919" target=_blank>Lee NN, <i>et al.</i> (2013 Nov 21)</a>. Mtr4-like protein coordinates nuclear RNA processing for heterochromatin assembly and for telomere maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24874881" target=_blank>Kallgren SP, <i>et al.</i> (2014)</a>. The proper splicing of RNAi factors is critical for pericentric heterochromatin assembly in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24442611" target=_blank>Chen W, <i>et al.</i> (2014 Mar)</a>. Endogenous U2·U5·U6 snRNA complexes in S. pombe are intron lariat spliceosomes.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02489_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02489_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02489_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02489_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02490<br>EOG87PVP7<br><br>sce:PRP2<br><br>Genes: 22
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>RNA degradation<br><br>
			<br><b>SGD Description</b><br>RNA-dependent DExD/H-box ATPase; required for activation of spliceosome before first transesterification step in RNA splicing; implicated in rearranging and proofreading snRNA structure in catalytic activation of spliceosome; ortholog of human protein DHX16<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Cdc28<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, ATP-dependent helicase activity, helicase activity, nucleic acid binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6943408" target=_blank>Nasmyth K, <i>et al.</i> (1981)</a>. Cell division cycle mutants altered in DNA replication and mitosis in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536958" target=_blank>Last RL, <i>et al.</i> (1986 Dec)</a>. Identification and nuclear localization of yeast pre-messenger RNA processing components: RNA2 and RNA3 proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16453724" target=_blank>Hirano T, <i>et al.</i> (1986 Nov)</a>. Isolation and characterization of Schizosaccharomyces pombe cutmutants that block nuclear division but not cytokinesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2147058" target=_blank>Chen JH, <i>et al.</i> (1990 Nov 11)</a>. The yeast PRP2 protein, a putative RNA-dependent ATPase, shares extensive sequence homology with two other pre-mRNA splicing factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2251118" target=_blank>King DS, <i>et al.</i> (1990 Nov 25)</a>. Interactions of PRP2 protein with pre-mRNA splicing complexes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1534753" target=_blank>Kim SH, <i>et al.</i> (1992 Jun)</a>. The purified yeast pre-mRNA splicing factor PRP2 is an RNA-dependent NTPase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8112301" target=_blank>Plumpton M, <i>et al.</i> (1994 Feb 15)</a>. A dominant negative mutation in the conserved RNA helicase motif 'SAT' causes splicing factor PRP2 to stall in spliceosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8112302" target=_blank>Teigelkamp S, <i>et al.</i> (1994 Feb 15)</a>. The splicing factor PRP2, a putative RNA helicase, interacts directly with pre-mRNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7493316" target=_blank>Roy J, <i>et al.</i> (1995 Jun)</a>. The final stages of spliceosome maturation require Spp2p that can interact with the DEAH box protein Prp2p and promote step 1 of splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8943336" target=_blank>Kim SH, <i>et al.</i> (1996 Dec)</a>. Spliceosome activation by PRP2 ATPase prior to the first transesterification reaction of pre-mRNA splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8862522" target=_blank>Lundgren K, <i>et al.</i> (1996 Jul)</a>. A connection between pre-mRNA splicing and the cell cycle in fission yeast: cdc28+ is allelic with prp8+ and encodes an RNA-dependent ATPase/helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003295" target=_blank>Urushiyama S, <i>et al.</i> (1996 Nov 27)</a>. Isolation of novel pre-mRNA splicing mutants of Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350031" target=_blank>Habara Y, <i>et al.</i> (2001 May)</a>. Mutation in the prp12+ gene encoding a homolog of SAP130/SF3b130 causes differential inhibition of pre-mRNA splicing and arrest of cell-cycle progression in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15542821" target=_blank>Silverman EJ, <i>et al.</i> (2004 Dec)</a>. Interaction between a G-patch protein and a spliceosomal DEXD/H-box ATPase that is critical for splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15620657" target=_blank>Wilkinson CR, <i>et al.</i> (2004 Dec 29)</a>. Ubiquitin-like protein Hub1 is required for pre-mRNA splicing and localization of an essential splicing factor in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730020" target=_blank>Edwalds-Gilbert G, <i>et al.</i> (2004 Feb)</a>. Definition of a spliceosome interaction domain in yeast Prp2 ATPase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16049013" target=_blank>Shimada M, <i>et al.</i> (2005 Sep 23)</a>. Regulation of Cdc2p and Cdc13p is required for cell cycle arrest induced by defective RNA splicing in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20018856" target=_blank>Chinen M, <i>et al.</i> (2010 Feb 19)</a>. Involvement of the spliceosomal U4 small nuclear RNA in heterochromatic gene silencing at fission yeast centromeres.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22240020" target=_blank>Watanabe N, <i>et al.</i> (2012 Feb 3)</a>. Characterization of the ptr5+ gene involved in nuclear mRNA export in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24014766" target=_blank>Livesay SB, <i>et al.</i> (2013 Nov)</a>. Structural and functional characterization of the N terminus of Schizosaccharomyces pombe Cwf10.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26968627" target=_blank>Mayerle M, <i>et al.</i> (2016 May)</a>. Prp8 retinitis pigmentosa mutants cause defects in the transition between the catalytic steps of splicing.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02490_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02490_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02490_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02490_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius transmembrane predictions</a>
										<br>13 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG02491<br>EOG87PVP7<br><br>sce:DHR2<br><br>Genes: 25
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>RNA degradation<br><br>
			<br><b>SGD Description</b><br>Predominantly nucleolar DEAH-box ATP-dependent RNA helicase; required for 18S rRNA synthesis<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Prh1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, ATP-dependent helicase activity, helicase activity, nucleic acid binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982841" target=_blank>Colley A, <i>et al.</i> (2000 Oct)</a>. Dhr1p, a putative DEAH-box RNA helicase, is associated with the box C+D snoRNP U3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21941128" target=_blank>Choque E, <i>et al.</i> (2011 Nov-Dec)</a>. The nucleolar protein Nop19p interacts preferentially with Utp25p and Dhr2p and is essential for the production of the 40S ribosomal subunit in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02491_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02491_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02491_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02491_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG02492<br><br><br>sce:absent<br><br>Genes: 0
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02492_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02492_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02492_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02492_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG02493<br>EOG87PVP7<br><br>sce:ECM16<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Essential DEAH-box ATP-dependent RNA helicase specific to U3 snoRNP; predominantly nucleolar in distribution; required for 18S rRNA synthesis<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Dhr1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have RNA helicase activity, role in maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) and 90S preribosome, mitochondrion, nucleolus, small-subunit processome localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982841" target=_blank>Colley A, <i>et al.</i> (2000 Oct)</a>. Dhr1p, a putative DEAH-box RNA helicase, is associated with the box C+D snoRNP U3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12068309" target=_blank>Dragon F, <i>et al.</i> (2002 Jun 27)</a>. A large nucleolar U3 ribonucleoprotein required for 18S ribosomal RNA biogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02493_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02493_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02493_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02493_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG02494<br>EOG87PVP7<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>ATP-dependent RNA helicase, spliceosomal (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, ATP-dependent helicase activity, helicase activity, nucleic acid binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18684775" target=_blank>Kennedy PJ, <i>et al.</i> (2008 Nov)</a>. A genome-wide screen of genes involved in cadmium tolerance in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24874881" target=_blank>Kallgren SP, <i>et al.</i> (2014)</a>. The proper splicing of RNAi factors is critical for pericentric heterochromatin assembly in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26173815" target=_blank>Zhou H, <i>et al.</i> (2015 Oct)</a>. Genome-wide screen of fission yeast mutants for sensitivity to 6-azauracil, an inhibitor of transcriptional elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02494_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02494_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02494_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02494_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01052_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01052.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>