<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00200.html">HOG00200&emsp;&emsp;phosphoglycerate kinase family</a><br> 
																	HOG00201&emsp;&emsp;<br> 
																	<a href="HOG00202.html">HOG00202&emsp;&emsp;cofactor-independent phosphoglycerate mutase</a><br><br> 
																	<a href="../sequences/hog/HOG00201.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG00201_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG00201_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG00201_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG00201_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00292<br>EOG8XWDGV<br>GPM1<br>sce:GPM1<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cofactor-dependent phosphoglycerate mutase (GPM3) ortholog<br><br>
			<br><b>SGD Description</b><br>Tetrameric phosphoglycerate mutase; mediates the conversion of 3-phosphoglycerate to 2-phosphoglycerate during glycolysis and the reverse reaction during gluconeogenesis<br><br>
			<br><b>PomBase Description</b><br>monomeric 2,3-bisphosphoglycerate (BPG)-dependent phosphoglycerate mutase (PGAM), Gpm1<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6115412" target=_blank>Winn SI, <i>et al.</i> (1981 Jun 26)</a>. Structure and activity of phosphoglycerate mutase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6127696" target=_blank>Fothergill LA, <i>et al.</i> (1982 Apr 22)</a>. The amino acid sequence of yeast phosphoglycerate mutase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2996957" target=_blank>Price NC, <i>et al.</i> (1985)</a>. Purification and preliminary characterization of phosphoglycerate mutase from Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2831102" target=_blank>White MF, <i>et al.</i> (1988 Mar 14)</a>. Sequence of the gene encoding phosphoglycerate mutase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1834353" target=_blank>Heinisch J, <i>et al.</i> (1991 Jul)</a>. Sequence and localization of the gene encoding yeast phosphoglycerate mutase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1386023" target=_blank>White MF, <i>et al.</i> (1992 Jul 15)</a>. Development of a mutagenesis, expression and purification system for yeast phosphoglycerate mutase. Investigation of the role of active-site His181.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8462867" target=_blank>Rodicio R, <i>et al.</i> (1993 Mar 30)</a>. Transcriptional control of yeast phosphoglycerate mutase-encoding gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8240286" target=_blank>White MF, <i>et al.</i> (1993 Nov 1)</a>. Dissociation of the tetrameric phosphoglycerate mutase from yeast by a mutation in the subunit contact region.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8110200" target=_blank>Nairn J, <i>et al.</i> (1994 Feb 1)</a>. The amino acid sequence of the small monomeric phosphoglycerate mutase from the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8765231" target=_blank>Nairn J, <i>et al.</i> (1996 Aug 15)</a>. Phosphoglycerate mutase from Schizosaccharomyces pombe: development of an expression system and characterisation of three histidine mutants of the enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9509572" target=_blank>Norbeck J, <i>et al.</i> (1997 Dec)</a>. Two-dimensional electrophoretic separation of yeast proteins using a non-linear wide range (pH 3-10) immobilized pH gradient in the first dimension; reproducibility and evidence for isoelectric focusing of alkaline (pI &amp;gt; 7) proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9544241" target=_blank>Heinisch JJ, <i>et al.</i> (1998 Feb)</a>. Investigation of two yeast genes encoding putative isoenzymes of phosphoglycerate mutase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512715" target=_blank>Rigden DJ, <i>et al.</i> (1998 Feb 20)</a>. The 2.3 A X-ray crystal structure of S. cerevisiae phosphoglycerate mutase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10369755" target=_blank>Rigden DJ, <i>et al.</i> (1999 Jun 18)</a>. Polyanionic inhibitors of phosphoglycerate mutase: combined structural and biochemical analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10064712" target=_blank>Rigden DJ, <i>et al.</i> (1999 Mar 12)</a>. Sulphate ions observed in the 2.12 A structure of a new crystal form of S. cerevisiae phosphoglycerate mutase provide insights into understanding the catalytic mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10531478" target=_blank>Crowhurst GS, <i>et al.</i> (1999 Nov)</a>. Structure of a phosphoglycerate mutase:3-phosphoglyceric acid complex at 1.7 A.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10949142" target=_blank>Pardo M, <i>et al.</i> (2000 Jul)</a>. Cross-species identification of novel Candida albicans immunogenic proteins by combination of two-dimensional polyacrylamide gel electrophoresis and mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15234973" target=_blank>Zhang Y, <i>et al.</i> (2004 Aug 27)</a>. Cofactor-independent phosphoglycerate mutase has an essential role in Caenorhabditis elegans and is conserved in parasitic nematodes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15378761" target=_blank>Pitarch A, <i>et al.</i> (2004 Oct)</a>. Proteomics-based identification of novel Candida albicans antigens for diagnosis of systemic candidiasis in patients with underlying hematological malignancies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17016471" target=_blank>Tafforeau L, <i>et al.</i> (2006 Oct 4)</a>. Repression of ergosterol level during oxidative stress by fission yeast F-box protein Pof14 independently of SCF.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16962558" target=_blank>Brandina I, <i>et al.</i> (2006 Sep-Oct)</a>. Enolase takes part in a macromolecular complex associated to mitochondria in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23956636" target=_blank>Cho SJ, <i>et al.</i> (2010 Jun)</a>. Possible Roles of LAMMER Kinase Lkh1 in Fission Yeast by Comparative Proteome Analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22984289" target=_blank>VÃ¶gtle FN, <i>et al.</i> (2012 Dec)</a>. Intermembrane space proteome of yeast mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23297348" target=_blank>Chen JS, <i>et al.</i> (2013 May)</a>. Comprehensive proteomics analysis reveals new substrates and regulators of the fission yeast clp1/cdc14 phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00292_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00292_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00292_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00292_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00201_phobius.html#FOG00292">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00293<br>EOG8XWDGV<br>GPM2<br>sce:absent<br><br>Genes: 16
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized GPM1 paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00292<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00293_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00293_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00293_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00293_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00201_phobius.html#FOG00293">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG09640<br>EOG8VQ862<br>GPM3<br>sce:GPM2;GPM3<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>[Reserved for GPM3 ohnolog]<br><br>
			<br><b>SGD Description</b><br>Homolog of Gpm1p phosphoglycerate mutase; converts 3-phosphoglycerate to 2-phosphoglycerate in glycolysis; may be non-functional; GPM2 has a paralog, GPM3, that arose from the whole genome duplication|Homolog of Gpm1p phosphoglycerate mutase; converts 3-phosphoglycerate to 2-phosphoglycerate in glycolysis; may be non-functional; GPM3 has a paralog, GPM2, that arose from the whole genome duplication<br><br>
			<br><b>Suggested Analysis</b><br>Previously assigned to different ortholog group by OrthoMCL and OrthoDB. Integrate with GPM2 ortholog.<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9544241" target=_blank>Heinisch JJ, <i>et al.</i> (1998 Feb)</a>. Investigation of two yeast genes encoding putative isoenzymes of phosphoglycerate mutase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11502169" target=_blank>Grandier-Vazeille X, <i>et al.</i> (2001 Aug 21)</a>. Yeast mitochondrial dehydrogenases are associated in a supramolecular complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG09640_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG09640_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG09640_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG09640_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00201_phobius.html#FOG09640">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG00201.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>