<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00676.html">HOG00676&emsp;&emsp;PAPS reductase family. FAD1 subfamily</a><br> 
																	HOG00677&emsp;&emsp;<br> 
																	<a href="HOG00679.html">HOG00679&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00677.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00677_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00677_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00677_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00677_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01619<br>EOG81RNB6<br>EOG870S00<br>EOG8C2FRX<br><br>sce:PHO3;PHO5;PHO11;PHO12;DIA3<br><br>Genes: 54
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Constitutively expressed acid phosphatase similar to Pho5p; brought to the cell surface by transport vesicles; hydrolyzes thiamin phosphates in the periplasmic space, increasing cellular thiamin uptake; expression is repressed by thiamin|Repressible acid phosphatase; 1 of 3 repressible acid phosphatases that also mediates extracellular nucleotide-derived phosphate hydrolysis; secretory pathway derived cell surface glycoprotein; induced by phosphate starvation and coordinately regulated by PHO4 and PHO2|One of three repressible acid phosphatases; glycoprotein that is transported to the cell surface by the secretory pathway; induced by phosphate starvation and coordinately regulated by PHO4 and PHO2; PHO11 has a paralog, PHO12, that arose from a segmental duplication|One of three repressible acid phosphatases; glycoprotein that is transported to the cell surface by the secretory pathway; pregulated by phosphate starvation; PHO12 has a paralog, PHO11, that arose from a segmental duplication|Protein of unknown function; involved in invasive and pseudohyphal growth<br><br>
			<br><b>PomBase Description</b><br>acid phosphatase (predicted)|acid phosphatase Pho1|thiamine-repressible acid phosphatase Pho4<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have thiamine phosphate phosphatase activity|Phytase (myo-inositol-hexakisphosphate phosphohydrolase); narrow pH optimum at around 2.5, very little activity at pH 5.0<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6300772" target=_blank>Arima K, <i>et al.</i> (1983 Mar 25)</a>. The nucleotide sequence of the yeast PHO5 gene: a putative precursor of repressible acid phosphatase contains a signal peptide.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6093051" target=_blank>Bajwa W, <i>et al.</i> (1984 Oct 25)</a>. Structural analysis of the two tandemly repeated acid phosphatase genes in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4092931" target=_blank>Maundrell K, <i>et al.</i> (1985)</a>. Cloning and characterization of two genes restoring acid phosphatase activity in pho1- mutants of Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3447747" target=_blank>Coddington A, <i>et al.</i> (1986)</a>. Acid phosphatase deficient mutants of Schizosaccharomyces pombe are defective in tyrosine uptake.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3005272" target=_blank>Elliott S, <i>et al.</i> (1986 Feb 25)</a>. Isolation and characterization of the structural gene for secreted acid phosphatase from Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3732265" target=_blank>Schweingruber AM, <i>et al.</i> (1986 Jul 1)</a>. Glycosylation and secretion of acid phosphatase in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537710" target=_blank>Tait-Kamradt AG, <i>et al.</i> (1986 Jun)</a>. Reciprocal regulation of the tandemly duplicated PHO5/PHO3 gene cluster within the acid phosphatase multigene family of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2381421" target=_blank>Schwaninger R, <i>et al.</i> (1990 May)</a>. Effects of seven different mutations in the pho1 gene on enzymatic activity, glycosylation and secretion of acid phosphatase in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2249257" target=_blank>Yang JW, <i>et al.</i> (1990 Oct)</a>. The structural gene coding for thiamin-repressible acid phosphatase in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1868574" target=_blank>Schweingruber AM, <i>et al.</i> (1991 Apr)</a>. Thiamine in Schizosaccharomyces pombe: dephosphorylation, intracellular pool, biosynthesis and transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1394510" target=_blank>Schweingruber ME, <i>et al.</i> (1992 Oct)</a>. Regulation of pho1-encoded acid phosphatase of Schizosaccharomyces pombe by adenine and phosphate.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7969124" target=_blank>Althoff SM, <i>et al.</i> (1994 Dec)</a>. The Srp54 GTPase is essential for protein export in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7499352" target=_blank>Fankhauser H, <i>et al.</i> (1995 Nov 24)</a>. Schizosaccharomyces pombe thiamine pyrophosphokinase is encoded by gene tnr3 and is a regulator of thiamine metabolism, phosphate metabolism, mating, and growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9860839" target=_blank>Jannatipour M, <i>et al.</i> (1998 Dec 8)</a>. Calnexin and BiP interact with acid phosphatase independently of glucose trimming and reglucosylation in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618370" target=_blank>Decottignies A, <i>et al.</i> (2003 Mar)</a>. Schizosaccharomyces pombe essential genes: a pilot study.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16278456" target=_blank>Kennedy EJ, <i>et al.</i> (2005 Nov)</a>. Pho5p and newly identified nucleotide pyrophosphatases/ phosphodiesterases regulate extracellular nucleotide phosphate metabolism in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18622392" target=_blank>Monahan BJ, <i>et al.</i> (2008 Aug)</a>. Fission yeast SWI/SNF and RSC complexes show compositional and functional differences from budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20625380" target=_blank>Liu NN, <i>et al.</i> (2010 Aug)</a>. A genome-wide screen for Schizosaccharomyces pombe deletion mutants that affect telomere length.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19933844" target=_blank>Hogan CJ, <i>et al.</i> (2010 Feb)</a>. Fission yeast Iec1-ino80-mediated nucleosome eviction regulates nucleotide and phosphate metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21850271" target=_blank>Ma Y, <i>et al.</i> (2011)</a>. Genome-wide screening for genes associated with FK506 sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21899677" target=_blank>Estrav√≠s M, <i>et al.</i> (2011 Dec)</a>. Cdc42 regulates multiple membrane traffic events in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21169418" target=_blank>Henry TC, <i>et al.</i> (2011 Feb)</a>. Systematic screen of Schizosaccharomyces pombe deletion collection uncovers parallel evolution of the phosphate signal transduction pathway in yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22730331" target=_blank>Nie M, <i>et al.</i> (2012 Aug 24)</a>. Dual recruitment of Cdc48 (p97)-Ufd1-Npl4 ubiquitin-selective segregase by small ubiquitin-like modifier protein (SUMO) and ubiquitin in SUMO-targeted ubiquitin ligase-mediated genome stability functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23050226" target=_blank>Chen Z, <i>et al.</i> (2012 Oct)</a>. A genetic screen to discover pathways affecting cohesin function in Schizosaccharomyces pombe identifies chromatin effectors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24192486" target=_blank>Fontebasso Y, <i>et al.</i> (2013 Dec)</a>. The conserved Fanconi anemia nuclease Fan1 and the SUMO E3 ligase Pli1 act in two novel Pso2-independent pathways of DNA interstrand crosslink repair in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24210919" target=_blank>Lee NN, <i>et al.</i> (2013 Nov 21)</a>. Mtr4-like protein coordinates nuclear RNA processing for heterochromatin assembly and for telomere maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24493644" target=_blank>Shah S, <i>et al.</i> (2014 Feb 1)</a>. lncRNA recruits RNAi and the exosome to dynamically regulate pho1 expression in response to phosphate levels in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24463365" target=_blank>Rallis C, <i>et al.</i> (2014 Feb 15)</a>. Systematic screen for mutants resistant to TORC1 inhibition in fission yeast reveals genes involved in cellular ageing and growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojard√≠n L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547512" target=_blank>Estill M, <i>et al.</i> (2015 May)</a>. Dissection of the PHO pathway in Schizosaccharomyces pombe using epistasis and the alternate repressor adenine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27887640" target=_blank>Malecki M, <i>et al.</i> (2016 Nov 25)</a>. Functional and regulatory profiling of energy metabolism in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01619_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01619_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01619_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01619_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00677_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01620<br>EOG8C2FRX<br><br>sce:absent<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted acid phosphatase activity|Has domain(s) with predicted acid phosphatase activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01620_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01620_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01620_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01620_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00677_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG01621<br>EOG870S00<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Phytase (myo-inositol-hexakisphosphate phosphohydrolase), extracellular enzyme that hydrolyzes myo-inositol hexakisphosphate (phytate) to myo-inositol and inorganic phosphate; inhibited by vanadate anions; pH optima at 2.5 and 5.5<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01621_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01621_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01621_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01621_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00677_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG01622<br>EOG8C2FRX<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01622_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01622_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01622_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01622_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00677_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG01623<br>EOG8C2FRX<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01623_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01623_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01623_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01623_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00677_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG01624<br>EOG8C2FRX<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Phytase; expression repressed by tunicamycin and DTT<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01624_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01624_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01624_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01624_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00677_phobius.pdf">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00677.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>