<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00210.html">HOG00210&emsp;&emsp;zinc-containing alcohol dehydrogenase family</a><br> 
																	HOG00216&emsp;&emsp;<br> 
																	<a href="HOG00219.html">HOG00219&emsp;&emsp;pyruvate carboxylase </a><br><br> 
																	<a href="../sequences/hog/HOG00216.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG00216_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG00216_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG00216_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG00216_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00347<br>EOG8P8D0N<br>ALD2<br>sce:ALD2<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD2[c] 116+<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>SGD Description</b><br>Cytoplasmic aldehyde dehydrogenase; involved in ethanol oxidation and beta-alanine biosynthesis; uses NAD+ as the preferred coenzyme; expression is stress induced and glucose repressed; very similar to Ald3p<br><br>
			<br><b>PomBase Description</b><br>aldehyde dehydrogenase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Aldehyde dehydrogenase involved in ethanol utilization; repressed by growth on starch and lactate<br><br>
			<br><b>Suggested Analysis</b><br>Check origins of spo as a xenolog. ALD2 have insertion at 11<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3158573" target=_blank>Lockington RA, <i>et al.</i> (1985)</a>. Cloning and characterization of the ethanol utilization regulon in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00347_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00347_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00347_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00347_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00347">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00348<br>EOG8P8D0N<br>ALD2.2<br>sce:absent<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD2.2 116-<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>Parent</b><br>paralog:FOG00347<br><br>
			<br><b>AspGD Description</b><br>Putative aldehyde dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00348_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00348_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00348_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00348_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00348">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00349<br>EOG8P8D0N<br>ALD2.3<br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD2.3[m] NAD (assumed)<br><br>
			<br><b>Features</b><br>[m];NAD<br><br>
			<br><b>Parent</b><br>paralog:FOG00347|FOG00348<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00349_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00349_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00349_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00349_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00349">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00350<br>EOG8P8D0N<br>ALD2.4<br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD2.4[c] NADP (assumed)<br><br>
			<br><b>Features</b><br>[m];NADP<br><br>
			<br><b>Parent</b><br>paralog:FOG00347<br><br>
			<br><b>PomBase Description</b><br>aldehyde dehydrogenase (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16348101" target=_blank>Van Urk H, <i>et al.</i> (1990 Jan)</a>. Transient-state analysis of metabolic fluxes in crabtree-positive and crabtree-negative yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23956636" target=_blank>Cho SJ, <i>et al.</i> (2010 Jun)</a>. Possible Roles of LAMMER Kinase Lkh1 in Fission Yeast by Comparative Proteome Analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22119525" target=_blank>Singh NS, <i>et al.</i> (2011 Dec 6)</a>. SIN-inhibitory phosphatase complex promotes Cdc11p dephosphorylation and propagates SIN asymmetry in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23365689" target=_blank>Tay Z, <i>et al.</i> (2013)</a>. Cellular robustness conferred by genetic crosstalk underlies resistance against chemotherapeutic drug doxorubicin in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23297348" target=_blank>Chen JS, <i>et al.</i> (2013 May)</a>. Comprehensive proteomics analysis reveals new substrates and regulators of the fission yeast clp1/cdc14 phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24463365" target=_blank>Rallis C, <i>et al.</i> (2014 Feb 15)</a>. Systematic screen for mutants resistant to TORC1 inhibition in fission yeast reveals genes involved in cellular ageing and growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26791325" target=_blank>Nguyen TT, <i>et al.</i> (2016 Jan 21)</a>. Predicting chemotherapeutic drug combinations through gene network profiling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27687866" target=_blank>Noguchi C, <i>et al.</i> (2017 Jan 2)</a>. Genetic controls of DNA damage avoidance in response to acetaldehyde in fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00350_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00350_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00350_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00350_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00350">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00351<br><br>ALD2.5<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD2.5[c] NAD (assumed)<br><br>
			<br><b>Features</b><br>[c];NAD<br><br>
			<br><b>Parent</b><br>paralog:FOG00350<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00351_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00351_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00351_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00351_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00351">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00352<br><br>ALD2.6<br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Features</b><br>[c]<br><br>
			<br><b>Parent</b><br>paralog:FOG00347|FOG00348<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00352_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00352_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00352_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00352_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00352">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00353<br>EOG8P8D0N<br>ALD2.7<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Features</b><br>[c]<br><br>
			<br><b>Parent</b><br>xenolog:FOG00352<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00353_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00353_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00353_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00353_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00353">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG00354<br>EOG8P8D0N<br>ALD3<br>sce:ALD3<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD3[c]<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>Parent</b><br>paralog:FOG00347<br><br>
			<br><b>SGD Description</b><br>Cytoplasmic aldehyde dehydrogenase; involved in beta-alanine synthesis; uses NAD+ as the preferred coenzyme; very similar to Ald2p; expression is induced by stress and repressed by glucose<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5484465" target=_blank>Clark JF, <i>et al.</i> (1970 Nov 25)</a>. Yeast aldehyde dehydrogenase. 3. Preparation of three homogeneous species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2277591" target=_blank>Millán C, <i>et al.</i> (1990)</a>. Alcohol and aldehyde dehydrogenase from Saccharomyces cerevisiae: specific activity and influence on the production of acetic acid, ethanol and higher alcohols in the first 48 h of fermentation of grape must.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10407263" target=_blank>Navarro-Aviño JP, <i>et al.</i> (1999 Jul)</a>. A proposal for nomenclature of aldehyde dehydrogenases in Saccharomyces cerevisiae and characterization of the stress-inducible ALD2 and ALD3 genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586697" target=_blank>White WH, <i>et al.</i> (2003 Jan)</a>. Specialization of function among aldehyde dehydrogenases: the ALD2 and ALD3 genes are required for beta-alanine biosynthesis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256563" target=_blank>Saint-Prix F, <i>et al.</i> (2004 Jul)</a>. Functional analysis of the ALD gene family of Saccharomyces cerevisiae during anaerobic growth on glucose: the NADP+-dependent Ald6p and Ald5p isoforms play a major role in acetate formation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00354_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00354_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00354_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00354_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00354">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG00355<br>EOG8P8D0N<br>ALD4.2<br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD4.2[c] NAD (assumed)<br><br>
			<br><b>Features</b><br>[c];NAD<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00355_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00355_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00355_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00355_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00355">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG00356<br>EOG8P8D0N<br>ALD4.3<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD4.3[m] NAD (assumed)<br><br>
			<br><b>Features</b><br>[m];NAD<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00356_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00356_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00356_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00356_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00356">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG00359<br>EOG8P8D0N<br>ALD5<br>sce:ALD5<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD5[m]<br><br>
			<br><b>Features</b><br>[m];NADP<br><br>
			<br><b>Parent</b><br>paralog:FOG00355<br><br>
			<br><b>SGD Description</b><br>Mitochondrial aldehyde dehydrogenase; involved in regulation or biosynthesis of electron transport chain components and acetate formation; activated by K+; utilizes NADP+ as the preferred coenzyme; constitutively expressed<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9059631" target=_blank>Wang X, <i>et al.</i> (1997)</a>. Saccharomyces cerevisiae aldehyde dehydrogenases. Identification and expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473035" target=_blank>Wang X, <i>et al.</i> (1998 Feb)</a>. Molecular cloning, characterization, and potential roles of cytosolic and mitochondrial aldehyde dehydrogenases in ethanol metabolism in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10585550" target=_blank>Kurita O, <i>et al.</i> (1999 Dec 15)</a>. Involvement of mitochondrial aldehyde dehydrogenase ALD5 in maintenance of the mitochondrial electron transport chain in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256563" target=_blank>Saint-Prix F, <i>et al.</i> (2004 Jul)</a>. Functional analysis of the ALD gene family of Saccharomyces cerevisiae during anaerobic growth on glucose: the NADP+-dependent Ald6p and Ald5p isoforms play a major role in acetate formation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00359_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00359_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00359_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00359_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00359">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG00358<br>EOG8P8D0N<br>ALD4<br>sce:ALD4<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD4[m] NAD and NADP; used in ethanol consumption<br><br>
			<br><b>Features</b><br>[m];NAD(P)<br><br>
			<br><b>Parent</b><br>paralog:FOG00359<br><br>
			<br><b>SGD Description</b><br>Mitochondrial aldehyde dehydrogenase; required for growth on ethanol and conversion of acetaldehyde to acetate; phosphorylated; activity is K+ dependent; utilizes NADP+ or NAD+ equally as coenzymes; expression is glucose repressed; can substitute for cytosolic NADP-dependent aldehyde dehydrogenase when directed to the cytosol<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7002547" target=_blank>Brandt WF, <i>et al.</i> (1980 Sep)</a>. The histones of yeast. The isolation and partial structure of the core histones.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6310494" target=_blank>Woudt LP, <i>et al.</i> (1983 Aug 25)</a>. The genes coding for histone H3 and H4 in Neurospora crassa are unique and contain intervening sequences.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6355483" target=_blank>Smith MM, <i>et al.</i> (1983 Sep 25)</a>. DNA sequences of yeast H3 and H4 histone genes from two non-allelic gene sets encode identical H3 and H4 proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1989592" target=_blank>Chalmers RM, <i>et al.</i> (1991 Jan 1)</a>. Comparison of benzyl alcohol dehydrogenases and benzaldehyde dehydrogenases from the benzyl alcohol and mandelate pathways in Acinetobacter calcoaceticus and from the TOL-plasmid-encoded toluene pathway in Pseudomonas putida. N-terminal amino acid sequences, amino acid compositions and immunological cross-reactions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9391047" target=_blank>Keener J, <i>et al.</i> (1997 Dec 9)</a>. Histones H3 and H4 are components of upstream activation factor required for the high-level transcription of yeast rDNA by RNA polymerase I.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9150920" target=_blank>Larsson T, <i>et al.</i> (1997 Mar-Apr)</a>. Identification of two-dimensional gel electrophoresis resolved yeast proteins by matrix-assisted laser desorption ionization mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473035" target=_blank>Wang X, <i>et al.</i> (1998 Feb)</a>. Molecular cloning, characterization, and potential roles of cytosolic and mitochondrial aldehyde dehydrogenases in ethanol metabolism in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9675847" target=_blank>Tessier WD, <i>et al.</i> (1998 Jul 1)</a>. Identification and disruption of the gene encoding the K(+)-activated acetaldehyde dehydrogenase of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11080160" target=_blank>Owen DJ, <i>et al.</i> (2000 Nov 15)</a>. The structural basis for the recognition of acetylated histone H4 by the bromodomain of histone acetyltransferase gcn5p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11502169" target=_blank>Grandier-Vazeille X, <i>et al.</i> (2001 Aug 21)</a>. Yeast mitochondrial dehydrogenases are associated in a supramolecular complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566884" target=_blank>White CL, <i>et al.</i> (2001 Sep 17)</a>. Structure of the yeast nucleosome core particle reveals fundamental changes in internucleosome interactions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256563" target=_blank>Saint-Prix F, <i>et al.</i> (2004 Jul)</a>. Functional analysis of the ALD gene family of Saccharomyces cerevisiae during anaerobic growth on glucose: the NADP+-dependent Ald6p and Ald5p isoforms play a major role in acetate formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150415" target=_blank>Zhao K, <i>et al.</i> (2004 Jun 8)</a>. Structural basis for nicotinamide cleavage and ADP-ribose transfer by NAD(+)-dependent Sir2 histone/protein deacetylases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16631569" target=_blank>Bloom K, <i>et al.</i> (2006 Apr 18)</a>. The path of DNA in the kinetochore.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16768447" target=_blank>Cosgrove MS, <i>et al.</i> (2006 Jun 20)</a>. The structural basis of sirtuin substrate affinity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289592" target=_blank>Sanders BD, <i>et al.</i> (2007 Feb 9)</a>. Structural basis for nicotinamide inhibition and base exchange in Sir2 enzymes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17761666" target=_blank>Reinders J, <i>et al.</i> (2007 Nov)</a>. Profiling phosphoproteins of yeast mitochondria reveals a role of phosphorylation in assembly of the ATP synthase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19113941" target=_blank>Zhang K, <i>et al.</i> (2009 Feb)</a>. Identification and verification of lysine propionylation and butyrylation in yeast core histones using PTMap software.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21412236" target=_blank>Zhou Z, <i>et al.</i> (2011 Apr 14)</a>. Structural basis for recognition of centromere histone variant CenH3 by the chaperone Scm3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22020126" target=_blank>Yuan H, <i>et al.</i> (2012 Jan 4)</a>. MYST protein acetyltransferase activity requires active site lysine autoacetylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22389435" target=_blank>Xie Z, <i>et al.</i> (2012 May)</a>. Lysine succinylation and lysine malonylation in histones.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27105113" target=_blank>Goudarzi A, <i>et al.</i> (2016 Apr 21)</a>. Dynamic Competing Histone H4 K5K8 Acetylation and Butyrylation Are Hallmarks of Highly Active Gene Promoters.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00358_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00358_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00358_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00358_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00358">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG00360<br>EOG8P8D0N<br>ALD5.2<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD5.2[c] NADP<br><br>
			<br><b>Features</b><br>[c];NADP<br><br>
			<br><b>Parent</b><br>paralog:FOG00359<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00360_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00360_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00360_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00360_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00360">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG00361<br>EOG8P8D0N<br>ALD6.1<br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD6.1[c] NADP<br><br>
			<br><b>Features</b><br>[c];NADP<br><br>
			<br><b>Parent</b><br>paralog:FOG00359<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00361_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00361_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00361_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00361_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00361">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG00362<br>EOG8P8D0N<br>ALD6.2<br>sce:ALD6<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD6.2[c] NADP<br><br>
			<br><b>Features</b><br>[c];NADP<br><br>
			<br><b>Parent</b><br>paralog:FOG00361<br><br>
			<br><b>SGD Description</b><br>Cytosolic aldehyde dehydrogenase; activated by Mg2+ and utilizes NADP+ as the preferred coenzyme; required for conversion of acetaldehyde to acetate; constitutively expressed; locates to the mitochondrial outer surface upon oxidative stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9038161" target=_blank>Norbeck J, <i>et al.</i> (1997 Feb 28)</a>. Metabolic and regulatory changes associated with growth of Saccharomyces cerevisiae in 1.4 M NaCl. Evidence for osmotic induction of glycerol dissimilation via the dihydroxyacetone pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9392076" target=_blank>Meaden PG, <i>et al.</i> (1997 Nov)</a>. The ALD6 gene of Saccharomyces cerevisiae encodes a cytosolic, Mg(2+)-activated acetaldehyde dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473035" target=_blank>Wang X, <i>et al.</i> (1998 Feb)</a>. Molecular cloning, characterization, and potential roles of cytosolic and mitochondrial aldehyde dehydrogenases in ethanol metabolism in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9675847" target=_blank>Tessier WD, <i>et al.</i> (1998 Jul 1)</a>. Identification and disruption of the gene encoding the K(+)-activated acetaldehyde dehydrogenase of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10407263" target=_blank>Navarro-Aviño JP, <i>et al.</i> (1999 Jul)</a>. A proposal for nomenclature of aldehyde dehydrogenases in Saccharomyces cerevisiae and characterization of the stress-inducible ALD2 and ALD3 genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10919763" target=_blank>Remize F, <i>et al.</i> (2000 Aug)</a>. Engineering of the pyruvate dehydrogenase bypass in Saccharomyces cerevisiae: role of the cytosolic Mg(2+) and mitochondrial K(+) acetaldehyde dehydrogenases Ald6p and Ald4p in acetate formation during alcoholic fermentation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12872131" target=_blank>Peng J, <i>et al.</i> (2003 Aug)</a>. A proteomics approach to understanding protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00362_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00362_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00362_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00362_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00362">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG00363<br><br>ALD6.3<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD6.3[m] NADP, Dekkera paralog<br><br>
			<br><b>Features</b><br>[m];NADP<br><br>
			<br><b>Parent</b><br>paralog:FOG00361<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00363_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00363_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00363_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00363_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00363">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG00357<br>EOG8P8D0N<br>ALD6.4<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Features</b><br>[c];NADP<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00357_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00357_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00357_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00357_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00357">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG00364<br>EOG8P8D0N<br><br>sce:absent<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity, oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor activity and role in oxidation-reduction process|Has domain(s) with predicted oxidoreductase activity, oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor activity and role in oxidation-reduction process|Has domain(s) with predicted oxidoreductase activity, oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor activity and role in oxidation-reduction process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15328121" target=_blank>Graminha MA, <i>et al.</i> (2004 Sep)</a>. Terbinafine resistance mediated by salicylate 1-monooxygenase in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00364_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00364_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00364_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00364_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00364">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9e6c3" class="col">
			FOG00365<br>EOG8P8D0N<br>UGA2<br>sce:UGA2<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cytoplasmic succinate semialdehyde dehydrogenase<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>SGD Description</b><br>Succinate semialdehyde dehydrogenase; involved in the utilization of gamma-aminobutyrate (GABA) as a nitrogen source; part of the 4-aminobutyrate and glutamate degradation pathways; localized to the cytoplasm<br><br>
			<br><b>PomBase Description</b><br>succinate-semialdehyde dehydrogenase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Succinate-semialdehyde dehydrogenase (NAD(P)+)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=778641" target=_blank>Arst HN Jr, <i>et al.</i> (1976 Jul 15)</a>. Integrator gene in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=325361" target=_blank>Arst HN Jr, <i>et al.</i> (1977 Feb 28)</a>. Some genetical aspects of ornithine metabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7007164" target=_blank>Bailey CR, <i>et al.</i> (1980 Oct)</a>. A third gene affecting GABA transaminase levels in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23740823" target=_blank>Cao J, <i>et al.</i> (2013 Jul)</a>. GABA transaminases from Saccharomyces cerevisiae and Arabidopsis thaliana complement function in cytosol and mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25092794" target=_blank>Cao J, <i>et al.</i> (2014 Oct)</a>. Characterization of the recombinant succinic semi-aldehyde dehydrogenase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00365_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00365_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00365_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00365_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00365">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dbc4e1" class="col">
			FOG00366<br>EOG8P8D0N<br>UGA2.2<br>sce:absent<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial succinate semialdehyde dehydrogenase<br><br>
			<br><b>Features</b><br>[m]<br><br>
			<br><b>Parent</b><br>paralog:FOG00365<br><br>
			<br><b>PomBase Description</b><br>succinate-semialdehyde dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00366_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00366_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00366_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00366_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00366">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fad7ad" class="col">
			FOG00367<br>EOG8P8D0N<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00367_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00367_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00367_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00367_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00367">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb9" class="col">
			FOG00368<br>EOG8P8D0N<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity, oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor activity and role in oxidation-reduction process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00368_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00368_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00368_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00368_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00368">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dfc9ba" class="col">
			FOG00369<br>EOG8P8D0N<br>ALD7<br>sce:absent<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD7 msa [m]; degradation<br><br>
			<br><b>Features</b><br>[m]<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have mitochondrion localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00369_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00369_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00369_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00369_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00369">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8d4ea" class="col">
			FOG00370<br>EOG8P8D0N<br>ALD7.2<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD7 msa [c]<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>Parent</b><br>paralog:FOG00369<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00370_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00370_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00370_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00370_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00370">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eeadad" class="col">
			FOG00371<br><br>ALD7.3<br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD7 msa [c]<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>Parent</b><br>paralog:FOG00369<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00371_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00371_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00371_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00371_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00371">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bacde4" class="col">
			FOG00372<br><br>ALD7.4<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>ALD7 msa [c]<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>Parent</b><br>ortholog?:FOG00369<br><br>
			<br><b>Suggested Analysis</b><br>Assign to existing paralog?<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00372_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00372_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00372_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00372_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00372">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c1e3bb" class="col">
			FOG00373<br>EOG8P8D0N<br>MSC7<br>sce:MSC7<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>MSC7<br><br>
			<br><b>SGD Description</b><br>Protein of unknown function; green fluorescent protein (GFP)-fusion protein localizes to the endoplasmic reticulum; msc7 mutants are defective in directing meiotic recombination events to homologous chromatids<br><br>
			<br><b>PomBase Description</b><br>aldehyde dehydrogenase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Succinate-semialdehyde dehydrogenase (NAD(P)+); similar to ER protein; expression repressed by tunicamycin and DTT<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19756047" target=_blank>Kung LA, <i>et al.</i> (2009)</a>. Global analysis of the glycoproteome in Saccharomyces cerevisiae reveals new roles for protein glycosylation in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00373_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00373_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00373_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00373_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00373">Phobius transmembrane predictions</a>
										<br>28 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7bcdd" class="col">
			FOG00374<br>EOG8P8D0N<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Parent</b><br>ohnolog?:FOG00373<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00374_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00374_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00374_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00374_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00374">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d2a2" class="col">
			FOG00375<br>EOG8P8D0N<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity, oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor activity and role in oxidation-reduction process|Putative aldehyde dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19053082" target=_blank>Shimizu M, <i>et al.</i> (2009 Jan)</a>. Proteomic analysis of Aspergillus nidulans cultured under hypoxic conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22170104" target=_blank>Terabayashi Y, <i>et al.</i> (2012 Jan)</a>. Conserved and specific responses to hypoxia in Aspergillus oryzae and Aspergillus nidulans determined by comparative transcriptomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00375_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00375_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00375_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00375_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00375">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb0" class="col">
			FOG00376<br>EOG8P8D0N<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have betaine-aldehyde dehydrogenase activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00376_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00376_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00376_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00376_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00376">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dac1b0" class="col">
			FOG00377<br>EOG8P8D0N<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>aldehyde dehydrogenase Atd3 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative aldehyde dehydrogenase|Has domain(s) with predicted oxidoreductase activity, oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor activity and role in oxidation-reduction process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11376151" target=_blank>Watanabe T, <i>et al.</i> (2001 Jun 1)</a>. Comprehensive isolation of meiosis-specific genes identifies novel proteins and unusual non-coding transcripts in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27687866" target=_blank>Noguchi C, <i>et al.</i> (2017 Jan 2)</a>. Genetic controls of DNA damage avoidance in response to acetaldehyde in fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00377_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00377_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00377_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00377_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00216_phobius.html#FOG00377">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG00216.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>