<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00904.html">HOG00904&emsp;&emsp;cytidylyltransferase family</a><br> 
																	HOG00905&emsp;&emsp;<br> 
																	<a href="HOG00906.html">HOG00906&emsp;&emsp;lysophospholipase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00905.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00905_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00905_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00905_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02100<br>EOG8THT9H<br><br>sce:EPT1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>sn-1,2-diacylglycerol ethanolamine- and cholinephosphotranferase<br><br>
			<br><b>SGD Description</b><br>sn-1,2-diacylglycerol ethanolamine- and cholinephosphotranferase; not essential for viability; EPT1 has a paralog, CPT1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>diacylglycerol cholinephosphotransferase/ diacylglycerol ethanolaminesphotransferase Ept1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have diacylglycerol cholinephosphotransferase activity, ethanolaminephosphotransferase activity, role in phosphatidylcholine biosynthetic process, phosphatidylethanolamine biosynthetic process and Golgi apparatus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2848840" target=_blank>Hjelmstad RH, <i>et al.</i> (1988 Dec 25)</a>. The sn-1,2-diacylglycerol ethanolaminephosphotransferase activity of Saccharomyces cerevisiae. Isolation of mutants and cloning of the EPT1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848238" target=_blank>Hjelmstad RH, <i>et al.</i> (1991 Mar 15)</a>. sn-1,2-diacylglycerol choline- and ethanolaminephosphotransferases in Saccharomyces cerevisiae. Nucleotide sequence of the EPT1 gene and comparison of the CPT1 and EPT1 gene products.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1847919" target=_blank>Hjelmstad RH, <i>et al.</i> (1991 Mar 5)</a>. sn-1,2-diacylglycerol choline- and ethanolaminephosphotransferases in Saccharomyces cerevisiae. Mixed micellar analysis of the CPT1 and EPT1 gene products.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961445" target=_blank>McGee TP, <i>et al.</i> (1994 Nov)</a>. Functional redundancy of CDP-ethanolamine and CDP-choline pathway enzymes in phospholipid biosynthesis: ethanolamine-dependent effects on steady-state membrane phospholipid composition in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961735" target=_blank>McMaster CR, <i>et al.</i> (1994 Nov 11)</a>. Phosphatidylcholine biosynthesis in Saccharomyces cerevisiae. Regulatory insights from studies employing null and chimeric sn-1,2-diacylglycerol choline- and ethanolaminephosphotransferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961831" target=_blank>Morash SC, <i>et al.</i> (1994 Nov 18)</a>. Studies employing Saccharomyces cerevisiae cpt1 and ept1 null mutants implicate the CPT1 gene in coordinate regulation of phospholipid biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225629" target=_blank>Boumann HA, <i>et al.</i> (2004 Jul 2)</a>. The selective utilization of substrates in vivo by the phosphatidylethanolamine and phosphatidylcholine biosynthetic enzymes Ept1p and Cpt1p in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870366" target=_blank>Sims AH, <i>et al.</i> (2005 May)</a>. Transcriptome analysis of recombinant protein secretion by Aspergillus nidulans and the unfolded-protein response in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02100_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02100_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02100_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02100_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>33 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02101<br>EOG8THT9H<br><br>sce:CPT1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cholinephosphotransferase; required for phosphatidylcholine biosynthesis and for inositol-dependent regulation of EPT1 transcription; CPT1 has a paralog, EPT1, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3005242" target=_blank>Kuchler K, <i>et al.</i> (1986 Mar)</a>. Subcellular and submitochondrial localization of phospholipid-synthesizing enzymes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3029130" target=_blank>Hjelmstad RH, <i>et al.</i> (1987 Mar 15)</a>. Mutants of Saccharomyces cerevisiae defective in sn-1,2-diacylglycerol cholinephosphotransferase. Isolation, characterization, and cloning of the CPT1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2153142" target=_blank>Hjelmstad RH, <i>et al.</i> (1990 Jan 25)</a>. The sn-1,2-diacylglycerol cholinephosphotransferase of Saccharomyces cerevisiae. Nucleotide sequence, transcriptional mapping, and gene product analysis of the CPT1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1847919" target=_blank>Hjelmstad RH, <i>et al.</i> (1991 Mar 5)</a>. sn-1,2-diacylglycerol choline- and ethanolaminephosphotransferases in Saccharomyces cerevisiae. Mixed micellar analysis of the CPT1 and EPT1 gene products.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961445" target=_blank>McGee TP, <i>et al.</i> (1994 Nov)</a>. Functional redundancy of CDP-ethanolamine and CDP-choline pathway enzymes in phospholipid biosynthesis: ethanolamine-dependent effects on steady-state membrane phospholipid composition in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961735" target=_blank>McMaster CR, <i>et al.</i> (1994 Nov 11)</a>. Phosphatidylcholine biosynthesis in Saccharomyces cerevisiae. Regulatory insights from studies employing null and chimeric sn-1,2-diacylglycerol choline- and ethanolaminephosphotransferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961831" target=_blank>Morash SC, <i>et al.</i> (1994 Nov 18)</a>. Studies employing Saccharomyces cerevisiae cpt1 and ept1 null mutants implicate the CPT1 gene in coordinate regulation of phospholipid biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8543066" target=_blank>Leber A, <i>et al.</i> (1995 Dec 18)</a>. Phospholipid-synthesizing enzymes in Golgi membranes of the yeast, Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12844361" target=_blank>Brachat S, <i>et al.</i> (2003)</a>. Reinvestigation of the Saccharomyces cerevisiae genome annotation by comparison to the genome of a related fungus: Ashbya gossypii.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225629" target=_blank>Boumann HA, <i>et al.</i> (2004 Jul 2)</a>. The selective utilization of substrates in vivo by the phosphatidylethanolamine and phosphatidylcholine biosynthetic enzymes Ept1p and Cpt1p in yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02101_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02101_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02101_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02101_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02102<br>EOG8THT9H<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted phosphotransferase activity, for other substituted phosphate groups activity, role in phospholipid biosynthetic process and membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02102_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02102_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02102_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02102_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>9 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00905.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>