<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00252.html">HOG00252&emsp;&emsp;FAD-binding oxidoreductase/transferase type 4 family</a><br> 
																	HOG00253&emsp;&emsp;<br> 
																	<a href="HOG00254.html">HOG00254&emsp;&emsp;PDH1 E1 alpha subunit</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00253.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00253_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00253_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00253_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00582<br>EOG851C88<br>IRC21<br>sce:IRC21<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative protein of unknown function; may be involved in resistance to carboplatin and cisplatin; null mutant displays increase in spontaneous Rad52p foci; contains a lipid-binding domain and binds cardiolipin in a large-scale study<br><br>
			<br><b>PomBase Description</b><br>NADPH-hemoprotein reductase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted heme binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16121259" target=_blank>Lee W, <i>et al.</i> (2005 Aug)</a>. Genome-wide requirements for resistance to functionally distinct DNA-damaging agents.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18085829" target=_blank>Alvaro D, <i>et al.</i> (2007 Dec)</a>. Genome-wide analysis of Rad52 foci reveals diverse mechanisms impacting recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20537132" target=_blank>Han TX, <i>et al.</i> (2010)</a>. Global fitness profiling of fission yeast deletion strains by barcode sequencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22235339" target=_blank>Lando D, <i>et al.</i> (2012)</a>. The S. pombe histone H2A dioxygenase Ofd2 regulates gene expression during hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22840777" target=_blank>Kawashima SA, <i>et al.</i> (2012 Jul 27)</a>. Analyzing fission yeast multidrug resistance mechanisms to develop a genetically tractable model system for chemical biology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23658229" target=_blank>Sugiyama T, <i>et al.</i> (2013 Jul)</a>. Red5 and three nuclear pore components are essential for efficient suppression of specific mRNAs during vegetative growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27168121" target=_blank>Chen JS, <i>et al.</i> (2016 Sep)</a>. Discovery of genes involved in mitosis, cell division, cell wall integrity and chromosome segregation through construction of Schizosaccharomyces pombe deletion strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00582_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00582_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00582_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00582_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00583<br>EOG851C88<br>CYB2<br>sce:CYB2<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytochrome b2 (L-lactate cytochrome-c oxidoreductase); component of the mitochondrial intermembrane space, required for lactate utilization; expression is repressed by glucose and anaerobic conditions<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=165435" target=_blank>Guiard B, <i>et al.</i> (1975 May 29)</a>. More similarity between bakers'yeast L-(+)-lactate dehydrogenase and liver microsomal cytochrome b5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=776230" target=_blank>Guiard B, <i>et al.</i> (1976)</a>. Complete amino acid sequence of the heme-binding core in bakers' yeast cytochrome b2 (L-(+)-lactate dehydrogenase).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6365548" target=_blank>Ghrir R, <i>et al.</i> (1984 Feb 15)</a>. Primary structure of flavocytochrome b2 from baker's yeast. Purification by reverse-phase high-pressure liquid chromatography and sequencing of fragment alpha cyanogen bromide peptides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3004948" target=_blank>Guiard B, <i>et al.</i> (1985 Dec 1)</a>. Structure, expression and regulation of a nuclear gene encoding a mitochondrial protein: the yeast L(+)-lactate cytochrome c oxidoreductase (cytochrome b2).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3902473" target=_blank>Lederer F, <i>et al.</i> (1985 Oct 15)</a>. Complete amino acid sequence of flavocytochrome b2 from baker's yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2329585" target=_blank>Xia ZX, <i>et al.</i> (1990 Apr 20)</a>. Molecular structure of flavocytochrome b2 at 2.4 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11502169" target=_blank>Grandier-Vazeille X, <i>et al.</i> (2001 Aug 21)</a>. Yeast mitochondrial dehydrogenases are associated in a supramolecular complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11914072" target=_blank>Cunane LM, <i>et al.</i> (2002 Apr 2)</a>. Crystallographic study of the recombinant flavin-binding domain of Baker's yeast flavocytochrome b(2): comparison with the intact wild-type enzyme.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00583_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00583_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00583_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00583_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>18 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00584<br>EOG851C88<br><br>sce:absent<br><br>Genes: 11
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00584_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00584_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00584_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00584_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00585<br>EOG851C88<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Lactic acid dehydrogenase|Lactic acid dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00585_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00585_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00585_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00585_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00586<br>EOG851C88<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Lactic acid dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00586_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00586_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00586_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00586_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00587<br>EOG851C88<br>YNR1<br>sce:absent<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>NAD(P)H nitrate reductase<br><br>
			<br><b>AspGD Description</b><br>Nitrate reductase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14023503" target=_blank>COVE DJ, <i>et al.</i> (1963 Apr 20)</a>. Independently segregating genetic loci concerned with nitrate reductase activity in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14085568" target=_blank>PATEMAN JA, <i>et al.</i> (1964 Jan 4)</a>. A COMMON CO-FACTOR FOR NITRATE REDUCTASE AND  XANTHINE DEHYDROGENASE WHICH ALSO REGULATES THE SYNTHESIS OF NITRATE REDUCTASE.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5866385" target=_blank>Cove DJ, <i>et al.</i> (1965 Nov 22)</a>. Purification of nitrate reductase and cytochrome c reductase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5940632" target=_blank>Cove DJ, <i>et al.</i> (1966 Jan 11)</a>. The induction and repression of nitrate reductase in the fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4382427" target=_blank>Pateman JA, <i>et al.</i> (1967 Jul)</a>. Genetic and biochemical studies of nitrate reduction in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6049855" target=_blank>Cove DJ, <i>et al.</i> (1967 Sep)</a>. Kinetic studies of the induction of nitrate reductase and cytochrome c reductase in the fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4901826" target=_blank>Pateman JA, <i>et al.</i> (1969 Dec)</a>. Regulation of synthesis of glutamate dehydrogenase and glutamine synthetase in micro-organisms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5776531" target=_blank>Cove DJ, <i>et al.</i> (1969 Mar)</a>. Autoregulation of the synthesis of nitrate reductase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5796514" target=_blank>Hartley MJ, <i>et al.</i> (1969 Mar-Apr)</a>. Reversion of non-nitrate utilizing (nia D) mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5475567" target=_blank>Arst HN Jr, <i>et al.</i> (1970)</a>. Molybdate metabolism in Aspergillus nidulans. I. Mutations affecting nitrate reductase and-or xanthine dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4144577" target=_blank>Downey RJ, <i>et al.</i> (1973)</a>. The multimeric nature of NADPH-nitrate reductase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4144009" target=_blank>Downey RJ, <i>et al.</i> (1973 Feb 5)</a>. The role of molybdenum in formation of the NADPH-nitrate reductase by Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4581424" target=_blank>Clutterbuck AJ, <i>et al.</i> (1973 Jun)</a>. Gene symbols in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4589996" target=_blank>Hynes MJ, <i>et al.</i> (1973 Nov)</a>. The effect of lack of a carbon source on nitrate-reductase activity in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4274591" target=_blank>Hankinson O, <i>et al.</i> (1974 Apr 25)</a>. Regulation of the pentose phosphate pathway in the fungus Aspergillus nidulans. The effect of growth with nitrate.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4611763" target=_blank>MacDonlad DW, <i>et al.</i> (1974 Aug 15)</a>. Studies on temperature-sensitive mutants affecting the assimilatory nitrate reductase of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4153024" target=_blank>McDonald DW, <i>et al.</i> (1974 Jul 1)</a>. Properties of the assimilatory nitrate reductase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=129029" target=_blank>Davis RH, <i>et al.</i> (1975)</a>. Compartmentation and regulation of fungal metabolism: genetic approaches.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=241670" target=_blank>Dunn-Coleman NS, <i>et al.</i> (1975)</a>. The regulation of nitrate reductase in the fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=123779" target=_blank>Ketchum PA, <i>et al.</i> (1975 Apr 7)</a>. In vitro restoration of nitrate reductase: investigation of Aspergillus nidulans and Neurospora crassa nitrate reductase mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=773908" target=_blank>Cove DJ, <i>et al.</i> (1976 Apr)</a>. Cholorate toxicity in Aspergillus nidulans: the selection and characterisation of chlorate resistant mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=796678" target=_blank>Garrett RH, <i>et al.</i> (1976 Dec 8)</a>. Formation of NADPH-nitrate reductase activity in vitro from Aspergillus nidulans niaD and cnx mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8697" target=_blank>Cove DJ, <i>et al.</i> (1976 Jul 23)</a>. Chlorate toxicity in Aspergillus nidulans. Studies of mutants altered in nitrate assimilation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17826" target=_blank>Dunn-Coleman NS, <i>et al.</i> (1977 Apr 29)</a>. In vivo and in vitro studies of nitrate reductase regulation in Asperillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=330163" target=_blank>Lewis NJ, <i>et al.</i> (1977 Jun 15)</a>. The genetic control of molybdoflavoproteins in Aspergillus nidulans. A xanthine dehydrogenase I half-molecule in cnx- mutant strains of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=157059" target=_blank>Garrett RH, <i>et al.</i> (1978)</a>. Nitrate assimilation in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=389305" target=_blank>Cove DJ, <i>et al.</i> (1979 Aug)</a>. Genetic studies of nitrate assimilation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=389740" target=_blank>Tomsett AB, <i>et al.</i> (1979 Aug)</a>. Deletion mapping of the niiA niaD gene region of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=384164" target=_blank>Arst HN Jr, <i>et al.</i> (1979 Jul 2)</a>. Do the tightly linked structural genes for nitrate and nitrite reductases in Aspergillus nidulans form an operon? Evidence from an insertional translocation which separates them.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7042701" target=_blank>Minagawa N, <i>et al.</i> (1982 Mar)</a>. Purification and characterization of the assimilatory NADPH-nitrate reductase of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6817753" target=_blank>Downey R, <i>et al.</i> (1982 Oct 29)</a>. De novo formation of the niaD gene directed protomer in the NADPH-nitrate reductase of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6751405" target=_blank>Steiner FX, <i>et al.</i> (1982 Sep 7)</a>. Isoelectric focusing and two-dimensional analysis of purified nitrate reductase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6340750" target=_blank>Martinelli SD, <i>et al.</i> (1983 Jan)</a>. Diagnosis of nonsense mutations in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6350263" target=_blank>Brownlee AG, <i>et al.</i> (1983 Sep)</a>. Nitrate uptake in Aspergillus nidulans and involvement of the third gene of the nitrate assimilation gene cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6423961" target=_blank>Downey R, <i>et al.</i> (1984)</a>. A convener role for the cnxH gene specified component in the NADPH-nitrate reductase fron Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6381230" target=_blank>Martinelli SD, <i>et al.</i> (1984 Jun)</a>. Evidence for a nonsense mutation at the niaD locus of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6101115" target=_blank>Arst HN Jr, <i>et al.</i> (1984 Sep)</a>. Regulation of gene expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2869649" target=_blank>Wiame JM, <i>et al.</i> (1985)</a>. Nitrogen catabolite repression in yeasts and filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3323853" target=_blank>Downey RJ, <i>et al.</i> (1987)</a>. Monoclonal antibody probes for the niaD specified subunit in the NADPH-nitrate reductase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454152" target=_blank>Downey RJ, <i>et al.</i> (1988 Jan)</a>. Distinction of cnxH cofactor gene-specified protomers with monoclonal antibodies to Aspergillus nitrate reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3046761" target=_blank>Bratt R, <i>et al.</i> (1988 Jul)</a>. Every ribosomal suppressor mutation in Aspergillus nidulans has a unique and highly pleiotropic phenotype.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2791035" target=_blank>Campbell EI, <i>et al.</i> (1989 Jul)</a>. Improved transformation efficiency of Aspergillus niger using the homologous niaD gene for nitrate reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2673557" target=_blank>Daboussi MJ, <i>et al.</i> (1989 Jun)</a>. Transformation of seven species of filamentous fungi using the nitrate reductase gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670677" target=_blank>Malardier L, <i>et al.</i> (1989 May 15)</a>. Cloning of the nitrate reductase gene (niaD) of Aspergillus nidulans and its use for transformation of Fusarium oxysporum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2205530" target=_blank>Johnstone IL, <i>et al.</i> (1990 Jun 15)</a>. Isolation and characterisation of the crnA-niiA-niaD gene cluster for nitrate assimilation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2381424" target=_blank>Debets AJ, <i>et al.</i> (1990 May)</a>. Genetic analysis of Aspergillus niger: isolation of chlorate resistance mutants, their use in mitotic mapping and evidence for an eighth linkage group.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2277647" target=_blank>Horng JS, <i>et al.</i> (1990 Nov)</a>. Development of a homologous transformation system for Aspergillus parasiticus with the gene encoding nitrate reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2062247" target=_blank>Downey RJ, <i>et al.</i> (1991)</a>. Nitrate reactive structural gene mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1756977" target=_blank>Daboussi MJ, <i>et al.</i> (1991 Dec 20)</a>. Heterologous expression of the Aspergillus nidulans regulatory gene nirA in Fusarium oxysporum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1990284" target=_blank>Burger G, <i>et al.</i> (1991 Feb)</a>. Molecular cloning and functional characterization of the pathway-specific regulatory gene nirA, which controls nitrate assimilation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1986367" target=_blank>Unkles SE, <i>et al.</i> (1991 Jan 1)</a>. crnA encodes a nitrate transporter in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1916271" target=_blank>Punt PJ, <i>et al.</i> (1991 Jul 31)</a>. A twin-reporter vector for simultaneous analysis of expression signals of divergently transcribed, contiguous genes in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1922075" target=_blank>Burger G, <i>et al.</i> (1991 Nov)</a>. nirA, the pathway-specific regulatory gene of nitrate assimilation in Aspergillus nidulans, encodes a putative GAL4-type zinc finger protein and contains four introns in highly conserved regions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1745237" target=_blank>Lloyd AT, <i>et al.</i> (1991 Nov)</a>. Codon usage in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1538701" target=_blank>Hawker KL, <i>et al.</i> (1992 Feb)</a>. Nitrate reductase and nitrite reductase transcript levels in various mutants of Aspergillus nidulans: confirmation of autogenous regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1541396" target=_blank>Unkles SE, <i>et al.</i> (1992 Feb 15)</a>. The Aspergillus niger niaD gene encoding nitrate reductase: upstream nucleotide and amino acid sequence comparisons.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8257101" target=_blank>Marzluf GA, <i>et al.</i> (1993)</a>. Regulation of sulfur and nitrogen metabolism in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765124" target=_blank>Kinghorn JR, <i>et al.</i> (1994)</a>. Inorganic nitrogen assimilation: molecular aspects.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8121403" target=_blank>Gems D, <i>et al.</i> (1994 Feb)</a>. An 'instant gene bank' method for gene cloning by mutant complementation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896804" target=_blank>Garde J, <i>et al.</i> (1995 Mar 24)</a>. Site-directed mutagenesis of nitrate reductase from Aspergillus nidulans. Identification of some essential and some nonessential amino acids among conserved residues.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7770039" target=_blank>Thijs H, <i>et al.</i> (1995 May 10)</a>. Polarity of meiotic gene conversion is 5' to 3' within the niaD gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565720" target=_blank>Punt PJ, <i>et al.</i> (1995 Oct)</a>. The intergenic region between the divergently transcribed niiA and niaD genes of Aspergillus nidulans contains multiple NirA binding sites which act bidirectionally.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8520125" target=_blank>Kitamoto N, <i>et al.</i> (1995 Sep)</a>. The nitrate reductase gene from a shoyu koji mold, Aspergillus oryzae KBN616.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8625427" target=_blank>Amutan M, <i>et al.</i> (1996 Apr)</a>. Identification and cloning of a mobile transposon from Aspergillus niger var. awamori.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8662212" target=_blank>Chang PK, <i>et al.</i> (1996 Jun)</a>. Characterization of the Aspergillus parasiticus niaD and niiA gene cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8950182" target=_blank>Haas H, <i>et al.</i> (1996 Nov 11)</a>. Sequence analysis and expression of the Penicillium chrysogenum nitrate reductase encoding gene (niaD).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8891357" target=_blank>Zucchi TM, <i>et al.</i> (1996 Sep)</a>. RNA-mediated transformation in Aspergillus nidulans recovers gene functions lost by deletion or by point mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236780" target=_blank>Bird D, <i>et al.</i> (1997 Jun)</a>. Gene targeting is locus dependent in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352912" target=_blank>Lamb HK, <i>et al.</i> (1997 Nov)</a>. Deletion of the N-terminal region of the AREA protein is correlated with a derepressed phenotype with respect to nitrogen metabolite repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10075929" target=_blank>Muro-Pastor MI, <i>et al.</i> (1999 Mar 15)</a>. The GATA factor AreA is essential for chromatin remodelling in a eukaryotic bidirectional promoter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760588" target=_blank>Chang PK, <i>et al.</i> (2000 Apr 25)</a>. Characterization of the Aspergillus parasiticus major nitrogen regulatory gene, areA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689191" target=_blank>Dessen P, <i>et al.</i> (2000 Feb 22)</a>. The PAUSE software for analysis of translational control over protein targeting: application to E. nidulans membrane proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10632892" target=_blank>Ramón A, <i>et al.</i> (2000 Jan)</a>. Deletion of the unique gene encoding a typical histone H1 has no apparent phenotype in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10846218" target=_blank>Hall N, <i>et al.</i> (2000 Jun)</a>. Structure-function analysis of NADPH:nitrate reductase from Aspergillus nidulans: analysis of altered pyridine nucleotide specificity in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11055928" target=_blank>Zadra I, <i>et al.</i> (2000 Nov)</a>. xylP promoter-based expression system and its use for antisense downregulation of the Penicillium chrysogenum nitrogen regulator NRE.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12019757" target=_blank>Sandhu SS, <i>et al.</i> (2001 Jul)</a>. Transformation system of Beauveria bassiana and Metarhizium anisopliae using nitrate reductase gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11707396" target=_blank>Unkles SE, <i>et al.</i> (2001 Nov 15)</a>. Apparent genetic redundancy facilitates ecological plasticity for nitrate transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972792" target=_blank>Narendja F, <i>et al.</i> (2002 Apr)</a>. Nitrate and the GATA factor AreA are necessary for in vivo binding of NirA, the pathway-specific transcriptional activator of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15075269" target=_blank>Muro-Pastor MI, <i>et al.</i> (2004 Apr)</a>. A paradoxical mutant GATA factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15118339" target=_blank>Takasaki K, <i>et al.</i> (2004 Apr)</a>. Unusual transcription regulation of the niaD gene under anaerobic conditions supporting fungal ammonia fermentation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14722082" target=_blank>Takasaki K, <i>et al.</i> (2004 Mar 26)</a>. Fungal ammonia fermentation, a novel metabolic mechanism that couples the dissimilatory and assimilatory pathways of both nitrate and ethanol. Role of acetyl CoA synthetase in anaerobic ATP synthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375695" target=_blank>Takahashi T, <i>et al.</i> (2004 Oct)</a>. Efficient gene disruption in the koji-mold Aspergillus sojae using a novel variation of the positive-negative method.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020584" target=_blank>Caddick MX, <i>et al.</i> (2006 Oct)</a>. Opposing signals differentially regulate transcript stability in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16699756" target=_blank>Lubertozzi D, <i>et al.</i> (2006 Oct)</a>. Marker and promoter effects on heterologous expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17688585" target=_blank>Wang Y, <i>et al.</i> (2007)</a>. Evidence for post-translational regulation of NrtA, the Aspergillus nidulans high-affinity nitrate transporter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116695" target=_blank>Bernreiter A, <i>et al.</i> (2007 Feb)</a>. Nuclear export of the transcription factor NirA is a regulatory checkpoint for nitrate induction in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673441" target=_blank>Berger H, <i>et al.</i> (2008 Sep)</a>. Dissecting individual steps of nitrogen transcription factor cooperation in the Aspergillus nidulans nitrate cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19562269" target=_blank>Navarrete K, <i>et al.</i> (2009)</a>. Molecular characterization of the niaD and pyrG genes from Penicillium camemberti, and their use as transformation markers.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251760" target=_blank>Basheer A, <i>et al.</i> (2009 Apr)</a>. A library-based method to rapidly analyse chromatin accessibility at multiple genomic regions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19129650" target=_blank>Takaya N, <i>et al.</i> (2009 Jan)</a>. Response to hypoxia, reduction of electron acceptors, and subsequent survival by filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19596074" target=_blank>Hagiwara D, <i>et al.</i> (2009 Nov)</a>. Transcriptional profiling for Aspergillusnidulans HogA MAPK signaling pathway in response to fludioxonil and osmotic stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20969648" target=_blank>Schinko T, <i>et al.</i> (2010 Nov)</a>. Transcriptome analysis of nitrate assimilation in Aspergillus nidulans reveals connections to nitric oxide metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21088867" target=_blank>Meyer V, <i>et al.</i> (2011 Mar)</a>. Aspergillus as a multi-purpose cell factory: current status and perspectives.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22170104" target=_blank>Terabayashi Y, <i>et al.</i> (2012 Jan)</a>. Conserved and specific responses to hypoxia in Aspergillus oryzae and Aspergillus nidulans determined by comparative transcriptomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22306112" target=_blank>Sarkar A, <i>et al.</i> (2012 Jul 31)</a>. Differential expression of silent polyketide biosynthesis gene clusters in chemostat cultures of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23154970" target=_blank>Szilágyi M, <i>et al.</i> (2013 Jan)</a>. Transcriptome changes initiated by carbon starvation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23454548" target=_blank>Schinko T, <i>et al.</i> (2013 May)</a>. Pseudo-constitutivity of nitrate-responsive genes in nitrate reductase mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23841692" target=_blank>Krol K, <i>et al.</i> (2013 Sep)</a>. RrmA regulates the stability of specific transcripts in response to both nitrogen source and oxidative stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23973960" target=_blank>Scazzocchio C, <i>et al.</i> (2013 Sep-Oct)</a>. In praise of erroneous hypotheses.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25506342" target=_blank>Tudzynski B, <i>et al.</i> (2014)</a>. Nitrogen regulation of fungal secondary metabolism in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26353949" target=_blank>Marcos AT, <i>et al.</i> (2016 Jan)</a>. Nitric oxide synthesis by nitrate reductase is regulated during development in Aspergillus.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00587_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00587_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00587_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00587_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00588<br>EOG851C88<br>CBR1<br>sce:CBR1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Microsomal cytochrome b reductase; not essential for viability; also detected in mitochondria; mutation in conserved NADH binding domain of the human ortholog results in type I methemoglobinemia<br><br>
			<br><b>PomBase Description</b><br>cytochrome b5 reductase Cbr1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have endoplasmic reticulum, plasma membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14930" target=_blank>Kubota S, <i>et al.</i> (1977 Jan)</a>. Studies on the microsomal electron-transport system of anaerobically grown yeast. IV. Purification and characterization of NADH-cytochrome b5 reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6442167" target=_blank>Bertrand JC, <i>et al.</i> (1984 Jul-Aug)</a>. Influence of oxygen on the microsomal electron transport system in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8307010" target=_blank>Csukai M, <i>et al.</i> (1994 Jan 15)</a>. Isolation and complete sequence of CBR, a gene encoding a putative cytochrome b reductase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10622712" target=_blank>Lamb DC, <i>et al.</i> (1999 Dec 3)</a>. Biodiversity of the P450 catalytic cycle: yeast cytochrome b5/NADH cytochrome b5 reductase complex efficiently drives the entire sterol 14-demethylation (CYP51) reaction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15905473" target=_blank>Zhang Z, <i>et al.</i> (2005)</a>. Mapping of transcription start sites in Saccharomyces cerevisiae using 5' SAGE.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16407407" target=_blank>Zahedi RP, <i>et al.</i> (2006 Mar)</a>. Proteomic analysis of the yeast mitochondrial outer membrane reveals accumulation of a subclass of preproteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17895367" target=_blank>Tiedje C, <i>et al.</i> (2007 Oct 15)</a>. Proteins involved in sterol synthesis interact with Ste20 and regulate cell polarity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00588_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00588_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00588_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00588_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>27 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG00589<br>EOG851C88<br>MCR1<br>sce:MCR1<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mitochondrial NADH-cytochrome b5 reductase; involved in ergosterol biosynthesis<br><br>
			<br><b>AspGD Description</b><br>Cytochrome-b5 reductase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8001120" target=_blank>Hahne K, <i>et al.</i> (1994 Dec 2)</a>. Incomplete arrest in the outer membrane sorts NADH-cytochrome b5 reductase to two different submitochondrial compartments.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199337" target=_blank>Haucke V, <i>et al.</i> (1997 Jul)</a>. Analysis of the sorting signals directing NADH-cytochrome b5 reductase to two locations within yeast mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9866716" target=_blank>Martin H, <i>et al.</i> (1998 Dec 1)</a>. The yeast mitochondrial intermembrane space: purification and analysis of two distinct fractions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11420140" target=_blank>Lee JS, <i>et al.</i> (2001 Jul 2)</a>. Mitochondrial NADH-cytochrome b(5) reductase plays a crucial role in the reduction of D-erythroascorbyl free radical in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15590813" target=_blank>Krantz M, <i>et al.</i> (2004 Dec)</a>. Anaerobicity prepares Saccharomyces cerevisiae cells for faster adaptation to osmotic shock.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16689936" target=_blank>Burri L, <i>et al.</i> (2006 Apr)</a>. Integral membrane proteins in the mitochondrial outer membrane of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16407407" target=_blank>Zahedi RP, <i>et al.</i> (2006 Mar)</a>. Proteomic analysis of the yeast mitochondrial outer membrane reveals accumulation of a subclass of preproteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17244705" target=_blank>Juneau K, <i>et al.</i> (2007 Jan 30)</a>. High-density yeast-tiling array reveals previously undiscovered introns and extensive regulation of meiotic splicing.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00589_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00589_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00589_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00589_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>15 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG00590<br>EOG851C88<br>PGA3<br>sce:PGA3<br><br>Genes: 22
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative cytochrome b5 reductase, localized to the plasma membrane; may be involved in regulation of lifespan; required for maturation of Gas1p and Pho8p, proposed to be involved in protein trafficking; PGA3 has a paralog, AIM33, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16943325" target=_blank>Yu L, <i>et al.</i> (2006 Nov)</a>. A survey of essential gene function in the yeast cell division cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19239415" target=_blank>Jiménez-Hidalgo M, <i>et al.</i> (2009 Apr)</a>. NQR1 controls lifespan by regulating the promotion of respiratory metabolism in yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00590_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00590_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00590_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00590_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>21 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG00591<br>EOG851C88<br><br>sce:absent<br><br>Genes: 20
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Lactic acid dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00591_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00591_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00591_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00591_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG00592<br>EOG851C88<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted heme binding, oxidoreductase activity and role in oxidation-reduction process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00592_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00592_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00592_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00592_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG00593<br>EOG851C88<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted heme binding, oxidoreductase activity and role in oxidation-reduction process|Has domain(s) with predicted heme binding, oxidoreductase activity and role in oxidation-reduction process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00593_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00593_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00593_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00593_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG00594<br>EOG851C88<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted electron carrier activity, heme binding, metal ion binding, molybdenum ion binding, oxidoreductase activity and role in nitrate assimilation, oxidation-reduction process|Has domain(s) with predicted electron carrier activity, heme binding, metal ion binding, molybdenum ion binding, oxidoreductase activity and role in nitrate assimilation, oxidation-reduction process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00594_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00594_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00594_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00594_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG00595<br>EOG851C88<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative cytochrome-b5 reductase; expression repressed by tunicamycin and DTT<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00595_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00595_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00595_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00595_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG00596<br>EOG851C88<br>CYB5<br>sce:CYB5<br><br>Genes: 38
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytochrome b5; involved in the sterol and lipid biosynthesis pathways; acts as an electron donor to support sterol C5-6 desaturation<br><br>
			<br><b>PomBase Description</b><br>cytochrome b5 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Cytochrome b5<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8181746" target=_blank>Truan G, <i>et al.</i> (1994 May 3)</a>. Cloning and characterization of a yeast cytochrome b5-encoding gene which suppresses ketoconazole hypersensitivity in a NADPH-P-450 reductase-deficient strain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12237855" target=_blank>Tallada VA, <i>et al.</i> (2002 Sep 30)</a>. Genome-wide search of Schizosaccharomyces pombe genes causing overexpression-mediated cell cycle defects.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21850271" target=_blank>Ma Y, <i>et al.</i> (2011)</a>. Genome-wide screening for genes associated with FK506 sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23861937" target=_blank>Zhang L, <i>et al.</i> (2013)</a>. Genome-wide screening for genes associated with valproic acid sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24463365" target=_blank>Rallis C, <i>et al.</i> (2014 Feb 15)</a>. Systematic screen for mutants resistant to TORC1 inhibition in fission yeast reveals genes involved in cellular ageing and growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25373780" target=_blank>Graml V, <i>et al.</i> (2014 Oct 27)</a>. A genomic Multiprocess survey of machineries that control and link cell shape, microtubule organization, and cell-cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojardín L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26628015" target=_blank>Zhang X, <i>et al.</i> (2015 Oct 9)</a>. Characterization of Tamoxifen as an Antifungal Agent Using the Yeast Schizosaccharomyces Pombe Model Organism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00596_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00596_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00596_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00596_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>34 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG00597<br>EOG851C88<br><br>sce:absent<br><br>Genes: 13
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00597_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00597_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00597_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00597_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG00598<br>EOG851C88<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00598_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00598_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00598_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00598_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG00599<br>EOG851C88<br>AIM33<br>sce:AIM33<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative protein of unknown function, highly conserved across species; homolog of human CYB5R4; null mutant displays reduced frequency of mitochondrial genome loss; AIM33 has a paralog, PGA3, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>mitochondrial cytochrome c-heme linkage protein Cyc2 (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19300474" target=_blank>Hess DC, <i>et al.</i> (2009 Mar)</a>. Computationally driven, quantitative experiments discover genes required for mitochondrial biogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20133687" target=_blank>Takeda K, <i>et al.</i> (2010 Feb 23)</a>. Synergistic roles of the proteasome and autophagy for mitochondrial maintenance and chronological lifespan in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22235339" target=_blank>Lando D, <i>et al.</i> (2012)</a>. The S. pombe histone H2A dioxygenase Ofd2 regulates gene expression during hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27887640" target=_blank>Malecki M, <i>et al.</i> (2016 Nov 25)</a>. Functional and regulatory profiling of energy metabolism in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00599_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00599_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00599_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00599_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00253.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>