<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00947.html">HOG00947&emsp;&emsp;group II decarboxylase family. Sphingosine-1-phosphate lyase subfamily</a><br> 
																	HOG00948&emsp;&emsp;<br> 
																	<a href="HOG00949.html">HOG00949&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00948.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00948_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00948_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00948_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02175<br>EOG8HMGTC<br><br>sce:LCB3;YSR3<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Long-chain base-1-phosphate phosphatase; specific for dihydrosphingosine-1-phosphate, regulates ceramide and long-chain base phosphates levels, involved in incorporation of exogenous long chain bases in sphingolipids; LCB3 has a paralog, YSR3, that arose from the whole genome duplication|Dihydrosphingosine 1-phosphate phosphatase; membrane protein involved in sphingolipid metabolism; YSR3 has a paralog, LCB3, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>sphingosine-1-phosphate phosphatase<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted catalytic activity and membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703236" target=_blank>Wiesenberger G, <i>et al.</i> (1991 Jan 5)</a>. MRS3 and MRS4, two suppressors of mtRNA splicing defects in yeast, are new members of the mitochondrial carrier family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9195906" target=_blank>Qie L, <i>et al.</i> (1997 Jun 27)</a>. Identification of a Saccharomyces gene, LCB3, necessary for incorporation of exogenous long chain bases into sphingolipids.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9353337" target=_blank>Mao C, <i>et al.</i> (1997 Nov 7)</a>. Identification and characterization of Saccharomyces cerevisiae dihydrosphingosine-1-phosphate phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9419344" target=_blank>Mandala SM, <i>et al.</i> (1998 Jan 6)</a>. Sphingoid base 1-phosphate phosphatase: a key regulator of sphingolipid metabolism and stress response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10477278" target=_blank>Mao C, <i>et al.</i> (1999 Sep 15)</a>. The dihydrosphingosine-1-phosphate phosphatases of Saccharomyces cerevisiae are important regulators of cell proliferation and heat stress responses.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10563329" target=_blank>Mao C, <i>et al.</i> (2000)</a>. Yeast sphingosine-1-phosphate phosphatases: assay, expression, deletion, purification, and cellular localization by GFP tagging.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856228" target=_blank>Zanolari B, <i>et al.</i> (2000 Jun 15)</a>. Sphingoid base synthesis requirement for endocytosis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278643" target=_blank>Birchwood CJ, <i>et al.</i> (2001 Apr 13)</a>. Calcium influx and signaling in yeast stimulated by intracellular sphingosine 1-phosphate accumulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11967828" target=_blank>Ferguson-Yankey SR, <i>et al.</i> (2002 May)</a>. Mutant analysis reveals complex regulation of sphingolipid long chain base phosphates and long chain bases during heat stress in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12684378" target=_blank>Kobayashi SD, <i>et al.</i> (2003 Apr)</a>. Ceramide/long-chain base phosphate rheostat in Saccharomyces cerevisiae: regulation of ceramide synthesis by Elo3p and Cka2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12493772" target=_blank>Funato K, <i>et al.</i> (2003 Feb 28)</a>. Lcb4p is a key regulator of ceramide synthesis from exogenous long chain sphingoid base in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12786943" target=_blank>Kihara A, <i>et al.</i> (2003 Jun)</a>. Transmembrane topology of sphingoid long-chain base-1-phosphate phosphatase, Lcb3p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02175_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02175_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02175_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02175_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>34 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00948.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>