<html>
<head><title>HOG00209</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00208.html">HOG00208&emsp;&emsp;phosphoenolpyruvate carboxykinase (ATP) family</a><br> 
																	HOG00209&emsp;&emsp;<br> 
																	<a href="HOG00210.html">HOG00210&emsp;&emsp;zinc-containing alcohol dehydrogenase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00209.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00209_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00209_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00209_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG00209.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00311<br>EOG8S4MZ4<br>PDC1<br>sce:PDC1<br><br>Genes: 40
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Main isozyme of pyruvate decarboxylase<br><br>
			<br><b>SGD Description</b><br>Major of three pyruvate decarboxylase isozymes; key enzyme in alcoholic fermentation; decarboxylates pyruvate to acetaldehyde; involved in amino acid catabolism; subject to glucose-, ethanol-, and autoregulation; activated by phosphorylation in response to glucose levels; N-terminally propionylated in vivo<br><br>
			<br><b>PomBase Description</b><br>pyruvate decarboxylase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Pyruvate decarboxylase; protein levels influenced by presence of starch<br><br>
			<br><b>Suggested Analysis</b><br>Investigate ancestry of with with HOG00462 (IVLB family)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4687392" target=_blank>Lehmann H, <i>et al.</i> (1973 Jan 3)</a>. The influence of steric and electronic parameters on the substrate behavior of  -oxo acids to yeast pyruvate decarboxylase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7050079" target=_blank>Schmitt HD, <i>et al.</i> (1982 Sep)</a>. Genetic analysis of the pyruvate decarboxylase reaction in yeast glycolysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537965" target=_blank>Kellermann E, <i>et al.</i> (1986 Nov 25)</a>. Analysis of the primary structure and promoter function of a pyruvate decarboxylase gene (PDC1) from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2185016" target=_blank>Hohmann S, <i>et al.</i> (1990 Mar 30)</a>. Autoregulation may control the expression of yeast pyruvate decarboxylase structural genes PDC1 and PDC5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8512926" target=_blank>Dyda F, <i>et al.</i> (1993 Jun 22)</a>. Catalytic centers in the thiamin diphosphate dependent enzyme pyruvate decarboxylase at 2.4-A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8181725" target=_blank>Sanchis V, <i>et al.</i> (1994 Apr 1)</a>. A pyruvate decarboxylase gene from Aspergillus parasiticus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7725793" target=_blank>Holloway P, <i>et al.</i> (1994 Dec)</a>. The nucleotide sequence and initial characterization of pyruvate decarboxylase from the yeast Hanseniaspora uvarum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7737086" target=_blank>Norbeck J, <i>et al.</i> (1995 Jan)</a>. Gene linkage of two-dimensional polyacrylamide gel electrophoresis resolved proteins from isogene families in Saccharomyces cerevisiae by microsequencing of in-gel trypsin generated peptides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866484" target=_blank>Liesen T, <i>et al.</i> (1996 Aug)</a>. ERA, a novel cis-acting element required for autoregulation and ethanol repression of PDC1 transcription in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821934" target=_blank>Bianchi MM, <i>et al.</i> (1996 Jan)</a>. The 'petite-negative' yeast Kluyveromyces lactis has a single gene expressing pyruvate decarboxylase activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8604141" target=_blank>Arjunan P, <i>et al.</i> (1996 Mar 1)</a>. Crystal structure of the thiamin diphosphate-dependent enzyme pyruvate decarboxylase from the yeast Saccharomyces cerevisiae at 2.3 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9298649" target=_blank>Garrels JI, <i>et al.</i> (1997 Aug)</a>. Proteome studies of Saccharomyces cerevisiae: identification and characterization of abundant proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210590" target=_blank>Lockington RA, <i>et al.</i> (1997 May 20)</a>. Pyruvate decarboxylase and anaerobic survival in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9341119" target=_blank>Dickinson JR, <i>et al.</i> (1997 Oct 24)</a>. A 13C nuclear magnetic resonance investigation of the metabolism of leucine to isoamyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9435065" target=_blank>Lu P, <i>et al.</i> (1998 Jan)</a>. Cloning and characterization of two pyruvate decarboxylase genes from Pichia stipitis CBS 6054.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9655908" target=_blank>Hohmann S, <i>et al.</i> (1998 Jun 29)</a>. Thiamin metabolism and thiamin diphosphate-dependent enzymes in the yeast Saccharomyces cerevisiae: genetic regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9655924" target=_blank>Iding H, <i>et al.</i> (1998 Jun 29)</a>. Application of alpha-keto acid decarboxylases in biotransformations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748245" target=_blank>Dickinson JR, <i>et al.</i> (1998 Oct 2)</a>. An investigation of the metabolism of valine to isobutyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10219994" target=_blank>Destruelle M, <i>et al.</i> (1999 Mar 30)</a>. Regulation of the expression of the Kluyveromyces lactis PDC1 gene: carbon source-responsive elements and autoregulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10231381" target=_blank>Eberhardt I, <i>et al.</i> (1999 May)</a>. Autoregulation of yeast pyruvate decarboxylase gene expression requires the enzyme but not its catalytic activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10234824" target=_blank>Flikweert MT, <i>et al.</i> (1999 May 1)</a>. Growth requirements of pyruvate-decarboxylase-negative Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10545125" target=_blank>Polevoda B, <i>et al.</i> (1999 Nov 1)</a>. Identification and specificities of N-terminal acetyltransferases from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10753893" target=_blank>Dickinson JR, <i>et al.</i> (2000 Apr 14)</a>. An investigation of the metabolism of isoleucine to active Amyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11141278" target=_blank>Neuser F, <i>et al.</i> (2000 Dec)</a>. Generation of odorous acyloins by yeast pyruvate decarboxylases and their occurrence in sherry and soy sauce.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651824" target=_blank>Lu G, <i>et al.</i> (2000 Feb)</a>. The structural basis of substrate activation in yeast pyruvate decarboxylase. A crystallographic and kinetic study.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12902239" target=_blank>Vuralhan Z, <i>et al.</i> (2003 Aug)</a>. Identification and characterization of phenylpyruvate decarboxylase genes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12499363" target=_blank>Dickinson JR, <i>et al.</i> (2003 Mar 7)</a>. The catabolism of amino acids to long chain and complex alcohols in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15378761" target=_blank>Pitarch A, <i>et al.</i> (2004 Oct)</a>. Proteomics-based identification of novel Candida albicans antigens for diagnosis of systemic candidiasis in patients with underlying hematological malignancies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15625313" target=_blank>Wang Q, <i>et al.</i> (2004 Oct)</a>. Purification, characterization, cloning and expression of pyruvate decarboxylase from Torulopsis glabrata IFO005.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21062371" target=_blank>Nahlik K, <i>et al.</i> (2010 Nov)</a>. The COP9 signalosome mediates transcriptional and metabolic response to hormones, oxidative stress protection and cell wall rearrangement during fungal development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22792266" target=_blank>Etxebeste O, <i>et al.</i> (2012)</a>. GmcA is a putative glucose-methanol-choline oxidoreductase required for the induction of asexual development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00311_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00311_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00311_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00311_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>20 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00312<br>EOG8S4MZ4<br>PDC5<br>sce:PDC5<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>PDC5 paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00311<br><br>
			<br><b>SGD Description</b><br>Minor isoform of pyruvate decarboxylase; key enzyme in alcoholic fermentation, decarboxylates pyruvate to acetaldehyde, regulation is glucose- and ethanol-dependent, repressed by thiamine, involved in amino acid catabolism<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2185016" target=_blank>Hohmann S, <i>et al.</i> (1990 Mar 30)</a>. Autoregulation may control the expression of yeast pyruvate decarboxylase structural genes PDC1 and PDC5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9341119" target=_blank>Dickinson JR, <i>et al.</i> (1997 Oct 24)</a>. A 13C nuclear magnetic resonance investigation of the metabolism of leucine to isoamyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9655908" target=_blank>Hohmann S, <i>et al.</i> (1998 Jun 29)</a>. Thiamin metabolism and thiamin diphosphate-dependent enzymes in the yeast Saccharomyces cerevisiae: genetic regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9655924" target=_blank>Iding H, <i>et al.</i> (1998 Jun 29)</a>. Application of alpha-keto acid decarboxylases in biotransformations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748245" target=_blank>Dickinson JR, <i>et al.</i> (1998 Oct 2)</a>. An investigation of the metabolism of valine to isobutyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10338141" target=_blank>Muller EH, <i>et al.</i> (1999 Apr 23)</a>. Thiamine repression and pyruvate decarboxylase autoregulation independently control the expression of the Saccharomyces cerevisiae PDC5 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10231381" target=_blank>Eberhardt I, <i>et al.</i> (1999 May)</a>. Autoregulation of yeast pyruvate decarboxylase gene expression requires the enzyme but not its catalytic activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10234824" target=_blank>Flikweert MT, <i>et al.</i> (1999 May 1)</a>. Growth requirements of pyruvate-decarboxylase-negative Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10753893" target=_blank>Dickinson JR, <i>et al.</i> (2000 Apr 14)</a>. An investigation of the metabolism of isoleucine to active Amyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11141278" target=_blank>Neuser F, <i>et al.</i> (2000 Dec)</a>. Generation of odorous acyloins by yeast pyruvate decarboxylases and their occurrence in sherry and soy sauce.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12902239" target=_blank>Vuralhan Z, <i>et al.</i> (2003 Aug)</a>. Identification and characterization of phenylpyruvate decarboxylase genes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12499363" target=_blank>Dickinson JR, <i>et al.</i> (2003 Mar 7)</a>. The catabolism of amino acids to long chain and complex alcohols in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00312_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00312_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00312_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00312_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00313<br>EOG8S4MZ4<br>PDC6<br>sce:PDC6<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>PDC6 paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00311<br><br>
			<br><b>SGD Description</b><br>Minor isoform of pyruvate decarboxylase; decarboxylates pyruvate to acetaldehyde, involved in amino acid catabolism; transcription is glucose- and ethanol-dependent, and is strongly induced during sulfur limitation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1744053" target=_blank>Hohmann S, <i>et al.</i> (1991 Dec)</a>. Characterization of PDC6, a third structural gene for pyruvate decarboxylase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9341119" target=_blank>Dickinson JR, <i>et al.</i> (1997 Oct 24)</a>. A 13C nuclear magnetic resonance investigation of the metabolism of leucine to isoamyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9655908" target=_blank>Hohmann S, <i>et al.</i> (1998 Jun 29)</a>. Thiamin metabolism and thiamin diphosphate-dependent enzymes in the yeast Saccharomyces cerevisiae: genetic regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9655924" target=_blank>Iding H, <i>et al.</i> (1998 Jun 29)</a>. Application of alpha-keto acid decarboxylases in biotransformations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748245" target=_blank>Dickinson JR, <i>et al.</i> (1998 Oct 2)</a>. An investigation of the metabolism of valine to isobutyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10234824" target=_blank>Flikweert MT, <i>et al.</i> (1999 May 1)</a>. Growth requirements of pyruvate-decarboxylase-negative Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10753893" target=_blank>Dickinson JR, <i>et al.</i> (2000 Apr 14)</a>. An investigation of the metabolism of isoleucine to active Amyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11141278" target=_blank>Neuser F, <i>et al.</i> (2000 Dec)</a>. Generation of odorous acyloins by yeast pyruvate decarboxylases and their occurrence in sherry and soy sauce.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12902239" target=_blank>Vuralhan Z, <i>et al.</i> (2003 Aug)</a>. Identification and characterization of phenylpyruvate decarboxylase genes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12499363" target=_blank>Dickinson JR, <i>et al.</i> (2003 Mar 7)</a>. The catabolism of amino acids to long chain and complex alcohols in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00313_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00313_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00313_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00313_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00314<br>EOG8S4MZ4<br>PDC1.2<br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized PDC1 paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00311<br><br>
			<br><b>Suggested Analysis</b><br>Review impact of inserition in msa<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00314_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00314_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00314_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00314_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00315<br>EOG8S4MZ4<br>PDC1.3<br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized PDC1 paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00311<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00315_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00315_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00315_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00315_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00316<br>EOG8S4MZ4<br>THI3<br>sce:THI3<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>THI3 regulator present in SCE clade<br><br>
			<br><b>Parent</b><br>paralog:FOG00311<br><br>
			<br><b>SGD Description</b><br>Regulatory protein that binds Pdc2p and Thi2p transcription factors; activates thiamine biosynthesis transcription factors Pdc2p and Thi2p by binding to them, but releases and de-activates them upon binding to thiamine pyrophosphate (TPP), the end product of the pathway; has similarity to decarboxylases but enzymatic activity is not detected<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624458" target=_blank>Nishimura H, <i>et al.</i> (1992 Jul)</a>. A positive regulatory gene, THI3, is required for thiamine metabolism in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9341119" target=_blank>Dickinson JR, <i>et al.</i> (1997 Oct 24)</a>. A 13C nuclear magnetic resonance investigation of the metabolism of leucine to isoamyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748245" target=_blank>Dickinson JR, <i>et al.</i> (1998 Oct 2)</a>. An investigation of the metabolism of valine to isobutyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10753893" target=_blank>Dickinson JR, <i>et al.</i> (2000 Apr 14)</a>. An investigation of the metabolism of isoleucine to active Amyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12902239" target=_blank>Vuralhan Z, <i>et al.</i> (2003 Aug)</a>. Identification and characterization of phenylpyruvate decarboxylase genes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12499363" target=_blank>Dickinson JR, <i>et al.</i> (2003 Mar 7)</a>. The catabolism of amino acids to long chain and complex alcohols in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14648197" target=_blank>Møller K, <i>et al.</i> (2004 Jan)</a>. Pyruvate decarboxylases from the petite-negative yeast Saccharomyces kluyveri.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00316_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00316_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00316_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00316_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00317<br>EOG879CQB<br>EOG8S4MZ4<br>ARO10<br>sce:ARO10<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Phenylpyruvate decarboxylase<br><br>
			<br><b>Parent</b><br>paralog:FOG00311<br><br>
			<br><b>SGD Description</b><br>Phenylpyruvate decarboxylase; catalyzes decarboxylation of phenylpyruvate to phenylacetaldehyde, which is the first specific step in the Ehrlich pathway; involved in protein N-terminal Met and Ala catabolism<br><br>
			<br><b>AspGD Description</b><br>Pyruvate decarboxylase<br><br>
			<br><b>Suggested Analysis</b><br>ARO10 appears to be derived from PDC, which may be derived from IVLB. Further studies are needed.<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10207060" target=_blank>Iraqui I, <i>et al.</i> (1999 May)</a>. Transcriptional induction by aromatic amino acids in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10753893" target=_blank>Dickinson JR, <i>et al.</i> (2000 Apr 14)</a>. An investigation of the metabolism of isoleucine to active Amyl alcohol in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12872131" target=_blank>Peng J, <i>et al.</i> (2003 Aug)</a>. A proteomics approach to understanding protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12902239" target=_blank>Vuralhan Z, <i>et al.</i> (2003 Aug)</a>. Identification and characterization of phenylpyruvate decarboxylase genes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12499363" target=_blank>Dickinson JR, <i>et al.</i> (2003 Mar 7)</a>. The catabolism of amino acids to long chain and complex alcohols in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15933030" target=_blank>Vuralhan Z, <i>et al.</i> (2005 Jun)</a>. Physiological characterization of the ARO10-dependent, broad-substrate-specificity 2-oxo acid decarboxylase activity of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16423070" target=_blank>Perpète P, <i>et al.</i> (2006 Jan)</a>. Methionine catabolism in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00317_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00317_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00317_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00317_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>16 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG00318<br>EOG879CQB<br>ARO10.2<br>sce:absent<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized ARO10 paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00317<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00318_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00318_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00318_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00318_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG00319<br>EOG879CQB<br>ARO10.3<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized ARO10 vpo ohnolog<br><br>
			<br><b>Parent</b><br>ohnolog?:FOG00317<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00319_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00319_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00319_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00319_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00209_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00209.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>