<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01611.html">HOG01611&emsp;&emsp;COA2 family</a><br> 
																	HOG01612&emsp;&emsp;<br> 
																	<a href="HOG01613.html">HOG01613&emsp;&emsp;peptidase M14 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01612.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01612_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01612_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01612_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG04584<br>EOG8R22D0<br><br>sce:HAT2<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of the Hat1p-Hat2p histone acetyltransferase complex; required for high affinity binding of the complex to free histone H4, thereby enhancing Hat1p activity; similar to human RbAp46 and 48; has a role in telomeric silencing<br><br>
			<br><b>PomBase Description</b><br>kinetochore protein Mis16<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have sequence-specific DNA binding activity and role in CENP-A containing chromatin organization, kinetochore assembly, mitotic sister chromatid segregation, negative regulation of G0 to G1 transition<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8858151" target=_blank>Parthun MR, <i>et al.</i> (1996 Oct 4)</a>. The major cytoplasmic histone acetyltransferase in yeast: links to chromatin replication and histone metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9575221" target=_blank>Ruiz-Garc√≠a AB, <i>et al.</i> (1998 May 15)</a>. HAT1 and HAT2 proteins are components of a yeast nuclear histone acetyltransferase enzyme specific for free histone H4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10075411" target=_blank>Machida K, <i>et al.</i> (1999 Feb)</a>. Farnesol-induced growth inhibition in Saccharomyces cerevisiae by a cell cycle mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982821" target=_blank>Kelly TJ, <i>et al.</i> (2000 Oct)</a>. Type B histone acetyltransferase Hat1p participates in telomeric silencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751918" target=_blank>Taricani L, <i>et al.</i> (2002 Mar 22)</a>. The fission yeast ES2 homologue, Bis1, interacts with the Ish1 stress-responsive nuclear envelope protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14761951" target=_blank>Poveda A, <i>et al.</i> (2004 Apr 16)</a>. Hif1 is a component of yeast histone acetyltransferase B, a complex mainly localized in the nucleus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15099519" target=_blank>Ai X, <i>et al.</i> (2004 Apr 23)</a>. The nuclear Hat1p/Hat2p complex: a molecular link between type B histone acetyltransferases and chromatin assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15369671" target=_blank>Hayashi T, <i>et al.</i> (2004 Sep 17)</a>. Mis16 and Mis18 are required for CENP-A loading and histone deacetylation at centromeres.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023114" target=_blank>Rosaleny LE, <i>et al.</i> (2005 Aug 1)</a>. Yeast HAT1 and HAT2 deletions have different life-span and transcriptome phenotypes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18362178" target=_blank>Nakazawa N, <i>et al.</i> (2008 Mar 24)</a>. Dissection of the essential steps for condensin accumulation at kinetochores and rDNAs during fission yeast mitosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18493607" target=_blank>Natsume T, <i>et al.</i> (2008 May 21)</a>. A DNA polymerase alpha accessory protein, Mcl1, is required for propagation of centromere structures in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19217404" target=_blank>Pidoux AL, <i>et al.</i> (2009 Feb 13)</a>. Fission yeast Scm3: A CENP-A receptor required for integrity of subkinetochore chromatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19217403" target=_blank>Williams JS, <i>et al.</i> (2009 Feb 13)</a>. Fission yeast Scm3 mediates stable assembly of Cnp1/CENP-A into centromeric chromatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19366728" target=_blank>Sajiki K, <i>et al.</i> (2009 May 1)</a>. Genetic control of cellular quiescence in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23091701" target=_blank>Nakamura T, <i>et al.</i> (2012 Sep)</a>. Impaired coenzyme A synthesis in fission yeast causes defective mitosis, quiescence-exit failure, histone hypoacetylation and fragile DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22771823" target=_blank>Tong K, <i>et al.</i> (2012 Sep)</a>. Schizosaccharomyces pombe Hat1 (Kat1) is associated with Mis16 and is required for telomeric silencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375240" target=_blank>Hirai H, <i>et al.</i> (2014)</a>. The kinetochore protein Kis1/Eic1/Mis19 ensures the integrity of mitotic spindles through maintenance of kinetochore factors Mis6/CENP-I and CENP-A.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24789708" target=_blank>Subramanian L, <i>et al.</i> (2014 Apr 30)</a>. Eic1 links Mis18 with the CCAN/Mis6/Ctf19 complex to promote CENP-A assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24774534" target=_blank>Hayashi T, <i>et al.</i> (2014 Jul)</a>. Schizosaccharomyces pombe centromere protein Mis19 links Mis16 and Mis18 to recruit CENP-A through interacting with NMD factors and the SWI/SNF complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835250" target=_blank>Li Y, <i>et al.</i> (2014 Jun 1)</a>. Hat2p recognizes the histone H3 tail to specify the acetylation of the newly synthesized H3/H4 heterodimer by the Hat1p/Hat2p complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26343758" target=_blank>An S, <i>et al.</i> (2015 Oct 9)</a>. Mis16 Independently Recognizes Histone H4 and the CENP-ACnp1-Specific Chaperone Scm3sp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27666591" target=_blank>Dong Q, <i>et al.</i> (2016 Oct 6)</a>. Ccp1 Homodimer Mediates Chromatin Integrity by Antagonizing CENP-A Loading.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28827290" target=_blank>Gao C, <i>et al.</i> (2017 Oct)</a>. Kinetochore Components Required for Centromeric Chromatin Assembly Are Impacted by Msc1 in &lt;i&gt;Schizosaccharomyces pombe&lt;/i&gt;.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04584_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04584_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04584_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04584_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01612_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG04585<br>EOG8BZKJJ<br>EOG8R22D0<br><br>sce:MSI1<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of chromatin assembly factor I (CAF-1); chromatin assembly by CAF-1 affects multiple processes including silencing at telomeres, mating type loci, and rDNA; maintenance of kinetochore structure; deactivation of DNA damage checkpoint after DNA repair; chromatin dynamics during transcription; and repression of divergent noncoding transcription; Msi1p localizes to nucleus and cytoplasm and independently regulates the RAS/cAMP pathway via sequestration of Npr1p kinase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2554329" target=_blank>Ruggieri R, <i>et al.</i> (1989 Nov)</a>. MSI1, a negative regulator of the RAS-cAMP pathway in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9030687" target=_blank>Kaufman PD, <i>et al.</i> (1997 Feb 1)</a>. Ultraviolet radiation sensitivity and reduction of telomeric silencing in Saccharomyces cerevisiae cells lacking chromatin assembly factor-I.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238915" target=_blank>Johnston SD, <i>et al.</i> (2001 Mar)</a>. CAC3(MSI1) suppression of RAS2(G19V) is independent of chromatin assembly factor I and mediated by NPR1.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04585_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04585_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04585_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04585_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01612_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG04586<br>EOG8R22D0<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>Clr6 histone deacetylase complex subunit Prw1<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773392" target=_blank>Nakayama J, <i>et al.</i> (2003 Jun 2)</a>. Alp13, an MRG family protein, is a component of fission yeast Clr6 histone deacetylase required for genomic integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17450151" target=_blank>Nicolas E, <i>et al.</i> (2007 May)</a>. Distinct roles of HDAC complexes in promoter silencing, antisense suppression and DNA damage protection.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19040720" target=_blank>Shevchenko A, <i>et al.</i> (2008)</a>. Chromatin Central: towards the comparative proteome by accurate mapping of the yeast proteomic environment.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23738021" target=_blank>Zhou X, <i>et al.</i> (2013)</a>. A genome-wide screening of potential target genes to enhance the antifungal activity of micafungin in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25002536" target=_blank>Zilio N, <i>et al.</i> (2014 Sep 15)</a>. A novel histone deacetylase complex in the control of transcription and genome stability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojard√≠n L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27887640" target=_blank>Malecki M, <i>et al.</i> (2016 Nov 25)</a>. Functional and regulatory profiling of energy metabolism in fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04586_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04586_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04586_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04586_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01612_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01612.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>