<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01245.html">HOG01245&emsp;&emsp;glycosyl hydrolase 16 family. CRH1 subfamily</a><br> 
																	HOG01246&emsp;&emsp;<br> 
																	<a href="HOG01247.html">HOG01247&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01246.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01246_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01246_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01246_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03484<br>EOG8S4N15<br><br>sce:SIW14<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Tyrosine phosphatase involved in actin organization and endocytosis; localized to the cytoplasm<br><br>
			<br><b>PomBase Description</b><br>phosphoprotein phosphatase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative tyrosine phosphatase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15020461" target=_blank>Care A, <i>et al.</i> (2004 Feb)</a>. A synthetic lethal screen identifies a role for the cortical actin patch/endocytosis complex in the response to nutrient deprivation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19181872" target=_blank>Son S, <i>et al.</i> (2009 Apr)</a>. Analysis of all protein phosphatase genes in Aspergillus nidulans identifies a new mitotic regulator, fcp1.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03484_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03484_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03484_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03484_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01246_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03485<br>EOG8S4N15<br><br>sce:OCA2<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative protein of unknown function; similar to predicted tyrosine phosphatases Oca1p and Siw14p; green fluorescent protein (GFP)-fusion protein localizes to the cytoplasm; YNL056W is not an essential gene<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03485_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03485_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03485_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03485_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01246_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03486<br>EOG8S4N15<br><br>sce:OCA6<br><br>Genes: 27
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic protein required for replication of Brome mosaic virus; S. cerevisiae is a model system for studying positive-strand RNA virus replication; null mutation confers sensitivity to tunicamycin and DTT<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14671320" target=_blank>Kushner DB, <i>et al.</i> (2003 Dec 23)</a>. Systematic, genome-wide identification of host genes affecting replication of a positive-strand RNA virus.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03486_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03486_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03486_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03486_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01246_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03487<br>EOG8S4N15<br><br>sce:OCA1<br><br>Genes: 27
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative protein tyrosine phosphatase; required for cell cycle arrest in response to oxidative damage of DNA<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9757831" target=_blank>Wishart MJ, <i>et al.</i> (1998 Aug)</a>. Gathering STYX: phosphatase-like form predicts functions for unique protein-interaction domains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11408586" target=_blank>Alic N, <i>et al.</i> (2001 Jun)</a>. Identification of a Saccharomyces cerevisiae gene that is required for G1 arrest in response to the lipid oxidation product linoleic acid hydroperoxide.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12912987" target=_blank>Alic N, <i>et al.</i> (2003 Oct 24)</a>. Lipid hydroperoxides activate the mitogen-activated protein kinase Mpk1p in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16273108" target=_blank>Deutschbauer AM, <i>et al.</i> (2005 Dec)</a>. Quantitative trait loci mapped to single-nucleotide resolution in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16429126" target=_blank>Gavin AC, <i>et al.</i> (2006 Mar 30)</a>. Proteome survey reveals modularity of the yeast cell machinery.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18205808" target=_blank>Fong CS, <i>et al.</i> (2008 May)</a>. Oxidant-induced cell-cycle delay in Saccharomyces cerevisiae: the involvement of the SWI6 transcription factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03487_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03487_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03487_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03487_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01246_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03488<br>EOG8S4N15<br><br>sce:OCA4<br><br>Genes: 27
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic protein required for replication of Brome mosaic virus; S. cerevisiae is a model system for studying replication of positive-strand RNA viruses in their natural hosts<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1574927" target=_blank>Sor F, <i>et al.</i> (1992 Mar)</a>. Sequence of the HMR region on chromosome III of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14671320" target=_blank>Kushner DB, <i>et al.</i> (2003 Dec 23)</a>. Systematic, genome-wide identification of host genes affecting replication of a positive-strand RNA virus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18845848" target=_blank>Addinall SG, <i>et al.</i> (2008 Dec)</a>. A genomewide suppressor and enhancer analysis of cdc13-1 reveals varied cellular processes influencing telomere capping in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18205808" target=_blank>Fong CS, <i>et al.</i> (2008 May)</a>. Oxidant-induced cell-cycle delay in Saccharomyces cerevisiae: the involvement of the SWI6 transcription factor.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03488_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03488_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03488_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03488_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01246_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01246.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>