<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01255.html">HOG01255&emsp;&emsp;Mediator complex subunit 13 family</a><br> 
																	HOG01256&emsp;&emsp;<br> 
																	<a href="HOG01257.html">HOG01257&emsp;&emsp;AIM19 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01256.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01256_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01256_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01256_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03525<br>EOG8MW6QG<br><br>sce:GRX2<br><br>Genes: 40
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic glutaredoxin; thioltransferase, glutathione-dependent disulfide oxidoreductase involved in maintaining redox state of target proteins, also exhibits glutathione peroxidase activity, expression induced in response to stress; GRX2 has two in-frame start codons resulting in a shorter isoform that is retained in the cytosol and a longer form translocated to the mitochondrial matrix; GRX2 has a paralog, GRX1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>glutaredoxin Grx1|glutaredoxin Grx2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have glutathione disulfide oxidoreductase activity, glutathione peroxidase activity, glutathione transferase activity and role in glutathione metabolic process, removal of superoxide radicals<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2189409" target=_blank>Gan ZR, <i>et al.</i> (1990 May 16)</a>. Complete amino acid sequence of yeast thioltransferase (glutaredoxin).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1530649" target=_blank>Gan ZR, <i>et al.</i> (1992 Sep 16)</a>. Cloning and sequencing of a gene encoding yeast thioltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9571241" target=_blank>Luikenhuis S, <i>et al.</i> (1998 May)</a>. The yeast Saccharomyces cerevisiae contains two glutaredoxin genes that are required for protection against reactive oxygen species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10672936" target=_blank>Kim HG, <i>et al.</i> (1999 Dec 31)</a>. Cloning, nucleotide sequence and expression of thioltransferase (glutaredoxin) cDNA from Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118633" target=_blank>Cho YW, <i>et al.</i> (2000 Dec 15)</a>. Cloning, expression and regulation of Schizosaccharomyces pombe gene encoding thioltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10786615" target=_blank>Grant CM, <i>et al.</i> (2000 Jan 31)</a>. Differential regulation of glutaredoxin gene expression in response to stress conditions in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12521308" target=_blank>Lim CJ, <i>et al.</i> (2002 Dec 31)</a>. Pap1-dependent regulation of the GSTII gene from the fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12297001" target=_blank>Kim M, <i>et al.</i> (2002 Jul 31)</a>. Transcription of Schizosaccharomyces pombe thioltransferase-1 in response to stress conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11958675" target=_blank>Pedrajas JR, <i>et al.</i> (2002 Jun 15)</a>. Two isoforms of Saccharomyces cerevisiae glutaredoxin 2 are expressed in vivo and localize to different subcellular compartments.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11875065" target=_blank>Collinson EJ, <i>et al.</i> (2002 May 10)</a>. The yeast glutaredoxins are active as glutathione peroxidases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14503856" target=_blank>Lim CJ, <i>et al.</i> (2003 Aug 31)</a>. The thioltransferase (glutaredoxin) 1 gene of fission yeast is regulated by Atf1 and Pap1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12684511" target=_blank>Collinson EJ, <i>et al.</i> (2003 Jun 20)</a>. Role of yeast glutaredoxins as glutathione S-transferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15358115" target=_blank>Chung WH, <i>et al.</i> (2004 Sep 3)</a>. Differential expression and role of two dithiol glutaredoxins Grx1 and Grx2 in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15865206" target=_blank>Kim HG, <i>et al.</i> (2005 Mar)</a>. Carbon source-dependent regulation of a second gene encoding glutaredoxin from the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15796926" target=_blank>Chung WH, <i>et al.</i> (2005 May 6)</a>. Localization and function of three monothiol glutaredoxins in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18992757" target=_blank>Discola KF, <i>et al.</i> (2009 Jan 23)</a>. Structural aspects of the distinct biochemical properties of glutaredoxin 1 and glutaredoxin 2 from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20085751" target=_blank>Kim KD, <i>et al.</i> (2010 Feb 12)</a>. Monothiol glutaredoxin Grx5 interacts with Fe-S scaffold proteins Isa1 and Isa2 and supports Fe-S assembly and DNA integrity in mitochondria of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20417731" target=_blank>Li WF, <i>et al.</i> (2010 Jul)</a>. Structural basis for the different activities of yeast Grx1 and Grx2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23950735" target=_blank>Sun LL, <i>et al.</i> (2013)</a>. Global analysis of fission yeast mating genes reveals new autophagy factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24118096" target=_blank>García-Santamarina S, <i>et al.</i> (2013 Dec)</a>. Methionine sulphoxide reductases revisited: free methionine as a primary target of H₂O₂stress in auxotrophic fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26173815" target=_blank>Zhou H, <i>et al.</i> (2015 Oct)</a>. Genome-wide screen of fission yeast mutants for sensitivity to 6-azauracil, an inhibitor of transcriptional elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27887640" target=_blank>Malecki M, <i>et al.</i> (2016 Nov 25)</a>. Functional and regulatory profiling of energy metabolism in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28640807" target=_blank>Boronat S, <i>et al.</i> (2017 Jun)</a>. Lack of a peroxiredoxin suppresses the lethality of cells devoid of electron donors by channelling electrons to oxidized ribonucleotide reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28945192" target=_blank>Guydosh NR, <i>et al.</i> (2017 Sep 25)</a>. Regulated Ire1-dependent mRNA decay requires no-go mRNA degradation to maintain endoplasmic reticulum homeostasis in &lt;i&gt;S. pombe&lt;/i&gt;.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03525_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03525_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03525_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03525_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01256_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03526<br>EOG8MW6QG<br><br>sce:GRX7;GRX6<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cis-golgi localized monothiol glutaredoxin; more similar in activity to dithiol than other monothiol glutaredoxins; involved in the oxidative stress response; does not bind metal ions; GRX7 has a paralog, GRX6, that arose from the whole genome duplication|Cis-golgi localized monothiol glutaredoxin, binds Fe-S cluster; more similar in activity to dithiol than other monothiol glutaredoxins; involved in the oxidative stress response; GRX6 has a paralog, GRX7, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>monothiol glutaredoxin Grx3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have glutathione-disulfide reductase activity, protein homodimerization activity and role in cellular response to oxidative stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16258244" target=_blank>Moon JS, <i>et al.</i> (2005 Aug 31)</a>. Characterization and regulation of the gene encoding monothiol glutaredoxin 3 in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15796926" target=_blank>Chung WH, <i>et al.</i> (2005 May 6)</a>. Localization and function of three monothiol glutaredoxins in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18182845" target=_blank>Lee JH, <i>et al.</i> (2007 Dec 31)</a>. Expression, characterization and regulation of a Saccharomyces cerevisiae monothiol glutaredoxin (Grx6) gene in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18684775" target=_blank>Kennedy PJ, <i>et al.</i> (2008 Nov)</a>. A genome-wide screen of genes involved in cadmium tolerance in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03526_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03526_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03526_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03526_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01256_phobius.pdf">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03527<br>EOG8MW6QG<br><br>sce:PRM4<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Pheromone-regulated protein proposed to be involved in mating; predicted to have 1 transmembrane segment; transcriptionally regulated by Ste12p during mating and by Cat8p during the diauxic shift<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11062271" target=_blank>Heiman MG, <i>et al.</i> (2000 Oct 30)</a>. Prm1p, a pheromone-regulated multispanning membrane protein, facilitates plasma membrane fusion during yeast mating.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03527_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03527_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03527_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03527_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01256_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03528<br>EOG8MW6QG<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03528_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03528_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03528_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03528_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01256_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03529<br>EOG8MW6QG<br><br>sce:GRX1<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Glutathione-dependent disulfide oxidoreductase; hydroperoxide and superoxide-radical responsive, heat-stable, with active site cysteine pair; protects cells from oxidative damage; GRX1 has a paralog, GRX2, that arose from the whole genome duplication; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9571241" target=_blank>Luikenhuis S, <i>et al.</i> (1998 May)</a>. The yeast Saccharomyces cerevisiae contains two glutaredoxin genes that are required for protection against reactive oxygen species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10786615" target=_blank>Grant CM, <i>et al.</i> (2000 Jan 31)</a>. Differential regulation of glutaredoxin gene expression in response to stress conditions in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11875065" target=_blank>Collinson EJ, <i>et al.</i> (2002 May 10)</a>. The yeast glutaredoxins are active as glutathione peroxidases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12684511" target=_blank>Collinson EJ, <i>et al.</i> (2003 Jun 20)</a>. Role of yeast glutaredoxins as glutathione S-transferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17327665" target=_blank>Håkansson KO, <i>et al.</i> (2007 Mar)</a>. Structure of glutaredoxin Grx1p C30S mutant from yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18473363" target=_blank>Yu J, <i>et al.</i> (2008 Aug 15)</a>. Glutathionylation-triggered conformational changes of glutaredoxin Grx1 from the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18992757" target=_blank>Discola KF, <i>et al.</i> (2009 Jan 23)</a>. Structural aspects of the distinct biochemical properties of glutaredoxin 1 and glutaredoxin 2 from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20417731" target=_blank>Li WF, <i>et al.</i> (2010 Jul)</a>. Structural basis for the different activities of yeast Grx1 and Grx2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03529_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03529_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03529_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03529_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01256_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01256.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>