<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00564.html">HOG00564&emsp;&emsp;PAPS reductase family. CysH subfamily</a><br> 
																	HOG00565&emsp;&emsp;<br> 
																	<a href="HOG00566.html">HOG00566&emsp;&emsp;peroxidase family. Cytochrome c peroxidase subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00565.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00565_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00565_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00565_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01360<br>EOG8SQVCB<br><br>sce:MET5<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Sulfite reductase beta subunit; involved in amino acid biosynthesis, transcription repressed by methionine<br><br>
			<br><b>PomBase Description</b><br>sulfite reductase Sir1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in hydrogen sulfide biosynthetic process, sulfate assimilation, sulfur amino acid biosynthetic process and cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14009264" target=_blank>YOSHIMOTO A, <i>et al.</i> (1961 Dec)</a>. A sulfite reductase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6751400" target=_blank>Kobayashi K, <i>et al.</i> (1982 Aug 10)</a>. Studies on yeast sulfite reductase. IV. Structure and steady-state kinetics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8962070" target=_blank>Shevchenko A, <i>et al.</i> (1996 Dec 10)</a>. Linking genome and proteome by mass spectrometry: large-scale identification of yeast proteins from two dimensional gels.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335584" target=_blank>Lussier M, <i>et al.</i> (1997 Oct)</a>. Large scale identification of genes involved in cell surface biosynthesis and architecture in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20955810" target=_blank>Brzywczy J, <i>et al.</i> (2011 Feb)</a>. Novel mutations reveal two important regions in Aspergillus nidulans transcriptional activator MetR.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01360_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01360_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01360_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01360_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01361<br>EOG8SQVCB<br><br>sce:MET10<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit alpha of assimilatory sulfite reductase; complex converts sulfite into sulfide<br><br>
			<br><b>PomBase Description</b><br>sulfite reductase NADPH flavoprotein subunit (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative alpha subunit of assimilatory sulfite reductase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14009264" target=_blank>YOSHIMOTO A, <i>et al.</i> (1961 Dec)</a>. A sulfite reductase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6751400" target=_blank>Kobayashi K, <i>et al.</i> (1982 Aug 10)</a>. Studies on yeast sulfite reductase. IV. Structure and steady-state kinetics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928966" target=_blank>Hansen J, <i>et al.</i> (1994 Oct)</a>. Two divergent MET10 genes, one from Saccharomyces cerevisiae and one from Saccharomyces carlsbergensis, encode the alpha subunit of sulfite reductase and specify potential binding sites for FAD and NADPH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7788524" target=_blank>Hosseini-Mazinani SM, <i>et al.</i> (1995)</a>. Cloning and sequencing of sulfite reductase alpha subunit gene from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7698648" target=_blank>Strunnikov AV, <i>et al.</i> (1995 Mar 1)</a>. SMC2, a Saccharomyces cerevisiae gene essential for chromosome segregation and condensation, defines a subgroup within the SMC family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18684775" target=_blank>Kennedy PJ, <i>et al.</i> (2008 Nov)</a>. A genome-wide screen of genes involved in cadmium tolerance in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19053082" target=_blank>Shimizu M, <i>et al.</i> (2009 Jan)</a>. Proteomic analysis of Aspergillus nidulans cultured under hypoxic conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19171936" target=_blank>Sato I, <i>et al.</i> (2009 Mar 20)</a>. The glutathione system of Aspergillus nidulans involves a fungus-specific glutathione S-transferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24634168" target=_blank>Bernal M, <i>et al.</i> (2014 Jun)</a>. Proteome-wide search for PP2A substrates in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26152728" target=_blank>Mathiassen SG, <i>et al.</i> (2015 Aug 21)</a>. A Two-step Protein Quality Control Pathway for a Misfolded DJ-1 Variant in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01361_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01361_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01361_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01361_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG01362<br>EOG8SQVCB<br><br>sce:NCP1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>NADP-cytochrome P450 reductase; involved in ergosterol biosynthesis; associated and coordinately regulated with Erg11p<br><br>
			<br><b>PomBase Description</b><br>NADPH-cytochrome p450 reductase<br><br>
			<br><b>AspGD Description</b><br>Putative cytochrome P450 reductase; NADPH-ferrihemoprotein reductase; induced by benzoic acid; longer cprA-beta transcript carries a small uORF<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139648" target=_blank>Yabusaki Y, <i>et al.</i> (1988 Jun)</a>. Primary structure of Saccharomyces cerevisiae NADPH-cytochrome P450 reductase deduced from nucleotide sequence of its cloned gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730736" target=_blank>Turi TG, <i>et al.</i> (1992 Jan 25)</a>. Multiple regulatory elements control expression of the gene encoding the Saccharomyces cerevisiae cytochrome P450, lanosterol 14 alpha-demethylase (ERG11).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9368374" target=_blank>Lesuisse E, <i>et al.</i> (1997 Nov 1)</a>. Cytochrome P-450 reductase is responsible for the ferrireductase activity associated with isolated plasma membranes of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9468503" target=_blank>Venkateswarlu K, <i>et al.</i> (1998 Feb 20)</a>. The N-terminal membrane domain of yeast NADPH-cytochrome P450 (CYP) oxidoreductase is not required for catalytic activity in sterol biosynthesis or in reconstitution of CYP activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11485306" target=_blank>Lamb DC, <i>et al.</i> (2001 Aug 10)</a>. Activities and kinetic mechanisms of native and soluble NADPH-cytochrome P450 reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14557538" target=_blank>Hitchcock AL, <i>et al.</i> (2003 Oct 28)</a>. A subset of membrane-associated proteins is ubiquitinated in response to mutations in the endoplasmic reticulum degradation machinery.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15082539" target=_blank>Keniry ME, <i>et al.</i> (2004 Mar)</a>. The identification of Pcl1-interacting proteins that genetically interact with Cla4 may indicate a link between G1 progression and mitotic exit.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16407065" target=_blank>Lamb DC, <i>et al.</i> (2006 Jan)</a>. A second FMN binding site in yeast NADPH-cytochrome P450 reductase suggests a mechanism of electron transfer by diflavin reductases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16823961" target=_blank>Reinders J, <i>et al.</i> (2006 Jul)</a>. Toward the complete yeast mitochondrial proteome: multidimensional separation techniques for mitochondrial proteomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16407407" target=_blank>Zahedi RP, <i>et al.</i> (2006 Mar)</a>. Proteomic analysis of the yeast mitochondrial outer membrane reveals accumulation of a subclass of preproteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19483672" target=_blank>Aigrain L, <i>et al.</i> (2009 Jul)</a>. Structure of the open conformation of a functional chimeric NADPH cytochrome P450 reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20557022" target=_blank>Ivanov AS, <i>et al.</i> (2010 Aug 20)</a>. FMN binding site of yeast NADPH-cytochrome P450 reductase exposed at the surface is highly specific.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01362_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01362_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01362_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01362_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>16 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG01363<br>EOG8SQVCB<br><br>sce:TAH18<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Conserved NAPDH-dependent diflavin reductase; component of an early step in the cytosolic Fe-S protein assembly (CIA) machinery; transfers electrons from NADPH to the Fe-S cluster of Dre2p; plays a pro-death role under oxidative stress; Tah18p-dependent nitric oxide synthesis confers high-temperature stress tolerance; possible target for development of antifungal drugs<br><br>
			<br><b>PomBase Description</b><br>NADPH-dependent diflavin oxidoreductase, involved in iron-sulfur cluster assembly Tah18 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted FMN binding, iron ion binding, oxidoreductase activity and role in oxidation-reduction process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12759774" target=_blank>Chanet R, <i>et al.</i> (2003 Aug)</a>. Characterization of mutations that are synthetic lethal with pol3-13, a mutated allele of DNA polymerase delta in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19194512" target=_blank>Vernis L, <i>et al.</i> (2009)</a>. A newly identified essential complex, Dre2-Tah18, controls mitochondria integrity and cell death after oxidative stress in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20802492" target=_blank>Netz DJ, <i>et al.</i> (2010 Oct)</a>. Tah18 transfers electrons to Dre2 in cytosolic iron-sulfur protein biogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21902732" target=_blank>Soler N, <i>et al.</i> (2011 Oct)</a>. Interaction between the reductase Tah18 and highly conserved Fe-S containing Dre2 C-terminus is essential for yeast viability.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01363_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01363_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01363_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01363_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG01364<br>EOG8SQVCB<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>NADPH-ferrihemoprotein reductase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01364_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01364_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01364_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01364_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00565.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>