<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01172.html">HOG01172&emsp;&emsp;TRAFAC class myosin-kinesin ATPase superfamily. Myosin family; protein kinase superfamily. CAMK Ser/Thr protein kinase family. CHEK2 subfamily; protein kinase superfamily. CAMK Ser/Thr protein kinase family. CaMK subfamily; protein kinase superfamily. CAMK Ser/Thr protein kinase family. NIM1 subfamily; protein kinase superfamily. CAMK Ser/Thr protein kinase family. SNF1 subfamily; protein kinase superfamily. Ser/Thr protein kinase family; protein kinase superfamily. Ser/Thr protein kinase family. Aurora subfamily; protein kinase superfamily. Ser/Thr protein kinase family. CDC5/Polo subfamily</a><br> 
																	HOG01173&emsp;&emsp;<br> 
																	<a href="HOG01174.html">HOG01174&emsp;&emsp;protein kinase superfamily. CMGC Ser/Thr protein kinase family. CDC2/CDKX subfamily; protein kinase superfamily. CMGC Ser/Thr protein kinase family. MAP kinase subfamily; protein kinase superfamily. Ser/Thr protein kinase family; protein kinase superfamily. Ser/Thr protein kinase family. MAP kinase subfamily. HOG1 sub-subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01173.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01173_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01173_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02959<br>EOG88SF7W<br><br>sce:BAG7;SAC7<br><br>Genes: 39
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Rho GTPase activating protein (RhoGAP); stimulates the intrinsic GTPase activity of Rho1p, which plays a bud growth by regulating actin cytoskeleton organization and cell wall biosynthesis, resulting in the downregulation of Rho1p; structurally and functionally related to Sac7p; BAG7 has a paralog, SAC7, that arose from the whole genome duplication|GTPase activating protein (GAP) for Rho1p; regulator of a Tor2p-mediated, Rho1p GTPase switch that controls organization of the actin cytoskeleton; negative regulator of the RHO1-PKC1-MAPK cell integrity (CWI) and membrane fluidity homeostasis signaling pathways; potential Cdc28p substrate; SAC7 has a paralog, BAG7, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>Rho-type GTPase activating protein Rga5<br><br>
			<br><b>AspGD Description</b><br>Rho-GTPase activating protein (Rho-GAP); induced by caspofungin<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183030" target=_blank>Dunn TM, <i>et al.</i> (1990 May)</a>. Null alleles of SAC7 suppress temperature-sensitive actin mutations in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9038344" target=_blank>Schmidt A, <i>et al.</i> (1997 Feb 21)</a>. The yeast phosphatidylinositol kinase homolog TOR2 activates RHO1 and RHO2 via the exchange factor ROM2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11591390" target=_blank>Roumanie O, <i>et al.</i> (2001 Oct 5)</a>. Functional characterization of the Bag7, Lrg1 and Rgd2 RhoGAP proteins from Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02959_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02959_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02959_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02959_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02960<br>EOG88SF7W<br><br>sce:RGD1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>GTPase-activating protein (RhoGAP) for Rho3p and Rho4p; possibly involved in control of actin cytoskeleton organization<br><br>
			<br><b>PomBase Description</b><br>Rho-type GTPase activating protein Rga7<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have GTPase activator activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9769859" target=_blank>Barthe C, <i>et al.</i> (1998 Jun)</a>. First characterization of the gene RGD1 in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10590461" target=_blank>de Bettignies G, <i>et al.</i> (1999 Dec)</a>. RGD1 genetically interacts with MID2 and SLG1, encoding two putative sensors for cell integrity signalling in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10526184" target=_blank>Doignon F, <i>et al.</i> (1999 Oct 15)</a>. The yeast Rgd1p is a GTPase activating protein of the Rho3 and rho4 proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11779787" target=_blank>de Bettignies G, <i>et al.</i> (2001 Dec)</a>. Overactivation of the protein kinase C-signaling pathway suppresses the defects of cells lacking the Rho3/Rho4-GAP Rgd1p in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16087742" target=_blank>Claret S, <i>et al.</i> (2005 Aug)</a>. The Rgd1p Rho GTPase-activating protein and the Mid2p cell wall sensor are required at low pH for protein kinase C pathway activation and cell survival in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15922872" target=_blank>Gatti X, <i>et al.</i> (2005 May 23)</a>. RGD1, encoding a RhoGAP involved in low-pH survival, is an Msn2p/Msn4p regulated gene in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23454383" target=_blank>Vieillemard A, <i>et al.</i> (2013 Mar 29)</a>. The Saccharomyces cerevisiae RhoGAP Rgd1 is phosphorylated by the Aurora B like kinase Ipl1.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02960_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02960_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02960_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02960_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02961<br>EOG88SF7W<br><br>sce:LRG1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>GTPase-activating protein (GAP); contains Rho1p-specific GAP activity, interacting with activated forms of Rho1p; functions along with Sac7p as a negative regulator of the Pkc1p-mediated cell wall integrity signaling pathway; negative regulator of cell wall 1,3-beta-glucan biosynthesis; required for efficient cell fusion; contains a RhoGAP domain and three Lin-11-Isl1-Mec-3 (LIM) domains<br><br>
			<br><b>PomBase Description</b><br>Rho-type GTPase activating protein Rga1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have GTPase activator activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065929" target=_blank>Müller L, <i>et al.</i> (1994 Aug 11)</a>. LRG1 is expressed during sporulation in Saccharomyces cerevisiae and contains motifs similar to LIM and rho/racGAP domains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11447600" target=_blank>Watanabe D, <i>et al.</i> (2001 Jul)</a>. Yeast Lrg1p acts as a specialized RhoGAP regulating 1,3-beta-glucan synthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11591390" target=_blank>Roumanie O, <i>et al.</i> (2001 Oct 5)</a>. Functional characterization of the Bag7, Lrg1 and Rgd2 RhoGAP proteins from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02961_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02961_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02961_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02961_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02962<br>EOG88SF7W<br><br>sce:BEM3<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Rho GTPase activating protein (RhoGAP); involved in control of the cytoskeleton organization; targets the essential Rho-GTPase Cdc42p, which controls establishment and maintenance of cell polarity, including bud-site assembly<br><br>
			<br><b>PomBase Description</b><br>Rho-type GTPase activating protein Rga2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have GTPase activator activity, role in negative regulation of Rho protein signal transduction, positive regulation of GTPase activity and actomyosin contractile ring, growing cell tip localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8227021" target=_blank>Zheng Y, <i>et al.</i> (1993 Nov 25)</a>. Biochemical comparisons of the Saccharomyces cerevisiae Bem2 and Bem3 proteins. Delineation of a limit Cdc42 GTPase-activating protein domain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300560" target=_blank>Zheng Y, <i>et al.</i> (1994 Jan 28)</a>. Control of the yeast bud-site assembly GTPase Cdc42. Catalysis of guanine nucleotide exchange by Cdc24 and stimulation of GTPase activity by Bem3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02962_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02962_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02962_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02962_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>9 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG02963<br>EOG88SF7W<br><br>sce:PXL1<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein that localizes to sites of polarized growth; required for selection and/or maintenance of polarized growth sites, may modulate signaling by the GTPases Cdc42p and Rho1p; contains LIM domains and has similarity to metazoan paxillin; relocalizes from bud neck to cytoplasm upon DNA replication stress<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted zinc ion binding activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02963_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02963_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02963_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02963_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG02964<br>EOG88SF7W<br><br>sce:BEM2<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Rho GTPase activating protein (RhoGAP); involved in the control of cytoskeleton organization and cellular morphogenesis; required for bud emergence; potential GAP for Rho4p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8293973" target=_blank>Chan CS, <i>et al.</i> (1993 Nov)</a>. Isolation and characterization of chromosome-gain and increase-in-ploidy mutants in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7962097" target=_blank>Kim YJ, <i>et al.</i> (1994 Dec)</a>. Control of cellular morphogenesis by the Ip12/Bem2 GTPase-activating protein: possible role of protein phosphorylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7962098" target=_blank>Peterson J, <i>et al.</i> (1994 Dec)</a>. Interactions between the bud emergence proteins Bem1p and Bem2p and Rho-type GTPases in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10751152" target=_blank>Wendland J, <i>et al.</i> (2000 May)</a>. Determination of cell polarity in germinated spores and hyphal tips of the filamentous ascomycete Ashbya gossypii requires a rhoGAP homolog.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02964_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02964_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02964_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02964_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>16 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG02965<br>EOG88SF7W<br><br>sce:RGA1;RGA2<br><br>Genes: 24
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>GTPase-activating protein for polarity-establishment protein Cdc42p; implicated in control of septin organization, pheromone response, and haploid invasive growth; relocalizes from bud neck to cytoplasm upon DNA replication stress; RGA1 has a paralog, RGA2, that arose from the whole genome duplication|GTPase-activating protein for polarity-establishment protein Cdc42p; implicated in control of septin organization, pheromone response, and haploid invasive growth; regulated by Pho85p and Cdc28p; RGA2 has a paralog, RGA1, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1454852" target=_blank>Ramer SW, <i>et al.</i> (1992 Dec 1)</a>. Dominant genetics using a yeast genomic library under the control of a strong inducible promoter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7498791" target=_blank>Stevenson BJ, <i>et al.</i> (1995 Dec 1)</a>. Mutation of RGA1, which encodes a putative GTPase-activating protein for the polarity-establishment protein Cdc42p, activates the pheromone-response pathway in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8657111" target=_blank>Chen GC, <i>et al.</i> (1996 Apr)</a>. The LIM domain-containing Dbm1 GTPase-activating protein is required for normal cellular morphogenesis in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02965_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02965_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02965_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02965_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG02966<br>EOG88SF7W<br>EOG89S4Q0<br><br>sce:ECM25<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Non-essential protein of unknown function; promoter contains a consensus binding sequence for factor Abf1p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8341697" target=_blank>Arenas JE, <i>et al.</i> (1993 Jul 15)</a>. The Saccharomyces cerevisiae PRP21 gene product is an integral component of the prespliceosome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335584" target=_blank>Lussier M, <i>et al.</i> (1997 Oct)</a>. Large scale identification of genes involved in cell surface biosynthesis and architecture in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02966_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02966_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02966_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02966_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG02967<br>EOG88SF7W<br><br>sce:absent<br><br>Genes: 11
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>Rho-type GTPase activating protein Rga4<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have GTPase activator activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11737264" target=_blank>Nakano K, <i>et al.</i> (2001 Dec)</a>. Characterization of GTPase-activating proteins for the function of the Rho-family small GTPases in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17377067" target=_blank>Das M, <i>et al.</i> (2007 Jun)</a>. Regulation of cell diameter, For3p localization, and cell symmetry by fission yeast Rho-GAP Rga4p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18328707" target=_blank>Tatebe H, <i>et al.</i> (2008 Mar 11)</a>. Pom1 DYRK regulates localization of the Rga4 GAP to ensure bipolar activation of Cdc42 in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19646873" target=_blank>Das M, <i>et al.</i> (2009 Aug 11)</a>. The conserved NDR kinase Orb6 controls polarized cell growth by spatial regulation of the small GTPase Cdc42.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20164182" target=_blank>Soto T, <i>et al.</i> (2010 Apr 9)</a>. Rga4 modulates the activity of the fission yeast cell integrity MAPK pathway by acting as a Rho2 GTPase-activating protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20870879" target=_blank>Dodgson J, <i>et al.</i> (2010 Nov)</a>. Reorganization of the growth pattern of Schizosaccharomyces pombe in invasive filament formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21057634" target=_blank>Cansado J, <i>et al.</i> (2010 Sep)</a>. Rga4, a Rho-GAP from fission yeast: Finding specificity within promiscuity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21340088" target=_blank>Arita Y, <i>et al.</i> (2011 May)</a>. Microarray-based target identification using drug hypersensitive fission yeast expressing ORFeome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849474" target=_blank>Kelly FD, <i>et al.</i> (2011 Oct)</a>. Spatial control of Cdc42 activation determines cell width in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24146635" target=_blank>Vjestica A, <i>et al.</i> (2013)</a>. Hsp70-Hsp40 chaperone complex functions in controlling polarized growth by repressing Hsf1-driven heat stress-associated transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23615447" target=_blank>Deng L, <i>et al.</i> (2013 Jun)</a>. Compartmentalized nodes control mitotic entry signaling in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24554432" target=_blank>Kokkoris K, <i>et al.</i> (2014 May 1)</a>. The Tea4-PP1 landmark promotes local growth by dual Cdc42 GEF recruitment and GAP exclusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26246599" target=_blank>Das M, <i>et al.</i> (2015 Oct 1)</a>. Phosphorylation-dependent inhibition of Cdc42 GEF Gef1 by 14-3-3 protein Rad24 spatially regulates Cdc42 GTPase activity and oscillatory dynamics during cell morphogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26960792" target=_blank>Revilla-Guarinos MT, <i>et al.</i> (2016 Mar 9)</a>. Rga6 is a Fission Yeast Rho GAP Involved in Cdc42 Regulation of Polarized Growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27168121" target=_blank>Chen JS, <i>et al.</i> (2016 Sep)</a>. Discovery of genes involved in mitosis, cell division, cell wall integrity and chromosome segregation through construction of Schizosaccharomyces pombe deletion strains.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02967_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02967_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02967_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02967_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG02968<br>EOG89S4Q0<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02968_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02968_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02968_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02968_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG02969<br>EOG88SF7W<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in signal transduction and intracellular localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02969_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02969_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02969_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02969_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG02970<br>EOG88SF7W<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted zinc ion binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02970_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02970_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02970_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02970_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG02971<br>EOG88SF7W<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>paxillin-like protein Pxl1<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18272786" target=_blank>Ge W, <i>et al.</i> (2008 Apr)</a>. Pxl1p, a paxillin-related protein, stabilizes the actomyosin ring during cytokinesis in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18256290" target=_blank>Pinar M, <i>et al.</i> (2008 Apr)</a>. Schizosaccharomyces pombe Pxl1 is a paxillin homologue that modulates Rho1 activity and participates in cytokinesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19139265" target=_blank>Roberts-Galbraith RH, <i>et al.</i> (2009 Jan 12)</a>. The SH3 domains of two PCH family members cooperate in assembly of the Schizosaccharomyces pombe contractile ring.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23861937" target=_blank>Zhang L, <i>et al.</i> (2013)</a>. Genome-wide screening for genes associated with valproic acid sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25052092" target=_blank>Martín-García R, <i>et al.</i> (2014 Oct 1)</a>. F-BAR domain protein Rga7 collaborates with Cdc15 and Imp2 to ensure proper cytokinesis in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojardín L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26702831" target=_blank>McDonald NA, <i>et al.</i> (2015 Dec 21)</a>. Oligomerization but Not Membrane Bending Underlies the Function of Certain F-BAR Proteins in Cell Motility and Cytokinesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25428987" target=_blank>Ren L, <i>et al.</i> (2015 Jan 15)</a>. The Cdc15 and Imp2 SH3 domains cooperatively scaffold a network of proteins that redundantly ensure efficient cell division in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26132084" target=_blank>Cortés JC, <i>et al.</i> (2015 Jul)</a>. Cooperation between Paxillin-like Protein Pxl1 and Glucan Synthase Bgs1 Is Essential for Actomyosin Ring Stability and Septum Formation in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27385337" target=_blank>Liu Y, <i>et al.</i> (2016 Aug 15)</a>. Roles of the novel coiled-coil protein Rng10 in septum formation during fission yeast cytokinesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27053105" target=_blank>Burr R, <i>et al.</i> (2016 Jun 3)</a>. Mga2 Transcription Factor Regulates an Oxygen-responsive Lipid Homeostasis Pathway in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27749909" target=_blank>Sethi K, <i>et al.</i> (2016 Oct)</a>. A New Membrane Protein Sbg1 Links the Contractile Ring Apparatus and Septum Synthesis Machinery in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28784611" target=_blank>Snider CE, <i>et al.</i> (2017 Oct 2)</a>. Phosphoinositide-mediated ring anchoring resists perpendicular forces to promote medial cytokinesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28914606" target=_blank>McDonald NA, <i>et al.</i> (2017 Sep 15)</a>. Nanoscale architecture of the &lt;i&gt;Schizosaccharomyces pombe&lt;/i&gt; contractile ring.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02971_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02971_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02971_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02971_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG02972<br>EOG88SF7W<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>Rho-type GTPase activating protein (predicted)|Rho-type GTPase activating protein Rga3|Rho-type GTPase activating protein Rga6 (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10759889" target=_blank>Ding DQ, <i>et al.</i> (2000 Mar)</a>. Large-scale screening of intracellular protein localization in living fission yeast cells by the use of a GFP-fusion genomic DNA library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11737264" target=_blank>Nakano K, <i>et al.</i> (2001 Dec)</a>. Characterization of GTPase-activating proteins for the function of the Rho-family small GTPases in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618370" target=_blank>Decottignies A, <i>et al.</i> (2003 Mar)</a>. Schizosaccharomyces pombe essential genes: a pilot study.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16169489" target=_blank>Gregan J, <i>et al.</i> (2005 Sep 20)</a>. Novel genes required for meiotic chromosome segregation are identified by a high-throughput knockout screen in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20404563" target=_blank>Rumpf C, <i>et al.</i> (2010 May)</a>. High-throughput knockout screen in Schizosaccharomyces pombe identifies a novel gene required for efficient homolog disjunction during meiosis I.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27168121" target=_blank>Chen JS, <i>et al.</i> (2016 Sep)</a>. Discovery of genes involved in mitosis, cell division, cell wall integrity and chromosome segregation through construction of Schizosaccharomyces pombe deletion strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02972_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02972_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02972_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02972_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01173_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01173.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>