<html>
<head><title>HOG01179</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01178.html">HOG01178&emsp;&emsp;UPF0057 (PMP3) family</a><br> 
																	HOG01179&emsp;&emsp;<br> 
																	<a href="HOG01180.html">HOG01180&emsp;&emsp;DEAD box helicase family. DDX10/DBP4 subfamily; DEAD box helicase family. DDX18/HAS1 subfamily; DEAD box helicase family. DDX27/DRS1 subfamily; DEAD box helicase family. DDX31/DBP7 subfamily; DEAD box helicase family. DDX47/RRP3 subfamily; DEAD box helicase family. DDX49/DBP8 subfamily; DEAD box helicase family. DDX54/DBP10 subfamily; DEAD box helicase family. DDX55/SPB4 subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01179.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01179_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01179_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG01179.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03028<br>EOG81NS34<br>EOG8MCVN8<br><br>sce:KIP1;CIN8<br><br>Genes: 43
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Kinesin-related motor protein; required for mitotic spindle assembly, chromosome segregation, and 2 micron plasmid partitioning; functionally redundant with Cin8p for chromosomal but not plasmid functions|Kinesin motor protein; involved in mitotic spindle assembly and chromosome segregation<br><br>
			<br><b>PomBase Description</b><br>kinesin-like protein Cut7<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have microtubule binding, microtubule motor activity and role in mitotic spindle disassembly, mitotic spindle midzone assembly, spindle pole body separation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=773766" target=_blank>Morris NR, <i>et al.</i> (1975 Dec)</a>. Mitotic mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2138511" target=_blank>Enos AP, <i>et al.</i> (1990 Mar 23)</a>. Mutation of a gene that encodes a kinesin-like protein blocks nuclear division in A. nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1819513" target=_blank>Enos AP, <i>et al.</i> (1991)</a>. Kinesin-like proteins of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1710028" target=_blank>Le Guellec R, <i>et al.</i> (1991 Jun)</a>. Cloning by differential screening of a Xenopus cDNA that encodes a kinesin-related protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1745237" target=_blank>Lloyd AT, <i>et al.</i> (1991 Nov)</a>. Codon usage in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1643659" target=_blank>Saunders WS, <i>et al.</i> (1992 Aug 7)</a>. Kinesin-related proteins required for structural integrity of the mitotic spindle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1618897" target=_blank>Hoyt MA, <i>et al.</i> (1992 Jul)</a>. Two Saccharomyces cerevisiae kinesin-related gene products required for mitotic spindle assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1618910" target=_blank>Roof DM, <i>et al.</i> (1992 Jul)</a>. Kinesin-related proteins required for assembly of the mitotic spindle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1336015" target=_blank>Doonan JH, <i>et al.</i> (1992 Nov)</a>. Cell division in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416986" target=_blank>O'Connell MJ, <i>et al.</i> (1993 Jan)</a>. Suppression of the bimC4 mitotic spindle defect by deletion of klpA, a gene encoding a KAR3-related kinesin-like protein in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7704303" target=_blank>Okano K, <i>et al.</i> (1994 Nov)</a>. cDNA structure and characterization of a kinesin-like protein from the silkworm Bombyx mori.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7867803" target=_blank>Rousselet G, <i>et al.</i> (1995 Feb 13)</a>. A second nitrogen permease regulator in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9268050" target=_blank>Kashina AS, <i>et al.</i> (1997 Jul 24)</a>. The bimC family of kinesins: essential bipolar mitotic motors driving centrosome separation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9490630" target=_blank>Drummond DR, <i>et al.</i> (1998 Apr)</a>. Mutations in the bimC box of Cut7 indicate divergence of regulation within the bimC family of kinesin related proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9584089" target=_blank>Efimov VP, <i>et al.</i> (1998 May)</a>. A screen for dynein synthetic lethals in Aspergillus nidulans identifies spindle assembly checkpoint genes and other genes involved in mitosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14530265" target=_blank>Stock MF, <i>et al.</i> (2003 Dec 26)</a>. The kinesin family member BimC contains a second microtubule binding region attached to the N terminus of the motor domain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12525161" target=_blank>DeBonis S, <i>et al.</i> (2003 Jan 21)</a>. Interaction of the mitotic inhibitor monastrol with human kinesin Eg5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998523" target=_blank>Osmani SA, <i>et al.</i> (2004 Apr)</a>. The early impact of genetics on our understanding of cell cycle regulation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998524" target=_blank>Xiang X, <i>et al.</i> (2004 Apr)</a>. Nuclear migration and positioning in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15189985" target=_blank>Rischitor PE, <i>et al.</i> (2004 Jun)</a>. The Kip3-like kinesin KipB moves along microtubules and determines spindle position during synchronized mitoses in Aspergillus nidulans hyphae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17191952" target=_blank>Bevan DR, <i>et al.</i> (2005 Nov)</a>. Application of molecular modeling to analysis of inhibition of kinesin motor proteins of the BimC subfamily by monastrol and related compounds.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22645705" target=_blank>Pe√±alva MA, <i>et al.</i> (2012 Jan 1)</a>. Searching for gold beyond mitosis: Mining intracellular membrane traffic in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26117688" target=_blank>Wang B, <i>et al.</i> (2015 Sep)</a>. The Aspergillus nidulans bimC4 mutation provides an excellent tool for identification of kinesin-14 inhibitors.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03028_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03028_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03028_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03028_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03029<br>EOG81NS34<br><br>sce:KIP2<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Kinesin-related motor protein involved in mitotic spindle positioning; stabilizes microtubules by targeting Bik1p to the plus end; Kip2p levels are controlled during the cell cycle<br><br>
			<br><b>PomBase Description</b><br>kinesin-like protein Tea2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent microtubule motor activity, plus-end-directed, microtubule plus-end binding, plus-end directed microfilament motor activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1618910" target=_blank>Roof DM, <i>et al.</i> (1992 Jul)</a>. Kinesin-related proteins required for assembly of the mitotic spindle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10875281" target=_blank>Suelmann R, <i>et al.</i> (2000 May)</a>. Nuclear migration in fungi--different motors at work.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998524" target=_blank>Xiang X, <i>et al.</i> (2004 Apr)</a>. Nuclear migration and positioning in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15189985" target=_blank>Rischitor PE, <i>et al.</i> (2004 Jun)</a>. The Kip3-like kinesin KipB moves along microtubules and determines spindle position during synchronized mitoses in Aspergillus nidulans hyphae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15563609" target=_blank>Konzack S, <i>et al.</i> (2005 Feb)</a>. The role of the kinesin motor KipA in microtubule organization and polarized growth of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16467375" target=_blank>Efimov VP, <i>et al.</i> (2006 Apr)</a>. CLIP-170 homologue and NUDE play overlapping roles in NUDF localization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17237365" target=_blank>Enke C, <i>et al.</i> (2007 Mar)</a>. Aspergillus nidulans Dis1/XMAP215 protein AlpA localizes to spindle pole bodies and microtubule plus ends and contributes to growth directionality.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18003978" target=_blank>Takeshita N, <i>et al.</i> (2008 Jan)</a>. Apical sterol-rich membranes are essential for localizing cell end markers that determine growth directionality in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18399939" target=_blank>Fischer R, <i>et al.</i> (2008 May)</a>. Polarized growth in fungi--interplay between the cytoskeleton, positional markers and membrane domains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19037104" target=_blank>Zekert N, <i>et al.</i> (2009 Jan)</a>. The Aspergillus nidulans kinesin-3 UncA motor moves vesicles along a subpopulation of microtubules.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19429780" target=_blank>Higashitsuji Y, <i>et al.</i> (2009 Jul)</a>. The cell end marker protein TeaC is involved in growth directionality and septation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20034586" target=_blank>Chen S, <i>et al.</i> (2010 Mar)</a>. Localization and function of calmodulin in live-cells of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21392133" target=_blank>Herrero S, <i>et al.</i> (2011 May)</a>. The Aspergillus nidulans CENP-E kinesin motor KipA interacts with the fungal homologue of the centromere-associated protein CENP-H at the kinetochore.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21785949" target=_blank>Schunck T, <i>et al.</i> (2011 Oct)</a>. The Aspergillus nidulans CENP-E kinesin KipA is able to dimerize and to move processively along microtubules.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127389" target=_blank>Egan MJ, <i>et al.</i> (2012 Dec)</a>. Microtubule-based transport in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24101725" target=_blank>Takeshita N, <i>et al.</i> (2013 Dec 1)</a>. The cell-end marker TeaA and the microtubule polymerase AlpA contribute to microtubule guidance at the hyphal tip cortex of Aspergillus nidulans to provide polarity maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24951440" target=_blank>Herrero S, <i>et al.</i> (2014 Aug)</a>. F-box protein RcyA controls turnover of the kinesin-7 motor KipA in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26117688" target=_blank>Wang B, <i>et al.</i> (2015 Sep)</a>. The Aspergillus nidulans bimC4 mutation provides an excellent tool for identification of kinesin-14 inhibitors.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03029_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03029_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03029_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03029_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03030<br>EOG81NS34<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have role in mitochondrion transport along microtubule, vesicle transport along microtubule and cytoplasmic microtubule, cytoplasmic vesicle, early endosome localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15189985" target=_blank>Rischitor PE, <i>et al.</i> (2004 Jun)</a>. The Kip3-like kinesin KipB moves along microtubules and determines spindle position during synchronized mitoses in Aspergillus nidulans hyphae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19037104" target=_blank>Zekert N, <i>et al.</i> (2009 Jan)</a>. The Aspergillus nidulans kinesin-3 UncA motor moves vesicles along a subpopulation of microtubules.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22194856" target=_blank>Zhang J, <i>et al.</i> (2011)</a>. Aspergillus myosin-V supports polarized growth in the absence of microtubule-based transport.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22363525" target=_blank>Seidel C, <i>et al.</i> (2012)</a>. The Aspergillus nidulans kinesin-3 tail is necessary and sufficient to recognize modified microtubules.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24376868" target=_blank>Etxebeste O, <i>et al.</i> (2013)</a>. Cytoplasmic dynamics of the general nuclear import machinery in apically growing syncytial cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23687116" target=_blank>Seidel C, <i>et al.</i> (2013 Jul)</a>. Neurospora crassa NKIN2, a kinesin-3 motor, transports early endosomes and is required for polarized growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25955346" target=_blank>Takeshita N, <i>et al.</i> (2015)</a>. Transportation of Aspergillus nidulans Class III and V Chitin Synthases to the Hyphal Tips Depends on Conventional Kinesin.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03030_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03030_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03030_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03030_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03031<br>EOG81NS34<br>EOG8Q2BXV<br><br>sce:absent<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>kinesin-like protein Klp8<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have microtubule motor activity, protein homodimerization activity and cell tip, kinesin complex, medial cortical node, mitotic spindle pole body, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18931302" target=_blank>Dixon SJ, <i>et al.</i> (2008 Oct 28)</a>. Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19474789" target=_blank>Moseley JB, <i>et al.</i> (2009 Jun 11)</a>. A spatial gradient coordinates cell size and mitotic entry in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24508166" target=_blank>Deng L, <i>et al.</i> (2014 Feb 17)</a>. Dueling kinases regulate cell size at division through the SAD kinase Cdr2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28539404" target=_blank>Akamatsu M, <i>et al.</i> (2017 Nov 7)</a>. Analysis of interphase node proteins in fission yeast by quantitative and superresolution fluorescence microscopy.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03031_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03031_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03031_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03031_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03032<br>EOG81NS34<br><br>sce:KIP3<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Kinesin-related antiparallel sliding motor protein; involved in mitotic spindle positioning; sliding activity promotes bipolar spindle assembly and maintenance of genome stability; inhibits spindle elongation, destabilizing late anaphase spindle microtubules that polymerize beyond the midzone<br><br>
			<br><b>PomBase Description</b><br>kinesin-like protein Klp5|kinesin-like protein Klp6<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent microtubule motor activity, plus-end-directed activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998524" target=_blank>Xiang X, <i>et al.</i> (2004 Apr)</a>. Nuclear migration and positioning in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15189985" target=_blank>Rischitor PE, <i>et al.</i> (2004 Jun)</a>. The Kip3-like kinesin KipB moves along microtubules and determines spindle position during synchronized mitoses in Aspergillus nidulans hyphae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15563609" target=_blank>Konzack S, <i>et al.</i> (2005 Feb)</a>. The role of the kinesin motor KipA in microtubule organization and polarized growth of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21131437" target=_blank>de Castro Pimentel Figueiredo B, <i>et al.</i> (2011 Feb)</a>. The Aspergillus nidulans nucA(EndoG) homologue is not involved in cell death.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127389" target=_blank>Egan MJ, <i>et al.</i> (2012 Dec)</a>. Microtubule-based transport in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23001566" target=_blank>Cederquist GY, <i>et al.</i> (2012 Dec 15)</a>. An inherited TUBB2B mutation alters a kinesin-binding site and causes polymicrogyria, CFEOM and axon dysinnervation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03032_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03032_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03032_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03032_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03033<br>EOG81NS34<br><br>sce:KAR3<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Minus-end-directed microtubule motor; functions in mitosis and meiosis, localizes to the spindle pole body and localization is dependent on functional Cik1p, required for nuclear fusion during mating; potential Cdc28p substrate<br><br>
			<br><b>PomBase Description</b><br>minus-end directed microtubule motor kinesin-14, Klp2|minus-end directed microtubule motor, kinesin Pkl1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent microtubule motor activity, minus-end-directed activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2138512" target=_blank>Meluh PB, <i>et al.</i> (1990 Mar 23)</a>. KAR3, a kinesin-related gene required for yeast nuclear fusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416986" target=_blank>O'Connell MJ, <i>et al.</i> (1993 Jan)</a>. Suppression of the bimC4 mitotic spindle defect by deletion of klpA, a gene encoding a KAR3-related kinesin-like protein in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8224825" target=_blank>Hoyt MA, <i>et al.</i> (1993 Sep)</a>. Loss of function of Saccharomyces cerevisiae kinesin-related CIN8 and KIP1 is suppressed by KAR3 motor domain mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7912193" target=_blank>Endow SA, <i>et al.</i> (1994 Jun 1)</a>. Yeast Kar3 is a minus-end microtubule motor protein that destabilizes microtubules preferentially at the minus ends.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9485302" target=_blank>Gulick AM, <i>et al.</i> (1998 Feb 17)</a>. X-ray crystal structure of the yeast Kar3 motor domain complexed with Mg.ADP to 2.3 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10087265" target=_blank>Manning BD, <i>et al.</i> (1999 Mar 22)</a>. Differential regulation of the Kar3p kinesin-related protein by two associated proteins, Cik1p and Vik1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11729143" target=_blank>Shanks RM, <i>et al.</i> (2001 Nov)</a>. The Kar3-interacting protein Cik1p plays a critical role in passage through meiosis I in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11598200" target=_blank>Prigozhina NL, <i>et al.</i> (2001 Oct)</a>. Gamma-tubulin and the C-terminal motor domain kinesin-like protein, KLPA, function in the establishment of spindle bipolarity in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15189985" target=_blank>Rischitor PE, <i>et al.</i> (2004 Jun)</a>. The Kip3-like kinesin KipB moves along microtubules and determines spindle position during synchronized mitoses in Aspergillus nidulans hyphae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23043140" target=_blank>Duan D, <i>et al.</i> (2012 Nov 23)</a>. Neck rotation and neck mimic docking in the noncatalytic Kar3-associated protein Vik1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23137538" target=_blank>Delorme C, <i>et al.</i> (2012 Nov 30)</a>. Crystal structure of the Candida albicans Kar3 kinesin motor domain fused to maltose-binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26117688" target=_blank>Wang B, <i>et al.</i> (2015 Sep)</a>. The Aspergillus nidulans bimC4 mutation provides an excellent tool for identification of kinesin-14 inhibitors.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03033_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03033_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03033_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03033_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG03034<br>EOG81NS34<br><br>sce:absent<br><br>Genes: 25
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>kinesin-like protein Klp3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent microtubule motor activity, plus-end-directed, protein dimerization activity, tubulin binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10641037" target=_blank>Brazer SC, <i>et al.</i> (2000 Jan 30)</a>. A fission yeast kinesin affects Golgi membrane recycling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11018050" target=_blank>Browning H, <i>et al.</i> (2000 Oct 2)</a>. Tea2p is a kinesin-like protein required to generate polarized growth in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12132578" target=_blank>Jeong JW, <i>et al.</i> (2002 Jun 30)</a>. Cloning and characterization of the kinesin-related protein, Krp1p, in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15850809" target=_blank>Rhee DK, <i>et al.</i> (2005 Jun 3)</a>. ATP-binding motifs play key roles in Krp1p, kinesin-related protein 1, function for bi-polar growth control in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18931302" target=_blank>Dixon SJ, <i>et al.</i> (2008 Oct 28)</a>. Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19686686" target=_blank>Fu C, <i>et al.</i> (2009 Aug)</a>. Phospho-regulated interaction between kinesin-6 Klp9p and microtubule bundler Ase1p promotes spindle elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03034_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03034_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03034_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03034_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG03035<br>EOG81NS34<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, microtubule binding, microtubule motor activity, role in microtubule-based movement and microtubule associated complex localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03035_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03035_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03035_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03035_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG03036<br>EOG81NS34<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, DNA binding, microtubule binding, microtubule motor activity, role in DNA repair, microtubule-based movement and microtubule associated complex localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03036_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03036_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03036_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03036_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG03037<br>EOG81NS34<br><br>sce:SMY1<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Kinesin-like myosin passenger-protein; interacts with Myo2p; controls actin cable structure and dynamics; proposed to be involved in exocytosis<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730668" target=_blank>Beltr√°n C, <i>et al.</i> (1992 Jan 15)</a>. Cloning and mutational analysis of the gene encoding subunit C of yeast vacuolar H(+)-ATPase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1549181" target=_blank>Lillie SH, <i>et al.</i> (1992 Mar 26)</a>. Suppression of a myosin defect by a kinesin-related gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03037_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03037_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03037_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03037_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG03038<br>EOG81NS34<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03038_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03038_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03038_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03038_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01179_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01179.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>