<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG03379.html">HOG03379&emsp;&emsp;OST3/OST6 family</a><br> 
																	HOG03380&emsp;&emsp;<br> 
																	<a href="HOG03381.html">HOG03381&emsp;&emsp;tricalbin family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG03380.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG03380_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG03380_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG03380_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG07326<br>EOG8BNZTP<br><br>sce:PMS1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATP-binding protein required for mismatch repair; required for both mitosis and meiosis; functions as a heterodimer with Mlh1p; binds double- and single-stranded DNA via its N-terminal domain, similar to E. coli MutL<br><br>
			<br><b>PomBase Description</b><br>MutL family mismatch-repair protein Pms1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP binding, ATPase activity, dinucleotide insertion or deletion binding, heteroduplex DNA loop binding, single-stranded DNA binding activity and role in maintenance of DNA repeat elements, meiotic mismatch repair<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676974" target=_blank>Kramer W, <i>et al.</i> (1989 Oct)</a>. Cloning and nucleotide sequence of DNA mismatch repair gene PMS1 from Saccharomyces cerevisiae: homology of PMS1 to procaryotic MutL and HexB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9234704" target=_blank>Pang Q, <i>et al.</i> (1997 Aug)</a>. Functional domains of the Saccharomyces cerevisiae Mlh1p and Pms1p DNA mismatch repair proteins and their relevance to human hereditary nonpolyposis colorectal cancer-associated mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9258673" target=_blank>Sch√§r P, <i>et al.</i> (1997 Aug)</a>. Mismatch repair in Schizosaccharomyces pombe requires the mutL homologous gene pms1: molecular cloning and functional analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9545323" target=_blank>Habraken Y, <i>et al.</i> (1998 Apr 17)</a>. ATP-dependent assembly of a ternary complex consisting of a DNA mismatch and the yeast MSH2-MSH6 and MLH1-PMS1 protein complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871115" target=_blank>Rudolph C, <i>et al.</i> (1998 Dec)</a>. Schizosaccharomyces pombe exo1 is involved in the same mismatch repair pathway as msh2 and pms1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10080187" target=_blank>Fleck O, <i>et al.</i> (1999 Mar)</a>. Involvement of nucleotide-excision repair in msh2 pms1-independent mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10570173" target=_blank>Wang TF, <i>et al.</i> (1999 Nov 23)</a>. Functional specificity of MutL homologs in yeast: evidence for three Mlh1-based heterocomplexes with distinct roles during meiosis in recombination and mismatch correction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10938116" target=_blank>Tran PT, <i>et al.</i> (2000 Sep)</a>. Functional studies on the candidate ATPase domains of Saccharomyces cerevisiae MutLalpha.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333219" target=_blank>Mansour AA, <i>et al.</i> (2001 May)</a>. Control of GT repeat stability in Schizosaccharomyces pombe by mismatch repair factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11800554" target=_blank>Kunz C, <i>et al.</i> (2001 Oct 19)</a>. Role of the DNA repair nucleases Rad13, Rad2 and Uve1 of Schizosaccharomyces pombe in mismatch correction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717305" target=_blank>Hall MC, <i>et al.</i> (2002 Feb 1)</a>. Differential ATP binding and intrinsic ATP hydrolysis by amino-terminal domains of the yeast Mlh1 and Pms1 proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12222686" target=_blank>Drotschmann K, <i>et al.</i> (2002 Jun)</a>. DNA binding properties of the yeast Msh2-Msh6 and Mlh1-Pms1 heterodimers.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11907579" target=_blank>Steinmetz LM, <i>et al.</i> (2002 Mar 21)</a>. Dissecting the architecture of a quantitative trait locus in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12682353" target=_blank>Hall MC, <i>et al.</i> (2003 Apr 15)</a>. DNA binding by yeast Mlh1 and Pms1: implications for DNA mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12713814" target=_blank>Marti TM, <i>et al.</i> (2003 May 13)</a>. Different frameshift mutation spectra in non-repetitive DNA of MutSalpha- and MutLalpha-deficient fission yeast cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16273108" target=_blank>Deutschbauer AM, <i>et al.</i> (2005 Dec)</a>. Quantitative trait loci mapped to single-nucleotide resolution in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16227575" target=_blank>Erdeniz N, <i>et al.</i> (2005 Nov)</a>. Novel PMS1 alleles preferentially affect the repair of primer strand loops during DNA replication.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16492773" target=_blank>Heck JA, <i>et al.</i> (2006 Feb 28)</a>. Negative epistasis between natural variants of the Saccharomyces cerevisiae MLH1 and PMS1 genes results in a defect in mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22806344" target=_blank>Ryuko S, <i>et al.</i> (2012 Aug)</a>. Genome-wide screen reveals novel mechanisms for regulating cobalt uptake and detoxification in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23861937" target=_blank>Zhang L, <i>et al.</i> (2013)</a>. Genome-wide screening for genes associated with valproic acid sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23738021" target=_blank>Zhou X, <i>et al.</i> (2013)</a>. A genome-wide screening of potential target genes to enhance the antifungal activity of micafungin in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24463365" target=_blank>Rallis C, <i>et al.</i> (2014 Feb 15)</a>. Systematic screen for mutants resistant to TORC1 inhibition in fission yeast reveals genes involved in cellular ageing and growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26628015" target=_blank>Zhang X, <i>et al.</i> (2015 Oct 9)</a>. Characterization of Tamoxifen as an Antifungal Agent Using the Yeast Schizosaccharomyces Pombe Model Organism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27255861" target=_blank>Hu L, <i>et al.</i> (2016 Aug)</a>. Genetic evidence for involvement of membrane trafficking in the action of 5-fluorouracil.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07326_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07326_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07326_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG07326_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG07327<br>EOG8BNZTP<br><br>sce:MLH1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein required for mismatch repair in mitosis and meiosis; also required for crossing over during meiosis; forms a complex with Pms1p and Msh2p-Msh3p during mismatch repair; human homolog is associated with hereditary non-polyposis colon cancer<br><br>
			<br><b>PomBase Description</b><br>MutL family protein Mlh1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP binding, ATPase activity, dinucleotide insertion or deletion binding, heteroduplex DNA loop binding, single-stranded DNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8264608" target=_blank>Prolla TA, <i>et al.</i> (1994 Jan)</a>. Dual requirement in yeast DNA mismatch repair for MLH1 and PMS1, two homologs of the bacterial mutL gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9234704" target=_blank>Pang Q, <i>et al.</i> (1997 Aug)</a>. Functional domains of the Saccharomyces cerevisiae Mlh1p and Pms1p DNA mismatch repair proteins and their relevance to human hereditary nonpolyposis colorectal cancer-associated mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9545323" target=_blank>Habraken Y, <i>et al.</i> (1998 Apr 17)</a>. ATP-dependent assembly of a ternary complex consisting of a DNA mismatch and the yeast MSH2-MSH6 and MLH1-PMS1 protein complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10570173" target=_blank>Wang TF, <i>et al.</i> (1999 Nov 23)</a>. Functional specificity of MutL homologs in yeast: evidence for three Mlh1-based heterocomplexes with distinct roles during meiosis in recombination and mismatch correction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10938116" target=_blank>Tran PT, <i>et al.</i> (2000 Sep)</a>. Functional studies on the candidate ATPase domains of Saccharomyces cerevisiae MutLalpha.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11555625" target=_blank>Ellison AR, <i>et al.</i> (2001 Sep 1)</a>. Functional analysis of human MLH1 and MSH2 missense variants and hybrid human-yeast MLH1 proteins in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717305" target=_blank>Hall MC, <i>et al.</i> (2002 Feb 1)</a>. Differential ATP binding and intrinsic ATP hydrolysis by amino-terminal domains of the yeast Mlh1 and Pms1 proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12222686" target=_blank>Drotschmann K, <i>et al.</i> (2002 Jun)</a>. DNA binding properties of the yeast Msh2-Msh6 and Mlh1-Pms1 heterodimers.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12682353" target=_blank>Hall MC, <i>et al.</i> (2003 Apr 15)</a>. DNA binding by yeast Mlh1 and Pms1: implications for DNA mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529393" target=_blank>Argueso JL, <i>et al.</i> (2003 Feb)</a>. Systematic mutagenesis of the Saccharomyces cerevisiae MLH1 gene reveals distinct roles for Mlh1p in meiotic crossing over and in vegetative and meiotic mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998526" target=_blank>Goldman GH, <i>et al.</i> (2004 Apr)</a>. Aspergillus nidulans as a model system to characterize the DNA damage response in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16492773" target=_blank>Heck JA, <i>et al.</i> (2006 Feb 28)</a>. Negative epistasis between natural variants of the Saccharomyces cerevisiae MLH1 and PMS1 genes results in a defect in mismatch repair.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07327_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07327_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07327_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG07327_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG03380.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>