<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00467.html">HOG00467&emsp;&emsp;aconitase/IPM isomerase family</a><br> 
																	HOG00468&emsp;&emsp;<br> 
																	<a href="HOG00469.html">HOG00469&emsp;&emsp;class-IV pyridoxal-phosphate-dependent aminotransferase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00468.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00468_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00468_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00468_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00468_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01084<br>EOG8MSBFR<br><br>sce:LEU2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Beta-isopropylmalate dehydrogenase (IMDH)<br><br>
			<br><b>SGD Description</b><br>Beta-isopropylmalate dehydrogenase (IMDH); catalyzes the third step in the leucine biosynthesis pathway; can additionally catalyze the conversion of &#946;-ethylmalate into &#945;-ketovalerate<br><br>
			<br><b>PomBase Description</b><br>3-isopropylmalate dehydrogenase Leu1<br><br>
			<br><b>AspGD Description</b><br>Putative beta-isopropylmalate dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6330094" target=_blank>Andreadis A, <i>et al.</i> (1984 Jul 10)</a>. Yeast LEU2. Repression of mRNA levels by leucine and primary structure of the gene product.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6396161" target=_blank>Froman BE, <i>et al.</i> (1984 Nov)</a>. Nucleotide sequence of the 3' terminal region of the LEU2 gene from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836077" target=_blank>Davidow LS, <i>et al.</i> (1987)</a>. The Yarrowia lipolytica LEU2 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2453299" target=_blank>Gaillardin C, <i>et al.</i> (1987)</a>. LEU2 directed expression of beta-galactosidase activity and phleomycin resistance in Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1441757" target=_blank>Zhang YP, <i>et al.</i> (1992 Sep)</a>. LEU2 gene homolog in Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9534253" target=_blank>Lu P, <i>et al.</i> (1998 Feb)</a>. Cloning and disruption of the beta-isopropylmalate dehydrogenase gene (LEU2) of Pichia stipitis with URA3 and recovery of the double auxotroph.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11447605" target=_blank>Sychrov√° H, <i>et al.</i> (2001 Jul)</a>. Molecular cloning and sequence analysis of the Zygosaccharomyces rouxiiLEU2 gene encoding a beta-isopropylmalate dehydrogenase.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01084_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01084_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01084_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01084_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00468_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01085<br>EOG8MSBFR<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized LEU3 paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG01084<br><br>
			<br><b>AspGD Description</b><br>Putative beta-isopropylmalate dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20969648" target=_blank>Schinko T, <i>et al.</i> (2010 Nov)</a>. Transcriptome analysis of nitrate assimilation in Aspergillus nidulans reveals connections to nitric oxide metabolism.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01085_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01085_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01085_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01085_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00468_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00468.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>