<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00216.html">HOG00216&emsp;&emsp;aldehyde dehydrogenase family</a><br> 
																	HOG00219&emsp;&emsp;<br> 
																	<a href="HOG00222.html">HOG00222&emsp;&emsp;malic enzymes family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00219.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00219_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00219_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00219_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00219_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00378<br>EOG8GB5N3<br><br>sce:PYC2<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Pyruvate carboxylase isoform. No obvious phenotype is observed when either PYC1 or PYC2 is disrupted singly, but when both genes are disrupted cells are unable to grow with glucose as the sole carbon source unless aspartate is added to the medium instead of ammonia; Aspergillus orthologs have [c], [m] and [m]/[c] localization<br><br>
			<br><b>SGD Description</b><br>Pyruvate carboxylase isoform; cytoplasmic enzyme that converts pyruvate to oxaloacetate; differentially regulated than isoform Pyc1p; mutations in the human homolog are associated with lactic acidosis; PYC2 has a paralog, PYC1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>pyruvate carboxylase Pyr1<br><br>
			<br><b>AspGD Description</b><br>Cytoplasmic pyruvate carboxylase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11946370" target=_blank>Skinner VM, <i>et al.</i> (1972 Jan 15)</a>. Mutants of Aspergillus nidulans lacking pyruvate carboxylase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=776698" target=_blank>Payton M, <i>et al.</i> (1976 Jul 1)</a>. Mutants of Aspergillus nidulans lacking pyruvate kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=778333" target=_blank>Payton M, <i>et al.</i> (1976 May)</a>. Agar as a carbon source and its effect on the utilization of other carbon sources by acetate non-utilizing (acu) mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=320455" target=_blank>Hynes MJ, <i>et al.</i> (1977 Jan 18)</a>. Pleiotropic mutants of Aspergillus nidulans altered in carbon metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7026241" target=_blank>Osmani SA, <i>et al.</i> (1981 Aug)</a>. Pyruvate carboxylase from Aspergillus nidulans. Regulatory properties.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6345155" target=_blank>Osmani SA, <i>et al.</i> (1983 Jul 1)</a>. The sub-cellular localisation of pyruvate carboxylase and of some other enzymes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6421580" target=_blank>Osmani SA, <i>et al.</i> (1984 Mar 15)</a>. Pyruvate carboxylase from Aspergillus nidulans. Effects of regulatory modifiers on the structure of the enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3315006" target=_blank>Dijkema C, <i>et al.</i> (1987 Dec 10)</a>. 13C-NMR analysis of Aspergillus mutants disturbed in pyruvate metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2383004" target=_blank>Bercovitz A, <i>et al.</i> (1990 Jun)</a>. Localization of pyruvate carboxylase in organic acid-producing Aspergillus strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1921979" target=_blank>Stucka R, <i>et al.</i> (1991 Oct)</a>. DNA sequences in chromosomes II and VII code for pyruvate carboxylase isoenzymes in Saccharomyces cerevisiae: analysis of pyruvate carboxylase-deficient strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8554526" target=_blank>Val DL, <i>et al.</i> (1995 Dec 15)</a>. Polymorphism of the yeast pyruvate carboxylase 2 gene and protein: effects on protein biotinylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26152728" target=_blank>Mathiassen SG, <i>et al.</i> (2015 Aug 21)</a>. A Two-step Protein Quality Control Pathway for a Misfolded DJ-1 Variant in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26644575" target=_blank>Halim A, <i>et al.</i> (2015 Dec 22)</a>. Discovery of a nucleocytoplasmic O-mannose glycoproteome in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00378_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00378_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00378_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00378_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00219_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00379<br>EOG8GB5N3<br><br>sce:PYC1<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Pyruvate carboxylase isoform. No obvious phenotype is observed when either PYC1 or PYC2 is disrupted singly, but when both genes are disrupted cells are unable to grow with glucose as the sole carbon source unless aspartate is added to the medium instead of ammonia; Aspergillus orthologs have [c], [m] and [m]/[c] localization<br><br>
			<br><b>Parent</b><br>ohnolog:FOG00378<br><br>
			<br><b>SGD Description</b><br>Pyruvate carboxylase isoform; cytoplasmic enzyme that converts pyruvate to oxaloacetate; differentially regulated than isoform Pyc2p; mutations in the human homolog are associated with lactic acidosis; PYC1 has a paralog, PYC2, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3036126" target=_blank>Morris CP, <i>et al.</i> (1987 May 29)</a>. Yeast pyruvate carboxylase: gene isolation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042770" target=_blank>Lim F, <i>et al.</i> (1988 Aug 15)</a>. Sequence and domain structure of yeast pyruvate carboxylase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2383004" target=_blank>Bercovitz A, <i>et al.</i> (1990 Jun)</a>. Localization of pyruvate carboxylase in organic acid-producing Aspergillus strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9234674" target=_blank>Feuermann M, <i>et al.</i> (1997 Jul)</a>. The characterization of two new clusters of duplicated genes suggests a 'Lego' organization of the yeast Saccharomyces cerevisiae chromosomes.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00379_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00379_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00379_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00379_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00219_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00219.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>