<html>
<head><title>HOG01359</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01358.html">HOG01358&emsp;&emsp;</a><br> 
																	HOG01359&emsp;&emsp;<br> 
																	<a href="HOG01360.html">HOG01360&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01359.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01359_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01359_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01359_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG01359.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03918<br>EOG8N5TBS<br><br>sce:MSH3<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mismatch repair protein; forms dimers with Msh2p that mediate repair of insertion or deletion mutations and removal of nonhomologous DNA ends, contains a PCNA (Pol30p) binding motif required for genome stability<br><br>
			<br><b>PomBase Description</b><br>MutS protein homolog 3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in gene conversion at mating-type locus, termination of copy-synthesis, maintenance of DNA repeat elements, reciprocal meiotic recombination and nucleus, site of double-strand break localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8510668" target=_blank>New L, <i>et al.</i> (1993 May)</a>. The yeast gene MSH3 defines a new class of eukaryotic MutS homologues.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8600025" target=_blank>Marsischky GT, <i>et al.</i> (1996 Feb 15)</a>. Redundancy of Saccharomyces cerevisiae MSH3 and MSH6 in MSH2-dependent mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8910404" target=_blank>Johnson RE, <i>et al.</i> (1996 Nov 8)</a>. Evidence for involvement of yeast proliferating cell nuclear antigen in DNA mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8805366" target=_blank>Habraken Y, <i>et al.</i> (1996 Sep 1)</a>. Binding of insertion/deletion DNA mismatches by the heterodimer of yeast mismatch repair proteins MSH2 and MSH3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256462" target=_blank>Sugawara N, <i>et al.</i> (1997 Aug 19)</a>. Role of Saccharomyces cerevisiae Msh2 and Msh3 repair proteins in double-strand break-induced recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9111357" target=_blank>Sia EA, <i>et al.</i> (1997 May)</a>. Microsatellite instability in yeast: dependence on repeat unit size and DNA mismatch repair genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9368761" target=_blank>Habraken Y, <i>et al.</i> (1997 Oct 1)</a>. Enhancement of MSH2-MSH3-mediated mismatch recognition by the yeast MLH1-PMS1 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770499" target=_blank>Flores-Rozas H, <i>et al.</i> (1998 Oct 13)</a>. The Saccharomyces cerevisiae MLH3 gene functions in MSH3-dependent suppression of frameshift mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10523644" target=_blank>Studamire B, <i>et al.</i> (1999 Nov)</a>. Separation-of-function mutations in Saccharomyces cerevisiae MSH2 that confer mismatch repair defects but do not affect nonhomologous-tail removal during recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11005803" target=_blank>Clark AB, <i>et al.</i> (2000 Nov 24)</a>. Functional interaction of proliferating cell nuclear antigen with MSH2-MSH6 and MSH2-MSH3 complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16702432" target=_blank>Stone JE, <i>et al.</i> (2006 Jul)</a>. Analysis of the proteins involved in the in vivo repair of base-base mismatches and four-base loops formed during meiotic recombination in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16781730" target=_blank>Surtees JA, <i>et al.</i> (2006 Jul 14)</a>. Mismatch repair factor MSH2-MSH3 binds and alters the conformation of branched DNA structures predicted to form during genetic recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17157869" target=_blank>Lee SD, <i>et al.</i> (2007 Feb 9)</a>. Saccharomyces cerevisiae MSH2-MSH3 and MSH2-MSH6 complexes display distinct requirements for DNA binding domain I in mismatch recognition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17573527" target=_blank>Shell SS, <i>et al.</i> (2007 Jun 26)</a>. Chimeric Saccharomyces cerevisiae Msh6 protein with an Msh3 mispair-binding domain combines properties of both proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17636021" target=_blank>Harrington JM, <i>et al.</i> (2007 Sep)</a>. Saccharomyces cerevisiae Msh2-Msh3 acts in repair of base-base mispairs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18471978" target=_blank>Li F, <i>et al.</i> (2008 May 9)</a>. Microarray-based genetic screen defines SAW1, a gene required for Rad1/Rad10-dependent processing of recombination intermediates.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03918_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03918_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03918_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03918_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01359_phobius.pdf">Phobius transmembrane predictions</a>
										<br>9 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03919<br>EOG8N5TBS<br><br>sce:MSH2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein that binds to DNA mismatches; forms heterodimers with Msh3p and Msh6p that bind to DNA mismatches to initiate the mismatch repair process; contains a Walker ATP-binding motif required for repair activity and involved in interstrand cross-link repair; Msh2p-Msh6p binds to and hydrolyzes ATP<br><br>
			<br><b>PomBase Description</b><br>MutS protein homolog 2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP binding, ATPase activity, Y-form DNA binding, double-strand/single-strand DNA junction binding, four-way junction DNA binding and guanine/thymine mispair binding, more<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1334021" target=_blank>Reenan RA, <i>et al.</i> (1992 Dec)</a>. Characterization of insertion mutations in the Saccharomyces cerevisiae MSH1 and MSH2 genes: evidence for separate mitochondrial and nuclear functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1459447" target=_blank>Reenan RA, <i>et al.</i> (1992 Dec)</a>. Isolation and characterization of two Saccharomyces cerevisiae genes encoding homologs of the bacterial HexA and MutS mismatch repair proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8066446" target=_blank>Prolla TA, <i>et al.</i> (1994 Aug 19)</a>. MLH1, PMS1, and MSH2 interactions during the initiation of DNA mismatch repair in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8600025" target=_blank>Marsischky GT, <i>et al.</i> (1996 Feb 15)</a>. Redundancy of Saccharomyces cerevisiae MSH3 and MSH6 in MSH2-dependent mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8816473" target=_blank>Alani E, <i>et al.</i> (1996 Oct)</a>. The Saccharomyces cerevisiae Msh2 and Msh6 proteins form a complex that specifically binds to duplex oligonucleotides containing mismatched DNA base pairs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8858149" target=_blank>Umar A, <i>et al.</i> (1996 Oct 4)</a>. Requirement for PCNA in DNA mismatch repair at a step preceding DNA resynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8805366" target=_blank>Habraken Y, <i>et al.</i> (1996 Sep 1)</a>. Binding of insertion/deletion DNA mismatches by the heterodimer of yeast mismatch repair proteins MSH2 and MSH3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256462" target=_blank>Sugawara N, <i>et al.</i> (1997 Aug 19)</a>. Role of Saccharomyces cerevisiae Msh2 and Msh3 repair proteins in double-strand break-induced recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9111357" target=_blank>Sia EA, <i>et al.</i> (1997 May)</a>. Microsatellite instability in yeast: dependence on repeat unit size and DNA mismatch repair genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9368761" target=_blank>Habraken Y, <i>et al.</i> (1997 Oct 1)</a>. Enhancement of MSH2-MSH3-mediated mismatch recognition by the yeast MLH1-PMS1 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9545323" target=_blank>Habraken Y, <i>et al.</i> (1998 Apr 17)</a>. ATP-dependent assembly of a ternary complex consisting of a DNA mismatch and the yeast MSH2-MSH6 and MLH1-PMS1 protein complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819445" target=_blank>Studamire B, <i>et al.</i> (1998 Dec)</a>. Saccharomyces cerevisiae Msh2p and Msh6p ATPase activities are both required during mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770499" target=_blank>Flores-Rozas H, <i>et al.</i> (1998 Oct 13)</a>. The Saccharomyces cerevisiae MLH3 gene functions in MSH3-dependent suppression of frameshift mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469597" target=_blank>Drotschmann K, <i>et al.</i> (1999 Aug 26)</a>. Mutator phenotypes of common polymorphisms and missense mutations in MSH2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10066781" target=_blank>Marsischky GT, <i>et al.</i> (1999 Mar 12)</a>. 'Saccharomyces cerevisiae MSH2/6 complex interacts with Holliday junctions and facilitates their cleavage by phage resolution enzymes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10077621" target=_blank>Drotschmann K, <i>et al.</i> (1999 Mar 16)</a>. Mutator phenotypes of yeast strains heterozygous for mutations in the MSH2 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10523644" target=_blank>Studamire B, <i>et al.</i> (1999 Nov)</a>. Separation-of-function mutations in Saccharomyces cerevisiae MSH2 that confer mismatch repair defects but do not affect nonhomologous-tail removal during recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10518225" target=_blank>Ni TT, <i>et al.</i> (1999 Sep)</a>. MSH2 and MSH6 are required for removal of adenine misincorporated opposite 8-oxo-guanine in S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10480869" target=_blank>Marsischky GT, <i>et al.</i> (1999 Sep 17)</a>. Biochemical characterization of the interaction between the Saccharomyces cerevisiae MSH2-MSH6 complex and mispaired bases in DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11005803" target=_blank>Clark AB, <i>et al.</i> (2000 Nov 24)</a>. Functional interaction of proliferating cell nuclear antigen with MSH2-MSH6 and MSH2-MSH3 complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11641390" target=_blank>Drotschmann K, <i>et al.</i> (2001 Dec 7)</a>. Asymmetric recognition of DNA local distortion. Structure-based functional studies of eukaryotic Msh2-Msh6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11237611" target=_blank>Bowers J, <i>et al.</i> (2001 Mar 9)</a>. MSH-MLH complexes formed at a DNA mismatch are disrupted by the PCNA sliding clamp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11555625" target=_blank>Ellison AR, <i>et al.</i> (2001 Sep 1)</a>. Functional analysis of human MLH1 and MSH2 missense variants and hybrid human-yeast MLH1 proteins in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12509278" target=_blank>Drotschmann K, <i>et al.</i> (2002 Sep 4)</a>. Evidence for sequential action of two ATPase active sites in yeast Msh2-Msh6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12875840" target=_blank>Kijas AW, <i>et al.</i> (2003 Aug 1)</a>. Msh2 separation of function mutations confer defects in the initiation steps of mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12435741" target=_blank>Lau PJ, <i>et al.</i> (2003 Jan 3)</a>. Transfer of the MSH2.MSH6 complex from proliferating cell nuclear antigen to mispaired bases in DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12820877" target=_blank>Antony E, <i>et al.</i> (2003 Jul 1)</a>. Mismatch recognition-coupled stabilization of Msh2-Msh6 in an ATP-bound state at the initiation of DNA repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15513922" target=_blank>Clark AB, <i>et al.</i> (2004 Dec 24)</a>. Cadmium inhibits the functions of eukaryotic MutS complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15746000" target=_blank>Banerjee S, <i>et al.</i> (2005)</a>. Cadmium inhibits mismatch repair by blocking the ATPase activity of the MSH2-MSH6 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16337600" target=_blank>Jiang J, <i>et al.</i> (2005 Dec 9)</a>. Detection of high-affinity and sliding clamp modes for MSH2-MSH6 by single-molecule unzipping force analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15811858" target=_blank>Mendillo ML, <i>et al.</i> (2005 Jun 10)</a>. Analysis of the interaction between the Saccharomyces cerevisiae MSH2-MSH6 and MLH1-PMS1 complexes with DNA using a reversible DNA end-blocking system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600868" target=_blank>Mazur DJ, <i>et al.</i> (2006 Apr 7)</a>. Inhibition of Msh6 ATPase activity by mispaired DNA induces a Msh2(ATP)-Msh6(ATP) state capable of hydrolysis-independent movement along DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16214425" target=_blank>Antony E, <i>et al.</i> (2006 Feb 3)</a>. Contribution of Msh2 and Msh6 subunits to the asymmetric ATPase and DNA mismatch binding activities of Saccharomyces cerevisiae Msh2-Msh6 mismatch repair protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16702432" target=_blank>Stone JE, <i>et al.</i> (2006 Jul)</a>. Analysis of the proteins involved in the in vivo repair of base-base mismatches and four-base loops formed during meiotic recombination in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16781730" target=_blank>Surtees JA, <i>et al.</i> (2006 Jul 14)</a>. Mismatch repair factor MSH2-MSH3 binds and alters the conformation of branched DNA structures predicted to form during genetic recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17157869" target=_blank>Lee SD, <i>et al.</i> (2007 Feb 9)</a>. Saccharomyces cerevisiae MSH2-MSH3 and MSH2-MSH6 complexes display distinct requirements for DNA binding domain I in mismatch recognition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17636021" target=_blank>Harrington JM, <i>et al.</i> (2007 Sep)</a>. Saccharomyces cerevisiae Msh2-Msh3 acts in repair of base-base mispairs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18471978" target=_blank>Li F, <i>et al.</i> (2008 May 9)</a>. Microarray-based genetic screen defines SAW1, a gene required for Rad1/Rad10-dependent processing of recombination intermediates.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03919_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03919_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03919_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03919_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01359_phobius.pdf">Phobius transmembrane predictions</a>
										<br>8 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03920<br>EOG8N5TBS<br><br>sce:MSH6<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein required for mismatch repair in mitosis and meiosis; forms a complex with Msh2p to repair both single-base & insertion-deletion mispairs; also involved in interstrand cross-link repair; potentially phosphorylated by Cdc28p<br><br>
			<br><b>PomBase Description</b><br>MutS protein homolog<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in maintenance of DNA repeat elements, mismatch repair and cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8723353" target=_blank>Iaccarino I, <i>et al.</i> (1996 Apr 1)</a>. MSH6, a Saccharomyces cerevisiae protein that binds to mismatches as a heterodimer with MSH2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8600025" target=_blank>Marsischky GT, <i>et al.</i> (1996 Feb 15)</a>. Redundancy of Saccharomyces cerevisiae MSH3 and MSH6 in MSH2-dependent mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8816473" target=_blank>Alani E, <i>et al.</i> (1996 Oct)</a>. The Saccharomyces cerevisiae Msh2 and Msh6 proteins form a complex that specifically binds to duplex oligonucleotides containing mismatched DNA base pairs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9111357" target=_blank>Sia EA, <i>et al.</i> (1997 May)</a>. Microsatellite instability in yeast: dependence on repeat unit size and DNA mismatch repair genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9545323" target=_blank>Habraken Y, <i>et al.</i> (1998 Apr 17)</a>. ATP-dependent assembly of a ternary complex consisting of a DNA mismatch and the yeast MSH2-MSH6 and MLH1-PMS1 protein complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819445" target=_blank>Studamire B, <i>et al.</i> (1998 Dec)</a>. Saccharomyces cerevisiae Msh2p and Msh6p ATPase activities are both required during mismatch repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770499" target=_blank>Flores-Rozas H, <i>et al.</i> (1998 Oct 13)</a>. The Saccharomyces cerevisiae MLH3 gene functions in MSH3-dependent suppression of frameshift mutations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347163" target=_blank>Bowers J, <i>et al.</i> (1999 Jun 4)</a>. A mutation in the MSH6 subunit of the Saccharomyces cerevisiae MSH2-MSH6 complex disrupts mismatch recognition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10066781" target=_blank>Marsischky GT, <i>et al.</i> (1999 Mar 12)</a>. 'Saccharomyces cerevisiae MSH2/6 complex interacts with Holliday junctions and facilitates their cleavage by phage resolution enzymes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537275" target=_blank>Kolodner RD, <i>et al.</i> (1999 Oct 15)</a>. Germ-line msh6 mutations in colorectal cancer families.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10518225" target=_blank>Ni TT, <i>et al.</i> (1999 Sep)</a>. MSH2 and MSH6 are required for removal of adenine misincorporated opposite 8-oxo-guanine in S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10480869" target=_blank>Marsischky GT, <i>et al.</i> (1999 Sep 17)</a>. Biochemical characterization of the interaction between the Saccharomyces cerevisiae MSH2-MSH6 complex and mispaired bases in DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10615127" target=_blank>Das Gupta R, <i>et al.</i> (2000 Jan)</a>. Novel dominant mutations in Saccharomyces cerevisiae MSH6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11062484" target=_blank>Flores-Rozas H, <i>et al.</i> (2000 Nov)</a>. Proliferating cell nuclear antigen and Msh2p-Msh6p interact to form an active mispair recognition complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11005803" target=_blank>Clark AB, <i>et al.</i> (2000 Nov 24)</a>. Functional interaction of proliferating cell nuclear antigen with MSH2-MSH6 and MSH2-MSH3 complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10970737" target=_blank>Bowers J, <i>et al.</i> (2000 Sep 15)</a>. Analysis of yeast MSH2-MSH6 suggests that the initiation of mismatch repair can be separated into discrete steps.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11641390" target=_blank>Drotschmann K, <i>et al.</i> (2001 Dec 7)</a>. Asymmetric recognition of DNA local distortion. Structure-based functional studies of eukaryotic Msh2-Msh6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11237611" target=_blank>Bowers J, <i>et al.</i> (2001 Mar 9)</a>. MSH-MLH complexes formed at a DNA mismatch are disrupted by the PCNA sliding clamp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333219" target=_blank>Mansour AA, <i>et al.</i> (2001 May)</a>. Control of GT repeat stability in Schizosaccharomyces pombe by mismatch repair factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333218" target=_blank>Tornier C, <i>et al.</i> (2001 May)</a>. Requirement for Msh6, but not for Swi4 (Msh3), in Msh2-dependent repair of base-base mismatches and mononucleotide loops in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11986324" target=_blank>Hess MT, <i>et al.</i> (2002 Jul 12)</a>. Dominant Saccharomyces cerevisiae msh6 mutations cause increased mispair binding and decreased dissociation from mispairs by Msh2-Msh6 in the presence of ATP.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12509278" target=_blank>Drotschmann K, <i>et al.</i> (2002 Sep 4)</a>. Evidence for sequential action of two ATPase active sites in yeast Msh2-Msh6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12435741" target=_blank>Lau PJ, <i>et al.</i> (2003 Jan 3)</a>. Transfer of the MSH2.MSH6 complex from proliferating cell nuclear antigen to mispaired bases in DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12820877" target=_blank>Antony E, <i>et al.</i> (2003 Jul 1)</a>. Mismatch recognition-coupled stabilization of Msh2-Msh6 in an ATP-bound state at the initiation of DNA repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15513922" target=_blank>Clark AB, <i>et al.</i> (2004 Dec 24)</a>. Cadmium inhibits the functions of eukaryotic MutS complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15746000" target=_blank>Banerjee S, <i>et al.</i> (2005)</a>. Cadmium inhibits mismatch repair by blocking the ATPase activity of the MSH2-MSH6 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16337600" target=_blank>Jiang J, <i>et al.</i> (2005 Dec 9)</a>. Detection of high-affinity and sliding clamp modes for MSH2-MSH6 by single-molecule unzipping force analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15811858" target=_blank>Mendillo ML, <i>et al.</i> (2005 Jun 10)</a>. Analysis of the interaction between the Saccharomyces cerevisiae MSH2-MSH6 and MLH1-PMS1 complexes with DNA using a reversible DNA end-blocking system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600868" target=_blank>Mazur DJ, <i>et al.</i> (2006 Apr 7)</a>. Inhibition of Msh6 ATPase activity by mispaired DNA induces a Msh2(ATP)-Msh6(ATP) state capable of hydrolysis-independent movement along DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16214425" target=_blank>Antony E, <i>et al.</i> (2006 Feb 3)</a>. Contribution of Msh2 and Msh6 subunits to the asymmetric ATPase and DNA mismatch binding activities of Saccharomyces cerevisiae Msh2-Msh6 mismatch repair protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16407100" target=_blank>Hess MT, <i>et al.</i> (2006 Jan 17)</a>. Biochemical basis for dominant mutations in the Saccharomyces cerevisiae MSH6 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16702432" target=_blank>Stone JE, <i>et al.</i> (2006 Jul)</a>. Analysis of the proteins involved in the in vivo repair of base-base mismatches and four-base loops formed during meiotic recombination in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17567610" target=_blank>Clark AB, <i>et al.</i> (2007)</a>. Multiple functions for the N-terminal region of Msh6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17157869" target=_blank>Lee SD, <i>et al.</i> (2007 Feb 9)</a>. Saccharomyces cerevisiae MSH2-MSH3 and MSH2-MSH6 complexes display distinct requirements for DNA binding domain I in mismatch recognition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17573527" target=_blank>Shell SS, <i>et al.</i> (2007 Jun 26)</a>. Chimeric Saccharomyces cerevisiae Msh6 protein with an Msh3 mispair-binding domain combines properties of both proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17141577" target=_blank>Holmes SF, <i>et al.</i> (2007 Mar 1)</a>. Specialized mismatch repair function of Glu339 in the Phe-X-Glu motif of yeast Msh6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18931302" target=_blank>Dixon SJ, <i>et al.</i> (2008 Oct 28)</a>. Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22806344" target=_blank>Ryuko S, <i>et al.</i> (2012 Aug)</a>. Genome-wide screen reveals novel mechanisms for regulating cobalt uptake and detoxification in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22308326" target=_blank>Zamir L, <i>et al.</i> (2012 Feb 14)</a>. Tight coevolution of proliferating cell nuclear antigen (PCNA)-partner interaction networks in fungi leads to interspecies network incompatibility.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23050226" target=_blank>Chen Z, <i>et al.</i> (2012 Oct)</a>. A genetic screen to discover pathways affecting cohesin function in Schizosaccharomyces pombe identifies chromatin effectors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23779158" target=_blank>Saito Y, <i>et al.</i> (2013)</a>. The proteasome factor Bag101 binds to Rad22 and suppresses homologous recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27611590" target=_blank>McDonald KR, <i>et al.</i> (2016 Sep)</a>. Pfh1 Is an Accessory Replicative Helicase that Interacts with the Replisome to Facilitate Fork Progression and Preserve Genome Integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03920_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03920_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03920_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03920_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01359_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03921<br>EOG8N5TBS<br><br>sce:MSH1<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>DNA-binding protein of the mitochondria; involved in repair of mitochondrial DNA; has ATPase activity and binds to DNA mismatches; has homology to E. coli MutS; transcription is induced during meiosis<br><br>
			<br><b>PomBase Description</b><br>mitochondrial MutS protein Msh1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have mitochondrion localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1334021" target=_blank>Reenan RA, <i>et al.</i> (1992 Dec)</a>. Characterization of insertion mutations in the Saccharomyces cerevisiae MSH1 and MSH2 genes: evidence for separate mitochondrial and nuclear functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1459447" target=_blank>Reenan RA, <i>et al.</i> (1992 Dec)</a>. Isolation and characterization of two Saccharomyces cerevisiae genes encoding homologs of the bacterial HexA and MutS mismatch repair proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961998" target=_blank>Chi NW, <i>et al.</i> (1994 Nov 25)</a>. Purification and characterization of MSH1, a yeast mitochondrial protein that binds to DNA mismatches.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961999" target=_blank>Chi NW, <i>et al.</i> (1994 Nov 25)</a>. The effect of DNA mismatches on the ATPase activity of MSH1, a protein in yeast mitochondria that recognizes DNA mismatches.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9726974" target=_blank>Vanderstraeten S, <i>et al.</i> (1998 Sep 11)</a>. The role of 3'-5' exonucleolytic proofreading and mismatch repair in yeast mitochondrial DNA error avoidance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11862493" target=_blank>Koprowski P, <i>et al.</i> (2002 Feb)</a>. A dominant mitochondrial mutator phenotype of Saccharomyces cerevisiae conferred by msh1 alleles altered in the sequence encoding the ATP-binding domain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15010316" target=_blank>Dzierzbicki P, <i>et al.</i> (2004 Apr 1)</a>. Repair of oxidative damage in mitochondrial DNA of Saccharomyces cerevisiae: involvement of the MSH1-dependent pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15611870" target=_blank>Mookerjee SA, <i>et al.</i> (2005 Feb)</a>. Analysis of the functional domains of the mismatch repair homologue Msh1p and its role in mitochondrial genome maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16337661" target=_blank>Mookerjee SA, <i>et al.</i> (2006 Mar 20)</a>. Overlapping contributions of Msh1p and putative recombination proteins Cce1p, Din7p, and Mhr1p in large-scale recombination and genome sorting events in the mitochondrial genome of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028377" target=_blank>Choi ES, <i>et al.</i> (2012 Sep)</a>. Factors that promote H3 chromatin integrity during transcription prevent promiscuous deposition of CENP-A(Cnp1) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27611590" target=_blank>McDonald KR, <i>et al.</i> (2016 Sep)</a>. Pfh1 Is an Accessory Replicative Helicase that Interacts with the Replisome to Facilitate Fork Progression and Preserve Genome Integrity.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03921_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03921_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03921_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03921_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01359_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01359.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>