<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00233.html">HOG00233&emsp;&emsp;acireductone dioxygenase (ARD) family; short-chain dehydrogenases/reductases (SDR) family</a><br> 
																	HOG00234&emsp;&emsp;<br> 
																	<a href="HOG00235.html">HOG00235&emsp;&emsp;alcohol acetyltransferase</a><br><br> 
																	<a href="../sequences/hog/HOG00234.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG00234_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG00234_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG00234_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG00234_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00443<br>EOG81RN97<br>EOG86Q583<br>EOG8HDR9F<br><br>sce:ATF2<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol acetyltransferase<br><br>
			<br><b>SGD Description</b><br>Alcohol acetyltransferase; may play a role in steroid detoxification; forms volatile esters during fermentation, which is important for brewing and winemaking<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00443_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00443_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00443_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00443_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00443">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00442<br>EOG86Q583<br><br>sce:ATF1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alcohol acetyltransferase<br><br>
			<br><b>Parent</b><br>ohnolog:FOG00443<br><br>
			<br><b>SGD Description</b><br>Alcohol acetyltransferase; responsible for the major part of volatile acetate ester production during fermentation; main enzyme involved in terpenyl acetate synthesis; potential roles in lipid and sterol metabolism<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7764365" target=_blank>Minetoki T, <i>et al.</i> (1993 Dec)</a>. The purification, properties and internal peptide sequences of alcohol acetyltransferase isolated from Saccharomyces cerevisiae Kyokai No. 7.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8085822" target=_blank>Fujii T, <i>et al.</i> (1994 Aug)</a>. Molecular cloning, sequence analysis, and expression of the yeast alcohol acetyltransferase gene.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00442_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00442_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00442_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00442_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00442">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00444<br>EOG8HDR9F<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Suggested Analysis</b><br>consolidate<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00444_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00444_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00444_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00444_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00444">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00445<br>EOG81RN97<br>EOG8W6MBB<br><br>sce:CDC4<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>F-box protein required for both the G1/S and G2/M phase transitions; modular substrate specificity factor which associates with core SCF (Cdc53p, Skp1p and Hrt1p/Rbx1p) to form the SCFCdc4 complex; SCFCdc4 acts as a ubiquitin-protein ligase directing ubiquitination of cyclin-dependent kinase (CDK) phosphorylated substrates, such as: Sic1p, Far1p, Cdc6p, Clb6p, and Cln3p<br><br>
			<br><b>PomBase Description</b><br>F-box/WD repeat protein Pop2|cullin 1 adaptor protein Pop1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have protein binding, bridging, ubiquitin-protein transferase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3309335" target=_blank>Yochem J, <i>et al.</i> (1987 May 20)</a>. Structural comparison of the yeast cell division cycle gene CDC4 and a related pseudogene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7813440" target=_blank>Kornitzer D, <i>et al.</i> (1994 Dec 15)</a>. Regulated degradation of the transcription factor Gcn4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8706131" target=_blank>Bai C, <i>et al.</i> (1996 Jul 26)</a>. SKP1 connects cell cycle regulators to the ubiquitin proteolysis machinery through a novel motif, the F-box.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312054" target=_blank>Drury LS, <i>et al.</i> (1997 Oct 1)</a>. The Cdc4/34/53 pathway targets Cdc6p for proteolysis in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9346239" target=_blank>Feldman RM, <i>et al.</i> (1997 Oct 17)</a>. A complex of Cdc4p, Skp1p, and Cdc53p/cullin catalyzes ubiquitination of the phosphorylated CDK inhibitor Sic1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9346238" target=_blank>Skowyra D, <i>et al.</i> (1997 Oct 17)</a>. F-box proteins are receptors that recruit phosphorylated substrates to the SCF ubiquitin-ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312022" target=_blank>Li FN, <i>et al.</i> (1997 Sep 15)</a>. Grr1 of Saccharomyces cerevisiae is connected to the ubiquitin proteolysis machinery through Skp1: coupling glucose sensing to gene expression and the cell cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9499404" target=_blank>Patton EE, <i>et al.</i> (1998 Mar 1)</a>. Cdc53 is a scaffold protein for multiple Cdc34/Skp1/F-box proteincomplexes that regulate cell division and methionine biosynthesis in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9736614" target=_blank>Jaquenoud M, <i>et al.</i> (1998 Sep 15)</a>. The Cdc42p effector Gic2p is targeted for ubiquitin-dependent degradation by the SCFGrr1 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10213692" target=_blank>Skowyra D, <i>et al.</i> (1999 Apr 23)</a>. Reconstitution of G1 cyclin ubiquitination with complexes containing SCFGrr1 and Rbx1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10409741" target=_blank>Goh PY, <i>et al.</i> (1999 Aug)</a>. Cdc4, a protein required for the onset of S phase, serves an essential function during G(2)/M transition in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10385629" target=_blank>Seol JH, <i>et al.</i> (1999 Jun 15)</a>. Cdc53/cullin and the essential Hrt1 RING-H2 subunit of SCF define a ubiquitin ligase module that activates the E2 enzyme Cdc34.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11080155" target=_blank>Blondel M, <i>et al.</i> (2000 Nov 15)</a>. Nuclear-specific degradation of Far1 is controlled by the localization of the F-box protein Cdc4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11500370" target=_blank>Jäger S, <i>et al.</i> (2001 Aug 15)</a>. Cic1, an adaptor protein specifically linking the 26S proteasome to its substrate, the SCF component Cdc4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14747994" target=_blank>Kus BM, <i>et al.</i> (2004 Feb 15)</a>. Functional interaction of 13 yeast SCF complexes with a set of yeast E2 enzymes in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17434132" target=_blank>Hao B, <i>et al.</i> (2007 Apr 13)</a>. Structure of a Fbw7-Skp1-cyclin E complex: multisite-phosphorylated substrate recognition by SCF ubiquitin ligases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22142781" target=_blank>Colabardini AC, <i>et al.</i> (2012 Feb)</a>. Molecular characterization of the Aspergillus nidulans fbxA encoding an F-box protein involved in xylanase induction.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00445_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00445_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00445_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00445_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00445">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00446<br>EOG81RN97<br><br>sce:PFS2<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Integral subunit of the pre-mRNA CPF complex; the cleavage and polyadenylation factor (CPF) complex plays an essential role in mRNA 3'-end formation by bridging different processing factors and thereby promoting the assembly of the processing complex<br><br>
			<br><b>PomBase Description</b><br>WD repeat protein Pfs2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in mRNA 3'-end processing and cytosol, mRNA cleavage and polyadenylation specificity factor complex, nuclear chromatin localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10619842" target=_blank>Ohnacker M, <i>et al.</i> (2000 Jan 4)</a>. The WD-repeat protein pfs2p bridges two essential factors within the yeast pre-mRNA 3'-end-processing complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12819204" target=_blank>Nedea E, <i>et al.</i> (2003 Aug 29)</a>. Organization and function of APT, a subcomplex of the yeast cleavage and polyadenylation factor involved in the formation of mRNA and small nucleolar RNA 3'-ends.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00446_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00446_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00446_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00446_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00446">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00447<br>EOG81RN97<br><br>sce:PRP4<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Splicing factor; component of the U4/U6-U5 snRNP complex<br><br>
			<br><b>PomBase Description</b><br>U4/U6 x U5 tri-snRNP complex subunit Prp4 family, Rna4<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have U4/U6 x U5 tri-snRNP complex, cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2528687" target=_blank>Banroques J, <i>et al.</i> (1989 Sep)</a>. PRP4: a protein of the yeast U4/U6 small nuclear ribonucleoprotein particle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2528686" target=_blank>Bjørn SP, <i>et al.</i> (1989 Sep)</a>. PRP4 (RNA4) from Saccharomyces cerevisiae: its gene product is associated with the U4/U6 small nuclear ribonucleoprotein particle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10449419" target=_blank>Gottschalk A, <i>et al.</i> (1999 Aug 16)</a>. Identification by mass spectrometry and functional analysis of novel proteins of the yeast [U4/U6.U5] tri-snRNP.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755920" target=_blank>Carnahan RH, <i>et al.</i> (2005 Mar)</a>. Dim1p is required for efficient splicing and export of mRNA encoding lid1p, a component of the fission yeast anaphase-promoting complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00447_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00447_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00447_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00447_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00447">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00448<br>EOG81RN97<br><br>sce:TAF5<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit (90 kDa) of TFIID and SAGA complexes; involved in RNA polymerase II transcription initiation and in chromatin modification<br><br>
			<br><b>PomBase Description</b><br>SAGA complex subunit/TATA-binding protein associated factor/transcription factor TFIID complex subunit Taf5<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have SAGA complex localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7935765" target=_blank>Reese JC, <i>et al.</i> (1994 Oct 6)</a>. Yeast TAFIIS in a multisubunit complex required for activated transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7667272" target=_blank>Poon D, <i>et al.</i> (1995 Aug 29)</a>. Identification and characterization of a TFIID-like multiprotein complex from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674426" target=_blank>Grant PA, <i>et al.</i> (1998 Jul 10)</a>. A subset of TAF(II)s are integral components of the SAGA complex required for nucleosome acetylation and transcriptional stimulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10026213" target=_blank>Grant PA, <i>et al.</i> (1999 Feb 26)</a>. Expanded lysine acetylation specificity of Gcn5 in native complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10788514" target=_blank>Sanders SL, <i>et al.</i> (2000 May 5)</a>. Identification of two novel TAF subunits of the yeast Saccharomyces cerevisiae TFIID complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12138208" target=_blank>Sanders SL, <i>et al.</i> (2002 Aug)</a>. Molecular characterization of Saccharomyces cerevisiae TFIID.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12516863" target=_blank>Martinez E, <i>et al.</i> (2002 Dec)</a>. Multi-protein complexes in eukaryotic gene transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446794" target=_blank>Pray-Grant MG, <i>et al.</i> (2002 Dec)</a>. The novel SLIK histone acetyltransferase complex functions in the yeast retrograde response pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12052880" target=_blank>Sanders SL, <i>et al.</i> (2002 Jul)</a>. Proteomics of the eukaryotic transcription machinery: identification of proteins associated with components of yeast TFIID by multidimensional mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093743" target=_blank>Leurent C, <i>et al.</i> (2002 Jul 1)</a>. Mapping histone fold TAFs within yeast TFIID.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12186975" target=_blank>Sterner DE, <i>et al.</i> (2002 Sep 3)</a>. SALSA, a variant of yeast SAGA, contains truncated Spt7, which correlates with activated transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15260971" target=_blank>Wu PY, <i>et al.</i> (2004 Jul 23)</a>. Molecular architecture of the S. cerevisiae SAGA complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15647753" target=_blank>Pray-Grant MG, <i>et al.</i> (2005 Jan 27)</a>. Chd1 chromodomain links histone H3 methylation with SAGA- and SLIK-dependent acetylation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00448_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00448_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00448_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00448_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00448">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG00449<br>EOG83TXBC<br>EOG8W6MBB<br><br>sce:TUP1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>General repressor of transcription; forms complex with Cyc8p, involved in the establishment of repressive chromatin structure through interactions with histones H3 and H4, appears to enhance expression of some genes<br><br>
			<br><b>PomBase Description</b><br>transcriptional corepressor Tup11|transcriptional corepressor Tup12<br><br>
			<br><b>AspGD Description</b><br>Protein of unknown function|Transcriptional repressor that controls expression of genes involved in cell wall biosynthesis, development, and nitrogen source utilization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2197185" target=_blank>Fujita A, <i>et al.</i> (1990 Apr 30)</a>. Cloning of the yeast SFL2 gene: its disruption results in pleiotropic phenotypes characteristic for tup1 mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2247069" target=_blank>Williams FE, <i>et al.</i> (1990 Dec)</a>. Characterization of TUP1, a mediator of glucose repression in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1901558" target=_blank>Kearsey S, <i>et al.</i> (1991 Feb 1)</a>. The SFL2 (TUP1?) protein of Saccharomyces cerevisiae contains a repeating motif homologous to beta subunits of G proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1900249" target=_blank>Zhang M, <i>et al.</i> (1991 Jan 15)</a>. A yeast protein with homology to the beta-subunit of G proteins is involved in control of heme-regulated and catabolite-repressed genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7995523" target=_blank>Komachi K, <i>et al.</i> (1994 Dec 1)</a>. The WD repeats of Tup1 interact with the homeo domain protein alpha 2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8943325" target=_blank>Varanasi US, <i>et al.</i> (1996 Dec)</a>. The Cyc8 (Ssn6)-Tup1 corepressor complex is composed of one Cyc8 and four Tup1 subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8675011" target=_blank>Edmondson DG, <i>et al.</i> (1996 May 15)</a>. Repression domain of the yeast global repressor Tup1 interacts directly with histones H3 and H4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9204892" target=_blank>Braun BR, <i>et al.</i> (1997 Jul 4)</a>. Control of filament formation in Candida albicans by the transcriptional repressor TUP1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9809417" target=_blank>Hoover CI, <i>et al.</i> (1998 Oct 15)</a>. Cloning and regulated expression of the Candida albicans phospholipase B (PLB1) gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9741624" target=_blank>Huang M, <i>et al.</i> (1998 Sep 4)</a>. The DNA replication and damage checkpoint pathways induce transcription by inhibition of the Crt1 repressor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10567571" target=_blank>Mukai Y, <i>et al.</i> (1999 Dec)</a>. Conservation of histone binding and transcriptional repressor functions in a Schizosaccharomyces pombe Tup1p homolog.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10816502" target=_blank>Phan QT, <i>et al.</i> (2000 Jun)</a>. Role of hyphal formation in interactions of Candida albicans with endothelial cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856245" target=_blank>Sprague ER, <i>et al.</i> (2000 Jun 15)</a>. Structure of the C-terminal domain of Tup1, a corepressor of transcription in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10722672" target=_blank>Papamichos-Chronakis M, <i>et al.</i> (2000 Mar 24)</a>. Hrs1/Med3 is a Cyc8-Tup1 corepressor target in the RNA polymerase II holoenzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10790384" target=_blank>Braun BR, <i>et al.</i> (2000 May)</a>. TUP1, CPH1 and EFG1 make independent contributions to filamentation in candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11069890" target=_blank>Watson AD, <i>et al.</i> (2000 Nov 1)</a>. Ssn6-Tup1 interacts with class I histone deacetylases required for repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10978273" target=_blank>Braun BR, <i>et al.</i> (2000 Sep)</a>. Identification and characterization of TUP1-regulated genes in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259598" target=_blank>Kadosh D, <i>et al.</i> (2001 Apr)</a>. Rfg1, a protein related to the Saccharomyces cerevisiae hypoxic regulator Rox1, controls filamentous growth and virulence in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11172717" target=_blank>Wu J, <i>et al.</i> (2001 Jan)</a>. TUP1 utilizes histone H3/H2B-specific HDA1 deacetylase to repress gene activity in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11452028" target=_blank>Taricani L, <i>et al.</i> (2001 Jul 15)</a>. Expression of hsp16 in response to nucleotide depletion is regulated via the spc1 MAPK pathway in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11260466" target=_blank>Hicks J, <i>et al.</i> (2001 Mar)</a>. RcoA has pleiotropic effects on Aspergillus nidulans cellular development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238405" target=_blank>Janoo RT, <i>et al.</i> (2001 Mar)</a>. Transcriptional regulators of the Schizosaccharomyces pombe fbp1 gene include two redundant Tup1p-like corepressors and the CCAAT binding factor activation complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230135" target=_blank>Proft M, <i>et al.</i> (2001 Mar 1)</a>. Regulation of the Sko1 transcriptional repressor by the Hog1 MAP kinase in response to osmotic stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11737641" target=_blank>Murad AM, <i>et al.</i> (2001 Nov)</a>. Transcript profiling in Candida albicans reveals new cellular functions for the transcriptional repressors CaTup1, CaMig1 and CaNrg1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11500510" target=_blank>Pascual-Ahuir A, <i>et al.</i> (2001 Oct 5)</a>. Multiple levels of control regulate the yeast cAMP-response element-binding protein repressor Sko1p in response to stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532938" target=_blank>Murad AM, <i>et al.</i> (2001 Sep 3)</a>. NRG1 represses yeast-hypha morphogenesis and hypha-specific gene expression in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11784848" target=_blank>Davie JK, <i>et al.</i> (2002 Feb)</a>. Histone-dependent association of Tup1-Ssn6 with repressed genes in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12455984" target=_blank>Zhao R, <i>et al.</i> (2002 Jun)</a>. Roles of TUP1 in switching, phase maintenance, and phase-specific gene expression in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11956219" target=_blank>Pelletier B, <i>et al.</i> (2002 Jun 21)</a>. Fep1, an iron sensor regulating iron transporter gene expression in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12221110" target=_blank>Greenall A, <i>et al.</i> (2002 Sep)</a>. Role of fission yeast Tup1-like repressors and Prr1 transcription factor in response to salt stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12657047" target=_blank>Todd RB, <i>et al.</i> (2003 Apr)</a>. TupA, the Penicillium marneffei Tup1p homologue, represses both yeast and spore development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665463" target=_blank>Mennella TA, <i>et al.</i> (2003 Dec)</a>. Recruitment of Tup1-Ssn6 by yeast hypoxic genes and chromatin-independent exclusion of TATA binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14525981" target=_blank>Davie JK, <i>et al.</i> (2003 Dec 12)</a>. Tup1-Ssn6 interacts with multiple class I histone deacetylases in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12637515" target=_blank>Mukai Y, <i>et al.</i> (2003 May 23)</a>. Physical and functional interaction of the yeast corepressor Tup1 with mRNA 5'-triphosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14573465" target=_blank>Hirota K, <i>et al.</i> (2003 Oct)</a>. Fission yeast Tup1-like repressors repress chromatin remodeling at the fbp1+ promoter and the ade6-M26 recombination hotspot.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14762213" target=_blank>Hirota K, <i>et al.</i> (2004)</a>. Fission yeast global repressors regulate the specificity of chromatin alteration in response to distinct environmental stresses.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16313617" target=_blank>Park YN, <i>et al.</i> (2005 Dec)</a>. Candida albicans MTLalpha tup1Delta mutants can reversibly switch to mating-competent, filamentous growth forms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15673737" target=_blank>Cao YY, <i>et al.</i> (2005 Feb)</a>. cDNA microarray analysis of differential gene expression in Candida albicans biofilm exposed to farnesol.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15632072" target=_blank>Fagerström-Billai F, <i>et al.</i> (2005 Jan)</a>. Functional comparison of the Tup11 and Tup12 transcriptional corepressors in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15814840" target=_blank>Kadosh D, <i>et al.</i> (2005 Jun)</a>. Induction of the Candida albicans filamentous growth program by relief of transcriptional repression: a genome-wide analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980390" target=_blank>Todd RB, <i>et al.</i> (2006 Nov)</a>. The Aspergillus nidulans rcoA gene is required for veA-dependent sexual development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17101775" target=_blank>Fagerström-Billai F, <i>et al.</i> (2007 Feb)</a>. Individual subunits of the Ssn6-Tup11/12 corepressor are selectively required for repression of different target genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18622392" target=_blank>Monahan BJ, <i>et al.</i> (2008 Aug)</a>. Fission yeast SWI/SNF and RSC complexes show compositional and functional differences from budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054105" target=_blank>García I, <i>et al.</i> (2008 Dec)</a>. Roles of the Aspergillus nidulans homologues of Tup1 and Ssn6 in chromatin structure and cell viability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18457967" target=_blank>Calvo AM, <i>et al.</i> (2008 Jul)</a>. The VeA regulatory system and its role in morphological and chemical development in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18424510" target=_blank>Kebaara BW, <i>et al.</i> (2008 Jun)</a>. Candida albicans Tup1 is involved in farnesol-mediated inhibition of filamentous-growth induction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18703157" target=_blank>Miskei M, <i>et al.</i> (2009 Mar)</a>. Annotation of stress-response proteins in the aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23983529" target=_blank>Han KH, <i>et al.</i> (2009 Sep)</a>. Molecular Genetics of Emericella nidulans Sexual Development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20200159" target=_blank>Kang WH, <i>et al.</i> (2010 Apr 30)</a>. The LAMMER kinase homolog, Lkh1, regulates Tup transcriptional repressors through phosphorylation in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20396879" target=_blank>Matsuzawa T, <i>et al.</i> (2010 Jun)</a>. The gld1+ gene encoding glycerol dehydrogenase is required for glycerol metabolism in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20544037" target=_blank>Ferreira ME, <i>et al.</i> (2010 Jun 8)</a>. WD40 domain divergence is important for functional differences between the fission yeast Tup11 and Tup12 co-repressor proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22098069" target=_blank>Matsuzawa T, <i>et al.</i> (2011 Dec)</a>. Identification of a galactose-specific flocculin essential for non-sexual flocculation and filamentous growth in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21511999" target=_blank>Rhind N, <i>et al.</i> (2011 May 20)</a>. Comparative functional genomics of the fission yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21658084" target=_blank>Bayram O, <i>et al.</i> (2012 Jan)</a>. Coordination of secondary metabolism and development in fungi: the velvet family of regulatory proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22091779" target=_blank>Dyer PS, <i>et al.</i> (2012 Jan)</a>. Sexual development and cryptic sexuality in fungi: insights from Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22226946" target=_blank>Kitamura K, <i>et al.</i> (2012 Mar)</a>. The Ubiquitin ligase Ubr11 is essential for oligopeptide utilization in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23365689" target=_blank>Tay Z, <i>et al.</i> (2013)</a>. Cellular robustness conferred by genetic crosstalk underlies resistance against chemotherapeutic drug doxorubicin in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23861937" target=_blank>Zhang L, <i>et al.</i> (2013)</a>. Genome-wide screening for genes associated with valproic acid sensitivity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23738021" target=_blank>Zhou X, <i>et al.</i> (2013)</a>. A genome-wide screening of potential target genes to enhance the antifungal activity of micafungin in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633078" target=_blank>Sun X, <i>et al.</i> (2013 May)</a>. PyrG is required for maintaining stable cellular uracil level and normal sporulation pattern under excess uracil stress in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25373780" target=_blank>Graml V, <i>et al.</i> (2014 Oct 27)</a>. A genomic Multiprocess survey of machineries that control and link cell shape, microtubule organization, and cell-cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26210809" target=_blank>Dzikowska A, <i>et al.</i> (2015 Dec 1)</a>. KAEA (SUDPRO), a member of the ubiquitous KEOPS/EKC protein complex, regulates the arginine catabolic pathway and the expression of several other genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25669599" target=_blank>Nguyen TT, <i>et al.</i> (2015 Feb 11)</a>. Fitness profiling links topoisomerase II regulation of centromeric integrity to doxorubicin resistance in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25535331" target=_blank>Asada R, <i>et al.</i> (2015 Mar)</a>. Antagonistic controls of chromatin and mRNA start site selection by Tup family corepressors and the CCAAT-binding factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25701403" target=_blank>Tang MY, <i>et al.</i> (2015 May 10)</a>. Two fission yeast high mobility group box proteins in the maintenance of genomic integrity following doxorubicin insult.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26791325" target=_blank>Nguyen TT, <i>et al.</i> (2016 Jan 21)</a>. Predicting chemotherapeutic drug combinations through gene network profiling.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00449_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00449_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00449_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00449_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00449">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG00450<br>EOG81RN97<br><br>sce:RSA4<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>WD-repeat protein involved in ribosome biogenesis; may interact with ribosomes; required for maturation and efficient intra-nuclear transport or pre-60S ribosomal subunits, localizes to the nucleolus<br><br>
			<br><b>PomBase Description</b><br>notchless-like protein (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular response to drug, ribosomal large subunit assembly and nucleolus, ribosome localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16221974" target=_blank>de la Cruz J, <i>et al.</i> (2005)</a>. The essential WD-repeat protein Rsa4p is required for rRNA processing and intra-nuclear transport of 60S ribosomal subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16702403" target=_blank>Fleischer TC, <i>et al.</i> (2006 May 15)</a>. Systematic identification and functional screens of uncharacterized proteins associated with eukaryotic ribosomal complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18658244" target=_blank>Lebreton A, <i>et al.</i> (2008 Sep)</a>. 60S ribosomal subunit assembly dynamics defined by semi-quantitative mass spectrometry of purified complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19737519" target=_blank>Ulbrich C, <i>et al.</i> (2009 Sep 4)</a>. Mechanochemical removal of ribosome biogenesis factors from nascent 60S ribosomal subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20542003" target=_blank>Bassler J, <i>et al.</i> (2010 Jun 11)</a>. The AAA-ATPase Rea1 drives removal of biogenesis factors during multiple stages of 60S ribosome assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24662372" target=_blank>Leidig C, <i>et al.</i> (2014 Mar 24)</a>. 60S ribosome biogenesis requires rotation of the 5S ribonucleoprotein particle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25404745" target=_blank>Baßler J, <i>et al.</i> (2014 Nov 24)</a>. A network of assembly factors is involved in remodeling rRNA elements during preribosome maturation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26150393" target=_blank>Baßler J, <i>et al.</i> (2015 Jul 6)</a>. A network of assembly factors is involved in remodeling rRNA elements during preribosome maturation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26619264" target=_blank>Barrio-Garcia C, <i>et al.</i> (2016 Jan)</a>. Architecture of the Rix1-Rea1 checkpoint machinery during pre-60S-ribosome remodeling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27251291" target=_blank>Wu S, <i>et al.</i> (2016 Jun 2)</a>. Diverse roles of assembly factors revealed by structures of late nuclear pre-60S ribosomes.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00450_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00450_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00450_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00450_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00450">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG00451<br>EOG81RN97<br><br>sce:ASC1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>G-protein beta subunit and guanine dissociation inhibitor for Gpa2p; ortholog of RACK1 that inhibits translation; core component of the small (40S) ribosomal subunit; regulates P-body formation induced by replication stress; represses Gcn4p in the absence of amino acid starvation<br><br>
			<br><b>PomBase Description</b><br>RACK1 ortholog Cpc2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have G-protein alpha-subunit binding, GDP-dissociation inhibitor activity, protein kinase C binding, ribosome binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9038161" target=_blank>Norbeck J, <i>et al.</i> (1997 Feb 28)</a>. Metabolic and regulatory changes associated with growth of Saccharomyces cerevisiae in 1.4 M NaCl. Evidence for osmotic induction of glycerol dissimilation via the dihydroxyacetone pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931303" target=_blank>Hoffmann B, <i>et al.</i> (2000 Jul)</a>. c-Jun and RACK1 homologues regulate a control point for sexual development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12207217" target=_blank>Melin P, <i>et al.</i> (2002 Aug)</a>. Proteome analysis of Aspergillus nidulans reveals proteins associated with the response to the antibiotic concanamycin A, produced by Streptomyces species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15012629" target=_blank>Baum S, <i>et al.</i> (2004 Jun 15)</a>. Asc1p, a WD40-domain containing adaptor protein, is required for the interaction of the RNA-binding protein Scp160p with polysomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15378761" target=_blank>Pitarch A, <i>et al.</i> (2004 Oct)</a>. Proteomics-based identification of novel Candida albicans antigens for diagnosis of systemic candidiasis in patients with underlying hematological malignancies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15334071" target=_blank>Sengupta J, <i>et al.</i> (2004 Oct)</a>. Identification of the versatile scaffold protein RACK1 on the eukaryotic ribosome by cryo-EM.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15340087" target=_blank>Gerbasi VR, <i>et al.</i> (2004 Sep)</a>. Yeast Asc1p and mammalian RACK1 are functionally orthologous core 40S ribosomal proteins that repress gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15937140" target=_blank>Chibana H, <i>et al.</i> (2005 Aug)</a>. Sequence finishing and gene mapping for Candida albicans chromosome 7 and syntenic analysis against the Saccharomyces cerevisiae genome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919853" target=_blank>Oh YT, <i>et al.</i> (2010 Mar)</a>. Proteomic analysis of early phase of conidia germination in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20929924" target=_blank>Liu X, <i>et al.</i> (2010 Nov)</a>. Asc1, a WD-repeat protein, is required for hyphal development and virulence in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21109664" target=_blank>Ben-Shem A, <i>et al.</i> (2010 Nov 26)</a>. Crystal structure of the eukaryotic ribosome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21704636" target=_blank>Yatime L, <i>et al.</i> (2011 Aug 12)</a>. Structure of the RACK1 dimer from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096102" target=_blank>Ben-Shem A, <i>et al.</i> (2011 Dec 16)</a>. The structure of the eukaryotic ribosome at 3.0 Å resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21553327" target=_blank>Freitas JS, <i>et al.</i> (2011 Sep)</a>. Transcription of the Hsp30, Hsp70, and Hsp90 heat shock protein genes is modulated by the PalA protein in response to acid pH-sensing in the fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092920" target=_blank>Park HS, <i>et al.</i> (2012 Dec)</a>. Genetic control of asexual sporulation in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22091779" target=_blank>Dyer PS, <i>et al.</i> (2012 Jan)</a>. Sexual development and cryptic sexuality in fungi: insights from Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23936193" target=_blank>Kong Q, <i>et al.</i> (2013)</a>. Gβ-like CpcB plays a crucial role for growth and development of Aspergillus nidulans and Aspergillus fumigatus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24120530" target=_blank>Martins I, <i>et al.</i> (2013 Dec 6)</a>. Proteomic alterations induced by ionic liquids in Aspergillus nidulans and Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24524803" target=_blank>Ban N, <i>et al.</i> (2014 Feb)</a>. A new system for naming ribosomal proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25417110" target=_blank>Hussain T, <i>et al.</i> (2014 Oct 23)</a>. Structural changes enable start codon recognition by the eukaryotic translation initiation complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25892048" target=_blank>Cai ZD, <i>et al.</i> (2015 Aug)</a>. The Gβ-like protein CpcB is required for hyphal growth, conidiophore morphology and pathogenicity in Aspergillus fumigatus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26212456" target=_blank>Llácer JL, <i>et al.</i> (2015 Aug 6)</a>. Conformational Differences between Open and Closed States of the Eukaryotic Translation Initiation Complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27159451" target=_blank>Murray J, <i>et al.</i> (2016 May 9)</a>. Structural characterization of ribosome recruitment and translocation by type IV IRES.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00451_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00451_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00451_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00451_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00451">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG00452<br>EOG81RN97<br><br>sce:SEC27<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Essential beta'-coat protein of the COPI coatomer; involved in ER-to-Golgi and Golgi-to-ER transport; contains WD40 domains that mediate cargo selective interactions; 45% sequence identity to mammalian beta'-COP<br><br>
			<br><b>PomBase Description</b><br>coatomer beta' subunit (predicted)<br><br>
			<br><b>AspGD Description</b><br>COPI complex, beta' subunit; expression enhanced by maltose<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8405452" target=_blank>Harter C, <i>et al.</i> (1993 Oct 11)</a>. Yeast coatomer contains a subunit homologous to mammalian beta'-COP.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7929113" target=_blank>Duden R, <i>et al.</i> (1994 Sep 30)</a>. Yeast beta- and beta'-coat proteins (COP). Two coatomer subunits essential for endoplasmic reticulum-to-Golgi protein traffic.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870366" target=_blank>Sims AH, <i>et al.</i> (2005 May)</a>. Transcriptome analysis of recombinant protein secretion by Aspergillus nidulans and the unfolded-protein response in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17101773" target=_blank>Gabriely G, <i>et al.</i> (2007 Jan)</a>. Involvement of specific COPI subunits in protein sorting from the late endosome to the vacuole in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00452_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00452_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00452_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00452_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00452">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG00453<br>EOG8W6MBB<br><br>sce:SWD3<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Essential subunit of the COMPASS (Set1C) complex; COMPASS methylates histone H3 on lysine 4 and is required in transcriptional silencing near telomeres; WD40 beta propeller superfamily member and ortholog of mammalian WDR5<br><br>
			<br><b>PomBase Description</b><br>WD repeat protein Swd3<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN3926, A. fumigatus Af293 : Afu6g08380, A. oryzae RIB40 : AO090003000092, Aspergillus wentii : Aspwe1_0045950 and Aspergillus sydowii : Aspsy1_0044421<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742990" target=_blank>Roguev A, <i>et al.</i> (2001 Dec 17)</a>. The Saccharomyces cerevisiae Set1 complex includes an Ash2 homologue and methylates histone 3 lysine 4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11687631" target=_blank>Miller T, <i>et al.</i> (2001 Nov 6)</a>. COMPASS: a complex of proteins associated with a trithorax-related SET domain protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11805083" target=_blank>Krogan NJ, <i>et al.</i> (2002 Mar 29)</a>. COMPASS, a histone H3 (Lysine 4) methyltransferase required for telomeric silencing of gene expression.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00453_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00453_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00453_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00453_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00453">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG00454<br>EOG81RN97<br><br>sce:COP1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Alpha subunit of COPI vesicle coatomer complex; complex surrounds transport vesicles in the early secretory pathway<br><br>
			<br><b>PomBase Description</b><br>coatomer alpha subunit Cop1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>COPI complex. alpha subunit; expression enhanced by maltose<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8001155" target=_blank>Letourneur F, <i>et al.</i> (1994 Dec 30)</a>. Coatomer is essential for retrieval of dilysine-tagged proteins to the endoplasmic reticulum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7724544" target=_blank>Gerich B, <i>et al.</i> (1995 Apr 11)</a>. Non-clathrin-coat protein alpha is a conserved subunit of coatomer and in Saccharomyces cerevisiae is essential for growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17101773" target=_blank>Gabriely G, <i>et al.</i> (2007 Jan)</a>. Involvement of specific COPI subunits in protein sorting from the late endosome to the vacuole in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19726565" target=_blank>Sato K, <i>et al.</i> (2009 Oct)</a>. Kei1: a novel subunit of inositolphosphorylceramide synthase, essential for its enzyme activity and Golgi localization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00454_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00454_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00454_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00454_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00454">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG00455<br>EOG81RN97<br><br>sce:MET30<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>F-box protein containing five copies of the WD40 motif; controls cell cycle function, sulfur metabolism, and methionine biosynthesis as part of the ubiquitin ligase complex; interacts with and regulates Met4p, localizes within the nucleus; dissociation of Met30p from SCF complex in response to cadmium stress is regulated by Cdc48p<br><br>
			<br><b>PomBase Description</b><br>F-box/WD repeat protein protein Pof1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have protein binding, bridging, ubiquitin binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765129" target=_blank>Paszewski A, <i>et al.</i> (1994)</a>. Sulphur metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8524217" target=_blank>Thomas D, <i>et al.</i> (1995 Dec)</a>. Met30p, a yeast transcriptional inhibitor that responds to S-adenosylmethionine, is an essential protein with WD40 repeats.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9520259" target=_blank>Natorff R, <i>et al.</i> (1998 Feb)</a>. The Aspergillus nidulans sulphur regulatory gene sconB encodes a protein with WD40 repeats and an F-box.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9499404" target=_blank>Patton EE, <i>et al.</i> (1998 Mar 1)</a>. Cdc53 is a scaffold protein for multiple Cdc34/Skp1/F-box proteincomplexes that regulate cell division and methionine biosynthesis in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10637232" target=_blank>Rouillon A, <i>et al.</i> (2000 Jan 17)</a>. Feedback-regulated degradation of the transcriptional activator Met4 is triggered by the SCF(Met30 )complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11027256" target=_blank>Smothers DB, <i>et al.</i> (2000 Nov)</a>. The abundance of Met30p limits SCF(Met30p) complex activity and is regulated by methionine availability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11085267" target=_blank>Piotrowska M, <i>et al.</i> (2000 Oct)</a>. sconC, a gene involved in the regulation of sulphur metabolism in Aspergillus nidulans, belongs to the SKP1 gene family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11820777" target=_blank>Ikebe C, <i>et al.</i> (2002 Feb 8)</a>. Isolation and characterization of a novel F-box protein Pof10 in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12890030" target=_blank>Natorff R, <i>et al.</i> (2003 Aug)</a>. The Aspergillus nidulans metR gene encodes a bZIP protein which activates transcription of sulphur metabolism genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14660673" target=_blank>Brunson LE, <i>et al.</i> (2004 Feb 20)</a>. The amino-terminal portion of the F-box protein Met30p mediates its nuclear import and assimilation into an SCF complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15147268" target=_blank>Lehmann A, <i>et al.</i> (2004 May)</a>. Molecular interactions of fission yeast Skp1 and its role in the DNA damage checkpoint.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15689486" target=_blank>Yen JL, <i>et al.</i> (2005 Apr)</a>. The yeast ubiquitin ligase SCFMet30 regulates heavy metal response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15660136" target=_blank>Harrison C, <i>et al.</i> (2005 Feb 9)</a>. SCF(Pof1)-ubiquitin and its target Zip1 transcription factor mediate cadmium response in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15883825" target=_blank>Brunson LE, <i>et al.</i> (2005 Jun)</a>. Identification of residues in the WD-40 repeat motif of the F-box protein Met30p required for interaction with its substrate Met4p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870262" target=_blank>Su NY, <i>et al.</i> (2005 May)</a>. The F-box protein Met30 is required for multiple steps in the budding yeast cell cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17257865" target=_blank>Sieńko M, <i>et al.</i> (2007 Jul)</a>. Two Aspergillus nidulans genes encoding methylenetetrahydrofolate reductases are up-regulated by homocysteine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19748355" target=_blank>Schmidt MW, <i>et al.</i> (2009 Sep 11)</a>. F-box-directed CRL complex assembly and regulation by the CSN and CAND1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20955810" target=_blank>Brzywczy J, <i>et al.</i> (2011 Feb)</a>. Novel mutations reveal two important regions in Aspergillus nidulans transcriptional activator MetR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22142781" target=_blank>Colabardini AC, <i>et al.</i> (2012 Feb)</a>. Molecular characterization of the Aspergillus nidulans fbxA encoding an F-box protein involved in xylanase induction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23535663" target=_blank>Wu S, <i>et al.</i> (2013)</a>. CAND1 controls in vivo dynamics of the cullin 1-RING ubiquitin ligase repertoire.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24513272" target=_blank>Sieńko M, <i>et al.</i> (2014 Apr)</a>. Regulatory mutations affecting sulfur metabolism induce environmental stress response in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00455_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00455_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00455_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00455_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00455">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG00456<br>EOG81RN97<br><br>sce:PAC1<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Involved in nuclear migration, part of the dynein/dynactin pathway; targets dynein to microtubule tips, which is necessary for sliding of microtubules along bud cortex; serves at interface between dynein's ATPase site and its microtubule binding stalk, causing individual dynein motors to remain attached to microtubules for long periods; synthetic lethal with bni1; homolog of human LIS1, mutations in which cause the severe brain disorder lissencephaly<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have microtubule plus-end binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824456" target=_blank>Xiang X, <i>et al.</i> (1995)</a>. Analysis of nuclear migration in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8601474" target=_blank>Willins DA, <i>et al.</i> (1995 Dec)</a>. An alpha tubulin mutation suppresses nuclear migration mutations in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14732112" target=_blank>Morris NR, <i>et al.</i> (1995 Jul)</a>. Nuclear migration advances in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7612965" target=_blank>Xiang X, <i>et al.</i> (1995 Mar)</a>. NudF, a nuclear migration gene in Aspergillus nidulans, is similar to the human LIS-1 gene required for neuronal migration.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8647384" target=_blank>Chiu YH, <i>et al.</i> (1995 Oct)</a>. Extragenic suppressors of nudC3, a mutation that blocks nuclear migration in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568239" target=_blank>Xiang X, <i>et al.</i> (1995 Oct 10)</a>. Characterization and localization of the cytoplasmic dynein heavy chain in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8856514" target=_blank>Wolkow TD, <i>et al.</i> (1996 Aug)</a>. Cytokinesis in Aspergillus nidulans is controlled by cell size, nuclear positioning and mitosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8896271" target=_blank>Chéret G, <i>et al.</i> (1996 Sep)</a>. DNA sequence analysis of the VPH1-SNF2 region on chromosome XV of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9013770" target=_blank>Morris SM, <i>et al.</i> (1997 Feb)</a>. A prolactin-inducible T cell gene product is structurally similar to the Aspergillus nidulans nuclear movement protein NUDC.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9201714" target=_blank>Geiser JR, <i>et al.</i> (1997 Jun)</a>. Saccharomyces cerevisiae genes required in the absence of the CIN8-encoded spindle motor act in functionally diverse mitotic pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236777" target=_blank>Willins DA, <i>et al.</i> (1997 Jun)</a>. Mutations in the heavy chain of cytoplasmic dynein suppress the nudF nuclear migration mutation of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9861667" target=_blank>Morris NR, <i>et al.</i> (1998 Dec)</a>. Nuclear migration, nucleokinesis and lissencephaly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10077853" target=_blank>Fischer R, <i>et al.</i> (1999 Jan)</a>. Nuclear movement in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10498683" target=_blank>Liu Z, <i>et al.</i> (1999 Oct)</a>. Lis1, the Drosophila homolog of a human lissencephaly disease gene, is required for germline cell division and oocyte differentiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931877" target=_blank>Efimov VP, <i>et al.</i> (2000 Aug 7)</a>. The LIS1-related NUDF protein of Aspergillus nidulans interacts with the coiled-coil domain of the NUDE/RO11 protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11163258" target=_blank>Feng Y, <i>et al.</i> (2000 Dec)</a>. LIS1 regulates CNS lamination by interacting with mNudE, a central component of the centrosome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11163260" target=_blank>Niethammer M, <i>et al.</i> (2000 Dec)</a>. NUDEL is a novel Cdk5 substrate that associates with LIS1 and cytoplasmic dynein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11134054" target=_blank>Ahn C, <i>et al.</i> (2001 Mar 30)</a>. Nudf, a fungal homolog of the human LIS1 protein, functions as a dimer in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11369237" target=_blank>Han G, <i>et al.</i> (2001 May 1)</a>. The Aspergillus cytoplasmic dynein heavy chain and NUDF localize to microtubule ends and affect microtubule dynamics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11509576" target=_blank>Hoffmann B, <i>et al.</i> (2001 Oct 19)</a>. The LIS1-related protein NUDF of Aspergillus nidulans and its interaction partner NUDE bind directly to specific subunits of dynein and dynactin and to alpha- and gamma-tubulin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11685578" target=_blank>Dawe AL, <i>et al.</i> (2001 Sep)</a>. Evolutionarily conserved nuclear migration genes required for early embryonic development in Caenorhabditis elegans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972777" target=_blank>Zhang J, <i>et al.</i> (2002 Apr)</a>. Cytoplasmic dynein intermediate chain and heavy chain are dependent upon each other for microtubule end localization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12686603" target=_blank>Zhang J, <i>et al.</i> (2003 Apr)</a>. Accumulation of cytoplasmic dynein and dynactin at microtubule plus ends in Aspergillus nidulans is kinesin dependent.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12566428" target=_blank>Lee WL, <i>et al.</i> (2003 Feb 3)</a>. The role of the lissencephaly protein Pac1 during nuclear migration in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12631710" target=_blank>Efimov VP, <i>et al.</i> (2003 Mar)</a>. Roles of NUDE and NUDF proteins of Aspergillus nidulans: insights from intracellular localization and overexpression effects.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998524" target=_blank>Xiang X, <i>et al.</i> (2004 Apr)</a>. Nuclear migration and positioning in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14712821" target=_blank>Hoffmann B, <i>et al.</i> (2004 Jan 2)</a>. The LIS1-related protein NUDF of Aspergillus nidulans and its interaction partner NUDE bind directly to specific subunits of dynein and dynactin and to alpha- and gamma-tubulin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15232582" target=_blank>Holzbaur EL, <i>et al.</i> (2004 Jul)</a>. Tangled NUDELs?<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15930134" target=_blank>Li S, <i>et al.</i> (2005 Aug)</a>. Cytoplasmic dynein's mitotic spindle pole localization requires a functional anaphase-promoting complex, gamma-tubulin, and NUDF/LIS1 in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16105883" target=_blank>Veith D, <i>et al.</i> (2005 Aug 15)</a>. Role of the spindle-pole-body protein ApsB and the cortex protein ApsA in microtubule organization and nuclear migration in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15965467" target=_blank>Li J, <i>et al.</i> (2005 Jul)</a>. NudEL targets dynein to microtubule ends through LIS1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16467375" target=_blank>Efimov VP, <i>et al.</i> (2006 Apr)</a>. CLIP-170 homologue and NUDE play overlapping roles in NUDF localization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17237365" target=_blank>Enke C, <i>et al.</i> (2007 Mar)</a>. Aspergillus nidulans Dis1/XMAP215 protein AlpA localizes to spindle pole bodies and microtubule plus ends and contributes to growth directionality.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17237507" target=_blank>Zhuang L, <i>et al.</i> (2007 Mar)</a>. Point mutations in the stem region and the fourth AAA domain of cytoplasmic dynein heavy chain partially suppress the phenotype of NUDF/LIS1 loss in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18390647" target=_blank>Helmstaedt K, <i>et al.</i> (2008 Jun)</a>. The nuclear migration protein NUDF/LIS1 forms a complex with NUDC and BNFA at spindle pole bodies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18399939" target=_blank>Fischer R, <i>et al.</i> (2008 May)</a>. Polarized growth in fungi--interplay between the cytoskeleton, positional markers and membrane domains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19185494" target=_blank>Markus SM, <i>et al.</i> (2009 Feb 10)</a>. Motor- and tail-dependent targeting of dynein to microtubule plus ends and the cell cortex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20876661" target=_blank>Zhang J, <i>et al.</i> (2010 Oct 15)</a>. The microtubule plus-end localization of Aspergillus dynein is important for dynein-early-endosome interaction but not for dynein ATPase activation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20675372" target=_blank>Zhu XJ, <i>et al.</i> (2010 Sep 24)</a>. The L279P mutation of nuclear distribution gene C (NudC) influences its chaperone activity and lissencephaly protein 1 (LIS1) stability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21708978" target=_blank>Zhang J, <i>et al.</i> (2011 Jun 27)</a>. The p25 subunit of the dynactin complex is required for dynein-early endosome interaction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127389" target=_blank>Egan MJ, <i>et al.</i> (2012 Dec)</a>. Microtubule-based transport in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22645705" target=_blank>Peñalva MA, <i>et al.</i> (2012 Jan 1)</a>. Searching for gold beyond mitosis: Mining intracellular membrane traffic in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23212922" target=_blank>Qiu R, <i>et al.</i> (2013 Jan 25)</a>. Identification of a novel site in the tail of dynein heavy chain important for dynein function in vivo.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00456_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00456_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00456_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00456_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00456">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG00457<br>EOG81RN97<br><br>sce:LST8<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein required for the transport of Gap1p; required for the transport of amino acid permease Gap1p from the Golgi to the cell surface; component of the TOR signaling pathway; associates with both Tor1p and Tor2p; contains a WD-repeat<br><br>
			<br><b>PomBase Description</b><br>WD repeat protein Pop3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in microtubule cytoskeleton organization, mitotic sister chromatid segregation and TORC1 complex, TORC2 complex, cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=958201" target=_blank>Nurse P, <i>et al.</i> (1976 Jul 23)</a>. Genetic control of the cell division cycle in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19473" target=_blank>Bañuelos M, <i>et al.</i> (1977 Sep 25)</a>. Activation by phosphate of yeast phosphofructokinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6490749" target=_blank>Streiblová E, <i>et al.</i> (1984 Jul)</a>. Septum pattern in ts mutants of Schizosaccharomyces pombe defective in genes cdc3, cdc4, cdc8 and cdc12.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409822" target=_blank>Roberg KJ, <i>et al.</i> (1997 Dec)</a>. Control of amino acid permease sorting in the late secretory pathway of Saccharomyces cerevisiae by SEC13, LST4, LST7 and LST8.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9108274" target=_blank>Kemp JT, <i>et al.</i> (1997 Mar 26)</a>. A wat1 mutant of fission yeast is defective in cell morphology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11686295" target=_blank>Ochotorena IL, <i>et al.</i> (2001 Aug)</a>. Conserved Wat1/Pop3 WD-repeat protein of fission yeast secures genome stability through microtubule integrity and may be involved in mRNA maturation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742997" target=_blank>Liu Z, <i>et al.</i> (2001 Dec 17)</a>. RTG-dependent mitochondria to nucleus signaling is negatively regulated by the seven WD-repeat protein Lst8p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12408816" target=_blank>Loewith R, <i>et al.</i> (2002 Sep)</a>. Two TOR complexes, only one of which is rapamycin sensitive, have distinct roles in cell growth control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12719473" target=_blank>Chen EJ, <i>et al.</i> (2003 Apr 28)</a>. LST8 negatively regulates amino acid biosynthesis as a component of the TOR pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12631735" target=_blank>Wedaman KP, <i>et al.</i> (2003 Mar)</a>. Tor kinases are in distinct membrane-associated protein complexes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15507118" target=_blank>Yuasa T, <i>et al.</i> (2004 Nov)</a>. An interactive gene network for securin-separase, condensin, cohesin, Dis1/Mtc1 and histones constructed by mass transformation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16002396" target=_blank>Wullschleger S, <i>et al.</i> (2005 Sep 2)</a>. Molecular organization of target of rapamycin complex 2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17046992" target=_blank>Alvarez B, <i>et al.</i> (2006 Nov 1)</a>. Fission yeast Tor2 promotes cell growth and represses cell differentiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17261596" target=_blank>Matsuo T, <i>et al.</i> (2007 Apr)</a>. Loss of the TOR kinase Tor2 mimics nitrogen starvation and activates the sexual development pathway in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18076573" target=_blank>Hayashi T, <i>et al.</i> (2007 Dec)</a>. Rapamycin sensitivity of the Schizosaccharomyces pombe tor2 mutant and organization of two highly phosphorylated TOR complexes by specific and common subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18076567" target=_blank>Kanoh J, <i>et al.</i> (2007 Dec)</a>. Tel2: a common partner of PIK-related kinases and a link between DNA checkpoint and nutritional response?<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18235227" target=_blank>Ikeda K, <i>et al.</i> (2008 Feb 1)</a>. Fission yeast TOR complex 2 activates the AGC-family Gad8 kinase essential for stress resistance and cell cycle control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21035342" target=_blank>Tatebe H, <i>et al.</i> (2010 Nov 23)</a>. Rab-family GTPase regulates TOR complex 2 signaling in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22891259" target=_blank>Cortés JC, <i>et al.</i> (2012 Aug 20)</a>. Fission yeast Ags1 confers the essential septum strength needed for safe gradual cell abscission.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22976295" target=_blank>Nakashima A, <i>et al.</i> (2012 Dec 1)</a>. Psk1, an AGC kinase family member in fission yeast, is directly phosphorylated and controlled by TORC1 and functions as S6 kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695164" target=_blank>Das J, <i>et al.</i> (2013 May 21)</a>. Cross-species protein interactome mapping reveals species-specific wiring of stress response pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24586893" target=_blank>Verma SK, <i>et al.</i> (2014)</a>. Wat1/pop3, a conserved WD repeat containing protein acts synergistically with checkpoint kinase Chk1 to maintain genome ploidy in fission yeast S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25373780" target=_blank>Graml V, <i>et al.</i> (2014 Oct 27)</a>. A genomic Multiprocess survey of machineries that control and link cell shape, microtubule organization, and cell-cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25590601" target=_blank>Hatano T, <i>et al.</i> (2015)</a>. Fission yeast Ryh1 GTPase activates TOR Complex 2 in response to glucose.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25411338" target=_blank>Saitoh S, <i>et al.</i> (2015 Jan 15)</a>. Mechanisms of expression and translocation of major fission yeast glucose transporters regulated by CaMKK/phosphatases, nuclear shuttling, and TOR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206859" target=_blank>Baker K, <i>et al.</i> (2016 Jul 1)</a>. TOR complex 2 localises to the cytokinetic actomyosin ring and controls the fidelity of cytokinesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28264193" target=_blank>Tatebe H, <i>et al.</i> (2017 Mar 7)</a>. Substrate specificity of TOR complex 2 is determined by a ubiquitin-fold domain of the Sin1 subunit.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28821619" target=_blank>Burr R, <i>et al.</i> (2017 Sep 29)</a>. Dsc E3 ligase localization to the Golgi requires the ATPase Cdc48 and cofactor Ufd1 for activation of sterol regulatory element-binding protein in fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00457_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00457_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00457_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00457_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00457">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG00458<br>EOG8W6MBB<br><br>sce:MDV1<br><br>Genes: 27
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Peripheral protein of cytosolic face of mitochondrial outer membrane; required for mitochondrial fission; interacts with Fis1p and with the dynamin-related GTPase Dnm1p; contains WD repeats; MDV1 has a paralog, CAF4, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>CCR4-Not complex subunit Caf4/Mdv1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ubiquitin binding activity, role in mitochondrial fission, mitochondrial genome maintenance, peroxisome fission and mitochondrial outer membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11038180" target=_blank>Fekkes P, <i>et al.</i> (2000 Oct 16)</a>. Gag3p, an outer membrane protein required for fission of mitochondrial tubules.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11038183" target=_blank>Mozdy AD, <i>et al.</i> (2000 Oct 16)</a>. Dnm1p GTPase-mediated mitochondrial fission is a multi-step process requiring the novel integral membrane component Fis1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11038182" target=_blank>Tieu Q, <i>et al.</i> (2000 Oct 16)</a>. Mdv1p is a WD repeat protein that interacts with the dynamin-related GTPase, Dnm1p, to trigger mitochondrial division.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12163467" target=_blank>Tieu Q, <i>et al.</i> (2002 Aug 5)</a>. The WD repeat protein, Mdv1p, functions as a molecular adaptor by interacting with Dnm1p and Fis1p during mitochondrial fission.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14517324" target=_blank>Cerveny KL, <i>et al.</i> (2003 Oct)</a>. The WD-repeats of Net2p interact with Dnm1p and Fis1p to regulate division of mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16009724" target=_blank>Griffin EE, <i>et al.</i> (2005 Jul 18)</a>. The WD40 protein Caf4p is a component of the mitochondrial fission machinery and recruits Dnm1p to mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15809300" target=_blank>Suzuki M, <i>et al.</i> (2005 Jun 3)</a>. Novel structure of the N terminus in yeast Fis1 correlates with a specialized function in mitochondrial fission.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16247028" target=_blank>Karren MA, <i>et al.</i> (2005 Oct 24)</a>. The role of Fis1p-Mdv1p interactions in mitochondrial fission complex assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16272155" target=_blank>Naylor K, <i>et al.</i> (2006 Jan 27)</a>. Mdv1 interacts with assembled dnm1 to promote mitochondrial division.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16601120" target=_blank>Bhar D, <i>et al.</i> (2006 Jun 23)</a>. Dimeric Dnm1-G385D interacts with Mdv1 on mitochondria and can be stimulated to assemble into fission complexes containing Mdv1 and Fis1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16407407" target=_blank>Zahedi RP, <i>et al.</i> (2006 Mar)</a>. Proteomic analysis of the yeast mitochondrial outer membrane reveals accumulation of a subclass of preproteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23530241" target=_blank>Koirala S, <i>et al.</i> (2013 Apr 9)</a>. Interchangeable adaptors regulate mitochondrial dynamin assembly for membrane scission.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00458_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00458_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00458_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00458_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00458">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG00459<br>EOG812JNC<br>EOG81RN97<br>EOG8K6DK2<br>EOG8KPRTF<br>EOG8RV164<br>EOG8SBCFX<br>EOG8T1G2H<br><br>sce:absent<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ADP binding, catalytic activity, microtubule motor activity, nucleoside-triphosphatase activity, nucleotide binding activity, role in nucleoside metabolic process and kinesin complex localization|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted ADP binding, catalytic activity and role in defense response, nucleoside metabolic process|Ortholog of A. fumigatus Af293 : Afu6g09450, Neosartorya fischeri NRRL 181 : NFIA_112440, Aspergillus wentii : Aspwe1_0731886 and Aspergillus versicolor : Aspve1_0056875, Aspve1_0755767|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted ADP binding, catalytic activity, microtubule motor activity, nucleoside-triphosphatase activity, nucleotide binding activity, role in nucleoside metabolic process and kinesin complex localization|Has domain(s) with predicted ADP binding, catalytic activity and role in defense response, nucleoside metabolic process|Ortholog of A. oryzae RIB40 : AO090038000491, Aspergillus wentii : Aspwe1_0036466, Aspergillus clavatus NRRL 1 : ACLA_041520 and Aspergillus niger ATCC 1015 : 55214-mRNA|Has domain(s) with predicted ADP binding, catalytic activity and role in defense response, nucleoside metabolic process|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00459_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00459_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00459_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00459_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00459">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9e6c3" class="col">
			FOG00460<br>EOG870RZR<br>EOG8KPRTF<br><br>sce:absent<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ADP binding, catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00460_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00460_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00460_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00460_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00460">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dbc4e1" class="col">
			FOG00461<br>EOG81RN97<br>EOG8W6MBB<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>F-box protein Pof11<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN2861, A. fumigatus Af293 : Afu3g11870, A. oryzae RIB40 : AO090003000750, Aspergillus wentii : Aspwe1_0105255 and Aspergillus sydowii : Aspsy1_0042742<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15147268" target=_blank>Lehmann A, <i>et al.</i> (2004 May)</a>. Molecular interactions of fission yeast Skp1 and its role in the DNA damage checkpoint.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16303567" target=_blank>Martín-Castellanos C, <i>et al.</i> (2005 Nov 22)</a>. A large-scale screen in S. pombe identifies seven novel genes required for critical meiotic events.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18931302" target=_blank>Dixon SJ, <i>et al.</i> (2008 Oct 28)</a>. Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22142781" target=_blank>Colabardini AC, <i>et al.</i> (2012 Feb)</a>. Molecular characterization of the Aspergillus nidulans fbxA encoding an F-box protein involved in xylanase induction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23535663" target=_blank>Wu S, <i>et al.</i> (2013)</a>. CAND1 controls in vivo dynamics of the cullin 1-RING ubiquitin ligase repertoire.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00461_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00461_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00461_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00461_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00461">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fad7ad" class="col">
			FOG00462<br>EOG8W6MBB<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN6217, A. fumigatus Af293 : Afu2g12060, A. oryzae RIB40 : AO090026000440, Aspergillus wentii : Aspwe1_0053972 and Aspergillus sydowii : Aspsy1_0152133<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00462_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00462_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00462_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00462_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00462">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb9" class="col">
			FOG00463<br>EOG81RN97<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Ortholog of A. niger CBS 513.88 : An08g12200, Aspergillus flavus NRRL 3357 : AFL2T_08825, Neosartorya fischeri NRRL 181 : NFIA_005170 and Aspergillus wentii : Aspwe1_0161656|Ortholog of A. nidulans FGSC A4 : AN8468, AN2021, A. fumigatus Af293 : Afu7g00550 and A. niger CBS 513.88 : An15g05630, An11g03460, An11g07530, An09g00640|Ortholog of A. nidulans FGSC A4 : AN8468, AN2021, A. fumigatus Af293 : Afu7g00550 and A. niger CBS 513.88 : An15g05630, An11g03460, An11g07530, An04g03850|Ortholog of A. nidulans FGSC A4 : AN8468, AN2021, A. fumigatus Af293 : Afu7g00550 and A. niger CBS 513.88 : An15g05630, An11g03460, An04g03850, An09g00640<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00463_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00463_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00463_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00463_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00463">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dfc9ba" class="col">
			FOG00464<br>EOG81RN97<br>EOG8NCJTH<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN8468, AN2021, A. fumigatus Af293 : Afu7g00550 and A. niger CBS 513.88 : An15g05630, An11g07530, An04g03850, An09g00640<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00464_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00464_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00464_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00464_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00464">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8d4ea" class="col">
			FOG00465<br>EOG8T1G2H<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Ortholog of A. niger CBS 513.88 : An01g01380, A. oryzae RIB40 : AO090038000267, Aspergillus niger ATCC 1015 : 36409-mRNA, Aspergillus zonatus : Aspzo1_0017959 and Aspergillus terreus NIH2624 : ATET_06412|Ortholog of Aspergillus tubingensis : Asptu1_0061269, Aspergillus kawachii : Aspka1_0181383 and Aspergillus acidus : Aspfo1_0064915<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00465_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00465_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00465_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00465_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00465">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eeadad" class="col">
			FOG00466<br>EOG8BVQBN<br>EOG8KPRTF<br>EOG8WWQ6N<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function|Ortholog of Aspergillus niger ATCC 1015 : 52980-mRNA and Aspergillus aculeatus ATCC16872 : Aacu16872_029599|Has domain(s) with predicted cysteine-type endopeptidase activity and role in proteolysis|Ortholog of A. oryzae RIB40 : AO090012000390, AO090009000464, Aspergillus brasiliensis : Aspbr1_0130524, Aspergillus flavus NRRL 3357 : AFL2T_03297 and Neosartorya fischeri NRRL 181 : NFIA_006040|Protein of unknown function<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00466_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00466_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00466_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00466_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00466">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bacde4" class="col">
			FOG00467<br>EOG847D82<br>EOG8KKWHW<br>EOG8KPRTF<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN10921, A. fumigatus Af293 : Afu3g07460, Neosartorya fischeri NRRL 181 : NFIA_069710 and Aspergillus niger ATCC 1015 : 119074-mRNA<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00467_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00467_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00467_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00467_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00467">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c1e3bb" class="col">
			FOG00468<br>EOG8T1G2H<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00468_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00468_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00468_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00468_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00468">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7bcdd" class="col">
			FOG00469<br>EOG8W6MBB<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular response to light stimulus, protein ubiquitination, regulation of circadian rhythm and SCF ubiquitin ligase complex, cytosol localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00469_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00469_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00469_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00469_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00469">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d2a2" class="col">
			FOG00470<br>EOG8KPRTF<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00470_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00470_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00470_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00470_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00470">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb0" class="col">
			FOG00471<br>EOG8KPRTF<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function|Ortholog of Aspergillus tubingensis : Asptu1_0120187<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00471_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00471_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00471_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00471_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00471">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dac1b0" class="col">
			FOG00472<br>EOG84J10K<br>EOG8K6DK2<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ADP binding, catalytic activity and role in defense response, nucleoside metabolic process|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00472_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00472_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00472_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00472_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00472">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cee7" class="col">
			FOG00473<br>EOG8T1G2H<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of Aspergillus tubingensis : Asptu1_0109568 and Aspergillus kawachii : Aspka1_0176829|Ortholog of A. nidulans FGSC A4 : AN5168, AN1130, A. fumigatus Af293 : Afu6g07030, Afu4g13872, A. niger CBS 513.88 : An07g01930 and A. oryzae RIB40 : AO090701000916<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00473_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00473_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00473_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00473_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00473">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eca3a4" class="col">
			FOG00474<br>EOG8J0ZX2<br>EOG8T1G2H<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00474_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00474_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00474_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00474_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00474">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b2c7e1" class="col">
			FOG00475<br>EOG8T1G2H<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00475_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00475_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00475_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00475_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00475">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bae0b3" class="col">
			FOG00476<br>EOG8K6DK2<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00476_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00476_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00476_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00476_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00476">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d2b4d9" class="col">
			FOG00477<br>EOG8KPRTF<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00477_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00477_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00477_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00477_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00477">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cc97" class="col">
			FOG00478<br>EOG870RZR<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. niger CBS 513.88 : An16g06750, A. oryzae RIB40 : AO090120000239, Aspergillus wentii : Aspwe1_0027728 and Aspergillus versicolor : Aspve1_0055646, Aspve1_0145949<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00478_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00478_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00478_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00478_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00478">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcffa6" class="col">
			FOG00479<br>EOG8T1G2H<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00479_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00479_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00479_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00479_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00479">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d6baa7" class="col">
			FOG00480<br>EOG8W6MBB<br><br>sce:CAF4;RRT13<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>WD40 repeat-containing protein associated with the CCR4-NOT complex; interacts in a Ccr4p-dependent manner with Ssn2p; also interacts with Fis1p, Mdv1p and Dnm1p and plays a role in mitochondrial fission; CAF4 has a paralog, MDV1, that arose from the whole genome duplication|Putative protein of unknown function; non-essential gene identified in a screen for mutants with decreased levels of rDNA transcription<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11113136" target=_blank>Liu HY, <i>et al.</i> (2001 Mar 9)</a>. Characterization of CAF4 and CAF16 reveals a functional connection between the CCR4-NOT complex and a subset of SRB proteins of the RNA polymerase II holoenzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16009724" target=_blank>Griffin EE, <i>et al.</i> (2005 Jul 18)</a>. The WD40 protein Caf4p is a component of the mitochondrial fission machinery and recruits Dnm1p to mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16835275" target=_blank>Schauss AC, <i>et al.</i> (2006 Aug 1)</a>. Fis1p and Caf4p, but not Mdv1p, determine the polar localization of Dnm1p clusters on the mitochondrial surface.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17998537" target=_blank>Zhang Y, <i>et al.</i> (2007 Nov 20)</a>. Structural basis for recruitment of mitochondrial fission complexes by Fis1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19270272" target=_blank>Hontz RD, <i>et al.</i> (2009 May)</a>. Genetic identification of factors that modulate ribosomal DNA transcription in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00480_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00480_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00480_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00480_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00480">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f7c8e5" class="col">
			FOG00481<br>EOG8KPRTF<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted microtubule motor activity and kinesin complex localization|Has domain(s) with predicted ADP binding activity|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted microtubule motor activity and kinesin complex localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00481_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00481_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00481_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00481_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00481">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ea999a" class="col">
			FOG00482<br>EOG8K6DK2<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process|Has domain(s) with predicted catalytic activity and role in nucleoside metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00482_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00482_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00482_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00482_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00482">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#aac1de" class="col">
			FOG00483<br>EOG8KKWHW<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00483_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00483_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00483_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00483_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00483">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b3dcab" class="col">
			FOG00484<br>EOG8RV164<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00484_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00484_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00484_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00484_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00484">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cdacd5" class="col">
			FOG00485<br>EOG81RN97<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>transcription factor TFIID complex subunit Taf5-like<br><br>
			<br><b>AspGD Description</b><br>Protein of unknown function|Ortholog of Aspergillus tubingensis : Asptu1_0122161, Aspergillus kawachii : Aspka1_0174991, Aspergillus acidus : Aspfo1_0162529 and Aspergillus niger ATCC 1015 : 41351-mRNA|Ortholog of A. nidulans FGSC A4 : AN8468, AN2021, A. fumigatus Af293 : Afu7g00550 and A. niger CBS 513.88 : An11g03460, An11g07530, An04g03850, An09g00640<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00485_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00485_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00485_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00485_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00485">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8c78c" class="col">
			FOG00486<br>EOG812JNC<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted nucleoside-triphosphatase activity, nucleotide binding activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00486_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00486_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00486_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00486_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00234_phobius.html#FOG00486">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG00234.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>