<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG05095.html">HOG05095&emsp;&emsp;PIH1 family</a><br> 
																	HOG05100&emsp;&emsp;<br> 
																	<a href="HOG05101.html">HOG05101&emsp;&emsp;ECM29 family</a><br><br> 
																	<a href="../sequences/hog/HOG05100.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG05100_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG05100_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG05100_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG05100_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG09313<br>EOG8WM38T<br><br>sce:PAT1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Deadenylation-dependent mRNA-decapping factor; also required for faithful chromosome transmission, maintenance of rDNA locus stability, and protection of mRNA 3'-UTRs from trimming; associated with topoisomerase II; binds to mRNAs under glucose starvation, most often in the 3' UTR; functionally linked to Pab1p; forms cytoplasmic foci upon DNA replication stress; phosphorylation by PKA inhibits P body foci formation<br><br>
			<br><b>PomBase Description</b><br>topoisomerase II-associated deadenylation-dependent mRNA-decapping factor Pdc2 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytoplasmic mRNA processing body, cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7645349" target=_blank>Rodriguez-Cousino N, <i>et al.</i> (1995 May)</a>. Identification and initial characterization of the cytosolic protein Ycr77p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8972867" target=_blank>Wang X, <i>et al.</i> (1996 Dec 1)</a>. Pat1: a topoisomerase II-associated protein required for faithful chromosome transmission in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8816497" target=_blank>Hatfield L, <i>et al.</i> (1996 Oct)</a>. Mutations in trans-acting factors affecting mRNA decapping in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394921" target=_blank>Wang X, <i>et al.</i> (1999 Jun)</a>. The topoisomerase II-associated protein, Pat1p, is required for maintenance of rDNA locus stability in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10523645" target=_blank>Zhang S, <i>et al.</i> (1999 Nov)</a>. Mutations in VPS16 and MRT1 stabilize mRNAs by activating an inhibitor of the decapping enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10747033" target=_blank>Bouveret E, <i>et al.</i> (2000 Apr 3)</a>. A Sm-like protein complex that participates in mRNA degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913177" target=_blank>Bonnerot C, <i>et al.</i> (2000 Aug)</a>. The two proteins Pat1p (Mrt1p) and Spb8p interact in vivo, are required for mRNA decay, and are functionally linked to Pab1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10761922" target=_blank>Tharun S, <i>et al.</i> (2000 Mar 30)</a>. Yeast Sm-like proteins function in mRNA decapping and decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10779343" target=_blank>Wyers F, <i>et al.</i> (2000 May)</a>. Deletion of the PAT1 gene affects translation initiation and suppresses a PAB1 gene deletion in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11027264" target=_blank>Schwartz DC, <i>et al.</i> (2000 Nov)</a>. mRNA decapping in yeast requires dissociation of the cap binding protein, eukaryotic translation initiation factor 4E.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11514438" target=_blank>He W, <i>et al.</i> (2001 Aug)</a>. The yeast cytoplasmic LsmI/Pat1p complex protects mRNA 3' termini from partial degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11780629" target=_blank>Coller JM, <i>et al.</i> (2001 Dec)</a>. The DEAD box helicase, Dhh1p, functions in mRNA decapping and interacts with both the decapping and deadenylase complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741542" target=_blank>Tharun S, <i>et al.</i> (2001 Nov)</a>. Targeting an mRNA for decapping: displacement of translation factors and association of the Lsm1p-7p complex on deadenylated yeast mRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12032091" target=_blank>Fischer N, <i>et al.</i> (2002 Jun 3)</a>. The DEAD box protein Dhh1 stimulates the decapping enzyme Dcp1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773554" target=_blank>Noueiry AO, <i>et al.</i> (2003 Jun)</a>. Yeast Lsm1p-7p/Pat1p deadenylation-dependent mRNA-decapping factors are required for brome mosaic virus genomic RNA translation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730603" target=_blank>Sheth U, <i>et al.</i> (2003 May 2)</a>. Decapping and decay of messenger RNA occur in cytoplasmic processing bodies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15703442" target=_blank>Teixeira D, <i>et al.</i> (2005 Apr)</a>. Processing bodies require RNA for assembly and contain nontranslating mRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16357218" target=_blank>Lotan R, <i>et al.</i> (2005 Dec 15)</a>. The RNA polymerase II subunit Rpb4p mediates decay of a specific class of mRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16179257" target=_blank>Coller J, <i>et al.</i> (2005 Sep 23)</a>. General translational repression by activators of mRNA decapping.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16777600" target=_blank>Sheth U, <i>et al.</i> (2006 Jun 16)</a>. Targeting of aberrant mRNAs to cytoplasmic processing bodies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513695" target=_blank>Chowdhury A, <i>et al.</i> (2007 Jul)</a>. The decapping activator Lsm1p-7p-Pat1p complex has the intrinsic ability to distinguish between oligoadenylated and polyadenylated RNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17429074" target=_blank>Teixeira D, <i>et al.</i> (2007 Jun)</a>. Analysis of P-body assembly in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17875743" target=_blank>Lotan R, <i>et al.</i> (2007 Sep 24)</a>. The Rpb7p subunit of yeast RNA polymerase II plays roles in the two major cytoplasmic mRNA decay mechanisms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19111170" target=_blank>Tharun S, <i>et al.</i> (2008)</a>. Purification and analysis of the decapping activator Lsm1p-7p-Pat1p complex from yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18086885" target=_blank>Pilkington GR, <i>et al.</i> (2008 Feb)</a>. Pat1 contains distinct functional domains that promote P-body assembly and activation of decapping.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233300" target=_blank>Morozov IY, <i>et al.</i> (2010 Apr)</a>. Distinct roles for Caf1, Ccr4, Edc3 and CutA in the co-ordination of transcript deadenylation, decapping and P-body formation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19901074" target=_blank>Checkley MA, <i>et al.</i> (2010 Jan)</a>. P-body components are required for Ty1 retrotransposition during assembly of retrotransposition-competent virus-like particles.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20832728" target=_blank>Nissan T, <i>et al.</i> (2010 Sep 10)</a>. Decapping activators in Saccharomyces cerevisiae act by multiple mechanisms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24139796" target=_blank>Sharif H, <i>et al.</i> (2013 Oct 31)</a>. Architecture of the Lsm1-7-Pat1 complex: a conserved assembly in eukaryotic mRNA turnover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24247251" target=_blank>Wu D, <i>et al.</i> (2014 Feb)</a>. Lsm2 and Lsm3 bridge the interaction of the Lsm1-7 complex with Pat1 for decapping activation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG09313_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG09313_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG09313_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG09313_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG05100_phobius.html#FOG09313">Phobius transmembrane predictions</a>
										<br>18 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG05100.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>