<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00994<br>EOG8GQNMB<br>LEU9<br>sce:LEU9<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial alpha-isopropylmalate synthase II<br><br>
			<br><b>Features</b><br>[m]<br><br>
			<br><b>SGD Description</b><br>Alpha-isopropylmalate synthase II (2-isopropylmalate synthase); catalyzes the first step in the leucine biosynthesis pathway; the minor isozyme, responsible for the residual alpha-IPMS activity detected in a leu4 null mutant; LEU9 has a paralog, LEU4, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>2-isopropylmalate synthase Leu3<br><br>
			<br><b>AspGD Description</b><br>Alpha-isopropylmalate synthase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17248775" target=_blank>Kohli J, <i>et al.</i> (1977 Nov)</a>. Genetic Mapping in SCHIZOSACCHAROMYCES POMBE by Mitotic and Meiotic Analysis and Induced Haploidization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10790691" target=_blank>Casalone E, <i>et al.</i> (2000 Apr)</a>. Identification by functional analysis of the gene encoding alpha-isopropylmalate synthase II (LEU9) in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20657649" target=_blank>Larson EM, <i>et al.</i> (2010 Jul 15)</a>. Two origins for the gene encoding alpha-isopropylmalate synthase in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23671279" target=_blank>Tarumoto Y, <i>et al.</i> (2013 Jun 28)</a>. Receptor for activated C-kinase (RACK1) homolog Cpc2 facilitates the general amino acid control response through Gcn2 kinase in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00994_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00994_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00994_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00994_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00995<br>EOG8GQNMB<br><br>sce:absent<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cytoplasmic alpha-isopropylmalate synthase<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>AspGD Description</b><br>Alpha-isopropylmalate synthase<br><br>
			<br><b>Suggested Analysis</b><br>Explore if ang copy is a xenolog<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00995_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00995_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00995_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00995_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00996<br>EOG8GQNMB<br>LEU4<br>sce:LEU4<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial and cytoplasmic alpha-isopropylmalate synthase<br><br>
			<br><b>Features</b><br>[m];[c]<br><br>
			<br><b>Parent</b><br>ohnolog?:FOG00994<br><br>
			<br><b>SGD Description</b><br>Alpha-isopropylmalate synthase (2-isopropylmalate synthase); the main isozyme responsible for the first step in the leucine biosynthesis pathway; LEU4 has a paralog, LEU9, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2420798" target=_blank>Beltzer JP, <i>et al.</i> (1986 Apr 15)</a>. Structure of yeast LEU4. The 5' flanking region contains features that predict two modes of control and two productive translation starts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3275644" target=_blank>Beltzer JP, <i>et al.</i> (1988 Jan 5)</a>. Yeast LEU4 encodes mitochondrial and nonmitochondrial forms of alpha-isopropylmalate synthase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446029" target=_blank>Mountain HA, <i>et al.</i> (1993 Jan)</a>. The general amino acid control regulates MET4, which encodes a methionine-pathway-specific transcriptional activator of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00996_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00996_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00996_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00996_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00997<br>EOG8GQNMB<br>EOG8GQNNW<br><br>sce:CAR1<br><br>Genes: 37
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Arginase, catabolizes arginine to ornithine and urea; expression responds to both induction by arginine and nitrogen catabolite repression; disruption decreases production of carcinogen ethyl carbamate during wine fermentation and also enhances freeze tolerance<br><br>
			<br><b>PomBase Description</b><br>arginase Aru1|arginase Car1<br><br>
			<br><b>AspGD Description</b><br>Arginase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5764741" target=_blank>Piotrowska M, <i>et al.</i> (1969 Feb)</a>. Mutants of the arginine-proline pathway in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4576496" target=_blank>Cybis J, <i>et al.</i> (1972 Oct)</a>. Arginase induction in Aspergillus nidulans. The appearance and decay of the coding capacity of messenger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4591372" target=_blank>Bartnik E, <i>et al.</i> (1973 Oct 16)</a>. Ammonium and glucose repression of the arginine catabolic enzymes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4603558" target=_blank>Wegleński P, <i>et al.</i> (1974)</a>. [Genetic regulation of arginine metabolism in fungi (author's transl)].<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=325361" target=_blank>Arst HN Jr, <i>et al.</i> (1977 Feb 28)</a>. Some genetical aspects of ornithine metabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6094498" target=_blank>Sumrada RA, <i>et al.</i> (1984 Dec)</a>. Nucleotide sequence of the Saccharomyces cerevisiae arginase gene (CAR1) and its transcription under various physiological conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2983306" target=_blank>Sumrada RA, <i>et al.</i> (1985 Feb)</a>. Point mutation generates constitutive expression of an inducible eukaryotic gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1939179" target=_blank>Green SM, <i>et al.</i> (1991 Nov 15)</a>. Roles of metal ions in the maintenance of the tertiary and quaternary structure of arginase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7732765" target=_blank>Dzikowska A, <i>et al.</i> (1994)</a>. Purification of arginase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8625426" target=_blank>Clement DJ, <i>et al.</i> (1996 Apr)</a>. Evidence for sltA1 as a salt-sensitive allele of the arginase gene (agaA) in the ascomycete Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10547040" target=_blank>Borsuk P, <i>et al.</i> (1999)</a>. Structure of the arginase coding gene and its transcript in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11810230" target=_blank>Empel J, <i>et al.</i> (2001 Dec)</a>. arcA, the regulatory gene for the arginine catabolic pathway in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17719249" target=_blank>Olszewska A, <i>et al.</i> (2007 Dec)</a>. Arginine catabolism in Aspergillus nidulans is regulated by the rrmA gene coding for the RNA-binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17261076" target=_blank>Borsuk P, <i>et al.</i> (2007 Feb)</a>. L-arginine influences the structure and function of arginase mRNA in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22300944" target=_blank>Macios M, <i>et al.</i> (2012 Mar)</a>. The GATA factors AREA and AREB together with the co-repressor NMRA, negatively regulate arginine catabolism in Aspergillus nidulans in response to nitrogen and carbon source.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23841692" target=_blank>Krol K, <i>et al.</i> (2013 Sep)</a>. RrmA regulates the stability of specific transcripts in response to both nitrogen source and oxidative stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24912400" target=_blank>Romagnoli G, <i>et al.</i> (2014 Jul)</a>. An alternative, arginase-independent pathway for arginine metabolism in Kluyveromyces lactis involves guanidinobutyrase as a key enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26210809" target=_blank>Dzikowska A, <i>et al.</i> (2015 Dec 1)</a>. KAEA (SUDPRO), a member of the ubiquitous KEOPS/EKC protein complex, regulates the arginine catabolic pathway and the expression of several other genes in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00997_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00997_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00997_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00997_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00998<br>EOG83BK4N<br>EOG873N7J<br>EOG8GQNMB<br>EOG8WWQ0K<br><br>sce:SUC2<br><br>Genes: 20
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Invertase; sucrose hydrolyzing enzyme; a secreted, glycosylated form is regulated by glucose repression, and an intracellular, nonglycosylated enzyme is produced constitutively<br><br>
			<br><b>PomBase Description</b><br>beta-fructofuranosidase (predicted)|external invertase, beta-fructofuranosidase<br><br>
			<br><b>AspGD Description</b><br>Extracellular exo-inulinase; highly expressed on maltose; induced by inulin or sucrose and repressed by fructose or glucose<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7039847" target=_blank>Carlson M, <i>et al.</i> (1982 Jan)</a>. Two differentially regulated mRNAs with different 5' ends encode secreted with intracellular forms of yeast invertase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6341817" target=_blank>Carlson M, <i>et al.</i> (1983 Mar)</a>. The secreted form of invertase in Saccharomyces cerevisiae is synthesized from mRNA encoding a signal sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6300785" target=_blank>Taussig R, <i>et al.</i> (1983 Mar 25)</a>. Nucleotide sequence of the yeast SUC2 gene for invertase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6396505" target=_blank>Sarokin L, <i>et al.</i> (1984 Dec)</a>. Upstream region required for regulated expression of the glucose-repressible SUC2 gene of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537718" target=_blank>Kaiser CA, <i>et al.</i> (1986 Jul)</a>. Secretion-defective mutations in the signal sequence for Saccharomyces cerevisiae invertase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3284881" target=_blank>Reddy VA, <i>et al.</i> (1988 May 25)</a>. Characterization of the glycosylation sites in yeast external invertase. I. N-linked oligosaccharide content of the individual sequons.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2113524" target=_blank>Reddy VA, <i>et al.</i> (1990 Jul 5)</a>. Identification of an active-site residue in yeast invertase by affinity labeling and site-directed mutagenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2112088" target=_blank>Hengst L, <i>et al.</i> (1990 Jun)</a>. The ryh1 gene in the fission yeast Schizosaccharomyces pombe encoding a GTP-binding protein related to ras, rho and ypt: structure, expression and identification of its human homologue.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187435" target=_blank>Moreno S, <i>et al.</i> (1990 May 1)</a>. Purification and characterization of the invertase from Schizosaccharomyces pombe. A comparative analysis with the invertase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1888528" target=_blank>Xie KW, <i>et al.</i> (1991)</a>. [The effects of upstream region of SUC2 gene on its expression].<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9535817" target=_blank>Tanaka N, <i>et al.</i> (1998 Apr 7)</a>. Isolation and characterization of an invertase and its repressor genes from Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394924" target=_blank>Georis I, <i>et al.</i> (1999 Jun)</a>. Glucose repression of the Kluyveromyces lactis invertase gene KlINV1 does not require Mig1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136464" target=_blank>Reinders A, <i>et al.</i> (2001 Jan)</a>. Functional characterization of the alpha-glucoside transporter Sut1p from Schizosaccharomyces pombe, the first fungal homologue of plant sucrose transporters.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16049679" target=_blank>Donoso I, <i>et al.</i> (2005 Sep)</a>. Mpg1, a fission yeast protein required for proper septum structure, is involved in cell cycle progression through cell-size checkpoint.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16624901" target=_blank>Nakase Y, <i>et al.</i> (2006 Jun)</a>. A defect in protein farnesylation suppresses a loss of Schizosaccharomyces pombe tsc2+, a homolog of the human gene predisposing to tuberous sclerosis complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17111131" target=_blank>Gong F, <i>et al.</i> (2007 Mar)</a>. Inulinase production by a marine yeast Pichia guilliermondii and inulin hydrolysis by the crude inulinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19107534" target=_blank>Guo N, <i>et al.</i> (2009 Apr)</a>. Enhanced inulinase production in solid state fermentation by a mutant of the marine yeast Pichia guilliermondii using surface response methodology and inulin hydrolysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19620394" target=_blank>Murai T, <i>et al.</i> (2009 Oct)</a>. Distinctive responses to nitrogen starvation in the dominant active mutants of the fission yeast Rheb GTPase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22416758" target=_blank>Muñoz-Centeno MC, <i>et al.</i> (2012 Aug)</a>. Mpg2 interacts and cooperates with Mpg1 to maintain yeast glycosylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22140232" target=_blank>Matsuzawa T, <i>et al.</i> (2012 Feb)</a>. Snf1-like protein kinase Ssp2 regulates glucose derepression in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27481777" target=_blank>Vishwanatha A, <i>et al.</i> (2016 Sep)</a>. Identification of nuclear genes affecting 2-Deoxyglucose resistance in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00998_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00998_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00998_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00998_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00999<br>EOG851C7V<br>EOG8GQNMB<br>BTS1<br>sce:BTS1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Geranylgeranyl diphosphate synthase (GGPS); increases the intracellular pool of geranylgeranyl diphosphate, suppressor of bet2 mutation that causes defective geranylgeranylation of small GTP-binding proteins that mediate vesicular traffic<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in isoprenoid biosynthetic process|Ortholog(s) have farnesyltranstransferase activity, role in terpenoid biosynthetic process and mitochondrion localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665600" target=_blank>Jiang Y, <i>et al.</i> (1995 Sep 15)</a>. BTS1 encodes a geranylgeranyl diphosphate synthase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15296494" target=_blank>Shiflett SL, <i>et al.</i> (2004 Sep)</a>. Bph1p, the Saccharomyces cerevisiae homologue of CHS1/beige, functions in cell wall formation and protein sorting.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16554305" target=_blank>Chang TH, <i>et al.</i> (2006 May 26)</a>. Crystal structure of type-III geranylgeranyl pyrophosphate synthase from Saccharomyces cerevisiae and the mechanism of product chain length determination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17535895" target=_blank>Guo RT, <i>et al.</i> (2007 Jun 12)</a>. Bisphosphonates target multiple sites in both cis- and trans-prenyltransferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245203" target=_blank>Lo CH, <i>et al.</i> (2009 Mar 25)</a>. Combined experimental and theoretical study of long-range interactions modulating dimerization and activity of yeast geranylgeranyl diphosphate synthase.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00999_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00999_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00999_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00999_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG01000<br>EOG851C7V<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in isoprenoid biosynthetic process|Has domain(s) with predicted role in isoprenoid biosynthetic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01000_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01000_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01000_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01000_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG01001<br>EOG87H454<br>EOG8GQNMB<br><br>sce:CAT2<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Carnitine acetyl-CoA transferase; present in both mitochondria and peroxisomes; transfers activated acetyl groups to carnitine to form acetylcarnitine which can be shuttled across membranes<br><br>
			<br><b>AspGD Description</b><br>Mitochondrial carnitine acetyltransferase; induced by fenpropimorph<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=778333" target=_blank>Payton M, <i>et al.</i> (1976 May)</a>. Agar as a carbon source and its effect on the utilization of other carbon sources by acetate non-utilizing (acu) mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8420957" target=_blank>Kispal G, <i>et al.</i> (1993 Jan 25)</a>. Cloning and sequencing of a cDNA encoding Saccharomyces cerevisiae carnitine acetyltransferase. Use of the cDNA in gene disruption studies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8472927" target=_blank>Midgley M, <i>et al.</i> (1993 Mar 15)</a>. Carnitine acetyltransferase is absent from acuJ mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7628448" target=_blank>Elgersma Y, <i>et al.</i> (1995 Jul 17)</a>. Peroxisomal and mitochondrial carnitine acetyltransferases of Saccharomyces cerevisiae are encoded by a single gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9385142" target=_blank>De Lucas JR, <i>et al.</i> (1997 Dec)</a>. Characterization of oleate-nonutilizing mutants of Aspergillus nidulans isolated by the 3-amino-1,2,4-triazole positive selection method.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829933" target=_blank>Stemple CJ, <i>et al.</i> (1998 Dec)</a>. The facC gene of Aspergillus nidulans encodes an acetate-inducible carnitine acetyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741859" target=_blank>Hynes MJ, <i>et al.</i> (2002 Jan)</a>. Regulation of the acuF gene, encoding phosphoenolpyruvate carboxykinase in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16682457" target=_blank>Hynes MJ, <i>et al.</i> (2006 May)</a>. Regulatory genes controlling fatty acid catabolism and peroxisomal functions in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20495057" target=_blank>Hynes MJ, <i>et al.</i> (2010 Jul)</a>. ATP-citrate lyase is required for production of cytosolic acetyl coenzyme A and development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21296915" target=_blank>Hynes MJ, <i>et al.</i> (2011 Apr)</a>. Role of carnitine acetyltransferases in acetyl coenzyme A metabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01001_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01001_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01001_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01001_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG01002<br>EOG8CVDP0<br>EOG8GQNMB<br><br>sce:ALA1<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic and mitochondrial alanyl-tRNA synthetase; required for protein synthesis; point mutation (cdc64-1 allele) causes cell cycle arrest at G1; lethality of null mutation is functionally complemented by human homolog AARS; mutations in human homolog AARS are associated with autoimmune disease polymyositis/dermatomyositis<br><br>
			<br><b>PomBase Description</b><br>mitochondrial and cytoplasmic alanine-tRNA ligase Ala1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have alanine-tRNA ligase activity, role in alanyl-tRNA aminoacylation and cytosol, mitochondrion localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7761427" target=_blank>Ripmaster TL, <i>et al.</i> (1995 May 23)</a>. Wide cross-species aminoacyl-tRNA synthetase replacement in vivo: yeast cytoplasmic alanine enzyme replaced by human polymyositis serum antigen.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15358761" target=_blank>Tang HL, <i>et al.</i> (2004 Nov 26)</a>. Translation of a yeast mitochondrial tRNA synthetase initiated at redundant non-AUG codons.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556230" target=_blank>Huang HY, <i>et al.</i> (2006 Apr)</a>. An unusual pattern of protein expression and localization of yeast alanyl-tRNA synthetase isoforms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16928688" target=_blank>Huang HY, <i>et al.</i> (2006 Oct 20)</a>. Cross-species and cross-compartmental aminoacylation of isoaccepting tRNAs by a class II tRNA synthetase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603077" target=_blank>Roberts-Galbraith RH, <i>et al.</i> (2010 Jul 9)</a>. Dephosphorylation of F-BAR protein Cdc15 modulates its conformation and stimulates its scaffolding activity at the cell division site.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22119525" target=_blank>Singh NS, <i>et al.</i> (2011 Dec 6)</a>. SIN-inhibitory phosphatase complex promotes Cdc11p dephosphorylation and propagates SIN asymmetry in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23293578" target=_blank>Azzedine H, <i>et al.</i> (2012 Nov)</a>. Molecular genetics of charcot-marie-tooth disease: from genes to genomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01002_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01002_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01002_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01002_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG01003<br>EOG8GQNMB<br>EOG8J9KDS<br><br>sce:VAS1<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mitochondrial and cytoplasmic valyl-tRNA synthetase<br><br>
			<br><b>PomBase Description</b><br>cytoplasmic valine-tRNA ligase Vrs1/Vas1|mitochondrial valine-tRNA ligase Vrs2/Vas2<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, aminoacyl-tRNA editing activity, aminoacyl-tRNA ligase activity, valine-tRNA ligase activity and role in tRNA aminoacylation for protein translation, valyl-tRNA aminoacylation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294828" target=_blank>Jordana X, <i>et al.</i> (1987 May 25)</a>. Structure of the yeast valyl-tRNA synthetase gene (VASI) and the homology of its translated amino acid sequence with Escherichia coli isoleucyl-tRNA synthetase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3275649" target=_blank>Chatton B, <i>et al.</i> (1988 Jan 5)</a>. The yeast VAS1 gene encodes both mitochondrial and cytoplasmic valyl-tRNA synthetases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20106903" target=_blank>Chiu WC, <i>et al.</i> (2010 Jun)</a>. Schizosaccharomyces pombe possesses two paralogous valyl-tRNA synthetase genes of mitochondrial origin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24634168" target=_blank>Bernal M, <i>et al.</i> (2014 Jun)</a>. Proteome-wide search for PP2A substrates in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01003_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01003_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01003_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01003_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>10 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG01004<br>EOG8GQNMB<br>EOG8MGQPX<br><br>sce:GRS1<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic and mitochondrial glycyl-tRNA synthase; ligates glycine to the cognate anticodon-bearing tRNA; transcription termination factor that may interact with the 3'-end of pre-mRNA to promote 3'-end formation; GRS1 has a paralog, GRS2, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>mitochondrial and cytoplasmic glycine-tRNA ligase Grs1<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, aminoacyl-tRNA ligase activity, glycine-tRNA ligase activity, role in glycyl-tRNA aminoacylation, tRNA aminoacylation for protein translation and cytoplasm localization|Ortholog(s) have glycine-tRNA ligase activity, role in DNA-templated transcription, termination, mitochondrial glycyl-tRNA aminoacylation and cytosol, mitochondrion localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10874035" target=_blank>Turner RJ, <i>et al.</i> (2000 Sep 8)</a>. One of two genes encoding glycyl-tRNA synthetase in Saccharomyces cerevisiae provides mitochondrial and cytoplasmic functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14734560" target=_blank>Chang KJ, <i>et al.</i> (2004 Apr 2)</a>. Translation initiation from a naturally occurring non-AUG codon in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01004_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01004_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01004_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01004_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG01005<br>EOG8MGQPX<br><br>sce:GRS2<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Glycine-tRNA synthetase, not expressed under normal growth conditions; expression is induced under heat, oxidative, pH, or ethanol stress conditions; more stable than the major glycine-tRNA synthetase Grs1p at 37 deg C; GRS2 has a paralog, GRS1, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10874035" target=_blank>Turner RJ, <i>et al.</i> (2000 Sep 8)</a>. One of two genes encoding glycyl-tRNA synthetase in Saccharomyces cerevisiae provides mitochondrial and cytoplasmic functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01005_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01005_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01005_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01005_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG01006<br>EOG8GQNMB<br>EOG8JH9X7<br><br>sce:HTS1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic and mitochondrial histidine tRNA synthetase; efficient mitochondrial localization requires both a presequence and an amino-terminal sequence; mutations in human ortholog HARS2 are associated with Perrault syndrome<br><br>
			<br><b>PomBase Description</b><br>mitochondrial and cytoplasmic histidine-tRNA ligase Hrs1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3521891" target=_blank>Natsoulis G, <i>et al.</i> (1986 Jul 18)</a>. The HTS1 gene encodes both the cytoplasmic and mitochondrial histidine tRNA synthetases of S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1459448" target=_blank>Chiu MI, <i>et al.</i> (1992 Dec)</a>. HTS1 encodes both the cytoplasmic and mitochondrial histidyl-tRNA synthetase of Saccharomyces cerevisiae: mutations alter the specificity of compartmentation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01006_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01006_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01006_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01006_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG01007<br>EOG8CRJFX<br>EOG8GQNMB<br><br>sce:MOD5<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Delta 2-isopentenyl pyrophosphate:tRNA isopentenyl transferase; required for biosynthesis of isopentenyladenosine in mitochondrial and cytoplasmic tRNAs; also has a role in tRNA gene-mediated silencing; gene encodes two isozymic forms; converts to a prion form, prion conversion contributes to azole antifungal resistance by upregulating ergosterol biosynthesis; homolog of human TRIT1, a mutation in which is associated with severe combined respiratory chain defects<br><br>
			<br><b>PomBase Description</b><br>tRNA isopentenyltransferase Tit1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have tRNA binding, tRNA dimethylallyltransferase activity, role in tRNA modification, transfer RNA gene-mediated silencing and cytosol, mitochondrion, nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3031456" target=_blank>Dihanich ME, <i>et al.</i> (1987 Jan)</a>. Isolation and characterization of MOD5, a gene required for isopentenylation of cytoplasmic and mitochondrial tRNAs of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3031457" target=_blank>Najarian D, <i>et al.</i> (1987 Jan)</a>. DNA sequence and transcript mapping of MOD5: features of the 5' region which suggest two translational starts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1850093" target=_blank>Gillman EC, <i>et al.</i> (1991 May)</a>. MOD5 translation initiation sites determine N6-isopentenyladenosine modification of mitochondrial and cytoplasmic tRNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1946403" target=_blank>Slusher LB, <i>et al.</i> (1991 Nov 1)</a>. mRNA leader length and initiation codon context determine alternative AUG selection for the yeast gene MOD5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8139535" target=_blank>Boguta M, <i>et al.</i> (1994 Apr)</a>. Subcellular locations of MOD5 proteins: mapping of sequences sufficient for targeting to mitochondria and demonstration that mitochondrial and nuclear isoforms commingle in the cytosol.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8896271" target=_blank>Chéret G, <i>et al.</i> (1996 Sep)</a>. DNA sequence analysis of the VPH1-SNF2 region on chromosome XV of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10618371" target=_blank>Benko AL, <i>et al.</i> (2000 Jan 4)</a>. Competition between a sterol biosynthetic enzyme and tRNA modification in addition to changes in the protein synthesis machinery causes altered nonsense suppression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18852462" target=_blank>Zhou C, <i>et al.</i> (2008 Oct 21)</a>. Crystallographic snapshots of eukaryotic dimethylallyltransferase acting on tRNA: insight into tRNA recognition and reaction mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21873461" target=_blank>Lamichhane TN, <i>et al.</i> (2011 Oct)</a>. Plasticity and diversity of tRNA anticodon determinants of substrate recognition by eukaryotic A37 isopentenyltransferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22517861" target=_blank>Suzuki G, <i>et al.</i> (2012 Apr 20)</a>. A yeast prion, Mod5, promotes acquired drug resistance and cell survival under environmental stress.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01007_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01007_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01007_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01007_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG01008<br>EOG8GQNMB<br>EOG8NP5M1<br><br>sce:TRM1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>tRNA methyltransferase; two forms of protein are made by alternative translation starts; localizes to both nucleus and mitochondrion to produce modified base N2,N2-dimethylguanosine in tRNAs in both compartments; nuclear Trm1p is evenly distributed around inner nuclear membrane in WT, but mislocalizes as puncta near ER-nucleus junctions in spindle pole body (SPB) mutants; both Trm1p inner nuclear membrane targeting and maintenance depend upon SPB<br><br>
			<br><b>PomBase Description</b><br>tRNA (guanine-N2-)-methyltransferase Trm1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have tRNA (guanine-N2-)-methyltransferase activity, role in tRNA N2-guanine methylation and mitochondrion, nuclear inner membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3299379" target=_blank>Ellis SR, <i>et al.</i> (1987 Aug)</a>. Amino-terminal extension generated from an upstream AUG codon is not required for mitochondrial import of yeast N2,N2-dimethylguanosine-specific tRNA methyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7599275" target=_blank>Rose AM, <i>et al.</i> (1995)</a>. Location of N2,N2-dimethylguanosine-specific tRNA methyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9801306" target=_blank>Liu J, <i>et al.</i> (1998 Nov 15)</a>. Point and deletion mutations eliminate one or both methyl group transfers catalysed by the yeast TRM1 encoded tRNA (m22G26)dimethyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10611485" target=_blank>Niederberger C, <i>et al.</i> (1999 Dec 24)</a>. The tRNA N2,N2-dimethylguanosine-26 methyltransferase encoded by gene trm1 increases efficiency of suppression of an ochre codon in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048958" target=_blank>Liu J, <i>et al.</i> (1999 Jan 8)</a>. Caenorhabditis elegans ZC376.5 encodes a tRNA (m2/2G(26))dimethyltransferance in which (246)arginine is important for the enzyme activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15911569" target=_blank>Murthi A, <i>et al.</i> (2005 Aug)</a>. Genome-wide screen for inner nuclear membrane protein targeting in Saccharomyces cerevisiae: roles for N-acetylation and an integral membrane protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16987955" target=_blank>Osmani AH, <i>et al.</i> (2006 Dec)</a>. Systematic deletion and mitotic localization of the nuclear pore complex proteins of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19602197" target=_blank>Lai TP, <i>et al.</i> (2009 Sep)</a>. Mechanism and a peptide motif for targeting peripheral proteins to the yeast inner nuclear membrane.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26720005" target=_blank>Arimbasseri AG, <i>et al.</i> (2015 Dec)</a>. RNA Polymerase III Output Is Functionally Linked to tRNA Dimethyl-G26 Modification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26628015" target=_blank>Zhang X, <i>et al.</i> (2015 Oct 9)</a>. Characterization of Tamoxifen as an Antifungal Agent Using the Yeast Schizosaccharomyces Pombe Model Organism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28977649" target=_blank>Vakiloroayaei A, <i>et al.</i> (2017 Nov 2)</a>. The RNA chaperone La promotes pre-tRNA maturation via indiscriminate binding of both native and misfolded targets.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01008_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01008_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01008_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01008_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG01009<br>EOG8FJ6RJ<br>EOG8GQNMB<br><br>sce:KAR2;SSA1;SSA2;SSA3;SSA4;SSB1;SSB2;SSC1;ECM10;SSQ1<br><br>Genes: 190
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATPase involved in protein import into the ER; also acts as a chaperone to mediate protein folding in the ER and may play a role in ER export of soluble proteins; regulates the unfolded protein response via interaction with Ire1p|ATPase involved in protein folding and NLS-directed nuclear transport; member of HSP70 family; forms chaperone complex with Ydj1p; localized to nucleus, cytoplasm, and cell wall; 98% identical with paralog Ssa2p, but subtle differences between the two proteins provide functional specificity with respect to propagation of yeast [URE3] prions and vacuolar-mediated degradations of gluconeogenesis enzymes; general targeting factor of Hsp104p to prion fibrils|ATP-binding protein; involved in protein folding and vacuolar import of proteins; member of heat shock protein 70 (HSP70) family; associated with the chaperonin-containing T-complex; present in the cytoplasm, vacuolar membrane and cell wall; 98% identical with paralog Ssa1p, but subtle differences between the two proteins provide functional specificity with respect to propagation of yeast [URE3] prions and vacuolar-mediated degradations of gluconeogenesis enzymes|ATPase involved in protein folding and the response to stress; plays a role in SRP-dependent cotranslational protein-membrane targeting and translocation; member of the heat shock protein 70 (HSP70) family; localized to the cytoplasm; SSA3 has a paralog, SSA4, that arose from the whole genome duplication|Heat shock protein that is highly induced upon stress; plays a role in SRP-dependent cotranslational protein-membrane targeting and translocation; member of the HSP70 family; cytoplasmic protein that concentrates in nuclei upon starvation; SSA4 has a paralog, SSA3, that arose from the whole genome duplication|Cytoplasmic ATPase that is a ribosome-associated molecular chaperone; functions with J-protein partner Zuo1p; may be involved in folding of newly-made polypeptide chains; member of the HSP70 family; interacts with phosphatase subunit Reg1p; SSB1 has a paralog, SSB2, that arose from the whole genome duplication|Cytoplasmic ATPase that is a ribosome-associated molecular chaperone; functions with J-protein partner Zuo1p; may be involved in the folding of newly-synthesized polypeptide chains; member of the HSP70 family; SSB2 has a paralog, SSB1, that arose from the whole genome duplication|Hsp70 family ATPase; constituent of the import motor component of the Translocase of the Inner Mitochondrial membrane (TIM23 complex); involved in protein translocation and folding; subunit of SceI endonuclease; SSC1 has a paralog, ECM10, that arose from the whole genome duplication|Heat shock protein of the Hsp70 family; localized in mitochondrial nucleoids, plays a role in protein translocation, interacts with Mge1p in an ATP-dependent manner; overexpression induces extensive mitochondrial DNA aggregations; ECM10 has a paralog, SSC1, that arose from the whole genome duplication|Mitochondrial hsp70-type molecular chaperone; required for assembly of iron/sulfur clusters into proteins at a step after cluster synthesis, and for maturation of Yfh1p, which is a homolog of human frataxin implicated in Friedreich's ataxia<br><br>
			<br><b>PomBase Description</b><br>ER heat shock protein BiP|heat shock protein Ssa1 (predicted)|heat shock protein Ssa2|heat shock protein, ribosome associated molecular chaperone Sks2|mitochondrial heat shock protein Hsp70<br><br>
			<br><b>AspGD Description</b><br>Hsp70 protein; induced by growth on starch and lactate|DnaK-type molecular chaperone; ER chaperone; unfolded protein response (UPR) target gene; expression enhanced by maltose, by heat shock, DTT and by tunicamycin treatment|Putative mitochondrial heat shock protein; induced by growth on starch and lactate|DnaK-type molecular chaperone<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6096826" target=_blank>Ogden RC, <i>et al.</i> (1984 Dec 21)</a>. Transfer RNA splicing in Saccharomyces cerevisiae: defining the substrates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2644626" target=_blank>Slater MR, <i>et al.</i> (1989 Jan 25)</a>. The SSA1 and SSA2 genes of the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2674677" target=_blank>Craig EA, <i>et al.</i> (1989 Jul)</a>. SSC1, an essential member of the yeast HSP70 multigene family, encodes a mitochondrial protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2661019" target=_blank>Normington K, <i>et al.</i> (1989 Jun 30)</a>. S. cerevisiae encodes an essential protein homologous in sequence and function to mammalian BiP.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2661018" target=_blank>Rose MD, <i>et al.</i> (1989 Jun 30)</a>. KAR2, a karyogamy gene, is the yeast homolog of the mammalian BiP/GRP78 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265609" target=_blank>Scherer PE, <i>et al.</i> (1990 Dec)</a>. A precursor protein partly translocated into yeast mitochondria is bound to a 70 kd mitochondrial stress protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2105497" target=_blank>Nicholson RC, <i>et al.</i> (1990 Feb)</a>. An essential member of the HSP70 gene family of Saccharomyces cerevisiae is homologous to immunoglobulin heavy chain binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2188113" target=_blank>Boorstein WR, <i>et al.</i> (1990 Jun)</a>. Transcriptional regulation of SSA3, an HSP70 gene from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2243798" target=_blank>Lewis MJ, <i>et al.</i> (1990 Nov 11)</a>. The sequence of the Kluyveromyces lactis BiP gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2121731" target=_blank>Boorstein WR, <i>et al.</i> (1990 Nov 5)</a>. Structure and regulation of the SSA4 HSP70 gene of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1922071" target=_blank>Nakazawa N, <i>et al.</i> (1991 Nov)</a>. AAR2, a gene for splicing pre-mRNA of the MATa1 cistron in cell type control of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7941740" target=_blank>Clark MW, <i>et al.</i> (1994 Apr)</a>. Sequencing of chromosome I of Saccharomyces cerevisiae: analysis of the 42 kbp SPO7-CENI-CDC15 region.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7992509" target=_blank>Logghe M, <i>et al.</i> (1994 Aug)</a>. The two genes encoding yeast ribosomal protein S8 reside on different chromosomes, and are closely linked to the hsp70 stress protein genes SSA3 and SSA4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8151709" target=_blank>Boorstein WR, <i>et al.</i> (1994 Jan)</a>. Molecular evolution of the HSP70 multigene family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7625280" target=_blank>Shibata T, <i>et al.</i> (1995)</a>. Multi-site-specific endonucleases and the initiation of homologous genetic recombination in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7744244" target=_blank>Eroles P, <i>et al.</i> (1995 Apr 15)</a>. Cloning of a DNA fragment encoding part of a 70-kDa heat shock protein of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7558317" target=_blank>La Valle R, <i>et al.</i> (1995 Oct)</a>. Molecular cloning and expression of a 70-kilodalton heat shock protein of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8674973" target=_blank>Lee DW, <i>et al.</i> (1996 Apr 15)</a>. Quantitative analysis of gene expression in sexual structures of Aspergillus nidulans by sequencing of 3'-directed cDNA clones.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8757872" target=_blank>López-Ribot JL, <i>et al.</i> (1996 Aug)</a>. Evidence for presence in the cell wall of Candida albicans of a protein related to the hsp70 family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8935650" target=_blank>Norbeck J, <i>et al.</i> (1996 Mar 15)</a>. Protein expression during exponential growth in 0.7 M NaCl medium of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9298649" target=_blank>Garrels JI, <i>et al.</i> (1997 Aug)</a>. Proteome studies of Saccharomyces cerevisiae: identification and characterization of abundant proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9200817" target=_blank>Maneu V, <i>et al.</i> (1997 Jun 15)</a>. Molecular cloning of a Candida albicans gene (SSB1) coding for a protein related to the Hsp70 family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9178502" target=_blank>Lee IH, <i>et al.</i> (1997 May)</a>. Yarrowia lipolytica SRP54 homolog and translocation of Kar2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9660806" target=_blank>Knight SA, <i>et al.</i> (1998 Jul 17)</a>. Mt-Hsp70 homolog, Ssc2p, required for maturation of yeast frataxin and mitochondrial iron homeostasis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11150530" target=_blank>Baumann F, <i>et al.</i> (2000 Dec 29)</a>. Ecm10, a novel hsp70 homolog in the mitochondrial matrix of the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10835388" target=_blank>Osherov N, <i>et al.</i> (2000 Jun)</a>. Conidial germination in Aspergillus nidulans requires RAS signaling and protein synthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10779357" target=_blank>Voisine C, <i>et al.</i> (2000 May)</a>. Role of the mitochondrial Hsp70s, Ssc1 and Ssq1, in the maturation of Yfh1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279042" target=_blank>Horton LE, <i>et al.</i> (2001 Apr 27)</a>. The yeast hsp70 homologue Ssa is required for translation and interacts with Sis1 and Pab1 on translating ribosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11273703" target=_blank>Lutz T, <i>et al.</i> (2001 Mar 30)</a>. The mitochondrial proteins Ssq1 and Jac1 are required for the assembly of iron sulfur clusters in mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12032075" target=_blank>Strub A, <i>et al.</i> (2002 Jun 3)</a>. The Hsp70 peptide-binding domain determines the interaction of the ATPase domain with Tim44 in mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12872131" target=_blank>Peng J, <i>et al.</i> (2003 Aug)</a>. A proteomics approach to understanding protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12761219" target=_blank>Li XS, <i>et al.</i> (2003 Aug 1)</a>. Candida albicans Ssa1/2p is the cell envelope binding protein for human salivary histatin 5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12756240" target=_blank>Dutkiewicz R, <i>et al.</i> (2003 Aug 8)</a>. Ssq1, a mitochondrial Hsp70 involved in iron-sulfur (Fe/S) center biogenesis. Similarities to and differences from its bacterial counterpart.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581366" target=_blank>Saloheimo M, <i>et al.</i> (2003 Feb)</a>. Activation mechanisms of the HAC1-mediated unfolded protein response in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14517234" target=_blank>Mokranjac D, <i>et al.</i> (2003 Oct 1)</a>. Tim14, a novel key component of the import motor of the TIM23 protein translocase of mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12943685" target=_blank>Sakasegawa Y, <i>et al.</i> (2003 Sep 12)</a>. Ecm10p localizes in yeast mitochondrial nucleoids and its overexpression induces extensive mitochondrial DNA aggregations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15449589" target=_blank>Sims AH, <i>et al.</i> (2004 Aug)</a>. Combining transcriptome data with genomic and cDNA sequence alignments to make confident functional assignments for Aspergillus nidulans genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15123690" target=_blank>Dutkiewicz R, <i>et al.</i> (2004 Jul 9)</a>. Sequence-specific interaction between mitochondrial Fe-S scaffold protein Isu and Hsp70 Ssq1 is essential for their in vivo function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15378761" target=_blank>Pitarch A, <i>et al.</i> (2004 Oct)</a>. Proteomics-based identification of novel Candida albicans antigens for diagnosis of systemic candidiasis in patients with underlying hematological malignancies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15958384" target=_blank>Knieszner H, <i>et al.</i> (2005 Aug 12)</a>. Compensation for a defective interaction of the hsp70 ssq1 with the mitochondrial Fe-S cluster scaffold isu.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870366" target=_blank>Sims AH, <i>et al.</i> (2005 May)</a>. Transcriptome analysis of recombinant protein secretion by Aspergillus nidulans and the unfolded-protein response in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16278451" target=_blank>Jenkins CC, <i>et al.</i> (2005 Nov)</a>. Activation of AP-1-dependent transcription by a truncated translation initiation factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16107694" target=_blank>van der Laan M, <i>et al.</i> (2005 Sep)</a>. Pam17 is required for architecture and translocation activity of the mitochondrial protein import motor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16002399" target=_blank>Kimura T, <i>et al.</i> (2005 Sep 9)</a>. Interactions among yeast protein-disulfide isomerase proteins and endoplasmic reticulum chaperone proteins influence their activities.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16873065" target=_blank>Denic V, <i>et al.</i> (2006 Jul 28)</a>. A luminal surveillance complex that selects misfolded glycoproteins for ER-associated degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16431909" target=_blank>Dutkiewicz R, <i>et al.</i> (2006 Mar 24)</a>. The Hsp70 chaperone Ssq1p is dispensable for iron-sulfur cluster formation on the scaffold protein Isu1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17761666" target=_blank>Reinders J, <i>et al.</i> (2007 Nov)</a>. Profiling phosphoproteins of yeast mitochondria reveals a role of phosphorylation in assembly of the ATP synthase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17258477" target=_blank>Kim Y, <i>et al.</i> (2007 Sep)</a>. Proteome map of Aspergillus nidulans during osmoadaptation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18086883" target=_blank>Calvert ME, <i>et al.</i> (2008 Feb)</a>. Phosphorylation by casein kinase 2 regulates Nap1 localization and function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17890114" target=_blank>Araújo-Bazán L, <i>et al.</i> (2008 Mar)</a>. NapA and NapB are the Aspergillus nidulans Nap/SET family members and NapB is a nuclear protein specifically interacting with importin alpha.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18794354" target=_blank>Fukumoto Y, <i>et al.</i> (2008 Nov)</a>. Schizosaccharomyces pombe Ddb1 recruits substrate-specific adaptor proteins through a novel protein motif, the DDB-box.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19053082" target=_blank>Shimizu M, <i>et al.</i> (2009 Jan)</a>. Proteomic analysis of Aspergillus nidulans cultured under hypoxic conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20534811" target=_blank>Abenza JF, <i>et al.</i> (2010 Aug 1)</a>. Aspergillus RabB Rab5 integrates acquisition of degradative identity with the long distance movement of early endosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21091509" target=_blank>Colabardini AC, <i>et al.</i> (2010 Dec)</a>. Involvement of the Aspergillus nidulans protein kinase C with farnesol tolerance is related to the unfolded protein response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19756689" target=_blank>Spirek M, <i>et al.</i> (2010 Feb)</a>. SUMOylation is required for normal development of linear elements and wild-type meiotic recombination in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23956636" target=_blank>Cho SJ, <i>et al.</i> (2010 Jun)</a>. Possible Roles of LAMMER Kinase Lkh1 in Fission Yeast by Comparative Proteome Analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919853" target=_blank>Oh YT, <i>et al.</i> (2010 Mar)</a>. Proteomic analysis of early phase of conidia germination in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20231270" target=_blank>Selvanathan SP, <i>et al.</i> (2010 May 7)</a>. Schizosaccharomyces pombe Dss1p is a DNA damage checkpoint protein that recruits Rad24p, Cdc25p, and Rae1p to DNA double-strand breaks.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21553327" target=_blank>Freitas JS, <i>et al.</i> (2011 Sep)</a>. Transcription of the Hsp30, Hsp70, and Hsp90 heat shock protein genes is modulated by the PalA protein in response to acid pH-sensing in the fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22835028" target=_blank>Saykhedkar S, <i>et al.</i> (2012 Jul 26)</a>. A time course analysis of the extracellular proteome of Aspergillus nidulans growing on sorghum stover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22561085" target=_blank>Gerke J, <i>et al.</i> (2012 Jun)</a>. Fungal S-adenosylmethionine synthetase and the control of development and secondary metabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23122962" target=_blank>Drogat J, <i>et al.</i> (2012 Nov 29)</a>. Cdk11-cyclinL controls the assembly of the RNA polymerase II mediator complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23779158" target=_blank>Saito Y, <i>et al.</i> (2013)</a>. The proteasome factor Bag101 binds to Rad22 and suppresses homologous recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24146635" target=_blank>Vjestica A, <i>et al.</i> (2013)</a>. Hsp70-Hsp40 chaperone complex functions in controlling polarized growth by repressing Hsf1-driven heat stress-associated transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23766336" target=_blank>Coradetti ST, <i>et al.</i> (2013 Aug)</a>. Analysis of a conserved cellulase transcriptional regulator reveals inducer-independent production of cellulolytic enzymes in Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24634168" target=_blank>Bernal M, <i>et al.</i> (2014 Jun)</a>. Proteome-wide search for PP2A substrates in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24442611" target=_blank>Chen W, <i>et al.</i> (2014 Mar)</a>. Endogenous U2·U5·U6 snRNA complexes in S. pombe are intron lariat spliceosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24768994" target=_blank>Oda Y, <i>et al.</i> (2014 May 21)</a>. The Schizosaccharomyces pombe Hikeshi/Opi10 protein has similar biochemical functions to its human homolog but acts in different physiological contexts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26152728" target=_blank>Mathiassen SG, <i>et al.</i> (2015 Aug 21)</a>. A Two-step Protein Quality Control Pathway for a Misfolded DJ-1 Variant in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27966061" target=_blank>Poulsen EG, <i>et al.</i> (2017 Jan)</a>. UBL/BAG-domain co-chaperones cause cellular stress upon overexpression through constitutive activation of Hsf1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28765280" target=_blank>Kampmeyer C, <i>et al.</i> (2017 Sep 15)</a>. The exocyst subunit Sec3 is regulated by a protein quality control pathway.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01009_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01009_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01009_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01009_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG01010<br>EOG82BVQW<br>EOG8CVDR9<br><br>sce:MNN2<br><br>Genes: 60
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Alpha-1,2-mannosyltransferase; responsible for addition of the first alpha-1,2-linked mannose to form the branches on the mannan backbone of oligosaccharides, localizes to an early Golgi compartment<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have alpha-1,2-mannosyltransferase activity, role in protein glycosylation and Golgi apparatus localization|Has domain(s) with predicted transferase activity, transferring glycosyl groups activity and role in protein glycosylation|Ortholog(s) have role in mannan biosynthetic process, protein mannosylation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4124122" target=_blank>Ballou CE, <i>et al.</i> (1973 Jul 10)</a>. Genetic control of yeast mannan structure. Complementation studies and properties of mannan mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4578089" target=_blank>Hawkins ER, <i>et al.</i> (1973 Jul 10)</a>. A comparison of yeast mannan mutants by electron microscopy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4578088" target=_blank>Raschke WC, <i>et al.</i> (1973 Jul 10)</a>. Genetic control of yeast mannan structure. Isolation and characterization of mannan mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4612040" target=_blank>Nakajima T, <i>et al.</i> (1974 Dec 10)</a>. Characterization of the carbohydrate fragments obtained from Saccharomyces cerevisiae mannan by alkaline degradation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4612041" target=_blank>Nakajima T, <i>et al.</i> (1974 Dec 10)</a>. Structure of the linkage region between the polysaccharide and protein parts of Saccharomyces cerevisiae mannan.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1097420" target=_blank>Ballou DL, <i>et al.</i> (1975 Aug)</a>. Genetic control of yeast mannan structure: mapping genes mnn2 and mnn4 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=355255" target=_blank>Karson EM, <i>et al.</i> (1978 Sep 25)</a>. Biosynthesis of yeast mannan. Properties of a mannosylphosphate transferase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=381304" target=_blank>Parodi AJ, <i>et al.</i> (1979 Sep 10)</a>. Biosynthesis of yeast mannoproteins. Synthesis of mannan outer chain and of dolichol derivatives.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6802821" target=_blank>Cohen RE, <i>et al.</i> (1982 May 25)</a>. Effects of mannoprotein mutations on Saccharomyces cerevisiae core oligosaccharide structure.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2082155" target=_blank>Devlin C, <i>et al.</i> (1990 Nov)</a>. Identification and characterization of a gene and protein required for glycosylation in the yeast Golgi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7992511" target=_blank>Romero PA, <i>et al.</i> (1994 Aug)</a>. The nucleotide sequence of TTP1, a gene encoding a predicted type II membrane protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9413431" target=_blank>Nakamura T, <i>et al.</i> (1997 Nov)</a>. Yeast Crv4/Ttp1, a predicted type II membrane protein, is involved in an event important for growth, functionally overlapping with the event regulated by calcineurin- and Mpk1-mediated pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9756928" target=_blank>Rayner JC, <i>et al.</i> (1998 Oct 9)</a>. Identification of the MNN2 and MNN5 mannosyltransferases required for forming and extending the mannose branches of the outer chain mannans of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16107716" target=_blank>Inadome H, <i>et al.</i> (2005 Sep)</a>. Immunoisolaton of the yeast Golgi subcompartments and characterization of a novel membrane protein, Svp26, discovered in the Sed5-containing compartments.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18410729" target=_blank>Schmitz KR, <i>et al.</i> (2008 Apr)</a>. Golgi localization of glycosyltransferases requires a Vps74p oligomer.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20236934" target=_blank>Noda Y, <i>et al.</i> (2010 May 14)</a>. Svp26 facilitates endoplasmic reticulum to golgi transport of a set of mannosyltransferases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633946" target=_blank>Hall RA, <i>et al.</i> (2013)</a>. The Mnn2 mannosyltransferase family modulates mannoprotein fibril length, immune recognition and virulence of Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01010_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01010_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01010_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01010_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>26 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG01011<br>EOG82BVQW<br>EOG8CVDR9<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030247" target=_blank>Hromatka BS, <i>et al.</i> (2005 Oct)</a>. Transcriptional response of Candida albicans to nitric oxide and the role of the YHB1 gene in nitrosative stress and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16102003" target=_blank>Urban C, <i>et al.</i> (2005 Sep)</a>. The moonlighting protein Tsa1p is implicated in oxidative stress response and in cell wall biogenesis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16339080" target=_blank>Enjalbert B, <i>et al.</i> (2006 Feb)</a>. Role of the Hog1 stress-activated protein kinase in the global transcriptional response to stress in the fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633946" target=_blank>Hall RA, <i>et al.</i> (2013)</a>. The Mnn2 mannosyltransferase family modulates mannoprotein fibril length, immune recognition and virulence of Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01011_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01011_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01011_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01011_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9e6c3" class="col">
			FOG01012<br>EOG82BVQW<br><br>sce:absent<br><br>Genes: 11
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01012_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01012_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01012_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01012_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dbc4e1" class="col">
			FOG01013<br>EOG82BVQW<br><br>sce:MNN5<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Alpha-1,2-mannosyltransferase; responsible for addition of the second alpha-1,2-linked mannose of the branches on the mannan backbone of oligosaccharides, localizes to an early Golgi compartment<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6995454" target=_blank>Cohen RE, <i>et al.</i> (1980 Aug 25)</a>. Saccharomyces cerevisiae mannoprotein mutants. Isolation of the mnn5 mutant and comparison with the mnn3 strain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6802821" target=_blank>Cohen RE, <i>et al.</i> (1982 May 25)</a>. Effects of mannoprotein mutations on Saccharomyces cerevisiae core oligosaccharide structure.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9756928" target=_blank>Rayner JC, <i>et al.</i> (1998 Oct 9)</a>. Identification of the MNN2 and MNN5 mannosyltransferases required for forming and extending the mannose branches of the outer chain mannans of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20236934" target=_blank>Noda Y, <i>et al.</i> (2010 May 14)</a>. Svp26 facilitates endoplasmic reticulum to golgi transport of a set of mannosyltransferases in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01013_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01013_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01013_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01013_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fad7ad" class="col">
			FOG01014<br>EOG82BVQW<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01014_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01014_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01014_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01014_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb9" class="col">
			FOG01015<br>EOG8QRFK1<br><br>sce:TPK1;TPK2;TPK3<br><br>Genes: 48
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>cAMP-dependent protein kinase catalytic subunit; promotes vegetative growth in response to nutrients via the Ras-cAMP signaling pathway; inhibited by regulatory subunit Bcy1p in the absence of cAMP; phosphorylates and inhibits Whi3p to promote G1/S phase passage; partially redundant with Tpk2p and Tpk3p; phosphorylates pre-Tom40p, which impairs its import into mitochondria under non-respiratory conditions; TPK1 has a paralog, TPK3, that arose from the whole genome duplication|cAMP-dependent protein kinase catalytic subunit; promotes vegetative growth in response to nutrients via the Ras-cAMP signaling pathway; partially redundant with Tpk1p and Tpk3p; localizes to P-bodies during stationary phase; relocalizes to the cytosol in response to hypoxia|cAMP-dependent protein kinase catalytic subunit; promotes vegetative growth in response to nutrients via the Ras-cAMP signaling pathway; partially redundant with Tpk1p and Tpk2p; localizes to P-bodies during stationary phase; TPK3 has a paralog, TPK1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>cAMP-dependent protein kinase catalytic subunit Pka1<br><br>
			<br><b>AspGD Description</b><br>Catalytic subunit of cAMP-dependent protein kinase; expressed in dormant conidia<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823100" target=_blank>Cannon JF, <i>et al.</i> (1987 Aug)</a>. Characterization of Saccharomyces cerevisiae genes encoding subunits of cyclic AMP-dependent protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3546292" target=_blank>Lisziewicz J, <i>et al.</i> (1987 Feb 25)</a>. Isolation and nucleotide sequence of a Saccharomyces cerevisiae protein kinase gene suppressing the cell cycle start mutation cdc25.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3036373" target=_blank>Toda T, <i>et al.</i> (1987 Jul 17)</a>. Three different genes in S. cerevisiae encode the catalytic subunits of the cAMP-dependent protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3308514" target=_blank>Ohno S, <i>et al.</i> (1987 Oct 5)</a>. A yeast gene coding for a putative protein kinase homologous to cdc25 suppressing protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1544898" target=_blank>Fearon K, <i>et al.</i> (1992 Mar 15)</a>. Structure and function of MRP20 and MRP49, the nuclear genes for two proteins of the 54 S subunit of the yeast mitochondrial ribosome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11156981" target=_blank>Shimizu K, <i>et al.</i> (2001 Feb)</a>. Genetic involvement of a cAMP-dependent protein kinase in a G protein signaling pathway regulating morphological and chemical transitions in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11368172" target=_blank>Mashhoon N, <i>et al.</i> (2001 Mar 1)</a>. Structure of the unliganded cAMP-dependent protein kinase catalytic subunit from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12398295" target=_blank>Vanzela AP, <i>et al.</i> (2002)</a>. Evidence for carbon source regulated protein kinase A and protein kinase C signaling in the duplication cycle, polarization and septum formation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12046590" target=_blank>Fillinger S, <i>et al.</i> (2002 May)</a>. cAMP and ras signalling independently control spore germination in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14668367" target=_blank>Shimizu K, <i>et al.</i> (2003 Nov)</a>. Pka, Ras and RGS protein interactions regulate activity of AflR, a Zn(II)2Cys6 transcription factor in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14982628" target=_blank>Han KH, <i>et al.</i> (2004 Mar)</a>. A putative G protein-coupled receptor negatively controls sexual development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15905473" target=_blank>Zhang Z, <i>et al.</i> (2005)</a>. Mapping of transcription start sites in Saccharomyces cerevisiae using 5' SAGE.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16087751" target=_blank>Ni M, <i>et al.</i> (2005 Aug)</a>. The pkaB gene encoding the secondary protein kinase A catalytic subunit has a synthetic lethal interaction with pkaA and plays overlapping and opposite roles in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16728950" target=_blank>Yu JH, <i>et al.</i> (2006 Apr)</a>. Heterotrimeric G protein signaling and RGSs in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16376592" target=_blank>Greenstein S, <i>et al.</i> (2006 Jan)</a>. Analysis of the Aspergillus nidulans thaumatin-like cetA gene and evidence for transcriptional repression of pyr4 expression in the cetA-disrupted strain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16426964" target=_blank>Schardl CL, <i>et al.</i> (2006 Jan)</a>. A global view of metabolites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16546420" target=_blank>Lafon A, <i>et al.</i> (2006 Jul)</a>. G-protein and cAMP-mediated signaling in aspergilli: a genomic perspective.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17030989" target=_blank>Yu JH, <i>et al.</i> (2006 Oct)</a>. Growth and developmental control in the model and pathogenic aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19648579" target=_blank>Mircus G, <i>et al.</i> (2009 Oct)</a>. Identification of novel cell wall destabilizing antifungal compounds using a conditional Aspergillus nidulans protein kinase C mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19889092" target=_blank>Binder U, <i>et al.</i> (2010 Jan)</a>. The antifungal protein PAF interferes with PKC/MPK and cAMP/PKA signalling of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21741477" target=_blank>Gao L, <i>et al.</i> (2011 Nov)</a>. Osmotic stabilizer-coupled suppression of NDR defects is dependent on the calcium-calcineurin signaling cascade in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21658084" target=_blank>Bayram O, <i>et al.</i> (2012 Jan)</a>. Coordination of secondary metabolism and development in fungi: the velvet family of regulatory proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23002107" target=_blank>Kwon NJ, <i>et al.</i> (2012 Nov)</a>. The putative guanine nucleotide exchange factor RicA mediates upstream signaling for growth and development in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23800192" target=_blank>Brown NA, <i>et al.</i> (2013 Jun 25)</a>. Functional characterisation of the non-essential protein kinases and phosphatases regulating Aspergillus nidulans hydrolytic enzyme production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23450714" target=_blank>Krijgsheld P, <i>et al.</i> (2013 Mar 15)</a>. Development in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25116090" target=_blank>Ramsubramaniam N, <i>et al.</i> (2014 Nov)</a>. The phosphoproteome of Aspergillus nidulans reveals functional association with cellular processes involved in morphology and secretion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26690721" target=_blank>de Assis LJ, <i>et al.</i> (2015)</a>. Aspergillus nidulans protein kinase A plays an important role in cellulase production.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01015_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01015_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01015_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01015_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>25 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dfc9ba" class="col">
			FOG01016<br>EOG8QRFK1<br><br>sce:YPK3<br><br>Genes: 42
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>AGC kinase; phosphorylated by cAMP-dependent protein kinase (PKA) in a TORC1-dependent manner<br><br>
			<br><b>PomBase Description</b><br>serine/threonine protein kinase Psk1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11062466" target=_blank>Zhu H, <i>et al.</i> (2000 Nov)</a>. Analysis of yeast protein kinases using protein chips.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16172400" target=_blank>Budovskaya YV, <i>et al.</i> (2005 Sep 27)</a>. An evolutionary proteomics approach identifies substrates of the cAMP-dependent protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20702584" target=_blank>Soulard A, <i>et al.</i> (2010 Oct 1)</a>. The rapamycin-sensitive phosphoproteome reveals that TOR controls protein kinase A toward some but not all substrates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25767889" target=_blank>González A, <i>et al.</i> (2015)</a>. TORC1 promotes phosphorylation of ribosomal protein S6 via the AGC kinase Ypk3 in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01016_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01016_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01016_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01016_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8d4ea" class="col">
			FOG01017<br>EOG8GQNMB<br>EOG8QJQ38<br><br>sce:absent<br><br>Genes: 40
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative transcription factor, as suggested by computational analysis; green fluorescent protein (GFP)-fusion protein localizes to both the cytoplasm and the nucleus and is induced in response to the DNA-damaging agent MMS<br><br>
			<br><b>PomBase Description</b><br>zinc finger protein, transcription factor Zas1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have sequence-specific DNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171100" target=_blank>Böhm S, <i>et al.</i> (1997 Jun 15)</a>. Variations of the C2H2 zinc finger motif in the yeast genome and classification of yeast zinc finger proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01017_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01017_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01017_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01017_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>39 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eeadad" class="col">
			FOG01018<br>EOG8QRFK1<br><br>sce:YPK1;YPK2<br><br>Genes: 39
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Serine/threonine protein kinase; phosphorylates and downregulates flippase activator Fpk1p; inactivates Orm1p and Orm2p (inhibitors of serine:palmitoyl-coenzyme A transferase) by phosphorylation in response to compromised sphingolipid synthesis; mutations affect receptor-mediated endocytosis and sphingolipid-mediated and cell integrity signaling pathways; homolog of mammalian kinase SGK; YPK1 has a paralog, YPK2, that arose from the whole genome duplication|Protein kinase similar to serine/threonine protein kinase Ypk1p; functionally redundant with YPK1 at the genetic level; participates in a signaling pathway required for optimal cell wall integrity; homolog of mammalian kinase SGK; YPK2 has a paralog, YPK1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>AGC family protein kinase Gad8<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have protein serine/threonine kinase activity, protein serine/threonine kinase inhibitor activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2850145" target=_blank>Maurer RA, <i>et al.</i> (1988 Sep)</a>. Isolation of a yeast protein kinase gene by screening with a mammalian protein kinase cDNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2663649" target=_blank>Kubo K, <i>et al.</i> (1989 Mar 15)</a>. A novel yeast gene coding for a putative protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437590" target=_blank>Chen P, <i>et al.</i> (1993 Jan)</a>. A pair of putative protein kinase genes (YPK1 and YPK2) is required for cell growth in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074427" target=_blank>Casamayor A, <i>et al.</i> (1999 Feb 25)</a>. Functional counterparts of mammalian protein kinases PDK1 and SGK in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10825204" target=_blank>Sun Y, <i>et al.</i> (2000 Jun)</a>. Sli2 (Ypk1), a homologue of mammalian protein kinase SGK, is a downstream kinase in the sphingolipid-mediated signaling pathway of yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11807089" target=_blank>deHart AK, <i>et al.</i> (2002 Jan 21)</a>. The conserved Pkh-Ypk kinase cascade is required for endocytosis in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12221112" target=_blank>Roelants FM, <i>et al.</i> (2002 Sep)</a>. Pkh1 and Pkh2 differentially phosphorylate and activate Ypk1 and Ykr2 and define protein kinase modules required for maintenance of cell wall integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15025559" target=_blank>Momoi M, <i>et al.</i> (2004 Jul 1)</a>. SLI1 (YGR212W) is a major gene conferring resistance to the sphingolipid biosynthesis inhibitor ISP-1, and encodes an ISP-1 N-acetyltransferase in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15470109" target=_blank>Roelants FM, <i>et al.</i> (2004 Oct)</a>. Differential roles of PDK1- and PDK2-phosphorylation sites in the yeast AGC kinases Ypk1, Pkc1 and Sch9.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16055732" target=_blank>Kamada Y, <i>et al.</i> (2005 Aug)</a>. Tor2 directly phosphorylates the AGC kinase Ypk2 to regulate actin polarization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15840588" target=_blank>Liu K, <i>et al.</i> (2005 Jun 17)</a>. The sphingoid long chain base phytosphingosine activates AGC-type protein kinases in Saccharomyces cerevisiae including Ypk1, Ypk2, and Sch9.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15820214" target=_blank>Tanoue D, <i>et al.</i> (2005 May 1)</a>. The requirement for the hydrophobic motif phosphorylation of Ypk1 in yeast differs depending on the downstream events, including endocytosis, cell growth, and resistance to a sphingolipid biosynthesis inhibitor, ISP-1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16598003" target=_blank>Herrmann M, <i>et al.</i> (2006 Apr)</a>. Protein kinase C (PkcA) of Aspergillus nidulans is involved in penicillin production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19966303" target=_blank>Roelants FM, <i>et al.</i> (2010 Jan 5)</a>. A protein kinase network regulates the function of aminophospholipid flippases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20526834" target=_blank>Chellegatti MA, <i>et al.</i> (2010 May)</a>. Role of phospholipase C and protein kinase C in Aspergillus nidulans during growth on pectin or glucose: Effects on germination and duplication cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23472095" target=_blank>Colabardini AC, <i>et al.</i> (2013)</a>. Functional characterization of Aspergillus nidulans ypkA, a homologue of the mammalian kinase SGK.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01018_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01018_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01018_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01018_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>11 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bacde4" class="col">
			FOG01019<br>EOG8QRFK1<br><br>sce:SCH9<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>AGC family protein kinase; functional ortholog of mammalian S6 kinase; phosphorylated by Tor1p and required for TORC1-mediated regulation of ribosome biogenesis, translation initiation, and entry into G0 phase; involved in transactivation of osmostress-responsive genes; regulates G1 progression, cAPK activity and nitrogen activation of the FGM pathway; integrates nutrient signals and stress signals from sphingolipids to regulate lifespan<br><br>
			<br><b>PomBase Description</b><br>serine/threonine protein kinase Sck1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have protein serine/threonine kinase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3290050" target=_blank>Toda T, <i>et al.</i> (1988 May)</a>. SCH9, a gene of Saccharomyces cerevisiae that encodes a protein distinct from, but functionally and structurally related to, cAMP-dependent protein kinase catalytic subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8442384" target=_blank>di Blasi F, <i>et al.</i> (1993 Jan)</a>. The SCH9 protein kinase mRNA contains a long 5' leader with a small open reading frame.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14663094" target=_blank>Nobile CJ, <i>et al.</i> (2003 Dec)</a>. Genetic control of chlamydospore formation in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17560372" target=_blank>Urban J, <i>et al.</i> (2007 Jun 8)</a>. Sch9 is a major target of TORC1 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345900" target=_blank>Liu W, <i>et al.</i> (2010 Jun)</a>. The protein kinase CaSch9p is required for the cell growth, filamentation and virulence in the human fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20702584" target=_blank>Soulard A, <i>et al.</i> (2010 Oct 1)</a>. The rapamycin-sensitive phosphoproteome reveals that TOR controls protein kinase A toward some but not all substrates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21335533" target=_blank>Stichternoth C, <i>et al.</i> (2011 Apr)</a>. Sch9 kinase integrates hypoxia and CO2 sensing to suppress hyphal morphogenesis in Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01019_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01019_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01019_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01019_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>8 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c1e3bb" class="col">
			FOG01020<br>EOG8QRFK1<br><br>sce:FPK1;KIN82<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ser/Thr protein kinase; phosphorylates several aminophospholipid translocase family members, regulating phospholipid translocation and membrane asymmetry; phosphorylates and inhibits upstream inhibitory kinase, Ypk1p; localizes to the cytoplasm, early endosome/TGN compartments and thplasma membrane; localizes to the shmoo tip where it has a redundant role in the cellular response to mating pheromone; FPK1 has a paralog, KIN82, that arose from the whole genome duplication|Putative serine/threonine protein kinase; implicated in the regulation of phospholipid asymmetry through the activation of phospholipid translocases (flippases); involved in the phosphorylation of upstream inhibitory kinase Ypk1p along with Fpk1p; has a redundant role in the cellular response to mating pheromone; KIN82 has a paralog, FPK1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>serine/threonine protein kinase Ppk14 (predicted)|serine/threonine protein kinase Ppk22 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in ascospore formation, conidiophore development, conidium formation, hyphal growth, sporocarp development involved in sexual reproduction<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1580103" target=_blank>Wilson C, <i>et al.</i> (1992 Jan)</a>. A putative serine/threonine protein kinase gene on chromosome III of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11337509" target=_blank>Burchett SA, <i>et al.</i> (2001 Jul 13)</a>. Identification of novel pheromone-response regulators through systematic overexpression of 120 protein kinases in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18199685" target=_blank>Nakano K, <i>et al.</i> (2008 Apr)</a>. Protein kinases Fpk1p and Fpk2p are novel regulators of phospholipid asymmetry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19966303" target=_blank>Roelants FM, <i>et al.</i> (2010 Jan 5)</a>. A protein kinase network regulates the function of aminophospholipid flippases.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01020_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01020_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01020_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01020_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7bcdd" class="col">
			FOG01021<br>EOG837Q17<br>EOG8GQNMB<br><br>sce:CPR5<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Peptidyl-prolyl cis-trans isomerase (cyclophilin) of the ER; catalyzes the cis-trans isomerization of peptide bonds N-terminal to proline residues; transcriptionally induced in response to unfolded proteins in the ER; CPR5 has a paralog, CPR2, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>cyclophilin family peptidyl-prolyl cis-trans isomerase Cyp4 (predicted)<br><br>
			<br><b>AspGD Description</b><br>ER peptidyl-prolyl cis-trans isomerase; cyclophilin; has a functional endoplasmic reticulum (ER) retention sequence<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8377189" target=_blank>Frigerio G, <i>et al.</i> (1993 Sep 5)</a>. A Saccharomyces cerevisiae cyclophilin resident in the endoplasmic reticulum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14644438" target=_blank>Pemberton TJ, <i>et al.</i> (2003 Dec 4)</a>. Cyclophilin sensitivity to sanglifehrin A can be correlated to the same specific tryptophan residue as cyclosporin A.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23050226" target=_blank>Chen Z, <i>et al.</i> (2012 Oct)</a>. A genetic screen to discover pathways affecting cohesin function in Schizosaccharomyces pombe identifies chromatin effectors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01021_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01021_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01021_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01021_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d2a2" class="col">
			FOG01022<br>EOG8QRFK1<br><br>sce:DBF20;DBF2<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Ser/Thr kinase involved in late nuclear division; one of the mitotic exit network (MEN) proteins; necessary for the execution of cytokinesis; also plays a role in regulating the stability of SWI5 and CLB2 mRNAs; DBF20 has a paralog, DBF2, that arose from the whole genome duplication|Ser/Thr kinase involved in transcription and stress response; functions as part of a network of genes in exit from mitosis; localization is cell cycle regulated; activated by Cdc15p during the exit from mitosis; also plays a role in regulating the stability of SWI5 and CLB2 mRNAs; phosphorylates Chs2p to regulate primary septum formation and Hof1p to regulate cytokinesis; DBF2 has a paralog, DBF20, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>NDR kinase Sid2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have protein serine/threonine kinase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181271" target=_blank>Johnston LH, <i>et al.</i> (1990 Apr)</a>. The product of the Saccharomyces cerevisiae cell cycle gene DBF2 has homology with protein kinases and is periodically expressed in the cell cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1916278" target=_blank>Toyn JH, <i>et al.</i> (1991 Jul 31)</a>. The cell-cycle-regulated budding yeast gene DBF2, encoding a putative protein kinase, has a homologue that is not under cell-cycle control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8131744" target=_blank>Toyn JH, <i>et al.</i> (1994 Mar 1)</a>. The Dbf2 and Dbf20 protein kinases of budding yeast are activated after the metaphase to anaphase cell cycle transition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613578" target=_blank>Grandin N, <i>et al.</i> (1998 Apr)</a>. The Cdc14 phosphatase is functionally associated with the Dbf2 protein kinase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9528782" target=_blank>Komarnitsky SI, <i>et al.</i> (1998 Apr)</a>. DBF2 protein kinase binds to and acts through the cell cycle-regulated MOB1 protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10228157" target=_blank>Fesquet D, <i>et al.</i> (1999 May 4)</a>. A Bub2p-dependent spindle checkpoint pathway regulates the Dbf2p kinase in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10984431" target=_blank>Frenz LM, <i>et al.</i> (2000 Oct)</a>. The budding yeast Dbf2 protein kinase localises to the centrosome and moves to the bud neck in late mitosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11434459" target=_blank>Yoshida S, <i>et al.</i> (2001 Apr)</a>. Regulation of the localization of Dbf2 and mob1 during cell division of saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11404483" target=_blank>Mah AS, <i>et al.</i> (2001 Jun 19)</a>. Protein kinase Cdc15 activates the Dbf2-Mob1 kinase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11378390" target=_blank>Lee SE, <i>et al.</i> (2001 May 15)</a>. Order of function of the budding-yeast mitotic exit-network proteins Tem1, Cdc15, Mob1, Dbf2, and Cdc5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11598184" target=_blank>Visintin R, <i>et al.</i> (2001 Oct)</a>. Regulation of the mitotic exit protein kinases Cdc15 and Dbf2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12832486" target=_blank>Bardin AJ, <i>et al.</i> (2003 Jul)</a>. Mitotic exit regulation through distinct domains within the protein kinase Cdc15.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12937277" target=_blank>Hwa Lim H, <i>et al.</i> (2003 Nov)</a>. Inactivation of mitotic kinase triggers translocation of MEN components to mother-daughter neck in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14551257" target=_blank>Visintin R, <i>et al.</i> (2003 Nov)</a>. The role of the polo kinase Cdc5 in controlling Cdc14 localization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16176976" target=_blank>Stoepel J, <i>et al.</i> (2005 Dec)</a>. The mitotic exit network Mob1p-Dbf2p kinase complex localizes to the nucleus and regulates passenger protein localization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16242037" target=_blank>Mah AS, <i>et al.</i> (2005 Oct 21)</a>. Substrate specificity analysis of protein kinase complex Dbf2-Mob1 by peptide library and proteome array screening.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16624915" target=_blank>Kim JM, <i>et al.</i> (2006 Jun)</a>. Isolation of mutations that bypass the requirement of the septation initiation network for septum formation and conidiation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18048916" target=_blank>Makrantoni V, <i>et al.</i> (2007 Dec)</a>. A novel role for the yeast protein kinase Dbf2p in vacuolar H+-ATPase function and sorbic acid stress tolerance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19221193" target=_blank>Mohl DA, <i>et al.</i> (2009 Feb 23)</a>. Dbf2-Mob1 drives relocalization of protein phosphatase Cdc14 to the cytoplasm during exit from mitosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19477921" target=_blank>Côte P, <i>et al.</i> (2009 Jul)</a>. Transcriptional analysis of the Candida albicans cell cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19460099" target=_blank>González-Novo A, <i>et al.</i> (2009 Jun)</a>. Dbf2 is essential for cytokinesis and correct mitotic spindle formation in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19386763" target=_blank>Kim JM, <i>et al.</i> (2009 Jun)</a>. Timely septation requires SNAD-dependent spindle pole body localization of the septation initiation network components in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20123997" target=_blank>König C, <i>et al.</i> (2010 Feb 8)</a>. Mutual regulation of cyclin-dependent kinase and the mitotic exit network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20442249" target=_blank>Meitinger F, <i>et al.</i> (2010 Jun 1)</a>. Targeted localization of Inn1, Cyk3 and Chs2 by the mitotic-exit network regulates cytokinesis in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22196726" target=_blank>Trcek T, <i>et al.</i> (2011 Dec 23)</a>. Single-molecule mRNA decay measurements reveal promoter- regulated mRNA stability in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22573892" target=_blank>Oh Y, <i>et al.</i> (2012 Jul)</a>. Mitotic exit kinase Dbf2 directly phosphorylates chitin synthase Chs2 to regulate cytokinesis in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23447700" target=_blank>Meitinger F, <i>et al.</i> (2013 May)</a>. Dual function of the NDR-kinase Dbf2 in the regulation of the F-BAR protein Hof1 during cytokinesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375164" target=_blank>Eslami H, <i>et al.</i> (2014)</a>. Down-regulation of sidB gene by use of RNA interference in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01022_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01022_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01022_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01022_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb0" class="col">
			FOG01023<br>EOG86HDSW<br>EOG8GQNMB<br><br>sce:CCA1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATP (CTP):tRNA-specific tRNA nucleotidyltransferase; different forms targeted to the nucleus, cytosol, and mitochondrion are generated via the use of multiple transcriptional and translational start sites; human homolog TRNT1 complements yeast null mutant<br><br>
			<br><b>PomBase Description</b><br>ATP(CTP) tRNA nucleotidyltransferase Cca1 (predicted)|ATP(CTP) tRNA nucleotidyltransferase Cca2 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have tRNA adenylyltransferase activity, role in tRNA 3'-terminal CCA addition and mitochondrial matrix localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204621" target=_blank>Aebi M, <i>et al.</i> (1990 Sep 25)</a>. Isolation of a temperature-sensitive mutant with an altered tRNA nucleotidyltransferase and cloning of the gene encoding tRNA nucleotidyltransferase in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175766" target=_blank>Wolfe CL, <i>et al.</i> (1994 May 6)</a>. Interplay of heterogeneous transcriptional start sites and translational selection of AUGs dictate the production of mitochondrial and cytosolic/nuclear tRNA nucleotidyltransferase from the same gene in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12478587" target=_blank>Hanic-Joyce PJ, <i>et al.</i> (2002 Dec)</a>. Characterization of a gene encoding tRNA nucleotidyltransferase from Candida glabrata.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01023_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01023_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01023_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01023_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dac1b0" class="col">
			FOG01024<br>EOG837Q17<br><br>sce:CPR6<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Peptidyl-prolyl cis-trans isomerase (cyclophilin); catalyzes the cis-trans isomerization of peptide bonds N-terminal to proline residues; plays a role in determining prion variants; binds to Hsp82p and contributes to chaperone activity; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>cyclophilin family peptidyl-prolyl cis-trans isomerase Wis2<br><br>
			<br><b>AspGD Description</b><br>Estrogen receptor-binding cyclophilin<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8873448" target=_blank>Duina AA, <i>et al.</i> (1996 Aug)</a>. Identification of two CyP-40-like cyclophilins in Saccharomyces cerevisiae, one of which is required for normal growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870366" target=_blank>Sims AH, <i>et al.</i> (2005 May)</a>. Transcriptome analysis of recombinant protein secretion by Aspergillus nidulans and the unfolded-protein response in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16995943" target=_blank>Pemberton TJ, <i>et al.</i> (2006 Sep 22)</a>. Identification and comparative analysis of sixteen fungal peptidyl-prolyl cis/trans isomerase repertoires.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01024_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01024_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01024_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01024_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cee7" class="col">
			FOG01025<br>EOG8GQNMB<br>EOG8QRFK1<br><br>sce:CBK1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Serine/threonine protein kinase of the the RAM signaling network; Ndr/LATS family member; binds regulatory subunit Mob2p; involved in regulation of cellular morphogenesis, polarized growth, and septum destruction; phosphorylation by Cbk1p regulates localization and activity of Ace2p transcription factor and Ssd1p translational repressor; Cbk1p activity is regulated by both phosphorylation and specific localization; relocalizes to cytoplasm upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>serine/threonine protein kinase Orb6<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have protein serine/threonine kinase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8477753" target=_blank>Dehoux P, <i>et al.</i> (1993 Apr 15)</a>. Natural cycloheximide resistance in yeast. The role of ribosomal protein L41.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11889116" target=_blank>McNemar MD, <i>et al.</i> (2002 Apr)</a>. Conserved serine/threonine kinase encoded by CBK1 regulates expression of several hypha-associated transcripts and genes encoding cell wall proteins in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11854408" target=_blank>Du LL, <i>et al.</i> (2002 Feb)</a>. Pag1p, a novel protein associated with protein kinase Cbk1p, is required for cell morphogenesis and proliferation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12196508" target=_blank>Weiss EL, <i>et al.</i> (2002 Sep 2)</a>. The Saccharomyces cerevisiae Mob2p-Cbk1p kinase complex promotes polarized growth and acts with the mitotic exit network to facilitate daughter cell-specific localization of Ace2p transcription factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15588991" target=_blank>Zarrin M, <i>et al.</i> (2005 Jan)</a>. A rapid method for promoter exchange in Aspergillus nidulans using recombinant PCR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16506053" target=_blank>Johns SA, <i>et al.</i> (2006 Jun)</a>. Depletion of Aspergillus nidulans cotA causes a severe polarity defect which is not suppressed by the nuclear migration mutation nudA2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17189178" target=_blank>Feng B, <i>et al.</i> (2006 Nov)</a>. Identification of genes from Aspergillus oryzae that are preferentially expressed in membrane-surface liquid culture.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18832040" target=_blank>Shi J, <i>et al.</i> (2008 Dec)</a>. Depletion of the MobB and CotA complex in Aspergillus nidulans causes defects in polarity maintenance that can be suppressed by the environment stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18843050" target=_blank>Song Y, <i>et al.</i> (2008 Dec)</a>. Role of the RAM network in cell polarity and hyphal morphogenesis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21593210" target=_blank>Gutiérrez-Escribano P, <i>et al.</i> (2011 Jul 15)</a>. CDK-dependent phosphorylation of Mob2 is essential for hyphal development in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21741477" target=_blank>Gao L, <i>et al.</i> (2011 Nov)</a>. Osmotic stabilizer-coupled suppression of NDR defects is dependent on the calcium-calcineurin signaling cascade in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22589718" target=_blank>Gutiérrez-Escribano P, <i>et al.</i> (2012)</a>. The NDR/LATS kinase Cbk1 controls the activity of the transcriptional regulator Bcr1 during biofilm formation in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24599037" target=_blank>De Souza CP, <i>et al.</i> (2014)</a>. Application of a new dual localization-affinity purification tag reveals novel aspects of protein kinase biology in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01025_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01025_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01025_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01025_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eca3a4" class="col">
			FOG01026<br>EOG8FJ6RG<br>EOG8GQNMB<br><br>sce:LAP3<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cysteine aminopeptidase with homocysteine-thiolactonase activity; protects cells against homocysteine toxicity; has bleomycin hydrolase activity in vitro; transcription is regulated by galactose via Gal4p; orthologous to human BLMH<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have RNA polymerase II core promoter proximal region sequence-specific DNA binding, cysteine-type peptidase activity, double-stranded DNA binding, mRNA binding, single-stranded DNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1882548" target=_blank>Creutz CE, <i>et al.</i> (1991 Apr)</a>. Calcium-dependent secretory vesicle-binding and lipid-binding proteins of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400467" target=_blank>Kambouris NG, <i>et al.</i> (1992 Oct 25)</a>. Cloning and characterization of a cysteine proteinase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8463237" target=_blank>Enenkel C, <i>et al.</i> (1993 Apr 5)</a>. BLH1 codes for a yeast thiol aminopeptidase, the equivalent of mammalian bleomycin hydrolase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8424954" target=_blank>Magdolen U, <i>et al.</i> (1993 Jan 23)</a>. A yeast gene (BLH1) encodes a polypeptide with high homology to vertebrate bleomycin hydrolase, a family member of thiol proteinases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8063738" target=_blank>Xu HE, <i>et al.</i> (1994 Aug 19)</a>. Yeast bleomycin hydrolase is a DNA-binding cysteine protease. Identification, purification, biochemical characterization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7638617" target=_blank>Joshua-Tor L, <i>et al.</i> (1995 Aug 18)</a>. Crystal structure of a conserved protease that binds DNA: the bleomycin hydrolase, Gal6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9374524" target=_blank>Zheng W, <i>et al.</i> (1997 Nov 28)</a>. The cysteine-peptidase bleomycin hydrolase is a member of the galactose regulon in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9546396" target=_blank>Zheng W, <i>et al.</i> (1998 Apr 3)</a>. The unusual active site of Gal6/bleomycin hydrolase can act as a carboxypeptidase, aminopeptidase, and peptide ligase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9584198" target=_blank>Zheng W, <i>et al.</i> (1998 Jun)</a>. The nucleic acid binding activity of bleomycin hydrolase is involved in bleomycin detoxification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12555812" target=_blank>Wang H, <i>et al.</i> (2002)</a>. Cellular resistance to bleomycin in Saccharomyces cerevisiae is not affected by changes in bleomycin hydrolase levels.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16769724" target=_blank>Zimny J, <i>et al.</i> (2006 Aug 11)</a>. Protective mechanisms against homocysteine toxicity: the role of bleomycin hydrolase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17101987" target=_blank>Miura F, <i>et al.</i> (2006 Nov 21)</a>. A large-scale full-length cDNA analysis to explore the budding yeast transcriptome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01026_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01026_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01026_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01026_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b2c7e1" class="col">
			FOG01027<br>EOG8CVDRR<br>EOG8GQNMB<br><br>sce:RAD52<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein that stimulates strand exchange; stimulates strand exchange by facilitating Rad51p binding to single-stranded DNA; anneals complementary single-stranded DNA; involved in the repair of double-strand breaks in DNA during vegetative growth and meiosis and UV induced sister chromatid recombination<br><br>
			<br><b>PomBase Description</b><br>DNA recombination protein, Rad51 mediator Rad52 (previously Rad22)|Rad22 homolog Rti1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have single-stranded DNA binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6098821" target=_blank>Adzuma K, <i>et al.</i> (1984 Dec)</a>. Primary structure of the RAD52 gene in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8370524" target=_blank>Milne GT, <i>et al.</i> (1993 Sep)</a>. Dominant negative alleles of RAD52 reveal a DNA repair/recombination complex including Rad51 and Rad52.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512870" target=_blank>Suto K, <i>et al.</i> (1999 Oct)</a>. A double-strand break repair component is essential for S phase completion in fission yeast cell cycling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104907" target=_blank>van den Bosch M, <i>et al.</i> (2001 Jan 5)</a>. Characterization of RAD52 homologs in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11560889" target=_blank>Tsutsui Y, <i>et al.</i> (2001 Sep)</a>. Multiple interactions among the components of the recombinational DNA repair system in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11884628" target=_blank>van den Bosch M, <i>et al.</i> (2002 Mar 15)</a>. Differential expression and requirements for Schizosaccharomyces pombe RAD52 homologs in DNA repair and recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14551247" target=_blank>Catlett MG, <i>et al.</i> (2003 Nov)</a>. Schizosaccharomyces pombe Rdh54 (TID1) acts with Rhp54 (RAD54) to repair meiotic double-strand breaks.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13679150" target=_blank>Davis AP, <i>et al.</i> (2003 Oct 7)</a>. The Rad52-Rad59 complex interacts with Rad51 and replication protein A.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15238514" target=_blank>Young JA, <i>et al.</i> (2004 Jun)</a>. Conserved and nonconserved proteins for meiotic DNA breakage and repair in yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16707661" target=_blank>Antúnez de Mayolo A, <i>et al.</i> (2006)</a>. Multiple start codons and phosphorylation result in discrete Rad52 protein species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208257" target=_blank>de Vries FA, <i>et al.</i> (2007 Feb 3)</a>. Schizosaccharomyces pombe Rad22A and Rad22B have similar biochemical properties and form multimeric structures.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18471978" target=_blank>Li F, <i>et al.</i> (2008 May 9)</a>. Microarray-based genetic screen defines SAW1, a gene required for Rad1/Rad10-dependent processing of recombination intermediates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21149262" target=_blank>Sun W, <i>et al.</i> (2011 Mar)</a>. A failure of meiotic chromosome segregation in a fbh1Delta mutant correlates with persistent Rad51-DNA associations.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23779158" target=_blank>Saito Y, <i>et al.</i> (2013)</a>. The proteasome factor Bag101 binds to Rad22 and suppresses homologous recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01027_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01027_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01027_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01027_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bae0b3" class="col">
			FOG01028<br>EOG84XGXV<br>EOG8GQNMB<br><br>sce:HSH155<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>U2-snRNP associated splicing factor; forms extensive associations with the branch site-3' splice site-3' exon region upon prespliceosome formation; similarity to the mammalian U2 snRNP-associated splicing factor SAP155<br><br>
			<br><b>PomBase Description</b><br>U2 snRNP-associated protein Sap155<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have U2 snRNP localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11884590" target=_blank>Ohi MD, <i>et al.</i> (2002 Apr)</a>. Proteomics analysis reveals stable multiprotein complexes in both fission and budding yeasts containing Myb-related Cdc5p/Cef1p, novel pre-mRNA splicing factors, and snRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14517302" target=_blank>Wang Q, <i>et al.</i> (2003 Oct)</a>. Rds3p is required for stable U2 snRNP recruitment to the splicing apparatus.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01028_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01028_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01028_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01028_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>32 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d2b4d9" class="col">
			FOG01029<br>EOG8QRFK1<br><br>sce:PKH1;PKH2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Serine/threonine protein kinase; involved in sphingolipid-mediated signaling pathway that controls endocytosis; activates Ypk1p and Ykr2p, components of signaling cascade required for maintenance of cell wall integrity; redundant with Pkh2p; PKH1 has a paralog, PKH2, that arose from the whole genome duplication|Serine/threonine protein kinase; involved in sphingolipid-mediated signaling pathway that controls endocytosis; activates Ypk1p and Ykr2p, components of signaling cascade required for maintenance of cell wall integrity; redundant with Pkh1p; PKH2 has a paralog, PKH1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>serine/threonine protein kinase Ksg1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have protein serine/threonine kinase activity, role in fungal-type cell wall organization or biogenesis, hyphal growth, response to oxidative stress and cytoplasm localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10567559" target=_blank>Inagaki M, <i>et al.</i> (1999 Dec)</a>. PDK1 homologs activate the Pkc1-mitogen-activated protein kinase pathway in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10071224" target=_blank>Niederberger C, <i>et al.</i> (1999 Feb)</a>. A Schizosaccharomyces pombe gene, ksg1, that shows structural homology to the human phosphoinositide-dependent protein kinase PDK1, is essential for growth, mating and sporulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074427" target=_blank>Casamayor A, <i>et al.</i> (1999 Feb 25)</a>. Functional counterparts of mammalian protein kinases PDK1 and SGK in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11726514" target=_blank>Friant S, <i>et al.</i> (2001 Dec 3)</a>. Sphingoid base signaling via Pkh kinases is required for endocytosis in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12221112" target=_blank>Roelants FM, <i>et al.</i> (2002 Sep)</a>. Pkh1 and Pkh2 differentially phosphorylate and activate Ypk1 and Ykr2 and define protein kinase modules required for maintenance of cell wall integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14625898" target=_blank>Gräub R, <i>et al.</i> (2003)</a>. Ksg1, a homologue of the phosphoinositide-dependent protein kinase 1, controls cell wall integrity in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12805221" target=_blank>Matsuo T, <i>et al.</i> (2003 Jun 16)</a>. Schizosaccharomyces pombe AGC family kinase Gad8p forms a conserved signaling module with TOR and PDK1-like kinases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15470109" target=_blank>Roelants FM, <i>et al.</i> (2004 Oct)</a>. Differential roles of PDK1- and PDK2-phosphorylation sites in the yeast AGC kinases Ypk1, Pkc1 and Sch9.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18034155" target=_blank>Walther TC, <i>et al.</i> (2007 Dec 12)</a>. Pkh-kinases control eisosome assembly and organization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21163942" target=_blank>Luo G, <i>et al.</i> (2011 Mar 18)</a>. Nutrients and the Pkh1/2 and Pkc1 protein kinases control mRNA decay and P-body assembly in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22976295" target=_blank>Nakashima A, <i>et al.</i> (2012 Dec 1)</a>. Psk1, an AGC kinase family member in fission yeast, is directly phosphorylated and controlled by TORC1 and functions as S6 kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23472095" target=_blank>Colabardini AC, <i>et al.</i> (2013)</a>. Functional characterization of Aspergillus nidulans ypkA, a homologue of the mammalian kinase SGK.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24247430" target=_blank>Hálová L, <i>et al.</i> (2013 Nov 25)</a>. Phosphorylation of the TOR ATP binding domain by AGC kinase constitutes a novel mode of TOR inhibition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24281718" target=_blank>Douglas LM, <i>et al.</i> (2013 Nov 26)</a>. The MARVEL domain protein Nce102 regulates actin organization and invasive growth of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23911092" target=_blank>Pastor-Flores D, <i>et al.</i> (2013 Oct 18)</a>. PIF-pocket as a target for C. albicans Pkh selective inhibitors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25416816" target=_blank>Madrid M, <i>et al.</i> (2015 Jan 15)</a>. Multiple layers of regulation influence cell integrity control by the PKC ortholog Pck2 in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25411338" target=_blank>Saitoh S, <i>et al.</i> (2015 Jan 15)</a>. Mechanisms of expression and translocation of major fission yeast glucose transporters regulated by CaMKK/phosphatases, nuclear shuttling, and TOR.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01029_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01029_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01029_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01029_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cc97" class="col">
			FOG01030<br>EOG8BK3JS<br>EOG8GQNMB<br><br>sce:CDC9<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>DNA ligase found in the nucleus and mitochondria; an essential enzyme that joins Okazaki fragments during DNA replication; also acts in ribonucleotide excision repair, base excision repair, and recombination<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent DNA replication ligase Cdc17<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have DNA ligase (ATP) activity, role in DNA ligation, DNA repair, Okazaki fragment processing involved in mitotic DNA replication and mitochondrion, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3909103" target=_blank>Barker DG, <i>et al.</i> (1985 Dec 9)</a>. The nucleotide sequence of the DNA ligase gene (CDC9) from Saccharomyces cerevisiae: a gene which is cell-cycle regulated and induced in response to DNA damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483449" target=_blank>Wehner EP, <i>et al.</i> (1993 Mar)</a>. Molecular structure and genetic regulation of SFA, a gene responsible for resistance to formaldehyde in Saccharomyces cerevisiae, and characterization of its protein product.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10531002" target=_blank>Willer M, <i>et al.</i> (1999 Oct 7)</a>. The yeast CDC9 gene encodes both a nuclear and a mitochondrial form of DNA ligase I.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998526" target=_blank>Goldman GH, <i>et al.</i> (2004 Apr)</a>. Aspergillus nidulans as a model system to characterize the DNA damage response in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01030_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01030_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01030_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01030_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcffa6" class="col">
			FOG01031<br>EOG8QRFK1<br><br>sce:PKC1<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein serine/threonine kinase; essential for cell wall remodeling during growth; localized to sites of polarized growth and the mother-daughter bud neck; homolog of the alpha, beta, and gamma isoforms of mammalian protein kinase C (PKC)<br><br>
			<br><b>PomBase Description</b><br>protein kinase C (PKC)-like Pck1|protein kinase C (PKC)-like Pck2<br><br>
			<br><b>AspGD Description</b><br>Putative protein kinase C<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2196995" target=_blank>Levin DE, <i>et al.</i> (1990 Jul 27)</a>. A candidate protein kinase C gene, PKC1, is required for the S. cerevisiae cell cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1899230" target=_blank>Toda T, <i>et al.</i> (1991 Jan)</a>. Fission yeast genes that confer resistance to staurosporine encode an AP-1-like transcription factor and a protein kinase related to the mammalian ERK1/MAP2 and budding yeast FUS3 and KSS1 kinases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491190" target=_blank>Toda T, <i>et al.</i> (1993 May)</a>. Two novel protein kinase C-related genes of fission yeast are essential for cell viability and implicated in cell shape control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7929623" target=_blank>Kobori H, <i>et al.</i> (1994 May)</a>. Fission yeast protein kinase C gene homologues are required for protoplast regeneration: a functional link between cell wall formation and cell shape control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8943330" target=_blank>Toda T, <i>et al.</i> (1996 Dec)</a>. The fission yeast pmk1+ gene encodes a novel mitogen-activated protein kinase homolog which regulates cell integrity and functions coordinately with the protein kinase C pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8886983" target=_blank>Toda T, <i>et al.</i> (1996 Sep)</a>. The fission yeast sts5+ gene is required for maintenance of growth polarity and functionally interacts with protein kinase C and an osmosensing MAP-kinase pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199286" target=_blank>Sengar AS, <i>et al.</i> (1997 Jul)</a>. Mkh1, a MEK kinase required for cell wall integrity and proper response to osmotic and temperature stress in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9203581" target=_blank>Kominami K, <i>et al.</i> (1997 Jun 15)</a>. Fission yeast WD-repeat protein pop1 regulates genome ploidy through ubiquitin-proteasome-mediated degradation of the CDK inhibitor Rum1 and the S-phase initiator Cdc18.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135147" target=_blank>Zaitsevskaya-Carter T, <i>et al.</i> (1997 Mar 17)</a>. Spm1, a stress-activated MAP kinase that regulates morphogenesis in S.pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9133734" target=_blank>Baymiller J, <i>et al.</i> (1997 Mar 30)</a>. Saccharomyces cerevisiae cell lysis mutations cly5 and cly7 define temperature-sensitive alleles of PKC1, the gene encoding yeast protein kinase C.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9427748" target=_blank>Sugiura R, <i>et al.</i> (1998 Jan 2)</a>. pmp1+, a suppressor of calcineurin deficiency, encodes a novel MAP kinase phosphatase in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10087262" target=_blank>Katayama S, <i>et al.</i> (1999 Mar 22)</a>. Fission yeast alpha-glucan synthase Mok1 requires the actin cytoskeleton to localize the sites of growth and plays an essential role in cell morphogenesis downstream of protein kinase C function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10504305" target=_blank>Arellano M, <i>et al.</i> (1999 Oct)</a>. Schizosaccharomyces pombe protein kinase C homologues, pck1p and pck2p, are targets of rho1p and rho2p and differentially regulate cell integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11102532" target=_blank>Calonge TM, <i>et al.</i> (2000 Dec)</a>. Schizosaccharomyces pombe rho2p GTPase regulates cell wall alpha-glucan biosynthesis through the protein kinase pck2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651902" target=_blank>Sayers LG, <i>et al.</i> (2000 Jan)</a>. Rho-dependence of Schizosaccharomyces pombe Pck2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11016847" target=_blank>Carnero E, <i>et al.</i> (2000 Sep)</a>. Schizosaccharomyces pombe ehs1p is involved in maintaining cell wall integrity and in calcium uptake.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11263963" target=_blank>Won M, <i>et al.</i> (2001 Mar 23)</a>. Rkp1/Cpc2, a fission yeast RACK1 homolog, is involved in actin cytoskeleton organization through protein kinase C, Pck2, signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12398295" target=_blank>Vanzela AP, <i>et al.</i> (2002)</a>. Evidence for carbon source regulated protein kinase A and protein kinase C signaling in the duplication cycle, polarization and septum formation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14625898" target=_blank>Gräub R, <i>et al.</i> (2003)</a>. Ksg1, a homologue of the phosphoinositide-dependent protein kinase 1, controls cell wall integrity in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15923187" target=_blank>Deng L, <i>et al.</i> (2005 Jul 29)</a>. Phosphatidylinositol-4-phosphate 5-kinase regulates fission yeast cell integrity through a phospholipase C-mediated protein kinase C-independent pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16224022" target=_blank>Wu JQ, <i>et al.</i> (2005 Oct 14)</a>. Counting cytokinesis proteins globally and locally in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16598003" target=_blank>Herrmann M, <i>et al.</i> (2006 Apr)</a>. Protein kinase C (PkcA) of Aspergillus nidulans is involved in penicillin production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17005909" target=_blank>Ma Y, <i>et al.</i> (2006 Dec)</a>. Rho2 is a target of the farnesyltransferase Cpp1 and acts upstream of Pmk1 mitogen-activated protein kinase signaling in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16928959" target=_blank>Deng L, <i>et al.</i> (2006 Nov)</a>. Real-time monitoring of calcineurin activity in living cells: evidence for two distinct Ca2+-dependent pathways in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17601879" target=_blank>Fujioka T, <i>et al.</i> (2007 Aug)</a>. MpkA-Dependent and -independent cell wall integrity signaling in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881729" target=_blank>Takada H, <i>et al.</i> (2007 Dec)</a>. Atf1 is a target of the mitogen-activated protein kinase Pmk1 and regulates cell integrity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17118679" target=_blank>Teepe AG, <i>et al.</i> (2007 Jun)</a>. The protein kinase C orthologue PkcA plays a role in cell wall integrity and polarized growth in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17406869" target=_blank>Ronen R, <i>et al.</i> (2007 May)</a>. The Aspergillus nidulans pkcA gene is involved in polarized growth, morphogenesis and maintenance of cell wall integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17986778" target=_blank>Ichinomiya M, <i>et al.</i> (2007 Nov)</a>. A protein kinase C-encoding gene, pkcA, is essential to the viability of the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18793338" target=_blank>Villar-Tajadura MA, <i>et al.</i> (2008 Nov)</a>. Rga2 is a Rho2 GAP that regulates morphogenesis and cell integrity in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18681941" target=_blank>Savoldi M, <i>et al.</i> (2008 Oct)</a>. Farnesol induces the transcriptional accumulation of the Aspergillus nidulans Apoptosis-Inducing Factor (AIF)-like mitochondrial oxidoreductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19486165" target=_blank>Fang Y, <i>et al.</i> (2009 Jun)</a>. Pleiotropic phenotypes caused by an opal nonsense mutation in an essential gene encoding HMG-CoA reductase in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19585695" target=_blank>de Groot PW, <i>et al.</i> (2009 Mar)</a>. Comprehensive genomic analysis of cell wall genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20501954" target=_blank>Koyano T, <i>et al.</i> (2010)</a>. Search for kinases related to transition of growth polarity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20164182" target=_blank>Soto T, <i>et al.</i> (2010 Apr 9)</a>. Rga4 modulates the activity of the fission yeast cell integrity MAPK pathway by acting as a Rho2 GTPase-activating protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21091509" target=_blank>Colabardini AC, <i>et al.</i> (2010 Dec)</a>. Involvement of the Aspergillus nidulans protein kinase C with farnesol tolerance is related to the unfolded protein response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20526834" target=_blank>Chellegatti MA, <i>et al.</i> (2010 May)</a>. Role of phospholipase C and protein kinase C in Aspergillus nidulans during growth on pectin or glucose: Effects on germination and duplication cycle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20739711" target=_blank>Wu JQ, <i>et al.</i> (2010 Nov)</a>. Cooperation between the septins and the actomyosin ring and role of a cell-integrity pathway during cell division in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20624220" target=_blank>Cadou A, <i>et al.</i> (2010 Sep)</a>. Kin1 is a plasma membrane-associated kinase that regulates the cell surface in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21936849" target=_blank>Dinamarco TM, <i>et al.</i> (2011 Oct)</a>. Farnesol-induced cell death in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23209763" target=_blank>Katayama T, <i>et al.</i> (2012)</a>. Involvement of protein kinase C in the suppression of apoptosis and in polarity establishment in Aspergillus nidulans under conditions of heat stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23294323" target=_blank>Cadou A, <i>et al.</i> (2013 Mar)</a>. The Kin1 kinase and the calcineurin phosphatase cooperate to link actin ring assembly and septum synthesis in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25153325" target=_blank>Colabardini AC, <i>et al.</i> (2014)</a>. Protein kinase C overexpression suppresses calcineurin-associated defects in Aspergillus nidulans and is involved in mitochondrial function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24498240" target=_blank>Sánchez-Mir L, <i>et al.</i> (2014)</a>. Rho1 GTPase and PKC ortholog Pck1 are upstream activators of the cell integrity MAPK pathway in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25345444" target=_blank>Katayama T, <i>et al.</i> (2015)</a>. Protein kinase C regulates the expression of cell wall-related genes in RlmA-dependent and independent manners in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25651781" target=_blank>Doi A, <i>et al.</i> (2015 Apr)</a>. Geranylgeranyltransferase Cwg2-Rho4/Rho5 module is implicated in the Pmk1 MAP kinase-mediated cell wall integrity pathway in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25416816" target=_blank>Madrid M, <i>et al.</i> (2015 Jan 15)</a>. Multiple layers of regulation influence cell integrity control by the PKC ortholog Pck2 in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25724996" target=_blank>Jackson-Hayes L, <i>et al.</i> (2015 May-Jun)</a>. Two amino acid sequences direct Aspergillus nidulans protein kinase C (PkcA) localization to hyphal apices and septation sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27451356" target=_blank>Kanda Y, <i>et al.</i> (2016 Aug 15)</a>. Skb5, an SH3 adaptor protein, regulates Pmk1 MAPK signaling by controlling the intracellular localization of the MAPKKK Mkh1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27168121" target=_blank>Chen JS, <i>et al.</i> (2016 Sep)</a>. Discovery of genes involved in mitosis, cell division, cell wall integrity and chromosome segregation through construction of Schizosaccharomyces pombe deletion strains.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01031_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01031_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01031_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01031_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d6baa7" class="col">
			FOG01032<br>EOG837Q17<br>EOG8GQNMB<br><br>sce:CPR1<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic peptidyl-prolyl cis-trans isomerase (cyclophilin); catalyzes the cis-trans isomerization of peptide bonds N-terminal to proline residues; binds the drug cyclosporin A; N-terminally propionylated in vivo; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>cyclophilin family peptidyl-prolyl cis-trans isomerase Cyp2<br><br>
			<br><b>AspGD Description</b><br>Peptidyl-proline cis-trans isomerase; cyclophilin; putative cellular target for cyclosporin A<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2687115" target=_blank>Haendler B, <i>et al.</i> (1989 Nov 15)</a>. Yeast cyclophilin: isolation and characterization of the protein, cDNA and gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2269432" target=_blank>Koser PL, <i>et al.</i> (1990 Dec 15)</a>. A Candida albicans homolog of a human cyclophilin gene encodes a peptidyl-prolyl cis-trans isomerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183184" target=_blank>Dietmeier K, <i>et al.</i> (1990 Jan 25)</a>. Nucleotide sequence of a full-length cDNA coding for cyclophilin (peptidyl-prolyl cis-trans isomerase) of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8431466" target=_blank>Hasumi H, <i>et al.</i> (1993 Feb 13)</a>. Purification and properties of multiple molecular forms of yeast peptidyl prolyl cis-trans isomerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10899127" target=_blank>Arévalo-Rodríguez M, <i>et al.</i> (2000 Jul 17)</a>. Cyclophilin A and Ess1 interact with and regulate silencing by the Sin3-Rpd3 histone deacetylase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11641409" target=_blank>Brown CR, <i>et al.</i> (2001 Dec 21)</a>. Cyclophilin A mediates Vid22p function in the import of fructose-1,6-bisphosphatase into Vid vesicles.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11711434" target=_blank>Pijnappel WW, <i>et al.</i> (2001 Nov 15)</a>. The S. cerevisiae SET3 complex includes two histone deacetylases, Hos2 and Hst1, and is a meiotic-specific repressor of the sporulation gene program.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15937140" target=_blank>Chibana H, <i>et al.</i> (2005 Aug)</a>. Sequence finishing and gene mapping for Candida albicans chromosome 7 and syntenic analysis against the Saccharomyces cerevisiae genome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15643056" target=_blank>Arévalo-Rodríguez M, <i>et al.</i> (2005 Jan)</a>. Cyclophilin A is localized to the nucleus and controls meiosis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870366" target=_blank>Sims AH, <i>et al.</i> (2005 May)</a>. Transcriptome analysis of recombinant protein secretion by Aspergillus nidulans and the unfolded-protein response in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16995943" target=_blank>Pemberton TJ, <i>et al.</i> (2006 Sep 22)</a>. Identification and comparative analysis of sixteen fungal peptidyl-prolyl cis/trans isomerase repertoires.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22984289" target=_blank>Vögtle FN, <i>et al.</i> (2012 Dec)</a>. Intermembrane space proteome of yeast mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22835028" target=_blank>Saykhedkar S, <i>et al.</i> (2012 Jul 26)</a>. A time course analysis of the extracellular proteome of Aspergillus nidulans growing on sorghum stover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24120530" target=_blank>Martins I, <i>et al.</i> (2013 Dec 6)</a>. Proteomic alterations induced by ionic liquids in Aspergillus nidulans and Neurospora crassa.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01032_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01032_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01032_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01032_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f7c8e5" class="col">
			FOG01033<br>EOG87WM69<br>EOG8GQNMB<br><br>sce:TIF5<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Translation initiation factor eIF5; functions both as a GTPase-activating protein to mediate hydrolysis of ribosome-bound GTP and as a GDP dissociation inhibitor to prevent recycling of eIF2<br><br>
			<br><b>PomBase Description</b><br>translation initiation factor eIF5, Tif5(predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative to translation initiation factor eIF-5<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8486705" target=_blank>Chakravarti D, <i>et al.</i> (1993 May 15)</a>. Eukaryotic translation initiation factor 5 from Saccharomyces cerevisiae. Cloning, characterization, and expression of the gene encoding the 45,346-Da protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9671501" target=_blank>Phan L, <i>et al.</i> (1998 Aug)</a>. Identification of a translation initiation factor 3 (eIF3) core complex, conserved in yeast and mammals, that interacts with eIF5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10075937" target=_blank>Asano K, <i>et al.</i> (1999 Mar 15)</a>. Conserved bipartite motifs in yeast eIF5 and eIF2Bepsilon, GTPase-activating and GDP-GTP exchange factors in translation initiation, mediate binding to their common substrate eIF2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12651896" target=_blank>Valásek L, <i>et al.</i> (2003 Mar 15)</a>. The yeast eIF3 subunits TIF32/a, NIP1/c, and eIF5 make critical connections with the 40S ribosome in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01033_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01033_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01033_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01033_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ea999a" class="col">
			FOG01034<br>EOG8QRFK1<br><br>sce:RIM15<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein kinase involved in cell proliferation in response to nutrients; glucose-repressible; involved in signal transduction during cell proliferation in response to nutrients, specifically the establishment of stationary phase; identified as a regulator of IME2; phosphorylates Igo1p and Igo2p; substrate of Pho80p-Pho85p kinase<br><br>
			<br><b>PomBase Description</b><br>serine/threonine protein kinase Cek1|serine/threonine protein kinase Ppk18 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in conidiophore development, hyphal growth and cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9111339" target=_blank>Vidan S, <i>et al.</i> (1997 May)</a>. Stimulation of yeast meiotic gene expression by the glucose-repressible protein kinase Rim15p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9744870" target=_blank>Reinders A, <i>et al.</i> (1998 Sep 15)</a>. Saccharomyces cerevisiae cAMP-dependent protein kinase controls entry into stationary phase through the Rim15p protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16308562" target=_blank>Wanke V, <i>et al.</i> (2005 Dec 21)</a>. Regulation of G0 entry by the Pho80-Pho85 cyclin-CDK complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20471941" target=_blank>Talarek N, <i>et al.</i> (2010 May 14)</a>. Initiation of the TORC1-regulated G0 program requires Igo1/2, which license specific mRNAs to evade degradation via the 5'-3' mRNA decay pathway.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01034_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01034_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01034_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01034_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#aac1de" class="col">
			FOG01035<br>EOG8GQNMB<br>EOG8TDZ6W<br><br>sce:DAN1;PAU23;PAU17;PAU3;PAU19;PAU22;PAU21;PAU15;PAU20;PAU16;PAU5;PAU6;PAU18;PAU1;PAU14;PAU4;PAU12;PAU2;PAU13;PAU9;PAU8;PAU11;PAU24;PAU10;DAN4;PAU7<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cell wall mannoprotein; has similarity to Tir1p, Tir2p, Tir3p, and Tir4p; expressed under anaerobic conditions, completely repressed during aerobic growth|Cell wall mannoprotein; has similarity to Tir1p, Tir2p, Tir3p, and Tir4p; member of the seripauperin multigene family encoded mainly in subtelomeric regions; expressed under anaerobic conditions, completely repressed during aerobic growth|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions; YLL025W is not an essential gene|Member of the seripauperin multigene family; encoded mainly in subtelomeric region; active during alcoholic fermentation; regulated by anaerobiosis; negatively regulated by oxygen; repressed by heme |Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions; identical to Pau21p; encodes 2 proteins that are translated from 2 different start codons|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions; identical to Pau22p; encodes 2 proteins that are translated from 2 different start codons|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions; expression induced by low temperature and also by anaerobic conditions; induced during alcoholic fermentation|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions|Member of the seripauperin multigene family; encoded mainly in subtelomeric regions; induced during alcoholic fermentation; induced by low temperature and also by anaerobic conditions; negatively regulated by oxygen and repressed by heme|Member of the seripauperin multigene family; encoded mainly in subtelomeric regions; active during alcoholic fermentation; regulated by anaerobiosis; negatively regulated by oxygen; repressed by heme; identical to Pau18p|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions; identical to Pau6p|Member of the seripauperin multigene family; encoded mainly in subtelomeric regions; active during alcoholic fermentation; regulated by anaerobiosis, negatively regulated by oxygen; repressed by heme; identical to Pau14p|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions; identical to Pau1p|Member of the seripauperin multigene family; encoded mainly in subtelomeric regions; active during alcoholic fermentation; regulated by anaerobiosis; negatively regulated by oxygen; repressed by heme|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions|Member of the seripauperin multigene family; encoded mainly in subtelomeric region; active during alcoholic fermentation; regulated by anaerobiosis; negatively regulated by oxygen; repressed by heme |Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions; expression is induced after ethanol shock|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions|Putative protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions; mRNA expression appears to be regulated by SUT1 and UPC2|Cell wall mannoprotein; has similarity to Tir1p, Tir2p, Tir3p, and Tir4p; member of the seripauperin multigene family encoded mainly in subtelomeric regions; expressed under anaerobic conditions, completely repressed during aerobic growth|Protein of unknown function; member of the seripauperin multigene family encoded mainly in subtelomeric regions|Cell wall mannoprotein; has similarity to Tir1p, Tir2p, Tir3p, and Tir4p; expressed under anaerobic conditions, completely repressed during aerobic growth|Member of the seripauperin multigene family; active during alcoholic fermentation, regulated by anaerobiosis, inhibited by oxygen, repressed by heme<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304897" target=_blank>Bork P, <i>et al.</i> (1992 Dec)</a>. Comprehensive sequence analysis of the 182 predicted open reading frames of yeast chromosome III.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7926827" target=_blank>Viswanathan M, <i>et al.</i> (1994 Oct 11)</a>. Seripauperins of Saccharomyces cerevisiae: a new multigene family encoding serine-poor relatives of serine-rich proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7785338" target=_blank>Pryde FE, <i>et al.</i> (1995 Apr 15)</a>. Sequence analysis of the right end of chromosome XV in Saccharomyces cerevisiae: an insight into the structural and functional significance of sub-telomeric repeat sequences.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9224891" target=_blank>Sertil O, <i>et al.</i> (1997 Jun 19)</a>. The DAN1 gene of S. cerevisiae is regulated in parallel with the hypoxic genes, but by a different mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613572" target=_blank>Hamada K, <i>et al.</i> (1998 Apr)</a>. Screening for glycosylphosphatidylinositol (GPI)-dependent cell wall proteins in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10322008" target=_blank>Mrsa V, <i>et al.</i> (1999 May)</a>. Deletion of new covalently linked cell wall glycoproteins alters the electrophoretic mobility of phosphorylated wall components of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160904" target=_blank>Cohen BD, <i>et al.</i> (2001 Feb 1)</a>. Induction and repression of DAN1 and the family of anaerobic mannoprotein genes in Saccharomyces cerevisiae occurs through a complex array of regulatory sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292809" target=_blank>Abramova N, <i>et al.</i> (2001 May)</a>. Reciprocal regulation of anaerobic and aerobic cell wall mannoprotein gene expression in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16086015" target=_blank>Verstrepen KJ, <i>et al.</i> (2005 Sep)</a>. Intragenic tandem repeats generate functional variability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19762443" target=_blank>Luo Z, <i>et al.</i> (2009 Dec)</a>. Functional analyses of PAU genes in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01035_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01035_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01035_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01035_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b3dcab" class="col">
			FOG01036<br>EOG837Q17<br>EOG8KSN33<br><br>sce:CPR7<br><br>Genes: 22
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Peptidyl-prolyl cis-trans isomerase (cyclophilin); catalyzes the cis-trans isomerization of peptide bonds N-terminal to proline residues; binds to Hsp82p and contributes to chaperone activity; plays a role in determining prion variants<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8873448" target=_blank>Duina AA, <i>et al.</i> (1996 Aug)</a>. Identification of two CyP-40-like cyclophilins in Saccharomyces cerevisiae, one of which is required for normal growth.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01036_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01036_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01036_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01036_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cdacd5" class="col">
			FOG01037<br>EOG837Q17<br><br>sce:CPR3<br><br>Genes: 19
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mitochondrial peptidyl-prolyl cis-trans isomerase (cyclophilin); catalyzes the cis-trans isomerization of peptide bonds N-terminal to proline residues; involved in protein refolding after import into mitochondria<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1454795" target=_blank>Davis ES, <i>et al.</i> (1992 Dec 1)</a>. A yeast cyclophilin gene essential for lactate metabolism at high temperature.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1547957" target=_blank>McLaughlin MM, <i>et al.</i> (1992 Feb 1)</a>. The yeast cyclophilin multigene family: purification, cloning and characterization of a new isoform.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8431466" target=_blank>Hasumi H, <i>et al.</i> (1993 Feb 13)</a>. Purification and properties of multiple molecular forms of yeast peptidyl prolyl cis-trans isomerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603990" target=_blank>Matouschek A, <i>et al.</i> (1995 Jul 3)</a>. Cyclophilin catalyzes protein folding in yeast mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10025965" target=_blank>Scholz C, <i>et al.</i> (1999 Jan 29)</a>. R73A and H144Q mutants of the yeast mitochondrial cyclophilin Cpr3 exhibit a low prolyl isomerase activity in both peptide and protein-folding assays.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01037_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01037_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01037_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01037_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8c78c" class="col">
			FOG01038<br>EOG8QRFK1<br><br>sce:absent<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01038_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01038_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01038_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01038_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff9d" class="col">
			FOG01039<br>EOG831ZKS<br>EOG8GQNMB<br><br>sce:PIF1<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>DNA helicase, potent G-quadruplex DNA binder/unwinder; possesses strand annealing activity; promotes DNA synthesis during break-induced replication; important for crossover recombination; translation from different start sites produces mitochondrial and nuclear forms; nuclear form is a catalytic inhibitor of telomerase; mitochondrial form involved in DNA repair and recombination; mutations affect Zn, Fe homeostasis; regulated by Rad53p-dependent phosphorylation in rho0 cells<br><br>
			<br><b>PomBase Description</b><br>5' to 3' DNA helicase, involved in DNA recombination and repair Pif1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent 5'-3' DNA helicase activity, single-stranded DNA-dependent ATP-dependent DNA helicase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038524" target=_blank>Foury F, <i>et al.</i> (1987 May)</a>. Cloning and sequencing of the PIF gene involved in repair and recombination of yeast mitochondrial DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1849081" target=_blank>Lahaye A, <i>et al.</i> (1991 Apr)</a>. PIF1: a DNA helicase in yeast mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253734" target=_blank>Lahaye A, <i>et al.</i> (1993 Dec 15)</a>. PIF1 DNA helicase from Saccharomyces cerevisiae. Biochemical characterization of the enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8287473" target=_blank>Schulz VP, <i>et al.</i> (1994 Jan 14)</a>. The saccharomyces PIF1 DNA helicase inhibits telomere elongation and de novo telomere formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10926538" target=_blank>Zhou J, <i>et al.</i> (2000 Aug 4)</a>. Pif1p helicase, a catalytic inhibitor of telomerase in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10693764" target=_blank>Ivessa AS, <i>et al.</i> (2000 Feb 18)</a>. The Saccharomyces Pif1p DNA helicase and the highly related Rrm3p have opposite effects on replication fork progression in ribosomal DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11429610" target=_blank>Myung K, <i>et al.</i> (2001 Jun 28)</a>. Multiple pathways cooperate in the suppression of genome instability in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12024022" target=_blank>O'Rourke TW, <i>et al.</i> (2002 Jun)</a>. Mitochondrial dysfunction due to oxidative mitochondrial DNA damage is reduced through cooperative actions of diverse proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15907372" target=_blank>O'Rourke TW, <i>et al.</i> (2005 Jul 18)</a>. Differential involvement of the related DNA helicases Pif1p and Rrm3p in mtDNA point mutagenesis and stability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15923634" target=_blank>Doudican NA, <i>et al.</i> (2005 Jun)</a>. Oxidative DNA damage causes mitochondrial genomic instability in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16121131" target=_blank>Boulé JB, <i>et al.</i> (2005 Nov 3)</a>. The yeast Pif1p helicase removes telomerase from telomeric DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16537895" target=_blank>Budd ME, <i>et al.</i> (2006 Apr)</a>. Evidence suggesting that Pif1 helicase functions in DNA replication with the Dna2 helicase/nuclease and DNA polymerase delta.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16878131" target=_blank>Eugster A, <i>et al.</i> (2006 Aug)</a>. The finger subdomain of yeast telomerase cooperates with Pif1p to limit telomere elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16816432" target=_blank>Wagner M, <i>et al.</i> (2006 Oct)</a>. The absence of Top3 reveals an interaction between the Sgs1 and Pif1 DNA helicases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720711" target=_blank>Boulé JB, <i>et al.</i> (2007)</a>. The yeast Pif1p DNA helicase preferentially unwinds RNA DNA substrates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17590086" target=_blank>Vega LR, <i>et al.</i> (2007 Jun)</a>. Sensitivity of yeast strains with long G-tails to levels of telomere-bound telomerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17257907" target=_blank>Cheng X, <i>et al.</i> (2007 May)</a>. The role of Pif1p, a DNA helicase in Saccharomyces cerevisiae, in maintaining mitochondrial DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19424434" target=_blank>Ribeyre C, <i>et al.</i> (2009 May)</a>. The yeast Pif1 helicase prevents genomic instability caused by G-quadruplex-forming CEB1 sequences in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20225162" target=_blank>Boulé JB, <i>et al.</i> (2010)</a>. Characterization of the helicase activity and anti-telomerase properties of yeast Pif1p in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20655619" target=_blank>Cheng X, <i>et al.</i> (2010 Oct)</a>. Association of the yeast DNA helicase Pif1p with mitochondrial membranes and mitochondrial DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20795654" target=_blank>Barranco-Medina S, <i>et al.</i> (2010 Oct 5)</a>. DNA binding induces dimerization of Saccharomyces cerevisiae Pif1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21620135" target=_blank>Paeschke K, <i>et al.</i> (2011 May 27)</a>. DNA replication through G-quadruplex motifs is promoted by the Saccharomyces cerevisiae Pif1 DNA helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22927468" target=_blank>Crider DG, <i>et al.</i> (2012 Sep 3)</a>. Rad53 is essential for a mitochondrial DNA inheritance checkpoint regulating G1 to S progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23446274" target=_blank>Galletto R, <i>et al.</i> (2013 Apr)</a>. Translocation of Saccharomyces cerevisiae Pif1 helicase monomers on single-stranded DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23175612" target=_blank>Ramanagoudr-Bhojappa R, <i>et al.</i> (2013 Jan)</a>. Physical and functional interaction between yeast Pif1 helicase and Rim1 single-stranded DNA binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23657261" target=_blank>Paeschke K, <i>et al.</i> (2013 May 23)</a>. Pif1 family helicases suppress genome instability at G-quadruplex motifs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23596008" target=_blank>Ramanagoudr-Bhojappa R, <i>et al.</i> (2013 May 31)</a>. Yeast Pif1 helicase exhibits a one-base-pair stepping mechanism for unwinding duplex DNA.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01039_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01039_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01039_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01039_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d2b39e" class="col">
			FOG01040<br>EOG8THT93<br><br>sce:ASE1<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Mitotic spindle midzone-localized microtubule bundling protein; microtubule-associated protein (MAP) family member; required for spindle elongation and stabilization; undergoes cell cycle-regulated degradation by anaphase promoting complex; potential Cdc28p substrate; relative distribution to microtubules decreases upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>antiparallel microtubule cross-linking factor Ase1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cytokinesis after mitosis checkpoint, microtubule bundle formation involved in horsetail-astral microtubule organization, microtubule bundle formation involved in mitotic spindle midzone assembly<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559759" target=_blank>Pellman D, <i>et al.</i> (1995 Sep)</a>. Two microtubule-associated proteins required for anaphase spindle movement in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01040_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01040_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01040_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01040_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6c2e2" class="col">
			FOG01041<br>EOG82BVQW<br><br>sce:absent<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15725072" target=_blank>Bai C, <i>et al.</i> (2005 Jul 1)</a>. Identification and functional characterization of a novel Candida albicans gene CaMNN5 that suppresses the iron-dependent growth defect of Saccharomyces cerevisiae aft1Delta mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16467465" target=_blank>Bai C, <i>et al.</i> (2006 Feb)</a>. MNN5 encodes an iron-regulated alpha-1,2-mannosyltransferase important for protein glycosylation, cell wall integrity, morphogenesis, and virulence in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633946" target=_blank>Hall RA, <i>et al.</i> (2013)</a>. The Mnn2 mannosyltransferase family modulates mannoprotein fibril length, immune recognition and virulence of Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01041_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01041_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01041_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01041_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e88f90" class="col">
			FOG01042<br>EOG837Q17<br><br>sce:absent<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>cyclophilin family peptidyl-prolyl cis- trans isomerase Cyp3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have peptidyl-prolyl cis-trans isomerase activity, role in protein peptidyl-prolyl isomerization and cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16995943" target=_blank>Pemberton TJ, <i>et al.</i> (2006 Sep 22)</a>. Identification and comparative analysis of sixteen fungal peptidyl-prolyl cis/trans isomerase repertoires.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01042_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01042_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01042_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01042_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a2bbdb" class="col">
			FOG01043<br>EOG831ZKS<br><br>sce:RRM3<br><br>Genes: 15
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>DNA helicase involved in rDNA replication and Ty1 transposition; binds to and suppresses DNA damage at G4 motifs in vivo; relieves replication fork pauses at telomeric regions; structurally and functionally related to Pif1p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8293975" target=_blank>Keil RL, <i>et al.</i> (1993 Nov)</a>. A gene with specific and global effects on recombination of sequences from tandemly repeated genes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10693764" target=_blank>Ivessa AS, <i>et al.</i> (2000 Feb 18)</a>. The Saccharomyces Pif1p DNA helicase and the highly related Rrm3p have opposite effects on replication fork progression in ribosomal DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11779788" target=_blank>Scholes DT, <i>et al.</i> (2001 Dec)</a>. Multiple regulators of Ty1 transposition in Saccharomyces cerevisiae have conserved roles in genome maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12050116" target=_blank>Ivessa AS, <i>et al.</i> (2002 Jun 1)</a>. Saccharomyces Rrm3p, a 5' to 3' DNA helicase that promotes replication fork progression through telomeric and subtelomeric DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12239216" target=_blank>Schmidt KH, <i>et al.</i> (2002 Nov 22)</a>. Saccharomyces cerevisiae RRM3, a 5' to 3' DNA helicase, physically interacts with proliferating cell nuclear antigen.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14690605" target=_blank>Ivessa AS, <i>et al.</i> (2003 Dec)</a>. The Saccharomyces cerevisiae helicase Rrm3p facilitates replication past nonhistone protein-DNA complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12686542" target=_blank>Weitao T, <i>et al.</i> (2003 Jun 20)</a>. Dna2 helicase/nuclease causes replicative fork stalling and double-strand breaks in the ribosomal DNA of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15060145" target=_blank>Schmidt KH, <i>et al.</i> (2004 Apr)</a>. Requirement of Rrm3 helicase for repair of spontaneous DNA lesions in cells lacking Srs2 or Sgs1 helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15060144" target=_blank>Torres JZ, <i>et al.</i> (2004 Apr)</a>. Saccharomyces cerevisiae Rrm3p DNA helicase promotes genome integrity by preventing replication fork stalling: viability of rrm3 cells requires the intra-S-phase checkpoint and fork restart activities.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15542831" target=_blank>Gibson DG, <i>et al.</i> (2004 Dec)</a>. Diminished S-phase cyclin-dependent kinase function elicits vital Rad53-dependent checkpoint responses in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037547" target=_blank>Torres JZ, <i>et al.</i> (2004 Mar 1)</a>. Local chromatin structure at the ribosomal DNA causes replication fork pausing and genome instability in the absence of the S. cerevisiae DNA helicase Rrm3p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15082794" target=_blank>Makovets S, <i>et al.</i> (2004 May)</a>. Anatomy and dynamics of DNA replication fork movement in yeast telomeric regions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15579680" target=_blank>Bessler JB, <i>et al.</i> (2004 Nov)</a>. The amino terminus of the Saccharomyces cerevisiae DNA helicase Rrm3p modulates protein function altering replication and checkpoint activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16103218" target=_blank>Calzada A, <i>et al.</i> (2005 Aug 15)</a>. Molecular anatomy and regulation of a stable replisome at a paused eukaryotic DNA replication fork.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16327883" target=_blank>Budd ME, <i>et al.</i> (2005 Dec)</a>. A network of multi-tasking proteins at the DNA replication fork preserves genome stability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15863512" target=_blank>Chen YB, <i>et al.</i> (2005 Jul 1)</a>. Def1p is involved in telomere maintenance in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15907372" target=_blank>O'Rourke TW, <i>et al.</i> (2005 Jul 18)</a>. Differential involvement of the related DNA helicases Pif1p and Rrm3p in mtDNA point mutagenesis and stability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15829566" target=_blank>Taylor SD, <i>et al.</i> (2005 Jun)</a>. The conserved Mec1/Rad53 nuclear checkpoint pathway regulates mitochondrial DNA copy number in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15775982" target=_blank>Prado F, <i>et al.</i> (2005 Mar 23)</a>. Impairment of replication fork progression mediates RNA polII transcription-associated recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16137624" target=_blank>Szyjka SJ, <i>et al.</i> (2005 Sep 2)</a>. Mrc1 is required for normal progression of replication forks throughout chromatin in S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16631586" target=_blank>Luke B, <i>et al.</i> (2006 Apr 18)</a>. The cullin Rtt101p promotes replication fork progression through damaged DNA and natural pause sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17114583" target=_blank>Azvolinsky A, <i>et al.</i> (2006 Nov 15)</a>. The S. cerevisiae Rrm3p DNA helicase moves with the replication fork and affects replication of all yeast chromosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19414561" target=_blank>Stamenova R, <i>et al.</i> (2009 Jul)</a>. Rrm3 protects the Saccharomyces cerevisiae genome from instability at nascent sites of retrotransposition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19168400" target=_blank>de la Loza MC, <i>et al.</i> (2009 May 1)</a>. Stimulation of direct-repeat recombination by RNA polymerase III transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21172804" target=_blank>Hashash N, <i>et al.</i> (2011 Jan 15)</a>. Regulation of fragile sites expression in budding yeast by MEC1, RRM3 and hydroxyurea.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21087929" target=_blank>Bairwa NK, <i>et al.</i> (2011 Jan 28)</a>. The intra-S phase checkpoint protein Tof1 collaborates with the helicase Rrm3 and the F-box protein Dia2 to maintain genome stability in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23657261" target=_blank>Paeschke K, <i>et al.</i> (2013 May 23)</a>. Pif1 family helicases suppress genome instability at G-quadruplex motifs.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01043_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01043_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01043_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01043_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#abd9a3" class="col">
			FOG01044<br>EOG8TB2TG<br>EOG8THT93<br><br>sce:EST2<br><br>Genes: 13
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Reverse transcriptase subunit of the telomerase holoenzyme; essential for telomerase core catalytic activity, involved in other aspects of telomerase assembly and function; mutations in human homolog are associated with aplastic anemia<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9110970" target=_blank>Lingner J, <i>et al.</i> (1997 Apr 25)</a>. Reverse transcriptase motifs in the catalytic subunit of telomerase.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01044_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01044_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01044_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01044_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>10 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c8a4d1" class="col">
			FOG01045<br>EOG82BVQW<br>EOG87H473<br><br>sce:absent<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633946" target=_blank>Hall RA, <i>et al.</i> (2013)</a>. The Mnn2 mannosyltransferase family modulates mannoprotein fibril length, immune recognition and virulence of Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01045_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01045_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01045_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01045_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f7c181" class="col">
			FOG01046<br>EOG8QJQ38<br><br>sce:absent<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>transcription factor, zf-fungal binuclear cluster type Klf1<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted DNA binding, nucleic acid binding, zinc ion binding activity, role in transcription, DNA-templated and nucleus localization|Has domain(s) with predicted DNA binding, nucleic acid binding, zinc ion binding activity, role in transcription, DNA-templated and nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24385927" target=_blank>Mbogning J, <i>et al.</i> (2013)</a>. The PAF complex and Prf1/Rtf1 delineate distinct Cdk9-dependent pathways regulating transcription elongation in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24167631" target=_blank>Shimanuki M, <i>et al.</i> (2013)</a>. Klf1, a C2H2 zinc finger-transcription factor, is required for cell wall maintenance during long-term quiescence in differentiated G0 phase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695302" target=_blank>Vachon L, <i>et al.</i> (2013 Aug)</a>. Functional characterization of fission yeast transcription factors by overexpression analysis.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01046_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01046_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01046_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01046_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>10 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff93" class="col">
			FOG01047<br>EOG837Q17<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01047_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01047_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01047_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01047_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cdab94" class="col">
			FOG01048<br>EOG88PK32<br>EOG8QRFK1<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>serine/threonine protein kinase Ppk33 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, protein kinase activity, protein serine/threonine kinase activity, transferase activity, transferring phosphorus-containing groups activity and role in protein phosphorylation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15821139" target=_blank>Bimbó A, <i>et al.</i> (2005 Apr)</a>. Systematic deletion analysis of fission yeast protein kinases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16615890" target=_blank>Chikashige Y, <i>et al.</i> (2006 Apr 7)</a>. Meiotic proteins bqt1 and bqt2 tether telomeres to form the bouquet arrangement of chromosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19264558" target=_blank>Deshpande GP, <i>et al.</i> (2009 May 1)</a>. Screening a genome-wide S. pombe deletion library identifies novel genes and pathways involved in genome stability maintenance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20537132" target=_blank>Han TX, <i>et al.</i> (2010)</a>. Global fitness profiling of fission yeast deletion strains by barcode sequencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22235339" target=_blank>Lando D, <i>et al.</i> (2012)</a>. The S. pombe histone H2A dioxygenase Ofd2 regulates gene expression during hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22252817" target=_blank>Fang Y, <i>et al.</i> (2012 Apr)</a>. A genomewide screen in Schizosaccharomyces pombe for genes affecting the sensitivity of antifungal drugs that target ergosterol biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23050226" target=_blank>Chen Z, <i>et al.</i> (2012 Oct)</a>. A genetic screen to discover pathways affecting cohesin function in Schizosaccharomyces pombe identifies chromatin effectors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24167631" target=_blank>Shimanuki M, <i>et al.</i> (2013)</a>. Klf1, a C2H2 zinc finger-transcription factor, is required for cell wall maintenance during long-term quiescence in differentiated G0 phase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26628015" target=_blank>Zhang X, <i>et al.</i> (2015 Oct 9)</a>. Characterization of Tamoxifen as an Antifungal Agent Using the Yeast Schizosaccharomyces Pombe Model Organism.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01048_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01048_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01048_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01048_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f5bcdf" class="col">
			FOG01049<br>EOG8THT93<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01049_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01049_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01049_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01049_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e68586" class="col">
			FOG01050<br>EOG8BK3JS<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>DNA ligase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have nucleus localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01050_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01050_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01050_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01050_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#9ab5d8" class="col">
			FOG01051<br>EOG84QRMW<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01051_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01051_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01051_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01051_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a4d69b" class="col">
			FOG01052<br>EOG8SBCF8<br>EOG8THT93<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01052_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01052_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01052_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01052_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c39ccd" class="col">
			FOG01053<br>EOG8QRFK1<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01053_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01053_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01053_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01053_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6bc76" class="col">
			FOG01054<br>EOG8QRFK1<br><br>sce:PKH3<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein kinase with similarity to mammalian PDK1 and yeast Pkh1p/Phk2p; yeast Pkh1p and Pkh2p are two redundant upstream activators of Pkc1p; identified as a multicopy suppressor of a pkh1 pkh2 double mutant<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10567559" target=_blank>Inagaki M, <i>et al.</i> (1999 Dec)</a>. PDK1 homologs activate the Pkc1-mitogen-activated protein kinase pathway in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11062466" target=_blank>Zhu H, <i>et al.</i> (2000 Nov)</a>. Analysis of yeast protein kinases using protein chips.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01054_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01054_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01054_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01054_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff8a" class="col">
			FOG01055<br>EOG82BVQW<br>EOG8THT93<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633946" target=_blank>Hall RA, <i>et al.</i> (2013)</a>. The Mnn2 mannosyltransferase family modulates mannoprotein fibril length, immune recognition and virulence of Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01055_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01055_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01055_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01055_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9a48b" class="col">
			FOG01056<br>EOG8QRFK1<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular carbohydrate catabolic process, conidiophore development, fungal-type cell wall organization or biogenesis, spore germination<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16087751" target=_blank>Ni M, <i>et al.</i> (2005 Aug)</a>. The pkaB gene encoding the secondary protein kinase A catalytic subunit has a synthetic lethal interaction with pkaA and plays overlapping and opposite roles in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16728950" target=_blank>Yu JH, <i>et al.</i> (2006 Apr)</a>. Heterotrimeric G protein signaling and RGSs in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16546420" target=_blank>Lafon A, <i>et al.</i> (2006 Jul)</a>. G-protein and cAMP-mediated signaling in aspergilli: a genomic perspective.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17030989" target=_blank>Yu JH, <i>et al.</i> (2006 Oct)</a>. Growth and developmental control in the model and pathogenic aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21658084" target=_blank>Bayram O, <i>et al.</i> (2012 Jan)</a>. Coordination of secondary metabolism and development in fungi: the velvet family of regulatory proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23450714" target=_blank>Krijgsheld P, <i>et al.</i> (2013 Mar 15)</a>. Development in Aspergillus.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01056_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01056_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01056_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01056_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4b6dc" class="col">
			FOG01057<br>EOG8QRFK1<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, protein kinase activity, protein serine/threonine kinase activity, transferase activity, transferring phosphorus-containing groups activity and role in protein phosphorylation<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01057_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01057_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01057_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01057_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e47c7d" class="col">
			FOG01058<br>EOG8QJQ38<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01058_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01058_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01058_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01058_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#92b0d5" class="col">
			FOG01059<br>EOG84QRMW<br>EOG86Q5DS<br>EOG8GQNMB<br><br>sce:HAC1<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Basic leucine zipper (bZIP) transcription factor (ATF/CREB1 homolog); regulates the unfolded protein response, via UPRE binding, and membrane biogenesis; ER stress-induced splicing pathway facilitates efficient Hac1p synthesis; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7816617" target=_blank>Nojima H, <i>et al.</i> (1994 Dec 11)</a>. Hac1: a novel yeast bZIP protein binding to the CRE motif is a multicopy suppressor for cdc10 mutant of Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898193" target=_blank>Cox JS, <i>et al.</i> (1996 Nov 1)</a>. A novel mechanism for regulating activity of a transcription factor that controls the unfolded protein response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932376" target=_blank>Nikawa J, <i>et al.</i> (1996 Nov 1)</a>. Saccharomyces cerevisiae IRE2/HAC1 is involved in IRE1-mediated KAR2 expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898194" target=_blank>Sidrauski C, <i>et al.</i> (1996 Nov 1)</a>. tRNA ligase is required for regulated mRNA splicing in the unfolded protein response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9077435" target=_blank>Mori K, <i>et al.</i> (1996 Sep)</a>. Signalling from endoplasmic reticulum to nucleus: transcription factor with a basic-leucine zipper motif is required for the unfolded protein-response pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9348528" target=_blank>Kawahara T, <i>et al.</i> (1997 Oct)</a>. Endoplasmic reticulum stress-induced mRNA splicing permits synthesis of transcription factor Hac1p/Ern4p that activates the unfolded protein response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9323131" target=_blank>Sidrauski C, <i>et al.</i> (1997 Sep 19)</a>. The transmembrane kinase Ire1p is a site-specific endonuclease that initiates mRNA splicing in the unfolded protein response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11595189" target=_blank>Rüegsegger U, <i>et al.</i> (2001 Oct 5)</a>. Block of HAC1 mRNA translation by long-range base pairing is released by cytoplasmic splicing upon induction of the unfolded protein response.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01059_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01059_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01059_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01059_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#9dd393" class="col">
			FOG01060<br>EOG837Q17<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01060_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01060_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01060_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01060_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bf95ca" class="col">
			FOG01061<br>EOG837Q16<br>EOG8GQNMB<br><br>sce:PEX2<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>RING-finger peroxin and E3 ubiquitin ligase; peroxisomal membrane protein with a C-terminal zinc-binding RING domain, forms translocation subcomplex with Pex10p and Pex12p which functions in peroxisomal matrix protein import<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8686377" target=_blank>Liu Y, <i>et al.</i> (1996 Feb)</a>. Independent regulation of full-length and 5'-truncated PAS5 mRNAs in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01061_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01061_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01061_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01061_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6b76b" class="col">
			FOG01062<br>EOG831ZKS<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01062_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01062_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01062_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01062_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff81" class="col">
			FOG01063<br>EOG831ZKS<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01063_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01063_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01063_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01063_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c59d82" class="col">
			FOG01064<br>EOG84QRMW<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Transcription factor involved in regulation of the unfolded protein response; activates transcription of several genes involved in the UPR; during secretion stress, altered transcription start site relieves translational regulation of hacA<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01064_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01064_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01064_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01064_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4b1da" class="col">
			FOG01065<br>EOG8GQNMB<br>EOG8TB2TG<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>telomerase reverse transcriptase 1 protein Trt1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have RNA binding, telomeric template RNA reverse transcriptase activity and role in DNA biosynthetic process, telomere maintenance via telomerase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01065_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01065_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01065_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01065_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e17273" class="col">
			FOG01066<br>EOG8QJQ38<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01066_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01066_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01066_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01066_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#89aad1" class="col">
			FOG01067<br>EOG8QJQ38<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01067_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01067_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01067_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01067_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#95cf8a" class="col">
			FOG01068<br>EOG8QJQ38<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01068_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01068_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01068_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01068_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ba8dc6" class="col">
			FOG01069<br>EOG8GQNMB<br>EOG8VT4GM<br><br>sce:EST3<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Component of the telomerase holoenzyme; involved in telomere replication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8978029" target=_blank>Lendvay TS, <i>et al.</i> (1996 Dec)</a>. Senescence mutants of Saccharomyces cerevisiae with a defect in telomere replication identify three additional EST genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382847" target=_blank>Morris DK, <i>et al.</i> (1997 Dec 1)</a>. Programmed translational frameshifting in a gene required for yeast telomere replication.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9326584" target=_blank>Lingner J, <i>et al.</i> (1997 Oct 14)</a>. Three Ever Shorter Telomere (EST) genes are dispensable for in vitro yeast telomerase activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10619426" target=_blank>Diede SJ, <i>et al.</i> (1999 Dec 23)</a>. Telomerase-mediated telomere addition in vivo requires DNA primase and DNA polymerases alpha and delta.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10898986" target=_blank>Hughes TR, <i>et al.</i> (2000 Jun 29)</a>. The Est3 protein is a subunit of yeast telomerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16418502" target=_blank>Yang CP, <i>et al.</i> (2006)</a>. Saccharomyces cerevisiae Est3p dimerizes in vitro and dimerization contributes to efficient telomere replication in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16884717" target=_blank>Sharanov YS, <i>et al.</i> (2006 Aug 21)</a>. Saccharomyces cerevisiae telomerase subunit Est3p binds DNA and RNA and stimulates unwinding of RNA/DNA heteroduplexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17680761" target=_blank>Sharanov YS, <i>et al.</i> (2007 Jul)</a>. Isolation of active yeast telomerase protein Est3p and investigation of its dimerization in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19172754" target=_blank>Lee J, <i>et al.</i> (2008 Sep)</a>. The Est3 protein associates with yeast telomerase through an OB-fold domain.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01069_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01069_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01069_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01069_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f5b15f" class="col">
			FOG01070<br>EOG80000Z<br>EOG8GQNMB<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01070_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01070_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01070_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01070_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbff77" class="col">
			FOG01071<br>EOG84QRMW<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01071_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01071_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01071_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01071_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c09578" class="col">
			FOG01072<br>EOG8FJ6RJ<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01072_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01072_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01072_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01072_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f3abd7" class="col">
			FOG01073<br>EOG8QRFK1<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>meiosis specific protein kinase Mug27/Slk1|serine/threonine protein kinase Ppk21|serine/threonine protein kinase Sck2<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9560431" target=_blank>Fujita M, <i>et al.</i> (1998 Apr)</a>. S. pombe sck2+, a second homologue of S. cerevisiae SCH9 in fission yeast, encodes a putative protein kinase closely related to PKA in function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143612" target=_blank>Wang L, <i>et al.</i> (2005 Dec)</a>. Schizosaccharomyces pombe adenylate cyclase suppressor mutations suggest a role for cAMP phosphodiesterase regulation in feedback control of glucose/cAMP signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16303567" target=_blank>Martín-Castellanos C, <i>et al.</i> (2005 Nov 22)</a>. A large-scale screen in S. pombe identifies seven novel genes required for critical meiotic events.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18422613" target=_blank>Ohtsuka H, <i>et al.</i> (2008 Jun)</a>. A novel gene, ecl1(+), extends the chronological lifespan in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18411246" target=_blank>Ohtaka A, <i>et al.</i> (2008 May 1)</a>. Mug27 is a meiosis-specific protein kinase that functions in fission yeast meiosis II and sporulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18397994" target=_blank>Pérez-Hidalgo L, <i>et al.</i> (2008 May 1)</a>. Slk1 is a meiosis-specific Sid2-related kinase that coordinates meiotic nuclear division with growth of the forespore membrane.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19409973" target=_blank>Chen BR, <i>et al.</i> (2009 Aug)</a>. A new Schizosaccharomyces pombe chronological lifespan assay reveals that caloric restriction promotes efficient cell cycle exit and extends longevity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20501954" target=_blank>Koyano T, <i>et al.</i> (2010)</a>. Search for kinases related to transition of growth polarity in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20453258" target=_blank>Zuin A, <i>et al.</i> (2010 Apr)</a>. Living on the edge: stress and activation of stress responses promote lifespan extension.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20625380" target=_blank>Liu NN, <i>et al.</i> (2010 Aug)</a>. A genome-wide screen for Schizosaccharomyces pombe deletion mutants that affect telomere length.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20980623" target=_blank>Li WZ, <i>et al.</i> (2010 Dec)</a>. A novel role of Dma1 in regulating forespore membrane assembly and sporulation in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075862" target=_blank>Zuin A, <i>et al.</i> (2010 Mar 3)</a>. Lifespan extension by calorie restriction relies on the Sty1 MAP kinase stress pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291963" target=_blank>Tanae K, <i>et al.</i> (2012)</a>. Histone chaperone Asf1 plays an essential role in maintaining genomic stability in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22806344" target=_blank>Ryuko S, <i>et al.</i> (2012 Aug)</a>. Genome-wide screen reveals novel mechanisms for regulating cobalt uptake and detoxification in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23074192" target=_blank>Becker M, <i>et al.</i> (2012 Dec)</a>. Pmt1, a Dnmt2 homolog in Schizosaccharomyces pombe, mediates tRNA methylation in response to nutrient signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22976295" target=_blank>Nakashima A, <i>et al.</i> (2012 Dec 1)</a>. Psk1, an AGC kinase family member in fission yeast, is directly phosphorylated and controlled by TORC1 and functions as S6 kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23521895" target=_blank>Stephan J, <i>et al.</i> (2013 Aug)</a>. Chemical genetic screen in fission yeast reveals roles for vacuolar acidification, mitochondrial fission, and cellular GMP levels in lifespan extension.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24463365" target=_blank>Rallis C, <i>et al.</i> (2014 Feb 15)</a>. Systematic screen for mutants resistant to TORC1 inhibition in fission yeast reveals genes involved in cellular ageing and growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25102102" target=_blank>Kim JY, <i>et al.</i> (2014 Jul)</a>. A metabolic strategy to enhance long-term survival by Phx1 through stationary phase-specific pyruvate decarboxylases in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26776736" target=_blank>Chica N, <i>et al.</i> (2016 Feb 8)</a>. Nutritional Control of Cell Size by the Greatwall-Endosulfine-PP2A·B55 Pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28011631" target=_blank>Yang HJ, <i>et al.</i> (2017 Feb 15)</a>. Lipid droplet dynamics during &lt;i&gt;Schizosaccharomyces pombe&lt;/i&gt; sporulation and their role in spore survival.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01073_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01073_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01073_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01073_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#df6869" class="col">
			FOG01074<br>EOG851C7V<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted role in isoprenoid biosynthetic process|Has domain(s) with predicted magnesium ion binding, terpene synthase activity and role in isoprenoid biosynthetic process|Has domain(s) with predicted magnesium ion binding, terpene synthase activity and role in isoprenoid biosynthetic process|Farnesyl diphosphate synthetase (FPP synthetase)<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01074_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01074_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01074_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01074_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#81a4ce" class="col">
			FOG01075<br>EOG82BVQW<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01075_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01075_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01075_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01075_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#8ecc82" class="col">
			FOG01076<br>EOG80000Z<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01076_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01076_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01076_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01076_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b585c2" class="col">
			FOG01077<br>EOG84QRMW<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01077_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01077_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01077_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01077_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4ac54" class="col">
			FOG01078<br>EOG8VT4GM<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01078_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01078_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01078_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01078_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbff6e" class="col">
			FOG01079<br>EOG837Q16<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01079_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01079_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01079_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01079_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bc8e6f" class="col">
			FOG01080<br>EOG8GQNMB<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>Schizosaccharomyces specific protein<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11376151" target=_blank>Watanabe T, <i>et al.</i> (2001 Jun 1)</a>. Comprehensive isolation of meiosis-specific genes identifies novel proteins and unusual non-coding transcripts in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16169489" target=_blank>Gregan J, <i>et al.</i> (2005 Sep 20)</a>. Novel genes required for meiotic chromosome segregation are identified by a high-throughput knockout screen in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16615890" target=_blank>Chikashige Y, <i>et al.</i> (2006 Apr 7)</a>. Meiotic proteins bqt1 and bqt2 tether telomeres to form the bouquet arrangement of chromosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16823445" target=_blank>Harigaya Y, <i>et al.</i> (2006 Jul 6)</a>. Selective elimination of messenger RNA prevents an incidence of untimely meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23658229" target=_blank>Sugiyama T, <i>et al.</i> (2013 Jul)</a>. Red5 and three nuclear pore components are essential for efficient suppression of specific mRNAs during vegetative growth of fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01080_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01080_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01080_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01080_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2a5d4" class="col">
			FOG01081<br>EOG831ZKS<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01081_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01081_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01081_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01081_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dd5e5f" class="col">
			FOG01082<br>EOG837Q17<br><br>sce:CPR2<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Peptidyl-prolyl cis-trans isomerase (cyclophilin); catalyzes the cis-trans isomerization of peptide bonds N-terminal to proline residues; has a potential role in the secretory pathway; CPR2 has a paralog, CPR5, that arose from the whole genome duplication; suppresses toxicity of slow-folding human Z-type alpha1-antitrypsin variant associated with liver cirrhosis and emphysema<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183199" target=_blank>Koser PL, <i>et al.</i> (1990 Mar 25)</a>. A second cyclophilin-related gene in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1761234" target=_blank>Koser PL, <i>et al.</i> (1991 Dec 1)</a>. The CYP2 gene of Saccharomyces cerevisiae encodes a cyclosporin A-sensitive peptidyl-prolyl cis-trans isomerase with an N-terminal signal sequence.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01082_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01082_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01082_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01082_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00466_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<!--link to gblock-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00466.png"
	</div>
	<div id="footer">Footer</div>
</div>

</body></html>