<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00192.html">HOG00192&emsp;&emsp;phage and mitochondrial RNA polymerase family</a><br> 
																	HOG00193&emsp;&emsp;<br> 
																	<a href="HOG00194.html">HOG00194&emsp;&emsp;GPI family</a><br><br> 
																	<a href="../sequences/hog/HOG00193.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG00193_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG00193_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG00193_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG00193_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00269<br>EOG85HQDK<br>HXK2<br>sce:HXK2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>HXK II, enzyme and regulator<br><br>
			<br><b>Features</b><br>regulator;enzyme<br><br>
			<br><b>SGD Description</b><br>Hexokinase isoenzyme 2; catalyzes phosphorylation of glucose in the cytosol; predominant hexokinase during growth on glucose; functions in the nucleus to repress expression of HXK1 and GLK1 and to induce expression of its own gene; antiapoptotic; phosphorylation/dephosphorylation at serine-14 by protein kinase Snf1p and protein phosphatase Glc7p-Reg1p regulates nucleocytoplasmic shuttling of Hxk2p; HXK2 has a paralog, HXK1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>hexokinase 1<br><br>
			<br><b>AspGD Description</b><br>Hexokinase<br><br>
			<br><b>Suggested Analysis</b><br>Find duplication origin of GLK1/HXK2<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=355643" target=_blank>Anderson CM, <i>et al.</i> (1978 Jul 25)</a>. Sequencing a protein by x-ray crystallography. II. Refinement of yeast hexokinase B co-ordinates and sequence at 2.1 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6341351" target=_blank>Walsh RB, <i>et al.</i> (1983 May)</a>. Cloning of genes that complement yeast hexokinase and glucokinase mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3905511" target=_blank>Fröhlich KU, <i>et al.</i> (1985)</a>. The primary structure of the yeast hexokinase PII gene (HXK2) which is responsible for glucose repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3003701" target=_blank>Stachelek C, <i>et al.</i> (1986 Jan 24)</a>. Identification, cloning and sequence determination of the genes specifying hexokinase A and B from yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8321195" target=_blank>Prior C, <i>et al.</i> (1993 Jul)</a>. The hexokinase gene is required for transcriptional regulation of the glucose transporter gene RAG1 in Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8322518" target=_blank>Breitwieser W, <i>et al.</i> (1993 May)</a>. Identification of a gene encoding a novel zinc finger protein in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8286332" target=_blank>Kriegel TM, <i>et al.</i> (1994 Jan 11)</a>. In vivo phosphorylation site of hexokinase 2 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7737086" target=_blank>Norbeck J, <i>et al.</i> (1995 Jan)</a>. Gene linkage of two-dimensional polyacrylamide gel electrophoresis resolved proteins from isogene families in Saccharomyces cerevisiae by microsequencing of in-gel trypsin generated peptides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8549830" target=_blank>Petit T, <i>et al.</i> (1996 Jan 8)</a>. Schizosaccharomyces pombe possesses an unusual and a conventional hexokinase: biochemical and molecular characterization of both hexokinases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9047292" target=_blank>Heidrich K, <i>et al.</i> (1997 Feb 25)</a>. Autophosphorylation-inactivation site of hexokinase 2 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9718324" target=_blank>Behlke J, <i>et al.</i> (1998 Aug 25)</a>. Hexokinase 2 from Saccharomyces cerevisiae: regulation of oligomeric structure by in vivo phosphorylation at serine-14.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12882981" target=_blank>Bar D, <i>et al.</i> (2003 Oct 10)</a>. The unique hexokinase of Kluyveromyces lactis. Molecular and functional characterization and evaluation of a role in glucose signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15378761" target=_blank>Pitarch A, <i>et al.</i> (2004 Oct)</a>. Proteomics-based identification of novel Candida albicans antigens for diagnosis of systemic candidiasis in patients with underlying hematological malignancies.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00269_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00269_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00269_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00269_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00193_phobius.html#FOG00269">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00270<br>EOG85HQDK<br>HXK2.2<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized hexokinase paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00269<br><br>
			<br><b>AspGD Description</b><br>Putative hexokinase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17226029" target=_blank>Bernardo SM, <i>et al.</i> (2007 May)</a>. Characterization of regulatory non-catalytic hexokinases in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20453072" target=_blank>Fleck CB, <i>et al.</i> (2010 Jul)</a>. Aspergillus fumigatus catalytic glucokinase and hexokinase: expression analysis and importance for germination, growth, and conidiation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00270_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00270_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00270_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00270_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00193_phobius.html#FOG00270">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00271<br>EOG85HQDK<br>HXK3<br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized HXK3, Pichia clade paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00269<br><br>
			<br><b>Suggested Analysis</b><br>Probe role of 388-392 insertition. Does the protein maintain its regulator function in dbx which has lost HXK2 ortholog?<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00271_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00271_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00271_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00271_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00193_phobius.html#FOG00271">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00272<br>EOG85HQDK<br>HXK1<br>sce:HXK1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>HXK ohnolog, not regulator; highest expression on non-glucose carbon<br><br>
			<br><b>Parent</b><br>ohnolog:FOG00269<br><br>
			<br><b>SGD Description</b><br>Hexokinase isoenzyme 1; a cytosolic protein that catalyzes phosphorylation of glucose during glucose metabolism; expression is highest during growth on non-glucose carbon sources; glucose-induced repression involves hexokinase Hxk2p; HXK1 has a paralog, HXK2, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7001031" target=_blank>Bennett WS Jr, <i>et al.</i> (1980 Jun 25)</a>. Structure of a complex between yeast hexokinase A and glucose. I. Structure determination and refinement at 3.5 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3908224" target=_blank>Kopetzki E, <i>et al.</i> (1985)</a>. Complete nucleotide sequence of the hexokinase PI gene (HXK1) of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3003701" target=_blank>Stachelek C, <i>et al.</i> (1986 Jan 24)</a>. Identification, cloning and sequence determination of the genes specifying hexokinase A and B from yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3131329" target=_blank>Tamura JK, <i>et al.</i> (1988 Jun 5)</a>. The adenine nucleotide binding site on yeast hexokinase PII. Affinity labeling of Lys-111 by pyridoxal 5'-diphospho-5'-adenosine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572255" target=_blank>Petit T, <i>et al.</i> (1999 Nov)</a>. Molecular cloning and characterization of the gene HXK1 encoding the hexokinase from Yarrowia lipolytica.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00272_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00272_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00272_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00272_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00193_phobius.html#FOG00272">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00273<br>EOG85HQDK<br>GLK1<br>sce:GLK1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Glucokinase.<br><br>
			<br><b>Parent</b><br>paralog?:FOG00269<br><br>
			<br><b>SGD Description</b><br>Glucokinase; catalyzes the phosphorylation of glucose at C6 in the first irreversible step of glucose metabolism; one of three glucose phosphorylating enzymes; expression regulated by non-fermentable carbon sources; GLK1 has a paralog, EMI2, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>hexokinase 2<br><br>
			<br><b>AspGD Description</b><br>Putative glucokinase<br><br>
			<br><b>Suggested Analysis</b><br>Find origin of GLK1 or HXK2<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=799641" target=_blank>Felden RA, <i>et al.</i> (1976 Mar)</a>. Presence of histones in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6341351" target=_blank>Walsh RB, <i>et al.</i> (1983 May)</a>. Cloning of genes that complement yeast hexokinase and glucokinase mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3072253" target=_blank>Albig W, <i>et al.</i> (1988 Dec 15)</a>. Structure of yeast glucokinase, a strongly diverged specific aldo-hexose-phosphorylating isoenzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2274040" target=_blank>Ehinger A, <i>et al.</i> (1990 Jul)</a>. Sequence, organization and expression of the core histone genes of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8674973" target=_blank>Lee DW, <i>et al.</i> (1996 Apr 15)</a>. Quantitative analysis of gene expression in sexual structures of Aspergillus nidulans by sequencing of 3'-directed cDNA clones.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8549830" target=_blank>Petit T, <i>et al.</i> (1996 Jan 8)</a>. Schizosaccharomyces pombe possesses an unusual and a conventional hexokinase: biochemical and molecular characterization of both hexokinases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15311944" target=_blank>Trojer P, <i>et al.</i> (2004 Aug 24)</a>. Histone methyltransferases in Aspergillus nidulans: evidence for a novel enzyme with a unique substrate specificity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17573926" target=_blank>Kettner K, <i>et al.</i> (2007 Aug)</a>. Identification and characterization of a novel glucose-phosphorylating enzyme in Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18296621" target=_blank>Reyes-Dominguez Y, <i>et al.</i> (2008 Apr)</a>. Nucleosome positioning and histone H3 acetylation are independent processes in the Aspergillus nidulans prnD-prnB bidirectional promoter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673441" target=_blank>Berger H, <i>et al.</i> (2008 Sep)</a>. Dissecting individual steps of nitrogen transcription factor cooperation in the Aspergillus nidulans nitrate cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20453072" target=_blank>Fleck CB, <i>et al.</i> (2010 Jul)</a>. Aspergillus fumigatus catalytic glucokinase and hexokinase: expression analysis and importance for germination, growth, and conidiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132440" target=_blank>Reyes-Dominguez Y, <i>et al.</i> (2010 Jun)</a>. Heterochromatic marks are associated with the repression of secondary metabolism clusters in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23892751" target=_blank>Nützmann HW, <i>et al.</i> (2013 Oct)</a>. Distinct amino acids of histone H3 control secondary metabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24907261" target=_blank>Cánovas D, <i>et al.</i> (2014 Aug)</a>. The histone acetyltransferase GcnE (GCN5) plays a central role in the regulation of Aspergillus asexual development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835271" target=_blank>Govindaraghavan M, <i>et al.</i> (2014 Aug)</a>. The Set1/COMPASS histone H3 methyltransferase helps regulate mitosis with the CDK1 and NIMA mitotic kinases in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26213895" target=_blank>, <i>et al.</i> (2015 Aug)</a>. KdmA, a histone H3 demethylase with bipartite function, differentially regulates primary and secondary metabolism in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25980340" target=_blank>Hedtke M, <i>et al.</i> (2015 Aug)</a>. Light-dependent gene activation in Aspergillus nidulans is strictly dependent on phytochrome and involves the interplay of phytochrome and white collar-regulated histone H3 acetylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25712266" target=_blank>Gacek-Matthews A, <i>et al.</i> (2015 May)</a>. KdmA, a histone H3 demethylase with bipartite function, differentially regulates primary and secondary metabolism in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00273_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00273_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00273_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00273_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00193_phobius.html#FOG00273">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00274<br>EOG85HQDK<br>EMI2<br>sce:EMI2<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Non-catalytic protein<br><br>
			<br><b>Parent</b><br>ohnolog:FOG00273<br><br>
			<br><b>SGD Description</b><br>Non-essential protein of unknown function; required for transcriptional induction of the early meiotic-specific transcription factor IME1; required for sporulation; expression regulated by glucose-repression transcription factors Mig1/2p; EMI2 has a paralog, GLK1, that arose from the whole genome duplication; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9832517" target=_blank>Lutfiyya LL, <i>et al.</i> (1998 Dec)</a>. Characterization of three related glucose repressors and genes they regulate in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586695" target=_blank>Enyenihi AH, <i>et al.</i> (2003 Jan)</a>. Large-scale functional genomic analysis of sporulation and meiosis in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00274_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00274_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00274_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00274_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00193_phobius.html#FOG00274">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00275<br>EOG82Z36G<br>EOG84B8JM<br>EOG85HQDK<br><br>sce:absent<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Putative hexokinase<br><br>
			<br><b>Parent</b><br>paralog:FOG00269<br><br>
			<br><b>SGD Description</b><br>Putative hexokinase; transcript is upregulated during sporulation and the unfolded protein response; YLR446W is not an essential gene<br><br>
			<br><b>AspGD Description</b><br>Putative hexokinase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11114181" target=_blank>Kumar MJ, <i>et al.</i> (2000 Dec 19)</a>. The inducible N-acetylglucosamine catabolic pathway gene cluster in Candida albicans: discrete N-acetylglucosamine-inducible factors interact at the promoter of NAG1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298769" target=_blank>Yamada-Okabe T, <i>et al.</i> (2001 Apr)</a>. Identification and characterization of the genes for N-acetylglucosamine kinase and N-acetylglucosamine-phosphate deacetylase in the pathogenic fungus Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11705974" target=_blank>Singh P, <i>et al.</i> (2001 Dec)</a>. Attenuation of virulence and changes in morphology in Candida albicans by disruption of the N-acetylglucosamine catabolic pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17139615" target=_blank>Wendland J, <i>et al.</i> (2006)</a>. Use of the Porcine Intestinal Epithelium (PIE)-Assay to analyze early stages of colonization by the human fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16987174" target=_blank>Bennett RJ, <i>et al.</i> (2006 Oct)</a>. The role of nutrient regulation and the Gpa2 protein in the mating pheromone response of C. albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17226029" target=_blank>Bernardo SM, <i>et al.</i> (2007 May)</a>. Characterization of regulatory non-catalytic hexokinases in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19648376" target=_blank>Wendland J, <i>et al.</i> (2009 Sep)</a>. N-acetylglucosamine utilization by Saccharomyces cerevisiae based on expression of Candida albicans NAG genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20675577" target=_blank>Gunasekera A, <i>et al.</i> (2010 Oct)</a>. Identification of GIG1, a GlcNAc-induced gene in Candida albicans needed for normal sensitivity to the chitin synthase inhibitor nikkomycin Z.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21700702" target=_blank>Naseem S, <i>et al.</i> (2011 Aug 19)</a>. N-acetylglucosamine (GlcNAc) induction of hyphal morphogenesis and transcriptional responses in Candida albicans are not dependent on its metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23341961" target=_blank>Rao KH, <i>et al.</i> (2013)</a>. N-acetylglucosamine kinase, HXK1 is involved in morphogenetic transition and metabolic gene expression in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24491547" target=_blank>Rao KH, <i>et al.</i> (2014 Feb 28)</a>. N-acetylglucosamine kinase, HXK1 contributes to white-opaque morphological transition in Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00275_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00275_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00275_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00275_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00193_phobius.html#FOG00275">Phobius transmembrane predictions</a>
										<br>14 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG00193.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>