<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01456.html">HOG01456&emsp;&emsp;DASH complex ASK1 family</a><br> 
																	HOG01457&emsp;&emsp;<br> 
																	<a href="HOG01458.html">HOG01458&emsp;&emsp;MAD1 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01457.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01457_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01457_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01457_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG04209<br>EOG8280JQ<br><br>sce:absent<br><br>Genes: 19
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11069668" target=_blank>Schweizer A, <i>et al.</i> (2000 Nov)</a>. The TEA/ATTS transcription factor CaTec1p regulates hyphal development and virulence in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11595734" target=_blank>Lane S, <i>et al.</i> (2001 Dec 28)</a>. DNA array studies demonstrate convergent regulation of virulence factors by Cph1, Cph2, and Efg1 in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11533231" target=_blank>Lane S, <i>et al.</i> (2001 Oct)</a>. The basic helix-loop-helix transcription factor Cph2 regulates hyphal development in Candida albicans partly via TEC1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039365" target=_blank>Staib P, <i>et al.</i> (2004 Apr)</a>. Tec1p-independent activation of a hypha-associated Candida albicans virulence gene during infection.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15964282" target=_blank>Nobile CJ, <i>et al.</i> (2005 Jun 21)</a>. Regulation of cell-surface genes and biofilm formation by the C. albicans transcription factor Bcr1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755921" target=_blank>Bassilana M, <i>et al.</i> (2005 Mar)</a>. Regulation of the Cdc42/Cdc24 GTPase module during Candida albicans hyphal growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15819622" target=_blank>Biswas K, <i>et al.</i> (2005 May)</a>. The Mep2p ammonium permease controls nitrogen starvation-induced filamentous growth in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321041" target=_blank>Li F, <i>et al.</i> (2005 Nov-Dec)</a>. Identification of Candida albicans genes that induce Saccharomyces cerevisiae cell adhesion and morphogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16710830" target=_blank>Umeyama T, <i>et al.</i> (2006 May)</a>. Repression of CDC28 reduces the expression of the morphology-related transcription factors, Efg1p, Nrg1p, Rbf1p, Rim101p, Fkh2p and Tec1p and induces cell elongation in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17277173" target=_blank>Argimón S, <i>et al.</i> (2007 Apr)</a>. Developmental regulation of an adhesin gene during cellular morphogenesis in the fungal pathogen Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17257864" target=_blank>Eckert SE, <i>et al.</i> (2007 May)</a>. PGA4, a GAS homologue from Candida albicans, is up-regulated early in infection processes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19197361" target=_blank>Bastidas RJ, <i>et al.</i> (2009 Feb)</a>. The protein kinase Tor1 regulates adhesin gene expression in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19346360" target=_blank>Stichternoth C, <i>et al.</i> (2009 Jun)</a>. Hypoxic adaptation by Efg1 regulates biofilm formation by Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435697" target=_blank>Rosenbach A, <i>et al.</i> (2010 Jul)</a>. Adaptations of Candida albicans for growth in the mammalian intestinal tract.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20223293" target=_blank>Fuchs BB, <i>et al.</i> (2010 Jun)</a>. Role of filamentation in Galleria mellonella killing by Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20150241" target=_blank>Holcombe LJ, <i>et al.</i> (2010 May)</a>. Pseudomonas aeruginosa secreted factors impair biofilm development in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20454615" target=_blank>Sahni N, <i>et al.</i> (2010 May 4)</a>. Tec1 mediates the pheromone response of the white phenotype of Candida albicans: insights into the evolution of new signal transduction pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21357478" target=_blank>Shareck J, <i>et al.</i> (2011 Apr)</a>. Conjugated linoleic acid inhibits hyphal growth in Candida albicans by modulating Ras1p cellular levels and downregulating TEC1 expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21407800" target=_blank>Wächtler B, <i>et al.</i> (2011 Feb 23)</a>. From attachment to damage: defined genes of Candida albicans mediate adhesion, invasion and damage during interaction with oral epithelial cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21498643" target=_blank>Kim Y, <i>et al.</i> (2011 Jun)</a>. Killing of Candida albicans filaments by Salmonella enterica serovar Typhimurium is mediated by sopB effectors, parts of a type III secretion system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22276126" target=_blank>Du H, <i>et al.</i> (2012)</a>. Roles of Candida albicans Gat2, a GATA-type zinc finger transcription factor, in biofilm formation, filamentous growth and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23236295" target=_blank>Hnisz D, <i>et al.</i> (2012)</a>. A histone deacetylase adjusts transcription kinetics at coding sequences during Candida albicans morphogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22265407" target=_blank>Nobile CJ, <i>et al.</i> (2012 Jan 20)</a>. A recently evolved transcriptional network controls biofilm development in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22544909" target=_blank>Fanning S, <i>et al.</i> (2012 Jul)</a>. Divergent targets of Candida albicans biofilm regulator Bcr1 in vitro and in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23117819" target=_blank>Fox EP, <i>et al.</i> (2012 Nov-Dec)</a>. A sticky situation: untangling the transcriptional network controlling biofilm development in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22923045" target=_blank>Daniels KJ, <i>et al.</i> (2012 Oct)</a>. The "finger," a unique multicellular morphology of Candida albicans induced by CO2 and dependent upon the Ras1-cyclic AMP pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23637598" target=_blank>Lin CH, <i>et al.</i> (2013)</a>. Genetic control of conventional and pheromone-stimulated biofilm formation in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24260276" target=_blank>Zhao LX, <i>et al.</i> (2013)</a>. Effect of tetrandrine against Candida albicans biofilms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23766273" target=_blank>Bandara HM, <i>et al.</i> (2013 Aug)</a>. Secretory products of Escherichia coli biofilm modulate Candida biofilm formation and hyphal development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23808664" target=_blank>Guan G, <i>et al.</i> (2013 Aug)</a>. Bcr1 plays a central role in the regulation of opaque cell filamentation in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23966855" target=_blank>Znaidi S, <i>et al.</i> (2013 Aug)</a>. A comprehensive functional portrait of two heat shock factor-type transcriptional regulators involved in Candida albicans morphogenesis and virulence.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04209_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04209_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04209_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04209_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01457_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG04210<br>EOG8280JQ<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04210_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04210_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04210_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04210_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01457_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG04211<br>EOG8280JQ<br><br>sce:TEC1<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Transcription factor targeting filamentation genes and Ty1 expression; Ste12p activation of most filamentation gene promoters depends on Tec1p and Tec1p transcriptional activity is dependent on its association with Ste12p; binds to TCS elements upstream of filamentation genes, which are regulated by Tec1p/Ste12p/Dig1p complex; competes with Dig2p for binding to Ste12p/Dig1p; positive regulator of chronological life span; TEA/ATTS DNA-binding domain family member<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2192259" target=_blank>Laloux I, <i>et al.</i> (1990 Jul)</a>. TEC1, a gene involved in the activation of Ty1 and Ty1-mediated gene expression in Saccharomyces cerevisiae: cloning and molecular analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2070413" target=_blank>Bürglin TR, <i>et al.</i> (1991 Jul 12)</a>. The TEA domain: a novel, highly conserved DNA-binding motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04211_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04211_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04211_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04211_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01457_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG04212<br>EOG8280JQ<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative transcription factor similar to A. nidulans abaA; predicted role in conidiophore development; expressed in germinating conidia<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5366214" target=_blank>Clutterbuck AJ, <i>et al.</i> (1969 Oct)</a>. A mutational analysis of conidial development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4581424" target=_blank>Clutterbuck AJ, <i>et al.</i> (1973 Jun)</a>. Gene symbols in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=396353" target=_blank>Martinelli SD, <i>et al.</i> (1979 Oct)</a>. Phenotypes of double conidiation mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823119" target=_blank>Boylan MT, <i>et al.</i> (1987 Sep)</a>. Isolation and physical characterization of three essential conidiation genes from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3293800" target=_blank>Adams TH, <i>et al.</i> (1988 Jul 29)</a>. brlA is necessary and sufficient to direct conidiophore development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3076298" target=_blank>Timberlake WE, <i>et al.</i> (1988 Jun)</a>. Genetic regulation of development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2655931" target=_blank>Mirabito PM, <i>et al.</i> (1989 Jun 2)</a>. Interactions of three sequentially expressed genes control temporal and spatial specificity in Aspergillus development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2088177" target=_blank>Timberlake WE, <i>et al.</i> (1990)</a>. Molecular genetics of Aspergillus development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2152124" target=_blank>Sewall TC, <i>et al.</i> (1990 Aug)</a>. abaA controls phialide differentiation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2196567" target=_blank>Adams TH, <i>et al.</i> (1990 Jul)</a>. Developmental repression of growth and gene expression in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2117702" target=_blank>Adams TH, <i>et al.</i> (1990 Sep)</a>. Upstream elements repress premature expression of an Aspergillus developmental regulatory gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2283502" target=_blank>Clutterbuck AJ, <i>et al.</i> (1990 Sep)</a>. The genetics of conidiophore pigmentation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2253875" target=_blank>Mooney JL, <i>et al.</i> (1990 Sep)</a>. Light is required for conidiation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1820817" target=_blank>Andrianopoulos A, <i>et al.</i> (1991 Aug)</a>. ATTS, a new and conserved DNA binding domain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1901560" target=_blank>Birse CE, <i>et al.</i> (1991 Feb 1)</a>. Isolation and developmentally regulated expression of an Aspergillus nidulans phenol oxidase-encoding gene, ivoB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1986246" target=_blank>Marshall MA, <i>et al.</i> (1991 Jan)</a>. Aspergillus nidulans wetA activates spore-specific gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2070413" target=_blank>Bürglin TR, <i>et al.</i> (1991 Jul 12)</a>. The TEA domain: a novel, highly conserved DNA-binding motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1745237" target=_blank>Lloyd AT, <i>et al.</i> (1991 Nov)</a>. Codon usage in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840892" target=_blank>Timberlake WE, <i>et al.</i> (1991 Oct)</a>. Temporal and spatial controls of Aspergillus development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1504597" target=_blank>Yager LN, <i>et al.</i> (1992)</a>. Early developmental events during asexual and sexual sporulation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1516832" target=_blank>Miller KY, <i>et al.</i> (1992 Sep)</a>. StuA is required for cell pattern formation in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8316075" target=_blank>Aguirre J, <i>et al.</i> (1993 Apr)</a>. Spatial and temporal controls of the Aspergillus brlA developmental regulatory gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491194" target=_blank>Aramayo R, <i>et al.</i> (1993 May)</a>. The Aspergillus nidulans yA gene is regulated by abaA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12271040" target=_blank>Timberlake WE, <i>et al.</i> (1993 Oct)</a>. Translational Triggering and Feedback Fixation in the Control of Fungal Development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765135" target=_blank>Timberlake WE, <i>et al.</i> (1994)</a>. Genetic regulation of conidiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8139553" target=_blank>Andrianopoulos A, <i>et al.</i> (1994 Apr)</a>. The Aspergillus nidulans abaA gene encodes a transcriptional activator that acts as a genetic switch to control development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7704830" target=_blank>Sewall TC, <i>et al.</i> (1994 Dec)</a>. Cellular effects of misscheduled brlA, abaA, and wetA expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8056325" target=_blank>Gems DH, <i>et al.</i> (1994 May)</a>. Enhancers of conidiation mutants in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8078481" target=_blank>Prade RA, <i>et al.</i> (1994 Sep 1)</a>. The Penicillium chrysogenum and Aspergillus nidulans wetA developmental regulatory genes are functionally equivalent.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651135" target=_blank>Stringer MA, <i>et al.</i> (1995 Apr)</a>. dewA encodes a fungal hydrophobin component of the Aspergillus spore wall.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8598056" target=_blank>Navarro RE, <i>et al.</i> (1996 Mar)</a>. catA, a new Aspergillus nidulans gene encoding a developmentally regulated catalase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8722771" target=_blank>Busby TM, <i>et al.</i> (1996 May)</a>. Suppression and enhancement of the Aspergillus nidulans medusa mutation by altered dosage of the bristle and stunted genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969518" target=_blank>Karos M, <i>et al.</i> (1996 Nov)</a>. hymA (hypha-like metulae), a new developmental mutant of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9073485" target=_blank>Marhoul J, <i>et al.</i> (1997 Feb)</a>. Isolation of Aspergillus nidulans Mutants That Overcome brlA-Induced Growth Arrest<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9126620" target=_blank>Marhoul J, <i>et al.</i> (1997 Feb)</a>. Isolation of Aspergillus nidulans mutants that overcome brlA-induced growth arrest.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312029" target=_blank>Dutton JR, <i>et al.</i> (1997 Sep 15)</a>. StuAp is a sequence-specific transcription factor that regulates developmental complexity in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9717578" target=_blank>Guzmán-de-Peña D, <i>et al.</i> (1998 Feb)</a>. Correlation between the regulation of sterigmatocystin biosynthesis and asexual and sexual sporulation in Emericella nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529886" target=_blank>Adams TH, <i>et al.</i> (1998 Mar)</a>. Asexual sporulation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10441449" target=_blank>Stephens KE, <i>et al.</i> (1999 Jul-Aug)</a>. Functional analysis of DNA sequences required for conidium-specific expression of the SpoC1-C1C gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10792717" target=_blank>Vallim MA, <i>et al.</i> (2000 Apr)</a>. Aspergillus SteA (sterile12-like) is a homeodomain-C2/H2-Zn+2 finger transcription factor required for sexual reproduction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123677" target=_blank>Borneman AR, <i>et al.</i> (2000 Dec)</a>. The abaA homologue of Penicillium marneffei participates in two developmental programmes: conidiation and dimorphic growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931277" target=_blank>Pascon RC, <i>et al.</i> (2000 Jun)</a>. Morphogenesis in Aspergillus nidulans requires Dopey (DopA), a member of a novel family of leucine zipper-like proteins conserved from yeast to humans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11359914" target=_blank>Schier N, <i>et al.</i> (2001 Jun)</a>. A Pcl-like cyclin of Aspergillus nidulans is transcriptionally activated by developmental regulators and is involved in sporulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11683268" target=_blank>Han S, <i>et al.</i> (2001 Oct)</a>. Complex control of the developmental regulatory locus brlA in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12409104" target=_blank>Osherov N, <i>et al.</i> (2002 Nov)</a>. Identification of conidial-enriched transcripts in Aspergillus nidulans using suppression subtractive hybridization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12455692" target=_blank>Scherer M, <i>et al.</i> (2002 Oct)</a>. Aspergillus nidulans catalase-peroxidase gene (cpeA) is transcriptionally induced during sexual development through the transcription factor StuA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12553937" target=_blank>Ehrlich KC, <i>et al.</i> (2003 Feb)</a>. Sequence comparison of aflR from different Aspergillus species provides evidence for variability in regulation of aflatoxin production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670687" target=_blank>Park BC, <i>et al.</i> (2003 Mar 28)</a>. Activation of chsC transcription by AbaA during asexual development of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16082523" target=_blank>Ichinomiya M, <i>et al.</i> (2005 Sep)</a>. Expression of asexual developmental regulator gene abaA is affected in the double mutants of classes I and II chitin synthase genes, chsC and chsA, of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17427290" target=_blank>Cánovas D, <i>et al.</i> (2006 Dec)</a>. Developmental regulation of the glyoxylate cycle in the human pathogen Penicillium marneffei.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17030989" target=_blank>Yu JH, <i>et al.</i> (2006 Oct)</a>. Growth and developmental control in the model and pathogenic aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18975147" target=_blank>Emri T, <i>et al.</i> (2008 Dec)</a>. Regulation of autolysis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18651309" target=_blank>Horiuchi H, <i>et al.</i> (2009)</a>. Functional diversity of chitin synthases of Aspergillus nidulans in hyphal growth, conidiophore development and septum formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19627503" target=_blank>Harris SD, <i>et al.</i> (2009 Sep)</a>. The Spitzenkörper: a signalling hub for the control of fungal development?<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23956662" target=_blank>Yu JH, <i>et al.</i> (2010 Dec)</a>. Regulation of Development in Aspergillus nidulans and Aspergillus fumigatus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21152013" target=_blank>Sarikaya Bayram O, <i>et al.</i> (2010 Dec 2)</a>. LaeA control of velvet family regulatory proteins for light-dependent development and fungal cell-type specificity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19850144" target=_blank>Ogawa M, <i>et al.</i> (2010 Jan)</a>. Genetic analysis of conidiation regulatory pathways in koji-mold Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20624219" target=_blank>Kwon NJ, <i>et al.</i> (2010 Sep)</a>. FlbC is a putative nuclear C2H2 transcription factor regulating development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23049895" target=_blank>Park HS, <i>et al.</i> (2012)</a>. The role, interaction and regulation of the velvet regulator VelB in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092920" target=_blank>Park HS, <i>et al.</i> (2012 Dec)</a>. Genetic control of asexual sporulation in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22651396" target=_blank>Si H, <i>et al.</i> (2012 Jul)</a>. Morphogenetic and developmental functions of the Aspergillus nidulans homologues of the yeast bud site selection proteins Bud4 and Axl2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24039821" target=_blank>Son H, <i>et al.</i> (2013)</a>. AbaA regulates conidiogenesis in the ascomycete fungus Fusarium graminearum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24161728" target=_blank>Kang JY, <i>et al.</i> (2013 Dec)</a>. The MpkB MAP kinase plays a role in autolysis and conidiation of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24385358" target=_blank>Lee JY, <i>et al.</i> (2013 Dec)</a>. A putative APSES transcription factor is necessary for normal growth and development of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23264642" target=_blank>Garzia A, <i>et al.</i> (2013 Feb)</a>. Transcriptional changes in the transition from vegetative cells to asexual development in the model fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23450714" target=_blank>Krijgsheld P, <i>et al.</i> (2013 Mar 15)</a>. Development in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24587098" target=_blank>Park HS, <i>et al.</i> (2014)</a>. VelC positively controls sexual development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24794637" target=_blank>Yu Z, <i>et al.</i> (2014 Jul-Aug)</a>. A cyclin-like protein, ClgA, regulates development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24532783" target=_blank>Lee MK, <i>et al.</i> (2014 May)</a>. NsdD is a key repressor of asexual development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25530311" target=_blank>Wang M, <i>et al.</i> (2015 Jan)</a>. PdbrlA, PdabaA and PdwetA control distinct stages of conidiogenesis in Penicillium digitatum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25550299" target=_blank>Chung D, <i>et al.</i> (2015 Mar-Apr)</a>. Neurospora crassa ASM-1 complements the conidiation defect in a stuA mutant of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26283234" target=_blank>Hu P, <i>et al.</i> (2015 Oct)</a>. AcstuA, which encodes an APSES transcription regulator, is involved in conidiation, cephalosporin biosynthesis and cell wall integrity of Acremonium chrysogenum.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04212_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04212_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04212_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04212_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01457_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01457.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>