<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG03325.html">HOG03325&emsp;&emsp;</a><br> 
																	HOG03326&emsp;&emsp;<br> 
																	<a href="HOG03327.html">HOG03327&emsp;&emsp;ERT1/acuK family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG03326.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG03326_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG03326_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG03326_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG03326_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG07253<br>EOG8CJSZP<br><br>sce:ERO1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Thiol oxidase required for oxidative protein folding in the ER; essential for maintaining proper redox balance in ER; feedback regulation of Ero1p occurs via reduction and oxidation of Ero1p regulatory bonds; reduced Pdi1p activates Ero1p by direct reduction of Ero1p regulatory bonds; depletion of thiol substrates and accumulation of oxidized Pdi1p results in inactivation of Ero1p by both Pdi1p-mediated oxidation and autonomous oxidation of Ero1p regulatory bonds<br><br>
			<br><b>PomBase Description</b><br>ER protein folding oxidoreductin Ero1b<br><br>
			<br><b>AspGD Description</b><br>Putative thiol oxidase; expression enhanced by maltose; expression induced by tunicamycin and DTT<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9659913" target=_blank>Frand AR, <i>et al.</i> (1998 Jan)</a>. The ERO1 gene of yeast is required for oxidation of protein dithiols in the endoplasmic reticulum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9659914" target=_blank>Pollard MG, <i>et al.</i> (1998 Jan)</a>. Ero1p: a novel and ubiquitous protein with an essential role in oxidative protein folding in the endoplasmic reticulum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10549279" target=_blank>Frand AR, <i>et al.</i> (1999 Oct)</a>. Ero1p oxidizes protein disulfide isomerase in a pathway for disulfide bond formation in the endoplasmic reticulum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11090354" target=_blank>Tu BP, <i>et al.</i> (2000 Nov 24)</a>. Biochemical basis of oxidative protein folding in the endoplasmic reticulum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982384" target=_blank>Frand AR, <i>et al.</i> (2000 Sep)</a>. Two pairs of conserved cysteines are required for the oxidative activity of Ero1p in protein disulfide bond formation in the endoplasmic reticulum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11707280" target=_blank>Pagani M, <i>et al.</i> (2001 Nov 9)</a>. The C-terminal domain of yeast Ero1p mediates membrane localization and is essential for function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12453408" target=_blank>Tu BP, <i>et al.</i> (2002 Nov)</a>. The FAD- and O(2)-dependent reaction cycle of Ero1-mediated oxidative protein folding in the endoplasmic reticulum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14757749" target=_blank>Tu BP, <i>et al.</i> (2004 Feb 2)</a>. Oxidative protein folding in eukaryotes: mechanisms and consequences.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15163408" target=_blank>Gross E, <i>et al.</i> (2004 May 28)</a>. Structure of Ero1p, source of disulfide bonds for oxidative protein folding in the cell.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15449306" target=_blank>Kettner K, <i>et al.</i> (2004 Sep)</a>. Schizosaccharomyces pombe ER oxidoreductin-like proteins SpEro1a p and SpEro1b p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17448992" target=_blank>Sevier CS, <i>et al.</i> (2007 Apr 20)</a>. Modulation of cellular disulfide-bond formation and the ER redox environment by feedback regulation of Ero1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633078" target=_blank>Sun X, <i>et al.</i> (2013 May)</a>. PyrG is required for maintaining stable cellular uracil level and normal sporulation pattern under excess uracil stress in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07253_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07253_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07253_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG07253_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG03326_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG07254<br>EOG8CJSZP<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>ER oxidoreductin Ero1a<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07254_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07254_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07254_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG07254_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG03326_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG03326.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>