<html>
<head><title>HOG01217</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01216.html">HOG01216&emsp;&emsp;RRM GAR family</a><br> 
																	HOG01217&emsp;&emsp;<br> 
																	<a href="HOG01218.html">HOG01218&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01217.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01217_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01217_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01217_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG01217.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03338<br>EOG8N8PQH<br><br>sce:HAP5<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of the Hap2p/3p/4p/5p CCAAT-binding complex; complex is heme-activated and glucose repressed; complex is a transcriptional activator and global regulator of respiratory gene expression; required for assembly and DNA binding activity of the complex<br><br>
			<br><b>PomBase Description</b><br>CCAAT-binding factor complex subunit Php5<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2041742" target=_blank>van Heeswijck R, <i>et al.</i> (1991 May 25)</a>. The amdR product and a CCAAT-binding factor bind to adjacent, possibly overlapping DNA sequences in the promoter region of the Aspergillus nidulans amdS gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7828851" target=_blank>McNabb DS, <i>et al.</i> (1995 Jan 1)</a>. Cloning of yeast HAP5: a novel subunit of a heterotrimeric complex required for CCAAT binding.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9372932" target=_blank>McNabb DS, <i>et al.</i> (1997 Dec)</a>. The Saccharomyces cerevisiae Hap5p homolog from fission yeast reveals two conserved domains that are essential for assembly of heterotetrameric CCAAT-binding factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9858535" target=_blank>Steidl S, <i>et al.</i> (1999 Jan)</a>. AnCF, the CCAAT binding complex of Aspergillus nidulans, contains products of the hapB, hapC, and hapE genes and is required for activation by the pathway-specific regulatory gene amdR.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10441450" target=_blank>Brakhage AA, <i>et al.</i> (1999 Jul-Aug)</a>. HAP-Like CCAAT-binding complexes in filamentous fungi: implications for biotechnology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10490592" target=_blank>Narendja FM, <i>et al.</i> (1999 Oct)</a>. AnCF, the CCAAT binding complex of Aspergillus nidulans, is essential for the formation of a DNase I-hypersensitive site in the 5' region of the amdS gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11390369" target=_blank>Li Y, <i>et al.</i> (2001 Aug 10)</a>. A multiprotein complex that interacts with RNA polymerase II elongator.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12483563" target=_blank>Tsukagoshi N, <i>et al.</i> (2001 Feb)</a>. Regulation of the amylolytic and (hemi-)cellulolytic genes in aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11285740" target=_blank>Weidner G, <i>et al.</i> (2001 Feb)</a>. The Aspergillus nidulans homoaconitase gene lysF is negatively regulated by the multimeric CCAAT-binding complex AnCF and positively regulated by GATA sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11243777" target=_blank>Steidl S, <i>et al.</i> (2001 Mar 2)</a>. The Aspergillus nidulans multimeric CCAAT binding complex AnCF is negatively autoregulated via its hapB subunit gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11409179" target=_blank>Tanaka A, <i>et al.</i> (2001 May)</a>. AoHapB, AoHapC and AoHapE, subunits of the Aspergillus oryzae CCAAT-binding complex, are functionally interchangeable with the corresponding subunits in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11589578" target=_blank>Zeilinger S, <i>et al.</i> (2001 Sep)</a>. The Hypocrea jecorina HAP 2/3/5 protein complex binds to the inverted CCAAT-box (ATTGG) within the cbh2 (cellobiohydrolase II-gene) activating element.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11852085" target=_blank>Kato M, <i>et al.</i> (2002 Feb 13)</a>. A quantity control mechanism regulating levels of the HapE subunit of the Hap complex in Aspergillus nidulans: no accumulation of HapE in hapC deletion mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12031499" target=_blank>Tanaka A, <i>et al.</i> (2002 Jun 7)</a>. Isolation of genes encoding novel transcription factors which interact with the Hap complex from Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15719552" target=_blank>Brakhage AA, <i>et al.</i> (2004)</a>. Regulation of penicillin biosynthesis in filamentous fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15327951" target=_blank>Steidl S, <i>et al.</i> (2004 Sep 10)</a>. A single subunit of a heterotrimeric CCAAT-binding complex carries a nuclear localization signal: piggy back transport of the pre-assembled complex to the nucleus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15849404" target=_blank>Kato M, <i>et al.</i> (2005 Apr)</a>. An overview of the CCAAT-box binding factor in filamentous fungi: assembly, nuclear translocation, and transcriptional enhancement.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16163515" target=_blank>Goda H, <i>et al.</i> (2005 Nov)</a>. Nuclear translocation of the heterotrimeric CCAAT binding factor of Aspergillus oryzae is dependent on two redundant localising signals in a single subunit.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16278450" target=_blank>McNabb DS, <i>et al.</i> (2005 Nov)</a>. Assembly of the Hap2p/Hap3p/Hap4p/Hap5p-DNA complex in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098534" target=_blank>Tüncher A, <i>et al.</i> (2005 Sep 23)</a>. The CCAAT-binding complex of eukaryotes: evolution of a second NLS in the HapB subunit of the filamentous fungus Aspergillus nidulans despite functional conservation at the molecular level between yeast, A.nidulans and human.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16636442" target=_blank>Tanoue S, <i>et al.</i> (2006 Apr)</a>. The region in a subunit of the Aspergillus CCAAT-binding protein similar to the HAP4p-recruiting domain of Saccharomyces cerevisiae Hap5p is not essential for transcriptional enhancement.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18942174" target=_blank>Spröte P, <i>et al.</i> (2008 Oct)</a>. Identification of the novel penicillin biosynthesis gene aatB  of Aspergillus nidulans and its putative evolutionary  relationship to this fungal secondary metabolism  gene cluster.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19863978" target=_blank>Brakhage AA, <i>et al.</i> (2009 Oct-Nov)</a>. Aspects on evolution of fungal beta-lactam biosynthesis gene clusters and recruitment of trans-acting factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19965775" target=_blank>Thön M, <i>et al.</i> (2010 Mar)</a>. The CCAAT-binding complex coordinates the oxidative stress response in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22902862" target=_blank>Huber EM, <i>et al.</i> (2012 Oct 10)</a>. DNA minor groove sensing and widening by the CCAAT-binding complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25589790" target=_blank>Hortschansky P, <i>et al.</i> (2015 Mar 6)</a>. Deciphering the combinatorial DNA-binding code of the CCAAT-binding complex and the iron-regulatory basic region leucine zipper (bZIP) transcription factor HapX.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03338_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03338_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03338_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03338_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01217_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03339<br>EOG8N8PQH<br><br>sce:BUR6<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of a heterodimeric NC2 transcription regulator complex; complex binds to TBP and can repress transcription by preventing preinitiation complex assembly or stimulate activated transcription; homologous to human NC2alpha; complex also includes Ncb2p<br><br>
			<br><b>PomBase Description</b><br>DNA polymerase epsilon interactor Dpb3-like<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted protein heterodimerization activity, sequence-specific DNA binding activity and intracellular localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8948634" target=_blank>Goppelt A, <i>et al.</i> (1996 Nov 15)</a>. Characterization of the basal inhibitor of class II transcription NC2 from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096360" target=_blank>Gadbois EL, <i>et al.</i> (1997 Apr 1)</a>. Functional antagonism between RNA polymerase II holoenzyme and global negative regulator NC2 in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9023340" target=_blank>Kim S, <i>et al.</i> (1997 Feb 4)</a>. The Dr1/DRAP1 heterodimer is a global repressor of transcription in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12237409" target=_blank>Cang Y, <i>et al.</i> (2002 Oct 1)</a>. Direct stimulation of transcription by negative cofactor 2 (NC2) through TATA-binding protein (TBP).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15388803" target=_blank>Spiga MG, <i>et al.</i> (2004)</a>. Identification and cloning of two putative subunits of DNA polymerase epsilon in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03339_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03339_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03339_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03339_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01217_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03340<br>EOG8N8PQH<br><br>sce:absent<br><br>Genes: 19
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03340_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03340_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03340_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03340_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01217_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03341<br>EOG8N8PQH<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted protein heterodimerization activity, sequence-specific DNA binding activity and intracellular localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03341_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03341_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03341_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03341_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01217_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03342<br>EOG8N8PQH<br>EOG8P8D4K<br><br>sce:DLS1<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of ISW2/yCHRAC chromatin accessibility complex; ISW2/yCHRAC also includes Itc1p, Isw2p, and Dpb4p; involved in inheritance of telomeric silencing; DLS1 has a paralog, DPB3, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=776230" target=_blank>Guiard B, <i>et al.</i> (1976)</a>. Complete amino acid sequence of the heme-binding core in bakers' yeast cytochrome b2 (L-(+)-lactate dehydrogenase).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11081629" target=_blank>Goldmark JP, <i>et al.</i> (2000 Oct 27)</a>. The Isw2 chromatin remodeling complex represses early meiotic genes upon recruitment by Ume6p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238944" target=_blank>Gelbart ME, <i>et al.</i> (2001 Mar)</a>. Interactions of Isw2 chromatin remodeling complex with nucleosomal arrays: analyses using recombinant yeast histones and immobilized templates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11533234" target=_blank>Fazzio TG, <i>et al.</i> (2001 Oct)</a>. Widespread collaboration of Isw2 and Sin3-Rpd3 chromatin remodeling complexes in transcriptional repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489850" target=_blank>Sugiyama M, <i>et al.</i> (2001 Sep)</a>. The Saccharomyces cerevisiae Isw2p-Itc1p complex represses INO1 expression and maintains cell morphology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15024052" target=_blank>McConnell AD, <i>et al.</i> (2004 Apr)</a>. Histone fold protein Dls1p is required for Isw2-dependent chromatin remodeling in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14673157" target=_blank>Iida T, <i>et al.</i> (2004 Jan)</a>. Noncompetitive counteractions of DNA polymerase epsilon and ISW2/yCHRAC for epigenetic inheritance of telomere position effect in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03342_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03342_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03342_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03342_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01217_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03343<br>EOG8N8PQH<br><br>sce:DPB3<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Third-largest subunit of DNA polymerase II (DNA polymerase epsilon); required to maintain fidelity of chromosomal replication and also for inheritance of telomeric silencing; stabilizes the interaction of Pol epsilon with primer-template DNA, positively affecting the processivity of the polymerase and exonuclease activities of Pol epsilon; mRNA abundance peaks at the G1/S boundary of the cell cycle; DPB3 has a paralog, DLS1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>DNA polymerase epsilon Dpb3 (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1923754" target=_blank>Araki H, <i>et al.</i> (1991 Sep 25)</a>. Cloning DPB3, the gene encoding the third subunit of DNA polymerase II of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11024162" target=_blank>Ohya T, <i>et al.</i> (2000 Oct 15)</a>. Structure and function of the fourth subunit (Dpb4p) of DNA polymerase epsilon in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12124389" target=_blank>Shimizu K, <i>et al.</i> (2002 Oct 4)</a>. Fidelity of DNA polymerase epsilon holoenzyme from budding yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12571237" target=_blank>Chilkova O, <i>et al.</i> (2003 Apr 18)</a>. The quaternary structure of DNA polymerase epsilon from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03343_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03343_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03343_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03343_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01217_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01217.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>