<html>
<head><title>HOG01229</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01228.html">HOG01228&emsp;&emsp;AAA ATPase family</a><br> 
																	HOG01229&emsp;&emsp;<br> 
																	<a href="HOG01230.html">HOG01230&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01229.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01229_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01229_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01229_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG01229.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03411<br>EOG866T4M<br><br>sce:URE2<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Nitrogen catabolite repression transcriptional regulator; inhibits GLN3 transcription in good nitrogen source; role in sequestering Gln3p and Gat1p to the cytoplasm; has glutathione peroxidase activity and can mutate to acquire GST activity; altered form creates [URE3] prion<br><br>
			<br><b>PomBase Description</b><br>glutathione S-transferase Gst1|glutathione S-transferase Gst2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have glutathione peroxidase activity, glutathione transferase activity and intracellular localization|Ortholog(s) have glutathione peroxidase activity, glutathione transferase activity and cytosol, nucleus localization|Ortholog(s) have glutathione peroxidase activity, glutathione transferase activity, phosphoprotein binding, transcription corepressor activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1990286" target=_blank>Coschigano PW, <i>et al.</i> (1991 Feb)</a>. The URE2 gene product of Saccharomyces cerevisiae plays an important role in the cellular response to the nitrogen source and has homology to glutathione s-transferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7909170" target=_blank>Wickner RB, <i>et al.</i> (1994 Apr 22)</a>. [URE3] as an altered URE2 protein: evidence for a prion analog in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7569955" target=_blank>Masison DC, <i>et al.</i> (1995 Oct 6)</a>. Prion-inducing domain of yeast Ure2p and protease resistance of Ure2p in prion-containing cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755910" target=_blank>Blinder D, <i>et al.</i> (1996 Aug)</a>. Interaction of the GATA factor Gln3p with the nitrogen regulator Ure2p in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10604478" target=_blank>Beck T, <i>et al.</i> (1999 Dec 9)</a>. The TOR signalling pathway controls nuclear localization of nutrient-regulated transcription factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073991" target=_blank>Moriyama H, <i>et al.</i> (2000 Dec)</a>. [URE3] prion propagation in Saccharomyces cerevisiae: requirement for chaperone Hsp104 and curing by overexpressed chaperone Ydj1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10799523" target=_blank>Cunningham TS, <i>et al.</i> (2000 May 12)</a>. Nitrogen catabolite repression of DAL80 expression depends on the relative levels of Gat1p and Ure2p production in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11171973" target=_blank>Umland TC, <i>et al.</i> (2001 Feb 13)</a>. The crystal structure of the nitrogen regulation fragment of the yeast prion protein Ure2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11342133" target=_blank>Bousset L, <i>et al.</i> (2001 Jan 10)</a>. Structure of the globular region of the prion protein Ure2 from the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11695904" target=_blank>Bousset L, <i>et al.</i> (2001 Nov 13)</a>. Crystal structures of the yeast prion Ure2p functional region in complex with glutathione and related compounds.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177423" target=_blank>Edskes HK, <i>et al.</i> (2002 Dec 10)</a>. Conservation of a portion of the S. cerevisiae Ure2p prion domain that interacts with the full-length protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15371425" target=_blank>Bai M, <i>et al.</i> (2004 Nov 26)</a>. The yeast prion protein Ure2 shows glutathione peroxidase activity in both native and fibrillar forms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16060675" target=_blank>Chan JC, <i>et al.</i> (2005 Aug 9)</a>. Parallel beta-sheets and polar zippers in amyloid fibrils formed by residues 10-39 of the yeast prion protein Ure2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17953455" target=_blank>Baxa U, <i>et al.</i> (2007 Nov 13)</a>. Characterization of beta-sheet structure in Ure2p1-89 yeast prion fibrils by solid-state nuclear magnetic resonance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19321443" target=_blank>Zhang ZR, <i>et al.</i> (2009 May 22)</a>. Novel glutaredoxin activity of the yeast prion protein Ure2 reveals a native-like dimer within fibrils.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20489023" target=_blank>Breitkreutz A, <i>et al.</i> (2010 May 21)</a>. A global protein kinase and phosphatase interaction network in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03411_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03411_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03411_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03411_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01229_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03412<br>EOG866T4M<br><br>sce:absent<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have glutathione peroxidase activity, glutathione transferase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12039735" target=_blank>Fraser JA, <i>et al.</i> (2002 Jun)</a>. A gene from Aspergillus nidulans with similarity to URE2 of Saccharomyces cerevisiae encodes a glutathione S-transferase which contributes to heavy metal and xenobiotic resistance.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15749051" target=_blank>Burns C, <i>et al.</i> (2005 Apr)</a>. Identification, cloning, and functional expression of three glutathione transferase genes from Aspergillus fumigatus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16368011" target=_blank>PÃ³csi I, <i>et al.</i> (2005 Dec 20)</a>. Comparison of gene expression signatures of diamide, H2O2 and menadione exposed Aspergillus nidulans cultures--linking genome-wide transcriptional changes to cellular physiology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19171936" target=_blank>Sato I, <i>et al.</i> (2009 Mar 20)</a>. The glutathione system of Aspergillus nidulans involves a fungus-specific glutathione S-transferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24120530" target=_blank>Martins I, <i>et al.</i> (2013 Dec 6)</a>. Proteomic alterations induced by ionic liquids in Aspergillus nidulans and Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24041414" target=_blank>Montibus M, <i>et al.</i> (2015)</a>. Coupling of transcriptional response to oxidative stress and secondary metabolism regulation in filamentous fungi.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03412_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03412_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03412_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03412_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01229_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03413<br>EOG866T4M<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of Aspergillus tubingensis : Asptu1_0113414, Aspergillus kawachii : Aspka1_0177730, Aspergillus acidus : Aspfo1_0205181 and Aspergillus niger ATCC 1015 : 177976-mRNA|Ortholog of Aspergillus tubingensis : Asptu1_0052964, Aspergillus glaucus : Aspgl1_0181752, Aspergillus kawachii : Aspka1_0173366 and Aspergillus acidus : Aspfo1_0130243<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03413_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03413_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03413_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03413_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01229_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03414<br>EOG866T4M<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. fumigatus Af293 : Afu6g03390, Aspergillus niger ATCC 1015 : 126931-mRNA, Aspergillus zonatus : Aspzo1_0020462, Aspzo1_0132229 and Aspergillus carbonarius ITEM 5010 : Acar5010_127540<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03414_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03414_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03414_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03414_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01229_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03415<br>EOG866T4M<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have glutathione peroxidase activity, glutathione transferase activity and intracellular localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03415_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03415_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03415_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03415_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01229_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03416<br>EOG866T4M<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03416_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03416_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03416_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03416_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01229_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01229.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>