<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG02177.html">HOG02177&emsp;&emsp;</a><br> 
																	HOG02178&emsp;&emsp;<br> 
																	<a href="HOG02179.html">HOG02179&emsp;&emsp;</a><br><br> 
																	<a href="../sequences/hog/HOG02178.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG02178_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG02178_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="HOG02178_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG05674<br>EOG8X0KC1<br><br>sce:MMS4<br><br>Genes: 22
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of structure-specific Mms4p-Mus81p endonuclease; cleaves branched DNA; involved in recombination, DNA repair, and joint molecule formation/resolution during meiotic recombination; phosphorylation of the non-catalytic subunit Mms4p by Cdc28p and Cdc5p during mitotic cell cycle activates the function of Mms4p-Mus81p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9604884" target=_blank>Xiao W, <i>et al.</i> (1998 Apr)</a>. Mms4, a putative transcriptional (co)activator, protects Saccharomyces cerevisiae cells from endogenous and environmental DNA damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11779793" target=_blank>de los Santos T, <i>et al.</i> (2001 Dec)</a>. A role for MMS4 in the processing of recombination intermediates during meiosis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139495" target=_blank>Mullen JR, <i>et al.</i> (2001 Jan)</a>. Requirement for three novel protein complexes in the absence of the Sgs1 DNA helicase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11641278" target=_blank>Kaliraman V, <i>et al.</i> (2001 Oct 15)</a>. Functional overlap between Sgs1-Top3 and the Mms4-Mus81 endonuclease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12475932" target=_blank>Fabre F, <i>et al.</i> (2002 Dec 24)</a>. Alternate pathways involving Sgs1/Top3, Mus81/ Mms4, and Srs2 prevent formation of toxic recombination intermediates from single-stranded gaps created by DNA replication.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14642571" target=_blank>Fu Y, <i>et al.</i> (2003 Dec 9)</a>. Functional domains required for the Saccharomyces cerevisiae Mus81-Mms4 endonuclease complex formation and nuclear localization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12473680" target=_blank>Whitby MC, <i>et al.</i> (2003 Feb 28)</a>. Cleavage of model replication forks by fission yeast Mus81-Eme1 and budding yeast Mus81-Mms4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12724407" target=_blank>Bastin-Shanower SA, <i>et al.</i> (2003 May)</a>. The mechanism of Mus81-Mms4 cleavage site selection distinguishes it from the homologous endonuclease Rad1-Rad10.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12750322" target=_blank>de los Santos T, <i>et al.</i> (2003 May)</a>. The Mus81/Mms4 endonuclease acts independently of double-Holliday junction resolution to promote a distinct subset of crossovers during meiosis in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14527420" target=_blank>Osman F, <i>et al.</i> (2003 Sep)</a>. Generating crossovers by resolution of nicked Holliday junctions: a role for Mus81-Eme1 in meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15611158" target=_blank>Argueso JL, <i>et al.</i> (2004 Dec)</a>. Competing crossover pathways act during meiosis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14752007" target=_blank>Hollingsworth NM, <i>et al.</i> (2004 Jan 15)</a>. The Mus81 solution to resolution: generating meiotic crossovers without Holliday junctions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15590332" target=_blank>Fricke WM, <i>et al.</i> (2005 Feb 3)</a>. Substrate specificity of the Saccharomyces cerevisiae Mus81-Mms4 endonuclease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG05674_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG05674_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG05674_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG05674_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG02178_phobius.html#FOG05674">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG02178.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>