<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01049.html">HOG01049&emsp;&emsp;ABC transporter superfamily. ABCG family. PDR (TC 3.A.1.205) subfamily</a><br> 
																	HOG01050&emsp;&emsp;<br> 
																	<a href="HOG01051.html">HOG01051&emsp;&emsp;ENY2 family; archaeal/bacterial/fungal opsin family; oligopeptide OPT transporter family</a><br><br> 
																	<a href="../sequences/hog/HOG01050.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG01050_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG01050_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG01050_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG01050_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02448<br>EOG8HQC0N<br><br>sce:AGP1<br><br>Genes: 177
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Amino acid transporter<br><br>
			<br><b>SGD Description</b><br>Low-affinity amino acid permease with broad substrate range; involved in uptake of asparagine, glutamine, and other amino acids; expression regulated by SPS plasma membrane amino acid sensor system (Ssy1p-Ptr3p-Ssy5p); AGP1 has a paralog, GNP1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>amino acid permease Isp5|amino acid transmembrane transporter (predicted)|amino acid transmembrane transporter Aat1|amino acid transmembrane transporter, predicted Meu22|amino-acid permease, unknown|cationic amino acid transmembrane transporter Cat1|plasma membrane amino acid permease Per1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have L-lysine transmembrane transporter activity, L-proline transmembrane transporter activity, arginine transmembrane transporter activity, polyamine transmembrane transporter activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573211" target=_blank>Schreve JL, <i>et al.</i> (1998 May)</a>. The Saccharomyces cerevisiae YCC5 (YCL025c) gene encodes an amino acid permease, Agp1, which transports asparagine and glutamine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10654085" target=_blank>Regenberg B, <i>et al.</i> (1999 Dec)</a>. Substrate specificity and gene expression of the amino-acid permeases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9891035" target=_blank>Iraqui I, <i>et al.</i> (1999 Feb)</a>. Amino acid signaling in Saccharomyces cerevisiae: a permease-like sensor of external amino acids and F-Box protein Grr1p are required for transcriptional induction of the AGP1 gene, which encodes a broad-specificity amino acid permease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10467005" target=_blank>Düring-Olsen L, <i>et al.</i> (1999 Jul)</a>. Cysteine uptake by Saccharomyces cerevisiae is accomplished by multiple permeases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11154269" target=_blank>Forsberg H, <i>et al.</i> (2001 Feb)</a>. Genetic and biochemical analysis of the yeast plasma membrane Ssy1p-Ptr3p-Ssy5p sensor of extracellular amino acids.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11376151" target=_blank>Watanabe T, <i>et al.</i> (2001 Jun 1)</a>. Comprehensive isolation of meiosis-specific genes identifies novel proteins and unusual non-coding transcripts in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12136010" target=_blank>Matsumoto S, <i>et al.</i> (2002 Jul)</a>. Role of the Tsc1-Tsc2 complex in signaling and transport across the cell membrane in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12844361" target=_blank>Brachat S, <i>et al.</i> (2003)</a>. Reinvestigation of the Saccharomyces cerevisiae genome annotation by comparison to the genome of a related fungus: Ashbya gossypii.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12738880" target=_blank>Samuelsen CO, <i>et al.</i> (2003 May 27)</a>. TRAP230/ARC240 and TRAP240/ARC250 Mediator subunits are functionally conserved through evolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126393" target=_blank>Abdel-Sater F, <i>et al.</i> (2004 Apr)</a>. The external amino acid signaling pathway promotes activation of Stp1 and Uga35/Dal81 transcription factors for induction of the AGP1 gene in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14697254" target=_blank>Schreve JL, <i>et al.</i> (2004 Jan 16)</a>. Yeast Agp2p and Agp3p function as amino acid permeases in poor nutrient conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15797925" target=_blank>Pardo M, <i>et al.</i> (2005 Apr 15)</a>. The nuclear rim protein Amo1 is required for proper microtubule cytoskeleton organisation in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751107" target=_blank>Roth AF, <i>et al.</i> (2006 Jun 2)</a>. Global analysis of protein palmitoylation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630328" target=_blank>Breakspear A, <i>et al.</i> (2007 Sep)</a>. Aspergillus nidulans conidiation genes dewA, fluG, and stuA are differentially regulated in early vegetative growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18310029" target=_blank>Iwaki T, <i>et al.</i> (2008 Mar)</a>. Multiple functions of ergosterol in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18684775" target=_blank>Kennedy PJ, <i>et al.</i> (2008 Nov)</a>. A genome-wide screen of genes involved in cadmium tolerance in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18727152" target=_blank>Sasaki M, <i>et al.</i> (2008 Sep)</a>. The gap-filling sequence on the left arm of chromosome 2 in fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20388730" target=_blank>Nakase M, <i>et al.</i> (2010 May 1)</a>. Mannosylinositol phosphorylceramide is a major sphingolipid component and is required for proper localization of plasma-membrane proteins in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20969648" target=_blank>Schinko T, <i>et al.</i> (2010 Nov)</a>. Transcriptome analysis of nitrate assimilation in Aspergillus nidulans reveals connections to nitric oxide metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21760946" target=_blank>Takeda K, <i>et al.</i> (2011)</a>. Identification of genes affecting the toxicity of anti-cancer drug bortezomib by genome-wide screening in S. pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22235339" target=_blank>Lando D, <i>et al.</i> (2012)</a>. The S. pombe histone H2A dioxygenase Ofd2 regulates gene expression during hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22484924" target=_blank>Nakase M, <i>et al.</i> (2012)</a>. CUE domain-containing protein Vps901 is required for vacuolar protein transport in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22840777" target=_blank>Kawashima SA, <i>et al.</i> (2012 Jul 27)</a>. Analyzing fission yeast multidrug resistance mechanisms to develop a genetically tractable model system for chemical biology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22194353" target=_blank>Nakase M, <i>et al.</i> (2012 Mar)</a>. Intracellular trafficking and ubiquitination of the Schizosaccharomyces pombe amino acid permease Aat1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23813957" target=_blank>Nakase Y, <i>et al.</i> (2013 Sep 1)</a>. The fission yeast β-arrestin-like protein Any1 is involved in TSC-Rheb signaling and the regulation of amino acid transporters.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24454826" target=_blank>Fang Y, <i>et al.</i> (2014)</a>. E3 ubiquitin ligase Pub1 is implicated in endocytosis of a GPI-anchored protein Ecm33 in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25651869" target=_blank>Doi A, <i>et al.</i> (2015 Apr)</a>. Chemical genomics approach to identify genes associated with sensitivity to rapamycin in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02448_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02448_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02448_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02448_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02448">Phobius transmembrane predictions</a>
										<br>177 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02449<br>EOG8HQC0N<br><br>sce:TAT2<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>High affinity tryptophan and tyrosine permease; overexpression confers FK506 and FTY720 resistance<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8058037" target=_blank>Chen XH, <i>et al.</i> (1994 Aug 2)</a>. SCM2, a tryptophan permease in Saccharomyces cerevisiae, is important for cell growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7523855" target=_blank>Schmidt A, <i>et al.</i> (1994 Oct)</a>. Two FK506 resistance-conferring genes in Saccharomyces cerevisiae, TAT1 and TAT2, encode amino acid permeases mediating tyrosine and tryptophan uptake.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10654085" target=_blank>Regenberg B, <i>et al.</i> (1999 Dec)</a>. Substrate specificity and gene expression of the amino-acid permeases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10467005" target=_blank>Düring-Olsen L, <i>et al.</i> (1999 Jul)</a>. Cysteine uptake by Saccharomyces cerevisiae is accomplished by multiple permeases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02449_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02449_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02449_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02449_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02449">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02450<br>EOG8HQC0N<br><br>sce:SAM3<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>High-affinity S-adenosylmethionine permease; required for utilization of S-adenosylmethionine as a sulfur source; has similarity to S-methylmethionine permease Mmp1p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497160" target=_blank>Rouillon A, <i>et al.</i> (1999 Oct 1)</a>. Transport of sulfonium compounds. Characterization of the s-adenosylmethionine and s-methylmethionine permeases from the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02450_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02450_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02450_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02450_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02450">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02451<br>EOG8HQC0N<br><br>sce:MMP1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>High-affinity S-methylmethionine permease; required for utilization of S-methylmethionine as a sulfur source; has similarity to S-adenosylmethionine permease Sam3p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497160" target=_blank>Rouillon A, <i>et al.</i> (1999 Oct 1)</a>. Transport of sulfonium compounds. Characterization of the s-adenosylmethionine and s-methylmethionine permeases from the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02451_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02451_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02451_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02451_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02451">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG02452<br>EOG8HQC0N<br><br>sce:HIP1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>High-affinity histidine permease; also involved in the transport of manganese ions<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3905514" target=_blank>Tanaka J, <i>et al.</i> (1985)</a>. The histidine permease gene (HIP1) of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02452_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02452_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02452_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02452_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02452">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG02453<br>EOG8HQC0N<br><br>sce:GAP1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>General amino acid permease; Gap1p senses the presence of amino acid substrates to regulate localization to the plasma membrane when needed; essential for invasive growth<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5474888" target=_blank>Grenson M, <i>et al.</i> (1970 Sep)</a>. Multiplicity of the amino acid permeases in Saccharomyces cerevisiae. IV. Evidence for a general amino acid permease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2194797" target=_blank>Jauniaux JC, <i>et al.</i> (1990 May 31)</a>. GAP1, the general amino acid permease gene of Saccharomyces cerevisiae. Nucleotide sequence, protein similarity with the other bakers yeast amino acid permeases, and nitrogen catabolite repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7798155" target=_blank>Stanbrough M, <i>et al.</i> (1995 Jan)</a>. Transcriptional and posttranslational regulation of the general amino acid permease of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636059" target=_blank>Stanbrough M, <i>et al.</i> (1996 Apr)</a>. Two transcription factors, Gln3p and Nil1p, use the same GATAAG sites to activate the expression of GAP1 of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9614172" target=_blank>Springael JY, <i>et al.</i> (1998 Jun)</a>. Nitrogen-regulated ubiquitination of the Gap1 permease of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10467005" target=_blank>Düring-Olsen L, <i>et al.</i> (1999 Jul)</a>. Cysteine uptake by Saccharomyces cerevisiae is accomplished by multiple permeases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10373490" target=_blank>Stolz J, <i>et al.</i> (1999 Jun 25)</a>. The fenpropimorph resistance gene FEN2 from Saccharomyces cerevisiae encodes a plasma membrane H+-pantothenate symporter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10194416" target=_blank>Springael JY, <i>et al.</i> (1999 May)</a>. NH4+-induced down-regulation of the Saccharomyces cerevisiae Gap1p permease involves its ubiquitination with lysine-63-linked chains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10953083" target=_blank>Regenberg B, <i>et al.</i> (2000 Sep 15)</a>. GAP1, a novel selection and counter-selection marker for multiple gene disruptions in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11352928" target=_blank>Helliwell SB, <i>et al.</i> (2001 May 14)</a>. Components of a ubiquitin ligase complex specify polyubiquitination and intracellular trafficking of the general amino acid permease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11500494" target=_blank>Soetens O, <i>et al.</i> (2001 Nov 23)</a>. Ubiquitin is required for sorting to the vacuole of the yeast general amino acid permease, Gap1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12499351" target=_blank>Malkus P, <i>et al.</i> (2002 Dec 23)</a>. Concentrative sorting of secretory cargo proteins into COPII-coated vesicles.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12417748" target=_blank>Chen EJ, <i>et al.</i> (2002 Nov 12)</a>. Amino acids regulate the intracellular trafficking of the general amino acid permease of Saccharomycescerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12872131" target=_blank>Peng J, <i>et al.</i> (2003 Aug)</a>. A proteomics approach to understanding protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523026" target=_blank>Nikko E, <i>et al.</i> (2003 Dec 12)</a>. Permease recycling and ubiquitination status reveal a particular role for Bro1 in the multivesicular body pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14557538" target=_blank>Hitchcock AL, <i>et al.</i> (2003 Oct 28)</a>. A subset of membrane-associated proteins is ubiquitinated in response to mutations in the endoplasmic reticulum degradation machinery.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14968425" target=_blank>Andréasson C, <i>et al.</i> (2004 Feb)</a>. Four permeases import proline and the toxic proline analogue azetidine-2-carboxylate into yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039776" target=_blank>Scott PM, <i>et al.</i> (2004 Mar)</a>. GGA proteins bind ubiquitin to facilitate sorting at the trans-Golgi network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15707981" target=_blank>Uemura T, <i>et al.</i> (2005 Mar 25)</a>. Uptake of putrescine and spermidine by Gap1p on the plasma membrane in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21471002" target=_blank>Cain NE, <i>et al.</i> (2011 Jun 1)</a>. Transport activity-dependent intracellular sorting of the yeast general amino acid permease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22844449" target=_blank>Torbensen R, <i>et al.</i> (2012)</a>. Amino acid transporter genes are essential for FLO11-dependent and FLO11-independent biofilm formation and invasive growth in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02453_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02453_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02453_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02453_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02453">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG02454<br>EOG8HQC0N<br><br>sce:TAT1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Amino acid transporter for valine, leucine, isoleucine, and tyrosine; low-affinity tryptophan and histidine transporter; overexpression confers FK506 and FTY720 resistance; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7523855" target=_blank>Schmidt A, <i>et al.</i> (1994 Oct)</a>. Two FK506 resistance-conferring genes in Saccharomyces cerevisiae, TAT1 and TAT2, encode amino acid permeases mediating tyrosine and tryptophan uptake.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10467005" target=_blank>Düring-Olsen L, <i>et al.</i> (1999 Jul)</a>. Cysteine uptake by Saccharomyces cerevisiae is accomplished by multiple permeases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14557538" target=_blank>Hitchcock AL, <i>et al.</i> (2003 Oct 28)</a>. A subset of membrane-associated proteins is ubiquitinated in response to mutations in the endoplasmic reticulum degradation machinery.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02454_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02454_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02454_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02454_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02454">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG02455<br>EOG8HQC0N<br><br>sce:GNP1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>High-affinity glutamine permease; also transports Leu, Ser, Thr, Cys, Met and Asn; expression is fully dependent on Grr1p and modulated by the Ssy1p-Ptr3p-Ssy5p (SPS) sensor of extracellular amino acids; GNP1 has a paralog, AGP1, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8660458" target=_blank>Zhu X, <i>et al.</i> (1996 Jul 31)</a>. GNP1, the high-affinity glutamine permease of S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10654085" target=_blank>Regenberg B, <i>et al.</i> (1999 Dec)</a>. Substrate specificity and gene expression of the amino-acid permeases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10467005" target=_blank>Düring-Olsen L, <i>et al.</i> (1999 Jul)</a>. Cysteine uptake by Saccharomyces cerevisiae is accomplished by multiple permeases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12872131" target=_blank>Peng J, <i>et al.</i> (2003 Aug)</a>. A proteomics approach to understanding protein ubiquitination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02455_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02455_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02455_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02455_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02455">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG02456<br>EOG8HQC0N<br><br>sce:BAP3<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Amino acid permease; involved in uptake of cysteine, leucine, isoleucine and valine; BAP3 has a paralog, BAP2, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8200527" target=_blank>Mai B, <i>et al.</i> (1994 May 27)</a>. Cloning and chromosomal organization of a gene encoding a putative amino-acid permease from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10654085" target=_blank>Regenberg B, <i>et al.</i> (1999 Dec)</a>. Substrate specificity and gene expression of the amino-acid permeases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10467005" target=_blank>Düring-Olsen L, <i>et al.</i> (1999 Jul)</a>. Cysteine uptake by Saccharomyces cerevisiae is accomplished by multiple permeases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02456_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02456_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02456_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02456_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02456">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG02457<br>EOG8HQC0N<br><br>sce:BAP2<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>High-affinity leucine permease; functions as a branched-chain amino acid permease involved in uptake of leucine, isoleucine and valine; contains 12 predicted transmembrane domains; BAP2 has a paralog, BAP3, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7495881" target=_blank>Grauslund M, <i>et al.</i> (1995 Nov 30)</a>. BAP2, a gene encoding a permease for branched-chain amino acids in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10654085" target=_blank>Regenberg B, <i>et al.</i> (1999 Dec)</a>. Substrate specificity and gene expression of the amino-acid permeases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10467005" target=_blank>Düring-Olsen L, <i>et al.</i> (1999 Jul)</a>. Cysteine uptake by Saccharomyces cerevisiae is accomplished by multiple permeases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02457_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02457_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02457_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02457_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02457">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG02458<br>EOG8HQC0N<br><br>sce:PUT4<br><br>Genes: 54
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Proline permease; required for high-affinity transport of proline; also transports the toxic proline analog azetidine-2-carboxylate (AzC); PUT4 transcription is repressed in ammonia-grown cells<br><br>
			<br><b>PomBase Description</b><br>proline specific plasma membrane permease Put4 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have L-proline transmembrane transporter activity, role in proline transport and Golgi apparatus, endoplasmic reticulum, fungal-type vacuole, plasma membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2687114" target=_blank>Vandenbol M, <i>et al.</i> (1989 Nov 15)</a>. Nucleotide sequence of the Saccharomyces cerevisiae PUT4 proline-permease-encoding gene: similarities between CAN1, HIP1 and PUT4 permeases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10654085" target=_blank>Regenberg B, <i>et al.</i> (1999 Dec)</a>. Substrate specificity and gene expression of the amino-acid permeases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718525" target=_blank>van Slegtenhorst M, <i>et al.</i> (2004 Mar 26)</a>. Tsc1+ and tsc2+ regulate arginine uptake and metabolism in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466417" target=_blank>Weisman R, <i>et al.</i> (2005 Feb)</a>. Regulation of leucine uptake by tor1+ in Schizosaccharomyces pombe is sensitive to rapamycin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115814" target=_blank>van Slegtenhorst M, <i>et al.</i> (2005 Oct 1)</a>. Pas1, a G1 cyclin, regulates amino acid uptake and rescues a delay in G1 arrest in Tsc1 and Tsc2 mutants in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17556368" target=_blank>van Slegtenhorst M, <i>et al.</i> (2007 Aug 24)</a>. The Birt-Hogg-Dube and tuberous sclerosis complex homologs have opposing roles in amino acid homeostasis in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24344203" target=_blank>Laor D, <i>et al.</i> (2014 Mar)</a>. Isp7 is a novel regulator of amino acid uptake in the TOR signaling pathway.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02458_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02458_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02458_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02458_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02458">Phobius transmembrane predictions</a>
										<br>54 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG02459<br>EOG8HQC0N<br><br>sce:AGP2<br><br>Genes: 51
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Plasma membrane regulator of polyamine and carnitine transport; has similarity to transporters but lacks transport activity; may act as a sensor that transduces environmental signals; has a positive or negative regulatory effect on transcription of many transporter genes<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in positive regulation of (R)-carnitine transmembrane transport, positive regulation of polyamine transmembrane transport<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7954890" target=_blank>Nasr F, <i>et al.</i> (1994 Jul)</a>. An analysis of the sequence of part of the right arm of chromosome II of S. cerevisiae reveals new genes encoding an amino-acid permease and a carboxypeptidase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10545096" target=_blank>van Roermund CW, <i>et al.</i> (1999 Nov 1)</a>. Molecular characterization of carnitine-dependent transport of acetyl-CoA from peroxisomes to mitochondria in Saccharomyces cerevisiae and identification of a plasma membrane carnitine transporter, Agp2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12132580" target=_blank>Lee J, <i>et al.</i> (2002 Jun 30)</a>. Carnitine uptake by AGP2 in yeast Saccharomyces cerevisiae is dependent on Hog1 MAP kinase pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02459_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02459_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02459_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02459_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02459">Phobius transmembrane predictions</a>
										<br>51 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG02460<br>EOG8HQC0N<br><br>sce:ALP1;CAN1;LYP1<br><br>Genes: 49
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Arginine transporter; expression is normally very low and it is unclear what conditions would induce significant expression; ALP1 has a paralog, CAN1, that arose from the whole genome duplication|Plasma membrane arginine permease; requires phosphatidyl ethanolamine (PE) for localization, exclusively associated with lipid rafts; mutation confers canavanine resistance; CAN1 has a paralog, ALP1, that arose from the whole genome duplication|Lysine permease; one of three amino acid permeases (Alp1p, Can1p, Lyp1p) responsible for uptake of cationic amino acids<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted role in amino acid transport, transmembrane transport and integral component of membrane, membrane localization|Ortholog(s) have basic amino acid transmembrane transporter activity, role in basic amino acid transport and mitochondrion localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3900064" target=_blank>Hoffmann W, <i>et al.</i> (1985 Sep 25)</a>. Molecular characterization of the CAN1 locus in Saccharomyces cerevisiae. A transmembrane protein without N-terminal hydrophobic signal sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3327612" target=_blank>Ahmad M, <i>et al.</i> (1986)</a>. Yeast arginine permease: nucleotide sequence of the CAN1 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8436127" target=_blank>Opekarová M, <i>et al.</i> (1993 Feb 1)</a>. Unidirectional arginine transport in reconstituted plasma-membrane vesicles from yeast overexpressing CAN1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8368011" target=_blank>Sychrova H, <i>et al.</i> (1993 Jul)</a>. Cloning and sequencing of the Saccharomyces cerevisiae gene LYP1 coding for a lysine-specific permease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7941748" target=_blank>Sychrova H, <i>et al.</i> (1994 May)</a>. APL1, a yeast gene encoding a putative permease for basic amino acids.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9231419" target=_blank>Opekarová M, <i>et al.</i> (1997 Jul 15)</a>. On the unidirectionality of arginine uptake in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9544242" target=_blank>Opekarová M, <i>et al.</i> (1998 Feb)</a>. Post-translational fate of CAN1 permease of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10654085" target=_blank>Regenberg B, <i>et al.</i> (1999 Dec)</a>. Substrate specificity and gene expression of the amino-acid permeases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11746604" target=_blank>Regenberg B, <i>et al.</i> (2001 Nov)</a>. Amino acid residues important for substrate specificity of the amino acid permeases Can1p and Gnp1p in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100990" target=_blank>Opekarová M, <i>et al.</i> (2002 Aug 19)</a>. Phosphatidyl ethanolamine is essential for targeting the arginine transporter Can1p to the plasma membrane of yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14551254" target=_blank>Malínská K, <i>et al.</i> (2003 Nov)</a>. Visualization of protein compartmentation within the plasma membrane of living yeast cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21223946" target=_blank>Hosiner D, <i>et al.</i> (2011 Apr)</a>. Pun1p is a metal ion-inducible, calcineurin/Crz1p-regulated plasma membrane protein required for cell wall integrity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21880895" target=_blank>Shi Y, <i>et al.</i> (2011 Nov)</a>. Two novel WD40 domain-containing proteins, Ere1 and Ere2, function in the retromer-mediated endosomal recycling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28228255" target=_blank>Beaupere C, <i>et al.</i> (2017 Feb 21)</a>. CAN1 Arginine Permease Deficiency Extends Yeast Replicative Lifespan via Translational Activation of Stress Response Genes.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02460_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02460_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02460_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02460_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02460">Phobius transmembrane predictions</a>
										<br>49 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG02461<br>EOG8HQC0N<br><br>sce:DIP5<br><br>Genes: 43
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Dicarboxylic amino acid permease; mediates high-affinity and high-capacity transport of L-glutamate and L-aspartate; also a transporter for Gln, Asn, Ser, Ala, and Gly; relocalizes from plasma membrane to vacuole upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>amino acid transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have L-aspartate transmembrane transporter activity, L-glutamate transmembrane transporter activity and role in L-aspartate transport, L-glutamate transport, cellular response to drug<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4561785" target=_blank>Robinson JH, <i>et al.</i> (1972 Apr)</a>. Regulation of the acidic amino acid permease of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4590000" target=_blank>Robinson JH, <i>et al.</i> (1973 Nov)</a>. The acidic amino-acid permease of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10654085" target=_blank>Regenberg B, <i>et al.</i> (1999 Dec)</a>. Substrate specificity and gene expression of the amino-acid permeases in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870366" target=_blank>Sims AH, <i>et al.</i> (2005 May)</a>. Transcriptome analysis of recombinant protein secretion by Aspergillus nidulans and the unfolded-protein response in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19168757" target=_blank>Apostolaki A, <i>et al.</i> (2009 Mar)</a>. AgtA, the dicarboxylic amino acid transporter of Aspergillus nidulans, is concertedly down-regulated by exquisite sensitivity to nitrogen metabolite repression and ammonium-elicited endocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20534811" target=_blank>Abenza JF, <i>et al.</i> (2010 Aug 1)</a>. Aspergillus RabB Rab5 integrates acquisition of degradative identity with the long distance movement of early endosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20362686" target=_blank>Hervás-Aguilar A, <i>et al.</i> (2010 Jul)</a>. Characterization of Aspergillus nidulans DidB Did2, a non-essential component of the multivesicular body pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20026236" target=_blank>Roumelioti K, <i>et al.</i> (2010 Mar)</a>. A cryptic role of a glycolytic-gluconeogenic enzyme (aldolase) in amino acid transporter turnover in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20969648" target=_blank>Schinko T, <i>et al.</i> (2010 Nov)</a>. Transcriptome analysis of nitrate assimilation in Aspergillus nidulans reveals connections to nitric oxide metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21226815" target=_blank>Pantazopoulou A, <i>et al.</i> (2011 Apr)</a>. Characterization of Aspergillus nidulans RabC/Rab6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22135362" target=_blank>Calcagno-Pizarelli AM, <i>et al.</i> (2011 Dec 1)</a>. Rescue of Aspergillus nidulans severely debilitating null mutations in ESCRT-0, I, II and III genes by inactivation of a salt-tolerance pathway allows examination of ESCRT gene roles in pH signalling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22489878" target=_blank>Apostolaki A, <i>et al.</i> (2012 May)</a>. Aspergillus nidulans CkiA is an essential casein kinase I required for delivery of amino acid transporters to the plasma membrane.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23490137" target=_blank>Karachaliou M, <i>et al.</i> (2013 Apr)</a>. The arrestin-like protein ArtA is essential for ubiquitination and endocytosis of the UapA transporter in response to both broad-range and specific signals.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24957674" target=_blank>Anver S, <i>et al.</i> (2014 Aug)</a>. Yeast X-chromosome-associated protein 5 (Xap5) functions with H2A.Z to suppress aberrant transcripts.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02461_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02461_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02461_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02461_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02461">Phobius transmembrane predictions</a>
										<br>42 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG02462<br>EOG8HQC0N<br><br>sce:AGP3<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Low-affinity amino acid permease; may act to supply the cell with amino acids as nitrogen source in nitrogen-poor conditions; transcription is induced under conditions of sulfur limitation; plays a role in regulating Ty1 transposition<br><br>
			<br><b>PomBase Description</b><br>amino acid transmembrane transporter (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02462_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02462_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02462_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02462_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02462">Phobius transmembrane predictions</a>
										<br>35 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG02463<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have L-proline transmembrane transporter activity, role in proline transport and fungal-type vacuole, plasma membrane localization|Ortholog(s) have L-proline transmembrane transporter activity, role in proline transport and fungal-type vacuole, plasma membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02463_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02463_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02463_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02463_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02463">Phobius transmembrane predictions</a>
										<br>21 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG02464<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>amino acid permease (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have L-cystine transmembrane transporter activity, role in amino acid transmembrane transport and endoplasmic reticulum localization|Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, amino acid transport, transmembrane transport and integral component of membrane, membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22840777" target=_blank>Kawashima SA, <i>et al.</i> (2012 Jul 27)</a>. Analyzing fission yeast multidrug resistance mechanisms to develop a genetically tractable model system for chemical biology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02464_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02464_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02464_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02464_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02464">Phobius transmembrane predictions</a>
										<br>18 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG02465<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, transmembrane transport and membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23472095" target=_blank>Colabardini AC, <i>et al.</i> (2013)</a>. Functional characterization of Aspergillus nidulans ypkA, a homologue of the mammalian kinase SGK.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02465_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02465_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02465_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02465_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02465">Phobius transmembrane predictions</a>
										<br>10 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9e6c3" class="col">
			FOG02466<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have L-proline transmembrane transporter activity, role in proline transport and fungal-type vacuole, plasma membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02466_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02466_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02466_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02466_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02466">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dbc4e1" class="col">
			FOG02467<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, transmembrane transport and membrane localization|Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, transmembrane transport and membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02467_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02467_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02467_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02467_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02467">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fad7ad" class="col">
			FOG02468<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02468_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02468_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02468_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02468_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02468">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb9" class="col">
			FOG02469<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02469_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02469_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02469_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02469_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02469">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dfc9ba" class="col">
			FOG02470<br>EOG8HQC0N<br><br>sce:SSY1<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Component of the SPS plasma membrane amino acid sensor system; senses external amino acid concentration and transmits intracellular signals that result in regulation of expression of amino acid permease genes; other members are Ssy1p, Ptr3p, and Ssy5p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9489675" target=_blank>Didion T, <i>et al.</i> (1998 Feb)</a>. The permease homologue Ssy1p controls the expression of amino acid and peptide transporter genes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9483800" target=_blank>Jørgensen MU, <i>et al.</i> (1998 Jan 30)</a>. Mutations in five loci affecting GAP1-independent uptake of neutral amino acids in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10409731" target=_blank>Klasson H, <i>et al.</i> (1999 Aug)</a>. Ssy1p and Ptr3p are plasma membrane components of a yeast system that senses extracellular amino acids.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9891035" target=_blank>Iraqui I, <i>et al.</i> (1999 Feb)</a>. Amino acid signaling in Saccharomyces cerevisiae: a permease-like sensor of external amino acids and F-Box protein Grr1p are required for transcriptional induction of the AGP1 gene, which encodes a broad-specificity amino acid permease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11154269" target=_blank>Forsberg H, <i>et al.</i> (2001 Feb)</a>. Genetic and biochemical analysis of the yeast plasma membrane Ssy1p-Ptr3p-Ssy5p sensor of extracellular amino acids.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11679080" target=_blank>Forsberg H, <i>et al.</i> (2001 Oct)</a>. The role of the yeast plasma membrane SPS nutrient sensor in the metabolic response to extracellular amino acids.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12502738" target=_blank>Andréasson C, <i>et al.</i> (2002 Dec 15)</a>. Receptor-mediated endoproteolytic activation of two transcription factors in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12073087" target=_blank>Kodama Y, <i>et al.</i> (2002 May)</a>. Genome-wide expression analysis of genes affected by amino acid sensor Ssy1p in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14555474" target=_blank>Gaber RF, <i>et al.</i> (2003 Oct)</a>. Constitutive and hyperresponsive signaling by mutant forms of Saccharomyces cerevisiae amino acid sensor Ssy1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17984223" target=_blank>Liu Z, <i>et al.</i> (2008 Jan)</a>. Activation of the SPS amino acid-sensing pathway in Saccharomyces cerevisiae correlates with the phosphorylation state of a sensor component, Ptr3.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02470_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02470_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02470_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02470_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02470">Phobius transmembrane predictions</a>
										<br>29 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8d4ea" class="col">
			FOG02471<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, amino acid transport, transmembrane transport and integral component of membrane, membrane localization|Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, amino acid transport, transmembrane transport and integral component of membrane, membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02471_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02471_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02471_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02471_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02471">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eeadad" class="col">
			FOG02472<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, transmembrane transport and membrane localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02472_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02472_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02472_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02472_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02472">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bacde4" class="col">
			FOG02473<br>EOG8HQC0N<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>APC amino acid transmembrane transporter (predicted)<br><br>
			<br><b>AspGD Description</b><br>Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, transmembrane transport and membrane localization|Has domain(s) with predicted amino acid transmembrane transporter activity, role in amino acid transmembrane transport, transmembrane transport and membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21832151" target=_blank>Kashiwazaki J, <i>et al.</i> (2011 Oct)</a>. Endocytosis is essential for dynamic translocation of a syntaxin 1 orthologue during fission yeast meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG02473_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02473_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG02473_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG02473_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01050_phobius.html#FOG02473">Phobius transmembrane predictions</a>
										<br>8 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG01050.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>