<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00427.html">HOG00427&emsp;&emsp;FGGY kinase family</a><br> 
																	HOG00428&emsp;&emsp;<br> 
																	<a href="HOG00429.html">HOG00429&emsp;&emsp;carbohydrate kinase PfkB family. Ribokinase subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00428.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00428_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00428_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00428_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00428_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00908<br>EOG81JWVD<br>XYL2<br>sce:XYL2<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Xylitol dehydrogenase<br><br>
			<br><b>SGD Description</b><br>Xylitol dehydrogenase; converts xylitol to D-xylulose; expression induced by xylose, even though this pentose sugar is not well utilized by S. cerevisiae; null mutant has cell wall defect<br><br>
			<br><b>PomBase Description</b><br>hexitol dehydrogenase (predicted)<br><br>
			<br><b>AspGD Description</b><br>D-xylulose reductase (xylitol dehydrogenase); L-iditol 2-dehydrogenase; XlnR regulated gene; upregulated on l-arabinose, not on d-galactose, galactitol or d-sorbitol<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2127555" target=_blank>KÃ¶tter P, <i>et al.</i> (1990 Dec)</a>. Isolation and characterization of the Pichia stipitis xylitol dehydrogenase gene, XYL2, and construction of a xylose-utilizing Saccharomyces cerevisiae transformant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8504864" target=_blank>Persson B, <i>et al.</i> (1993 Jun 7)</a>. Dual relationships of xylitol and alcohol dehydrogenases in families of two protein types.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7737676" target=_blank>Wagner P, <i>et al.</i> (1994 Dec)</a>. Isolation and characterization of two monoclonal antibodies raised against tms1 protein of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10486580" target=_blank>Richard P, <i>et al.</i> (1999 Aug 20)</a>. Evidence that the gene YLR070c of Saccharomyces cerevisiae encodes a xylitol dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10849789" target=_blank>Shi NQ, <i>et al.</i> (2000 Spring)</a>. Characterization and complementation of a Pichia stipitis mutant unable to grow on D-xylose or L-arabinose.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18332432" target=_blank>Andersen MR, <i>et al.</i> (2008 Mar 18)</a>. A trispecies Aspergillus microarray: comparative transcriptomics of three Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21484208" target=_blank>Battaglia E, <i>et al.</i> (2011 Jul)</a>. Regulation of pentose utilisation by AraR, but not XlnR, differs in Aspergillus nidulans and Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23766336" target=_blank>Coradetti ST, <i>et al.</i> (2013 Aug)</a>. Analysis of a conserved cellulase transcriptional regulator reveals inducer-independent production of cellulolytic enzymes in Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695164" target=_blank>Das J, <i>et al.</i> (2013 May 21)</a>. Cross-species protein interactome mapping reveals species-specific wiring of stress response pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26580075" target=_blank>Kowalczyk JE, <i>et al.</i> (2015)</a>. Genetic Interaction of Aspergillus nidulans galR, xlnR and araR in Regulating D-Galactose and L-Arabinose Release and Catabolism Gene Expression.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00908_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00908_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00908_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00908_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00428_phobius.pdf">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00909<br>EOG81JWVD<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>D-xylulose reductase (xylitol dehydrogenase); sorbitol (glucitol) dehydrogenase (NAD+)|D-xylulose reductase (xylitol dehydrogenase); sorbitol (glucitol) dehydrogenase (NAD+)<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00909_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00909_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00909_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00909_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00428_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00910<br>EOG81JWVD<br><br>sce:SOR1;SOR2<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>XYL2 unknown duplications<br><br>
			<br><b>SGD Description</b><br>Sorbitol dehydrogenase; expression is induced in the presence of sorbitol or xylose|Protein of unknown function; protein sequence is 99% identical to the Sor1p sorbitol dehydrogenase; computational analysis of large-scale protein-protein interaction data also suggests a role in fructose or mannose metabolism<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8125328" target=_blank>Sarthy AV, <i>et al.</i> (1994 Mar 11)</a>. Cloning and sequence determination of the gene encoding sorbitol dehydrogenase from Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00910_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00910_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00910_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00910_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00428_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00911<br>EOG81JWVD<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>L-arabinitol 4-dehydrogenase.<br><br>
			<br><b>AspGD Description</b><br>Putative L-arabitol dehydrogenase; D-sorbitol dehydrogenase; induced on l-arabinose but not on galactitol or d-galactose|D-sorbitol dehydrogenase|L-iditol 2-dehydrogenase (sorbitol dehydrogenase); exhibits galactitol-3-dehydrogenase activity; upregulated on d-galactose and galactitol, but not on l-arabinose or d-glucose<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7988903" target=_blank>de Vries RP, <i>et al.</i> (1994 Oct 15)</a>. Characterization of an Aspergillus nidulans L-arabitol dehydrogenase mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217508" target=_blank>Gielkens M, <i>et al.</i> (1999 Mar)</a>. The abfB gene encoding the major alpha-L-arabinofuranosidase of Aspergillus nidulans: nucleotide sequence, regulation and construction of a disrupted strain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14624333" target=_blank>Fekete E, <i>et al.</i> (2004 Jan)</a>. The alternative D-galactose degrading pathway of Aspergillus nidulans proceeds via L-sorbose.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16310740" target=_blank>Suzuki T, <i>et al.</i> (2005 Oct)</a>. Cloning and expression of NAD+-dependent L-arabinitol 4-dehydrogenase gene (ladA) of Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18332432" target=_blank>Andersen MR, <i>et al.</i> (2008 Mar 18)</a>. A trispecies Aspergillus microarray: comparative transcriptomics of three Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20414651" target=_blank>Kim B, <i>et al.</i> (2010 Jul)</a>. Cloning, characterization, and engineering of fungal L-arabinitol dehydrogenases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21821745" target=_blank>Christensen U, <i>et al.</i> (2011 Oct)</a>. Unique regulatory mechanism for D-galactose utilization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22155165" target=_blank>Mojzita D, <i>et al.</i> (2012 Feb)</a>. Identification of the galactitol dehydrogenase, LadB, that is part of the oxido-reductive D-galactose catabolic pathway in Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23472095" target=_blank>Colabardini AC, <i>et al.</i> (2013)</a>. Functional characterization of Aspergillus nidulans ypkA, a homologue of the mammalian kinase SGK.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26580075" target=_blank>Kowalczyk JE, <i>et al.</i> (2015)</a>. Genetic Interaction of Aspergillus nidulans galR, xlnR and araR in Regulating D-Galactose and L-Arabinose Release and Catabolism Gene Expression.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00911_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00911_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00911_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00911_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00428_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00912<br>EOG81JWVD<br><br>sce:absent<br><br>Genes: 14
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>unknown function; monophyletic<br><br>
			<br><b>AspGD Description</b><br>D-sorbitol dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00912_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00912_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00912_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00912_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00428_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00913<br>EOG81JWVD<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>unknown function; monophyletic<br><br>
			<br><b>AspGD Description</b><br>L-iditol 2-dehydrogenase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00913_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00913_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00913_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00913_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00428_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00914<br>EOG81JWVD<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Sorbitol dehydrogenase; enzyme of the last step of the oxido-reductive d-galactose pathway, oxidizes d-sorbitol to d-fructose; essential enzyme for growth on d-sorbitol as a sole carbon source; strongly upregulated on d-sorbitol|D-Xylulose reductase (xylitol dehydrogenase); sorbitol (glucitol) dehydrogenase (NAD+)<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00914_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00914_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00914_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00914_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00428_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00428.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>