<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01474.html">HOG01474&emsp;&emsp;</a><br> 
																	HOG01475&emsp;&emsp;<br> 
																	<a href="HOG01476.html">HOG01476&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01475.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01475_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01475_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01475_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG04260<br>EOG8ZS7MD<br><br>sce:SET1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Histone methyltransferase, subunit of the COMPASS (Set1C) complex; COMPASS methylates histone H3K4; Set1p-dependent H3K4 trimethylation recruits Nrd1p, allowing efficient termination of snoRNAs and cryptic unstable transcripts (CUTs) by Nrd1p-Nab3p-Sen1p pathway; modulates histone acetylation levels in promoter proximal regions to ensure efficient Nrd1p-dependent termination; required in transcriptional silencing near telomeres and at silent mating type loci; has a SET domain<br><br>
			<br><b>PomBase Description</b><br>histone lysine methyltransferase Set1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have histone methyltransferase activity (H3-K4 specific) activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9398665" target=_blank>Nislow C, <i>et al.</i> (1997 Dec)</a>. SET1, a yeast member of the trithorax family, functions in transcriptional silencing and diverse cellular processes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9988274" target=_blank>Corda Y, <i>et al.</i> (1999 Feb)</a>. Interaction between Set1p and checkpoint protein Mec3p in DNA repair and telomere functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751634" target=_blank>Briggs SD, <i>et al.</i> (2001 Dec 15)</a>. Histone H3 lysine 4 methylation is mediated by Set1 and required for cell growth and rDNA silencing in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742990" target=_blank>Roguev A, <i>et al.</i> (2001 Dec 17)</a>. The Saccharomyces cerevisiae Set1 complex includes an Ash2 homologue and methylates histone 3 lysine 4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11687631" target=_blank>Miller T, <i>et al.</i> (2001 Nov 6)</a>. COMPASS: a complex of proteins associated with a trithorax-related SET domain protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11818070" target=_blank>Bryk M, <i>et al.</i> (2002 Jan 22)</a>. Evidence that Set1, a factor required for methylation of histone H3, regulates rDNA silencing in S. cerevisiae by a Sir2-independent mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752412" target=_blank>Nagy PL, <i>et al.</i> (2002 Jan 8)</a>. A trithorax-group complex purified from Saccharomyces cerevisiae is required for methylation of histone H3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060701" target=_blank>Bernstein BE, <i>et al.</i> (2002 Jun 25)</a>. Methylation of histone H3 Lys 4 in coding regions of active genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11805083" target=_blank>Krogan NJ, <i>et al.</i> (2002 Mar 29)</a>. COMPASS, a histone H3 (Lysine 4) methyltransferase required for telomeric silencing of gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12353038" target=_blank>Santos-Rosa H, <i>et al.</i> (2002 Sep 26)</a>. Active genes are tri-methylated at K4 of histone H3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845608" target=_blank>Boa S, <i>et al.</i> (2003 Jul 15)</a>. Saccharomyces cerevisiae Set1p is a methyltransferase specific for lysine 4 of histone H3 and is required for efficient gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14636589" target=_blank>Santos-Rosa H, <i>et al.</i> (2003 Nov)</a>. Methylation of histone H3 K4 mediates association of the Isw1p ATPase with chromatin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15949446" target=_blank>Morillon A, <i>et al.</i> (2005 Jun 10)</a>. Dynamic lysine methylation on histone H3 defines the regulatory phase of gene transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16629671" target=_blank>Raman SB, <i>et al.</i> (2006 May)</a>. Candida albicans SET1 encodes a histone 3 lysine 4 methyltransferase that contributes to the pathogenesis of invasive candidiasis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04260_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04260_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04260_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04260_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01475_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG04261<br>EOG8ZS7MD<br><br>sce:SET2<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Histone methyltransferase with a role in transcriptional elongation; methylates H3 lysine 36 (H3K36), which suppresses incorporation of acetylated histones and signals for the deacetylation of these histones within transcribed genes; associates with the C-terminal domain(CTD) of Rpo21p; H3K36me3 (trimethylation) requires Spt6p, proline 38 on H3, CTD of Rpo21p, Ctk1p, and C-terminal SRI domain of Ste2p; relocalizes to the cytosol in response to hypoxia<br><br>
			<br><b>PomBase Description</b><br>histone lysine methyltransferase Set2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have histone methyltransferase activity (H3-K36 specific) activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802733" target=_blank>Macias MJ, <i>et al.</i> (2000 May)</a>. Structural analysis of WW domains and design of a WW prototype.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12381723" target=_blank>Li J, <i>et al.</i> (2002 Dec 20)</a>. Association of the histone methyltransferase Set2 with RNA polymerase II plays a role in transcription elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11839797" target=_blank>Strahl BD, <i>et al.</i> (2002 Mar)</a>. Set2 is a nucleosomal histone H3-selective methyltransferase that mediates transcriptional repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773564" target=_blank>Krogan NJ, <i>et al.</i> (2003 Jun)</a>. Methylation of histone H3 by Set2 in Saccharomyces cerevisiae is linked to transcriptional elongation by RNA polymerase II.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12629047" target=_blank>Xiao T, <i>et al.</i> (2003 Mar 1)</a>. Phosphorylation of RNA polymerase II CTD regulates H3 methylation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511561" target=_blank>Li B, <i>et al.</i> (2003 Mar 14)</a>. The Set2 histone methyltransferase functions through the phosphorylated carboxyl-terminal domain of RNA polymerase II.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12736296" target=_blank>Schaft D, <i>et al.</i> (2003 May 15)</a>. The histone 3 lysine 36 methyltransferase, SET2, is involved in transcriptional elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12917322" target=_blank>Landry J, <i>et al.</i> (2003 Sep)</a>. Set2-catalyzed methylation of histone H3 represses basal expression of GAL4 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15798214" target=_blank>Kizer KO, <i>et al.</i> (2005 Apr)</a>. A novel domain in Set2 mediates RNA polymerase II interaction and couples histone H3 K36 methylation with transcript elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16227595" target=_blank>Rao B, <i>et al.</i> (2005 Nov)</a>. Dimethylation of histone H3 at lysine 36 demarcates regulatory and nonregulatory chromatin genome-wide.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16329992" target=_blank>Tripic T, <i>et al.</i> (2006 Jan 20)</a>. The Set2 methyltransferase associates with Ssn6 yet Tup1-Ssn6 repression is independent of histone methylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16286474" target=_blank>Vojnic E, <i>et al.</i> (2006 Jan 6)</a>. Structure and carboxyl-terminal domain (CTD) binding of the Set2 SRI domain that couples histone H3 Lys36 methylation to transcription.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04261_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04261_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04261_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04261_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01475_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG04262<br>EOG8ZS7MD<br><br>sce:absent<br><br>Genes: 15
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted histone-lysine N-methyltransferase activity and nucleus localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04262_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04262_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04262_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04262_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01475_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01475.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>