<html>
<head><title>HOG02525</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG02524.html">HOG02524&emsp;&emsp;</a><br> 
																	HOG02525&emsp;&emsp;<br> 
																	<a href="HOG02526.html">HOG02526&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG02525.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG02525_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG02525_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG02525_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG02525.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG06146<br>EOG8RFJ96<br><br>sce:SAE2<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Endonuclease required for telomere elongation; also required for telomeric 5' C-rich strand resection; involved in processing hairpin DNA structures with MRX complex; involved in double-strand break repair; sumoylation and phosphorylation both required for function; exists in form of inactive oligomers that are transiently released into smaller active units by a series of phosphorylations; DNA damage triggers removal of Sae2p, so active Sae2p is present only transiently<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9215888" target=_blank>McKee AH, <i>et al.</i> (1997 Jul)</a>. A general method for identifying recessive diploid-specific mutations in Saccharomyces cerevisiae, its application to the isolation of mutants blocked at intermediate stages of meiotic prophase and characterization of a new gene SAE2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9215887" target=_blank>Prinz S, <i>et al.</i> (1997 Jul)</a>. Isolation of COM1, a new gene required to complete meiotic double-strand break-induced recombination in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11052944" target=_blank>Borde V, <i>et al.</i> (2000 Oct 27)</a>. Direct coupling between meiotic DNA replication and recombination initiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333222" target=_blank>Rattray AJ, <i>et al.</i> (2001 May)</a>. Fidelity of mitotic double-strand-break repair in Saccharomyces cerevisiae: a role for SAE2/COM1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11983174" target=_blank>Neale MJ, <i>et al.</i> (2002 Apr)</a>. Wild-type levels of Spo11-induced DSBs are required for normal single-strand resection during meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11832209" target=_blank>Lobachev KS, <i>et al.</i> (2002 Jan 25)</a>. The Mre11 complex is required for repair of hairpin-capped double-strand breaks and prevention of chromosome rearrangements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12925751" target=_blank>Viscardi V, <i>et al.</i> (2003 Aug)</a>. Sudden telomere lengthening triggers a Rad53-dependent checkpoint in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12839620" target=_blank>Fukuda T, <i>et al.</i> (2003 Jul)</a>. VDE-initiated intein homing in Saccharomyces cerevisiae proceeds in a meiotic recombination-like manner.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14729978" target=_blank>Yu J, <i>et al.</i> (2004 Feb)</a>. Microhomology-dependent end joining and repair of transposon-induced DNA hairpins by host factors in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15121837" target=_blank>Baroni E, <i>et al.</i> (2004 May)</a>. The functions of budding yeast Sae2 in the DNA damage response require Mec1- and Tel1-dependent phosphorylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15369670" target=_blank>Lisby M, <i>et al.</i> (2004 Sep 17)</a>. Choreography of the DNA damage response: spatiotemporal relationships among checkpoint and repair proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15655113" target=_blank>Prieler S, <i>et al.</i> (2005 Jan 15)</a>. The control of Spo11's interaction with meiotic recombination hotspots.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15834151" target=_blank>Deng C, <i>et al.</i> (2005 Jun)</a>. Multiple endonucleases function to repair covalent topoisomerase I complexes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15937224" target=_blank>Rattray AJ, <i>et al.</i> (2005 Jun 1)</a>. A mechanism of palindromic gene amplification in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16162495" target=_blank>Clerici M, <i>et al.</i> (2005 Nov 18)</a>. The Saccharomyces cerevisiae Sae2 protein promotes resection and bridging of double strand break ends.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16374511" target=_blank>Clerici M, <i>et al.</i> (2006 Feb)</a>. The Saccharomyces cerevisiae Sae2 protein negatively regulates DNA damage checkpoint signalling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16861895" target=_blank>Cartagena-Lirola H, <i>et al.</i> (2006 Jul)</a>. Budding Yeast Sae2 is an In Vivo Target of the Mec1 and Tel1 Checkpoint Kinases During Meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17565964" target=_blank>Lee K, <i>et al.</i> (2007 Aug)</a>. Saccharomyces cerevisiae Sae2- and Tel1-dependent single-strand DNA formation at DNA break promotes microhomology-mediated end joining.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17989249" target=_blank>Suter B, <i>et al.</i> (2007 Dec)</a>. Examining protein protein interactions using endogenously tagged yeast arrays: the cross-and-capture system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18042458" target=_blank>Lengsfeld BM, <i>et al.</i> (2007 Nov 30)</a>. Sae2 is an endonuclease that processes hairpin DNA cooperatively with the Mre11/Rad50/Xrs2 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18670132" target=_blank>Terasawa M, <i>et al.</i> (2008 Jun)</a>. Sae2p phosphorylation is crucial for cooperation with Mre11p for resection of DNA double-strand break ends during meiotic recombination in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18716619" target=_blank>Huertas P, <i>et al.</i> (2008 Oct 2)</a>. CDK targets Sae2 to control DNA-end resection and homologous recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18806779" target=_blank>Mimitou EP, <i>et al.</i> (2008 Oct 9)</a>. Sae2, Exo1 and Sgs1 collaborate in DNA double-strand break processing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361851" target=_blank>Doksani Y, <i>et al.</i> (2009 Apr 17)</a>. Replicon dynamics, dormant origin firing, and terminal fork integrity after double-strand break formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19124276" target=_blank>LisniÄ‡ B, <i>et al.</i> (2009 Mar 1)</a>. Size-dependent palindrome-induced intrachromosomal recombination in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20061370" target=_blank>Janke R, <i>et al.</i> (2010 Apr)</a>. A truncated DNA-damage-signaling response is activated after DSB formation in the G1 phase of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20150422" target=_blank>Manfrini N, <i>et al.</i> (2010 Apr 9)</a>. Processing of meiotic DNA double strand breaks requires cyclin-dependent kinase and multiple nucleases.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06146_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06146_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06146_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG06146_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG02525_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG02525.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>