<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00162.html">HOG00162&emsp;&emsp;DNA polymerase type-B family</a><br> 
																	HOG00163&emsp;&emsp;<br> 
																	<a href="HOG00164.html">HOG00164&emsp;&emsp;helicase family. UvrD subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00163.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00163_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00163_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00163_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00163_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00220<br>EOG8VX0MD<br><br>sce:RAD54<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>DNA repair<br><br>
			<br><b>SGD Description</b><br>DNA-dependent ATPase that stimulates strand exchange; modifies the topology of double-stranded DNA; involved in the recombinational repair of double-strand breaks in DNA during vegetative growth and meiosis; member of the SWI/SNF family of DNA translocases; forms nuclear foci upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>DNA-dependent ATPase Rad54/Rhp54<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in DNA repair, ascospore formation, meiotic gene conversion, reciprocal meiotic recombination and mitotic spindle, nucleus, site of double-strand break localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1916269" target=_blank>Emery HS, <i>et al.</i> (1991 Jul 31)</a>. Sequence of RAD54, a Saccharomyces cerevisiae gene involved in recombination and repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7610042" target=_blank>Johnston LH, <i>et al.</i> (1995 Jun 25)</a>. The DNA repair genes RAD54 and UNG1 are cell cycle regulated in budding yeast but MCB promoter elements have no essential role in the DNA damage response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9590697" target=_blank>Petukhova G, <i>et al.</i> (1998 May 7)</a>. Catalysis of homologous DNA pairing by yeast Rad51 and Rad54 proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12453424" target=_blank>Solinger JA, <i>et al.</i> (2002 Nov)</a>. Rad54, a Swi2/Snf2-like recombinational repair protein, disassembles Rad51:dsDNA filaments.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00220_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00220_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00220_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00220_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00163_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00221<br>EOG8VX0MD<br><br>sce:RAD26<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>DNA repair<br><br>
			<br><b>SGD Description</b><br>Protein involved in transcription-coupled nucleotide excision repair; repairs UV-induced DNA lesions; recruitment to DNA lesions is dependent on an elongating RNA polymerase II; homolog of human CSB protein<br><br>
			<br><b>PomBase Description</b><br>SNF2 family ATP-dependent DNA helicase Rhp26<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in transcription-coupled nucleotide-excision repair and cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8198589" target=_blank>Huang ME, <i>et al.</i> (1994 May 30)</a>. A possible yeast homolog of human active-gene-repairing helicase ERCC6+.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957102" target=_blank>van Gool AJ, <i>et al.</i> (1994 Nov 15)</a>. RAD26, the functional S. cerevisiae homolog of the Cockayne syndrome B gene ERCC6.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10480889" target=_blank>Yasuhira S, <i>et al.</i> (1999 Sep 17)</a>. Transcription dependence and the roles of two excision repair pathways for UV damage in fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12807767" target=_blank>Marti TM, <i>et al.</i> (2003 Jun)</a>. Repair of damaged and mismatched DNA by the XPC homologues Rhp41 and Rhp42 of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18794354" target=_blank>Fukumoto Y, <i>et al.</i> (2008 Nov)</a>. Schizosaccharomyces pombe Ddb1 recruits substrate-specific adaptor proteins through a novel protein motif, the DDB-box.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20456449" target=_blank>Jaakkola E, <i>et al.</i> (2010 Dec)</a>. ERCC6 founder mutation identified in Finnish patients with COFS syndrome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21670547" target=_blank>Kanamitsu K, <i>et al.</i> (2011)</a>. Fission yeast homologs of human XPC and CSB, rhp41 and rhp26, are involved in transcription-coupled repair of methyl methanesulfonate-induced DNA damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22658721" target=_blank>Latypov VF, <i>et al.</i> (2012 Jul 13)</a>. Atl1 regulates choice between global genome and transcription-coupled repair of O(6)-alkylguanines.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25512493" target=_blank>Wang L, <i>et al.</i> (2014 Dec 30)</a>. Regulation of the Rhp26ERCC6/CSB chromatin remodeler by a novel conserved leucine latch motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24875629" target=_blank>Ding L, <i>et al.</i> (2014 May 28)</a>. Essential domains of Schizosaccharomyces pombe Rad8 required for DNA damage response.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00221_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00221_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00221_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00221_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00163_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00222<br>EOG8VX0MD<br><br>sce:RDH54<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>DNA repair<br><br>
			<br><b>SGD Description</b><br>DNA-dependent ATPase; DNA recombination/repair translocase, supercoils DNA and promotes DNA strand opening; stimulates strand exchange by modifying dsDNA topology; involved in recombinational repair of DNA double-strand breaks (DSBs) during mitosis and meiosis; phosphorylated in Mec1p-, Rad53p-dependent way in response to one DSB; contributes to remodelling of nucleosomes; proposed to be involved in crossover interference; interacts with Dmc1p; stimulates Dmc1p and Rad51p<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent DNA helicase Rdh54<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have DNA translocase activity, double-stranded DNA binding activity and role in DNA duplex unwinding, DNA repair, heteroduplex formation, meiotic sister chromatid segregation, reciprocal meiotic recombination<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409819" target=_blank>Klein HL, <i>et al.</i> (1997 Dec)</a>. RDH54, a RAD54 homologue in Saccharomyces cerevisiae, is required for mitotic diploid-specific recombination and repair and for meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409820" target=_blank>Shinohara M, <i>et al.</i> (1997 Dec)</a>. Characterization of the roles of the Saccharomyces cerevisiae RAD54 gene and a homologue of RAD54, RDH54/TID1, in mitosis and meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335591" target=_blank>Dresser ME, <i>et al.</i> (1997 Oct)</a>. DMC1 functions in a Saccharomyces cerevisiae meiotic pathway that is largely independent of the RAD51 pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10970884" target=_blank>Petukhova G, <i>et al.</i> (2000 Sep 1)</a>. Promotion of Rad51-dependent D-loop formation by yeast recombination factor Rdh54/Tid1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11005857" target=_blank>Shinohara M, <i>et al.</i> (2000 Sep 26)</a>. Tid1/Rdh54 promotes colocalization of rad51 and dmc1 during meiotic recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11470411" target=_blank>Lee SE, <i>et al.</i> (2001 Jul 10)</a>. The Saccharomyces recombination protein Tid1p is required for adaptation from G2/M arrest induced by a double-strand break.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12702674" target=_blank>Shinohara M, <i>et al.</i> (2003 Apr)</a>. Crossover interference in Saccharomyces cerevisiae requires a TID1/RDH54- and DMC1-dependent pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14551247" target=_blank>Catlett MG, <i>et al.</i> (2003 Nov)</a>. Schizosaccharomyces pombe Rdh54 (TID1) acts with Rhp54 (RAD54) to repair meiotic double-strand breaks.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998526" target=_blank>Goldman GH, <i>et al.</i> (2004 Apr)</a>. Aspergillus nidulans as a model system to characterize the DNA damage response in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16303567" target=_blank>Martín-Castellanos C, <i>et al.</i> (2005 Nov 22)</a>. A large-scale screen in S. pombe identifies seven novel genes required for critical meiotic events.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19028606" target=_blank>Chi P, <i>et al.</i> (2009 Feb 1)</a>. Functional interactions of meiotic recombination factors Rdh54 and Dmc1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21084840" target=_blank>Tougan T, <i>et al.</i> (2010 Dec 1)</a>. The Mek1 phosphorylation cascade plays a role in meiotic recombination of Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25293972" target=_blank>Mastro TL, <i>et al.</i> (2014 Dec)</a>. Increased meiotic crossovers and reduced genome stability in absence of Schizosaccharomyces pombe Rad16 (XPF).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25373780" target=_blank>Graml V, <i>et al.</i> (2014 Oct 27)</a>. A genomic Multiprocess survey of machineries that control and link cell shape, microtubule organization, and cell-cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojardín L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00222_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00222_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00222_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00222_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00163_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00223<br>EOG8VX0MD<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized paralog / ohnolog<br><br>
			<br><b>Parent</b><br>ohnolog?:FOG00222<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00223_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00223_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00223_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00223_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00163_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00163.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>