<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01185.html">HOG01185&emsp;&emsp;HRD1 family</a><br> 
																	HOG01186&emsp;&emsp;<br> 
																	<a href="HOG01187.html">HOG01187&emsp;&emsp;glycosyl hydrolase 76 family</a><br><br> 
																	<a href="../sequences/hog/HOG01186.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG01186_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG01186_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="HOG01186_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03094<br>EOG80K6G0<br><br>sce:DRN1<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Splicing factor that modulates turnover of branched RNAs by Dbr1p; interacts with spliceosomal components and branched RNA splicing products; enhances Dbr1p debranching in vitro; conserved protein with domain organization identical from yeast to human; N-terminal homology to Dbr1p N-terminus, but Dbr1p catalytic residues not conserved; relocalizes to the cytosol in response to hypoxia<br><br>
			<br><b>PomBase Description</b><br>CwfJ family protein, splicing factor (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22932476" target=_blank>Dastidar RG, <i>et al.</i> (2012 Aug 29)</a>. The nuclear localization of SWI/SNF proteins is subjected to oxygen regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24919400" target=_blank>Garrey SM, <i>et al.</i> (2014 Aug)</a>. A homolog of lariat-debranching enzyme modulates turnover of branched RNA.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03094_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03094_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03094_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03094_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03094">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03095<br>EOG80K6G0<br><br>sce:MSL5<br><br>Genes: 30
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Component of commitment complex; which defines first step in splicing pathway; essential protein that interacts with Mud2p and Prp40p, forming a bridge between the intron ends; also involved in nuclear retention of pre-mRNA; relocalizes to the cytosol in response to hypoxia<br><br>
			<br><b>PomBase Description</b><br>zinc finger splicing factor Bpb1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have U2-type prespliceosome, commitment complex, cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9150140" target=_blank>Abovich N, <i>et al.</i> (1997 May 2)</a>. Cross-intron bridging interactions in the yeast commitment complex are conserved in mammals.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10376880" target=_blank>Rutz B, <i>et al.</i> (1999 Jun)</a>. Transient interaction of BBP/ScSF1 and Mud2 with the splicing machinery affects the kinetics of spliceosome assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10775271" target=_blank>Rutz B, <i>et al.</i> (2000 Apr 17)</a>. A dual role for BBP/ScSF1 in nuclear pre-mRNA retention and splicing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10923022" target=_blank>Beales M, <i>et al.</i> (2000 Aug)</a>. Mutations in the large subunit of U2AF disrupt pre-mRNA splicing, cell cycle progression and nuclear structure.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12374752" target=_blank>Huang T, <i>et al.</i> (2002 Oct 15)</a>. Pre-spliceosome formation in S.pombe requires a stable complex of SF1-U2AF(59)-U2AF(23).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730023" target=_blank>Banerjee H, <i>et al.</i> (2004 Feb)</a>. The conserved RNA recognition motif 3 of U2 snRNA auxiliary factor (U2AF 65) is essential in vivo but dispensable for activity in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15470237" target=_blank>Kupfer DM, <i>et al.</i> (2004 Oct)</a>. Introns and splicing elements of five diverse fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16120600" target=_blank>Kofler M, <i>et al.</i> (2005 Nov)</a>. GYF domain proteomics reveals interaction sites in known and novel target proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17130122" target=_blank>Haraguchi N, <i>et al.</i> (2007 Jan 26)</a>. Mutations in the SF1-U2AF59-U2AF23 complex cause exon skipping in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20970342" target=_blank>Asakawa H, <i>et al.</i> (2010 Nov 9)</a>. Virtual breakdown of the nuclear envelope in fission yeast meiosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24014766" target=_blank>Livesay SB, <i>et al.</i> (2013 Nov)</a>. Structural and functional characterization of the N terminus of Schizosaccharomyces pombe Cwf10.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26302002" target=_blank>Sasaki-Haraguchi N, <i>et al.</i> (2015)</a>. Cwf16p Associating with the Nineteen Complex Ensures Ordered Exon Joining in Constitutive Pre-mRNA Splicing in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26167880" target=_blank>Lipp JJ, <i>et al.</i> (2015 Aug)</a>. SR protein kinases promote splicing of nonconsensus introns.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27984725" target=_blank>Swaffer MP, <i>et al.</i> (2016 Dec 15)</a>. CDK Substrate Phosphorylation and Ordering the Cell Cycle.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03095_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03095_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03095_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03095_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03095">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03096<br>EOG805QJR<br>EOG80K6G0<br><br>sce:AIR2;AIR1<br><br>Genes: 28
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>RNA-binding subunit of the TRAMP nuclear RNA surveillance complex; involved in nuclear RNA processing and degradation; involved in TRAMP complex assembly as a bridge between Mtr4p and Trf4p; stimulates the poly(A) polymerase activity of Pap2p in vitro; has 5 zinc knuckle motifs; AIR2 has a paralog, AIR1, that arose from the whole genome duplication; Air2p and Air1p have nonredundant roles in regulation of substrate specificity of the exosome|Zinc knuckle protein; involved in nuclear RNA processing and degradation as a component of the TRAMP complex; stimulates the poly(A) polymerase activity of Pap2p in vitro; AIR1 has a paralog, AIR2, that arose from the whole genome duplication; although Air1p and Air2p are homologous TRAMP subunits, they have nonredundant roles in regulation of substrate specificity of the exosome<br><br>
			<br><b>PomBase Description</b><br>zinc knuckle TRAMP complex subunit Air1<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10896665" target=_blank>Inoue K, <i>et al.</i> (2000 Oct 20)</a>. Novel RING finger proteins, Air1p and Air2p, interact with Hmt1p and inhibit the arginine methylation of Npl3p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15828860" target=_blank>Vanácová S, <i>et al.</i> (2005 Jun)</a>. A new yeast poly(A) polymerase complex involved in RNA quality control.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15935758" target=_blank>LaCava J, <i>et al.</i> (2005 Jun 3)</a>. RNA degradation by the exosome is promoted by a nuclear polyadenylation complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15935759" target=_blank>Wyers F, <i>et al.</i> (2005 Jun 3)</a>. Cryptic pol II transcripts are degraded by a nuclear quality control pathway involving a new poly(A) polymerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16374505" target=_blank>Houseley J, <i>et al.</i> (2006 Feb)</a>. Yeast Trf5p is a nuclear poly(A) polymerase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20696927" target=_blank>Hamill S, <i>et al.</i> (2010 Aug 24)</a>. Structure and function of the polymerase core of TRAMP, a RNA surveillance complex.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03096_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03096_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03096_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03096_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03096">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03097<br>EOG80K6G0<br><br>sce:GIS2<br><br>Genes: 25
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Translational activator for mRNAs with internal ribosome entry sites; associates with polysomes and binds to a specific subset of mRNAs; localizes to RNA processing bodies (P bodies) and to stress granules; may have a role in translation regulation under stress conditions; ortholog of human ZNF9/CNBP, a gene involved in myotonic dystrophy type 2<br><br>
			<br><b>PomBase Description</b><br>translational activator, zf-CCHC type zinc finger protein (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nucleolus localization|Ortholog(s) have mRNA binding, single-stranded DNA binding, translation regulator activity and role in cellular response to cold, cellular response to drug, cellular response to osmotic stress, positive regulation of translation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1515675" target=_blank>Xu HP, <i>et al.</i> (1992 Jul)</a>. A gene encoding a protein with seven zinc finger domains acts on the sexual differentiation pathways of Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8534915" target=_blank>Kanoh J, <i>et al.</i> (1995 Sep)</a>. Schizosaccharomyces pombe zfs1+ encoding a zinc-finger protein functions in the mating pheromone recognition pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10628841" target=_blank>Balciunas D, <i>et al.</i> (1999 Dec)</a>. Yeast genes GIS1-4: multicopy suppressors of the Gal- phenotype of snf1 mig1 srb8/10/11 cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10924454" target=_blank>Cullen CF, <i>et al.</i> (2000 Aug)</a>. A new genetic method for isolating functionally interacting genes: high plo1(+)-dependent mutants and their suppressors define genes in mitotic and septation pathways in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03097_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03097_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03097_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03097_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03097">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03098<br>EOG80K6G0<br>EOG8JHB08<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03098_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03098_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03098_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03098_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03098">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03099<br>EOG80K6G0<br>EOG8JHB08<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03099_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03099_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03099_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03099_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03099">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG03100<br>EOG80K6G0<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03100_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03100_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03100_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03100_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03100">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG03101<br>EOG80K6G0<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted nucleic acid binding, zinc ion binding activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03101_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03101_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03101_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03101_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03101">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG03102<br>EOG80K6G0<br><br>sce:absent<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>zf-CCHC type zinc finger protein (predicted)<br><br>
			<br><b>AspGD Description</b><br>Protein of unknown function|Protein of unknown function|Protein of unknown function|Protein of unknown function<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG03102_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03102_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG03102_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG03102_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG01186_phobius.html#FOG03102">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG01186.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>