<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01159.html">HOG01159&emsp;&emsp;helicase family. SKI2 subfamily</a><br> 
																	HOG01160&emsp;&emsp;<br> 
																	<a href="HOG01161.html">HOG01161&emsp;&emsp;peptidase M1 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01160.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01160_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01160_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02842<br>EOG8RV17C<br><br>sce:TIF1; TIF2<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>translation initiation factor eIF4A (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3057442" target=_blank>Linder P, <i>et al.</i> (1988 Nov 11)</a>. Sequence of the genes TIF1 and TIF2 from Saccharomyces cerevisiae coding for a translation initiation factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648398" target=_blank>Linder P, <i>et al.</i> (1989 Apr)</a>. An essential yeast protein, encoded by duplicated genes TIF1 and TIF2 and homologous to the mammalian translation initiation factor eIF-4A, can suppress a mitochondrial missense mutation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668952" target=_blank>Blum S, <i>et al.</i> (1989 Aug)</a>. Translation in Saccharomyces cerevisiae: initiation factor 4A-dependent cell-free system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2169890" target=_blank>Altmann M, <i>et al.</i> (1990 Aug 27)</a>. Translation initiation factor-dependent extracts from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2046664" target=_blank>Schmid SR, <i>et al.</i> (1991 Jul)</a>. Translation initiation factor 4A from Saccharomyces cerevisiae: analysis of residues conserved in the D-E-A-D family of RNA helicases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2041753" target=_blank>Foreman PK, <i>et al.</i> (1991 May 25)</a>. The Saccharomyces cerevisiae RPB4 gene is tightly linked to the TIF2 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1502180" target=_blank>Blum S, <i>et al.</i> (1992 Aug 15)</a>. ATP hydrolysis by initiation factor 4A is required for translation initiation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1408806" target=_blank>Smiley JK, <i>et al.</i> (1992 Sep 25)</a>. The 66 kDa component of yeast SFI, stimulatory factor I, is hsp60.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8524228" target=_blank>Cheng C, <i>et al.</i> (1995 Dec)</a>. Requirement of the self-glucosylating initiator proteins Glg1p and Glg2p for glycogen accumulation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8971628" target=_blank>Mirbod F, <i>et al.</i> (1996 Nov-Dec)</a>. Molecular cloning of a gene encoding translation initiation factor (TIF) from Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9298649" target=_blank>Garrels JI, <i>et al.</i> (1997 Aug)</a>. Proteome studies of Saccharomyces cerevisiae: identification and characterization of abundant proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10409745" target=_blank>Neff CL, <i>et al.</i> (1999 Aug)</a>. Eukaryotic translation initiation factors 4G and 4A from Saccharomyces cerevisiae interact physically and functionally.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10606264" target=_blank>Johnson ER, <i>et al.</i> (1999 Dec)</a>. Crystallographic structure of the amino terminal domain of yeast initiation factor 4A, a representative DEAD-box RNA helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10404596" target=_blank>Benz J, <i>et al.</i> (1999 Jun 15)</a>. Crystal structure of the ATPase domain of translation initiation factor 4A from Saccharomyces cerevisiae--the prototype of the DEAD box protein family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10462529" target=_blank>Daga RR, <i>et al.</i> (1999 Sep)</a>. Translational control of the cdc25 cell cycle phosphatase: a molecular mechanism coupling mitosis to cell growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10480875" target=_blank>Dominguez D, <i>et al.</i> (1999 Sep 17)</a>. Interaction of translation initiation factor eIF4G with eIF4A in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11087862" target=_blank>Caruthers JM, <i>et al.</i> (2000 Nov 21)</a>. Crystal structure of yeast initiation factor 4A, a DEAD-box RNA helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535527" target=_blank>Tanner NK, <i>et al.</i> (2003 Jan)</a>. The Q motif: a newly identified motif in DEAD box helicases may regulate ATP binding and hydrolysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919853" target=_blank>Oh YT, <i>et al.</i> (2010 Mar)</a>. Proteomic analysis of early phase of conidia germination in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22119525" target=_blank>Singh NS, <i>et al.</i> (2011 Dec 6)</a>. SIN-inhibitory phosphatase complex promotes Cdc11p dephosphorylation and propagates SIN asymmetry in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02842_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02842_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02842_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02842_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02843<br>EOG8RV17C<br><br>sce:DHH1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic DExD/H-box helicase, stimulates mRNA decapping; coordinates distinct steps in mRNA function and decay, interacts with both the decapping and deadenylase complexes, role in translational repression, mRNA decay, and processing body dynamics; may have a role in mRNA export; C-terminus of Dhh1p interacts with Ngr1p and promotes POR1, but not EDC1 mRNA decay; forms cytoplasmic foci upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Ste13<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have chromatin binding, mRNA binding, translation regulator activity, nucleic acid binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8511971" target=_blank>Strahl-Bolsinger S, <i>et al.</i> (1993 Apr)</a>. A yeast gene encoding a putative RNA helicase of the "DEAD"-box family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8078473" target=_blank>Maekawa H, <i>et al.</i> (1994 Sep 1)</a>. The ste13+ gene encoding a putative RNA helicase is essential for nitrogen starvation-induced G1 arrest and initiation of sexual development in the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9504907" target=_blank>Hata H, <i>et al.</i> (1998 Feb)</a>. Dhh1p, a putative RNA helicase, associates with the general transcription factors Pop2p and Ccr4p from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11780629" target=_blank>Coller JM, <i>et al.</i> (2001 Dec)</a>. The DEAD box helicase, Dhh1p, functions in mRNA decapping and interacts with both the decapping and deadenylase complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11696541" target=_blank>Maillet L, <i>et al.</i> (2002 Jan 25)</a>. Interaction between Not1p, a component of the Ccr4-not complex, a global regulator of transcription, and Dhh1p, a putative RNA helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12032091" target=_blank>Fischer N, <i>et al.</i> (2002 Jun 3)</a>. The DEAD box protein Dhh1 stimulates the decapping enzyme Dcp1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730603" target=_blank>Sheth U, <i>et al.</i> (2003 May 2)</a>. Decapping and decay of messenger RNA occur in cytoplasmic processing bodies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12930949" target=_blank>Tseng-Rogenski SS, <i>et al.</i> (2003 Sep 1)</a>. Functional conservation of Dhh1p, a cytoplasmic DExD/H-box protein present in large complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15166134" target=_blank>Bergkessel M, <i>et al.</i> (2004 May)</a>. An essential role for the Saccharomyces cerevisiae DEAD-box helicase DHH1 in G1/S DNA-damage checkpoint recovery.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15703442" target=_blank>Teixeira D, <i>et al.</i> (2005 Apr)</a>. Processing bodies require RNA for assembly and contain nontranslating mRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15987810" target=_blank>Cheng Z, <i>et al.</i> (2005 Aug)</a>. Crystal structure and functional analysis of DEAD-box protein Dhh1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15706350" target=_blank>Muhlrad D, <i>et al.</i> (2005 Mar 9)</a>. The yeast EDC1 mRNA undergoes deadenylation-independent decapping stimulated by Not2p, Not4p, and Not5p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19336419" target=_blank>Lemieux C, <i>et al.</i> (2009 Jun)</a>. Cotranscriptional recruitment of the nuclear poly(A)-binding protein Pab2 to nascent transcripts and association with translating mRNPs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20471941" target=_blank>Talarek N, <i>et al.</i> (2010 May 14)</a>. Initiation of the TORC1-regulated G0 program requires Igo1/2, which license specific mRNAs to evade degradation via the 5'-3' mRNA decay pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23319050" target=_blank>Wang CY, <i>et al.</i> (2013 Mar)</a>. Pdc1 functions in the assembly of P bodies in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26098573" target=_blank>Hu G, <i>et al.</i> (2015 Jul)</a>. A conserved mechanism of TOR-dependent RCK-mediated mRNA degradation regulates autophagy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27354705" target=_blank>Wurm JP, <i>et al.</i> (2016 Sep)</a>. The S. pombe mRNA decapping complex recruits cofactors and an Edc1-like activator through a single dynamic surface.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02843_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02843_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02843_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02843_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02844<br>EOG8RV17C<br><br>sce:DBP5<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cytoplasmic ATP-dependent RNA helicase of the DEAD-box family; involved in mRNA export from the nucleus, remodeling messenger ribonucleoprotein particles (mRNPs), with ATPase activity stimulated by Gle1p, IP6 and Nup159p; involved in translation termination along with Sup45p (eRF1); role in the cellular response to heat stress<br><br>
			<br><b>PomBase Description</b><br>cytoplasmic ATP-dependent RNA helicase Dbp5 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nuclear envelope localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2406722" target=_blank>Chang TH, <i>et al.</i> (1990 Feb)</a>. Identification of five putative yeast RNA helicase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9564048" target=_blank>Snay-Hodge CA, <i>et al.</i> (1998 May 1)</a>. Dbp5p/Rat8p is a yeast nuclear pore-associated DEAD-box protein essential for RNA export.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9564047" target=_blank>Tseng SS, <i>et al.</i> (1998 May 1)</a>. Dbp5p, a cytosolic RNA helicase, is required for poly(A)+ RNA export.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10428971" target=_blank>Schmitt C, <i>et al.</i> (1999 Aug 2)</a>. Dbp5, a DEAD-box protein required for mRNA export, is recruited to the cytoplasmic fibrils of nuclear pore complex via a conserved interaction with CAN/Nup159p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10523319" target=_blank>Hodge CA, <i>et al.</i> (1999 Oct 15)</a>. Rat8p/Dbp5p is a shuttling transport factor that interacts with Rat7p/Nup159p and Gle1p and suppresses the mRNA export defect of xpo1-1 cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10610322" target=_blank>Strahm Y, <i>et al.</i> (1999 Oct 15)</a>. The RNA export factor Gle1p is located on the cytoplasmic fibrils of the NPC and physically interacts with the FG-nucleoporin Rip1p, the DEAD-box protein Rat8p/Dbp5p and a new protein Ymr 255p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350039" target=_blank>Hilleren P, <i>et al.</i> (2001 May)</a>. Defects in the mRNA export factors Rat7p, Gle1p, Mex67p, and Rat8p cause hyperadenylation during 3'-end formation of nascent transcripts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12192043" target=_blank>Hammell CM, <i>et al.</i> (2002 Sep)</a>. Coupling of termination, 3' processing, and mRNA export.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12686617" target=_blank>Estruch F, <i>et al.</i> (2003 Apr)</a>. An early function during transcription for the yeast mRNA export factor Dbp5p/Rat8p suggested by its genetic and physical interactions with transcription factor IIH components.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15280434" target=_blank>Takemura R, <i>et al.</i> (2004 Aug 15)</a>. Stress response in yeast mRNA export factor: reversible changes in Rat8p localization are caused by ethanol stress but not heat shock.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15574330" target=_blank>Weirich CS, <i>et al.</i> (2004 Dec 3)</a>. The N-terminal domain of Nup159 forms a beta-propeller that functions in mRNA export by tethering the helicase Dbp5 to the nuclear pore.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15507118" target=_blank>Yuasa T, <i>et al.</i> (2004 Nov)</a>. An interactive gene network for securin-separase, condensin, cohesin, Dis1/Mtc1 and histones constructed by mass transformation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15619606" target=_blank>Estruch F, <i>et al.</i> (2005 Mar 11)</a>. Physical and genetic interactions link the yeast protein Zds1p with mRNA nuclear export.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22119525" target=_blank>Singh NS, <i>et al.</i> (2011 Dec 6)</a>. SIN-inhibitory phosphatase complex promotes Cdc11p dephosphorylation and propagates SIN asymmetry in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21386897" target=_blank>Ren L, <i>et al.</i> (2011 Feb 28)</a>. Systematic two-hybrid and comparative proteomic analyses reveal novel yeast pre-mRNA splicing factors connected to Prp19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21340088" target=_blank>Arita Y, <i>et al.</i> (2011 May)</a>. Microarray-based target identification using drug hypersensitive fission yeast expressing ORFeome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22540037" target=_blank>Pancaldi V, <i>et al.</i> (2012 Apr)</a>. Predicting the fission yeast protein interaction network.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02844_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02844_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02844_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02844_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02845<br>EOG8RV17C<br><br>sce:FAL1<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Nucleolar protein required for maturation of 18S rRNA; member of the eIF4A subfamily of DEAD-box ATP-dependent RNA helicases<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9372960" target=_blank>Kressler D, <i>et al.</i> (1997 Dec)</a>. Fal1p is an essential DEAD-box protein involved in 40S-ribosomal-subunit biogenesis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18681941" target=_blank>Savoldi M, <i>et al.</i> (2008 Oct)</a>. Farnesol induces the transcriptional accumulation of the Aspergillus nidulans Apoptosis-Inducing Factor (AIF)-like mitochondrial oxidoreductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23950735" target=_blank>Sun LL, <i>et al.</i> (2013)</a>. Global analysis of fission yeast mating genes reveals new autophagy factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02845_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02845_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02845_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02845_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01160.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>