<html>
<head><title>HOG01708</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01707.html">HOG01707&emsp;&emsp;</a><br> 
																	HOG01708&emsp;&emsp;<br> 
																	<a href="HOG01709.html">HOG01709&emsp;&emsp;SOP4 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01708.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01708_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01708_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01708_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG01708.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG04772<br>EOG8K0P3C<br><br>sce:RAD16<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Nucleotide excision repair (NER) protein; binds damaged DNA during NER; binds DNA in an ATP-dependent manner (with Rad7p) during NER; required for NER of non-transcribed chromatin; subunit of Nucleotide Excision Repair Factor 4 (NEF4) and the Elongin-Cullin-Socs (ECS) ligase complex<br><br>
			<br><b>PomBase Description</b><br>Rad16 homolog ATP-dependent DNA helicase/ ubiquitin protein ligase E3 Rhp16<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in nucleotide-excision repair, pyrimidine dimer repair and nucleotide-excision repair complex localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1508678" target=_blank>Bang DD, <i>et al.</i> (1992 Aug 11)</a>. Molecular cloning of RAD16, a gene involved in differential repair in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1626431" target=_blank>Mannhaupt G, <i>et al.</i> (1992 May)</a>. Molecular analysis of yeast chromosome II between CMD1 and LYS2: the excision repair gene RAD16 located in this region belongs to a novel group of double-finger proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1626430" target=_blank>Schild D, <i>et al.</i> (1992 May)</a>. Identification of RAD16, a yeast excision repair gene homologous to the recombinational repair gene RAD54 and to the SNF2 gene involved in transcriptional activation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601031" target=_blank>Reed SH, <i>et al.</i> (1999 Dec 1)</a>. Yeast autonomously replicating sequence binding factor is involved in nucleotide excision repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15177043" target=_blank>Yu S, <i>et al.</i> (2004 Mar 4)</a>. The yeast Rad7/Rad16/Abf1 complex generates superhelical torsion in DNA that is required for nucleotide excision repair.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04772_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04772_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04772_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04772_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01708_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG04773<br>EOG8K0P3C<br><br>sce:RAD5<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>DNA helicase/Ubiquitin ligase; involved in error-free branch of DNA damage tolerance (DDT) pathway; proposed to promote replication fork regression during postreplication repair by template switching; stimulates synthesis of free and PCNA-bound polyubiquitin chains by Ubc13p-Mms2p; required for error-prone translesion synthesis; forms nuclear foci upon DNA replication stress; associates with native telomeres, cooperates with homologous recombination in senescent cells<br><br>
			<br><b>PomBase Description</b><br>ubiquitin-protein ligase E3 Rad8<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in nucleotide-excision repair, postreplication repair and nucleus, site of double-strand break localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1394508" target=_blank>Ahne F, <i>et al.</i> (1992 Oct)</a>. The REV2 gene of Saccharomyces cerevisiae: cloning and DNA sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1324406" target=_blank>Johnson RE, <i>et al.</i> (1992 Sep)</a>. Saccharomyces cerevisiae RAD5-encoded DNA repair protein contains DNA helicase and zinc-binding sequence motifs and affects the stability of simple repetitive sequences in the genome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961763" target=_blank>Johnson RE, <i>et al.</i> (1994 Nov 11)</a>. Yeast DNA repair protein RAD5 that promotes instability of simple repetitive sequences is a DNA-dependent ATPase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016623" target=_blank>Ahne F, <i>et al.</i> (1997 Feb 15)</a>. The RAD5 gene product is involved in the avoidance of non-homologous end-joining of DNA double strand breaks in the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10924462" target=_blank>Xiao W, <i>et al.</i> (2000 Aug)</a>. The Saccharomyces cerevisiae RAD6 group is composed of an error-prone and two error-free postreplication repair pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10880451" target=_blank>Ulrich HD, <i>et al.</i> (2000 Jul 3)</a>. Two RING finger proteins mediate cooperation between ubiquitin-conjugating enzymes in DNA repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11884624" target=_blank>Torres-Ramos CA, <i>et al.</i> (2002 Apr)</a>. Requirement of RAD5 and MMS2 for postreplication repair of UV-damaged DNA in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12226657" target=_blank>Hoege C, <i>et al.</i> (2002 Sep 12)</a>. RAD6-dependent DNA repair is linked to modification of PCNA by ubiquitin and SUMO.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12496280" target=_blank>Ulrich HD, <i>et al.</i> (2003 Feb 28)</a>. Protein-protein interactions within an E2-RING finger complex. Implications for ubiquitin-dependent DNA damage repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14998526" target=_blank>Goldman GH, <i>et al.</i> (2004 Apr)</a>. Aspergillus nidulans as a model system to characterize the DNA damage response in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16224103" target=_blank>Chen S, <i>et al.</i> (2005)</a>. The RING finger ATPase Rad5p of Saccharomyces cerevisiae contributes to DNA double-strand break repair in a ubiquitin-independent manner.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04773_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04773_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04773_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04773_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01708_phobius.pdf">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG04774<br>EOG8K0P3C<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, DNA binding, helicase activity, hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides and nucleic acid binding, more<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04774_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04774_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04774_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04774_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01708_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01708.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>