<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>																	<a href="HOG00498.html">HOG00498&emsp;&emsp;</a><br> 																	HOG00499&emsp;&emsp;asparaginase 1 family<br> 																	<a href="HOG00500.html">HOG00500&emsp;&emsp;</a><br><br> 																	<a href="../sequences/hog/HOG00499.fasta">FASTA</a><br> 																	<a href="../trees/by_hog/msa/HOG00499_msa.fasta">MAFFT sequence alignment<br> 																	<a href="../trees/by_hog/phyml/HOG00499_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 																	<a href="../trees/by_hog/gb/HOG00499_msa.fasta-gb.htm">Gblocks</a><br> 																	<a href="HOG00499_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01142<br>EOG873N8P<br><br>sce:ASP3-1;ASP3-2;ASP3-3;ASP3-4;ASP1<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Cell-wall L-asparaginase II involved in asparagine catabolism; expression induced during nitrogen starvation; ORF contains a short non-coding RNA that enhances expression of full-length gene; likely arose in via horizontal gene transfer from the wine yeast Wickerhamomyces anomalus or a close relative; reference strain S288C has four copies of ASP3; ASP3-1 has a paralog, ASP3-3, that arose from a segmental duplication|Cell-wall L-asparaginase II involved in asparagine catabolism; expression induced during nitrogen starvation; ORF contains a short non-coding RNA that enhances expression of full-length gene; likely arose in via horizontal gene transfer from the wine yeast Wickerhamomyces anomalus or a close relative; reference strain S288C has four copies of ASP3; ASP3-2 has a paralog, ASP3-4, that arose from a segmental duplication|Cell-wall L-asparaginase II involved in asparagine catabolism; expression induced during nitrogen starvation; ORF contains a short non-coding RNA that enhances expression of full-length gene; likely arose in via horizontal gene transfer from the wine yeast Wickerhamomyces anomalus or a close relative; reference strain S288C has four copies of ASP3; ASP3-3 has a paralog, ASP3-1, that arose from a segmental duplication|Cell-wall L-asparaginase II involved in asparagine catabolism; expression induced during nitrogen starvation; ORF contains a short non-coding RNA that enhances expression of full-length gene; likely arose in via horizontal gene transfer from the wine yeast Wickerhamomyces anomalus or a close relative; reference strain S288C has four copies of ASP3; ASP3-4 has a paralog, ASP3-2, that arose from a segmental duplication|Cytosolic L-asparaginase, involved in asparagine catabolism; catalyzes hydrolysis of L-asparagine to aspartic acid and ammonia, has an important role in therapy of acute lymphoblastic leukemia; synthesized constitutively<br><br>
			<br><b>PomBase Description</b><br>L-asparaginase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Asparaginase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=342521" target=_blank>Dunlop PC, <i>et al.</i> (1978 Feb 25)</a>. Characterization of two forms of asparaginase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6986375" target=_blank>Dunlop PC, <i>et al.</i> (1980 Feb 25)</a>. Reactions of asparaginase II of Saccharomyces cerevisiae. A mechanistic analysis of hydrolysis and hydroxylaminolysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6995441" target=_blank>Dunlop PC, <i>et al.</i> (1980 Jul)</a>. Nitrogen catabolite repression of asparaginase II in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042786" target=_blank>Kim KW, <i>et al.</i> (1988 Aug 25)</a>. Asparaginase II of Saccharomyces cerevisiae. Characterization of the ASP3 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026756" target=_blank>Sinclair K, <i>et al.</i> (1994 Jun 24)</a>. The ASP1 gene of Saccharomyces cerevisiae, encoding the intracellular isozyme of L-asparaginase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748633" target=_blank>Kellis M, <i>et al.</i> (2003 May 15)</a>. Sequencing and comparison of yeast species to identify genes and regulatory elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15456898" target=_blank>Martinez MJ, <i>et al.</i> (2004 Dec)</a>. Genomic analysis of stationary-phase and exit in Saccharomyces cerevisiae: gene expression and identification of novel essential genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22840777" target=_blank>Kawashima SA, <i>et al.</i> (2012 Jul 27)</a>. Analyzing fission yeast multidrug resistance mechanisms to develop a genetically tractable model system for chemical biology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01142_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01142_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01142_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01142_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01142">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01143<br>EOG8JQ2JH<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01143_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01143_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01143_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01143_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01143">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG01144<br>EOG84MW7X<br>EOG8DFN4F<br>EOG8Z8WCJ<br><br>sce:TY1A-A;TY1A-BL;TY1B-BL;TY1A-BR;TY1B-BR;TY2A-B;TY2A-C;TY1A-DR1;TY1B-DR1;TY1A-DR3;TY1B-DR3;TY1A-DR4;TY1B-DR4;TY1A-DR5;TY1B-DR5;TY1A-DR6;TY1B-DR6;TY1A-DR2;TY2A-DR1;TY2A-DR2;TY2A-DR3;TY1A-ER1;TY1B-ER1;TY1A-ER2;TY1B-ER2;TY2A-F;TY1A-GR1;TY1B-GR1;TY1A-GR2;TY1B-GR2;TY1A-GR3;TY1B-GR3;TY2A-GR1;TY2A-GR2;TY1A-H;TY1B-H;TY1A-JR1;TY1B-JR1;TY1A-JR2;TY1B-JR2;TY1A-LR1;TY1B-LR1;TY1A-LR2;TY1B-LR2;TY1A-LR3;TY1B-LR3;TY1A-LR4;TY1B-LR4;TY2A-LR1;TY2A-LR2;TY1A-ML1;TY1B-ML1;TY1A-ML2;TY1B-ML2;TY1A-MR1;TY1B-MR1;TY1A-MR2;TY1B-MR2;TY1A-NL1;TY1B-NL1;TY1A-NL2;TY1B-NL2;TY1A-OL;TY1B-OL;TY1A-OR;TY1B-OR;TY2A-OR1;TY2A-OR2;TY1A-PL;TY1B-PL;TY1A-PR1;TY1B-PR1;TY1A-PR2;TY1B-PR2;TY1A-PR3;TY1B-PR3;TY1B-A;TY2B-B;TY2B-C;TY2B-DR1;TY2B-DR2;TY2B-DR3;TY2B-F;TY2B-GR1;TY2B-GR2;TY2B-LR1;TY2B-LR2;TY2B-OR1;TY2B-OR2<br><br>Genes: 94
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Retrotransposon TYA Gag gene co-transcribed with TYB Pol; Gag processing produces capsid proteins; in YARCTy1-1 TYB is mutant and probably non-functional|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; Gag processing produces capsid proteins; in YDRCTy1-3 TYB is mutant and probably non-functional|Retrotransposon TYA Gag and TYB Pol genes; Gag processing produces capsid proteins, Pol is cleaved to produce protease, reverse transcriptase and integrase activities; in YDRCTy1-3 TYB is mutant and probably non-functional|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag; YDR170W-A is part of a mutant retrotransposon; distribution in the cytoplasm becomes irregular rather than punctate upon DNA replication stress|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes||Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes; YHR214C-B has a paralog, YAR070C, that arose from a segmental duplication|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes||Retrotransposon TYA Gag and TYB Pol genes; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); YLR035C-A is part of a mutant retrotransposon|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag||Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag||Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|TyB Gag-Pol protein; proteolytically processed to make the Gag, reverse transcriptase (RT), protease (PR), and integrase (IN) proteins that are required for retrotransposition|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag and TYB Pol genes; Gag processing produces capsid proteins, Pol is cleaved to produce protease, reverse transcriptase and integrase activities; in YARCTy1-1 TYB is mutant and probably non-functional; protein product forms cytoplasmic foci upon DNA replication stress|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes||Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes||Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6322112" target=_blank>Bowen BA, <i>et al.</i> (1984 Feb 10)</a>. Expression of Ty-lacZ fusions in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989787" target=_blank>Fulton AM, <i>et al.</i> (1985 Jun 11)</a>. Variants within the yeast Ty sequence family encode a class of structurally conserved proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997719" target=_blank>Warmington JR, <i>et al.</i> (1985 Sep 25)</a>. Nucleotide sequence characterization of Ty 1-17, a class II transposon from yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2825192" target=_blank>Liao XB, <i>et al.</i> (1987 Dec)</a>. The upstream activation site of a Ty2 element of yeast is necessary but not sufficient to promote maximal transcription of the element.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3547324" target=_blank>Felici F, <i>et al.</i> (1987 Jan 12)</a>. Structure of the Saccharomyces cerevisiae gene encoding tRNAIle (IAU).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2837641" target=_blank>Boeke JD, <i>et al.</i> (1988 Apr)</a>. The Saccharomyces cerevisiae genome contains functional and nonfunctional copies of transposon Ty1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7941740" target=_blank>Clark MW, <i>et al.</i> (1994 Apr)</a>. Sequencing of chromosome I of Saccharomyces cerevisiae: analysis of the 42 kbp SPO7-CENI-CDC15 region.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582191" target=_blank>Kim JM, <i>et al.</i> (1998 May)</a>. Transposable elements and genome organization: a comprehensive survey of retrotransposons revealed by the complete Saccharomyces cerevisiae genome sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11884596" target=_blank>Morillon A, <i>et al.</i> (2002 Apr)</a>. Differential effects of chromatin and Gcn4 on the 50-fold range of expression among individual yeast Ty1 retrotransposons.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16093660" target=_blank>Lesage P, <i>et al.</i> (2005)</a>. Happy together: the life and times of Ty retrotransposons and their hosts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16093680" target=_blank>Wilhelm FX, <i>et al.</i> (2005)</a>. Reverse transcriptase and integrase of the Saccharomyces cerevisiae Ty1 element.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01144_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01144_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01144_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01144_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01144">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG01145<br>EOG88933K<br>EOG8NS1TQ<br>EOG8Z8WCJ<br><br>sce:;TY4A-H;TY4B-H;TY4A-J;TY4B-J<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>TyA Gag protein; the main structural constituent of virus-like particles (VLPs); Gag processing produces capsid proteins|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes|Retrotransposon TYA Gag gene co-transcribed with TYB Pol; translated as TYA or TYA-TYB polyprotein; Gag is a nucleocapsid protein that is the structural constituent of virus-like particles (VLPs); similar to retroviral Gag|Retrotransposon TYA Gag and TYB Pol genes; transcribed/translated as one unit; polyprotein is processed to make a nucleocapsid-like protein (Gag), reverse transcriptase (RT), protease (PR), and integrase (IN); similar to retroviral genes<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328182" target=_blank>Janetzky B, <i>et al.</i> (1992 Oct 5)</a>. Ty4, a new retrotransposon from Saccharomyces cerevisiae, flanked by tau-elements.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582191" target=_blank>Kim JM, <i>et al.</i> (1998 May)</a>. Transposable elements and genome organization: a comprehensive survey of retrotransposons revealed by the complete Saccharomyces cerevisiae genome sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16093660" target=_blank>Lesage P, <i>et al.</i> (2005)</a>. Happy together: the life and times of Ty retrotransposons and their hosts.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01145_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01145_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01145_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01145_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01145">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG01146<br>EOG88933K<br>EOG8JWT4K<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01146_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01146_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01146_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01146_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01146">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG01147<br>EOG8254BZ<br>EOG8B5MW6<br>EOG8GB5R0<br>EOG8Z8WCJ<br><br>sce:TY5A<br><br>Genes: 41
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24279325" target=_blank>Ravin NV, <i>et al.</i> (2013 Nov 27)</a>. Genome sequence and analysis of methylotrophic yeast Hansenula polymorpha DL1.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01147_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01147_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01147_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01147_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01147">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG01148<br>EOG8STQPC<br><br>sce:PRB1<br><br>Genes: 37
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Vacuolar proteinase B (yscB) with H3 N-terminal endopeptidase activity; serine protease of the subtilisin family; involved in protein degradation in the vacuole and required for full protein degradation during sporulation; activity inhibited by Pbi2p; protein abundance increases in response to DNA replication stress; PRB1 has a paralog, YSP3, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>vacuolar serine protease Isp6|vacuolar serine protease Psp3 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative vacuolar serine proteinase; putative signal sequence for transport into the endoplasmic reticulum; secreted protein<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021321" target=_blank>Zubenko GS, <i>et al.</i> (1981 Jan)</a>. Protein degradation, meiosis and sporulation in proteinase-deficient mutants of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6764902" target=_blank>Zubenko GS, <i>et al.</i> (1982 Dec)</a>. Genetic properties of mutations at the PEP4 locus in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3325823" target=_blank>Moehle CM, <i>et al.</i> (1987 Dec)</a>. Protease B of the lysosomelike vacuole of the yeast Saccharomyces cerevisiae is homologous to the subtilisin family of serine proteases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2645294" target=_blank>Moehle CM, <i>et al.</i> (1989 Feb)</a>. Processing pathway for protease B of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407604" target=_blank>Moehle CM, <i>et al.</i> (1990 Jan)</a>. Consequences of growth media, gene copy number, and regulatory mutations on the expression of the PRB1 gene of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1744078" target=_blank>Nebes VL, <i>et al.</i> (1991 Dec 5)</a>. Activation of the proteinase B precursor of the yeast Saccharomyces cerevisiae by autocatalysis and by an internal sequence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923060" target=_blank>Roberts BT, <i>et al.</i> (2003 Sep 1)</a>. Heritable activity: a prion that propagates by covalent autoactivation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16336669" target=_blank>Fedorova ND, <i>et al.</i> (2005 Dec 8)</a>. Comparative analysis of programmed cell death pathways in filamentous fungi.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01148_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01148_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01148_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01148_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01148">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG01149<br>EOG8STQPC<br><br>sce:RRT12<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Probable subtilisin-family protease; role in formation of the dityrosine layer of spore walls; localizes to the spore wall and also the nuclear envelope and ER region in mature spores<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19756047" target=_blank>Kung LA, <i>et al.</i> (2009)</a>. Global analysis of the glycoproteome in Saccharomyces cerevisiae reveals new roles for protein glycosylation in eukaryotes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19270272" target=_blank>Hontz RD, <i>et al.</i> (2009 May)</a>. Genetic identification of factors that modulate ribosomal DNA transcription in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19779569" target=_blank>Suda Y, <i>et al.</i> (2009 Sep 25)</a>. A screen for spore wall permeability mutants identifies a secreted protease required for proper spore wall assembly.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01149_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01149_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01149_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01149_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01149">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG01150<br>EOG8M908T<br><br>sce:BDF1<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein involved in transcription initiation; functions at TATA-containing promoters; associates with the basal transcription factor TFIID; contains two bromodomains; corresponds to the C-terminal region of mammalian TAF1; redundant with Bdf2p; BDF1 has a paralog, BDF2, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>BET family double bromodomain protein Bdf2|Swr1 complex bromodomain subunit Brf1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have TFIID-class transcription factor binding, core promoter binding, lysine-acetylated histone binding, nucleosomal histone binding activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8321235" target=_blank>Widner WR, <i>et al.</i> (1993 Jul)</a>. Evidence that the SKI antiviral system of Saccharomyces cerevisiae acts by blocking expression of viral mRNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7816623" target=_blank>Lygerou Z, <i>et al.</i> (1994 Dec 11)</a>. The yeast BDF1 gene encodes a transcription factor involved in the expression of a broad class of genes including snRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791775" target=_blank>Chua P, <i>et al.</i> (1995 Jul)</a>. Bdf1, a yeast chromosomal protein required for sporulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175470" target=_blank>Jeanmougin F, <i>et al.</i> (1997 May)</a>. The bromodomain revisited.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10783167" target=_blank>Matangkasombut O, <i>et al.</i> (2000 Apr 15)</a>. Bromodomain factor 1 corresponds to a missing piece of yeast TFIID.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11343701" target=_blank>Pamblanco M, <i>et al.</i> (2001 May 4)</a>. Bromodomain factor 1 (Bdf1) protein interacts with histones.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12482937" target=_blank>Chang M, <i>et al.</i> (2002 Dec 24)</a>. A genome-wide screen for methyl methanesulfonate-sensitive mutants reveals genes required for S phase progression in the presence of DNA damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12455696" target=_blank>Bensen ES, <i>et al.</i> (2002 Oct)</a>. A forkhead transcription factor is important for true hyphal as well as yeast morphogenesis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620225" target=_blank>Ladurner AG, <i>et al.</i> (2003 Feb)</a>. Bromodomains mediate an acetyl-histone encoded antisilencing function at heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620224" target=_blank>Matangkasombut O, <i>et al.</i> (2003 Feb)</a>. Different sensitivities of bromodomain factors 1 and 2 to histone H4 acetylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255905" target=_blank>Bianchi MM, <i>et al.</i> (2004 Aug)</a>. The bromodomain-containing protein Bdf1p acts as a phenotypic and transcriptional multicopy suppressor of YAF9 deletion in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225549" target=_blank>Martinez-Campa C, <i>et al.</i> (2004 Jul 2)</a>. Precise nucleosome positioning and the TATA box dictate requirements for the histone H4 tail and the bromodomain factor Bdf1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15143168" target=_blank>Sawa C, <i>et al.</i> (2004 Jun)</a>. Bromodomain factor 1 (Bdf1) is phosphorylated by protein kinase CK2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15796975" target=_blank>Marchais V, <i>et al.</i> (2005 Apr 1)</a>. DNA array analysis of Candida albicans gene expression in response to adherence to polystyrene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16039996" target=_blank>Singh V, <i>et al.</i> (2005 Sep 9)</a>. Global analysis of altered gene expression during morphogenesis of Candida albicans in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164403" target=_blank>Fernández-Arenas E, <i>et al.</i> (2007 Mar)</a>. Integrated proteomics and genomics strategies bring new insight into Candida albicans response upon macrophage interaction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01150_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01150_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01150_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01150_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01150">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG01151<br>EOG8M908T<br><br>sce:CHS5<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Component of the exomer complex; exomer also contains Csh6p, Bch1p, Bch2p, and Bud7p and is involved in export of selected proteins, such as chitin synthase Chs3p, from the Golgi to the plasma membrane; Chs5p is the only protein with a BRCT domain that is not localized to the nucleus<br><br>
			<br><b>PomBase Description</b><br>Chs five related protein Cfr1<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN8710, A. fumigatus Af293 : Afu6g02510, A. oryzae RIB40 : AO090120000136, Aspergillus wentii : Aspwe1_0103156 and Aspergillus sydowii : Aspsy1_0392745<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8197125" target=_blank>Choi WJ, <i>et al.</i> (1994 May 24)</a>. Chitin synthase 3 from yeast has zymogenic properties that depend on both the CAL1 and the CAL3 genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8994808" target=_blank>Bateman A, <i>et al.</i> (1996 Dec 1)</a>. Fibronectin type III domains in yeast detected by a hidden Markov model.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9008706" target=_blank>Santos B, <i>et al.</i> (1997 Jan 13)</a>. Targeting of chitin synthase 3 to polarized growth sites in yeast requires Chs5p and Myo2p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9111317" target=_blank>Santos B, <i>et al.</i> (1997 May)</a>. CHS5, a gene involved in chitin synthesis and mating in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12912901" target=_blank>Santos B, <i>et al.</i> (2003 Aug)</a>. Specific protein targeting during cell differentiation: polarized localization of Fus1p during mating depends on Chs5p in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15715908" target=_blank>Lesage G, <i>et al.</i> (2005 Feb 16)</a>. An interactional network of genes involved in chitin synthesis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16818716" target=_blank>Lam KK, <i>et al.</i> (2006 Jul 3)</a>. Palmitoylation by the DHHC protein Pfa4 regulates the ER exit of Chs3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16498409" target=_blank>Trautwein M, <i>et al.</i> (2006 Mar 8)</a>. Arf1p, Chs5p and the ChAPs are required for export of specialized cargo from the Golgi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16855022" target=_blank>Sanchatjate S, <i>et al.</i> (2006 Oct)</a>. Chs5/6 complex: a multiprotein complex that interacts with and conveys chitin synthase III from the trans-Golgi network to the cell surface.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17000877" target=_blank>Wang CW, <i>et al.</i> (2006 Sep 25)</a>. Exomer: A coat complex for transport of select membrane proteins from the trans-Golgi network to the plasma membrane in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19585695" target=_blank>de Groot PW, <i>et al.</i> (2009 Mar)</a>. Comprehensive genomic analysis of cell wall genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23154970" target=_blank>Szilágyi M, <i>et al.</i> (2013 Jan)</a>. Transcriptome changes initiated by carbon starvation in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01151_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01151_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01151_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01151_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01151">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG01152<br>EOG8M908T<br><br>sce:TAF2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>TFIID subunit (150 kDa); involved in RNA polymerase II transcription initiation<br><br>
			<br><b>PomBase Description</b><br>TATA-binding protein associated factor Taf2 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cell division site, cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840512" target=_blank>Ray BL, <i>et al.</i> (1991 Jul)</a>. The TSM1 gene of Saccharomyces cerevisiae overlaps the MAT locus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7667272" target=_blank>Poon D, <i>et al.</i> (1995 Aug 29)</a>. Identification and characterization of a TFIID-like multiprotein complex from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10788514" target=_blank>Sanders SL, <i>et al.</i> (2000 May 5)</a>. Identification of two novel TAF subunits of the yeast Saccharomyces cerevisiae TFIID complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12138208" target=_blank>Sanders SL, <i>et al.</i> (2002 Aug)</a>. Molecular characterization of Saccharomyces cerevisiae TFIID.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12516863" target=_blank>Martinez E, <i>et al.</i> (2002 Dec)</a>. Multi-protein complexes in eukaryotic gene transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093743" target=_blank>Leurent C, <i>et al.</i> (2002 Jul 1)</a>. Mapping histone fold TAFs within yeast TFIID.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16372000" target=_blank>Galagan JE, <i>et al.</i> (2005 Dec 22)</a>. Sequencing of Aspergillus nidulans and comparative analysis with A. fumigatus and A. oryzae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01152_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01152_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01152_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01152_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01152">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG01153<br>EOG8QZ646<br><br>sce:YTM1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Constituent of 66S pre-ribosomal particles; forms a complex with Nop7p and Erb1p that is required for maturation of the large ribosomal subunit; has seven C-terminal WD repeats<br><br>
			<br><b>PomBase Description</b><br>ribosome biogenesis protein Ytm1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in cellular response to biotic stimulus, cellular response to starvation and chromosome organization, more<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8896271" target=_blank>Chéret G, <i>et al.</i> (1996 Sep)</a>. DNA sequence analysis of the VPH1-SNF2 region on chromosome XV of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12110181" target=_blank>Du YC, <i>et al.</i> (2002 Jun 28)</a>. Yph1p, an ORC-interacting protein: potential links between cell proliferation control, DNA replication, and ribosome biogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16287855" target=_blank>Miles TD, <i>et al.</i> (2005 Dec)</a>. Ytm1, Nop7, and Erb1 form a complex necessary for maturation of yeast 66S preribosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18448671" target=_blank>Tang L, <i>et al.</i> (2008 Jul)</a>. Interactions among Ytm1, Erb1, and Nop7 required for assembly of the Nop7-subcomplex in yeast preribosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20542003" target=_blank>Bassler J, <i>et al.</i> (2010 Jun 11)</a>. The AAA-ATPase Rea1 drives removal of biogenesis factors during multiple stages of 60S ribosome assembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22146723" target=_blank>Ikebe C, <i>et al.</i> (2011)</a>. Systematic localization study on novel proteins encoded by meiotically up-regulated ORFs in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26657628" target=_blank>Thoms M, <i>et al.</i> (2016 Jan 29)</a>. Concerted removal of the Erb1-Ytm1 complex in ribosome biogenesis relies on an elaborate interface.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26601951" target=_blank>Romes EM, <i>et al.</i> (2016 Jan 8)</a>. The Crystal Structure of the Ubiquitin-like Domain of Ribosome Assembly Factor Ytm1 and Characterization of Its Interaction with the AAA-ATPase Midasin.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01153_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01153_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01153_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01153_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01153">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG01154<br>EOG82JM6N<br>EOG83R241<br>EOG87D863<br>EOG87WM7B<br>EOG8G4F55<br>EOG8M908T<br>EOG8QNKDJ<br>EOG8TQK13<br>EOG8W0W1W<br>EOG8WSTRB<br><br>sce:absent<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01154_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01154_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01154_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01154_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01154">Phobius transmembrane predictions</a>
										<br>12 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG01155<br>EOG8W6MB2<br><br>sce:EFR3<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein required for Stt4-containing PI kinase complex localization; required for Stt4-containing phosphoinositide (PI) kinase patch assembly at the plasma membrane; recruited to the plasma membrane via a conserved basic patch near its N-terminus; exhibits synthetic lethal genetic interactions with PHO85; has sequence similarity to the Drosophila rolling blackout (RBO) gene<br><br>
			<br><b>PomBase Description</b><br>HEAT repeat protein, involved in protein localization to plasma membrane (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN2496, A. fumigatus Af293 : Afu3g14050, A. oryzae RIB40 : AO090012000722, Aspergillus wentii : Aspwe1_0408944 and Aspergillus sydowii : Aspsy1_0148044<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17761666" target=_blank>Reinders J, <i>et al.</i> (2007 Nov)</a>. Profiling phosphoproteins of yeast mitochondria reveals a role of phosphorylation in assembly of the ATP synthase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25552606" target=_blank>Chen JS, <i>et al.</i> (2014 Dec 31)</a>. Identification of new players in cell division, DNA damage response, and morphogenesis through construction of Schizosaccharomyces pombe deletion strains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28784611" target=_blank>Snider CE, <i>et al.</i> (2017 Oct 2)</a>. Phosphoinositide-mediated ring anchoring resists perpendicular forces to promote medial cytokinesis.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01155_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01155_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01155_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01155_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01155">Phobius transmembrane predictions</a>
										<br>21 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG01156<br>EOG8Q2BWF<br><br>sce:APE3<br><br>Genes: 30
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Vacuolar aminopeptidase Y; processed to mature form by Prb1p<br><br>
			<br><b>AspGD Description</b><br>Vacuolar aminopeptidase Y family secreted protease; expression repressed by tunicamycin and DTT<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175800" target=_blank>Nishizawa M, <i>et al.</i> (1994 May 6)</a>. Molecular cloning of the aminopeptidase Y gene of Saccharomyces cerevisiae. Sequence analysis and gene disruption of a new aminopeptidase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175799" target=_blank>Yasuhara T, <i>et al.</i> (1994 May 6)</a>. Aminopeptidase Y, a new aminopeptidase from Saccharomyces cerevisiae. Purification, properties, localization, and processing by protease B.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20969648" target=_blank>Schinko T, <i>et al.</i> (2010 Nov)</a>. Transcriptome analysis of nitrate assimilation in Aspergillus nidulans reveals connections to nitric oxide metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22835028" target=_blank>Saykhedkar S, <i>et al.</i> (2012 Jul 26)</a>. A time course analysis of the extracellular proteome of Aspergillus nidulans growing on sorghum stover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23472095" target=_blank>Colabardini AC, <i>et al.</i> (2013)</a>. Functional characterization of Aspergillus nidulans ypkA, a homologue of the mammalian kinase SGK.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23154970" target=_blank>Szilágyi M, <i>et al.</i> (2013 Jan)</a>. Transcriptome changes initiated by carbon starvation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25043916" target=_blank>Martins I, <i>et al.</i> (2014 Jul 21)</a>. Elucidating how the saprophytic fungus Aspergillus nidulans uses the plant polyester suberin as carbon source.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01156_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01156_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01156_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01156_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01156">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG01157<br>EOG82BVSV<br>EOG86Q5C8<br>EOG873N8P<br>EOG8KWH7N<br>EOG8M37QP<br>EOG8P8D0F<br>EOG8PK0SD<br>EOG8Q2BWF<br><br>sce:absent<br><br>Genes: 26
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01157_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01157_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01157_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01157_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01157">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG01158<br>EOG85TB5W<br>EOG869PGX<br>EOG8BZKJ3<br>EOG8CC2HV<br>EOG8FFBN0<br>EOG8H9W42<br>EOG8NCJV3<br>EOG8Q2BXK<br>EOG8QZ646<br>EOG8SF7PV<br><br>sce:absent<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10417199" target=_blank>Hoyer LL, <i>et al.</i> (1999 Aug)</a>. Detection of Als proteins on the cell wall of Candida albicans in murine tissues.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15128742" target=_blank>Sheppard DC, <i>et al.</i> (2004 Jul 16)</a>. Functional and structural diversity in the Als protein family of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17625934" target=_blank>Zhao X, <i>et al.</i> (2007 Dec)</a>. Analysis of ALS5 and ALS6 allelic variability in a geographically diverse collection of Candida albicans isolates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17510860" target=_blank>Klotz SA, <i>et al.</i> (2007 Jun)</a>. Candida albicans Als proteins mediate aggregation with bacteria and yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164403" target=_blank>Fernández-Arenas E, <i>et al.</i> (2007 Mar)</a>. Integrated proteomics and genomics strategies bring new insight into Candida albicans response upon macrophage interaction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20398368" target=_blank>Nailis H, <i>et al.</i> (2010 Apr 16)</a>. Real-time PCR expression profiling of genes encoding potential virulence factors in Candida albicans biofilms: identification of model-dependent and -independent gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22321066" target=_blank>Aoki W, <i>et al.</i> (2012 Jun)</a>. Profiling of adhesive properties of the agglutinin-like sequence (ALS) protein family, a virulent attribute of Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01158_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01158_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01158_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01158_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01158">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG01159<br>EOG8G4F55<br>EOG8JQ2F0<br>EOG8JQ2GC<br>EOG8M908T<br>EOG8NCJV3<br>EOG8NVX36<br>EOG8W6MD3<br><br>sce:FLO1;FLO5;FLO9<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Lectin-like protein involved in flocculation; cell wall protein that binds mannose chains on the surface of other cells, confers floc-forming ability that is chymotrypsin sensitive and heat resistant; FLO1 has a paralog, FLO5, that arose from a segmental duplication|Lectin-like cell wall protein (flocculin) involved in flocculation; binds mannose chains on the surface of other cells, confers floc-forming ability that is chymotrypsin resistant but heat labile; FLO5 has a paralog, FLO1, that arose from a segmental duplication|Lectin-like protein with similarity to Flo1p; thought to be expressed and involved in flocculation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8511970" target=_blank>Teunissen AW, <i>et al.</i> (1993 Apr)</a>. Sequence of the open reading frame of the FLO1 gene from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8203162" target=_blank>Watari J, <i>et al.</i> (1994 Feb)</a>. Molecular cloning and analysis of the yeast flocculation gene FLO1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7483845" target=_blank>Bidard F, <i>et al.</i> (1995 Jul)</a>. The Saccharomyces cerevisiae FLO1 flocculation gene encodes for a cell surface protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7668043" target=_blank>Teunissen AW, <i>et al.</i> (1995 Jun 30)</a>. Localization of the dominant flocculation genes FLO5 and FLO8 of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7502576" target=_blank>Teunissen AW, <i>et al.</i> (1995 Sep 15)</a>. Review: the dominant flocculation genes of Saccharomyces cerevisiae constitute a new subtelomeric gene family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8980634" target=_blank>van der Vaart JM, <i>et al.</i> (1996 Dec 6)</a>. The retention mechanism of cell wall proteins in Saccharomyces cerevisiae. Wall-bound Cwp2p is beta-1,6-glucosylated.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9244284" target=_blank>Bony M, <i>et al.</i> (1997 Aug)</a>. Localization and cell surface anchoring of the Saccharomyces cerevisiae flocculation protein Flo1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851992" target=_blank>Kobayashi O, <i>et al.</i> (1998 Dec)</a>. Region of FLO1 proteins responsible for sugar recognition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9483793" target=_blank>Bony M, <i>et al.</i> (1998 Jan 15)</a>. Distribution of the flocculation protein, flop, at the cell surface during yeast growth: the availability of flop determines the flocculation level.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11027318" target=_blank>Guo B, <i>et al.</i> (2000 Oct 24)</a>. A Saccharomyces gene family involved in invasive growth, cell-cell adhesion, and mating.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12698276" target=_blank>Verstrepen KJ, <i>et al.</i> (2003 May)</a>. Yeast flocculation: what brewers should know.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523127" target=_blank>Zhao X, <i>et al.</i> (2003 Oct)</a>. Allelic variation in the contiguous loci encoding Candida albicans ALS5, ALS1 and ALS9.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15128742" target=_blank>Sheppard DC, <i>et al.</i> (2004 Jul 16)</a>. Functional and structural diversity in the Als protein family of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15470092" target=_blank>Frieman MB, <i>et al.</i> (2004 Oct)</a>. Multiple sequence signals determine the distribution of glycosylphosphatidylinositol proteins between the plasma membrane and cell wall in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16086015" target=_blank>Verstrepen KJ, <i>et al.</i> (2005 Sep)</a>. Intragenic tandem repeats generate functional variability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487349" target=_blank>Cunha AF, <i>et al.</i> (2006 Mar)</a>. Control by sugar of Saccharomyces cerevisiae flocculation for industrial ethanol production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18001350" target=_blank>Fichtner L, <i>et al.</i> (2007 Dec)</a>. Differential Flo8p-dependent regulation of FLO1 and FLO11 for cell-cell and cell-substrate adherence of S. cerevisiae S288c.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600078" target=_blank>Zhao X, <i>et al.</i> (2007 Jul)</a>. Unequal contribution of ALS9 alleles to adhesion between Candida albicans and human vascular endothelial cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17510860" target=_blank>Klotz SA, <i>et al.</i> (2007 Jun)</a>. Candida albicans Als proteins mediate aggregation with bacteria and yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21896717" target=_blank>Salgado PS, <i>et al.</i> (2011 Sep 20)</a>. Structural basis for the broad specificity to host-cell ligands by the pathogenic fungus Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22321066" target=_blank>Aoki W, <i>et al.</i> (2012 Jun)</a>. Profiling of adhesive properties of the agglutinin-like sequence (ALS) protein family, a virulent attribute of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22429754" target=_blank>Monroy-Pérez E, <i>et al.</i> (2012 May)</a>. Frequency and expression of ALS and HWP1 genotypes in Candida albicans strains isolated from Mexican patients suffering from vaginal candidosis.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01159_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01159_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01159_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01159_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01159">Phobius transmembrane predictions</a>
										<br>9 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9e6c3" class="col">
			FOG01160<br>EOG86M94B<br>EOG8STQPC<br><br>sce:absent<br><br>Genes: 13
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01160_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01160_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01160_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01160_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01160">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dbc4e1" class="col">
			FOG01161<br>EOG8BZKJ3<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted choline dehydrogenase activity, flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in alcohol metabolic process, oxidation-reduction process|Has domain(s) with predicted choline dehydrogenase activity, flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in alcohol metabolic process, oxidation-reduction process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01161_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01161_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01161_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01161_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01161">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fad7ad" class="col">
			FOG01162<br>EOG8Q2BWF<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01162_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01162_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01162_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01162_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01162">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb9" class="col">
			FOG01163<br>EOG82V6ZX<br>EOG8S7H8Z<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have nitronate monooxygenase activity and role in denitrification pathway, detoxification of nitrogen compound|Predicted 2-nitropropane dioxygenase; induced by growth on starch and lactate<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01163_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01163_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01163_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01163_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01163">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dfc9ba" class="col">
			FOG01164<br>EOG8QZ646<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative centromere protein; expression enhanced by maltose<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01164_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01164_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01164_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01164_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01164">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8d4ea" class="col">
			FOG01165<br>EOG8Q2BWF<br>EOG8STQPC<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=870075" target=_blank>Ogrydziak DM, <i>et al.</i> (1977 Apr 27)</a>. Regulation of extracellular protease production in Candida lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7007321" target=_blank>Simms PC, <i>et al.</i> (1981 Jan)</a>. Structural gene for the alkaline extracellular protease of Saccharomycopsis lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6750031" target=_blank>Ogrydziak DM, <i>et al.</i> (1982 Jun)</a>. Alkaline extracellular protease produced by Saccharomycopsis lipolytica CX161-1B.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2443483" target=_blank>Davidow LS, <i>et al.</i> (1987 Oct)</a>. Cloning and sequencing of the alkaline extracellular protease gene of Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3211132" target=_blank>Matoba S, <i>et al.</i> (1988 Nov)</a>. Intracellular precursors and secretion of alkaline extracellular protease of Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649495" target=_blank>Matoba S, <i>et al.</i> (1989 Apr 15)</a>. A novel location for dipeptidyl aminopeptidase processing sites in the alkaline extracellular protease of Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1995632" target=_blank>Fabre E, <i>et al.</i> (1991 Feb 25)</a>. Role of the proregion in the production and secretion of the Yarrowia lipolytica alkaline extracellular protease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1634541" target=_blank>Fabre E, <i>et al.</i> (1992 Jul 25)</a>. Intracellular transit of a yeast protease is rescued by trans-complementation with its prodomain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8264600" target=_blank>Blanchin-Roland S, <i>et al.</i> (1994 Jan)</a>. Two upstream activation sequences control the expression of the XPR2 gene in the yeast Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8203153" target=_blank>Enderlin CS, <i>et al.</i> (1994 Jan)</a>. Cloning, nucleotide sequence and functions of XPR6, which codes for a dibasic processing endoprotease from the yeast Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7954894" target=_blank>Le Dall MT, <i>et al.</i> (1994 Jul)</a>. Multiple-copy integration in the yeast Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8842151" target=_blank>Otero RC, <i>et al.</i> (1996 Sep 13)</a>. Dominant mutations affecting expression of pH-regulated genes in Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8798620" target=_blank>Mamoun CB, <i>et al.</i> (1996 Sep 27)</a>. The TSR1 gene of Yarrowia lipolytica is involved in the signal recognition particle-dependent translocation pathway of secretory proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199331" target=_blank>Lambert M, <i>et al.</i> (1997 Jul)</a>. Genetic analysis of regulatory mutants affecting synthesis of extracellular proteinases in the yeast Yarrowia lipolytica: identification of a RIM101/pacC homolog.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9353927" target=_blank>Matoba S, <i>et al.</i> (1997 Oct)</a>. Dipeptidyl aminopeptidase processing and biosynthesis of alkaline extracellular protease from Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308186" target=_blank>Glover DJ, <i>et al.</i> (1997 Sep)</a>. pH-regulated expression of the acid and alkaline extracellular proteases of Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206713" target=_blank>Madzak C, <i>et al.</i> (1999 Jan)</a>. Functional analysis of upstream regulating regions from the Yarrowia lipolytica XPR2 promoter.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11861549" target=_blank>Gonzalez-Lopez CI, <i>et al.</i> (2002 Feb)</a>. Genetic control of extracellular protease synthesis in the yeast Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17432872" target=_blank>Poza M, <i>et al.</i> (2007 May 16)</a>. Cloning and expression of the XPR2 gene from Yarrowia lipolytica in Pichia pastoris.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22909173" target=_blank>Ogrydziak DM, <i>et al.</i> (2012 Dec)</a>. Characterization of Yarrowia lipolytica XPR2 multi-copy strains over-producing alkaline extracellular protease - a system for rapidly increasing secretory pathway cargo loads.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01165_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01165_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01165_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01165_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01165">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eeadad" class="col">
			FOG01166<br>EOG83XSQK<br>EOG8FTTFM<br>EOG8G4F55<br>EOG8JHB5Q<br>EOG8SXKZ1<br>EOG8W6MB2<br><br>sce:absent<br><br>Genes: 9
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01166_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01166_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01166_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01166_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01166">Phobius transmembrane predictions</a>
										<br>5 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bacde4" class="col">
			FOG01167<br>EOG8W0W07<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01167_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01167_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01167_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01167_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01167">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c1e3bb" class="col">
			FOG01168<br>EOG8JDFNS<br>EOG8Z8WCJ<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted nucleic acid binding, zinc ion binding activity and role in DNA integration|Has domain(s) with predicted nucleic acid binding, zinc ion binding activity and role in DNA integration|Has domain(s) with predicted nucleic acid binding activity and role in DNA integration|Has domain(s) with predicted nucleic acid binding, zinc ion binding activity and role in DNA integration|Has domain(s) with predicted nucleic acid binding, zinc ion binding activity and role in DNA integration|Has domain(s) with predicted nucleic acid binding, zinc ion binding activity and role in DNA integration|Has domain(s) with predicted nucleic acid binding, zinc ion binding activity and role in DNA integration<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01168_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01168_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01168_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01168_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01168">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7bcdd" class="col">
			FOG01169<br>EOG8H9W42<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of Aspergillus brasiliensis : Aspbr1_0139491|Ortholog of A. nidulans FGSC A4 : AN0602, A. oryzae RIB40 : AO090023000525, Aspergillus wentii : Aspwe1_0050808, Aspwe1_0063480 and Aspergillus sydowii : Aspsy1_0055411<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01169_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01169_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01169_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01169_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01169">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d2a2" class="col">
			FOG01170<br>EOG8G4F55<br>EOG8M908T<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9765564" target=_blank>Hoyer LL, <i>et al.</i> (1998 Oct)</a>. Identification of Candida albicans ALS2 and ALS4 and localization of als proteins to the fungal cell surface.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10417199" target=_blank>Hoyer LL, <i>et al.</i> (1999 Aug)</a>. Detection of Als proteins on the cell wall of Candida albicans in murine tissues.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256583" target=_blank>Zhao X, <i>et al.</i> (2004 Jul)</a>. ALS3 and ALS8 represent a single locus that encodes a Candida albicans adhesin; functional comparisons between Als3p and Als1p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15128742" target=_blank>Sheppard DC, <i>et al.</i> (2004 Jul 16)</a>. Functional and structural diversity in the Als protein family of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731087" target=_blank>Green CB, <i>et al.</i> (2005 Mar)</a>. Use of green fluorescent protein and reverse transcription-PCR to monitor Candida albicans agglutinin-like sequence gene expression in a murine model of disseminated candidiasis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870470" target=_blank>Zhao X, <i>et al.</i> (2005 May)</a>. Analysis of the Candida albicans Als2p and Als4p adhesins suggests the potential for compensatory function within the Als family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16839200" target=_blank>Nobile CJ, <i>et al.</i> (2006 Jul)</a>. Critical role of Bcr1-dependent adhesins in C. albicans biofilm formation in vitro and in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17510860" target=_blank>Klotz SA, <i>et al.</i> (2007 Jun)</a>. Candida albicans Als proteins mediate aggregation with bacteria and yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17766464" target=_blank>Bauer J, <i>et al.</i> (2007 Oct)</a>. Candida albicans Sfl1 suppresses flocculation and filamentation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18635358" target=_blank>Nobile CJ, <i>et al.</i> (2008 Jul 22)</a>. Complementary adhesin function in C. albicans biofilm formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18712765" target=_blank>Castillo L, <i>et al.</i> (2008 Sep)</a>. A study of the Candida albicans cell wall proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19291169" target=_blank>Beucher B, <i>et al.</i> (2009 Apr)</a>. Recognition of Candida albicans Als3 by the germ tube-specific monoclonal antibody 3D9.3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683080" target=_blank>Nailis H, <i>et al.</i> (2009 Jan)</a>. Monitoring ALS1 and ALS3 gene expression during in vitro Candida albicans biofilm formation under continuous flow conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20398368" target=_blank>Nailis H, <i>et al.</i> (2010 Apr 16)</a>. Real-time PCR expression profiling of genes encoding potential virulence factors in Candida albicans biofilms: identification of model-dependent and -independent gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21456055" target=_blank>Giacometti R, <i>et al.</i> (2011 Apr)</a>. Candida albicans Tpk1p and Tpk2p isoforms differentially regulate pseudohyphal development, biofilm structure, cell aggregation and adhesins expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20864472" target=_blank>Sosinska GJ, <i>et al.</i> (2011 Jan)</a>. Mass spectrometric quantification of the adaptations in the wall proteome of Candida albicans in response to ambient pH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22545115" target=_blank>Li Y, <i>et al.</i> (2012)</a>. In vivo inhibitory effect on the biofilm formation of Candida albicans by liverwort derived riccardin D.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22428031" target=_blank>Murciano C, <i>et al.</i> (2012)</a>. Evaluation of the role of Candida albicans agglutinin-like sequence (Als) proteins in human oral epithelial cell interactions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106872" target=_blank>Coleman DA, <i>et al.</i> (2012 Apr)</a>. A monoclonal antibody specific for Candida albicans Als4 demonstrates overlapping localization of Als family proteins on the fungal cell surface and highlights differences between Als localization in vitro and in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22544909" target=_blank>Fanning S, <i>et al.</i> (2012 Jul)</a>. Divergent targets of Candida albicans biofilm regulator Bcr1 in vitro and in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22321066" target=_blank>Aoki W, <i>et al.</i> (2012 Jun)</a>. Profiling of adhesive properties of the agglutinin-like sequence (ALS) protein family, a virulent attribute of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22429754" target=_blank>Monroy-Pérez E, <i>et al.</i> (2012 May)</a>. Frequency and expression of ALS and HWP1 genotypes in Candida albicans strains isolated from Mexican patients suffering from vaginal candidosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22882910" target=_blank>Fanning S, <i>et al.</i> (2012 Oct)</a>. Functional control of the Candida albicans cell wall by catalytic protein kinase A subunit Tpk1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23136884" target=_blank>Röhm M, <i>et al.</i> (2013 Jan)</a>. A family of secreted pathogenesis-related proteins in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630968" target=_blank>Fu Y, <i>et al.</i> (2013 Jul)</a>. Investigation of the function of Candida albicans Als3 by heterologous expression in Candida glabrata.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24152214" target=_blank>Alsteens D, <i>et al.</i> (2013 Nov 5)</a>. Quantifying the forces driving cell-cell adhesion in a fungal pathogen.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24466000" target=_blank>Delgado-Silva Y, <i>et al.</i> (2014)</a>. Participation of Candida albicans transcription factor RLM1 in cell wall biogenesis and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24796422" target=_blank>Feldman M, <i>et al.</i> (2014)</a>. Therapeutic potential of thiazolidinedione-8 as an antibiofilm agent against Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24514088" target=_blank>Li DD, <i>et al.</i> (2014)</a>. In vitro and in vivo activities of pterostilbene against Candida albicans biofilms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24736223" target=_blank>Dutton LC, <i>et al.</i> (2014 Apr 15)</a>. O-mannosylation in Candida albicans enables development of interkingdom biofilm communities.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24802757" target=_blank>Lin J, <i>et al.</i> (2014 Jun 27)</a>. The peptide-binding cavity is essential for Als3-mediated adhesion of Candida albicans to human cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24673895" target=_blank>Ding X, <i>et al.</i> (2014 Mar 28)</a>. Human serum inhibits adhesion and biofilm formation in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24623107" target=_blank>Rautela R, <i>et al.</i> (2014 May)</a>. Lipopeptides from Bacillus strain AR2 inhibits biofilm formation by Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01170_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01170_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01170_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01170_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01170">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb0" class="col">
			FOG01171<br>EOG80RXXC<br>EOG8905S6<br>EOG8D51CR<br>EOG8FTTFP<br>EOG8JDFNS<br>EOG8KD520<br>EOG8RJDHS<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function|Ortholog of Aspergillus tubingensis : Asptu1_0180055, Aspergillus kawachii : Aspka1_0180640, Aspergillus acidus : Aspfo1_0207656 and Aspergillus wentii : Aspwe1_0179513<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01171_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01171_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01171_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01171_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01171">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dac1b0" class="col">
			FOG01172<br>EOG82FQZG<br>EOG8M908T<br>EOG8Z8WCJ<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01172_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01172_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01172_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01172_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01172">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cee7" class="col">
			FOG01173<br>EOG8254CD<br>EOG8DNCKD<br>EOG8G4F55<br>EOG8JSXW5<br>EOG8M908T<br>EOG8WM3C4<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>cell agglutination protein Mam3|cell surface glycoprotein|cell surface glycoprotein (predicted)|cell surface glycoprotein (predicted), DIPSY family|cell surface glycoprotein, adhesion molecule (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8668157" target=_blank>Yabana N, <i>et al.</i> (1996 Jul)</a>. Schizosaccharomyces pombe map1+ encodes a MADS-box-family protein required for cell-type-specific gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12738880" target=_blank>Samuelsen CO, <i>et al.</i> (2003 May 27)</a>. TRAP230/ARC240 and TRAP240/ARC250 Mediator subunits are functionally conserved through evolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16079914" target=_blank>Liu X, <i>et al.</i> (2005 Aug 17)</a>. Molecular analysis of kinetochore architecture in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16615890" target=_blank>Chikashige Y, <i>et al.</i> (2006 Apr 7)</a>. Meiotic proteins bqt1 and bqt2 tether telomeres to form the bouquet arrangement of chromosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17032641" target=_blank>Mata J, <i>et al.</i> (2006 Oct 17)</a>. Global roles of Ste11p, cell type, and pheromone in the control of gene expression during early sexual differentiation in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17870620" target=_blank>Linder T, <i>et al.</i> (2008 Apr)</a>. Molecular phylogenetics of ascomycotal adhesins--a novel family of putative cell-surface adhesive proteins in fission yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20537132" target=_blank>Han TX, <i>et al.</i> (2010)</a>. Global fitness profiling of fission yeast deletion strains by barcode sequencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20625380" target=_blank>Liu NN, <i>et al.</i> (2010 Aug)</a>. A genome-wide screen for Schizosaccharomyces pombe deletion mutants that affect telomere length.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23236291" target=_blank>Kwon EJ, <i>et al.</i> (2012)</a>. Deciphering the transcriptional-regulatory network of flocculation in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22252817" target=_blank>Fang Y, <i>et al.</i> (2012 Apr)</a>. A genomewide screen in Schizosaccharomyces pombe for genes affecting the sensitivity of antifungal drugs that target ergosterol biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22907753" target=_blank>Wells ML, <i>et al.</i> (2012 Oct)</a>. Posttranscriptional regulation of cell-cell interaction protein-encoding transcripts by Zfs1p in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23297348" target=_blank>Chen JS, <i>et al.</i> (2013 May)</a>. Comprehensive proteomics analysis reveals new substrates and regulators of the fission yeast clp1/cdc14 phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24727291" target=_blank>Ucisik-Akkaya E, <i>et al.</i> (2014 Apr 11)</a>. A genome-wide screen for sporulation-defective mutants in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojardín L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27887640" target=_blank>Malecki M, <i>et al.</i> (2016 Nov 25)</a>. Functional and regulatory profiling of energy metabolism in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27558664" target=_blank>Guo L, <i>et al.</i> (2016 Oct 13)</a>. Global Fitness Profiling Identifies Arsenic and Cadmium Tolerance Mechanisms in Fission Yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886462" target=_blank>Su Y, <i>et al.</i> (2017 Jan)</a>. Loss of ppr3, ppr4, ppr6, or ppr10 perturbs iron homeostasis and leads to apoptotic cell death in Schizosaccharomyces pombe.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01173_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01173_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01173_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01173_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01173">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eca3a4" class="col">
			FOG01174<br>EOG84QRGQ<br>EOG8F4QW4<br>EOG8QFTVV<br>EOG8RN8ZP<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01174_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01174_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01174_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01174_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01174">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b2c7e1" class="col">
			FOG01175<br>EOG82V6ZX<br>EOG8KWH7N<br>EOG8QFTVV<br>EOG8QV9TQ<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01175_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01175_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01175_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01175_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01175">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bae0b3" class="col">
			FOG01176<br>EOG8CC2HV<br>EOG8FXPR5<br><br>sce:SAG1<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Alpha-agglutinin of alpha-cells; binds to Aga1p during agglutination, N-terminal half is homologous to the immunoglobulin superfamily and contains binding site for a-agglutinin, C-terminal half is highly glycosylated and contains GPI anchor<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677666" target=_blank>Lipke PN, <i>et al.</i> (1989 Aug)</a>. AG alpha 1 is the structural gene for the Saccharomyces cerevisiae alpha-agglutinin, a cell surface glycoprotein involved in cell-cell interactions during mating.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676603" target=_blank>Hauser K, <i>et al.</i> (1989 Sep 25)</a>. Purification of the inducible alpha-agglutinin of S. cerevisiae and molecular cloning of the gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455628" target=_blank>Wojciechowicz D, <i>et al.</i> (1993 Apr)</a>. Cell surface anchorage and ligand-binding domains of the Saccharomyces cerevisiae cell adhesion protein alpha-agglutinin, a member of the immunoglobulin superfamily.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8007981" target=_blank>Lu CF, <i>et al.</i> (1994 Jul)</a>. A pathway for cell wall anchorage of Saccharomyces cerevisiae alpha-agglutinin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7844147" target=_blank>Lu CF, <i>et al.</i> (1995 Feb)</a>. Glycosyl phosphatidylinositol-dependent cross-linking of alpha-agglutinin and beta 1,6-glucan in the Saccharomyces cerevisiae cell wall.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592821" target=_blank>Chen MH, <i>et al.</i> (1995 Nov 3)</a>. Structure of Saccharomyces cerevisiae alpha-agglutinin. Evidence for a yeast cell wall protein with multiple immunoglobulin-like domains with atypical disulfides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8741846" target=_blank>de Nobel H, <i>et al.</i> (1996 Jan)</a>. Identification of a ligand-binding site in an immunoglobulin fold domain of the Saccharomyces cerevisiae adhesion protein alpha-agglutinin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613572" target=_blank>Hamada K, <i>et al.</i> (1998 Apr)</a>. Screening for glycosylphosphatidylinositol (GPI)-dependent cell wall proteins in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292808" target=_blank>Zhao H, <i>et al.</i> (2001 May)</a>. Interaction of alpha-agglutinin and a-agglutinin, Saccharomyces cerevisiae sexual cell adhesion molecules.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01176_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01176_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01176_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01176_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01176">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d2b4d9" class="col">
			FOG01177<br>EOG8G4F55<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01177_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01177_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01177_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01177_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01177">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cc97" class="col">
			FOG01178<br>EOG8M908T<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01178_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01178_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01178_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01178_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01178">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcffa6" class="col">
			FOG01179<br>EOG8M37QP<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Beta-galactosidase|Beta-galactosidase|Beta-galactosidase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=393802" target=_blank>Bal J, <i>et al.</i> (1979 Dec)</a>. Allele specific and locus non-specific suppressors in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15848224" target=_blank>Storms R, <i>et al.</i> (2005 May)</a>. Plasmid vectors for protein production, gene expression and molecular manipulations in Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19618505" target=_blank>Coutinho PM, <i>et al.</i> (2009 Mar)</a>. Post-genomic insights into the plant polysaccharide degradation potential of  Aspergillus nidulans and comparison to Aspergillus niger and Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24037406" target=_blank>Dragosits M, <i>et al.</i> (2014 Apr)</a>. Recombinant Aspergillus β-galactosidases as a robust glycomic and biotechnological tool.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24690910" target=_blank>Jónás Á, <i>et al.</i> (2014 Jul)</a>. Extra- and intracellular lactose catabolism in Penicillium chrysogenum: phylogenetic and expression analysis of the putative permease and hydrolase genes.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01179_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01179_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01179_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01179_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01179">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d6baa7" class="col">
			FOG01180<br>EOG8CC2HV<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01180_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01180_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01180_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01180_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01180">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f7c8e5" class="col">
			FOG01181<br>EOG8M908T<br>EOG8NVX36<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01181_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01181_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01181_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01181_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01181">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ea999a" class="col">
			FOG01182<br>EOG8Q2BWF<br>EOG8V9S6M<br>EOG8W0W07<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01182_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01182_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01182_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01182_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01182">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#aac1de" class="col">
			FOG01183<br>EOG82V6ZX<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative enoyl-(acyl carrier protein) reductase; induced by fenpropimorph<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01183_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01183_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01183_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01183_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01183">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b3dcab" class="col">
			FOG01184<br>EOG8NS1TQ<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative arabinofuranosidase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16844780" target=_blank>Bauer S, <i>et al.</i> (2006 Jul 25)</a>. Development and application of a suite of polysaccharide-degrading enzymes for analyzing plant cell walls.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19618505" target=_blank>Coutinho PM, <i>et al.</i> (2009 Mar)</a>. Post-genomic insights into the plant polysaccharide degradation potential of  Aspergillus nidulans and comparison to Aspergillus niger and Aspergillus oryzae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01184_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01184_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01184_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01184_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01184">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cdacd5" class="col">
			FOG01185<br>EOG8STQPC<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01185_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01185_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01185_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01185_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01185">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8c78c" class="col">
			FOG01186<br>EOG8M908T<br><br>sce:SEG1<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Component of eisosome required for proper eisosome assembly; precedes Pil1p/Lsp1p during eisosome formation, controls eisosome length and shape; diffusely distributed, forms heterogeneous patches at plasma membrane in small buds, also found in medium and large buds; expression repressed by cAMP; similar to A. gossypii SEG gene and to S. pombe Sle1p, important for generating eisosomes; SEG1 has a paralog, SEG2, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11914276" target=_blank>Kumar A, <i>et al.</i> (2002 Mar 15)</a>. Subcellular localization of the yeast proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19269952" target=_blank>Deng C, <i>et al.</i> (2009 Jun)</a>. Unifying fluorescence microscopy and mass spectrometry for studying protein complexes in cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22368779" target=_blank>Douglas LM, <i>et al.</i> (2011 Dec 1)</a>. Membrane Compartment Occupied by Can1 (MCC) and Eisosome Subdomains of the Fungal Plasma Membrane.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21525038" target=_blank>Seger S, <i>et al.</i> (2011 May 15)</a>. Formation and stability of eisosomes in the filamentous fungus Ashbya gossypii.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22869600" target=_blank>Moreira KE, <i>et al.</i> (2012 Aug 6)</a>. Seg1 controls eisosome assembly and shape.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01186_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01186_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01186_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01186_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01186">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff9d" class="col">
			FOG01187<br>EOG8STQPC<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01187_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01187_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01187_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01187_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01187">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d2b39e" class="col">
			FOG01188<br>EOG8G4F55<br><br>sce:SPC105<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of a kinetochore-microtubule binding complex; complex bridges centromeric heterochromatin and kinetochore MAPs and motors; required for sister chromatid bi-orientation and kinetochore binding of SAC components; complex also includes Kre28p; modified by sumoylation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9585415" target=_blank>Wigge PA, <i>et al.</i> (1998 May 18)</a>. Analysis of the Saccharomyces spindle pole by matrix-assisted laser desorption/ionization (MALDI) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14565975" target=_blank>Nekrasov VS, <i>et al.</i> (2003 Dec)</a>. Interactions between centromere complexes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16715078" target=_blank>Joglekar AP, <i>et al.</i> (2006 Jun)</a>. Molecular architecture of a kinetochore-microtubule attachment site.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19893618" target=_blank>Pagliuca C, <i>et al.</i> (2009 Oct 28)</a>. Roles for the conserved spc105p/kre28p complex in kinetochore-microtubule binding and the spindle assembly checkpoint.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01188_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01188_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01188_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01188_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01188">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6c2e2" class="col">
			FOG01189<br>EOG8KPRTJ<br>EOG8SXKTQ<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in oxidation-reduction process|Ortholog(s) have cellobiose dehydrogenase (acceptor) activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01189_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01189_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01189_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01189_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01189">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e88f90" class="col">
			FOG01190<br>EOG8BZKJ3<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted choline dehydrogenase activity, flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in alcohol metabolic process, oxidation-reduction process|Has domain(s) with predicted flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in oxidation-reduction process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01190_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01190_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01190_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01190_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01190">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a2bbdb" class="col">
			FOG01191<br>EOG8M37QP<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Beta-galactosidase|Secreted, thermostable beta-galactosidase; XlnR regulated<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=393802" target=_blank>Bal J, <i>et al.</i> (1979 Dec)</a>. Allele specific and locus non-specific suppressors in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15848224" target=_blank>Storms R, <i>et al.</i> (2005 May)</a>. Plasmid vectors for protein production, gene expression and molecular manipulations in Aspergillus niger.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19618505" target=_blank>Coutinho PM, <i>et al.</i> (2009 Mar)</a>. Post-genomic insights into the plant polysaccharide degradation potential of  Aspergillus nidulans and comparison to Aspergillus niger and Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24037406" target=_blank>Dragosits M, <i>et al.</i> (2014 Apr)</a>. Recombinant Aspergillus β-galactosidases as a robust glycomic and biotechnological tool.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24690910" target=_blank>Jónás Á, <i>et al.</i> (2014 Jul)</a>. Extra- and intracellular lactose catabolism in Penicillium chrysogenum: phylogenetic and expression analysis of the putative permease and hydrolase genes.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01191_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01191_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01191_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01191_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01191">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#abd9a3" class="col">
			FOG01192<br>EOG8NVX36<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN4364, A. fumigatus Af293 : Afu4g06520, A. oryzae RIB40 : AO090023000934, Aspergillus wentii : Aspwe1_0252118 and Aspergillus sydowii : Aspsy1_0062282<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01192_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01192_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01192_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01192_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01192">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c8a4d1" class="col">
			FOG01193<br>EOG85QG0G<br>EOG8KWH7N<br>EOG8MPG8S<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12734798" target=_blank>Lee SA, <i>et al.</i> (2003 May)</a>. An analysis of the Candida albicans genome database for soluble secreted proteins using computer-based prediction algorithms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16950924" target=_blank>Srikantha T, <i>et al.</i> (2006 Oct)</a>. TOS9 regulates white-opaque switching in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17371861" target=_blank>Bates S, <i>et al.</i> (2007 Jun)</a>. Candida albicans Iff11, a secreted protein required for cell wall structure and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21841123" target=_blank>Boisramé A, <i>et al.</i> (2011 Oct)</a>. Unexpected role for a serine/threonine-rich domain in the Candida albicans Iff protein family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23490206" target=_blank>She X, <i>et al.</i> (2013 Sep)</a>. Cell surface changes in the Candida albicans mitochondrial mutant goa1Δ are associated with reduced recognition by innate immune cells.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01193_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01193_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01193_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01193_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01193">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f7c181" class="col">
			FOG01194<br>EOG82280N<br>EOG8JQ2F0<br>EOG8KPRTJ<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01194_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01194_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01194_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01194_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01194">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff93" class="col">
			FOG01195<br>EOG8JQ2F0<br><br>sce:FLO10;;;TDA8<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Member of the FLO family of cell wall flocculation proteins; not expressed in most lab strains; overproduction induces flocculation that can be inhibited by mannose, sucrose, or glucose; overproduction also promotes haploid invasive growth and diploid filamentous growth||Pseudogenic fragment with similarity to flocculins; YHR213W has a paralog, YAR062W, that arose from a segmental duplication|Putative protein of unknown function; null mutant is sensitive to expression of the top1-T722A allele; not an essential gene<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7502576" target=_blank>Teunissen AW, <i>et al.</i> (1995 Sep 15)</a>. Review: the dominant flocculation genes of Saccharomyces cerevisiae constitute a new subtelomeric gene family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11027318" target=_blank>Guo B, <i>et al.</i> (2000 Oct 24)</a>. A Saccharomyces gene family involved in invasive growth, cell-cell adhesion, and mating.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15016375" target=_blank>Halme A, <i>et al.</i> (2004 Feb 6)</a>. Genetic and epigenetic regulation of the FLO gene family generates cell-surface variation in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16086015" target=_blank>Verstrepen KJ, <i>et al.</i> (2005 Sep)</a>. Intragenic tandem repeats generate functional variability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21173034" target=_blank>Reid RJ, <i>et al.</i> (2011 Mar)</a>. Selective ploidy ablation, a high-throughput plasmid transfer protocol, identifies new genes affecting topoisomerase I-induced DNA damage.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01195_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01195_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01195_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01195_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01195">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cdab94" class="col">
			FOG01196<br>EOG8M908T<br>EOG8NVX36<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01196_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01196_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01196_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01196_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01196">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f5bcdf" class="col">
			FOG01197<br>EOG8QFTVV<br><br>sce:HPF1;;;<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Haze-protective mannoprotein; reduces the particle size of aggregated proteins in white wines|Putative protein of unknown function; serine/threonine rich and highly similar to YOL155C, a putative glucan alpha-1,4-glucosidase; transcript is induced in both high and low pH environments; YIL169C is a non-essential gene|Putative protein of unknown function; predicted to be a glycosylphosphatidylinositol-modified (GPI) protein; YHR214W has a paralog, YAR066W, that arose from a segmental duplication|Putative GPI protein; YAR066W has a paralog, YHR214W, that arose from a segmental duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8810044" target=_blank>Gamo FJ, <i>et al.</i> (1996 Jun 15)</a>. Analysis of the DNA sequence of a 15,500 bp fragment near the left telomere of chromosome XV from Saccharomyces cerevisiae reveals a putative sugar transporter, a carboxypeptidase homologue and two new open reading frames.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10775354" target=_blank>Dupin IV, <i>et al.</i> (2000 Apr)</a>. Saccharomyces cerevisiae mannoproteins that protect wine from protein haze: evaluation of extraction methods and immunolocalization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11748726" target=_blank>Horie T, <i>et al.</i> (2001 Dec)</a>. Cooperative functions of the mannoprotein-encoding genes in the biogenesis and maintenance of the cell wall in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11935221" target=_blank>Terashima H, <i>et al.</i> (2002 Feb)</a>. Sequence-based approach for identification of cell wall proteins in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16086015" target=_blank>Verstrepen KJ, <i>et al.</i> (2005 Sep)</a>. Intragenic tandem repeats generate functional variability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024473" target=_blank>Brown SL, <i>et al.</i> (2007 Jan)</a>. Reducing haziness in white wine by overexpression of Saccharomyces cerevisiae genes YOL155c and YDR055w.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01197_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01197_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01197_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01197_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01197">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e68586" class="col">
			FOG01198<br>EOG8F4QW4<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Secreted endo-1,4-xylanase; glycoside hydrolase family 11; highly expressed on xylose; expression repressed by tunicamycin and DTT; XlnR regulated<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065265" target=_blank>de Graaff LH, <i>et al.</i> (1994 May)</a>. Regulation of the xylanase-encoding xlnA gene of Aspergillus tubigensis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8787417" target=_blank>Pérez-Gonzalez JA, <i>et al.</i> (1996 Jun)</a>. Molecular cloning and expression in Saccharomyces cerevisiae of two Aspergillus nidulans xylanase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495775" target=_blank>MacCabe AP, <i>et al.</i> (1998 Mar)</a>. Opposite patterns of expression of two Aspergillus nidulans xylanase genes with respect to ambient pH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987120" target=_blank>Orejas M, <i>et al.</i> (1999 Jan)</a>. Carbon catabolite repression of the Aspergillus nidulans xlnA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10359487" target=_blank>Ganga MA, <i>et al.</i> (1999 Mar 15)</a>. Aroma improving in microvinification processes by the use of a recombinant wine yeast strain expressing the Aspergillus nidulans xlnA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12519784" target=_blank>Flipphi M, <i>et al.</i> (2003 Apr 4)</a>. Onset of carbon catabolite repression in Aspergillus nidulans. Parallel involvement of hexokinase and glucokinase in sugar signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15168093" target=_blank>Díaz M, <i>et al.</i> (2004 Sep)</a>. Streptomyces lividans and Brevibacterium lactofermentum as heterologous hosts for the production of X22 xylanase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16473771" target=_blank>Tanaka H, <i>et al.</i> (2005 Dec)</a>. Purification and properties of an extracellular endo-1,4-beta-xylanase from Penicillium citrinum and characterization of the encoding gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16844780" target=_blank>Bauer S, <i>et al.</i> (2006 Jul 25)</a>. Development and application of a suite of polysaccharide-degrading enzymes for analyzing plant cell walls.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16783565" target=_blank>Forment JV, <i>et al.</i> (2006 Sep)</a>. Consecutive gene deletions in Aspergillus nidulans: application of the Cre/loxP system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18457952" target=_blank>Peñalva MA, <i>et al.</i> (2008 Jun)</a>. Ambient pH gene regulation in fungi: making connections.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18420433" target=_blank>Tamayo EN, <i>et al.</i> (2008 Jun)</a>. CreA mediates repression of the regulatory gene xlnR which controls the production of xylanolytic enzymes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19618505" target=_blank>Coutinho PM, <i>et al.</i> (2009 Mar)</a>. Post-genomic insights into the plant polysaccharide degradation potential of  Aspergillus nidulans and comparison to Aspergillus niger and Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22142781" target=_blank>Colabardini AC, <i>et al.</i> (2012 Feb)</a>. Molecular characterization of the Aspergillus nidulans fbxA encoding an F-box protein involved in xylanase induction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23766336" target=_blank>Coradetti ST, <i>et al.</i> (2013 Aug)</a>. Analysis of a conserved cellulase transcriptional regulator reveals inducer-independent production of cellulolytic enzymes in Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23800192" target=_blank>Brown NA, <i>et al.</i> (2013 Jun 25)</a>. Functional characterisation of the non-essential protein kinases and phosphatases regulating Aspergillus nidulans hydrolytic enzyme production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25043916" target=_blank>Martins I, <i>et al.</i> (2014 Jul 21)</a>. Elucidating how the saprophytic fungus Aspergillus nidulans uses the plant polyester suberin as carbon source.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01198_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01198_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01198_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01198_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01198">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#9ab5d8" class="col">
			FOG01199<br>EOG82280N<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01199_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01199_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01199_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01199_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01199">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a4d69b" class="col">
			FOG01200<br>EOG8BZKJ3<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Glucose oxidase|Has domain(s) with predicted flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in oxidation-reduction process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19053082" target=_blank>Shimizu M, <i>et al.</i> (2009 Jan)</a>. Proteomic analysis of Aspergillus nidulans cultured under hypoxic conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19171936" target=_blank>Sato I, <i>et al.</i> (2009 Mar 20)</a>. The glutathione system of Aspergillus nidulans involves a fungus-specific glutathione S-transferase.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01200_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01200_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01200_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01200_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01200">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c39ccd" class="col">
			FOG01201<br>EOG8Z8WCJ<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01201_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01201_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01201_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01201_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01201">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6bc76" class="col">
			FOG01202<br>EOG82RBRB<br>EOG8JQ2F0<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01202_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01202_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01202_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01202_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01202">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff8a" class="col">
			FOG01203<br>EOG83FFBX<br>EOG8FTTFP<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of Aspergillus niger ATCC 1015 : 43178-mRNA<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01203_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01203_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01203_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01203_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01203">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9a48b" class="col">
			FOG01204<br>EOG8DBS1B<br>EOG8JQ2JH<br>EOG8W0W07<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01204_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01204_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01204_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01204_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01204">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4b6dc" class="col">
			FOG01205<br>EOG85QG0G<br>EOG8R7SVJ<br><br>sce:EGT2<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Glycosylphosphatidylinositol (GPI)-anchored cell wall endoglucanase; required for proper cell separation after cytokinesis; expression is activated by Swi5p and tightly regulated in a cell cycle-dependent manner<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8668141" target=_blank>Kovacech B, <i>et al.</i> (1996 Jul)</a>. EGT2 gene transcription is induced predominantly by Swi5 in early G1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613572" target=_blank>Hamada K, <i>et al.</i> (1998 Apr)</a>. Screening for glycosylphosphatidylinositol (GPI)-dependent cell wall proteins in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15452134" target=_blank>Fujita M, <i>et al.</i> (2004 Dec 10)</a>. GPI7 involved in glycosylphosphatidylinositol biosynthesis is essential for yeast cell separation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16086015" target=_blank>Verstrepen KJ, <i>et al.</i> (2005 Sep)</a>. Intragenic tandem repeats generate functional variability.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01205_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01205_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01205_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01205_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01205">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e47c7d" class="col">
			FOG01206<br>EOG8KWH7N<br>EOG8M908T<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387822" target=_blank>Lan CY, <i>et al.</i> (2004 Sep)</a>. Regulatory networks affected by iron availability in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17371861" target=_blank>Bates S, <i>et al.</i> (2007 Jun)</a>. Candida albicans Iff11, a secreted protein required for cell wall structure and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21841123" target=_blank>Boisramé A, <i>et al.</i> (2011 Oct)</a>. Unexpected role for a serine/threonine-rich domain in the Candida albicans Iff protein family.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01206_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01206_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01206_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01206_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01206">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#92b0d5" class="col">
			FOG01207<br>EOG8M908T<br>EOG8X69SG<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01207_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01207_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01207_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01207_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01207">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#9dd393" class="col">
			FOG01208<br>EOG82V6ZX<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Enoyl-ACP reductase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01208_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01208_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01208_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01208_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01208">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bf95ca" class="col">
			FOG01209<br>EOG8G4F55<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01209_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01209_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01209_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01209_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01209">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f6b76b" class="col">
			FOG01210<br>EOG8STQPC<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative subtilisin-like serine protease<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01210_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01210_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01210_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01210_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01210">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fcff81" class="col">
			FOG01211<br>EOG8M908T<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01211_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01211_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01211_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01211_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01211">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c59d82" class="col">
			FOG01212<br>EOG8M908T<br><br>sce:SAE3<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Meiosis-specific protein involved in meiotic recombination; involved in DMC1-dependent meiotic recombination; forms heterodimer with Mei5p; proposed to be an assembly factor for Dmc1p<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9215889" target=_blank>McKee AH, <i>et al.</i> (1997 Jul)</a>. Mutations in Saccharomyces cerevisiae that block meiotic prophase chromosome metabolism and confer cell cycle arrest at pachytene identify two new meiosis-specific genes SAE1 and SAE3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15620352" target=_blank>Hayase A, <i>et al.</i> (2004 Dec 29)</a>. A protein complex containing Mei5 and Sae3 promotes the assembly of the meiosis-specific RecA homolog Dmc1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15238514" target=_blank>Young JA, <i>et al.</i> (2004 Jun)</a>. Conserved and nonconserved proteins for meiotic DNA breakage and repair in yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15579681" target=_blank>Tsubouchi H, <i>et al.</i> (2004 Nov)</a>. The budding yeast mei5 and sae3 proteins act together with dmc1 during meiotic recombination.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16341884" target=_blank>Fukuda T, <i>et al.</i> (2006 Feb)</a>. Recruitment of RecA homologs Dmc1p and Rad51p to the double-strand break repair site initiated by meiosis-specific endonuclease VDE (PI-SceI).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01212_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01212_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01212_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01212_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01212">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4b1da" class="col">
			FOG01213<br>EOG8BZKJ3<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Glucose oxidase|Glucose oxidase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01213_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01213_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01213_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01213_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01213">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e17273" class="col">
			FOG01214<br>EOG8BZKJ3<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted choline dehydrogenase activity, flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in alcohol metabolic process, oxidation-reduction process|Has domain(s) with predicted choline dehydrogenase activity, flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in alcohol metabolic process, oxidation-reduction process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01214_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01214_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01214_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01214_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01214">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#89aad1" class="col">
			FOG01215<br>EOG82280N<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function|Ortholog of A. niger CBS 513.88 : An13g02420, Aspergillus wentii : Aspwe1_0105685, Aspergillus niger ATCC 1015 : 192085-mRNA, 42223-mRNA and Aspergillus terreus NIH2624 : ATET_07066<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01215_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01215_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01215_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01215_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01215">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#95cf8a" class="col">
			FOG01216<br>EOG8JQ2F0<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01216_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01216_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01216_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01216_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01216">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ba8dc6" class="col">
			FOG01217<br>EOG8JQ2F0<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01217_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01217_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01217_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01217_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01217">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f5b15f" class="col">
			FOG01218<br>EOG8254BZ<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01218_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01218_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01218_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01218_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01218">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbff77" class="col">
			FOG01219<br>EOG8G4F55<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01219_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01219_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01219_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01219_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01219">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c09578" class="col">
			FOG01220<br>EOG8F4QW4<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01220_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01220_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01220_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01220_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01220">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f3abd7" class="col">
			FOG01221<br>EOG87H46Z<br>EOG8JQ2F0<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01221_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01221_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01221_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01221_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01221">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#df6869" class="col">
			FOG01222<br>EOG8STQPC<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01222_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01222_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01222_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01222_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01222">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#81a4ce" class="col">
			FOG01223<br>EOG83XSQK<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01223_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01223_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01223_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01223_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01223">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#8ecc82" class="col">
			FOG01224<br>EOG8Z8WCJ<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01224_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01224_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01224_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01224_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01224">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b585c2" class="col">
			FOG01225<br>EOG87WM7B<br>EOG8NVX36<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01225_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01225_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01225_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01225_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01225">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4ac54" class="col">
			FOG01226<br>EOG8Z8WCJ<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01226_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01226_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01226_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01226_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01226">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbff6e" class="col">
			FOG01227<br>EOG8JQ2F0<br><br>sce:;<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>|Pseudogenic fragment with similarity to flocculins; identified by gene-trapping, microarray-based expression analysis, and genome-wide homology searching; YHR212W-A has a paralog, YAR061W, that arose from a segmental duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11753363" target=_blank>Kumar A, <i>et al.</i> (2002 Jan)</a>. An integrated approach for finding overlooked genes in yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01227_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01227_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01227_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01227_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01227">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bc8e6f" class="col">
			FOG01228<br>EOG8DNCKD<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>cell surface glycoprotein (predicted), DIPSY family<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01228_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01228_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01228_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01228_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01228">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2a5d4" class="col">
			FOG01229<br>EOG8M908T<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01229_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01229_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01229_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01229_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01229">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dd5e5f" class="col">
			FOG01230<br>EOG8M908T<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01230_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01230_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01230_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01230_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01230">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#799ecb" class="col">
			FOG01231<br>EOG8DNCKD<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN3397, Aspergillus niger ATCC 1015 : 51389-mRNA, Aspergillus zonatus : Aspzo1_0142097, Aspergillus sydowii : Aspsy1_1179377 and Aspergillus terreus NIH2624 : ATET_09008|Ortholog of Aspergillus tubingensis : Asptu1_0164855, Aspergillus brasiliensis : Aspbr1_0033917, Aspergillus kawachii : Aspka1_0178340 and Aspergillus acidus : Aspfo1_0146136<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01231_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01231_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01231_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01231_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01231">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#86c97a" class="col">
			FOG01232<br>EOG8KWH7N<br>EOG8QJQ3R<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN3519, AN2646, AN0587, AN0866, AN7370 and A. fumigatus Af293 : Afu1g15200, Afu3g03790, Afu4g14040, Afu5g06900, Afu7g08575|Ortholog of A. nidulans FGSC A4 : AN3519, AN2646, AN0587, AN0866, AN7370 and A. fumigatus Af293 : Afu1g15200, Afu3g03790, Afu4g14040, Afu5g06900, Afu7g08575<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01232_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01232_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01232_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01232_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01232">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b07dbe" class="col">
			FOG01233<br>EOG86Q5C8<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN3258, A. fumigatus Af293 : Afu8g07090, A. oryzae RIB40 : AO090001000070, Neosartorya fischeri NRRL 181 : NFIA_099870 and Aspergillus clavatus NRRL 1 : ACLA_017870<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01233_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01233_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01233_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01233_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01233">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f3a649" class="col">
			FOG01234<br>EOG8H9W42<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted viral capsid localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01234_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01234_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01234_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01234_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01234">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbff64" class="col">
			FOG01235<br>EOG8STQPC<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01235_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01235_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01235_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01235_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01235">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b78665" class="col">
			FOG01236<br>EOG8H9W42<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01236_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01236_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01236_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01236_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01236">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f19fd1" class="col">
			FOG01237<br>EOG8M908T<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01237_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01237_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01237_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01237_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01237">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#db5456" class="col">
			FOG01238<br>EOG8254CD<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01238_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01238_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01238_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01238_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01238">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#7198c8" class="col">
			FOG01239<br>EOG82JM6N<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01239_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01239_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01239_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01239_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01239">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#7fc572" class="col">
			FOG01240<br>EOG8G4F55<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>NMS complex subunit, Spc105/KNL-1 family member, blinkin, Spc7|cell surface agglutination protein Map4<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN5229, A. fumigatus Af293 : Afu2g04050, A. oryzae RIB40 : AO090005001534, Aspergillus wentii : Aspwe1_0515975 and Aspergillus sydowii : Aspsy1_0134382<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15371542" target=_blank>Kerres A, <i>et al.</i> (2004 Dec)</a>. The fission yeast kinetochore component Spc7 associates with the EB1 family member Mal3 and is required for kinetochore-spindle association.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15502821" target=_blank>Obuse C, <i>et al.</i> (2004 Nov)</a>. A conserved Mis12 centromere complex is linked to heterochromatic HP1 and outer kinetochore protein Zwint-1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16079914" target=_blank>Liu X, <i>et al.</i> (2005 Aug 17)</a>. Molecular analysis of kinetochore architecture in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17035632" target=_blank>Hayashi A, <i>et al.</i> (2006 Dec)</a>. Reconstruction of the kinetochore during meiosis in fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16855021" target=_blank>Kerres A, <i>et al.</i> (2006 Oct)</a>. Fta2, an essential fission yeast kinetochore component, interacts closely with the conserved Mal2 protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17442892" target=_blank>Kerres A, <i>et al.</i> (2007 Jul)</a>. The conserved Spc7 protein is required for spindle integrity and links kinetochore complexes in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21664573" target=_blank>Meadows JC, <i>et al.</i> (2011 Jun 14)</a>. Spindle checkpoint silencing requires association of PP1 to both Spc7 and kinesin-8 motors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22065639" target=_blank>Chen JS, <i>et al.</i> (2011 Nov 14)</a>. Cdk1 phosphorylation of the kinetochore protein Nsk1 prevents error-prone chromosome segregation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22711988" target=_blank>Jakopec V, <i>et al.</i> (2012 Aug)</a>. Sos7, an essential component of the conserved Schizosaccharomyces pombe Ndc80-MIND-Spc7 complex, identifies a new family of fungal kinetochore proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22660415" target=_blank>Yamagishi Y, <i>et al.</i> (2012 Jun 3)</a>. MPS1/Mph1 phosphorylates the kinetochore protein KNL1/Spc7 to recruit SAC components.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22521786" target=_blank>Shepperd LA, <i>et al.</i> (2012 May 22)</a>. Phosphodependent recruitment of Bub1 and Bub3 to Spc7/KNL1 by Mph1 kinase maintains the spindle checkpoint.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23166349" target=_blank>Hou H, <i>et al.</i> (2012 Nov 26)</a>. Csi1 links centromeres to the nuclear envelope for centromere clustering.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23512483" target=_blank>Funabiki H, <i>et al.</i> (2013 Jun)</a>. Making an effective switch at the kinetochore by phosphorylation and dephosphorylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24497846" target=_blank>Kriegenburg F, <i>et al.</i> (2014 Jan)</a>. A chaperone-assisted degradation pathway targets kinetochore proteins to ensure genome stability.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26483559" target=_blank>Yang HJ, <i>et al.</i> (2015 Oct 26)</a>. Nup132 modulates meiotic spindle attachment in fission yeast by regulating kinetochore assembly.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01240_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01240_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01240_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01240_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01240">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ab75ba" class="col">
			FOG01241<br>EOG8JQ2JH<br><br>sce:CCW14<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Covalently linked cell wall glycoprotein; present in the inner layer of the cell wall<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN1338, AN9450, AN4381, A. fumigatus Af293 : Afu1g03040, Afu1g09510, Afu4g06700 and A. niger CBS 513.88 : An01g05230|Protein of unknown function<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079899" target=_blank>Moukadiri I, <i>et al.</i> (1997 Apr)</a>. Identification of a mannoprotein present in the inner layer of the cell wall of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10322008" target=_blank>Mrsa V, <i>et al.</i> (1999 May)</a>. Deletion of new covalently linked cell wall glycoproteins alters the electrophoretic mobility of phosphorylated wall components of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12388749" target=_blank>Nantel A, <i>et al.</i> (2002 Oct)</a>. Transcription profiling of Candida albicans cells undergoing the yeast-to-hyphal transition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12904553" target=_blank>Garcerá A, <i>et al.</i> (2003 Aug)</a>. Identification and study of a Candida albicans protein homologous to Saccharomyces cerevisiae Ssr1p, an internal cell-wall protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12734798" target=_blank>Lee SA, <i>et al.</i> (2003 May)</a>. An analysis of the Candida albicans genome database for soluble secreted proteins using computer-based prediction algorithms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15302828" target=_blank>de Groot PW, <i>et al.</i> (2004 Aug)</a>. Proteomic analysis of Candida albicans cell walls reveals covalently bound carbohydrate-active enzymes and adhesins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15554973" target=_blank>Bensen ES, <i>et al.</i> (2004 Dec)</a>. Transcriptional profiling in Candida albicans reveals new adaptive responses to extracellular pH and functions for Rim101p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15814840" target=_blank>Kadosh D, <i>et al.</i> (2005 Jun)</a>. Induction of the Candida albicans filamentous growth program by relief of transcriptional repression: a genome-wide analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781460" target=_blank>Yin QY, <i>et al.</i> (2005 May 27)</a>. Comprehensive proteomic analysis of Saccharomyces cerevisiae cell walls: identification of proteins covalently attached via glycosylphosphatidylinositol remnants or mild alkali-sensitive linkages.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16024227" target=_blank>Garcerá A, <i>et al.</i> (2005 Nov)</a>. Anchorage of Candida albicans Ssr1 to the cell wall, and transcript profiling of the null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16151249" target=_blank>Murillo LA, <i>et al.</i> (2005 Sep)</a>. Genome-wide transcription profiling of the early phase of biofilm formation by Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16102003" target=_blank>Urban C, <i>et al.</i> (2005 Sep)</a>. The moonlighting protein Tsa1p is implicated in oxidative stress response and in cell wall biogenesis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16267276" target=_blank>Cao F, <i>et al.</i> (2006 Jan)</a>. The Flo8 transcription factor is essential for hyphal development and virulence in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17192409" target=_blank>Alvarez FJ, <i>et al.</i> (2007 Mar)</a>. Identification of an N-acetylglucosamine transporter that mediates hyphal induction in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17605131" target=_blank>Sexton JA, <i>et al.</i> (2007 Oct)</a>. Regulation of sugar transport and metabolism by the Candida albicans Rgt1 transcriptional repressor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227255" target=_blank>Sosinska GJ, <i>et al.</i> (2008 Feb)</a>. Hypoxic conditions and iron restriction affect the cell-wall proteome of Candida albicans grown under vagina-simulative conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18625069" target=_blank>Bader O, <i>et al.</i> (2008 Jul 14)</a>. Processing of predicted substrates of fungal Kex2 proteinases from Candida albicans, C. glabrata, Saccharomyces cerevisiae and Pichia pastoris.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765290" target=_blank>Plaine A, <i>et al.</i> (2008 Oct)</a>. Functional analysis of Candida albicans GPI-anchored proteins: roles in cell wall integrity and caspofungin sensitivity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18712765" target=_blank>Castillo L, <i>et al.</i> (2008 Sep)</a>. A study of the Candida albicans cell wall proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19032986" target=_blank>Walker LA, <i>et al.</i> (2009 Feb)</a>. Genome-wide analysis of Candida albicans gene expression patterns during infection of the mammalian kidney.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19585695" target=_blank>de Groot PW, <i>et al.</i> (2009 Mar)</a>. Comprehensive genomic analysis of cell wall genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20641015" target=_blank>Sorgo AG, <i>et al.</i> (2010 Aug)</a>. Mass spectrometric analysis of the secretome of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21496229" target=_blank>Chaudhuri R, <i>et al.</i> (2011 Apr 15)</a>. FungalRV: adhesin prediction and immunoinformatics portal for human fungal pathogens.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602216" target=_blank>Heilmann CJ, <i>et al.</i> (2011 Aug)</a>. Hyphal induction in the human fungal pathogen Candida albicans reveals a characteristic wall protein profile.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622905" target=_blank>Sorgo AG, <i>et al.</i> (2011 Aug)</a>. Effects of fluconazole on the secretome, the wall proteome, and wall integrity of the clinical fungus Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20864472" target=_blank>Sosinska GJ, <i>et al.</i> (2011 Jan)</a>. Mass spectrometric quantification of the adaptations in the wall proteome of Candida albicans in response to ambient pH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22544909" target=_blank>Fanning S, <i>et al.</i> (2012 Jul)</a>. Divergent targets of Candida albicans biofilm regulator Bcr1 in vitro and in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23472095" target=_blank>Colabardini AC, <i>et al.</i> (2013)</a>. Functional characterization of Aspergillus nidulans ypkA, a homologue of the mammalian kinase SGK.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23951271" target=_blank>Fox SJ, <i>et al.</i> (2013)</a>. Characterization of genetic determinants that modulate Candida albicans filamentation in the presence of bacteria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23516516" target=_blank>Martin R, <i>et al.</i> (2013)</a>. A core filamentation response network in Candida albicans is restricted to eight genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23242994" target=_blank>Carlisle PL, <i>et al.</i> (2013 Feb)</a>. A genome-wide transcriptional analysis of morphology determination in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23208712" target=_blank>Silva LV, <i>et al.</i> (2013 Feb)</a>. Milbemycins: more than efflux inhibitors for fungal pathogens.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01241_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01241_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01241_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01241_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01241">Phobius transmembrane predictions</a>
										<br>11 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f3a13e" class="col">
			FOG01242<br>EOG8P8D0F<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01242_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01242_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01242_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01242_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01242">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbff5b" class="col">
			FOG01243<br>EOG8Q2BWF<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01243_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01243_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01243_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01243_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01243">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#b37f5c" class="col">
			FOG01244<br>EOG8254BZ<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN8649, A. fumigatus Af293 : Afu1g04100, A. oryzae RIB40 : AO090009000519, Aspergillus wentii : Aspwe1_0037325, Aspwe1_0104267 and Aspergillus sydowii : Aspsy1_0048337<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01244_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01244_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01244_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01244_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01244">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f099ce" class="col">
			FOG01245<br>EOG8QNKDJ<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01245_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01245_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01245_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01245_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01245">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d94a4c" class="col">
			FOG01246<br>EOG88933K<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01246_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01246_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01246_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01246_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01246">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#6992c5" class="col">
			FOG01247<br>EOG87H46Z<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01247_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01247_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01247_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01247_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01247">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#78c26a" class="col">
			FOG01248<br>EOG8F4QW4<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted hydrolase activity, hydrolyzing O-glycosyl compounds activity and role in carbohydrate metabolic process|Endo-1,4-beta-xylanase|Putative xylanase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065265" target=_blank>de Graaff LH, <i>et al.</i> (1994 May)</a>. Regulation of the xylanase-encoding xlnA gene of Aspergillus tubigensis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8787417" target=_blank>Pérez-Gonzalez JA, <i>et al.</i> (1996 Jun)</a>. Molecular cloning and expression in Saccharomyces cerevisiae of two Aspergillus nidulans xylanase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495775" target=_blank>MacCabe AP, <i>et al.</i> (1998 Mar)</a>. Opposite patterns of expression of two Aspergillus nidulans xylanase genes with respect to ambient pH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987120" target=_blank>Orejas M, <i>et al.</i> (1999 Jan)</a>. Carbon catabolite repression of the Aspergillus nidulans xlnA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10359487" target=_blank>Ganga MA, <i>et al.</i> (1999 Mar 15)</a>. Aroma improving in microvinification processes by the use of a recombinant wine yeast strain expressing the Aspergillus nidulans xlnA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12519784" target=_blank>Flipphi M, <i>et al.</i> (2003 Apr 4)</a>. Onset of carbon catabolite repression in Aspergillus nidulans. Parallel involvement of hexokinase and glucokinase in sugar signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15168093" target=_blank>Díaz M, <i>et al.</i> (2004 Sep)</a>. Streptomyces lividans and Brevibacterium lactofermentum as heterologous hosts for the production of X22 xylanase from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16473771" target=_blank>Tanaka H, <i>et al.</i> (2005 Dec)</a>. Purification and properties of an extracellular endo-1,4-beta-xylanase from Penicillium citrinum and characterization of the encoding gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16844780" target=_blank>Bauer S, <i>et al.</i> (2006 Jul 25)</a>. Development and application of a suite of polysaccharide-degrading enzymes for analyzing plant cell walls.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16783565" target=_blank>Forment JV, <i>et al.</i> (2006 Sep)</a>. Consecutive gene deletions in Aspergillus nidulans: application of the Cre/loxP system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18457952" target=_blank>Peñalva MA, <i>et al.</i> (2008 Jun)</a>. Ambient pH gene regulation in fungi: making connections.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18420433" target=_blank>Tamayo EN, <i>et al.</i> (2008 Jun)</a>. CreA mediates repression of the regulatory gene xlnR which controls the production of xylanolytic enzymes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19618505" target=_blank>Coutinho PM, <i>et al.</i> (2009 Mar)</a>. Post-genomic insights into the plant polysaccharide degradation potential of  Aspergillus nidulans and comparison to Aspergillus niger and Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22142781" target=_blank>Colabardini AC, <i>et al.</i> (2012 Feb)</a>. Molecular characterization of the Aspergillus nidulans fbxA encoding an F-box protein involved in xylanase induction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23766336" target=_blank>Coradetti ST, <i>et al.</i> (2013 Aug)</a>. Analysis of a conserved cellulase transcriptional regulator reveals inducer-independent production of cellulolytic enzymes in Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23800192" target=_blank>Brown NA, <i>et al.</i> (2013 Jun 25)</a>. Functional characterisation of the non-essential protein kinases and phosphatases regulating Aspergillus nidulans hydrolytic enzyme production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25043916" target=_blank>Martins I, <i>et al.</i> (2014 Jul 21)</a>. Elucidating how the saprophytic fungus Aspergillus nidulans uses the plant polyester suberin as carbon source.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01248_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01248_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01248_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01248_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01248">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a76db6" class="col">
			FOG01249<br>EOG8M37QP<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01249_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01249_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01249_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01249_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01249">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f29c33" class="col">
			FOG01250<br>EOG85QG0G<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01250_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01250_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01250_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01250_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01250">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#faff51" class="col">
			FOG01251<br>EOG8NS1TQ<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01251_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01251_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01251_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01251_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01251">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#af7853" class="col">
			FOG01252<br>EOG8FFBN0<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773383" target=_blank>Uhl MA, <i>et al.</i> (2003 Jun 2)</a>. Haploinsufficiency-based large-scale forward genetic analysis of filamentous growth in the diploid human fungal pathogen C.albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14585977" target=_blank>Bennett RJ, <i>et al.</i> (2003 Nov)</a>. Identification and characterization of a Candida albicans mating pheromone.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21496229" target=_blank>Chaudhuri R, <i>et al.</i> (2011 Apr 15)</a>. FungalRV: adhesin prediction and immunoinformatics portal for human fungal pathogens.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21592964" target=_blank>Singh RP, <i>et al.</i> (2011 Jul 15)</a>. Cap2-HAP complex is a critical transcriptional regulator that has dual but contrasting roles in regulation of iron homeostasis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21414038" target=_blank>Bonhomme J, <i>et al.</i> (2011 May)</a>. Contribution of the glycolytic flux and hypoxia adaptation to efficient biofilm formation by Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01252_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01252_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01252_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01252_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01252">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ef93cc" class="col">
			FOG01253<br>EOG8M908T<br><br>sce:;HBT1;BDF2<br><br>Genes: 47
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative protein of unknown function; green fluorescent protein (GFP)-fusion protein localizes to the cytoplasm and bud; interacts with Spa2p; YFR016C is not an essential gene|Shmoo tip protein, substrate of Hub1p ubiquitin-like protein; mutants are defective for mating projection formation, thereby implicating Hbt1p in polarized cell morphogenesis; HBT1 has a paralog, YNL195C, that arose from the whole genome duplication|Protein involved in transcription initiation; acts at TATA-containing promoters; associates with the basal transcription factor TFIID; contains two bromodomains; corresponds to the C-terminal region of mammalian TAF1; redundant with Bdf1p; protein abundance increases in response to DNA replication stress; BDF2 has a paralog, BDF1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>Swi5 protein|cell surface glycoprotein, flocculin, related to Gsf2<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN0923, A. fumigatus Af293 : Afu1g15870, A. oryzae RIB40 : AO090005001113, Aspergillus sydowii : Aspsy1_0053441 and Aspergillus glaucus : Aspgl1_0063899|Ortholog of A. nidulans FGSC A4 : AN7587, A. fumigatus Af293 : Afu2g15180, A. oryzae RIB40 : AO090012000301, Aspergillus wentii : Aspwe1_0054076 and Aspergillus sydowii : Aspsy1_0058595|Ortholog(s) have role in DNA recombinase assembly, gene conversion at mating-type locus, meiotic DNA repair synthesis, meiotic joint molecule formation, positive regulation of DNA binding, strand invasion||Ortholog(s) have role in fungal-type cell wall organization and hyphal cell wall localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175470" target=_blank>Jeanmougin F, <i>et al.</i> (1997 May)</a>. The bromodomain revisited.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10783167" target=_blank>Matangkasombut O, <i>et al.</i> (2000 Apr 15)</a>. Bromodomain factor 1 corresponds to a missing piece of yeast TFIID.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923536" target=_blank>Dittmar GA, <i>et al.</i> (2002 Mar 29)</a>. Role of a ubiquitin-like modification in polarized morphogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620224" target=_blank>Matangkasombut O, <i>et al.</i> (2003 Feb)</a>. Different sensitivities of bromodomain factors 1 and 2 to histone H4 acetylation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15456898" target=_blank>Martinez MJ, <i>et al.</i> (2004 Dec)</a>. Genomic analysis of stationary-phase and exit in Saccharomyces cerevisiae: gene expression and identification of novel essential genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15143168" target=_blank>Sawa C, <i>et al.</i> (2004 Jun)</a>. Bromodomain factor 1 (Bdf1) is phosphorylated by protein kinase CK2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15814841" target=_blank>García-Sánchez S, <i>et al.</i> (2005 Jun)</a>. Global roles of Ssn6 in Tup1- and Nrg1-dependent gene regulation in the fungal pathogen, Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15917516" target=_blank>Liu TT, <i>et al.</i> (2005 Jun)</a>. Genome-wide expression profiling of the response to azole, polyene, echinocandin, and pyrimidine antifungal agents in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16102003" target=_blank>Urban C, <i>et al.</i> (2005 Sep)</a>. The moonlighting protein Tsa1p is implicated in oxidative stress response and in cell wall biogenesis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16455273" target=_blank>Castillo L, <i>et al.</i> (2006 Feb)</a>. Genomic response programs of Candida albicans following protoplasting and regeneration.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17371861" target=_blank>Bates S, <i>et al.</i> (2007 Jun)</a>. Candida albicans Iff11, a secreted protein required for cell wall structure and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164403" target=_blank>Fernández-Arenas E, <i>et al.</i> (2007 Mar)</a>. Integrated proteomics and genomics strategies bring new insight into Candida albicans response upon macrophage interaction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18434592" target=_blank>Rauceo JM, <i>et al.</i> (2008 Jul)</a>. Regulation of the Candida albicans cell wall damage response by transcription factor Sko1 and PAS kinase Psk1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20735781" target=_blank>Lohse MB, <i>et al.</i> (2010 Oct)</a>. Temporal anatomy of an epigenetic switch in cell programming: the white-opaque transition of C. albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21592964" target=_blank>Singh RP, <i>et al.</i> (2011 Jul 15)</a>. Cap2-HAP complex is a critical transcriptional regulator that has dual but contrasting roles in regulation of iron homeostasis in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21841123" target=_blank>Boisramé A, <i>et al.</i> (2011 Oct)</a>. Unexpected role for a serine/threonine-rich domain in the Candida albicans Iff protein family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343036" target=_blank>Gelis S, <i>et al.</i> (2012 Apr)</a>. Pga13 in Candida albicans is localized in the cell wall and influences cell surface properties, morphogenesis and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23613980" target=_blank>Palige K, <i>et al.</i> (2013)</a>. Global transcriptome sequencing identifies chlamydospore specific markers in Candida albicans and Candida dubliniensis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23731904" target=_blank>Yu Q, <i>et al.</i> (2013 Jul)</a>. The P-type ATPase Spf1 is required for endoplasmic reticulum functions and cell wall integrity in Candida albicans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01253_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01253_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01253_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01253_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01253">Phobius transmembrane predictions</a>
										<br>10 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d74142" class="col">
			FOG01254<br>EOG8WM3C4<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01254_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01254_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01254_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01254_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01254">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#608cc2" class="col">
			FOG01255<br>EOG8KWH7N<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of Aspergillus tubingensis : Asptu1_0199036, Aspergillus brasiliensis : Aspbr1_0203827, Aspergillus acidus : Aspfo1_0076661 and Aspergillus niger ATCC 1015 : 37023-mRNA<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15128742" target=_blank>Sheppard DC, <i>et al.</i> (2004 Jul 16)</a>. Functional and structural diversity in the Als protein family of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15870470" target=_blank>Zhao X, <i>et al.</i> (2005 May)</a>. Analysis of the Candida albicans Als2p and Als4p adhesins suggests the potential for compensatory function within the Als family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106872" target=_blank>Coleman DA, <i>et al.</i> (2012 Apr)</a>. A monoclonal antibody specific for Candida albicans Als4 demonstrates overlapping localization of Als family proteins on the fungal cell surface and highlights differences between Als localization in vitro and in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22321066" target=_blank>Aoki W, <i>et al.</i> (2012 Jun)</a>. Profiling of adhesive properties of the agglutinin-like sequence (ALS) protein family, a virulent attribute of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22429754" target=_blank>Monroy-Pérez E, <i>et al.</i> (2012 May)</a>. Frequency and expression of ALS and HWP1 genotypes in Candida albicans strains isolated from Mexican patients suffering from vaginal candidosis.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01255_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01255_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01255_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01255_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01255">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#70bf62" class="col">
			FOG01256<br>EOG8BZKJ3<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted choline dehydrogenase activity, flavin adenine dinucleotide binding, oxidoreductase activity, acting on CH-OH group of donors activity and role in alcohol metabolic process, oxidation-reduction process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01256_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01256_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01256_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01256_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01256">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a265b2" class="col">
			FOG01257<br>EOG82280N<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01257_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01257_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01257_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01257_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01257">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f19628" class="col">
			FOG01258<br>EOG8STQPC<br><br>sce:YSP3<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative precursor to the subtilisin-like protease III; YSP3 has a paralog, PRB1, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9784122" target=_blank>Chu S, <i>et al.</i> (1998 Oct 23)</a>. The transcriptional program of sporulation in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115118" target=_blank>Ladds G, <i>et al.</i> (2000 Nov)</a>. Identification of proteases with shared functions to the proprotein processing protease Krp1 in the fission yeast Schizosaccharomyces pombe.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01258_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01258_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01258_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01258_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01258">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#faff48" class="col">
			FOG01259<br>EOG8H9W42<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01259_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01259_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01259_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01259_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01259">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#aa7049" class="col">
			FOG01260<br>EOG8Z8WCJ<br><br>sce:absent<br><br>Genes: 13
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function|Protein of unknown function<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01260_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01260_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01260_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01260_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01260">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#ef8dc9" class="col">
			FOG01261<br>EOG83XSQK<br><br>sce:BSC1<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein of unconfirmed function; similar to cell surface flocculin Flo11p; ORF exhibits genomic organization compatible with a translational readthrough-dependent mode of expression<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12711673" target=_blank>Namy O, <i>et al.</i> (2003 May 1)</a>. Identification of stop codon readthrough genes in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01261_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01261_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01261_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01261_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01261">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d53738" class="col">
			FOG01262<br>EOG8DFN4F<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01262_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01262_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01262_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01262_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01262">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#5886be" class="col">
			FOG01263<br>EOG82FQZG<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01263_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01263_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01263_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01263_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01263">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#69bb5a" class="col">
			FOG01264<br>EOG8QV9TQ<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01264_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01264_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01264_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01264_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01264">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#9d5dae" class="col">
			FOG01265<br>EOG8NVX36<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01265_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01265_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01265_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01265_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01265">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f1911d" class="col">
			FOG01266<br>EOG8GB5R0<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01266_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01266_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01266_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01266_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01266">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#faff3e" class="col">
			FOG01267<br>EOG8JQ2F0<br><br>sce:;<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative protein of unknown function; shows sequence similarity to FLO1 and other flocculins|Putative protein of unknown function; YFL051C is not an essential gene<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10591965" target=_blank>Kobayashi O, <i>et al.</i> (1999 Nov)</a>. Analysis of the genes activated by the FLO8 gene in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01267_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01267_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01267_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01267_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01267">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#a66940" class="col">
			FOG01268<br>EOG86Q5C8<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>cell wall protein Pwp1|conserved fungal protein|conserved fungal protein Adg2<br><br>
			<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN3310, AN1449, AN7654, A. fumigatus Af293 : Afu2g01140, Afu3g00880, Afu8g04370 and A. niger CBS 513.88 : An10g00430|Protein of unknown function|Ortholog of A. nidulans FGSC A4 : AN1941, A. fumigatus Af293 : Afu6g07855, Neosartorya fischeri NRRL 181 : NFIA_053540 and Aspergillus versicolor : Aspve1_0025340<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12845604" target=_blank>De Groot PW, <i>et al.</i> (2003 Jul 15)</a>. Genome-wide identification of fungal GPI proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15689498" target=_blank>Alonso-Nuñez ML, <i>et al.</i> (2005 Apr)</a>. Ace2p controls the expression of genes required for cell separation in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16303567" target=_blank>Martín-Castellanos C, <i>et al.</i> (2005 Nov 22)</a>. A large-scale screen in S. pombe identifies seven novel genes required for critical meiotic events.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17230583" target=_blank>de Groot PW, <i>et al.</i> (2007 Apr)</a>. Mass spectrometric identification of covalently bound cell wall proteins from the fission yeast Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23050226" target=_blank>Chen Z, <i>et al.</i> (2012 Oct)</a>. A genetic screen to discover pathways affecting cohesin function in Schizosaccharomyces pombe identifies chromatin effectors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26167880" target=_blank>Lipp JJ, <i>et al.</i> (2015 Aug)</a>. SR protein kinases promote splicing of nonconsensus introns.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG01268_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01268_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG01268_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG01268_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00499_phobius.html#FOG01268">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<!--link to gblock-->
	<img src="../img/trees/HOG00499.png"
	</div>
	<div id="footer">Footer</div>
</div>

</body></html>