<html>
<head><title>HOG00448</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00447.html">HOG00447&emsp;&emsp;SSM1 family</a><br> 
																	HOG00448&emsp;&emsp;<br> 
																	<a href="HOG00449.html">HOG00449&emsp;&emsp;group II decarboxylase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00448.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00448_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00448_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00448_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00448_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG00448.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00970<br>EOG8N02WM<br>MSL0<br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Malate synthase, located in glyoxysome in fungi. Sequences have SKL; fungal gene have introns, <br><br>
			<br><b>Features</b><br>[x];[c]?<br><br>
			<br><b>AspGD Description</b><br>Malate synthase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11945599" target=_blank>Armitt S, <i>et al.</i> (1971 Jan 30)</a>. Mutants of Aspergillus nidulans lacking malate synthase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3622" target=_blank>Armitt S, <i>et al.</i> (1976 Feb)</a>. Analysis of acetate non-utilizing (acu) mutants in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=778333" target=_blank>Payton M, <i>et al.</i> (1976 May)</a>. Agar as a carbon source and its effect on the utilization of other carbon sources by acetate non-utilizing (acu) mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3551935" target=_blank>Hynes MJ, <i>et al.</i> (1986 Sep)</a>. The amdS gene of Aspergillus nidulans: control by multiple regulatory signals.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054423" target=_blank>Thomas GH, <i>et al.</i> (1988 Sep)</a>. Molecular cloning, identification and transcriptional analysis of genes involved in acetate utilization in Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2571070" target=_blank>Sandeman RA, <i>et al.</i> (1989 Jul)</a>. Isolation of the facA (acetyl-coenzyme A synthetase) and acuE (malate synthase) genes of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1832736" target=_blank>Sandeman RA, <i>et al.</i> (1991 Sep)</a>. Molecular organisation of the malate synthase genes of Aspergillus nidulans and Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1454530" target=_blank>Hartig A, <i>et al.</i> (1992 Nov 11)</a>. Differentially regulated malate synthase genes participate in carbon and nitrogen metabolism of S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7614370" target=_blank>Kawasaki L, <i>et al.</i> (1995 Mar)</a>. Aspergillus nidulans mutants affected in acetate metabolism isolated as lipid nonutilizers.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9385142" target=_blank>De Lucas JR, <i>et al.</i> (1997 Dec)</a>. Characterization of oleate-nonutilizing mutants of Aspergillus nidulans isolated by the 3-amino-1,2,4-triazole positive selection method.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524126" target=_blank>Todd RB, <i>et al.</i> (1998 Apr 1)</a>. FacB, the Aspergillus nidulans activator of acetate utilization genes, binds dissimilar DNA sequences.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829933" target=_blank>Stemple CJ, <i>et al.</i> (1998 Dec)</a>. The facC gene of Aspergillus nidulans encodes an acetate-inducible carnitine acetyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9818356" target=_blank>Valenciano S, <i>et al.</i> (1998 Oct)</a>. Characterization of Aspergillus nidulans peroxisomes by immunoelectron microscopy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14732266" target=_blank>Sims AH, <i>et al.</i> (2004 Feb)</a>. Use of expressed sequence tag analysis and cDNA microarrays of the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17107606" target=_blank>David H, <i>et al.</i> (2006)</a>. Metabolic network driven analysis of genome-wide transcription data from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16682457" target=_blank>Hynes MJ, <i>et al.</i> (2006 May)</a>. Regulatory genes controlling fatty acid catabolism and peroxisomal functions in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245820" target=_blank>Hynes MJ, <i>et al.</i> (2008 Mar)</a>. Genetic analysis of the role of peroxisomes in the utilization of acetate and fatty acids in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20348383" target=_blank>Zekert N, <i>et al.</i> (2010 May)</a>. Interaction of the Aspergillus nidulans microtubule-organizing center (MTOC) component ApsB with gamma-tubulin and evidence for a role of a subclass of peroxisomes in the formation of septal MTOCs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835019" target=_blank>Flipphi M, <i>et al.</i> (2014 Jul)</a>. The Aspergillus nidulans acuL gene encodes a mitochondrial carrier required for the utilization of carbon sources that are metabolized via the TCA cycle.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00970_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00970_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00970_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00970_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00448_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00971<br>EOG8N02WM<br>MLS1.2<br>sce:absent<br><br>Genes: 14
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Peroxisomal malate synthase<br><br>
			<br><b>Features</b><br>[x]<br><br>
			<br><b>Parent</b><br>ortholog:FOG00970<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00971_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00971_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00971_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00971_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00448_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00972<br>EOG8N02WM<br>MSL1<br>sce:MLS1<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cytoplasmic malate synthase, non-fermentable carbon sources; cytosolic but sce ortholog can localize in [x] when grown on oleate<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>Parent</b><br>paralog:FOG00970<br><br>
			<br><b>SGD Description</b><br>Malate synthase, enzyme of the glyoxylate cycle; involved in utilization of non-fermentable carbon sources; expression is subject to carbon catabolite repression; localizes in peroxisomes during growth on oleic acid, otherwise cytosolic; can accept butyryl-CoA as acyl-CoA donor in addition to traditional substrate acetyl-CoA<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1454530" target=_blank>Hartig A, <i>et al.</i> (1992 Nov 11)</a>. Differentially regulated malate synthase genes participate in carbon and nitrogen metabolism of S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16103919" target=_blank>Fay JC, <i>et al.</i> (2005 Jul)</a>. Evidence for domesticated and wild populations of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18780730" target=_blank>Sinha H, <i>et al.</i> (2008 Nov)</a>. Sequential elimination of major-effect contributors identifies additional quantitative trait loci conditioning high-temperature growth in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00972_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00972_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00972_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00972_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00448_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00973<br>EOG8N02WM<br>DAL7<br>sce:DAL7<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>DAL7, nitrogenous compound allantoin<br><br>
			<br><b>Parent</b><br>onholog:FOG00972<br><br>
			<br><b>SGD Description</b><br>Malate synthase; can accept butyryl-CoA as acyl-CoA donor in addition to traditional substrate acetyl-CoA; recycles glyoxylate generated during allantoin degradation; expression sensitive to nitrogen catabolite repression and induced by allophanate, an intermediate in allantoin degradation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2552287" target=_blank>Yoo HS, <i>et al.</i> (1989 Aug)</a>. The DAL7 promoter consists of multiple elements that cooperatively mediate regulation of the gene's expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1454530" target=_blank>Hartig A, <i>et al.</i> (1992 Nov 11)</a>. Differentially regulated malate synthase genes participate in carbon and nitrogen metabolism of S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8462696" target=_blank>Fernandez E, <i>et al.</i> (1993 Apr 12)</a>. Two structural genes are encoding malate synthase isoenzymes in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00973_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00973_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00973_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00973_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00448_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00448.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>