<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG02811.html">HOG02811&emsp;&emsp;</a><br> 
																	HOG02812&emsp;&emsp;<br> 
																	<a href="HOG02813.html">HOG02813&emsp;&emsp;BRE1 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG02812.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG02812_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG02812_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG02812_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG02812_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG06554<br>EOG8MKKXH<br><br>sce:DCP2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Catalytic subunit of the Dcp1p-Dcp2p decapping enzyme complex; removes the 5' cap structure from mRNAs prior to their degradation; also enters the nucleus and positively regulates transcription initiation; nudix hydrolase family member; forms cytoplasmic foci upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>mRNA decapping complex catalytic subunit Dcp2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytoplasmic mRNA processing body localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7883168" target=_blank>He F, <i>et al.</i> (1995 Feb 15)</a>. Identification of a novel component of the nonsense-mediated mRNA decay pathway by use of an interacting protein screen.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10508173" target=_blank>Dunckley T, <i>et al.</i> (1999 Oct 1)</a>. The DCP2 protein is required for mRNA decapping in Saccharomyces cerevisiae and contains a functional MutT motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10900456" target=_blank>Fromont-Racine M, <i>et al.</i> (2000 Jun 30)</a>. Genome-wide protein interaction screens reveal functional networks involving Sm-like proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139489" target=_blank>Dunckley T, <i>et al.</i> (2001 Jan)</a>. Two related proteins, Edc1p and Edc2p, stimulate mRNA decapping in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741542" target=_blank>Tharun S, <i>et al.</i> (2001 Nov)</a>. Targeting an mRNA for decapping: displacement of translation factors and association of the Lsm1p-7p complex on deadenylated yeast mRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12554866" target=_blank>Steiger M, <i>et al.</i> (2003 Feb)</a>. Analysis of recombinant yeast decapping enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730603" target=_blank>Sheth U, <i>et al.</i> (2003 May 2)</a>. Decapping and decay of messenger RNA occur in cytoplasmic processing bodies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14758354" target=_blank>She M, <i>et al.</i> (2004 Mar)</a>. Crystal structure of Dcp1p and its functional implications in mRNA decapping.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18280238" target=_blank>Deshmukh MV, <i>et al.</i> (2008 Feb 15)</a>. mRNA decapping is promoted by an RNA-binding channel in Dcp2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18780730" target=_blank>Sinha H, <i>et al.</i> (2008 Nov)</a>. Sequential elimination of major-effect contributors identifies additional quantitative trait loci conditioning high-temperature growth in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18678652" target=_blank>Ling SH, <i>et al.</i> (2008 Oct)</a>. Crystal structure of human Edc3 and its functional implications.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26098573" target=_blank>Hu G, <i>et al.</i> (2015 Jul)</a>. A conserved mechanism of TOR-dependent RCK-mediated mRNA degradation regulates autophagy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27694841" target=_blank>Charenton C, <i>et al.</i> (2016 Nov)</a>. Structure of the active form of Dcp1-Dcp2 decapping enzyme bound to m&lt;sup&gt;7&lt;/sup&gt;GDP and its Edc3 activator.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06554_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06554_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06554_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG06554_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG02812_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG06555<br>EOG8MKKXH<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN11076, A. fumigatus Af293 : Afu3g02220, Neosartorya fischeri NRRL 181 : NFIA_003350, Aspergillus wentii : Aspwe1_0171305 and Aspergillus versicolor : Aspve1_0056749<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06555_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06555_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG06555_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG06555_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG02812_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG02812.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>