<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01993.html">HOG01993&emsp;&emsp;</a><br> 
																	HOG01994&emsp;&emsp;<br> 
																	<a href="HOG01995.html">HOG01995&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01994.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01994_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01994_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01994_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG05384<br>EOG82FR3P<br><br>sce:HHT1; HHT2<br><br>Genes: 49
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>histone H3 h3.1|histone H3 h3.3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in DNA methylation, global genome nucleotide-excision repair, mitotic spindle assembly checkpoint, rRNA transcription, regulation of DNA methylation, sexual sporulation resulting in formation of a cellular spore<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24279325" target=_blank>Ravin NV, <i>et al.</i> (2013 Nov 27)</a>. Genome sequence and analysis of methylotrophic yeast Hansenula polymorpha DL1.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05384_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05384_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05384_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG05384_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01994_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG05385<br>EOG82FR3P<br><br>sce:CSE4<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Centromere protein that resembles histone H3; associated with promoters, accessible chromatin and RNA polymerase II-bound regions; phosphorylated Cse4p associates with centromeres; required for proper kinetochore function; levels regulated by E3 ubiquitin ligase Psh1p; phosphorylation of Cse4p may destabilize defective kinetochores to promote bi-orientation; ubiquitination of N terminus regulates Cse4p proteolysis for faithful chromosome segregation; human CENP-A homolog<br><br>
			<br><b>PomBase Description</b><br>centromere-specific histone H3 CENP-A<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in CENP-A containing chromatin organization and CENP-A containing nucleosome, chromosome, centromeric core domain, condensed nuclear chromosome, centromeric region localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7698647" target=_blank>Stoler S, <i>et al.</i> (1995 Mar 1)</a>. A mutation in CSE4, an essential gene encoding a novel chromatin-associated protein in yeast, causes chromosome nondisjunction and cell cycle arrest at mitosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9584087" target=_blank>Baker RE, <i>et al.</i> (1998 May)</a>. Mutations synthetically lethal with cep1 target S. cerevisiae kinetochore components.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9741625" target=_blank>Meluh PB, <i>et al.</i> (1998 Sep 4)</a>. Cse4p is a component of the core centromere of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10323865" target=_blank>Ortiz J, <i>et al.</i> (1999 May 1)</a>. A putative protein complex consisting of Ctf19, Mcm21, and Okp1 represents a missing link in the budding yeast kinetochore.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10454560" target=_blank>Keith KC, <i>et al.</i> (1999 Sep)</a>. Analysis of primary structural determinants that distinguish the centromere-specific function of histone variant Cse4p from histone H3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10499801" target=_blank>Tanaka T, <i>et al.</i> (1999 Sep 17)</a>. Identification of cohesin association sites at centromeres and along chromosome arms.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10891506" target=_blank>Glowczewski L, <i>et al.</i> (2000 Aug)</a>. Histone-histone interactions and centromere function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11063678" target=_blank>Keith KC, <i>et al.</i> (2000 Nov)</a>. CSE4 genetically interacts with the Saccharomyces cerevisiae centromere DNA elements CDE I and CDE II but not CDE III. Implications for the path of the centromere dna around a cse4p variant nucleosome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10958698" target=_blank>Chen Y, <i>et al.</i> (2000 Sep)</a>. The N terminus of the centromere H3-like protein Cse4p performs an essential function distinct from that of the histone fold domain.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11257125" target=_blank>Pearson CG, <i>et al.</i> (2001 Mar 19)</a>. Budding yeast chromosome structure and dynamics during mitosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11606525" target=_blank>Biggins S, <i>et al.</i> (2001 Oct)</a>. Genes involved in sister chromatid separation and segregation in the budding yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14581449" target=_blank>Westermann S, <i>et al.</i> (2003 Oct 27)</a>. Architecture of the budding yeast kinetochore reveals a conserved molecular core.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15590827" target=_blank>Morey L, <i>et al.</i> (2004 Dec)</a>. The histone fold domain of Cse4 is sufficient for CEN targeting and propagation of active centromeres in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15530401" target=_blank>Collins KA, <i>et al.</i> (2004 Nov 9)</a>. Proteolysis contributes to the exclusive centromere localization of the yeast Cse4/CENP-A histone H3 variant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15470243" target=_blank>Stoyan T, <i>et al.</i> (2004 Oct)</a>. Inner kinetochore of the pathogenic yeast Candida glabrata.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207811" target=_blank>Collins KA, <i>et al.</i> (2005 Dec)</a>. De novo kinetochore assembly requires the centromeric histone H3 variant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16631569" target=_blank>Bloom K, <i>et al.</i> (2006 Apr 18)</a>. The path of DNA in the kinetochore.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16715078" target=_blank>Joglekar AP, <i>et al.</i> (2006 Jun)</a>. Molecular architecture of a kinetochore-microtubule attachment site.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16966420" target=_blank>Hajra S, <i>et al.</i> (2006 Sep 11)</a>. The centromere-specific histone variant Cse4p (CENP-A) is essential for functional chromatin architecture at the yeast 2-microm circle partitioning locus and promotes equal plasmid segregation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05385_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05385_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05385_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG05385_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01994_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG05386<br>EOG82FR3P<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05386_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05386_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05386_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG05386_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01994_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01994.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>