<html>
<head><title>HOG00928</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00927.html">HOG00927&emsp;&emsp;</a><br> 
																	HOG00928&emsp;&emsp;<br> 
																	<a href="HOG00929.html">HOG00929&emsp;&emsp;inositol 1,4,5-trisphosphate 5-phosphatase family; synaptojanin family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00928.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00928_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00928_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00928_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG00928.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02140<br>EOG8C8689<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, ATPase activity, nucleoside-triphosphatase activity, nucleotide binding activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02140_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02140_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02140_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02140_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00928_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02141<br>EOG8C8689<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>ATPase, involved in cytoplasmic translational initiation (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23738021" target=_blank>Zhou X, <i>et al.</i> (2013)</a>. A genome-wide screening of potential target genes to enhance the antifungal activity of micafungin in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25963819" target=_blank>Lucena R, <i>et al.</i> (2015 May 11)</a>. Nucleocytoplasmic transport in the midzone membrane domain controls yeast mitotic spindle disassembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27918601" target=_blank>Malecki M, <i>et al.</i> (2016)</a>. Identifying genes required for respiratory growth of fission yeast.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02141_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02141_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02141_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02141_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00928_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02142<br>EOG8C8689<br><br>sce:GCN20<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Positive regulator of the Gcn2p kinase activity; forms a complex with Gcn1p; proposed to stimulate Gcn2p activation by an uncharged tRNA<br><br>
			<br><b>PomBase Description</b><br>AAA family ATPase Gcn20 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7621831" target=_blank>Vazquez de Aldana CR, <i>et al.</i> (1995 Jul 3)</a>. GCN20, a novel ATP binding cassette protein, and GCN1 reside in a complex that mediates activation of the eIF-2 alpha kinase GCN2 in amino acid-starved cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9234705" target=_blank>Marton MJ, <i>et al.</i> (1997 Aug)</a>. Evidence that GCN1 and GCN20, translational regulators of GCN4, function on elongating ribosomes in activation of eIF2alpha kinase GCN2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10775272" target=_blank>Garcia-Barrio M, <i>et al.</i> (2000 Apr 17)</a>. Association of GCN1-GCN20 regulatory complex with the N-terminus of eIF2alpha kinase GCN2 is required for GCN2 activation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101534" target=_blank>Sattlegger E, <i>et al.</i> (2000 Dec 1)</a>. Separate domains in GCN1 for binding protein kinase GCN2 and ribosomes are required for GCN2 activation in amino acid-starved cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15722345" target=_blank>Sattlegger E, <i>et al.</i> (2005 Apr 22)</a>. Polyribosome binding by GCN1 is required for full activation of eukaryotic translation initiation factor 2{alpha} kinase GCN2 during amino acid starvation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02142_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02142_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02142_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02142_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00928_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02143<br>EOG8C8689<br><br>sce:ARB1<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATPase of the ATP-binding cassette (ABC) family; involved in 40S and 60S ribosome biogenesis, has similarity to Gcn20p; shuttles from nucleus to cytoplasm, physically interacts with Tif6p, Lsg1p<br><br>
			<br><b>PomBase Description</b><br>ribosome biogenesis ATPase, Arb family ABCF2-like (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative ABC transporter<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16260602" target=_blank>Dong J, <i>et al.</i> (2005 Nov)</a>. The novel ATP-binding cassette protein ARB1 is a shuttling factor that stimulates 40S and 60S ribosome biogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02143_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02143_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02143_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02143_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00928_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG02144<br>EOG8C8689<br><br>sce:NEW1<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATP binding cassette protein; cosediments with polysomes and is required for biogenesis of the small ribosomal subunit; Asn/Gln-rich rich region supports [NU+] prion formation and susceptibility to [PSI+] prion induction<br><br>
			<br><b>PomBase Description</b><br>AAA family ATPase Elf1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP binding, ATPase activity, mRNA binding activity and role in poly(A)+ mRNA export from nucleus, ribosomal small subunit biogenesis<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9020838" target=_blank>Decottignies A, <i>et al.</i> (1997 Feb)</a>. Complete inventory of the yeast ABC proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10660050" target=_blank>Santoso A, <i>et al.</i> (2000 Jan 21)</a>. Molecular basis of a yeast prion species barrier.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11511345" target=_blank>Derkatch IL, <i>et al.</i> (2001 Jul 27)</a>. Prions affect the appearance of other prions: the story of [PIN(+)].<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11511346" target=_blank>Osherovich LZ, <i>et al.</i> (2001 Jul 27)</a>. Multiple Gln/Asn-rich prion domains confer susceptibility to induction of the yeast [PSI(+)] prion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02144_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02144_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02144_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02144_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00928_phobius.pdf">Phobius transmembrane predictions</a>
										<br>6 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG02145<br>EOG8C8689<br><br>sce:YEF3;HEF3<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Translation elongation factor 3; contains two ABC cassettes; binds and hydrolyzes ATP; YEF3 has a paralog, HEF3, that arose from the whole genome duplication|Translational elongation factor EF-3; member of the ABC superfamily; stimulates EF-1 alpha-dependent binding of aminoacyl-tRNA by the ribosome; normally expressed in zinc deficient cells; HEF3 has a paralog, YEF3, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>translation elongation factor eEF3<br><br>
			<br><b>AspGD Description</b><br>Protein similar to translation elongation factor 3<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6456269" target=_blank>Dasmahapatra B, <i>et al.</i> (1981 Oct 10)</a>. Protein synthesis in yeast. I. Purification and properties of elongation factor 3 from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1976386" target=_blank>Sandbaken M, <i>et al.</i> (1990 Aug 27)</a>. Isolation and characterization of the structural gene encoding elongation factor 3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404974" target=_blank>Qin SL, <i>et al.</i> (1990 Feb 5)</a>. Sequence analysis of the translational elongation factor 3 from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203789" target=_blank>Sandbaken MG, <i>et al.</i> (1990 Sep 15)</a>. Protein synthesis in yeast. Structural and functional analysis of the gene encoding elongation factor 3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1584022" target=_blank>Colthurst DR, <i>et al.</i> (1992 Apr)</a>. Elongation factor 3 (EF-3) from Candida albicans shows both structural and functional similarity to EF-3 from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1579463" target=_blank>Myers KK, <i>et al.</i> (1992 Apr 11)</a>. Isolation and sequence analysis of the gene for translation elongation factor 3 from Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1626427" target=_blank>Di Domenico BJ, <i>et al.</i> (1992 May)</a>. Isolation and sequence analysis of the gene encoding translation elongation factor 3 from Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957240" target=_blank>Kovalchuke O, <i>et al.</i> (1994 Nov 15)</a>. Comparative analysis of ribosome-associated adenosinetriphosphatase (ATPase) from pig liver and the ATPase of elongation factor 3 from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7657623" target=_blank>Triana-Alonso FJ, <i>et al.</i> (1995 Sep 1)</a>. The elongation factor 3 unique in higher fungi and essential for protein biosynthesis is an E site factor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9298649" target=_blank>Garrels JI, <i>et al.</i> (1997 Aug)</a>. Proteome studies of Saccharomyces cerevisiae: identification and characterization of abundant proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9553076" target=_blank>Gontarek RR, <i>et al.</i> (1998 Apr 24)</a>. The N terminus of eukaryotic translation elongation factor 3 interacts with 18 S rRNA and 80 S ribosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990316" target=_blank>Kovalchuke O, <i>et al.</i> (1998 Dec 15)</a>. Competition and cooperation amongst yeast elongation factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9544245" target=_blank>Sarthy AV, <i>et al.</i> (1998 Feb)</a>. Identification and kinetic analysis of a functional homolog of elongation factor 3, YEF3 in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9782488" target=_blank>Nakayama H, <i>et al.</i> (1998 Sep)</a>. A controllable gene-expression system for the pathogenic fungus Candida glabrata.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9778796" target=_blank>Maurice TC, <i>et al.</i> (1998 Sep 15)</a>. A highly conserved intraspecies homolog of the Saccharomyces cerevisiae elongation factor-3 encoded by the HEF3 gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10361693" target=_blank>Uritani M, <i>et al.</i> (1999 Apr)</a>. Detection and analysis of translation elongation factor 3 genes from various yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10523624" target=_blank>Futcher B, <i>et al.</i> (1999 Nov)</a>. A sampling of the yeast proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11452028" target=_blank>Taricani L, <i>et al.</i> (2001 Jul 15)</a>. Expression of hsp16 in response to nucleotide depletion is regulated via the spc1 MAPK pathway in Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12493761" target=_blank>Anand M, <i>et al.</i> (2003 Feb 28)</a>. Functional interactions between yeast translation eukaryotic elongation factor (eEF) 1A and eEF3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15378761" target=_blank>Pitarch A, <i>et al.</i> (2004 Oct)</a>. Proteomics-based identification of novel Candida albicans antigens for diagnosis of systemic candidiasis in patients with underlying hematological malignancies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18065650" target=_blank>Asp E, <i>et al.</i> (2008 Feb)</a>. Fission yeast mitogen-activated protein kinase Sty1 interacts with translation factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20976105" target=_blank>Anderson HE, <i>et al.</i> (2010 Oct 18)</a>. Silencing mediated by the Schizosaccharomyces pombe HIRA complex is dependent upon the Hpc2-like protein, Hip4.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522802" target=_blank>Couttas TA, <i>et al.</i> (2012 Apr)</a>. Methylation of translation-associated proteins in Saccharomyces cerevisiae: Identification of methylated lysines and their methyltransferases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22888004" target=_blank>Visweswaraiah J, <i>et al.</i> (2012 Nov 2)</a>. Overexpression of eukaryotic translation elongation factor 3 impairs Gcn2 protein activation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24818994" target=_blank>Jongjitwimol J, <i>et al.</i> (2014)</a>. The S. pombe translation initiation factor eIF4G is Sumoylated and associates with the SUMO protease Ulp2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26167880" target=_blank>Lipp JJ, <i>et al.</i> (2015 Aug)</a>. SR protein kinases promote splicing of nonconsensus introns.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25963819" target=_blank>Lucena R, <i>et al.</i> (2015 May 11)</a>. Nucleocytoplasmic transport in the midzone membrane domain controls yeast mitotic spindle disassembly.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02145_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02145_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02145_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02145_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00928_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00928.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>