<html>
<head><title>HOG00781</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00780.html">HOG00780&emsp;&emsp;UDPGP type 1 family</a><br> 
																	HOG00781&emsp;&emsp;<br> 
																	<a href="HOG00782.html">HOG00782&emsp;&emsp;chitin synthase family. Class IV subfamily</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00781.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00781_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00781_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00781_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00781_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG00781.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01831<br>EOG86WWQH<br><br>sce:absent<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative chitin synthase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01831_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01831_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01831_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01831_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00781_phobius.pdf">Phobius transmembrane predictions</a>
										<br>32 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01832<br>EOG86WWQH<br><br>sce:CHS2<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Chitin synthase II; catalyzes transfer of N-acetylglucosamine (GlcNAc) to chitin upon activation of zymogenic form; required for chitin synthesis in the primary septum during cytokinesis; localization regulated by Cdk1p during mitosis; phosphorylation by Dbf2p kinase regulates its dynamics and chitin synthesis during cytokinesis<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2533436" target=_blank>Silverman SJ, <i>et al.</i> (1989 Nov-Dec)</a>. Similar and different domains of chitin synthases 1 and 2 of S. cerevisiae: two isozymes with distinct functions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7775457" target=_blank>Nagahashi S, <i>et al.</i> (1995 Jun 9)</a>. Characterization of chitin synthase 2 of Saccharomyces cerevisiae. Implication of two highly conserved domains as possible catalytic sites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01832_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01832_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01832_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01832_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00781_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG01833<br>EOG86WWQH<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>chitin synthase homolog Chs2<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01833_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01833_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01833_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01833_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00781_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG01834<br>EOG86WWQH<br><br>sce:absent<br><br>Genes: 10
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative chitin synthase|Putative chitin synthase, class III; induced by caspofungin|Putative chitin synthase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765508" target=_blank>Yanai K, <i>et al.</i> (1994 Oct)</a>. Isolation and characterization of two chitin synthase genes from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8953267" target=_blank>Borgia PT, <i>et al.</i> (1996 Sep)</a>. The chsB gene of Aspergillus nidulans is necessary for normal hyphal growth and development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9141670" target=_blank>Tatsuno K, <i>et al.</i> (1997 Apr 15)</a>. Properties of yeast expressed Aspergillus nidulans chitin synthase B which is essential for hyphal growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913868" target=_blank>Sreenivasaprasad S, <i>et al.</i> (2000 Aug 1)</a>. Cloning and characterisation of a chitin synthase gene cDNA from the cultivated mushroom Agaricus bisporus and its expression during morphogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689191" target=_blank>Dessen P, <i>et al.</i> (2000 Feb 22)</a>. The PAUSE software for analysis of translational control over protein targeting: application to E. nidulans membrane proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12480906" target=_blank>Müller C, <i>et al.</i> (2002 Dec)</a>. Altering the expression of two chitin synthase genes differentially affects the growth and morphology of Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11988507" target=_blank>Ichinomiya M, <i>et al.</i> (2002 May)</a>. Repression of chsB expression reveals the functional importance of class IV chitin synthase gene chsD in hyphal growth and conidiation of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12420146" target=_blank>Ichinomiya M, <i>et al.</i> (2002 Oct)</a>. Different functions of the class I and class II chitin synthase genes, chsC and chsA, are revealed by repression of chsB expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14732266" target=_blank>Sims AH, <i>et al.</i> (2004 Feb)</a>. Use of expressed sequence tag analysis and cDNA microarrays of the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15121085" target=_blank>Lee JI, <i>et al.</i> (2004 Jun)</a>. Differential expression of the chitin synthase genes of Aspergillus nidulans, chsA, chsB, and chsC, in response to developmental status and environmental factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17601879" target=_blank>Fujioka T, <i>et al.</i> (2007 Aug)</a>. MpkA-Dependent and -independent cell wall integrity signaling in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17409079" target=_blank>Li S, <i>et al.</i> (2007 May)</a>. basA regulates cell wall organization and asexual/sexual sporulation ratio in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18471095" target=_blank>Spielvogel A, <i>et al.</i> (2008 Sep 15)</a>. Two zinc finger transcription factors, CrzA and SltA, are involved in cation homoeostasis and detoxification in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18651309" target=_blank>Horiuchi H, <i>et al.</i> (2009)</a>. Functional diversity of chitin synthases of Aspergillus nidulans in hyphal growth, conidiophore development and septum formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411617" target=_blank>Fukuda K, <i>et al.</i> (2009 Jul)</a>. Class III chitin synthase ChsB of Aspergillus nidulans localizes at the sites of polarized cell wall synthesis and is required for conidial development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19585695" target=_blank>de Groot PW, <i>et al.</i> (2009 Mar)</a>. Comprehensive genomic analysis of cell wall genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21462551" target=_blank>Li S, <i>et al.</i> (2009 Mar-Apr)</a>. Induction of cell wall thickening by the antifungal compound dihydromaltophilin disrupts fungal growth and is mediated by sphingolipid biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23365684" target=_blank>Yoshimi A, <i>et al.</i> (2013)</a>. Functional analysis of the α-1,3-glucan synthase genes agsA and agsB in Aspergillus nidulans: agsB is the major α-1,3-glucan synthase in this fungus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23154970" target=_blank>Szilágyi M, <i>et al.</i> (2013 Jan)</a>. Transcriptome changes initiated by carbon starvation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25955346" target=_blank>Takeshita N, <i>et al.</i> (2015)</a>. Transportation of Aspergillus nidulans Class III and V Chitin Synthases to the Hyphal Tips Depends on Conventional Kinesin.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01834_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01834_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01834_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01834_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00781_phobius.pdf">Phobius transmembrane predictions</a>
										<br>10 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG01835<br>EOG86WWQH<br><br>sce:CHS1<br><br>Genes: 52
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Chitin synthase I; requires activation from zymogenic form in order to catalyze the transfer of N-acetylglucosamine (GlcNAc) to chitin; required for repairing the chitin septum during cytokinesis; transcription activated by mating factor<br><br>
			<br><b>PomBase Description</b><br>chitin synthase I<br><br>
			<br><b>AspGD Description</b><br>Chitin synthase class II<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2941152" target=_blank>Bulawa CE, <i>et al.</i> (1986 Jul 18)</a>. The S. cerevisiae structural gene for chitin synthase is not required for chitin synthesis in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765719" target=_blank>Motoyama T, <i>et al.</i> (1994 Dec)</a>. Isolation of a chitin synthase gene (chsC) of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709948" target=_blank>Motoyama T, <i>et al.</i> (1996 Jun 24)</a>. The Aspergillus nidulans genes chsA and chsD encode chitin synthases which have redundant functions in conidia formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10524253" target=_blank>Hintz WE, <i>et al.</i> (1999 Sep 3)</a>. Sequence analysis of the chitin synthase A gene of the Dutch elm pathogen Ophiostoma novo-ulmi indicates a close association with the human pathogen Sporothrix schenckii.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10731706" target=_blank>Fujiwara M, <i>et al.</i> (2000 Mar)</a>. Evidence that the Aspergillus nidulans class I and class II chitin synthase genes, chsC and chsA, share critical roles in hyphal wall integrity and conidiophore development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11705928" target=_blank>Wang Z, <i>et al.</i> (2001 Dec)</a>. WdChs2p, a class I chitin synthase, together with WdChs3p (class III) contributes to virulence in Wangiella (Exophiala) dermatitidis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12480906" target=_blank>Müller C, <i>et al.</i> (2002 Dec)</a>. Altering the expression of two chitin synthase genes differentially affects the growth and morphology of Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12420146" target=_blank>Ichinomiya M, <i>et al.</i> (2002 Oct)</a>. Different functions of the class I and class II chitin synthase genes, chsC and chsA, are revealed by repression of chsB expression in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670687" target=_blank>Park BC, <i>et al.</i> (2003 Mar 28)</a>. Activation of chsC transcription by AbaA during asexual development of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15121085" target=_blank>Lee JI, <i>et al.</i> (2004 Jun)</a>. Differential expression of the chitin synthase genes of Aspergillus nidulans, chsA, chsB, and chsC, in response to developmental status and environmental factors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665472" target=_blank>Yamada E, <i>et al.</i> (2005 Jan)</a>. The class V chitin synthase gene csmA is crucial for the growth of the chsA chsC double mutant in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15947204" target=_blank>Ichinomiya M, <i>et al.</i> (2005 Jun)</a>. Class I and class II chitin synthases are involved in septum formation in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16082523" target=_blank>Ichinomiya M, <i>et al.</i> (2005 Sep)</a>. Expression of asexual developmental regulator gene abaA is affected in the double mutants of classes I and II chitin synthase genes, chsC and chsA, of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17601879" target=_blank>Fujioka T, <i>et al.</i> (2007 Aug)</a>. MpkA-Dependent and -independent cell wall integrity signaling in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18651309" target=_blank>Horiuchi H, <i>et al.</i> (2009)</a>. Functional diversity of chitin synthases of Aspergillus nidulans in hyphal growth, conidiophore development and septum formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19585695" target=_blank>de Groot PW, <i>et al.</i> (2009 Mar)</a>. Comprehensive genomic analysis of cell wall genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21462551" target=_blank>Li S, <i>et al.</i> (2009 Mar-Apr)</a>. Induction of cell wall thickening by the antifungal compound dihydromaltophilin disrupts fungal growth and is mediated by sphingolipid biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23365684" target=_blank>Yoshimi A, <i>et al.</i> (2013)</a>. Functional analysis of the α-1,3-glucan synthase genes agsA and agsB in Aspergillus nidulans: agsB is the major α-1,3-glucan synthase in this fungus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23450714" target=_blank>Krijgsheld P, <i>et al.</i> (2013 Mar 15)</a>. Development in Aspergillus.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01835_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01835_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01835_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01835_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00781_phobius.pdf">Phobius transmembrane predictions</a>
										<br>49 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00781.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>