<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01517.html">HOG01517&emsp;&emsp;</a><br> 
																	HOG01518&emsp;&emsp;<br> 
																	<a href="HOG01519.html">HOG01519&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01518.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01518_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01518_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01518_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG04343<br>EOG82BVSN<br>EOG8WH74N<br><br>sce:SUR7<br><br>Genes: 30
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Plasma membrane protein, component of eisosomes; long-lived protein that remains stable in eisosomes of mother cells while other eisosome proteins, Pil1p and Lsp1p, turn over; may function to anchor the eisosome in place; sporulation and plasma membrane sphingolipid content are altered in mutants; localizes to furrow-like invaginations (MCC patches)<br><br>
			<br><b>AspGD Description</b><br>Protein similar to multicopy suppressor Sur7p; expression repressed by tunicamycin and DTT<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9219339" target=_blank>Sivadon P, <i>et al.</i> (1997 Jun 30)</a>. Cloning of the multicopy suppressor gene SUR7: evidence for a functional relationship between the yeast actin-binding protein Rvs167 and a putative membranous protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12469340" target=_blank>Navarre C, <i>et al.</i> (2002 Dec)</a>. Subproteomics: identification of plasma membrane proteins from the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11784867" target=_blank>Young ME, <i>et al.</i> (2002 Feb)</a>. The Sur7p family defines novel cortical domains in Saccharomyces cerevisiae, affects sphingolipid metabolism, and is involved in sporulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15536122" target=_blank>Malinska K, <i>et al.</i> (2004 Dec 1)</a>. Distribution of Can1p into stable domains reflects lateral protein segregation within the plasma membrane of living S. cerevisiae cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16496001" target=_blank>Walther TC, <i>et al.</i> (2006 Feb 23)</a>. Eisosomes mark static sites of endocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18799621" target=_blank>Alvarez FJ, <i>et al.</i> (2008 Dec)</a>. The Sur7 protein regulates plasma membrane organization and prevents intracellular cell wall growth in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18923548" target=_blank>Silva EM, <i>et al.</i> (2008 Oct)</a>. Identification of genes differentially expressed in a strain of the mold Aspergillus nidulans carrying a loss-of-function mutation in the palA gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19824013" target=_blank>Cabezón V, <i>et al.</i> (2009 Oct)</a>. Analysis of Candida albicans plasma membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20433738" target=_blank>Bernardo SM, <i>et al.</i> (2010 Apr 30)</a>. Candida albicans SUR7 contributes to secretion, biofilm formation, and macrophage killing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20693301" target=_blank>Vangelatos I, <i>et al.</i> (2010 Oct)</a>. Eisosome organization in the filamentous ascomycete Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622905" target=_blank>Sorgo AG, <i>et al.</i> (2011 Aug)</a>. Effects of fluconazole on the secretome, the wall proteome, and wall integrity of the clinical fungus Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21509182" target=_blank>Scazzocchio C, <i>et al.</i> (2011 Jan)</a>. Eisosomes and membrane compartments in the ascomycetes: A view from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115741" target=_blank>Wang HX, <i>et al.</i> (2011 Jan)</a>. The Candida albicans Sur7 protein is needed for proper synthesis of the fibrillar component of the cell wall that confers strength.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22202230" target=_blank>Douglas LM, <i>et al.</i> (2012)</a>. Sur7 promotes plasma membrane organization and is needed for resistance to stressful conditions and to the invasive growth and virulence of Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23395641" target=_blank>Athanasopoulos A, <i>et al.</i> (2013 Apr)</a>. Eisosome distribution and localization in the meiotic progeny of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23490206" target=_blank>She X, <i>et al.</i> (2013 Sep)</a>. Cell surface changes in the Candida albicans mitochondrial mutant goa1Δ are associated with reduced recognition by innate immune cells.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26468899" target=_blank>Athanasopoulos A, <i>et al.</i> (2015 Oct 15)</a>. Characterization of AnNce102 and its role in eisosome stability and sphingolipid biosynthesis.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04343_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04343_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04343_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04343_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01518_phobius.pdf">Phobius transmembrane predictions</a>
										<br>30 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG04344<br>EOG880GDV<br>EOG8WH74N<br><br>sce:FMP45;<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Integral membrane protein localized to mitochondria; required for sporulation and maintaining sphingolipid content; similar to SUR7; FMP45 has a paralog, YNL194C, that arose from the whole genome duplication|Integral membrane protein; required for sporulation and plasma membrane sphingolipid content; similar to SUR7; GFP-fusion protein is induced in response to the DNA-damaging agent MMS; GFP-fusion protein is more abundant at MCCs (membrane compartment occupied by Can1) in the presence of glycerol and oleate; YNL194C has a paralog, FMP45, that arose from the whole genome duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11784867" target=_blank>Young ME, <i>et al.</i> (2002 Feb)</a>. The Sur7p family defines novel cortical domains in Saccharomyces cerevisiae, affects sphingolipid metabolism, and is involved in sporulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12868060" target=_blank>Runner VM, <i>et al.</i> (2003 Jul 30)</a>. A genetic screen for yeast genes induced by sustained osmotic stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16847258" target=_blank>Kim H, <i>et al.</i> (2006 Jul 25)</a>. A global topology map of the Saccharomyces cerevisiae membrane proteome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17334841" target=_blank>Liu X, <i>et al.</i> (2007 Apr)</a>. Genetic and comparative transcriptome analysis of bromodomain factor 1 in the salt stress response of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17078969" target=_blank>White MA, <i>et al.</i> (2007 Jan 19)</a>. Characteristics affecting expression and solubilization of yeast membrane proteins.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04344_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04344_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04344_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04344_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01518_phobius.pdf">Phobius transmembrane predictions</a>
										<br>21 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG04345<br>EOG8WH74N<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04345_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04345_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04345_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04345_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01518_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01518.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>