<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01723.html">HOG01723&emsp;&emsp;</a><br> 
																	HOG01724&emsp;&emsp;<br> 
																	<a href="HOG01725.html">HOG01725&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01724.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01724_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01724_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01724_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG04813<br>EOG8JDFR0<br><br>sce:MKK1;MKK2<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>MAPKK involved in the protein kinase C signaling pathway; involved in control of cell integrity; upon activation by Bck1p phosphorylates downstream target, Slt2p; functionally redundant with Mkk2p; MKK1 has a paralog, MKK2, that arose from the whole genome duplication|MAPKK involved in the protein kinase C signaling pathway; involved in control of cell integrity; upon activation by Bck1p phosphorylates downstream target, Slt2p; functionally redundant with Mkk1p; MKK2 has a paralog, MKK1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>MAP kinase kinase Pek1<br><br>
			<br><b>AspGD Description</b><br>MAP kinase kinase (MAPKK) with a predicted role in the cell wall integrity pathway; induced by caspofungin<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8386320" target=_blank>Irie K, <i>et al.</i> (1993 May)</a>. MKK1 and MKK2, which encode Saccharomyces cerevisiae mitogen-activated protein kinase-kinase homologs, function in the pathway mediated by protein kinase C.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15905473" target=_blank>Zhang Z, <i>et al.</i> (2005)</a>. Mapping of transcription start sites in Saccharomyces cerevisiae using 5' SAGE.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19585695" target=_blank>de Groot PW, <i>et al.</i> (2009 Mar)</a>. Comprehensive genomic analysis of cell wall genes in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04813_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04813_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04813_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04813_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01724_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG04814<br>EOG8JDFR0<br><br>sce:PBS2<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>MAP kinase kinase of the HOG signaling pathway; activated under severe osmotic stress; mitophagy-specific regulator; plays a role in regulating Ty1 transposition<br><br>
			<br><b>PomBase Description</b><br>MAP kinase kinase Wis1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have MAP kinase kinase activity, MAP-kinase scaffold activity, non-membrane spanning protein tyrosine kinase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3039511" target=_blank>Boguslawski G, <i>et al.</i> (1987 Aug)</a>. Complete nucleotide sequence of a gene conferring polymyxin B resistance on yeast: similarity of the predicted polypeptide to protein kinases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1479360" target=_blank>Boguslawski G, <i>et al.</i> (1992 Nov)</a>. PBS2, a yeast gene encoding a putative protein kinase, interacts with the RAS2 pathway and affects osmotic sensitivity of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7681220" target=_blank>Brewster JL, <i>et al.</i> (1993 Mar 19)</a>. An osmosensing signal transduction pathway in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814365" target=_blank>Kahn RA, <i>et al.</i> (1995 Jan 6)</a>. Mutational analysis of Saccharomyces cerevisiae ARF1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7624781" target=_blank>Maeda T, <i>et al.</i> (1995 Jul 28)</a>. Activation of yeast PBS2 MAPKK by MAPKKKs or by binding of an SH3-containing osmosensor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10970855" target=_blank>Raitt DC, <i>et al.</i> (2000 Sep 1)</a>. Yeast Cdc42 GTPase and Ste20 PAK-like kinase regulate Sho1-dependent activation of the Hog1 MAPK pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11918796" target=_blank>Han KH, <i>et al.</i> (2002 Mar)</a>. Osmotic stress-coupled maintenance of polar growth in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14685261" target=_blank>Mapes J, <i>et al.</i> (2004 Jan 28)</a>. Nbp2 targets the Ptc1-type 2C Ser/Thr phosphatase to the HOG MAPK pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256499" target=_blank>Cullen PJ, <i>et al.</i> (2004 Jul 15)</a>. A signaling mucin at the head of the Cdc42- and MAPK-dependent filamentous growth pathway in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15200958" target=_blank>Marles JA, <i>et al.</i> (2004 Jun 18)</a>. Protein-protein interaction affinity plays a crucial role in controlling the Sho1p-mediated signal transduction pathway in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15882418" target=_blank>Furukawa K, <i>et al.</i> (2005 Jun)</a>. Aspergillus nidulans HOG pathway is activated only by two-component signalling pathway in response to osmotic stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18703157" target=_blank>Miskei M, <i>et al.</i> (2009 Mar)</a>. Annotation of stress-response proteins in the aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04814_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04814_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04814_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04814_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01724_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG04815<br>EOG8JDFR0<br><br>sce:STE7<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Signal transducing MAP kinase kinase; involved in pheromone response where it phosphorylates Fus3p; involved in the pseudohyphal/invasive growth pathway where it phosphorylates of Kss1p; phosphorylated by Ste11p; degraded by ubiquitin pathway<br><br>
			<br><b>PomBase Description</b><br>MAP kinase kinase Byr1<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have MAP kinase kinase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3532111" target=_blank>Teague MA, <i>et al.</i> (1986 Oct)</a>. Nucleotide sequence of the yeast regulatory gene STE7 predicts a protein homologous to protein kinases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1628833" target=_blank>Cairns BR, <i>et al.</i> (1992 Jul)</a>. Order of action of components in the yeast pheromone response pathway revealed with a dominant allele of the STE11 kinase and the multiple phosphorylation of the STE7 kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8131746" target=_blank>Zheng CF, <i>et al.</i> (1994 Mar 1)</a>. Activation of MEK family kinases requires phosphorylation of two conserved Ser/Thr residues.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8544826" target=_blank>Clark KL, <i>et al.</i> (1995 Dec 20)</a>. Constitutive activation of the Saccharomyces cerevisiae mating response pathway by a MAP kinase kinase from Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22829779" target=_blank>Bayram Ã–, <i>et al.</i> (2012)</a>. The Aspergillus nidulans MAPK module AnSte11-Ste50-Ste7-Fus3 controls development and secondary metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22091779" target=_blank>Dyer PS, <i>et al.</i> (2012 Jan)</a>. Sexual development and cryptic sexuality in fungi: insights from Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23505451" target=_blank>De Souza CP, <i>et al.</i> (2013)</a>. Functional analysis of the Aspergillus nidulans kinome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23084572" target=_blank>Gibbons JG, <i>et al.</i> (2013 Jan)</a>. The function and evolution of the Aspergillus genome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04815_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04815_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG04815_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG04815_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01724_phobius.pdf">Phobius transmembrane predictions</a>
										<br>7 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01724.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>