<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01824.html">HOG01824&emsp;&emsp;</a><br> 
																	HOG01825&emsp;&emsp;<br> 
																	<a href="HOG01826.html">HOG01826&emsp;&emsp;SEN15 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01825.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01825_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01825_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01825_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG05038<br>EOG8CFXQN<br><br>sce:absent<br><br>Genes: 25
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have calcium-dependent cysteine-type endopeptidase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14301537" target=_blank>DORN G, <i>et al.</i> (1965 Feb)</a>. GENETIC ANALYSIS OF THE PHOSPHATASES IN ASPERGILLUS NIDULANS.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5852975" target=_blank>Dorn G, <i>et al.</i> (1965 Nov 26)</a>. Phosphatase mutants in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5111886" target=_blank>Purnell DM, <i>et al.</i> (1971 Mar)</a>. Aspergillus nidulans: association of certain alkaline phosphatase mutations with decreased virulence for mice.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3016485" target=_blank>Caddick MX, <i>et al.</i> (1986 May)</a>. Regulation of gene expression by pH of the growth medium in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2037230" target=_blank>Shah AJ, <i>et al.</i> (1991 Jan 15)</a>. pH regulation of penicillin production in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7499363" target=_blank>Denison SH, <i>et al.</i> (1995 Dec 1)</a>. Signaling of ambient pH in Aspergillus involves a cysteine protease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7882981" target=_blank>Tilburn J, <i>et al.</i> (1995 Feb 15)</a>. The Aspergillus PacC zinc finger transcription factor mediates regulation of both acid- and alkaline-expressed genes by ambient pH.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7581026" target=_blank>Maccheroni Jùnior W, <i>et al.</i> (1995 Jan)</a>. Does gene palB regulate the transcription or the post-translational modification of Pi-repressible phosphatases of Aspergillus nidulans?<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9396712" target=_blank>Sorimachi H, <i>et al.</i> (1997 Dec 15)</a>. Structure and physiological function of calpains.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16232641" target=_blank>Futai E, <i>et al.</i> (1999)</a>. Aspergillus oryzae palBory encodes a calpain-like protease: homology to Emericella nidulans PalB and conservation of functional regions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10476033" target=_blank>Negrete-Urtasun S, <i>et al.</i> (1999 Sep)</a>. Ambient pH signal transduction in Aspergillus: completion of gene characterization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10919375" target=_blank>Denison SH, <i>et al.</i> (2000 Mar)</a>. pH regulation of gene expression in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11342116" target=_blank>Futai E, <i>et al.</i> (2001 Jan 26)</a>. Molecular cloning of PalBH, a mammalian homologue of the Aspergillus atypical calpain PalB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11861549" target=_blank>Gonzalez-Lopez CI, <i>et al.</i> (2002 Feb)</a>. Genetic control of extracellular protease synthesis in the yeast Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12208998" target=_blank>Peñalva MA, <i>et al.</i> (2002 Sep)</a>. Regulation of gene expression by ambient pH in filamentous fungi and yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12683976" target=_blank>Arst HN, <i>et al.</i> (2003 Apr)</a>. pH regulation in Aspergillus and parallels with higher eukaryotic regulatory systems.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620258" target=_blank>Nozawa SR, <i>et al.</i> (2003 Mar)</a>. Mutation in a calpain-like protease affects the posttranslational mannosylation of phosphatases in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15487944" target=_blank>Peñalva MA, <i>et al.</i> (2004)</a>. Recent advances in the characterization of ambient pH regulation of gene expression in filamentous fungi and yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15189995" target=_blank>Li M, <i>et al.</i> (2004 Jun)</a>. Candida albicans Rim13p, a protease required for Rim101p processing at acidic and alkaline pHs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16099830" target=_blank>Herranz S, <i>et al.</i> (2005 Aug 23)</a>. Arrestin-related proteins mediate pH signaling in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16963627" target=_blank>Morozov IY, <i>et al.</i> (2006 Nov)</a>. Nonsense-mediated mRNA decay mutation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416893" target=_blank>Peñas MM, <i>et al.</i> (2007 Jun)</a>. Further characterization of the signaling proteolysis step in the Aspergillus nidulans pH signal transduction pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17696968" target=_blank>Galindo A, <i>et al.</i> (2007 Oct)</a>. PalC, one of two Bro1 domain proteins in the fungal pH signalling pathway, localizes to cortical structures and binds Vps32.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18457952" target=_blank>Peñalva MA, <i>et al.</i> (2008 Jun)</a>. Ambient pH gene regulation in fungi: making connections.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19056728" target=_blank>Rodríguez-Galán O, <i>et al.</i> (2009 Feb 13)</a>. Physiological involvement in pH signaling of Vps24-mediated recruitment of Aspergillus PalB cysteine protease to ESCRT-III.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19850840" target=_blank>Hua X, <i>et al.</i> (2010 Mar)</a>. A fungal pH-responsive signaling pathway regulating Aspergillus adaptation and invasion into the cornea.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21985967" target=_blank>Trevisan GL, <i>et al.</i> (2011 Nov 4)</a>. Transcription of Aspergillus nidulans pacC is modulated by alternative RNA splicing of palB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22344261" target=_blank>Galindo A, <i>et al.</i> (2012 Apr 1)</a>. An ordered pathway for the assembly of fungal ESCRT-containing ambient pH signalling complexes at the plasma membrane.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22072140" target=_blank>Morya VK, <i>et al.</i> (2012 Jan)</a>. In silico characterization of alkaline proteases from different species of Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24265200" target=_blank>Rossi A, <i>et al.</i> (2013 Nov)</a>. Ambient pH sensing in filamentous fungi: pitfalls in elucidating regulatory hierarchical signaling networks.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25333968" target=_blank>Chinnici JL, <i>et al.</i> (2014)</a>. Neurospora crassa female development requires the PACC and other signal transduction pathways, transcription factors, chromatin remodeling, cell-to-cell fusion, and autophagy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25460796" target=_blank>Peñalva MA, <i>et al.</i> (2014 Dec)</a>. Liaison alcaline: Pals entice non-endosomal ESCRTs to  the plasma membrane for pH signaling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25841020" target=_blank>Lucena-Agell D, <i>et al.</i> (2015 Jun)</a>. Aspergillus nidulans Ambient pH Signaling Does Not Require Endocytosis.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05038_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05038_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05038_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG05038_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01825_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG05039<br>EOG8CFXQN<br><br>sce:RIM13<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Calpain-like cysteine protease; involved in proteolytic activation of Rim101p in response to alkaline pH; localizes to punctate structures in alkaline conditions and in vps4 mutant; has similarity to A. nidulans palB<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9017390" target=_blank>Li W, <i>et al.</i> (1997 Jan)</a>. Proteolytic activation of Rim1p, a positive regulator of yeast sporulation and invasive growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9928935" target=_blank>Futai E, <i>et al.</i> (1999 Jan)</a>. The protease activity of a calpain-like cysteine protease in Saccharomyces cerevisiae is required for alkaline adaptation and sporulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11283351" target=_blank>Ito T, <i>et al.</i> (2001 Apr 10)</a>. A comprehensive two-hybrid analysis to explore the yeast protein interactome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11050096" target=_blank>Lamb TM, <i>et al.</i> (2001 Jan 19)</a>. Alkaline response genes of Saccharomyces cerevisiae and their relationship to the RIM101 pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05039_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05039_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05039_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG05039_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01825_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG05040<br>EOG8CFXQN<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05040_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05040_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG05040_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG05040_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01825_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01825.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>