<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00663.html">HOG00663&emsp;&emsp;THI4 family</a><br> 
																	HOG00664&emsp;&emsp;<br> 
																	<a href="HOG00665.html">HOG00665&emsp;&emsp;Thz kinase family; thiamine-phosphate synthase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00664.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00664_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00664_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00664_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01596<br>EOG8QJQ3M<br><br>sce:THI20;THI21<br><br>Genes: 38
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Trifunctional enzyme of thiamine biosynthesis, degradation and salvage; has hydroxymethylpyrimidine (HMP) kinase, HMP-phosphate (HMP-P) kinase and thiaminase activities; member of a gene family with THI21 and THI22; HMP and HMP-P kinase activity redundant with Thi21p|Hydroxymethylpyrimidine (HMP) and HMP-phosphate kinase; involved in thiamine biosynthesis; member of a gene family with THI20 and THI22; functionally redundant with Thi20p<br><br>
			<br><b>PomBase Description</b><br>phosphomethylpyrimidine kinase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in thiamine biosynthetic process and cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5857597" target=_blank>KÃ¤fer E, <i>et al.</i> (1965 Jul)</a>. Origins of translocations in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4580921" target=_blank>Purnell DM, <i>et al.</i> (1973 Jul 31)</a>. The effects of specific auxotrophic mutations on the virulence of Aspergillus nidulans for mice.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4581424" target=_blank>Clutterbuck AJ, <i>et al.</i> (1973 Jun)</a>. Gene symbols in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383756" target=_blank>Llorente B, <i>et al.</i> (1999 Jun)</a>. Genetic redundancy and gene fusion in the genome of the Baker's yeast Saccharomyces cerevisiae: functional characterization of a three-member gene family involved in the thiamine biosynthetic pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20598542" target=_blank>Schoustra S, <i>et al.</i> (2010 Aug 10)</a>. Fitness-associated sexual reproduction in a filamentous fungus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23173672" target=_blank>Pan X, <i>et al.</i> (2012 Nov 23)</a>. Identification of novel genes involved in DNA damage response by screening a genome-wide Schizosaccharomyces pombe deletion library.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25373780" target=_blank>Graml V, <i>et al.</i> (2014 Oct 27)</a>. A genomic Multiprocess survey of machineries that control and link cell shape, microtubule organization, and cell-cycle progression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01596_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01596_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01596_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01596_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01597<br>EOG8QJQ3M<br><br>sce:THI22<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein with similarity to hydroxymethylpyrimidine phosphate kinases; member of a gene family with THI20 and THI21; not required for thiamine biosynthesis<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383756" target=_blank>Llorente B, <i>et al.</i> (1999 Jun)</a>. Genetic redundancy and gene fusion in the genome of the Baker's yeast Saccharomyces cerevisiae: functional characterization of a three-member gene family involved in the thiamine biosynthetic pathway.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01597_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01597_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01597_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01597_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG01598<br>EOG8QJQ3M<br><br>sce:DBP1;DED1<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative ATP-dependent RNA helicase of the DEAD-box protein family; mutants show reduced stability of the 40S ribosomal subunit scanning through 5' untranslated regions of mRNAs; protein abundance increases in response to DNA replication stress; DBP1 has a paralog, DED1, that arose from the whole genome duplication|ATP-dependent DEAD (Asp-Glu-Ala-Asp)-box RNA helicase; required for translation initiation of all yeast mRNAs; binds to mRNA cap-associated factors, and binding stimulates Ded1p RNA-dependent ATPase activity; mutation in human homolog DBY is associated with male infertility; human homolog DDX3X complements ded1 null mutation; DED1 has a paralog, DBP1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>translation initiation RNA helicase Sum3<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have mRNA binding, translation initiation factor activity, role in cytoplasmic translational initiation and cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3001645" target=_blank>Struhl K, <i>et al.</i> (1985 Dec 9)</a>. Nucleotide sequence and transcriptional mapping of the yeast pet56-his3-ded1 gene region.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2406722" target=_blank>Chang TH, <i>et al.</i> (1990 Feb)</a>. Identification of five putative yeast RNA helicase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1857205" target=_blank>Jamieson DJ, <i>et al.</i> (1991 Apr)</a>. A suppressor of yeast spp81/ded1 mutations encodes a very similar putative ATP-dependent RNA helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1996139" target=_blank>Jamieson DJ, <i>et al.</i> (1991 Feb 21)</a>. A suppressor of a yeast splicing mutation (prp8-1) encodes a putative ATP-dependent RNA helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7835345" target=_blank>Thuillier V, <i>et al.</i> (1995 Jan 16)</a>. A mutation in the C31 subunit of Saccharomyces cerevisiae RNA polymerase III affects transcription initiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9045610" target=_blank>Chuang RY, <i>et al.</i> (1997 Mar 7)</a>. Requirement of the DEAD-Box protein ded1p for messenger RNA translation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9144215" target=_blank>de la Cruz J, <i>et al.</i> (1997 May 13)</a>. The p20 and Ded1 proteins have antagonistic roles in eIF4E-dependent translation in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10364207" target=_blank>Iost I, <i>et al.</i> (1999 Jun 18)</a>. Ded1p, a DEAD-box protein required for translation initiation in Saccharomyces cerevisiae, is an RNA helicase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11069307" target=_blank>Noueiry AO, <i>et al.</i> (2000 Nov 21)</a>. A mutant allele of essential, general translation initiation factor DED1 selectively inhibits translation of a viral mRNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14671320" target=_blank>Kushner DB, <i>et al.</i> (2003 Dec 23)</a>. Systematic, genome-wide identification of host genes affecting replication of a positive-strand RNA virus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15064363" target=_blank>Chong JL, <i>et al.</i> (2004)</a>. Ded1p, a conserved DExD/H-box translation factor, can promote yeast L-A virus negative-strand RNA synthesis in vitro.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14763975" target=_blank>Berthelot K, <i>et al.</i> (2004 Feb)</a>. Dynamics and processivity of 40S ribosome scanning on mRNA in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15201868" target=_blank>Cordin O, <i>et al.</i> (2004 Jul 7)</a>. The newly discovered Q motif of DEAD-box RNA helicases regulates RNA-binding and helicase activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16216083" target=_blank>Yang Q, <i>et al.</i> (2005 Oct 18)</a>. ATP- and ADP-dependent modulation of RNA unwinding and strand annealing activities by the DEAD-box protein DED1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18332124" target=_blank>Banroques J, <i>et al.</i> (2008 May)</a>. A conserved phenylalanine of motif IV in superfamily 2 helicases is required for cooperative, ATP-dependent binding of RNA substrates in DEAD-box proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23865587" target=_blank>Low JK, <i>et al.</i> (2013 Sep 6)</a>. Analysis of the proteome of Saccharomyces cerevisiae for methylarginine.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01598_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01598_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01598_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01598_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG01599<br>EOG8QJQ3M<br><br>sce:DBP2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>ATP-dependent RNA helicase of the DEAD-box protein family; has a strong preference for dsRNA; interacts with YRA1; required for the assembly of Yra1p, Nab2p and Mex67p onto mRNA and formation of nuclear mRNP; involved in mRNA decay and rRNA processing; may be involved in suppression of transcription from cryptic initiation sites<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Dbp2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nucleolus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1869554" target=_blank>Dequard-Chablat M, <i>et al.</i> (1991 Aug 15)</a>. RPC19, the gene for a subunit common to yeast RNA polymerases A (I) and C (III).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1996094" target=_blank>Iggo RD, <i>et al.</i> (1991 Mar)</a>. p68 RNA helicase: identification of a nucleolar form and cloning of related genes containing a conserved intron in yeasts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7883168" target=_blank>He F, <i>et al.</i> (1995 Feb 15)</a>. Identification of a novel component of the nonsense-mediated mRNA decay pathway by use of an interacting protein screen.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11585918" target=_blank>Bond AT, <i>et al.</i> (2001 Nov)</a>. Absence of Dbp2p alters both nonsense-mediated mRNA decay and rRNA processing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12237855" target=_blank>Tallada VA, <i>et al.</i> (2002 Sep 30)</a>. Genome-wide search of Schizosaccharomyces pombe genes causing overexpression-mediated cell cycle defects.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20537132" target=_blank>Han TX, <i>et al.</i> (2010)</a>. Global fitness profiling of fission yeast deletion strains by barcode sequencing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20403971" target=_blank>Keller C, <i>et al.</i> (2010 Jun)</a>. Proteomic and functional analysis of the noncanonical poly(A) polymerase Cid14.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21436456" target=_blank>Zhang K, <i>et al.</i> (2011 Mar 25)</a>. Clr4/Suv39 and RNA quality control factors cooperate to trigger RNAi and suppress antisense RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23122962" target=_blank>Drogat J, <i>et al.</i> (2012 Nov 29)</a>. Cdk11-cyclinL controls the assembly of the RNA polymerase II mediator complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23022480" target=_blank>Shchepachev V, <i>et al.</i> (2012 Oct 25)</a>. Mpn1, mutated in poikiloderma with neutropenia protein 1, is a conserved 3'-to-5' RNA exonuclease processing U6 small nuclear RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013502" target=_blank>Wang J, <i>et al.</i> (2013 Sep 1)</a>. Epe1 recruits BET family bromodomain protein Bdf2 to establish heterochromatin boundaries.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24818994" target=_blank>Jongjitwimol J, <i>et al.</i> (2014)</a>. The S. pombe translation initiation factor eIF4G is Sumoylated and associates with the SUMO protease Ulp2.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26670050" target=_blank>Kilchert C, <i>et al.</i> (2015 Dec 22)</a>. Regulation of mRNA Levels by Decay-Promoting Introns that Recruit the Exosome Specificity Factor Mmi1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26404184" target=_blank>Nie M, <i>et al.</i> (2015 Sep 25)</a>. High Confidence Fission Yeast SUMO Conjugates Identified by Tandem Denaturing Affinity Purification.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01599_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01599_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01599_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01599_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG01600<br>EOG8QJQ3M<br><br>sce:DBP3<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>RNA-Dependent ATPase, member of DExD/H-box family; involved in cleavage of site A3 within the ITS1 spacer during rRNA processing; not essential for growth, but deletion causes severe slow-growth phenotype<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Dbp3 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATP-dependent RNA helicase activity and role in endonucleolytic cleavage in ITS1 upstream of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2406722" target=_blank>Chang TH, <i>et al.</i> (1990 Feb)</a>. Identification of five putative yeast RNA helicase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9032262" target=_blank>Weaver PL, <i>et al.</i> (1997 Mar)</a>. Dbp3p, a putative RNA helicase in Saccharomyces cerevisiae, is required for efficient pre-rRNA processing predominantly at site A3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01600_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01600_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01600_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01600_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG01601<br>EOG8QJQ3M<br><br>sce:PRP5<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>RNA helicase in the DEAD-box family; necessary for prespliceosome formation, bridges U1 and U2 snRNPs and enables stable U2 snRNP association with intron RNA<br><br>
			<br><b>PomBase Description</b><br>ATP-dependent RNA helicase Prp11<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have ATPase activity, U1 snRNP binding, U2 snRNP binding activity, role in U2-type prespliceosome assembly and U2-type prespliceosome, cytosol, mitotic spindle pole body localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2349233" target=_blank>Dalbadie-McFarland G, <i>et al.</i> (1990 Jun)</a>. PRP5: a helicase-like protein required for mRNA splicing in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8405998" target=_blank>Ruby SW, <i>et al.</i> (1993 Oct)</a>. Four yeast spliceosomal proteins (PRP5, PRP9, PRP11, and PRP21) interact to promote U2 snRNP binding to pre-mRNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065365" target=_blank>Wells SE, <i>et al.</i> (1994 Sep)</a>. Interactions between highly conserved U2 small nuclear RNA structures and Prp5p, Prp9p, Prp11p, and Prp21p proteins are required to ensure integrity of the U2 small nuclear ribonucleoprotein in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969184" target=_blank>O'Day CL, <i>et al.</i> (1996 Dec 27)</a>. The Saccharomyces cerevisiae Prp5 protein has RNA-dependent ATPase activity with specificity for U2 small nuclear RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969185" target=_blank>Wiest DK, <i>et al.</i> (1996 Dec 27)</a>. In vitro studies of the Prp9.Prp11.Prp21 complex indicate a pathway for U2 small nuclear ribonucleoprotein activation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8622683" target=_blank>Yan D, <i>et al.</i> (1996 Mar)</a>. Invariant U2 RNA sequences bordering the branchpoint recognition region are essential for interaction with yeast SF3a and SF3b subunits.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11927574" target=_blank>Abu Dayyeh BK, <i>et al.</i> (2002 Jun 7)</a>. Probing interactions between the U2 small nuclear ribonucleoprotein and the DEAD-box protein, Prp5.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14610285" target=_blank>Perriman R, <i>et al.</i> (2003 Nov 25)</a>. ATP requirement for Prp5p function is determined by Cus2p and the structure of U2 small nuclear RNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16314500" target=_blank>Wang Q, <i>et al.</i> (2005 Dec)</a>. Interactions of the yeast SF3b splicing factor.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01601_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01601_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01601_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01601_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG01602<br>EOG8QJQ3M<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted ATP binding, ATP-dependent helicase activity, helicase activity, nucleic acid binding activity<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01602_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01602_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01602_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01602_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG01603<br>EOG8QJQ3M<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>phosphomethylpyrimidine kinase (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25076038" target=_blank>Guo Y, <i>et al.</i> (2014 Jul)</a>. Large scale screening of genetic interaction with sgf73(+) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01603_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01603_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01603_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01603_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00664.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>