<html>
<head><title>HOG00203</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00202.html">HOG00202&emsp;&emsp;cofactor-independent phosphoglycerate mutase</a><br> 
																	HOG00203&emsp;&emsp;<br> 
																	<a href="HOG00204.html">HOG00204&emsp;&emsp;pyruvate kinase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00203.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00203_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00203_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00203_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00203_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG00203.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00295<br>EOG8JM664<br>ENO1<br>sce:ENO1;ENO2;ERR2;ERR3<br><br>Genes: 45
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Enolase. and local duplications; sce/vpo have ohnologs; 137-138 insertion in ctg clade starting with pta, suggets independent duplication in ctg clade?<br><br>
			<br><b>SGD Description</b><br>Enolase I, a phosphopyruvate hydratase; catalyzes conversion of 2-phosphoglycerate to phosphoenolpyruvate during glycolysis and the reverse reaction during gluconeogenesis; expression repressed in response to glucose; protein abundance increases in response to DNA replication stress; N-terminally propionylated in vivo; ENO1 has a paralog, ENO2, that arose from the whole genome duplication|Enolase II, a phosphopyruvate hydratase; catalyzes conversion of 2-phosphoglycerate to phosphoenolpyruvate during glycolysis and the reverse reaction during gluconeogenesis; expression induced in response to glucose; ENO2 has a paralog, ENO1, that arose from the whole genome duplication|Enolase, a phosphopyruvate hydratase; catalyzes the conversion of 2-phosphoglycerate to phosphoenolpyruvate; complements the growth defect of an ENO1 ENO2 double mutant|Enolase, a phosphopyruvate hydratase; catalyzes the conversion of 2-phosphoglycerate to phosphoenolpyruvate; complements the growth defect of an ENO1 ENO2 double mutant in glucose<br><br>
			<br><b>PomBase Description</b><br>enolase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Phosphopyruvate hydratase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=778333" target=_blank>Payton M, <i>et al.</i> (1976 May)</a>. Agar as a carbon source and its effect on the utilization of other carbon sources by acetate non-utilizing (acu) mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7005235" target=_blank>Chin CC, <i>et al.</i> (1981 Feb 10)</a>. The amino acid sequence of yeast enolase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6256394" target=_blank>Holland MJ, <i>et al.</i> (1981 Feb 10)</a>. The primary structures of two yeast enolase genes. Homology between the 5' noncoding flanking regions of yeast enolase and glyceraldehyde-3-phosphate dehydrogenase genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3374614" target=_blank>Lebioda L, <i>et al.</i> (1988 Jun 16)</a>. Crystal structure of enolase indicates that enolase and pyruvate kinase evolved from a common ancestor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2645275" target=_blank>Lebioda L, <i>et al.</i> (1989 Mar 5)</a>. The structure of yeast enolase at 2.25-A resolution. An 8-fold beta + alpha-barrel with a novel beta beta alpha alpha (beta alpha)6 topology.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2405163" target=_blank>Stec B, <i>et al.</i> (1990 Jan 5)</a>. Refined structure of yeast apo-enolase at 2.25 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400228" target=_blank>Sundstrom P, <i>et al.</i> (1992 Nov)</a>. Molecular cloning of cDNA and analysis of protein secondary structure of Candida albicans enolase, an abundant, immunodominant glycolytic enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8359671" target=_blank>Franklyn KM, <i>et al.</i> (1993 Jul 15)</a>. Cloning and nucleotide sequence analysis of the Candida albicans enolase gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8478328" target=_blank>Mason AB, <i>et al.</i> (1993 May)</a>. Molecular cloning and characterization of the Candida albicans enolase gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7737086" target=_blank>Norbeck J, <i>et al.</i> (1995 Jan)</a>. Gene linkage of two-dimensional polyacrylamide gel electrophoresis resolved proteins from isogene families in Saccharomyces cerevisiae by microsequencing of in-gel trypsin generated peptides.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8605183" target=_blank>Larsen TM, <i>et al.</i> (1996 Apr 9)</a>. A carboxylate oxygen of the substrate bridges the magnesium ions at the active site of enolase: structure of the yeast enzyme complexed with the equilibrium mixture of 2-phosphoglycerate and phosphoenolpyruvate at 1.8 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8634301" target=_blank>Poyner RR, <i>et al.</i> (1996 Feb 6)</a>. Toward identification of acid/base catalysts in the active site of enolase: comparison of the properties of K345A, E168Q, and E211Q variants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8935650" target=_blank>Norbeck J, <i>et al.</i> (1996 Mar 15)</a>. Protein expression during exponential growth in 0.7 M NaCl medium of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9376357" target=_blank>Zhang E, <i>et al.</i> (1997 Oct 14)</a>. Mechanism of enolase: the crystal structure of asymmetric dimer enolase-2-phospho-D-glycerate/enolase-phosphoenolpyruvate at 2.0 A resolution.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11027610" target=_blank>Brewer JM, <i>et al.</i> (2000 Oct 5)</a>. The H159A mutant of yeast enolase 1 has significant activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11502169" target=_blank>Grandier-Vazeille X, <i>et al.</i> (2001 Aug 21)</a>. Yeast mitochondrial dehydrogenases are associated in a supramolecular complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12054465" target=_blank>Poyner RR, <i>et al.</i> (2002 May 15)</a>. Functional and structural changes due to a serine to alanine mutation in the active-site flap of enolase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12846578" target=_blank>Sims PA, <i>et al.</i> (2003 Jul 15)</a>. Reverse protonation is the key to general acid-base catalysis in enolase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13678299" target=_blank>Brewer JM, <i>et al.</i> (2003 May)</a>. Enzymatic function of loop movement in enolase: preparation and some properties of H159N, H159A, H159F, and N207A enolases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15514048" target=_blank>Lemaire M, <i>et al.</i> (2004 Oct)</a>. Enolase and glycolytic flux play a role in the regulation of the glucose permease gene RAG1 of Kluyveromyces lactis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17339216" target=_blank>Hynes MJ, <i>et al.</i> (2007 May)</a>. Transcriptional control of gluconeogenesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17258477" target=_blank>Kim Y, <i>et al.</i> (2007 Sep)</a>. Proteome map of Aspergillus nidulans during osmoadaptation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23956636" target=_blank>Cho SJ, <i>et al.</i> (2010 Jun)</a>. Possible Roles of LAMMER Kinase Lkh1 in Fission Yeast by Comparative Proteome Analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919853" target=_blank>Oh YT, <i>et al.</i> (2010 Mar)</a>. Proteomic analysis of early phase of conidia germination in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22792266" target=_blank>Etxebeste O, <i>et al.</i> (2012)</a>. GmcA is a putative glucose-methanol-choline oxidoreductase required for the induction of asexual development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24167631" target=_blank>Shimanuki M, <i>et al.</i> (2013)</a>. Klf1, a C2H2 zinc finger-transcription factor, is required for cell wall maintenance during long-term quiescence in differentiated G0 phase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24120530" target=_blank>Martins I, <i>et al.</i> (2013 Dec 6)</a>. Proteomic alterations induced by ionic liquids in Aspergillus nidulans and Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28945192" target=_blank>Guydosh NR, <i>et al.</i> (2017 Sep 25)</a>. Regulated Ire1-dependent mRNA decay requires no-go mRNA degradation to maintain endoplasmic reticulum homeostasis in &lt;i&gt;S. pombe&lt;/i&gt;.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00295_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00295_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00295_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00295_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00203_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00296<br>EOG8JM664<br>ERR1<br>sce:ERR1<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized enolase ohnolog<br><br>
			<br><b>Parent</b><br>ohnolog:FOG00295<br><br>
			<br><b>SGD Description</b><br>Protein of unknown function; has similarity to enolases<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7785338" target=_blank>Pryde FE, <i>et al.</i> (1995 Apr 15)</a>. Sequence analysis of the right end of chromosome XV in Saccharomyces cerevisiae: an insight into the structural and functional significance of sub-telomeric repeat sequences.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00296_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00296_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00296_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00296_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00203_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00297<br>EOG8JM664<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized enolase paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00295<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00297_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00297_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00297_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00297_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00203_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00203.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>