<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00408.html">HOG00408&emsp;&emsp;NAD kinase family (mitochondrial)</a><br> 
																	HOG00409&emsp;&emsp;<br> 
																	<a href="HOG00410.html">HOG00410&emsp;&emsp;VTC1 family; VTC2/3 family; VTC4 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00409.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00409_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00409_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00409_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00409_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00869<br>EOG8VDNGT<br>IPP1<br>sce:IPP1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cytoplasmic inorganic pyrophosphatase (PPase)<br><br>
			<br><b>SGD Description</b><br>Cytoplasmic inorganic pyrophosphatase (PPase); homodimer that catalyzes the rapid exchange of oxygens from Pi with water, highly expressed and essential for viability, active-site residues show identity to those from E. coli PPase<br><br>
			<br><b>PomBase Description</b><br>inorganic pyrophosphatase Ipp1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Inorganic pyrophosphatase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=340461" target=_blank>Cohen SA, <i>et al.</i> (1978 Feb 10)</a>. Covalent structural analysis of yeast inorganic pyrophosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6101539" target=_blank>Bond MW, <i>et al.</i> (1980 Jan 8)</a>. Identification of an arginine important for enzymatic activity within the covalent structure of yeast inorganic pyrophosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2849749" target=_blank>Kolakowski LF Jr, <i>et al.</i> (1988 Nov 25)</a>. Cloning, molecular characterization and chromosome localization of the inorganic pyrophosphatase (PPA) gene from S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538971" target=_blank>Stark MJ, <i>et al.</i> (1989 Jan-Feb)</a>. Cloning and analysis of the Kluyveromyces lactis TRP1 gene: a chromosomal locus flanked by genes encoding inorganic pyrophosphatase and histone H3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160278" target=_blank>Lahti R, <i>et al.</i> (1990 May 8)</a>. Conservation of functional residues between yeast and E. coli inorganic pyrophosphatases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1322842" target=_blank>Raznikov AV, <i>et al.</i> (1992 Aug 10)</a>. Tyrosine-89 is important for enzymatic activity of S. cerevisiae inorganic pyrophosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8994974" target=_blank>Heikinheimo P, <i>et al.</i> (1996 Dec 15)</a>. The structural basis for pyrophosphatase catalysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8935650" target=_blank>Norbeck J, <i>et al.</i> (1996 Mar 15)</a>. Protein expression during exponential growth in 0.7 M NaCl medium of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878371" target=_blank>Tuominen V, <i>et al.</i> (1998 Dec 18)</a>. The R78K and D117E active-site variants of Saccharomyces cerevisiae soluble inorganic pyrophosphatase: structural studies and mechanistic implications.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15378761" target=_blank>Pitarch A, <i>et al.</i> (2004 Oct)</a>. Proteomics-based identification of novel Candida albicans antigens for diagnosis of systemic candidiasis in patients with underlying hematological malignancies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15665377" target=_blank>Gruhler A, <i>et al.</i> (2005 Mar)</a>. Quantitative phosphoproteomics applied to the yeast pheromone signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17258477" target=_blank>Kim Y, <i>et al.</i> (2007 Sep)</a>. Proteome map of Aspergillus nidulans during osmoadaptation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19053082" target=_blank>Shimizu M, <i>et al.</i> (2009 Jan)</a>. Proteomic analysis of Aspergillus nidulans cultured under hypoxic conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22119525" target=_blank>Singh NS, <i>et al.</i> (2011 Dec 6)</a>. SIN-inhibitory phosphatase complex promotes Cdc11p dephosphorylation and propagates SIN asymmetry in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22835028" target=_blank>Saykhedkar S, <i>et al.</i> (2012 Jul 26)</a>. A time course analysis of the extracellular proteome of Aspergillus nidulans growing on sorghum stover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23394829" target=_blank>Gupta S, <i>et al.</i> (2013 Feb 18)</a>. Identification of SIN pathway targets reveals mechanisms of crosstalk between NDR kinase pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23297348" target=_blank>Chen JS, <i>et al.</i> (2013 May)</a>. Comprehensive proteomics analysis reveals new substrates and regulators of the fission yeast clp1/cdc14 phosphatase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26412298" target=_blank>Beckley JR, <i>et al.</i> (2015 Dec)</a>. A Degenerate Cohort of Yeast Membrane Trafficking DUBs Mediates Cell Polarity and Survival.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26644575" target=_blank>Halim A, <i>et al.</i> (2015 Dec 22)</a>. Discovery of a nucleocytoplasmic O-mannose glycoproteome in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00869_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00869_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00869_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00869_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00409_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00870<br>EOG8VDNGT<br>PPA2<br>sce:PPA2<br><br>Genes: 23
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial inorganic pyrophosphatase<br><br>
			<br><b>SGD Description</b><br>Mitochondrial inorganic pyrophosphatase; required for mitochondrial function and possibly involved in energy generation from inorganic pyrophosphate<br><br>
			<br><b>PomBase Description</b><br>mitochondrial inorganic diphosphatase (predicted)<br><br>
			<br><b>Suggested Analysis</b><br>Investigate if protein is absent in some annotations; [m] enzyme is essential in sce<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648084" target=_blank>Lundin M, <i>et al.</i> (1991 Jul 5)</a>. Yeast PPA2 gene encodes a mitochondrial inorganic pyrophosphatase that is essential for mitochondrial function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1321599" target=_blank>Vihinen M, <i>et al.</i> (1992 Jul 15)</a>. Computer modeling of two inorganic pyrophosphatases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00870_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00870_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00870_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00870_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00409_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00409.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>