<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG01175.html">HOG01175&emsp;&emsp;glycosyltransferase 15 family</a><br> 
																	HOG01176&emsp;&emsp;<br> 
																	<a href="HOG01177.html">HOG01177&emsp;&emsp;calcineurin regulatory subunit family; calmodulin family; centrin family; recoverin family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01176.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01176_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01176_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG02996<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have role in negative regulation of sexual sporulation resulting in formation of a cellular spore, negative regulation of sterigmatocystin biosynthetic process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23341778" target=_blank>Palmer JM, <i>et al.</i> (2013)</a>. Secondary metabolism and development is mediated by LlmF control of VeA subcellular localization in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02996_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02996_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02996_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02996_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG02997<br>EOG86DJHT<br>EOG8CC2JF<br>EOG8H9W3W<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Putative protein of unknown function; non-essential gene identified in a screen for mutants affected in mannosylphophorylation of cell wall components<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02997_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02997_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02997_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02997_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG02998<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. fumigatus Af293 : Afu2g04890, A. oryzae RIB40 : AO090003000293, Neosartorya fischeri NRRL 181 : NFIA_081620 and Aspergillus clavatus NRRL 1 : ACLA_089760<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02998_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02998_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02998_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02998_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG02999<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Protein of unknown function<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02999_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02999_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG02999_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG02999_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03000<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN2165/llmA, A. fumigatus Af293 : Afu2g15860, A. oryzae RIB40 : AO090012000226, Aspergillus wentii : Aspwe1_0050672 and Aspergillus sydowii : Aspsy1_0054382<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03000_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03000_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03000_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03000_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03001<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. nidulans FGSC A4 : AN8833/llmI, Aspergillus wentii : Aspwe1_0261285, Aspergillus versicolor : Aspve1_0046299 and Aspergillus clavatus NRRL 1 : ACLA_009200<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23341778" target=_blank>Palmer JM, <i>et al.</i> (2013)</a>. Secondary metabolism and development is mediated by LlmF control of VeA subcellular localization in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03001_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03001_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03001_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03001_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG03002<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have methyltransferase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15075281" target=_blank>Bok JW, <i>et al.</i> (2004 Apr)</a>. LaeA, a regulator of secondary metabolism in Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16467480" target=_blank>Seo JA, <i>et al.</i> (2006 Feb)</a>. The phosducin-like protein PhnA is required for Gbetagamma-mediated signaling for vegetative growth, developmental control, and toxin biosynthesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16420358" target=_blank>Tsitsigiannis DI, <i>et al.</i> (2006 Feb)</a>. Oxylipins act as determinants of natural product biosynthesis and seed colonization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16426969" target=_blank>Bok JW, <i>et al.</i> (2006 Jan)</a>. Genomic mining for Aspergillus natural products.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16426964" target=_blank>Schardl CL, <i>et al.</i> (2006 Jan)</a>. A global view of metabolites.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17030989" target=_blank>Yu JH, <i>et al.</i> (2006 Oct)</a>. Growth and developmental control in the model and pathogenic aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16968230" target=_blank>Bok JW, <i>et al.</i> (2006 Sep)</a>. Secondary metabolic gene cluster silencing in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17291795" target=_blank>Bouhired S, <i>et al.</i> (2007 Nov)</a>. Accurate prediction of the Aspergillus nidulans terrequinone gene cluster boundaries using the transcriptional regulator LaeA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17686752" target=_blank>Rohlfs M, <i>et al.</i> (2007 Oct 22)</a>. Secondary chemicals protect mould from fungivory.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616629" target=_blank>Shwab EK, <i>et al.</i> (2007 Sep)</a>. Histone deacetylase activity regulates chemical diversity in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18457967" target=_blank>Calvo AM, <i>et al.</i> (2008 Jul)</a>. The VeA regulatory system and its role in morphological and chemical development in fungi.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18378656" target=_blank>Atoui A, <i>et al.</i> (2008 Jun)</a>. Aspergillus nidulans natural product biosynthesis is regulated by mpkB, a putative pheromone response mitogen-activated protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18556559" target=_blank>Bayram O, <i>et al.</i> (2008 Jun 13)</a>. VelB/VeA/LaeA complex coordinates light signal with fungal development and secondary metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18556538" target=_blank>Fischer R, <i>et al.</i> (2008 Jun 13)</a>. Developmental biology. Sex and poison in the dark.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18667168" target=_blank>Kale SP, <i>et al.</i> (2008 Oct)</a>. Requirement of LaeA for secondary metabolism and sclerotial production in Aspergillus flavus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411623" target=_blank>Amaike S, <i>et al.</i> (2009 Jul)</a>. Distinct roles for VeA and LaeA in development and pathogenesis of Aspergillus flavus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24031444" target=_blank>Delgado-Virgen F, <i>et al.</i> (2009 Oct)</a>. Mechanism of Sterigmatocystin Biosynthesis Regulation by pH in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20572938" target=_blank>Wiemann P, <i>et al.</i> (2010 Aug)</a>. FfVel1 and FfLae1, components of a velvet-like complex in Fusarium fujikuroi, affect differentiation, secondary metabolism and virulence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20934903" target=_blank>Braus GH, <i>et al.</i> (2010 Dec)</a>. Fungal development and the COP9 signalosome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21035346" target=_blank>Etxebeste O, <i>et al.</i> (2010 Dec)</a>. Aspergillus nidulans asexual development: making the most of cellular modules.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20935144" target=_blank>Shaaban MI, <i>et al.</i> (2010 Dec)</a>. Suppressor mutagenesis identifies a velvet complex remediator of Aspergillus nidulans secondary metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21152013" target=_blank>Sarikaya Bayram O, <i>et al.</i> (2010 Dec 2)</a>. LaeA control of velvet family regulatory proteins for light-dependent development and fungal cell-type specificity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132440" target=_blank>Reyes-Dominguez Y, <i>et al.</i> (2010 Jun)</a>. Heterochromatic marks are associated with the repression of secondary metabolism clusters in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20219692" target=_blank>Shaaban M, <i>et al.</i> (2010 May)</a>. Involvement of transposon-like elements in penicillin gene cluster regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22468035" target=_blank>Bhetariya PJ, <i>et al.</i> (2011 Apr)</a>. Allergens/Antigens, toxins and polyketides of important Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20433937" target=_blank>Brakhage AA, <i>et al.</i> (2011 Jan)</a>. Fungal secondary metabolites - strategies to activate silent gene clusters.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065618" target=_blank>Bayram O, <i>et al.</i> (2012)</a>. Identification of protein complexes from filamentous fungi with tandem affinity purification.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22829779" target=_blank>Bayram Ã–, <i>et al.</i> (2012)</a>. The Aspergillus nidulans MAPK module AnSte11-Ste50-Ste7-Fus3 controls development and secondary metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22713714" target=_blank>Jiang J, <i>et al.</i> (2012 Aug)</a>. FgVELB is associated with vegetative differentiation, secondary metabolism and virulence in Fusarium graminearum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783880" target=_blank>Ramamoorthy V, <i>et al.</i> (2012 Aug)</a>. veA-dependent RNA-pol II transcription elongation factor-like protein, RtfA, is associated with secondary metabolism and morphological development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289775" target=_blank>Chang PK, <i>et al.</i> (2012 Feb)</a>. Effects of laeA deletion on Aspergillus flavus conidial development and hydrophobicity may contribute to loss of aflatoxin production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21658084" target=_blank>Bayram O, <i>et al.</i> (2012 Jan)</a>. Coordination of secondary metabolism and development in fungi: the velvet family of regulatory proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22091779" target=_blank>Dyer PS, <i>et al.</i> (2012 Jan)</a>. Sexual development and cryptic sexuality in fungi: insights from Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23022264" target=_blank>Soukup AA, <i>et al.</i> (2012 Nov)</a>. NosA, a transcription factor important in Aspergillus fumigatus stress and developmental response, rescues the germination defect of a laeA deletion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22882998" target=_blank>Soukup AA, <i>et al.</i> (2012 Oct)</a>. Overexpression of the Aspergillus nidulans histone 4 acetyltransferase EsaA increases activation of secondary metabolite production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24023705" target=_blank>Caballero Ortiz S, <i>et al.</i> (2013)</a>. Induced fungal resistance to insect grazing: reciprocal fitness consequences and fungal gene expression in the Drosophila-Aspergillus model system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874628" target=_blank>Kim HK, <i>et al.</i> (2013)</a>. Functional roles of FgLaeA in controlling secondary metabolism, sexual development, and virulence in Fusarium graminearum.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23341778" target=_blank>Palmer JM, <i>et al.</i> (2013)</a>. Secondary metabolism and development is mediated by LlmF control of VeA subcellular localization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23729383" target=_blank>Kawauchi M, <i>et al.</i> (2013 Aug)</a>. Fungus-specific sirtuin HstD coordinates secondary metabolism and development through control of LaeA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23390613" target=_blank>Karimi-Aghcheh R, <i>et al.</i> (2013 Feb)</a>. Functional analyses of Trichoderma reesei LAE1 reveal conserved and contrasting roles of this regulator.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23084572" target=_blank>Gibbons JG, <i>et al.</i> (2013 Jan)</a>. The function and evolution of the Aspergillus genome.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23450714" target=_blank>Krijgsheld P, <i>et al.</i> (2013 Mar 15)</a>. Development in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633078" target=_blank>Sun X, <i>et al.</i> (2013 May)</a>. PyrG is required for maintaining stable cellular uracil level and normal sporulation pattern under excess uracil stress in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23532849" target=_blank>Patananan AN, <i>et al.</i> (2013 May 17)</a>. A novel automethylation reaction in the Aspergillus nidulans LaeA protein generates S-methylmethionine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23994319" target=_blank>Chang PK, <i>et al.</i> (2013 Sep-Oct)</a>. Aspergillus flavus VelB acts distinctly from VeA in conidiation and may coordinate with FluG to modulate sclerotial production.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25386652" target=_blank>Karimi Aghcheh R, <i>et al.</i> (2014)</a>. The VELVET A orthologue VEL1 of Trichoderma reesei regulates fungal development and is essential for cellulase gene expression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25762985" target=_blank>Calvo AM, <i>et al.</i> (2015)</a>. Association of fungal secondary metabolism and sclerotial biology.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03002_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03002_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03002_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03002_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG03003<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03003_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03003_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03003_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03003_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG03004<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03004_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03004_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03004_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03004_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG03005<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03005_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03005_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03005_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03005_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG03006<br>EOG8CC2JF<br><br>sce:absent<br><br>Genes: 14
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of A. oryzae RIB40 : AO090138000053, Aspergillus wentii : Aspwe1_0109942, Aspergillus versicolor : Aspve1_0029582, Aspve1_0086493 and Aspergillus clavatus NRRL 1 : ACLA_042390|Protein of unknown function|Protein of unknown function<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03006_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03006_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03006_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03006_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01176_phobius.pdf">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01176.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>