<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG04021.html">HOG04021&emsp;&emsp;PI3/PI4-kinase family. ATM subfamily; class-I pyridoxal-phosphate-dependent aminotransferase family</a><br> 
																	HOG04022&emsp;&emsp;<br> 
																	<a href="HOG04023.html">HOG04023&emsp;&emsp;PAF1 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG04022.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG04022_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG04022_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG04022_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG08198<br>EOG81JWZ0<br><br>sce:PSF2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of the GINS complex (Sld5p, Psf1p, Psf2p, Psf3p); complex is localized to DNA replication origins and implicated in assembly of the DNA replication machinery<br><br>
			<br><b>PomBase Description</b><br>GINS complex subunit Psf2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have role in mitotic DNA replication and cytosol, nuclear chromatin localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=776230" target=_blank>Guiard B, <i>et al.</i> (1976)</a>. Complete amino acid sequence of the heme-binding core in bakers' yeast cytochrome b2 (L-(+)-lactate dehydrogenase).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730134" target=_blank>Takayama Y, <i>et al.</i> (2003 May 1)</a>. GINS, a novel multiprotein complex required for chromosomal DNA replication in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466421" target=_blank>GÃ³mez EB, <i>et al.</i> (2005 Jan)</a>. A screen for Schizosaccharomyces pombe mutants defective in rereplication identifies new alleles of rad4+, cut9+ and psf2+.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15941470" target=_blank>Yang X, <i>et al.</i> (2005 Jun 7)</a>. Nuclear distribution and chromatin association of DNA polymerase alpha-primase is affected by TEV protease cleavage of Cdc23 (Mcm10) in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16199877" target=_blank>Huang HK, <i>et al.</i> (2005 Oct)</a>. Suppressors of Bir1p (Survivin) identify roles for the chromosomal passenger protein Pic1p (INCENP) and the replication initiation factor Psf2p in chromosome segregation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16990792" target=_blank>Yabuuchi H, <i>et al.</i> (2006 Oct 4)</a>. Ordered assembly of Sld3, GINS and Cdc45 is distinctly regulated by DDK and CDK for activation of replication origins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19109429" target=_blank>Pai CC, <i>et al.</i> (2009 Feb)</a>. GINS inactivation phenotypes reveal two pathways for chromatin association of replicative alpha and epsilon DNA polymerases in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20176980" target=_blank>Dolan WP, <i>et al.</i> (2010 May)</a>. Fission yeast Hsk1 (Cdc7) kinase is required after replication initiation for induced mutagenesis and proper response to DNA alkylation damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21593208" target=_blank>Fukuura M, <i>et al.</i> (2011 Jul 15)</a>. CDK promotes interactions of Sld3 and Drc1 with Cut5 for initiation of DNA replication in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22718908" target=_blank>Handa T, <i>et al.</i> (2012 Aug)</a>. DNA polymerization-independent functions of DNA polymerase epsilon in assembly and progression of the replisome in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22433840" target=_blank>Kanke M, <i>et al.</i> (2012 May 2)</a>. Mcm10 plays an essential role in origin DNA unwinding after loading of the CMG components.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23695164" target=_blank>Das J, <i>et al.</i> (2013 May 21)</a>. Cross-species protein interactome mapping reveals species-specific wiring of stress response pathways.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG08198_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG08198_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG08198_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG08198_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG04022.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>