<html>
<head><title>HOG00263</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG00262.html">HOG00262&emsp;&emsp;isocitrate and isopropylmalate dehydrogenases family</a><br> 
																	HOG00263&emsp;&emsp;<br> 
																	<a href="HOG00264.html">HOG00264&emsp;&emsp;alpha-ketoglutarate dehydrogenase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00263.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00263_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00263_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00263_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00263_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/kcorreia/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://kcorreia.github.io/aybrah/website/HOG00263.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00623<br>EOG8V41R6<br>IDH1<br>sce:IDH1<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Alpha subunit of mitochondrial NAD(+)-dependent isocitrate dehydrogenase<br><br>
			<br><b>SGD Description</b><br>Subunit of mitochondrial NAD(+)-dependent isocitrate dehydrogenase; complex catalyzes the oxidation of isocitrate to alpha-ketoglutarate in the TCA cycle<br><br>
			<br><b>PomBase Description</b><br>isocitrate dehydrogenase (NAD+) subunit 1 Idh1<br><br>
			<br><b>AspGD Description</b><br>Putative isocitrate dehydrogenase (NAD+)<br><br>
			<br><b>Suggested Analysis</b><br>Investigate origin of duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2198251" target=_blank>Keys DA, <i>et al.</i> (1990 Aug)</a>. Subunit structure, expression, and function of NAD(H)-specific isocitrate dehydrogenase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1644826" target=_blank>Cupp JR, <i>et al.</i> (1992 Aug 15)</a>. Cloning and characterization of the gene encoding the IDH1 subunit of NAD(+)-dependent isocitrate dehydrogenase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7903653" target=_blank>Barel I, <i>et al.</i> (1993 Nov 1)</a>. Enzyme defects in glutamate-requiring strains of Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7505425" target=_blank>Elzinga SD, <i>et al.</i> (1993 Nov 25)</a>. Yeast mitochondrial NAD(+)-dependent isocitrate dehydrogenase is an RNA-binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10975257" target=_blank>Elzinga SD, <i>et al.</i> (2000 Aug)</a>. Isolation and RNA-binding analysis of NAD+ -isocitrate dehydrogenases from Kluyveromyces lactis and Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21652630" target=_blank>Snaith HA, <i>et al.</i> (2011 Jul 1)</a>. Characterization of Mug33 reveals complementary roles for actin cable-dependent transport and exocyst regulators in fission yeast exocytosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23213482" target=_blank>Du W, <i>et al.</i> (2012 Sep 15)</a>. TORC2 and the AGC kinase Gad8 regulate phosphorylation of the ribosomal protein S6 in fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00623_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00623_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00623_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00623_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00263_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00624<br>EOG8V41R6<br>IDH2<br>sce:IDH2<br><br>Genes: 35
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Beta subunit of mitochondrial NAD(+)-dependent isocitrate dehydrogenase<br><br>
			<br><b>SGD Description</b><br>Subunit of mitochondrial NAD(+)-dependent isocitrate dehydrogenase; complex catalyzes the oxidation of isocitrate to alpha-ketoglutarate in the TCA cycle; phosphorylated<br><br>
			<br><b>PomBase Description</b><br>isocitrate dehydrogenase (NAD+) subunit 2<br><br>
			<br><b>AspGD Description</b><br>Putative isocitrate dehydrogenase (NAD+)<br><br>
			<br><b>Suggested Analysis</b><br>Investigate origin of duplication<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2198251" target=_blank>Keys DA, <i>et al.</i> (1990 Aug)</a>. Subunit structure, expression, and function of NAD(H)-specific isocitrate dehydrogenase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1939242" target=_blank>Cupp JR, <i>et al.</i> (1991 Nov 25)</a>. NAD(+)-dependent isocitrate dehydrogenase. Cloning, nucleotide sequence, and disruption of the IDH2 gene from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7505425" target=_blank>Elzinga SD, <i>et al.</i> (1993 Nov 25)</a>. Yeast mitochondrial NAD(+)-dependent isocitrate dehydrogenase is an RNA-binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10975257" target=_blank>Elzinga SD, <i>et al.</i> (2000 Aug)</a>. Isolation and RNA-binding analysis of NAD+ -isocitrate dehydrogenases from Kluyveromyces lactis and Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11502169" target=_blank>Grandier-Vazeille X, <i>et al.</i> (2001 Aug 21)</a>. Yeast mitochondrial dehydrogenases are associated in a supramolecular complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17761666" target=_blank>Reinders J, <i>et al.</i> (2007 Nov)</a>. Profiling phosphoproteins of yeast mitochondria reveals a role of phosphorylation in assembly of the ATP synthase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20878186" target=_blank>Masuo S, <i>et al.</i> (2010 Dec)</a>. Global gene expression analysis of Aspergillus nidulans reveals metabolic shift and transcription suppression under hypoxia.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24120530" target=_blank>Martins I, <i>et al.</i> (2013 Dec 6)</a>. Proteomic alterations induced by ionic liquids in Aspergillus nidulans and Neurospora crassa.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00624_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00624_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00624_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00624_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00263_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00625<br>EOG8V41R6<br>LYS12<br>sce:LYS12<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Homo-isocitrate dehydrogenase<br><br>
			<br><b>Features</b><br>[m];NAD<br><br>
			<br><b>Parent</b><br>paralog:FOG00624<br><br>
			<br><b>SGD Description</b><br>Homo-isocitrate dehydrogenase; an NAD-linked mitochondrial enzyme required for the fourth step in the biosynthesis of lysine, in which homo-isocitrate is oxidatively decarboxylated to alpha-ketoadipate<br><br>
			<br><b>PomBase Description</b><br>homoisocitrate dehydrogenase Lys12<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have NAD binding, homoisocitrate dehydrogenase activity, role in lysine biosynthetic process and mitochondrion localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5857597" target=_blank>KÃ¤fer E, <i>et al.</i> (1965 Jul)</a>. Origins of translocations in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4284830" target=_blank>Strassman M, <i>et al.</i> (1965 Nov)</a>. Enzymatic formation of alpha-ketoadipic acid from homoisocitric acid.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4395693" target=_blank>Rowley B, <i>et al.</i> (1970 Dec)</a>. Homoisocitric dehydrogenase from yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4581424" target=_blank>Clutterbuck AJ, <i>et al.</i> (1973 Jun)</a>. Gene symbols in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=393802" target=_blank>Bal J, <i>et al.</i> (1979 Dec)</a>. Allele specific and locus non-specific suppressors in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3939712" target=_blank>Urrestarazu LA, <i>et al.</i> (1985)</a>. General and specific controls of lysine biosynthesis in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7982501" target=_blank>Donnelly E, <i>et al.</i> (1994 Nov 28)</a>. Germinating conidiospores of Aspergillus amino acid auxotrophs are hypersensitive to heat shock, oxidative stress and DNA damage.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8962070" target=_blank>Shevchenko A, <i>et al.</i> (1996 Dec 10)</a>. Linking genome and proteome by mass spectrometry: large-scale identification of yeast proteins from two dimensional gels.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14576278" target=_blank>Sickmann A, <i>et al.</i> (2003 Nov 11)</a>. The proteome of Saccharomyces cerevisiae mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14732264" target=_blank>Bruggeman J, <i>et al.</i> (2004 Feb)</a>. Selection arena in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16698295" target=_blank>Mogensen J, <i>et al.</i> (2006 Aug)</a>. Transcription analysis using high-density micro-arrays of Aspergillus nidulans wild-type and creA mutant during growth on glucose or ethanol.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17223711" target=_blank>Lin Y, <i>et al.</i> (2007 Jan 23)</a>. Complete kinetic mechanism of homoisocitrate dehydrogenase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19530703" target=_blank>Lin Y, <i>et al.</i> (2009 Aug 4)</a>. Site-directed mutagenesis as a probe of the acid-base catalytic mechanism of homoisocitrate dehydrogenase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25557478" target=_blank>Kim JM, <i>et al.</i> (2015 Jan)</a>. Characterization of NpgA, a 4'-phosphopantetheinyl transferase of Aspergillus nidulans, and evidence of its involvement in fungal growth and formation of conidia and cleistothecia for development.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00625_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00625_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00625_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00625_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00263_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00626<br>EOG8V41R6<br>LYS12.2<br>sce:LYS12.2<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized paralog<br><br>
			<br><b>Parent</b><br>paralog:FOG00625<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00626_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00626_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG00626_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG00626_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00263_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00263.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>