<html>
<head><title>HOG03507</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG03506.html">HOG03506&emsp;&emsp;</a><br> 
																	HOG03507&emsp;&emsp;<br> 
																	<a href="HOG03508.html">HOG03508&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG03507.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG03507_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG03507_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG03507_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG03507_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG03507.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG07501<br>EOG88W9HG<br><br>sce:RPD3<br><br>Genes: 38
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Histone deacetylase, component of both the Rpd3S and Rpd3L complexes; regulates transcription, silencing, autophagy and other processes by influencing chromatin remodeling; forms at least two different complexes which have distinct functions and members; Rpd3(L) recruitment to the subtelomeric region is regulated by interaction with the arginine methyltransferase, Hmt1p<br><br>
			<br><b>PomBase Description</b><br>histone deacetylase (class I) Clr6<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have histone deacetylase activity, transcription coactivator activity, transcription corepressor activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1944291" target=_blank>Vidal M, <i>et al.</i> (1991 Dec)</a>. RPD3 encodes a second factor required to achieve maximum positive and negative transcriptional states in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8873448" target=_blank>Duina AA, <i>et al.</i> (1996 Aug)</a>. Identification of two CyP-40-like cyclophilins in Saccharomyces cerevisiae, one of which is required for normal growth.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8978024" target=_blank>Vannier D, <i>et al.</i> (1996 Dec)</a>. Evidence that the transcriptional regulators SIN3 and RPD3, and a novel gene (SDS3) with similar functions, are involved in transcriptional silencing in S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8962081" target=_blank>Rundlett SE, <i>et al.</i> (1996 Dec 10)</a>. HDA1 and RPD3 are members of distinct yeast histone deacetylase complexes that regulate silencing and transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9234741" target=_blank>Kasten MM, <i>et al.</i> (1997 Aug)</a>. A large protein complex containing the yeast Sin3p and Rpd3p transcriptional regulators.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9572144" target=_blank>Rundlett SE, <i>et al.</i> (1998 Apr 23)</a>. Transcriptional repression by UME6 involves deacetylation of lysine 5 of histone H4 by RPD3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512514" target=_blank>Kadosh D, <i>et al.</i> (1998 Mar 15)</a>. Histone deacetylase activity of Rpd3 is important for transcriptional repression in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9710596" target=_blank>Kadosh D, <i>et al.</i> (1998 Sep)</a>. Targeted recruitment of the Sin3-Rpd3 histone deacetylase complex generates a highly localized domain of repressed chromatin in vivo.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10388812" target=_blank>Sun ZW, <i>et al.</i> (1999 Jul)</a>. A general requirement for the Sin3-Rpd3 histone deacetylase complex in regulating silencing in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10359799" target=_blank>Burgess SM, <i>et al.</i> (1999 Jun 8)</a>. GCN5-dependent histone H3 acetylation and RPD3-dependent histone H4 deacetylation have distinct, opposing effects on IME2 transcription, during meiosis and during vegetative growth, in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10079324" target=_blank>Dora EG, <i>et al.</i> (1999 Mar)</a>. RPD3 (REC3) mutations affect mitotic recombination in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931932" target=_blank>Elkhaimi M, <i>et al.</i> (2000 Aug 15)</a>. Combinatorial regulation of phospholipid biosynthetic gene expression by the UME6, SIN3 and RPD3 genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10899127" target=_blank>Arévalo-Rodríguez M, <i>et al.</i> (2000 Jul 17)</a>. Cyclophilin A and Ess1 interact with and regulate silencing by the Sin3-Rpd3 histone deacetylase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004483" target=_blank>Graessle S, <i>et al.</i> (2000 Jun 21)</a>. Characterization of two putative histone deacetylase genes from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11069890" target=_blank>Watson AD, <i>et al.</i> (2000 Nov 1)</a>. Ssn6-Tup1 interacts with class I histone deacetylases required for repression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12089521" target=_blank>Kurdistani SK, <i>et al.</i> (2002 Jul)</a>. Genome-wide binding map of the histone deacetylase Rpd3 in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12192044" target=_blank>Deckert J, <i>et al.</i> (2002 Sep)</a>. Targeted recruitment of Rpd3 histone deacetylase represses transcription by inhibiting recruitment of Swi/Snf, SAGA, and TATA binding protein.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234935" target=_blank>Sandmeier JJ, <i>et al.</i> (2002 Sep 16)</a>. RPD3 is required for the inactivation of yeast ribosomal DNA genes in stationary phase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12110674" target=_blank>Adachi N, <i>et al.</i> (2002 Sep 20)</a>. A conserved motif common to the histone acetyltransferase Esa1 and the histone deacetylase Rpd3.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12808094" target=_blank>Scott KL, <i>et al.</i> (2003 Jul)</a>. Loss of Sin3/Rpd3 histone deacetylase restores the DNA damage response in checkpoint-deficient strains of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12853613" target=_blank>Trojer P, <i>et al.</i> (2003 Jul 15)</a>. Histone deacetylases in fungi: novel members, new facts.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12672825" target=_blank>Nourani A, <i>et al.</i> (2003 May 23)</a>. Opposite role of yeast ING family members in p53-dependent transcriptional activation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14711989" target=_blank>Jazayeri A, <i>et al.</i> (2004 Feb 10)</a>. Saccharomyces cerevisiae Sin3p facilitates DNA double-strand break repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14737171" target=_blank>De Nadal E, <i>et al.</i> (2004 Jan 22)</a>. The MAPK Hog1 recruits Rpd3 histone deacetylase to activate osmoresponsive genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15143171" target=_blank>Aparicio JG, <i>et al.</i> (2004 Jun)</a>. The Rpd3-Sin3 histone deacetylase regulates replication timing and enables intra-S origin control in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15141165" target=_blank>Schröder M, <i>et al.</i> (2004 Jun 2)</a>. The unfolded protein response represses differentiation through the RPD3-SIN3 histone deacetylase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15456858" target=_blank>Sabet N, <i>et al.</i> (2004 Oct)</a>. Genome-wide analysis of the relationship between transcriptional regulation by Rpd3p and the histone H3 and H4 amino termini in budding yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16314178" target=_blank>Carrozza MJ, <i>et al.</i> (2005 Nov 10)</a>. Stable incorporation of sequence specific repressors Ash1 and Ume6 into the Rpd3L complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16286008" target=_blank>Keogh MC, <i>et al.</i> (2005 Nov 18)</a>. Cotranscriptional set2 methylation of histone H3 lysine 36 recruits a repressive Rpd3 complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16215180" target=_blank>Tribus M, <i>et al.</i> (2005 Oct)</a>. HdaA, a major class 2 histone deacetylase of Aspergillus nidulans, affects growth under conditions of oxidative stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118613" target=_blank>Izawa M, <i>et al.</i> (2009 Dec)</a>. Inhibition of histone deacetylase causes reduction of appressorium formation in the rice blast fungus Magnaporthe oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18703157" target=_blank>Miskei M, <i>et al.</i> (2009 Mar)</a>. Annotation of stress-response proteins in the aspergilli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19940017" target=_blank>Tribus M, <i>et al.</i> (2010 Jan 15)</a>. A novel motif in fungal class 1 histone deacetylases is essential for growth and development of Aspergillus.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07501_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07501_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07501_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG07501_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG03507_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG07502<br>EOG88W9HG<br><br>sce:HOS2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Histone deacetylase and subunit of Set3 and Rpd3L complexes; required for gene activation via specific deacetylation of lysines in H3 and H4 histone tails; subunit of the Set3 complex, a meiotic-specific repressor of sporulation specific genes that contains deacetylase activity; co-localizes with Cmr1p in nuclear foci in response to DNA damage by MMS<br><br>
			<br><b>PomBase Description</b><br>histone deacetylase (class I) Hos2<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have NAD-dependent histone deacetylase activity, NAD-independent histone deacetylase activity, histone deacetylase activity (H4-K16 specific), tubulin deacetylase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8962081" target=_blank>Rundlett SE, <i>et al.</i> (1996 Dec 10)</a>. HDA1 and RPD3 are members of distinct yeast histone deacetylase complexes that regulate silencing and transcription.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004483" target=_blank>Graessle S, <i>et al.</i> (2000 Jun 21)</a>. Characterization of two putative histone deacetylase genes from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11711434" target=_blank>Pijnappel WW, <i>et al.</i> (2001 Nov 15)</a>. The S. cerevisiae SET3 complex includes two histone deacetylases, Hos2 and Hst1, and is a meiotic-specific repressor of the sporulation gene program.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12434058" target=_blank>Wang A, <i>et al.</i> (2002 Nov 15)</a>. Requirement of Hos2 histone deacetylase for gene activity in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12853613" target=_blank>Trojer P, <i>et al.</i> (2003 Jul 15)</a>. Histone deacetylases in fungi: novel members, new facts.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07502_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07502_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG07502_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG07502_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG03507_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG03507.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>