<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00242.html">HOG00242&emsp;&emsp;NAD(P)-dependent epimerase/dehydratase family. Dihydroflavonol-4-reductase subfamily</a><br> 
																	HOG00243&emsp;&emsp;<br> 
																	<a href="HOG00244.html">HOG00244&emsp;&emsp;short-chain dehydrogenases/reductases (SDR) family</a><br><br> 
																	<a href="../sequences/hog/HOG00243.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG00243_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG00243_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG00243_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG00243_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG00521<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 42
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Novel alcohol dehydrogenase; spo ortholog may have [m] localization<br><br>
			<br><b>PomBase Description</b><br>sorbose reductase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Putative to sorbitol utilization protein|L-xylulose reductase; xylitol dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9560244" target=_blank>Janbon G, <i>et al.</i> (1998 Apr 28)</a>. Monosomy of a specific chromosome determines L-sorbose utilization: a novel regulatory mechanism in Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529438" target=_blank>Chen D, <i>et al.</i> (2003 Jan)</a>. Global transcriptional responses of fission yeast to environmental stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16134116" target=_blank>Greenberg JR, <i>et al.</i> (2005 Sep)</a>. Candida albicans SOU1 encodes a sorbose reductase required for L-sorbose utilization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17435004" target=_blank>Nie Y, <i>et al.</i> (2007 Jun)</a>. Purification, characterization, gene cloning, and expression of a novel alcohol dehydrogenase with anti-prelog stereospecificity from Candida parapsilosis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23608762" target=_blank>Napora K, <i>et al.</i> (2013 Jun 1)</a>. Yarrowia lipolytica dehydrogenase/reductase: an enzyme tolerant for lipophilic compounds and carbohydrate substrates.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28410370" target=_blank>Dudin O, <i>et al.</i> (2017 Apr)</a>. A systematic screen for morphological abnormalities during fission yeast sexual reproduction identifies a mechanism of actin aster formation for cell fusion.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28218250" target=_blank>Lee J, <i>et al.</i> (2017 Feb 20)</a>. Chromatin remodeller Fun30&lt;sup&gt;Fft3&lt;/sup&gt; induces nucleosome disassembly to facilitate RNA polymerase II elongation.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00521_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00521_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00521_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00521_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00521">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG00522<br>EOG86Q59T<br>FOX2<br>sce:FOX2<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>3-hydroxyacyl-CoA dehydrogenase and enoyl-CoA hydratase; multifunctional enzyme of the peroxisomal fatty acid beta-oxidation pathway<br><br>
			<br><b>AspGD Description</b><br>Enoyl-CoA hydratase; peroxisomal multifunctional beta-oxidation protein; induced by fenpropimorph<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1551874" target=_blank>Hiltunen JK, <i>et al.</i> (1992 Apr 5)</a>. Peroxisomal multifunctional beta-oxidation protein of Saccharomyces cerevisiae. Molecular analysis of the fox2 gene and gene product.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15554960" target=_blank>Maggio-Hall LA, <i>et al.</i> (2004 Dec)</a>. Mitochondrial beta-oxidation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15590815" target=_blank>Tsitsigiannis DI, <i>et al.</i> (2004 Dec)</a>. Endogenous lipogenic regulators of spore balance in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16134890" target=_blank>Maggio-Hall LA, <i>et al.</i> (2005 Aug)</a>. Fundamental contribution of beta-oxidation to polyketide mycotoxin production in planta.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16682457" target=_blank>Hynes MJ, <i>et al.</i> (2006 May)</a>. Regulatory genes controlling fatty acid catabolism and peroxisomal functions in the filamentous fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20736083" target=_blank>Magliano P, <i>et al.</i> (2010 Dec)</a>. Repercussion of a deficiency in mitochondrial ß-oxidation on the carbon flux of short-chain fatty acids to the peroxisomal ß-oxidation cycle in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21998305" target=_blank>Magliano P, <i>et al.</i> (2011 Dec 9)</a>. Contributions of the peroxisome and β-oxidation cycle to biotin synthesis in fungi.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00522_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00522_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00522_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00522_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00522">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG00523<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Ortholog(s) have glyoxysome localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00523_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00523_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00523_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00523_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00523">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG00524<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 29
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have L-arabinitol 2-dehydrogenase activity and role in D-arabinose catabolic process, D-arabitol catabolic process to xylulose 5-phosphate<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7483848" target=_blank>Hallborn J, <i>et al.</i> (1995 Jul)</a>. A short-chain dehydrogenase gene from Pichia stipitis having D-arabinitol dehydrogenase activity.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00524_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00524_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00524_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00524_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00524">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG00525<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 24
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>D-arabitol dehydrogenase|Putative p-sulfobenzyl alcohol dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19784673" target=_blank>Salazar M, <i>et al.</i> (2009 Dec)</a>. Uncovering transcriptional regulation of glycerol metabolism in Aspergilli through genome-wide gene expression data analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22329759" target=_blank>Lo HC, <i>et al.</i> (2012 Mar 14)</a>. Two separate gene clusters encode the biosynthetic pathway for the meroterpenoids austinol and dehydroaustinol in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00525_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00525_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00525_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00525_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00525">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG00526<br>EOG86Q59T<br>EOG8F4QW9<br><br>sce:absent<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have tetrahydroxynaphthalene reductase activity, versicolorin reductase activity|Ortholog(s) have tetrahydroxynaphthalene reductase activity, versicolorin reductase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8017929" target=_blank>Keller NP, <i>et al.</i> (1994 May)</a>. Aspergillus nidulans verA is required for production of the mycotoxin sterigmatocystin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7546572" target=_blank>Keller NP, <i>et al.</i> (1995)</a>. Analysis of a mycotoxin gene cluster in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7486998" target=_blank>Keller NP, <i>et al.</i> (1995 Oct)</a>. stcS, a putative P-450 monooxygenase, is required for the conversion of versicolorin A to sterigmatocystin in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8643646" target=_blank>Brown DW, <i>et al.</i> (1996 Feb 20)</a>. Twenty-five coregulated transcripts define a sterigmatocystin gene cluster in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8662194" target=_blank>Yu JH, <i>et al.</i> (1996 May)</a>. Conservation of structure and function of the aflatoxin regulatory gene aflR from Aspergillus nidulans and A. flavus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9305634" target=_blank>Hicks JK, <i>et al.</i> (1997 Aug 15)</a>. Aspergillus sporulation and mycotoxin production both require inactivation of the FadA G alpha protein-dependent signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999832" target=_blank>Kelkar HS, <i>et al.</i> (1997 Jan 17)</a>. Aspergillus nidulans stcL encodes a putative cytochrome P-450 monooxygenase required for bisfuran desaturation during aflatoxin/sterigmatocystin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18945083" target=_blank>Keller NP, <i>et al.</i> (1997 Jun)</a>. pH Regulation of Sterigmatocystin and Aflatoxin Biosynthesis in Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15012504" target=_blank>Payne GA, <i>et al.</i> (1998)</a>. Genetics and physiology of aflatoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680223" target=_blank>Fernandes M, <i>et al.</i> (1998 Jun)</a>. Sequence-specific binding by Aspergillus nidulans AflR, a C6 zinc cluster protein regulating mycotoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11156981" target=_blank>Shimizu K, <i>et al.</i> (2001 Feb)</a>. Genetic involvement of a cAMP-dependent protein kinase in a G protein signaling pathway regulating morphological and chemical transitions in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11260466" target=_blank>Hicks J, <i>et al.</i> (2001 Mar)</a>. RcoA has pleiotropic effects on Aspergillus nidulans cellular development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12410837" target=_blank>Jin Y, <i>et al.</i> (2002 Nov)</a>. Requirement of spermidine for developmental transitions in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665453" target=_blank>Kato N, <i>et al.</i> (2003 Dec)</a>. The expression of sterigmatocystin and penicillin genes in Aspergillus nidulans is controlled by veA, a gene required for sexual development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15075281" target=_blank>Bok JW, <i>et al.</i> (2004 Apr)</a>. LaeA, a regulator of secondary metabolism in Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15022028" target=_blank>Yabe K, <i>et al.</i> (2004 Jun)</a>. Enzyme reactions and genes in aflatoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21148941" target=_blank>Wilkinson HH, <i>et al.</i> (2004 Nov-Dec)</a>. Increased conidiation associated with progression along the sterigmatocystin biosynthetic pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16134890" target=_blank>Maggio-Hall LA, <i>et al.</i> (2005 Aug)</a>. Fundamental contribution of beta-oxidation to polyketide mycotoxin production in planta.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16467480" target=_blank>Seo JA, <i>et al.</i> (2006 Feb)</a>. The phosducin-like protein PhnA is required for Gbetagamma-mediated signaling for vegetative growth, developmental control, and toxin biosynthesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16420358" target=_blank>Tsitsigiannis DI, <i>et al.</i> (2006 Feb)</a>. Oxylipins act as determinants of natural product biosynthesis and seed colonization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16426969" target=_blank>Bok JW, <i>et al.</i> (2006 Jan)</a>. Genomic mining for Aspergillus natural products.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16387865" target=_blank>Seo JA, <i>et al.</i> (2006 Mar)</a>. FluG-dependent asexual development in Aspergillus nidulans occurs via derepression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16968230" target=_blank>Bok JW, <i>et al.</i> (2006 Sep)</a>. Secondary metabolic gene cluster silencing in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616629" target=_blank>Shwab EK, <i>et al.</i> (2007 Sep)</a>. Histone deacetylase activity regulates chemical diversity in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18378656" target=_blank>Atoui A, <i>et al.</i> (2008 Jun)</a>. Aspergillus nidulans natural product biosynthesis is regulated by mpkB, a putative pheromone response mitogen-activated protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18457952" target=_blank>Peñalva MA, <i>et al.</i> (2008 Jun)</a>. Ambient pH gene regulation in fungi: making connections.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20816830" target=_blank>Atoui A, <i>et al.</i> (2010 Dec)</a>. Cross-talk between light and glucose regulation controls toxin production and morphogenesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20935144" target=_blank>Shaaban MI, <i>et al.</i> (2010 Dec)</a>. Suppressor mutagenesis identifies a velvet complex remediator of Aspergillus nidulans secondary metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132440" target=_blank>Reyes-Dominguez Y, <i>et al.</i> (2010 Jun)</a>. Heterochromatic marks are associated with the repression of secondary metabolism clusters in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21062371" target=_blank>Nahlik K, <i>et al.</i> (2010 Nov)</a>. The COP9 signalosome mediates transcriptional and metabolic response to hormones, oxidative stress protection and cell wall rearrangement during fungal development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783880" target=_blank>Ramamoorthy V, <i>et al.</i> (2012 Aug)</a>. veA-dependent RNA-pol II transcription elongation factor-like protein, RtfA, is associated with secondary metabolism and morphological development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24066102" target=_blank>Ramamoorthy V, <i>et al.</i> (2013)</a>. The putative C2H2 transcription factor MtfA is a novel regulator of secondary metabolism and morphogenesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23840895" target=_blank>Shantappa S, <i>et al.</i> (2013)</a>. Role of the zinc finger transcription factor SltA in morphogenesis and sterigmatocystin biosynthesis in the fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23658706" target=_blank>de Souza WR, <i>et al.</i> (2013)</a>. Identification of metabolic pathways influenced by the G-protein coupled receptors GprB and GprD in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00526_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00526_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00526_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00526_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00526">Phobius transmembrane predictions</a>
										<br>3 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e7d7cd" class="col">
			FOG00527<br>EOG85DV7Q<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 21
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have tetrahydroxynaphthalene reductase activity, versicolorin reductase activity|Ortholog(s) have tetrahydroxynaphthalene reductase activity, versicolorin reductase activity|L-xylo-3-hexulose reductase; 3-oxoacyl-[acyl-carrier protein] reductase|L-xylo-3-hexulose reductase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8017929" target=_blank>Keller NP, <i>et al.</i> (1994 May)</a>. Aspergillus nidulans verA is required for production of the mycotoxin sterigmatocystin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7546572" target=_blank>Keller NP, <i>et al.</i> (1995)</a>. Analysis of a mycotoxin gene cluster in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7486998" target=_blank>Keller NP, <i>et al.</i> (1995 Oct)</a>. stcS, a putative P-450 monooxygenase, is required for the conversion of versicolorin A to sterigmatocystin in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8643646" target=_blank>Brown DW, <i>et al.</i> (1996 Feb 20)</a>. Twenty-five coregulated transcripts define a sterigmatocystin gene cluster in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8662194" target=_blank>Yu JH, <i>et al.</i> (1996 May)</a>. Conservation of structure and function of the aflatoxin regulatory gene aflR from Aspergillus nidulans and A. flavus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9305634" target=_blank>Hicks JK, <i>et al.</i> (1997 Aug 15)</a>. Aspergillus sporulation and mycotoxin production both require inactivation of the FadA G alpha protein-dependent signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999832" target=_blank>Kelkar HS, <i>et al.</i> (1997 Jan 17)</a>. Aspergillus nidulans stcL encodes a putative cytochrome P-450 monooxygenase required for bisfuran desaturation during aflatoxin/sterigmatocystin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18945083" target=_blank>Keller NP, <i>et al.</i> (1997 Jun)</a>. pH Regulation of Sterigmatocystin and Aflatoxin Biosynthesis in Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15012504" target=_blank>Payne GA, <i>et al.</i> (1998)</a>. Genetics and physiology of aflatoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680223" target=_blank>Fernandes M, <i>et al.</i> (1998 Jun)</a>. Sequence-specific binding by Aspergillus nidulans AflR, a C6 zinc cluster protein regulating mycotoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11156981" target=_blank>Shimizu K, <i>et al.</i> (2001 Feb)</a>. Genetic involvement of a cAMP-dependent protein kinase in a G protein signaling pathway regulating morphological and chemical transitions in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11260466" target=_blank>Hicks J, <i>et al.</i> (2001 Mar)</a>. RcoA has pleiotropic effects on Aspergillus nidulans cellular development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12410837" target=_blank>Jin Y, <i>et al.</i> (2002 Nov)</a>. Requirement of spermidine for developmental transitions in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665453" target=_blank>Kato N, <i>et al.</i> (2003 Dec)</a>. The expression of sterigmatocystin and penicillin genes in Aspergillus nidulans is controlled by veA, a gene required for sexual development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15075281" target=_blank>Bok JW, <i>et al.</i> (2004 Apr)</a>. LaeA, a regulator of secondary metabolism in Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15022028" target=_blank>Yabe K, <i>et al.</i> (2004 Jun)</a>. Enzyme reactions and genes in aflatoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21148941" target=_blank>Wilkinson HH, <i>et al.</i> (2004 Nov-Dec)</a>. Increased conidiation associated with progression along the sterigmatocystin biosynthetic pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16134890" target=_blank>Maggio-Hall LA, <i>et al.</i> (2005 Aug)</a>. Fundamental contribution of beta-oxidation to polyketide mycotoxin production in planta.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16467480" target=_blank>Seo JA, <i>et al.</i> (2006 Feb)</a>. The phosducin-like protein PhnA is required for Gbetagamma-mediated signaling for vegetative growth, developmental control, and toxin biosynthesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16420358" target=_blank>Tsitsigiannis DI, <i>et al.</i> (2006 Feb)</a>. Oxylipins act as determinants of natural product biosynthesis and seed colonization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16426969" target=_blank>Bok JW, <i>et al.</i> (2006 Jan)</a>. Genomic mining for Aspergillus natural products.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16387865" target=_blank>Seo JA, <i>et al.</i> (2006 Mar)</a>. FluG-dependent asexual development in Aspergillus nidulans occurs via derepression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16968230" target=_blank>Bok JW, <i>et al.</i> (2006 Sep)</a>. Secondary metabolic gene cluster silencing in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616629" target=_blank>Shwab EK, <i>et al.</i> (2007 Sep)</a>. Histone deacetylase activity regulates chemical diversity in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18378656" target=_blank>Atoui A, <i>et al.</i> (2008 Jun)</a>. Aspergillus nidulans natural product biosynthesis is regulated by mpkB, a putative pheromone response mitogen-activated protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18457952" target=_blank>Peñalva MA, <i>et al.</i> (2008 Jun)</a>. Ambient pH gene regulation in fungi: making connections.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18332432" target=_blank>Andersen MR, <i>et al.</i> (2008 Mar 18)</a>. A trispecies Aspergillus microarray: comparative transcriptomics of three Aspergillus species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20816830" target=_blank>Atoui A, <i>et al.</i> (2010 Dec)</a>. Cross-talk between light and glucose regulation controls toxin production and morphogenesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20935144" target=_blank>Shaaban MI, <i>et al.</i> (2010 Dec)</a>. Suppressor mutagenesis identifies a velvet complex remediator of Aspergillus nidulans secondary metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132440" target=_blank>Reyes-Dominguez Y, <i>et al.</i> (2010 Jun)</a>. Heterochromatic marks are associated with the repression of secondary metabolism clusters in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21062371" target=_blank>Nahlik K, <i>et al.</i> (2010 Nov)</a>. The COP9 signalosome mediates transcriptional and metabolic response to hormones, oxidative stress protection and cell wall rearrangement during fungal development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783880" target=_blank>Ramamoorthy V, <i>et al.</i> (2012 Aug)</a>. veA-dependent RNA-pol II transcription elongation factor-like protein, RtfA, is associated with secondary metabolism and morphological development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24066102" target=_blank>Ramamoorthy V, <i>et al.</i> (2013)</a>. The putative C2H2 transcription factor MtfA is a novel regulator of secondary metabolism and morphogenesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23840895" target=_blank>Shantappa S, <i>et al.</i> (2013)</a>. Role of the zinc finger transcription factor SltA in morphogenesis and sterigmatocystin biosynthesis in the fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23658706" target=_blank>de Souza WR, <i>et al.</i> (2013)</a>. Identification of metabolic pathways influenced by the G-protein coupled receptors GprB and GprD in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26580075" target=_blank>Kowalczyk JE, <i>et al.</i> (2015)</a>. Genetic Interaction of Aspergillus nidulans galR, xlnR and araR in Regulating D-Galactose and L-Arabinose Release and Catabolism Gene Expression.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00527_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00527_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00527_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00527_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00527">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadff0" class="col">
			FOG00528<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 18
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>3-oxoacyl-[acyl-carrier protein] reductase|Has domain(s) with predicted oxidoreductase activity and role in metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00528_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00528_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00528_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00528_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00528">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f2c0c1" class="col">
			FOG00529<br>EOG86Q59T<br>EOG8X69T1<br><br>sce:absent<br><br>Genes: 17
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>3-oxoacyl-[acyl-carrier-protein] reductase|Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Has domain(s) with predicted oxidoreductase activity and role in metabolic process|3-oxoacyl-[acyl-carrier-protein] reductase|Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Has domain(s) with predicted oxidoreductase activity and role in metabolic process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22329759" target=_blank>Lo HC, <i>et al.</i> (2012 Mar 14)</a>. Two separate gene clusters encode the biosynthetic pathway for the meroterpenoids austinol and dehydroaustinol in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00529_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00529_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00529_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00529_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00529">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#cbd9eb" class="col">
			FOG00530<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 15
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Predicted mannitol dehydrogenase, expressed in conidiospores during sporulation<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17568774" target=_blank>Hortschansky P, <i>et al.</i> (2007 Jul 11)</a>. Interaction of HapX with the CCAAT-binding complex--a novel mechanism of gene regulation by iron.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00530_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00530_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00530_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00530_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00530">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d0eacb" class="col">
			FOG00531<br>EOG8PG4JC<br>EOG8X69T1<br><br>sce:absent<br><br>Genes: 15
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>3-oxoacyl-[acyl-carrier-protein]reductase (predicted)<br><br>
			<br><b>AspGD Description</b><br>3-oxoacyl-[acyl-carrier protein] reductase|Has domain(s) with predicted oxidoreductase activity and role in metabolic process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15328121" target=_blank>Graminha MA, <i>et al.</i> (2004 Sep)</a>. Terbinafine resistance mediated by salicylate 1-monooxygenase in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505728" target=_blank>Watanabe S, <i>et al.</i> (2008 Jul 18)</a>. Eukaryotic and bacterial gene clusters related to an alternative pathway of nonphosphorylated L-rhamnose metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18400031" target=_blank>Koivistoinen OM, <i>et al.</i> (2008 May)</a>. Identification in the yeast Pichia stipitis of the first L-rhamnose-1-dehydrogenase gene.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504829" target=_blank>Stewart EV, <i>et al.</i> (2011 Apr 22)</a>. Yeast SREBP cleavage activation requires the Golgi Dsc E3 ligase complex.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00531_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00531_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00531_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00531_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00531">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e0cce5" class="col">
			FOG00532<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 14
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Mannitol 2-dehydrogenase (NADP+)|Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Putative NAD-dependent D-arabinitol dehydrogenase|Has domain(s) with predicted oxidoreductase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18332432" target=_blank>Andersen MR, <i>et al.</i> (2008 Mar 18)</a>. A trispecies Aspergillus microarray: comparative transcriptomics of three Aspergillus species.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00532_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00532_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00532_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00532_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00532">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fadcb8" class="col">
			FOG00533<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>short chain dehydrogenase (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nucleus localization|Has domain(s) with predicted oxidoreductase activity and role in metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00533_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00533_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00533_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00533_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00533">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffc3" class="col">
			FOG00534<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 8
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00534_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00534_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00534_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00534_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00534">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e3d0c3" class="col">
			FOG00535<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity and role in metabolic process|3-oxoacyl-[acyl-carrier protein] reductase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00535_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00535_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00535_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00535_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00535">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d9ed" class="col">
			FOG00536<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>short chain dehydrogenase (predicted)<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18257517" target=_blank>Wilson-Grady JT, <i>et al.</i> (2008 Mar)</a>. Phosphoproteome analysis of fission yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19547744" target=_blank>Beltrao P, <i>et al.</i> (2009 Jun 16)</a>. Evolution of phosphoregulation: comparison of phosphorylation patterns across yeast species.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20531409" target=_blank>Amorim MJ, <i>et al.</i> (2010 Jun 8)</a>. Global coordination of transcriptional control and mRNA decay during cellular differentiation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25452419" target=_blank>Sideri T, <i>et al.</i> (2014 Dec 1)</a>. Parallel profiling of fission yeast deletion mutants for proliferation and for lifespan during long-term quiescence.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojardín L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00536_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00536_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00536_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00536_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00536">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f0b6b7" class="col">
			FOG00537<br>EOG86Q59T<br>EOG88PK3R<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog of Aspergillus tubingensis : Asptu1_0132215, Aspergillus brasiliensis : Aspbr1_0158085, Aspergillus kawachii : Aspka1_0180593 and Aspergillus acidus : Aspfo1_0135998<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00537_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00537_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00537_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00537_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00537">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c3d3e7" class="col">
			FOG00538<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 6
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Carbonyl reductase. Highest activity with 2,3-butanedione.<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15056898" target=_blank>Yamamoto H, <i>et al.</i> (2004 Mar)</a>. A novel NADH-dependent carbonyl reductase from Kluyveromyces aestuarii and comparison of NADH-regeneration system for the synthesis of ethyl (S)-4-chloro-3-hydroxybutanoate.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00538_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00538_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00538_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00538_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00538">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c9e6c3" class="col">
			FOG00539<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>3-oxoacyl-[acyl-carrier protein] reductase<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00539_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00539_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00539_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00539_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00539">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dbc4e1" class="col">
			FOG00540<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00540_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00540_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00540_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00540_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00540">Phobius transmembrane predictions</a>
										<br>4 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fad7ad" class="col">
			FOG00541<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have intracellular localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17568774" target=_blank>Hortschansky P, <i>et al.</i> (2007 Jul 11)</a>. Interaction of HapX with the CCAAT-binding complex--a novel mechanism of gene regulation by iron.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00541_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00541_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00541_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00541_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00541">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb9" class="col">
			FOG00542<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have tetrahydroxynaphthalene reductase activity, versicolorin reductase activity<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8017929" target=_blank>Keller NP, <i>et al.</i> (1994 May)</a>. Aspergillus nidulans verA is required for production of the mycotoxin sterigmatocystin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7546572" target=_blank>Keller NP, <i>et al.</i> (1995)</a>. Analysis of a mycotoxin gene cluster in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7486998" target=_blank>Keller NP, <i>et al.</i> (1995 Oct)</a>. stcS, a putative P-450 monooxygenase, is required for the conversion of versicolorin A to sterigmatocystin in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8643646" target=_blank>Brown DW, <i>et al.</i> (1996 Feb 20)</a>. Twenty-five coregulated transcripts define a sterigmatocystin gene cluster in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8662194" target=_blank>Yu JH, <i>et al.</i> (1996 May)</a>. Conservation of structure and function of the aflatoxin regulatory gene aflR from Aspergillus nidulans and A. flavus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9305634" target=_blank>Hicks JK, <i>et al.</i> (1997 Aug 15)</a>. Aspergillus sporulation and mycotoxin production both require inactivation of the FadA G alpha protein-dependent signaling pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999832" target=_blank>Kelkar HS, <i>et al.</i> (1997 Jan 17)</a>. Aspergillus nidulans stcL encodes a putative cytochrome P-450 monooxygenase required for bisfuran desaturation during aflatoxin/sterigmatocystin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18945083" target=_blank>Keller NP, <i>et al.</i> (1997 Jun)</a>. pH Regulation of Sterigmatocystin and Aflatoxin Biosynthesis in Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15012504" target=_blank>Payne GA, <i>et al.</i> (1998)</a>. Genetics and physiology of aflatoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680223" target=_blank>Fernandes M, <i>et al.</i> (1998 Jun)</a>. Sequence-specific binding by Aspergillus nidulans AflR, a C6 zinc cluster protein regulating mycotoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11156981" target=_blank>Shimizu K, <i>et al.</i> (2001 Feb)</a>. Genetic involvement of a cAMP-dependent protein kinase in a G protein signaling pathway regulating morphological and chemical transitions in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11260466" target=_blank>Hicks J, <i>et al.</i> (2001 Mar)</a>. RcoA has pleiotropic effects on Aspergillus nidulans cellular development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12410837" target=_blank>Jin Y, <i>et al.</i> (2002 Nov)</a>. Requirement of spermidine for developmental transitions in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665453" target=_blank>Kato N, <i>et al.</i> (2003 Dec)</a>. The expression of sterigmatocystin and penicillin genes in Aspergillus nidulans is controlled by veA, a gene required for sexual development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15075281" target=_blank>Bok JW, <i>et al.</i> (2004 Apr)</a>. LaeA, a regulator of secondary metabolism in Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15022028" target=_blank>Yabe K, <i>et al.</i> (2004 Jun)</a>. Enzyme reactions and genes in aflatoxin biosynthesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21148941" target=_blank>Wilkinson HH, <i>et al.</i> (2004 Nov-Dec)</a>. Increased conidiation associated with progression along the sterigmatocystin biosynthetic pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16134890" target=_blank>Maggio-Hall LA, <i>et al.</i> (2005 Aug)</a>. Fundamental contribution of beta-oxidation to polyketide mycotoxin production in planta.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16467480" target=_blank>Seo JA, <i>et al.</i> (2006 Feb)</a>. The phosducin-like protein PhnA is required for Gbetagamma-mediated signaling for vegetative growth, developmental control, and toxin biosynthesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16420358" target=_blank>Tsitsigiannis DI, <i>et al.</i> (2006 Feb)</a>. Oxylipins act as determinants of natural product biosynthesis and seed colonization in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16426969" target=_blank>Bok JW, <i>et al.</i> (2006 Jan)</a>. Genomic mining for Aspergillus natural products.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16387865" target=_blank>Seo JA, <i>et al.</i> (2006 Mar)</a>. FluG-dependent asexual development in Aspergillus nidulans occurs via derepression.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16968230" target=_blank>Bok JW, <i>et al.</i> (2006 Sep)</a>. Secondary metabolic gene cluster silencing in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616629" target=_blank>Shwab EK, <i>et al.</i> (2007 Sep)</a>. Histone deacetylase activity regulates chemical diversity in Aspergillus.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18378656" target=_blank>Atoui A, <i>et al.</i> (2008 Jun)</a>. Aspergillus nidulans natural product biosynthesis is regulated by mpkB, a putative pheromone response mitogen-activated protein kinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18457952" target=_blank>Peñalva MA, <i>et al.</i> (2008 Jun)</a>. Ambient pH gene regulation in fungi: making connections.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20816830" target=_blank>Atoui A, <i>et al.</i> (2010 Dec)</a>. Cross-talk between light and glucose regulation controls toxin production and morphogenesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20935144" target=_blank>Shaaban MI, <i>et al.</i> (2010 Dec)</a>. Suppressor mutagenesis identifies a velvet complex remediator of Aspergillus nidulans secondary metabolism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132440" target=_blank>Reyes-Dominguez Y, <i>et al.</i> (2010 Jun)</a>. Heterochromatic marks are associated with the repression of secondary metabolism clusters in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21062371" target=_blank>Nahlik K, <i>et al.</i> (2010 Nov)</a>. The COP9 signalosome mediates transcriptional and metabolic response to hormones, oxidative stress protection and cell wall rearrangement during fungal development.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783880" target=_blank>Ramamoorthy V, <i>et al.</i> (2012 Aug)</a>. veA-dependent RNA-pol II transcription elongation factor-like protein, RtfA, is associated with secondary metabolism and morphological development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24066102" target=_blank>Ramamoorthy V, <i>et al.</i> (2013)</a>. The putative C2H2 transcription factor MtfA is a novel regulator of secondary metabolism and morphogenesis in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23840895" target=_blank>Shantappa S, <i>et al.</i> (2013)</a>. Role of the zinc finger transcription factor SltA in morphogenesis and sterigmatocystin biosynthesis in the fungus Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23658706" target=_blank>de Souza WR, <i>et al.</i> (2013)</a>. Identification of metabolic pathways influenced by the G-protein coupled receptors GprB and GprD in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00542_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00542_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00542_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00542_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00542">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dfc9ba" class="col">
			FOG00543<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00543_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00543_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00543_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00543_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00543">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8d4ea" class="col">
			FOG00544<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Has domain(s) with predicted oxidoreductase activity and role in metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00544_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00544_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00544_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00544_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00544">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#eeadad" class="col">
			FOG00545<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22329759" target=_blank>Lo HC, <i>et al.</i> (2012 Mar 14)</a>. Two separate gene clusters encode the biosynthetic pathway for the meroterpenoids austinol and dehydroaustinol in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00545_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00545_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00545_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00545_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00545">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#bacde4" class="col">
			FOG00546<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity and role in metabolic process<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00546_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00546_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00546_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00546_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00546">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#c1e3bb" class="col">
			FOG00547<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 7
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Secoisolariciresinol dehydrogenase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00547_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00547_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00547_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00547_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00547">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7bcdd" class="col">
			FOG00548<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>3-hydroxyacyl-CoA dehydrogenase (predicted)<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00548_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00548_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00548_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00548_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00548">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f9d2a2" class="col">
			FOG00549<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 5
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Putative 3-oxoacyl-(acyl-carrier-protein) reductase; induced by fenpropimorph<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00549_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00549_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00549_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00549_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00549">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffb0" class="col">
			FOG00550<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Parent</b><br>paralog:FOG00548<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have cytosol, nucleus localization<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00550_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00550_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00550_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00550_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00550">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#dac1b0" class="col">
			FOG00551<br>EOG86Q59T<br>SPS19<br>sce:SPS19<br><br>Genes: 34
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Peroxisomal 2,4-dienoyl-CoA reductase; auxiliary enzyme of fatty acid beta-oxidation; homodimeric enzyme required for growth and sporulation on petroselineate medium; expression induced during late sporulation and in the presence of oleate<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have 2,4-dienoyl-CoA reductase (NADPH) activity, role in ascospore formation, fatty acid catabolic process and peroxisomal matrix localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7969036" target=_blank>Coe JG, <i>et al.</i> (1994 Sep 28)</a>. Identification of a sporulation-specific promoter regulating divergent transcription of two novel sporulation genes in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9268358" target=_blank>Gurvitz A, <i>et al.</i> (1997 Aug 29)</a>. The Saccharomyces cerevisiae peroxisomal 2,4-dienoyl-CoA reductase is encoded by the oleate-inducible gene SPS19.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9427398" target=_blank>Gurvitz A, <i>et al.</i> (1997 Nov)</a>. Regulation of the yeast SPS19 gene encoding peroxisomal 2,4-dienoyl-CoA reductase by the transcription factors Pip2p and Oaf1p: beta-oxidation is dispensable for Saccharomyces cerevisiae sporulation in acetate medium.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11170837" target=_blank>Gurvitz A, <i>et al.</i> (2000 Aug)</a>. Adr1p-dependent regulation of the oleic acid-inducible yeast gene SPS19 encoding the peroxisomal beta-oxidation auxiliary enzyme 2,4-dienoyl-CoA reductase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22106047" target=_blank>Starita LM, <i>et al.</i> (2012 Jan)</a>. Sites of ubiquitin attachment in Saccharomyces cerevisiae.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00551_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00551_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00551_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00551_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00551">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#f8cee7" class="col">
			FOG00552<br>EOG86Q59T<br><br>sce:absent<br><br>Genes: 14
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Has domain(s) with predicted 2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase activity, oxidoreductase activity and role in metabolic process, oxidation-reduction process, siderophore biosynthetic process|Has domain(s) with predicted oxidoreductase activity|Ortholog of A. nidulans FGSC A4 : AN6274, A. fumigatus Af293 : Afu7g04540, A. niger CBS 513.88 : An13g01130 and A. oryzae RIB40 : AO090005000138|Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Has domain(s) with predicted oxidoreductase activity and role in metabolic process|Putative levodione reductase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610199" target=_blank>Flipphi M, <i>et al.</i> (2009 Mar)</a>. Biodiversity and evolution of primary carbon metabolism in Aspergillus nidulans  and other Aspergillus spp.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG00552_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00552_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG00552_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG00552_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG00243_phobius.html#FOG00552">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG00243.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>