<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG00756.html">HOG00756&emsp;&emsp;esterase D family</a><br> 
																	HOG00757&emsp;&emsp;<br> 
																	<a href="HOG00759.html">HOG00759&emsp;&emsp;glycosyltransferase 20 family; trehalose phosphatase family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG00757.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG00757_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG00757_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG00757_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00757_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG01776<br>EOG851C79<br>FDH1<br>sce:FDH1<br><br>Genes: 38
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Cytoplasmic formate dehydrogenase<br><br>
			<br><b>Features</b><br>[c]<br><br>
			<br><b>SGD Description</b><br>NAD(+)-dependent formate dehydrogenase; may protect cells from exogenous formate<br><br>
			<br><b>AspGD Description</b><br>Putative formate dehydrogenase; acetate-inducible gene<br><br>
			<br><b>Suggested Analysis</b><br>zro/sce may have had HGT from CTG clade given conserved insertion seen in msa<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3916805" target=_blank>Atkinson PW, <i>et al.</i> (1985)</a>. Identification of the aciA gene controlled by the amdA regulatory gene in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3551935" target=_blank>Hynes MJ, <i>et al.</i> (1986 Sep)</a>. The amdS gene of Aspergillus nidulans: control by multiple regulatory signals.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1465107" target=_blank>Saleeba JA, <i>et al.</i> (1992 Nov)</a>. Characterization of the amdA-regulated aciA gene of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765128" target=_blank>Hynes MJ, <i>et al.</i> (1994)</a>. Regulation of acetamide utilization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596297" target=_blank>Lints R, <i>et al.</i> (1995 Mar)</a>. The positively acting amdA gene of Aspergillus nidulans encodes a protein with two C2H2 zinc-finger motifs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9126617" target=_blank>Andrianopoulos A, <i>et al.</i> (1997 Feb)</a>. The amdA regulatory gene of Aspergillus nidulans: characterization of gain-of-function mutations and identification of binding sites for the gene product.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9178506" target=_blank>van den Berg MA, <i>et al.</i> (1997 May)</a>. Expression cassettes for formaldehyde and fluoroacetate resistance, two dominant markers in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11921099" target=_blank>Overkamp KM, <i>et al.</i> (2002 Apr)</a>. Functional analysis of structural genes for NAD(+)-dependent formate dehydrogenase in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12144528" target=_blank>Serov AE, <i>et al.</i> (2002 Nov 1)</a>. Engineering of coenzyme specificity of formate dehydrogenase from Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19784673" target=_blank>Salazar M, <i>et al.</i> (2009 Dec)</a>. Uncovering transcriptional regulation of glycerol metabolism in Aspergilli through genome-wide gene expression data analysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20797444" target=_blank>Pusztahelyi T, <i>et al.</i> (2011 Feb)</a>. Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22792266" target=_blank>Etxebeste O, <i>et al.</i> (2012)</a>. GmcA is a putative glucose-methanol-choline oxidoreductase required for the induction of asexual development in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22835028" target=_blank>Saykhedkar S, <i>et al.</i> (2012 Jul 26)</a>. A time course analysis of the extracellular proteome of Aspergillus nidulans growing on sorghum stover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24120530" target=_blank>Martins I, <i>et al.</i> (2013 Dec 6)</a>. Proteomic alterations induced by ionic liquids in Aspergillus nidulans and Neurospora crassa.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23633078" target=_blank>Sun X, <i>et al.</i> (2013 May)</a>. PyrG is required for maintaining stable cellular uracil level and normal sporulation pattern under excess uracil stress in Aspergillus nidulans.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01776_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01776_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01776_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01776_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00757_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG01777<br>EOG851C79<br>FDH1.2<br>sce:absent<br><br>Genes: 2
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial formate dehydrogenase<br><br>
			<br><b>Features</b><br>[m]<br><br>
			<br><b>Parent</b><br>paralog:FOG01776<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01777_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01777_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01777_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01777_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00757_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG01778<br>EOG851C79<br><br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Uncharacterized paralog, does not have full FDH domain<br><br>
			<br><b>Parent</b><br>paralog:FOG01776<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01778_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01778_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01778_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01778_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00757_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG01779<br><br>FDH1.3<br>sce:absent<br><br>Genes: 1
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>Protein description</b><br>Mitochondrial formate dehydrogenase<br><br>
			<br><b>Features</b><br>[m]<br><br>
			<br><b>Parent</b><br>paralog:FOG01776<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01779_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01779_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG01779_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG01779_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG00757_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG00757.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>