<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG01206.html">HOG01206&emsp;&emsp;</a><br> 
																	HOG01207&emsp;&emsp;<br> 
																	<a href="HOG01208.html">HOG01208&emsp;&emsp;STB3 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG01207.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG01207_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG01207_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01207_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG03277<br>EOG8Z3509<br><br>sce:MVP1<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Protein required for sorting proteins to the vacuole; Mvp1p and Vps1p act in concert to promote membrane traffic to the vacuole; participates in transcription initiation and/or early elongation of specific genes; interacts with "foot domain" of RNA polymerase II; deletion results in abnormal CTD-Ser5 phosphorylation of RNA polymerase II at specific promoter regions; protein abundance increases in response to DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>sorting nexin Mvp1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have Golgi apparatus, fungal-type vacuole membrane localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862158" target=_blank>Ekena K, <i>et al.</i> (1995 Mar)</a>. The Saccharomyces cerevisiae MVP1 gene interacts with VPS1 and is required for vacuolar protein sorting.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15937140" target=_blank>Chibana H, <i>et al.</i> (2005 Aug)</a>. Sequence finishing and gene mapping for Candida albicans chromosome 7 and syntenic analysis against the Saccharomyces cerevisiae genome.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03277_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03277_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03277_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03277_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01207_phobius.pdf">Phobius transmembrane predictions</a>
										<br>2 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d3dfee" class="col">
			FOG03278<br>EOG8Z3509<br><br>sce:SNX3<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Sorting nexin for late-Golgi enzymes; required to maintain late-Golgi resident enzymes in their proper location by recycling molecules from the prevacuolar compartment; contains a PX domain and sequence similarity to human Snx3p<br><br>
			<br><b>PomBase Description</b><br>sorting nexin Snx3 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have phosphatidylinositol-3-phosphate binding activity, role in late endosome to Golgi transport, protein localization and cytosol, endosome, nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9456318" target=_blank>Voos W, <i>et al.</i> (1998 Feb 9)</a>. Retrieval of resident late-Golgi membrane proteins from the prevacuolar compartment of Saccharomyces cerevisiae is dependent on the function of Grd19p.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14514667" target=_blank>Zhou CZ, <i>et al.</i> (2003 Dec 12)</a>. Crystal structure of the yeast Phox homology (PX) domain protein Grd19p complexed to phosphatidylinositol-3-phosphate.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15189449" target=_blank>Koga T, <i>et al.</i> (2004 Jun)</a>. Sorting nexin homologues are targets of phosphatidylinositol 3-phosphate in sporulation of Schizosaccharomyces pombe.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15263065" target=_blank>Vollert CS, <i>et al.</i> (2004 Nov)</a>. The phox homology (PX) domain protein interaction network in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17287358" target=_blank>Chi A, <i>et al.</i> (2007 Feb 13)</a>. Analysis of phosphorylation sites on proteins from Saccharomyces cerevisiae by electron transfer dissociation (ETD) mass spectrometry.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22384365" target=_blank>Wendland J, <i>et al.</i> (2011 Dec)</a>. Genome evolution in the eremothecium clade of the Saccharomyces complex revealed by comparative genomics.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22634043" target=_blank>Wartenberg D, <i>et al.</i> (2012 Jul 16)</a>. Proteome analysis of the farnesol-induced stress response in Aspergillus nidulans--The role of a putative dehydrin.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24763107" target=_blank>Carpy A, <i>et al.</i> (2014 Aug)</a>. Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccharomyces pombe (Fission Yeast).<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25483073" target=_blank>Mojard√≠n L, <i>et al.</i> (2015)</a>. Chromosome segregation and organization are targets of 5'-Fluorouracil in eukaryotic cells.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03278_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03278_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03278_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03278_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01207_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#d7edd3" class="col">
			FOG03279<br>EOG8Z3509<br><br>sce:SNX4<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Sorting nexin; involved in retrieval of late-Golgi SNAREs from post-Golgi endosomes to the trans-Golgi network and in cytoplasm to vacuole transport; contains a PX phosphoinositide-binding domain; forms complexes with Snx41p and with Atg20p<br><br>
			<br><b>PomBase Description</b><br>autophagy associated protein Atg24 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Sorting nexin; predicted role in autophagy<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8663607" target=_blank>Harding TM, <i>et al.</i> (1996 Jul 26)</a>. Genetic and phenotypic overlap between autophagy and the cytoplasm to vacuole protein targeting pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12048214" target=_blank>Nice DC, <i>et al.</i> (2002 Aug 16)</a>. Cooperative binding of the cytoplasm to vacuole targeting pathway proteins, Cvt13 and Cvt20, to phosphatidylinositol 3-phosphate at the pre-autophagosomal structure is required for selective autophagy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12554655" target=_blank>Hettema EH, <i>et al.</i> (2003 Feb 3)</a>. Retromer and the sorting nexins Snx4/41/42 mediate distinct retrieval pathways from yeast endosomes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14536056" target=_blank>Klionsky DJ, <i>et al.</i> (2003 Oct)</a>. A unified nomenclature for yeast autophagy-related genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15752076" target=_blank>Muthuvijayan V, <i>et al.</i> (2004)</a>. In silico reconstruction of nutrient-sensing signal transduction pathways in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15800066" target=_blank>Krsmanovic T, <i>et al.</i> (2005 Jun)</a>. Control of Ste6 recycling by ubiquitination in the early endocytic pathway in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17420293" target=_blank>Strochlic TI, <i>et al.</i> (2007 Apr 9)</a>. Grd19/Snx3p functions as a cargo-specific adapter for retromer-dependent endocytic recycling.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18818209" target=_blank>Kanki T, <i>et al.</i> (2008 Nov 21)</a>. Mitophagy in yeast occurs through a selective mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19793921" target=_blank>Kanki T, <i>et al.</i> (2009 Nov)</a>. A genomic screen for yeast mutants defective in selective mitochondria autophagy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861302" target=_blank>Ohashi Y, <i>et al.</i> (2010 Nov 15)</a>. Membrane delivery to the yeast autophagosome from the Golgi-endosomal system.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729555" target=_blank>LeBlanc MA, <i>et al.</i> (2010 Oct 29)</a>. Lipid binding requirements for oxysterol-binding protein Kes1 inhibition of autophagy and endosome-trans-Golgi trafficking pathways.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21429936" target=_blank>Mendl N, <i>et al.</i> (2011 Apr 15)</a>. Mitophagy in yeast is independent of mitochondrial fission and requires the stress response gene WHI2.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03279_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03279_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03279_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03279_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01207_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#e5d4e9" class="col">
			FOG03280<br>EOG8Z3509<br><br>sce:absent<br><br>Genes: 11
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Has domain(s) with predicted phosphatidylinositol binding activity and role in cell communication<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03280_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03280_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03280_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03280_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01207_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fbe2c3" class="col">
			FOG03281<br>EOG8Z3509<br><br>sce:absent<br><br>Genes: 4
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
			&nbsp;<br><br><br><br><br><br>&nbsp;
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03281_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03281_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03281_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03281_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01207_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->

		<!--ortholog information:start-->
		<div id="left_col" style="background:#fdffcc" class="col">
			FOG03282<br>EOG8Z3509<br><br>sce:absent<br><br>Genes: 3
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>PomBase Description</b><br>autophagy associated protein (predicted)<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03282_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03282_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG03282_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG03282_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG01207_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG01207.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>