<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:225px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG12850.html">HOG12850&emsp;&emsp;</a><br> 
																	HOG12857&emsp;&emsp;<br> 
																	<a href="HOG12859.html">HOG12859&emsp;&emsp;WD repeat SWD2 family</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG12857.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG12857_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG12857_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG12857_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG12857_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG17070<br>EOG86M92V<br><br>sce:NTG1;NTG2<br><br>Genes: 33
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>DNA N-glycosylase and apurinic/apyrimidinic (AP) lyase; involved in base excision repair; acts in both nucleus and mitochondrion; creates a double-strand break at mtDNA origins that stimulates replication in response to oxidative stress; required for maintaining mitochondrial genome integrity; NTG1 has a paralog, NTG2, that arose from the whole genome duplication|DNA N-glycosylase and apurinic/apyrimidinic (AP) lyase; involved in base excision repair, localizes to the nucleus; sumoylated; NTG2 has a paralog, NTG1, that arose from the whole genome duplication<br><br>
			<br><b>PomBase Description</b><br>DNA endonuclease III<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have 5-formyluracil DNA N-glycosylase activity, 5-hydroxymethyluracil DNA N-glycosylase activity, 8-oxo-7,8-dihydroguanine DNA N-glycosylase activity, role in base-excision repair, AP site formation and nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144453" target=_blank>Barton AB, <i>et al.</i> (1994 Apr)</a>. Molecular cloning of chromosome I DNA from Saccharomyces cerevisiae: analysis of the genes in the FUN38-MAK16-SPO7 region.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8805338" target=_blank>Nash HM, <i>et al.</i> (1996 Aug 1)</a>. Cloning of a yeast 8-oxoguanine DNA glycosylase reveals the existence of a base-excision DNA-repair protein superfamily.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8855249" target=_blank>Eide L, <i>et al.</i> (1996 Oct 1)</a>. Base excision of oxidative purine and pyrimidine DNA damage in Saccharomyces cerevisiae by a DNA glycosylase with sequence similarity to endonuclease III from Escherichia coli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9020769" target=_blank>Augeri L, <i>et al.</i> (1997 Jan 28)</a>. Purification, characterization, gene cloning, and expression of Saccharomyces cerevisiae redoxyendonuclease, a homolog of Escherichia coli endonuclease III.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9558341" target=_blank>You HJ, <i>et al.</i> (1998 Apr 28)</a>. Saccharomyces cerevisiae possesses two functional homologues of Escherichia coli endonuclease III.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9826748" target=_blank>Sent√ºrker S, <i>et al.</i> (1998 Dec 1)</a>. Substrate specificities of the ntg1 and ntg2 proteins of Saccharomyces cerevisiae for oxidized DNA bases are not identical.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9545197" target=_blank>Bruner SD, <i>et al.</i> (1998 Mar 26)</a>. Repair of oxidatively damaged guanine in Saccharomyces cerevisiae by an alternative pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10471279" target=_blank>You HJ, <i>et al.</i> (1999 Aug 31)</a>. Saccharomyces cerevisiae Ntg1p and Ntg2p: broad specificity N-glycosylases for the repair of oxidative DNA damage in the nucleus and mitochondria.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10207101" target=_blank>Alseth I, <i>et al.</i> (1999 May)</a>. The Saccharomyces cerevisiae homologues of endonuclease III from Escherichia coli, Ntg1 and Ntg2, are both required for efficient repair of spontaneous and induced oxidative DNA damage in yeast.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11478780" target=_blank>Kim JE, <i>et al.</i> (2001 Aug 3)</a>. Ntg2 of Saccharomyces cerevisiae repairs the oxidation products of 8-hydroxyguanine.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12042306" target=_blank>Gellon L, <i>et al.</i> (2002 Aug 16)</a>. Ntg2p, a Saccharomyces cerevisiae DNA N-glycosylase/apurinic or apyrimidinic lyase involved in base excision repair of oxidative DNA damage, interacts with the DNA mismatch repair protein Mlh1p. Identification of a Mlh1p binding motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500818" target=_blank>Meadows KL, <i>et al.</i> (2003 Oct 1)</a>. Characterization of AP lyase activities of Saccharomyces cerevisiae Ntg1p and Ntg2p: implications for biological function.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14697759" target=_blank>Hanna M, <i>et al.</i> (2004 Jan 5)</a>. Involvement of two endonuclease III homologs in the base excision repair pathway for the processing of DNA alkylation damage in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15449310" target=_blank>Melo RG, <i>et al.</i> (2004 Sep)</a>. Role of OGG1 and NTG2 in the repair of oxidative DNA damage and mutagenesis induced by hydrogen peroxide in Saccharomyces cerevisiae: relationships with transition metals iron and copper.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15923634" target=_blank>Doudican NA, <i>et al.</i> (2005 Jun)</a>. Oxidative DNA damage causes mitochondrial genomic instability in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16730479" target=_blank>Phadnis N, <i>et al.</i> (2006 Jul 13)</a>. Ntg1p, the base excision repair protein, generates mutagenic intermediates in yeast mitochondrial DNA.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19029246" target=_blank>Griffiths LM, <i>et al.</i> (2009 Feb)</a>. Dynamic compartmentalization of base excision repair proteins in response to nuclear and mitochondrial oxidative stress.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074198" target=_blank>Hori A, <i>et al.</i> (2009 Feb)</a>. Reactive oxygen species regulate DNA copy number in isolated yeast mitochondria by triggering recombination-mediated replication.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18983898" target=_blank>Gasparutto D, <i>et al.</i> (2009 Jan 1)</a>. Excision of the oxidatively formed 5-hydroxyhydantoin and 5-hydroxy-5-methylhydantoin pyrimidine lesions by Escherichia coli and Saccharomyces cerevisiae DNA N-glycosylases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20194111" target=_blank>Swartzlander DB, <i>et al.</i> (2010 Jul)</a>. Regulation of base excision repair: Ntg1 nuclear and mitochondrial dynamic localization in response to genotoxic stress.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG17070_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG17070_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG17070_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG17070_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG12857_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG12857.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>