<html>
<head><title>HOG15394</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111512446-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111512446-1');
</script>


</head>





<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:265px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>

		<div>
			<form name="f" action="http://lmse.utoronto.ca/aybrah/hl.php" method="post">
				<b>Lookup String:</b> <input type="text" value="" size="15" name=text>
				<input type="submit" value="Search" name=search>
			</form>
		</div>


																	<a href="HOG15393.html">HOG15393&emsp;&emsp;NCA2 family</a><br> 
																	HOG15394&emsp;&emsp;<br> 
																	<a href="HOG15398.html">HOG15398&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG15394.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG15394_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG15394_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG15394_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG15394_phobius.pdf">Phobius predictions</a><br><br>
																	<a href="https://github.com/lmse/aybrah/issues/new?body=%23%20Description%20of%20the%20issue%0A%0A%0A%0A%23%20Page%0Ahttps://lmse.github.io/aybrah/website/HOG15394.html">Report suggestion on GitHub</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG19608<br>EOG802V70<br><br>sce:absent<br><br>Genes: 12
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>AspGD Description</b><br>Xanthine dehydrogenase|Ortholog(s) have xanthine dehydrogenase activity and role in purine nucleobase catabolic process<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14085568" target=_blank>PATEMAN JA, <i>et al.</i> (1964 Jan 4)</a>. A COMMON CO-FACTOR FOR NITRATE REDUCTASE AND  XANTHINE DEHYDROGENASE WHICH ALSO REGULATES THE SYNTHESIS OF NITRATE REDUCTASE.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5583651" target=_blank>Scazzocchio C, <i>et al.</i> (1967 Dec 18)</a>. The genetic control of xanthine dehydrogenase and urate oxidase synthess in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5680610" target=_blank>Scazzocchio C, <i>et al.</i> (1968 Sep 24)</a>. The induction and repression of the enzymes of purine breakdown in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5370743" target=_blank>Alderson T, <i>et al.</i> (1969 Nov-Dec)</a>. Spontaneous and induced reversion of ICR-170-induced xanthine dehydrogenase mutants of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5366006" target=_blank>Alderson T, <i>et al.</i> (1969 Sep-Oct)</a>. Specificity for spontaneous and induced forward mutation at several gene loci in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5475567" target=_blank>Arst HN Jr, <i>et al.</i> (1970)</a>. Molybdate metabolism in Aspergillus nidulans. I. Mutations affecting nitrate reductase and-or xanthine dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=5489968" target=_blank>Hartley MJ, <i>et al.</i> (1970 Sep)</a>. The frequency of reverse mutation at the XDH loci of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4581274" target=_blank>Scazzocchio C, <i>et al.</i> (1973 Jul 16)</a>. The genetic control of molybdoflavoproteins in Aspergillus nidulans. Allopurinol-resistant mutants constitutive for xanthine-dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4149466" target=_blank>Scazzocchio C, <i>et al.</i> (1973 Sep 5)</a>. The genetic control of molybdoflavoproteins in Aspergillus nidulans. II. Use of NADH dehydrogenase activity associated with xanthine dehydrogenase to investigate substrate and product inductions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=335238" target=_blank>Scott BR, <i>et al.</i> (1977 Oct)</a>. The induction of mutations to 2-thioxanthine resistance in inhibitor depleted conidia of Aspergillus nidulans by gamma-radiation in the presence of oxygen or nitrogen.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=363427" target=_blank>Lewis NJ, <i>et al.</i> (1978 Nov 2)</a>. The genetic control of the molybdoflavoproteins in Aspergillus nidulans. IV. A comparison between purine hydroxylase I and II.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=363429" target=_blank>Scazzocchio C, <i>et al.</i> (1978 Nov 2)</a>. A mutation in the xanthine dehydrogenase (purine hydroxylase I) of Aspergillus nidulans resulting in altered specificity. Implications for the geometry of the active site.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=362158" target=_blank>Sealy-Lewis HM, <i>et al.</i> (1978 Sep 8)</a>. A mutation defective in the xanthine alternative pathway of Aspergillus nidulans: its use to investigate the specificity of uaY mediated induction.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693595" target=_blank>Mehra RK, <i>et al.</i> (1989 Feb)</a>. Characterization of purine hydroxylase I from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7765126" target=_blank>Scazzocchio C, <i>et al.</i> (1994)</a>. The purine degradation pathway, genetics, biochemistry and regulation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7729421" target=_blank>Su√°rez T, <i>et al.</i> (1995 Apr 3)</a>. The sequence and binding specificity of UaY, the specific regulator of the purine utilization pathway in Aspergillus nidulans, suggest an evolutionary relationship with the PPR1 protein of Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7876088" target=_blank>Glatigny A, <i>et al.</i> (1995 Feb 24)</a>. Cloning and molecular characterization of hxA, the gene coding for the xanthine dehydrogenase (purine hydroxylase I) of Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7602582" target=_blank>Oestreicher N, <i>et al.</i> (1995 Jun 16)</a>. A single amino acid change in a pathway-specific transcription factor results in differing degrees of constitutivity, hyperinducibility and derepression of several structural genes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9299197" target=_blank>Clutterbuck AJ, <i>et al.</i> (1997 Jun)</a>. The validity of the Aspergillus nidulans linkage map.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9571062" target=_blank>Glatigny A, <i>et al.</i> (1998 May 1)</a>. Altered specificity mutations define residues essential for substrate positioning in xanthine dehydrogenase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689191" target=_blank>Dessen P, <i>et al.</i> (2000 Feb 22)</a>. The PAUSE software for analysis of translational control over protein targeting: application to E. nidulans membrane proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11810243" target=_blank>Ribard C, <i>et al.</i> (2001 Dec)</a>. The oxpA5 mutation of Aspergillus nidulans is an allele of adB, the gene encoding adenylosuccinate synthetase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16963627" target=_blank>Morozov IY, <i>et al.</i> (2006 Nov)</a>. Nonsense-mediated mRNA decay mutation in Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17880424" target=_blank>Malavazi I, <i>et al.</i> (2007 Oct)</a>. Transcriptome analysis of the Aspergillus nidulans AtmA (ATM, Ataxia-Telangiectasia mutated) null mutant.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG19608_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG19608_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG19608_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG19608_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG15394_phobius.pdf">Phobius transmembrane predictions</a>
										<br>1 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG15394.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>