<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG11913.html">HOG11913&emsp;&emsp;</a><br> 
																	HOG11914&emsp;&emsp;<br> 
																	<a href="HOG11917.html">HOG11917&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG11914.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG11914_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG11914_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG11914_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG16127<br>EOG895X7B<br><br>sce:PSO2<br><br>Genes: 31
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Nuclease required for DNA single- and double-strand break repair; acts at a post-incision step in repair of breaks that result from interstrand cross-links produced by a variety of mono- and bi-functional psoralen derivatives; induced by UV-irradiation; forms nuclear foci upon DNA replication stress<br><br>
			<br><b>PomBase Description</b><br>DNA 5' exonuclease (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have 5'-3' exonuclease activity, damaged DNA binding activity, role in interstrand cross-link repair and nucleus localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736091" target=_blank>Richter D, <i>et al.</i> (1992 Jan)</a>. Molecular structure of the DNA cross-link repair gene SNM1 (PSO2) of the yeast Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7526204" target=_blank>Niegemann E, <i>et al.</i> (1994 Nov)</a>. A single amino acid change in SNM1-encoded protein leads to thermoconditional deficiency for DNA cross-link repair in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8628215" target=_blank>Wolter R, <i>et al.</i> (1996 Feb 5)</a>. Regulation of SNM1, an inducible Saccharomyces cerevisiae gene required for repair of DNA cross-links.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10980408" target=_blank>Grossmann KF, <i>et al.</i> (2000 Sep 15)</a>. Saccharomyces cerevisiae lacking Snm1, Rev3 or Rad51 have a normal S-phase but arrest permanently in G2 after cisplatin treatment.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11738934" target=_blank>Grossmann KF, <i>et al.</i> (2001 Dec 19)</a>. S. cerevisiae has three pathways for DNA interstrand crosslink repair.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177301" target=_blank>Callebaut I, <i>et al.</i> (2002 Aug 15)</a>. Metallo-beta-lactamase fold within nucleic acids processing enzymes: the beta-CASP family.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12509272" target=_blank>Li X, <i>et al.</i> (2003 Jan 2)</a>. The beta-lactamase motif in Snm1 is required for repair of DNA double-strand breaks caused by interstrand crosslinks in S. cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14729978" target=_blank>Yu J, <i>et al.</i> (2004 Feb)</a>. Microhomology-dependent end joining and repair of transposon-induced DNA hairpins by host factors in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15590324" target=_blank>Li X, <i>et al.</i> (2005 Feb 3)</a>. The yeast Snm1 protein is a DNA 5'-exonuclease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743825" target=_blank>Barber LJ, <i>et al.</i> (2005 Mar)</a>. DNA interstrand cross-link repair in the Saccharomyces cerevisiae cell cycle: overlapping roles for PSO2 (SNM1) with MutS factors and EXO1 during S phase.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG16127_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG16127_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG16127_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG16127_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/.*_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG11914.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>