<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../img/aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../img/lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG10014.html">HOG10014&emsp;&emsp;</a><br> 
																	HOG10021&emsp;&emsp;<br> 
																	<a href="HOG10023.html">HOG10023&emsp;&emsp;</a><br><br> 
																	<a href="../sequences/hog/HOG10021.fasta">FASTA</a><br> 
																	<a href="../trees/by_hog/msa/HOG10021_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="../trees/by_hog/phyml/HOG10021_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="../trees/by_hog/gb/HOG10021_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="HOG10021_phobius.html">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG14235<br>EOG8G1K0G<br><br>sce:NMT1<br><br>Genes: 32
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>N-myristoyl transferase; catalyzes the cotranslational, covalent attachment of myristic acid to the N-terminal glycine residue of several proteins involved in cellular growth and signal transduction<br><br>
			<br><b>PomBase Description</b><br>N-myristoyltransferase Myr1 (predicted)<br><br>
			<br><b>AspGD Description</b><br>Ortholog(s) have glycylpeptide N-tetradecanoyltransferase activity, role in establishment of cell polarity and cytosol localization<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3106975" target=_blank>Towler DA, <i>et al.</i> (1987 May)</a>. Purification and characterization of yeast myristoyl CoA:protein N-myristoyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2644694" target=_blank>Duronio RJ, <i>et al.</i> (1989 Feb 10)</a>. Disruption of the yeast N-myristoyl transferase gene causes recessive lethality.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2045414" target=_blank>Duronio RJ, <i>et al.</i> (1991 Jun)</a>. Myristic acid auxotrophy caused by mutation of S. cerevisiae myristoyl-CoA:protein N-myristoyltransferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569105" target=_blank>Wiegand RC, <i>et al.</i> (1992 Apr 25)</a>. The Candida albicans myristoyl-CoA:protein N-myristoyltransferase gene. Isolation and expression in Saccharomyces cerevisiae and Escherichia coli.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8430078" target=_blank>Rudnick DA, <i>et al.</i> (1993 Feb 1)</a>. Use of photoactivatable peptide substrates of Saccharomyces cerevisiae myristoyl-CoA:protein N-myristoyltransferase (Nmt1p) to characterize a myristoyl-CoA-Nmt1p-peptide ternary complex and to provide evidence for an ordered reaction mechanism.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416952" target=_blank>Johnson DR, <i>et al.</i> (1993 Jan 5)</a>. Genetic and biochemical studies of a mutant Saccharomyces cerevisiae myristoyl-CoA:protein N-myristoyltransferase, nmt72pLeu99--&amp;gt;Pro, that produces temperature-sensitive myristic acid auxotrophy.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300631" target=_blank>Lodge JK, <i>et al.</i> (1994 Jan 28)</a>. Comparison of myristoyl-CoA:protein N-myristoyltransferases from three pathogenic fungi: Cryptococcus neoformans, Histoplasma capsulatum, and Candida albicans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955162" target=_blank>Zhang L, <i>et al.</i> (1996 Dec 20)</a>. Biochemical studies of Saccharomyces cerevisiae myristoyl-coenzyme A:protein N-myristoyltransferase mutants.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9115247" target=_blank>McWherter CA, <i>et al.</i> (1997 May 2)</a>. Scanning alanine mutagenesis and de-peptidization of a Candida albicans myristoyl-CoA:protein N-myristoyltransferase octapeptide substrate reveals three elements critical for molecular recognition.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9846880" target=_blank>Bhatnagar RS, <i>et al.</i> (1998 Dec)</a>. Structure of N-myristoyltransferase with bound myristoylCoA and peptide substrate analogs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9501915" target=_blank>Weston SA, <i>et al.</i> (1998 Mar)</a>. Crystal structure of the anti-fungal target N-myristoyl transferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9927451" target=_blank>Momany M, <i>et al.</i> (1999 Feb)</a>. Aspergillus nidulans swo mutants show defects in polarity establishment, polarity maintenance and hyphal morphogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11478885" target=_blank>Farazi TA, <i>et al.</i> (2001 Aug 7)</a>. Pre-steady-state kinetic studies of Saccharomyces cerevisiae myristoylCoA:protein N-myristoyltransferase mutants identify residues involved in catalysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11371195" target=_blank>Farazi TA, <i>et al.</i> (2001 May 29)</a>. Structures of Saccharomyces cerevisiae N-myristoyltransferase with bound myristoylCoA and peptide provide insights about substrate recognition and catalysis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12455958" target=_blank>Shaw BD, <i>et al.</i> (2002 Apr)</a>. Aspergillus nidulans swoF encodes an N-myristoyl transferase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12457701" target=_blank>Momany M, <i>et al.</i> (2002 Dec)</a>. Polarity in filamentous fungi: establishment, maintenance and new axes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12401496" target=_blank>Sogabe S, <i>et al.</i> (2002 Oct)</a>. Crystal structures of Candida albicans N-myristoyltransferase with two distinct inhibitors.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16110788" target=_blank>Momany M, <i>et al.</i> (2005 May)</a>. Growth control and polarization.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238925" target=_blank>Lee SC, <i>et al.</i> (2007 Feb)</a>. A novel interaction between N-myristoylation and the 26S proteasome during cell morphogenesis.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="../img/localization_hist_graphs/FOG14235_predotar.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG14235_targetp.png" width=190></td>
						<td><img src="../img/localization_hist_graphs/FOG14235_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="../hog_analysis/raw_localization/FOG14235_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="HOG10021_phobius.html#FOG14235">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="../img/trees/HOG10021.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>