<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG11212.html">HOG11212&emsp;&emsp;</a><br> 
																	HOG11214&emsp;&emsp;<br> 
																	<a href="HOG11215.html">HOG11215&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG11214.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG11214_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG11214_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG11214_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG11214_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG15427<br>EOG8BZKN3<br><br>sce:DCP1<br><br>Genes: 25
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Subunit of the Dcp1p-Dcp2p decapping enzyme complex; decapping complex removes the 5' cap structure from mRNAs prior to their degradation; enhances the activity of catalytic subunit Dcp2p; regulated by DEAD box protein Dhh1p; forms cytoplasmic foci upon DNA replication stress<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8757137" target=_blank>Beelman CA, <i>et al.</i> (1996 Aug 15)</a>. An essential component of the decapping enzyme required for normal rates of mRNA turnover.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8816497" target=_blank>Hatfield L, <i>et al.</i> (1996 Oct)</a>. Mutations in trans-acting factors affecting mRNA decapping in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9482746" target=_blank>Anderson JS, <i>et al.</i> (1998 Mar 2)</a>. The 3' to 5' degradation of yeast mRNAs is a general mechanism for mRNA turnover that requires the SKI2 DEVH box protein and 3' to 5' exonucleases of the exosome complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9482745" target=_blank>LaGrandeur TE, <i>et al.</i> (1998 Mar 2)</a>. Isolation and characterization of Dcp1p, the yeast mRNA decapping enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10101156" target=_blank>Tharun S, <i>et al.</i> (1999 Apr)</a>. Analysis of mutations in the yeast mRNA decapping enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10409716" target=_blank>Schwartz DC, <i>et al.</i> (1999 Aug)</a>. Mutations in translation initiation factors lead to increased rates of deadenylation and decapping of mRNAs in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10075882" target=_blank>Zhang S, <i>et al.</i> (1999 Jan)</a>. Monitoring mRNA decapping activity.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564284" target=_blank>Muhlrad D, <i>et al.</i> (1999 Nov)</a>. Recognition of yeast mRNAs as "nonsense containing" leads to both inhibition of mRNA translation and mRNA degradation: implications for the control of mRNA decapping.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10508173" target=_blank>Dunckley T, <i>et al.</i> (1999 Oct 1)</a>. The DCP2 protein is required for mRNA decapping in Saccharomyces cerevisiae and contains a functional MutT motif.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10944120" target=_blank>Vilela C, <i>et al.</i> (2000 Aug 15)</a>. The eukaryotic mRNA decapping protein Dcp1 interacts physically and functionally with the eIF4F translation initiation complex.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10900456" target=_blank>Fromont-Racine M, <i>et al.</i> (2000 Jun 30)</a>. Genome-wide protein interaction screens reveal functional networks involving Sm-like proteins.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10761922" target=_blank>Tharun S, <i>et al.</i> (2000 Mar 30)</a>. Yeast Sm-like proteins function in mRNA decapping and decay.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11780629" target=_blank>Coller JM, <i>et al.</i> (2001 Dec)</a>. The DEAD box helicase, Dhh1p, functions in mRNA decapping and interacts with both the decapping and deadenylase complexes.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139489" target=_blank>Dunckley T, <i>et al.</i> (2001 Jan)</a>. Two related proteins, Edc1p and Edc2p, stimulate mRNA decapping in Saccharomyces cerevisiae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741542" target=_blank>Tharun S, <i>et al.</i> (2001 Nov)</a>. Targeting an mRNA for decapping: displacement of translation factors and association of the Lsm1p-7p complex on deadenylated yeast mRNAs.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12032091" target=_blank>Fischer N, <i>et al.</i> (2002 Jun 3)</a>. The DEAD box protein Dhh1 stimulates the decapping enzyme Dcp1.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12054793" target=_blank>Ramirez CV, <i>et al.</i> (2002 May 10)</a>. Modulation of eukaryotic mRNA stability via the cap-binding translation complex eIF4F.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12554866" target=_blank>Steiger M, <i>et al.</i> (2003 Feb)</a>. Analysis of recombinant yeast decapping enzyme.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730603" target=_blank>Sheth U, <i>et al.</i> (2003 May 2)</a>. Decapping and decay of messenger RNA occur in cytoplasmic processing bodies.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15024087" target=_blank>Holmes LE, <i>et al.</i> (2004 Apr)</a>. Loss of translational control in yeast compromised for the major mRNA decay pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14758354" target=_blank>She M, <i>et al.</i> (2004 Mar)</a>. Crystal structure of Dcp1p and its functional implications in mRNA decapping.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22814378" target=_blank>Van Damme P, <i>et al.</i> (2012 Jul 31)</a>. N-terminal acetylome analyses and functional insights of the N-terminal acetyltransferase NatB.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27694841" target=_blank>Charenton C, <i>et al.</i> (2016 Nov)</a>. Structure of the active form of Dcp1-Dcp2 decapping enzyme bound to m&lt;sup&gt;7&lt;/sup&gt;GDP and its Edc3 activator.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG15427_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG15427_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG15427_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG15427_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG11214_phobius.pdf">Phobius transmembrane predictions</a>
										<br>0 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG11214.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>