<html>




<body>
<style>
#container {width:800px; margin:0 auto; background:#fff}
#header {width:100%; height:150px; border-bottom:1px solid #c7c7c7; background:white}
#logo {float:left; width:35%;height:80px; margin:0px;background:white}
/*#top_info {float:right; width:40px; height:80px; margin:10px; background:#ccc}*/
#top_info {float:right; width:20%;height:80px; margin:0px; background:white;text-align: right}
#navbar {display: inline-block; width:45%; ;height:80px; text-align: left;vertical-align: middle;display: table-cell;font-size: 22;font-family: Helvetica}
/*#navbar {height:20px clear:both;}*/
/*#navbar ul {margin:0; padding:0; list-style-type:none;}*/
/*#navbar ul li {padding:10px; float:left;}*/
/*#navbar ul li a {font-size:12px; float:left; padding: 0 0 0 20px;}*/
#banner { background: #bdbdbd; height:200px; padding:20px; clear:both;}
#left_col {float:left; width:115px; padding:20px;  border:1px solid #333; color:black;}
#right_col {float:left; width:600px; padding:20px;  border:1px solid #333; color:black;}
#footer { padding:20px; clear:both;}

table td,
table th {
	text-align: center;
}





</style>

<div id="container">
	<div id="header">
		<div id="logo">
			<img src="../aybrah_logo_small.png" height="80"></div>
		<div id="top_info">
			<img src="../lmse.jpg" height="80"></div>
		<div id="navbar">
			Analyzing Yeasts by Reconstructing Ancestry of Homologs
		</div>
		<!--<div id="navbar">
			<ul>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
				<li><a href="URL">Link</a></li>
		</div>-->
	</div>
	<div id="content_area">
		<div id="banner" class="row"><a href="../index.html">Home</a><br><br>
																	<a href="HOG12828.html">HOG12828&emsp;&emsp;</a><br> 
																	HOG12832&emsp;&emsp;<br> 
																	<a href="HOG12847.html">HOG12847&emsp;&emsp;</a><br><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/sequences/hog/HOG12832.fasta">FASTA</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/msa/HOG12832_msa.fasta">MAFFT sequence alignment<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/phyml/HOG12832_msa.phy_phyml_boot_trees.txt">Phyml trees<br> 
																	<a href="http://lmse.utoronto.ca/aybrah/trees/by_hog/gb/HOG12832_msa.fasta-gb.htm">Gblocks</a><br> 
																	<a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG12832_phobius.pdf">Phobius predictions</a></div>


		<!--ortholog information:start-->
		<div id="left_col" style="background:#f4cacb" class="col">
			FOG17045<br>EOG86Q57H<br><br>sce:KEX2<br><br>Genes: 36
			<br class="clear" />
		</div>
		<div id="right_col" class="col">
						<br><b>SGD Description</b><br>Kexin, a subtilisin-like protease (proprotein convertase); a calcium-dependent serine protease involved in the activation of proproteins of the secretory pathway<br><br>
			<br><b>PomBase Description</b><br>kexin<br><br>
			<br><b>AspGD Description</b><br>Putative kexin; furin-type endoprotease; endoproteolytic proprotein processing enzyme; processes dibasic cleavage sites in target proteins destined for secretory pathway; required for maturation of glucoamylase<br><br>
			<br><b>References</b><br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3292294" target=_blank>Tanguy-Rougeau C, <i>et al.</i> (1988 Jul 18)</a>. The Kluyveromyces lactis KEX1 gene encodes a subtilisin-type serine proteinase.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845974" target=_blank>Mizuno K, <i>et al.</i> (1988 Oct 14)</a>. Yeast KEX2 genes encodes an endopeptidase homologous to subtilisin-like serine proteases.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2646633" target=_blank>Fuller RS, <i>et al.</i> (1989 Mar)</a>. Yeast prohormone processing enzyme (KEX2 gene product) is a Ca2+-dependent serine protease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1918142" target=_blank>Wilcox CA, <i>et al.</i> (1991 Oct)</a>. Posttranslational processing of the prohormone-cleaving Kex2 protease in the Saccharomyces cerevisiae secretory pathway.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736307" target=_blank>Brenner C, <i>et al.</i> (1992 Feb 1)</a>. Structural and enzymatic characterization of a purified prohormone-processing enzyme: secreted, soluble Kex2 protease.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1544507" target=_blank>Germain D, <i>et al.</i> (1992 Mar 16)</a>. The pro-region of the Kex2 endoprotease of Saccharomyces cerevisiae is removed by self-processing.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8203153" target=_blank>Enderlin CS, <i>et al.</i> (1994 Jan)</a>. Cloning, nucleotide sequence and functions of XPR6, which codes for a dibasic processing endoprotease from the yeast Yarrowia lipolytica.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9360967" target=_blank>Newport G, <i>et al.</i> (1997 Nov 14)</a>. KEX2 influences Candida albicans proteinase secretion and hyphal formation.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11561725" target=_blank>Kwon BK, <i>et al.</i> (2001 Aug 31)</a>. Molecular cloning of kpcA gene encoding a Kex2p-like endoprotease from Aspergillus nidulans.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12779325" target=_blank>Holyoak T, <i>et al.</i> (2003 Jun 10)</a>. 2.4 A resolution crystal structure of the prototypical hormone-processing protease Kex2 in complex with an Ala-Lys-Arg boronic acid inhibitor.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15302836" target=_blank>Mizutani O, <i>et al.</i> (2004 Aug)</a>. Disordered cell integrity signaling caused by disruption of the kexB gene in Aspergillus oryzae.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19129662" target=_blank>Mizutani O, <i>et al.</i> (2009 Jan)</a>. Alternative processing of proproteins in Aspergilli kexB gene disruptants under hyperosmotic conditions.<br><br>
			<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19610202" target=_blank>Harris SD, <i>et al.</i> (2009 Mar)</a>. Morphology and development in Aspergillus nidulans: a complex puzzle.<br><br>
			<center>
			<table width=100%>
				<thead>
					<tr>
						<th colspan=3>Mitochondrial localization predictions</th>
					</tr>
				</theader>
				<tbody>
					<tr>
						<td><a href="https://urgi.versailles.inra.fr/predotar/" target="_blank">Predotar</a></td>
						<td><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">TargetP</a></td>
						<td><a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MitoProt</a></td></tr>
					<tr>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG17045_predotar.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG17045_targetp.png" width=190></td>
						<td><img src="http://lmse.utoronto.ca/aybrah/img/localization_hist_graphs/FOG17045_mitoprot.png" width=190></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/hog_analysis/raw_localization/FOG17045_local_raw.txt">Raw data</a>
							<hr></td>
					</tr>
					<tr>
						<td colspan=3><a href="http://lmse.utoronto.ca/aybrah/img/pdfs/HOG12832_phobius.pdf">Phobius transmembrane predictions</a>
										<br>35 genes with posterior transmembrane prediction > 50%</td>
					</tr>
				</tbody>
			</table>
			<br>
			</center>
			<br class="clear" />
		</div>
		<!--ortholog information:end-->
	<img src="http://lmse.utoronto.ca/aybrah/img/trees/HOG12832.png"
	</div>
	<div id="footer"></div>
</div>

</body></html>